{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "# Intermine-Python: Tutorial 5: Query Results"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This tutorial will talk about dealing with the results of our query. You can either store the results into a file (using a library like csv) or you can process the results immediately after you extract them. "
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We will write a short query and will then explore the Results class of intermine. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "from intermine.webservice import Service"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "service = Service(\"www.flymine.org/flymine/service\")\n",
    "query=service.new_query(\"Gene\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<intermine.query.Query at 0x7f9e100fd080>"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "query.select(\"publications.*\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<TernaryConstraint: Gene LOOKUP zen IN D. melanogaster>"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "query.add_constraint(\"Gene\",\"LOOKUP\",\"zen\",extra_value=\"D. melanogaster\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Once we have added our constraints and views, we are ready to look at the results. The results can be accessed in either a dictionary form, or a list, a ResultRow object(the most common one), or even as a list of strings (CSV or TSV)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Gene: publications.abstractText=\"A 631-bp fragment containing the 5'-flanking region of the Drosophila melanogaster proliferating-cell nuclear antigen (PCNA) gene was placed upstream of the chloramphenicol acetyltransferase (CAT) gene of a CAT vector. A transient expression assay of CAT activity in Drosophila Kc cells transfected with this plasmid and a set of 5'-deletion derivatives revealed that the promoter function resided within a 192-bp region (-168 to +24 with respect to the transcription initiation site). Cotransfection with a zerknüllt (zen)-expressing plasmid specifically repressed CAT expression. However, cotransfection with expression plasmids for a nonfunctional zen mutation, even-skipped, or bicoid showed no significant effect on CAT expression. RNase protection analysis revealed that the repression by zen was at the transcription step. The target sequence of zen was mapped within the 34-bp region (-119 to -86) of the PCNA gene promoter, even though it lacked zen protein-binding sites. Transgenic flies carrying the PCNA gene regulatory region (-607 to +137 or -168 to +137) fused with lacZ were established. When these flies were crossed with the zen mutant, ectopic expression of lacZ was observed in the dorsal region of gastrulating embryos carrying the transgene with either construct. These results indicate that zen indirectly represses PCNA gene expression, probably by regulating the expression of some transcription factor(s) that binds to the PCNA gene promoter.\" publications.doi=None publications.firstAuthor='Yamaguchi M' publications.id=1007881 publications.issue='10' publications.journal='Mol. Cell. Biol.' publications.month='Oct' publications.pages='4909-17' publications.pubMedId='1681423' publications.title='Repression of the Drosophila proliferating-cell nuclear antigen gene promoter by zerknüllt protein.' publications.volume='11' publications.year=1991\n",
      "Gene: publications.abstractText=\"A DNA replication-related element (DRE)-binding factor (DREF) has been revealed to be an important transcription factor for activating promoters of cell proliferation and differentiation related genes. The amino acid sequences of DREF are conserved in evolutionary separate Drosophila species, Drosophila melanogaster (Dm) and Drosophila virilis (Dv) in three regions. In the present study, evidence was obtained that there are several highly conserved regions in the 5' flanking region between the DmDREF and DvDREF genes. Band mobility shift assays using oligonucleotides corresponding to these conserved regions revealed that specific trans-acting factors can bind to at least three regions -554 to -543 (5'-TTTGTTCTTGCG), -81 to -70 (5'-GCCCACGTGGCT) and +225 to +234 (5'-GCAATCAGTG). Using a transient luciferase expression assay, we demonstrated that the region -554 to -543 functions as a negative regulatory element for DmDREF promoter activity, while the regions -77 to -70 (5'-ACGTGGCT) and +225 to +236 (5'-GCAATCAGTGTT) function as positive regulatory elements. In previous studies, we observed that expression of the homeodomain protein Zerknüllt (Zen) represses PCNA gene transcription, by reducing the DNA binding activity of DREF. Here we show Zen downregulates DREF gene promoter activity through action on the region between +241 and +254 (5'-AGAATACTCAACA). In addition, the DmDREF promoter contains five DREs. Using a double stranded RNA-mediated interference method, we generated evidence that expression of DmDREF could be auto-regulated by DREF through the third DRE located at +211 to +218. In living flies we obtained results consistent with those obtained in vitro and in cultured cells. The study thus indicates that DmDREF is effectively regulated via highly conserved regions between the DmDREF and DvDREF promoters, suggesting the existence of common regulatory factors, and that DmDREF can be positively regulated by itself via the third DRE located in its most highly conserved region.\" publications.doi=None publications.firstAuthor='Kwon Eunjeong' publications.id=1007877 publications.issue='2' publications.journal='Gene' publications.month='May' publications.pages='101-16' publications.pubMedId='12758126' publications.title='Transcription control of a gene for Drosophila transcription factor, DREF by DRE and cis-elements conserved between Drosophila melanogaster and virilis.' publications.volume='309' publications.year=2003\n",
      "Gene: publications.abstractText='A critical step in understanding the mechanisms of development is in defining the steps at the molecular, cellular, and organismal levels in the developmental program for a given organism-so that given the egg one can predict not only how the embryo will develop but also how that embryo evolved from its ancestors. Using methods employed by chemists and engineers in modeling hierarchical systems, I have integrated current theory and experiment into a calculational method that can model early Drosophila embryogenesis on a personal computer. This quantitative calculation tool is simple enough to be useful for experimentalists in designing experiments yet detailed enough for theoreticians to derive new insights on the evolution of developmental genetic networks. By integrating the strengths of theoretical and experimental methods into a single engineering model that can compute the cascade of genetic networks in a real organism, I provide a new calculational tool that can apply current theory to current experimental data to study the evolution of developmental programs.' publications.doi='10.1006/jtbi.1996.0328' publications.firstAuthor='Bodnar J W' publications.id=1007754 publications.issue='4' publications.journal='J. Theor. Biol.' publications.month='Oct' publications.pages='391-445' publications.pubMedId='9367733' publications.title='Programming the Drosophila embryo.' publications.volume='188' publications.year=1997\n",
      "Gene: publications.abstractText='A gradient of the maternal morphogen dorsal (dl) establishes dorsal-ventral (D-V) polarity in the early Drosophila embryo. The dl concentration gradient is initiated by regulated nuclear transport, and only protein that enters nuclei is active in the D-V patterning process. Here we show that dl is a DNA-binding protein that specifically interacts with distal sequences of the zerknüllt (zen) promoter, one of the genetic targets of the morphogen. These zen sequences have the properties of a silencer element and can act over long distances to repress the expression of a heterologous promoter. The dl protein recognizes a sequence motif similar to that of the mammalian transcriptional activator NF-kappa B, which was shown to contain extensive homology with dl and the oncoprotein rel. We present evidence that the DNA-binding activity of the dl protein is mediated by the region of homology (the rel domain) conserved in the rel and NF-kappa B proteins.' publications.doi=None publications.firstAuthor='Ip Y T' publications.id=1007836 publications.issue='2' publications.journal='Cell' publications.month='Jan' publications.pages='439-46' publications.pubMedId='1988156' publications.title='The dorsal morphogen is a sequence-specific DNA-binding protein that interacts with a long-range repression element in Drosophila.' publications.volume='64' publications.year=1991\n",
      "Gene: publications.abstractText='A gradient of the maternal morphogen dorsal (dl) initiates the differentiation of various tissues along the dorsal-ventral axis of early Drosophila embryos. dl is a sequence-specific DNA-binding protein that is related to the mammalian regulatory factor NF-kappa B. Previous studies suggest that dl can function as a transcriptional repressor. To determine how dl functions as an activator we have examined the promoter of the mesoderm determinant gene twist (twi). Genetic studies suggest that peak levels of dl protein in ventral regions of early embryos initiate twi expression. Using a combination of promoter fusion-P-transformation assays, and in vitro DNA-binding assays coupled with site-directed mutagenesis, we establish a direct link between dl-binding sites and twi expression in the early embryo. We also present evidence that the dorsal-ventral limits of twi expression depend on the number and affinity of dl-binding sites present in its promoter. A comparison of twi with a second dl target gene, zen, suggests a correlation between the affinities of dl-binding sites and response to different thresholds of dl morphogen.' publications.doi=None publications.firstAuthor='Jiang J' publications.id=1007687 publications.issue='10' publications.journal='Genes Dev.' publications.month='Oct' publications.pages='1881-91' publications.pubMedId='1655572' publications.title='The dorsal morphogen gradient regulates the mesoderm determinant twist in early Drosophila embryos.' publications.volume='5' publications.year=1991\n",
      "Gene: publications.abstractText=\"A member of a small group of genes in Drosophila that define the segmentation pattern of the early embryo even-skipped (eve), which plays a key role in a network of interactions among segmentation genes. It appears to control morphogenesis by regulating the expression of the segmentation gene engrailed (en), and by autoregulating its own expression (M. Frasch and M.L., in preparation). Here we show that these regulatory interactions could occur at the level of transcription as a full-length eve protein binds with high affinity to specific sequences located near the 5' ends of the eve and en genes. The en binding sites contain at least one copy of a 10-base pair consensus sequence: T-C-A-A-T-T-A-A-A-T. In contrast, the 5' eve binding sites are relatively G-C rich and do not share obvious similarities with the 10-base pair consensus sequence associated with en. Other homeo box proteins can recognize both classes of eve binding sites, lending support to the proposal that regulatory interactions among homeo box genes involve a competition of different homeo box proteins for similar cis regulatory sequences.\" publications.doi='10.1038/332858a0' publications.firstAuthor='Hoey T' publications.id=1007852 publications.issue='6167' publications.journal='Nature' publications.month='Apr' publications.pages='858-61' publications.pubMedId='2895896' publications.title='Divergent homeo box proteins recognize similar DNA sequences in Drosophila.' publications.volume='332' publications.year=1988\n",
      "Gene: publications.abstractText='A morphogen gradient is an important concept in developmental biology, because it describes a mechanism by which the emission of a signal from one part of an embryo can determine the location, differentiation and fate of many surrounding cells. The value of this idea has been clear for over half a century, but only recently have experimental systems and methods of analysis progressed to the point where we begin to understand how a cell can sense and respond to tiny changes in minute concentrations of extracellular signalling factors.' publications.doi='10.1038/35101500' publications.firstAuthor='Gurdon J B' publications.id=1004012 publications.issue='6858' publications.journal='Nature' publications.month='Oct' publications.pages='797-803' publications.pubMedId='11677596' publications.title='Morphogen gradient interpretation.' publications.volume='413' publications.year=2001\n",
      "Gene: publications.abstractText='A recent study reveals specific functions of Hox3/zerknüllt (zen) in the extraembryonic and embryonic primordia of the red flour beetle, Tribolium castaneum. The results shed light on the functional evolution of bicoid, a zen paralogue of higher flies, which determines anterior body parts.' publications.doi='10.1016/j.cub.2005.03.022' publications.firstAuthor='Schmidt-Ott Urs' publications.id=1007753 publications.issue='7' publications.journal='Curr. Biol.' publications.month='Apr' publications.pages='R245-7' publications.pubMedId='15823522' publications.title='Insect serosa: a head line in comparative developmental genetics.' publications.volume='15' publications.year=2005\n",
      "Gene: publications.abstractText=\"Although all bilaterian animals have a related set of Hox genes, the genomic organization of this gene complement comes in different flavors. In some unrelated species, Hox genes are clustered; in others, they are not. This indicates that the bilaterian ancestor had a clustered Hox gene family and that, subsequently, this genomic organization was either maintained or lost. Remarkably, the tightest organization is found in vertebrates, raising the embarrassingly finalistic possibility that vertebrates have maintained best this ancestral configuration. Alternatively, could they have co-evolved with an increased ;organization' of the Hox clusters, possibly linked to their genomic amplification, which would be at odds with our current perception of evolutionary mechanisms? When discussing the why's and how's of Hox gene clustering, we need to account for three points: the mechanisms of cluster evolution; the underlying biological constraints; and the developmental modes of the animals under consideration. By integrating these parameters, general conclusions emerge that can help solve the aforementioned dilemma.\" publications.doi='10.1242/dev.001065' publications.firstAuthor='Duboule Denis' publications.id=1007895 publications.issue='14' publications.journal='Development' publications.month='Jul' publications.pages='2549-60' publications.pubMedId='17553908' publications.title='The rise and fall of Hox gene clusters.' publications.volume='134' publications.year=2007\n",
      "Gene: publications.abstractText='Although genes involved in common developmental programs are usually scattered throughout the metazoan genome, there are some important examples of functionally interconnected regulatory genes that display close physical linkage. In particular the homeotic genes, which determine the identities of body parts, are clustered in the Hox complexes and clustering is thought to be crucial for the proper execution of their developmental programs. Here we describe the organization and functional properties of a more recently identified cluster of six homeobox genes at 93DE on the third chromosome of Drosophila. These genes, which include tinman, bagpipe, ladybird early, ladybird late, C15, and slouch, all participate in mesodermal patterning and differentiation programs and show multiple regulatory interactions among each other. We propose that their clustering, through unknown mechanisms, is functionally significant and discuss the similarities and differences between the 93DE homeobox gene cluster and the Hox complexes.' publications.doi='10.1002/1521-1878(200102)23:2<125::AID-BIES1019>3.0.CO;2-C' publications.firstAuthor='Jagla K' publications.id=1007883 publications.issue='2' publications.journal='Bioessays' publications.month='Feb' publications.pages='125-33' publications.pubMedId='11169585' publications.title='A cluster of Drosophila homeobox genes involved in mesoderm differentiation programs.' publications.volume='23' publications.year=2001\n",
      "Gene: publications.abstractText='Although the genetics of dorsal-ventral polarity which leads to mesoderm formation in Drosophila are understood in considerable detail, subsequent molecular mechanisms involved in patterning the mesoderm primordium into individual mesodermal subtypes are poorly understood. Two papers published recently suggest strongly that an inductive signal from dorsal ectoderm is involved in subdividing the underlying mesoderm, and present evidence that one of the signalling factors is Decapentaplegic (Dpp), a member of the bone morphogenetic protein subgroup of the Transforming Growth Factor-beta (TGF-beta) super family of proteins.' publications.doi='10.1002/bies.950170904' publications.firstAuthor='Venkatesh T V' publications.id=1007725 publications.issue='9' publications.journal='Bioessays' publications.month='Sep' publications.pages='754-7' publications.pubMedId='8763827' publications.title='How many signals does it take?' publications.volume='17' publications.year=1995\n",
      "Gene: publications.abstractText='Although the mechanisms of anterior-posterior axis formation are well understood in Drosophila, both embryological and molecular studies suggest significant variation in the mechanisms generating this axis within the Insecta class as a whole.' publications.doi=None publications.firstAuthor='Patel N H' publications.id=1007793 publications.issue='9' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Apr' publications.pages='4442-4' publications.pubMedId='10781040' publications.title=\"It's a bug's life.\" publications.volume='97' publications.year=2000\n",
      "Gene: publications.abstractText='Amino acid encoding genes contain character state information that may be useful for phylogenetic analysis on at least two levels. The nucleotide sequence and the translated amino acid sequences have both been employed separately as character states for cladistic studies of various taxa, including studies of the genealogy of genes in multigene families. In essence, amino acid sequences and nucleic acid sequences are two different ways of character coding the information in a gene. Silent positions in the nucleotide sequence (first or third positions in codons that can accrue change without changing the identity of the amino acid that the triplet codes for) may accrue change relatively rapidly and become saturated, losing the pattern of historical divergence. On the other hand, non-silent nucleotide alterations and their accompanying amino acid changes may evolve too slowly to reveal relationships among closely related taxa. In general, the dynamics of sequence change in silent and non-silent positions in protein coding genes result in homoplasy and lack of resolution, respectively. We suggest that the combination of nucleic acid and the translated amino acid coded character states into the same data matrix for phylogenetic analysis addresses some of the problems caused by the rapid change of silent nucleotide positions and overall slow rate of change of non-silent nucleotide positions and slowly changing amino acid positions. One major theoretical problem with this approach is the apparent non-independence of the two sources of characters. However, there are at least three possible outcomes when comparing protein coding nucleic acid sequences with their translated amino acids in a phylogenetic context on a codon by codon basis. First, the two character sets for a codon may be entirely congruent with respect to the information they convey about the relationships of a certain set of taxa. Second, one character set may display no information concerning a phylogenetic hypothesis while the other character set may impact information to a hypothesis. These two possibilities are cases of non-independence, however, we argue that congruence in such cases can be thought of as increasing the weight of the particular phylogenetic hypothesis that is supported by those characters. In the third case, the two sources of character information for a particular codon may be entirely incongruent with respect to phylogenetic hypotheses concerning the taxa examined. In this last case the two character sets are independent in that information from neither can predict the character states of the other. Examples of these possibilities are discussed and the general applicability of combining these two sources of information for protein coding genes is presented using sequences from the homeobox region of 46 homeobox genes from Drosophila melanogaster to develop a hypothesis of genealogical relationship of these genes in this large multigene family.' publications.doi=None publications.firstAuthor='Agosti D' publications.id=1007873 publications.issue=None publications.journal='Cladistics' publications.month=None publications.pages='65-82' publications.pubMedId='11541749' publications.title='On combining protein sequences and nucleic acid sequences in phylogenetic analysis: the homeobox protein case.' publications.volume='12' publications.year=1996\n",
      "Gene: publications.abstractText='Among the maternally active genes of Drosophila, cactus is the only one whose loss of function mutations specifically produce ventralized embryos. Its product inhibits nuclear translocation of the dorsal morphogen in the dorsal region of the embryo. Here we report the cloning of cactus and the sequencing of its maternal transcript. The identity of our clones was verified by induction of phenocopies with antisense RNA and rescue of the mutant phenotype with sense RNA. cactus is predicted to encode an acidic, cytoplasmic protein with seven ankyrin repeats. The sequence has similarity to the I kappa B proteins that inhibit the vertebrate transcription factor NF-kappa B. In analogy to results obtained with I kappa B and NF-kappa B, bacterially expressed cactus protein can inhibit DNA binding of dorsal protein in vitro.' publications.doi=None publications.firstAuthor='Geisler R' publications.id=1007832 publications.issue='4' publications.journal='Cell' publications.month='Nov' publications.pages='613-21' publications.pubMedId='1423618' publications.title='cactus, a gene involved in dorsoventral pattern formation of Drosophila, is related to the I kappa B gene family of vertebrates.' publications.volume='71' publications.year=1992\n",
      "Gene: publications.abstractText='An extracellular serine protease cascade generates the ligand that activates the Toll signaling pathway to establish dorsoventral polarity in the Drosophila embryo. We show here that this cascade is regulated by a serpin-type serine protease inhibitor, which plays an essential role in confining Toll signaling to the ventral side of the embryo. This role is strikingly analogous to the function of the mammalian serpin antithrombin in localizing the blood-clotting cascade, suggesting that serpin inhibition of protease activity may be a general mechanism for achieving spatial control in diverse biological processes.' publications.doi=None publications.firstAuthor='Hashimoto Carl' publications.id=1007691 publications.issue='6' publications.journal='Dev. Cell' publications.month='Dec' publications.pages='945-50' publications.pubMedId='14667416' publications.title='Spatial regulation of developmental signaling by a serpin.' publications.volume='5' publications.year=2003\n",
      "Gene: publications.abstractText='An old idea about the relationship between arthropod and vertebrate body plans has been given new life by studies of the signalling genes controlling dorsal and ventral development in Drosophila and Xenopus.' publications.doi=None publications.firstAuthor='Jones C M' publications.id=1007783 publications.issue='6' publications.journal='Curr. Biol.' publications.month='Jun' publications.pages='574-6' publications.pubMedId='7552158' publications.title='Inductive signals. Revolving vertebrates.' publications.volume='5' publications.year=1995\n",
      "Gene: publications.abstractText='An oligonucleotide based on the cdc 10/SWI6 repeats of the Drosophila Notch protein was used to isolate other Drosophila genes with these repeats. One of these genes is the cactus locus, 1 of 11 genes required maternally for the establishment in embryos of a gradient of nuclear localization of dorsal protein, a rel-like transcription factor. Previous work showed that in cactus mutants more dorsal protein enters the nucleus in dorsal regions, resulting in a ventralized phenotype. It is now shown that the cactus locus produces two proteins that differ at their carboxy termini; both contain six cdc 10/SWI6 repeats that are sufficient for binding to dorsal and for inhibiting the ability of dorsal to bind DNA. The site on dorsal to which cactus binds was localized to the rel homology domain, where it overlaps with, or is adjacent to, the nuclear localization signal. In vivo the bulk of the cactus protein associated with dorsal is phosphorylated. This, or the association with dorsal, appears to stabilize the maternal cactus protein.' publications.doi=None publications.firstAuthor='Kidd S' publications.id=1007831 publications.issue='4' publications.journal='Cell' publications.month='Nov' publications.pages='623-35' publications.pubMedId='1423619' publications.title='Characterization of the Drosophila cactus locus and analysis of interactions between cactus and dorsal proteins.' publications.volume='71' publications.year=1992\n",
      "Gene: publications.abstractText='At least 13 genes control the establishment of dorsoventral polarity in the Drosophila embryo and more than 30 genes control the anteroposterior pattern of body segments. Each group of genes is thought to control pattern formation along one body axis, independently of the other group. We have used the expression of the fushi tarazu (ftz) segmentation gene as a positional marker to investigate the relationship between the dorsoventral and anteroposterior axes. The ftz gene is normally expressed in seven transverse stripes. Changes in the striped pattern in embryos mutant for other genes (or progeny of females homozygous for maternal-effect mutations) can reveal alterations of cell fate resulting from such mutations. We show that in the absence of any of ten maternal-effect dorsoventral polarity gene functions, the characteristic stripes of ftz protein are altered. Normally there is a difference between ftz stripe spacing on the dorsal and ventral sides of the embryo; in dorsalized mutant embryos the ftz stripes appear to be altered so that dorsal-type spacing occurs on all sides of the embryo. These results indicate that cells respond to dorsoventral positional information in establishing early patterns of gene expression along the anteroposterior axis and that there may be more significant interactions between the different axes of positional information than previously determined.' publications.doi=None publications.firstAuthor='Carroll S B' publications.id=1002632 publications.issue='3' publications.journal='Development' publications.month='Mar' publications.pages='327-32' publications.pubMedId='3653004' publications.title='Genes that control dorsoventral polarity affect gene expression along the anteroposterior axis of the Drosophila embryo.' publications.volume='99' publications.year=1987\n",
      "Gene: publications.abstractText='Axis formation is a key step in development, but studies indicate that genes involved in insect axis formation are relatively fast evolving. Orthodenticle genes have conserved roles, often with hunchback, in maternal anterior patterning in several insect species. We show that two orthodenticle genes, otd1 and otd2, and hunchback act as maternal anterior patterning genes in the honeybee (Apis mellifera) but, unlike other insects, act to pattern the majority of the anteroposterior axis. These genes regulate the expression domains of anterior, central and posterior gap genes and may directly regulate the anterior gap gene giant. We show otd1 and hunchback also influence dorsoventral patterning by regulating zerknült (zen) as they do in Tribolium, but that zen does not regulate the expression of honeybee gap genes. This suggests that interactions between anteroposterior and dorsal-ventral patterning are ancestral in holometabolous insects. Honeybee axis formation, and the function of the conserved anterior patterning gene orthodenticle, displays unique characters that indicate that, even when conserved genes pattern the axis, their regulatory interactions differ within orders of insects, consistent with relatively fast evolution in axis formation pathways.' publications.doi='10.1242/dev.067926' publications.firstAuthor='Wilson Megan J' publications.id=1007771 publications.issue='16' publications.journal='Development' publications.month='Aug' publications.pages='3497-507' publications.pubMedId='21771808' publications.title='Diversity in insect axis formation: two orthodenticle genes and hunchback act in anterior patterning and influence dorsoventral organization in the honeybee (Apis mellifera).' publications.volume='138' publications.year=2011\n",
      "Gene: publications.abstractText='Bioinformatics methods have identified enhancers that mediate restricted expression in the Drosophila embryo. However, only a small fraction of the predicted enhancers actually work when tested in vivo. In the present study, co-regulated neurogenic enhancers that are activated by intermediate levels of the Dorsal regulatory gradient are shown to contain several shared sequence motifs. These motifs permitted the identification of new neurogenic enhancers with high precision: five out of seven predicted enhancers direct restricted expression within ventral regions of the neurogenic ectoderm. Mutations in some of the shared motifs disrupt enhancer function, and evidence is presented that the Twist and Su(H) regulatory proteins are essential for the specification of the ventral neurogenic ectoderm prior to gastrulation. The regulatory model of neurogenic gene expression defined in this study permitted the identification of a neurogenic enhancer in the distant Anopheles genome. We discuss the prospects for deciphering regulatory codes that link primary DNA sequence information with predicted patterns of gene expression.' publications.doi='10.1242/dev.01124' publications.firstAuthor='Markstein Michele' publications.id=1430828 publications.issue='10' publications.journal='Development' publications.month='May' publications.pages='2387-94' publications.pubMedId='15128669' publications.title='A regulatory code for neurogenic gene expression in the Drosophila embryo.' publications.volume='131' publications.year=2004\n",
      "Gene: publications.abstractText='Bone morphogenetic protein (BMP) signalling regulates embryonic dorsal-ventral cell fate decisions in flies, frogs and fish. BMP activity is controlled by several secreted factors including the antagonists chordin and short gastrulation (SOG). Here we show that a second secreted protein, Twisted gastrulation (Tsg), enhances the antagonistic activity of Sog/chordin. In Drosophila, visualization of BMP signalling using anti-phospho-Smad staining shows that the tsg and sog loss-of-function phenotypes are very similar. In S2 cells and imaginal discs, TSG and SOG together make a more effective inhibitor of BMP signalling than either of them alone. Blocking Tsg function in zebrafish with morpholino oligonucleotides causes ventralization similar to that produced by chordin mutants. Co-injection of sub-inhibitory levels of morpholines directed against both Tsg and chordin synergistically enhances the penetrance of the ventralized phenotype. We show that Tsgs from different species are functionally equivalent, and conclude that Tsg is a conserved protein that functions with SOG/chordin to antagonize BMP signalling.' publications.doi='10.1038/35068578' publications.firstAuthor='Ross J J' publications.id=1007781 publications.issue='6827' publications.journal='Nature' publications.month='Mar' publications.pages='479-83' publications.pubMedId='11260716' publications.title='Twisted gastrulation is a conserved extracellular BMP antagonist.' publications.volume='410' publications.year=2001\n",
      "Gene: publications.abstractText='Bone morphogenetic proteins (BMP) direct dorsal-ventral patterning in both invertebrate and vertebrate embryos, with strong evolutionary conservation of molecular components of the pathway. Dorsal-ventral patterning of the early Drosophila embryo is a powerful experimental system to probe mechanisms of BMP gradient formation and interpretation. Recent studies have found that spatial patterns of activated BMP signal transducers in Drosophila go through an unexpected transition: a shallow gradient of weak responses at mid-cellularization changes to a step gradient of stronger responses in cellularized embryos. The transition between two gradients of different shape yields new insights into the progression of Drosophila dorsal-ventral patterning and raises new issues about the mechanisms ofgradient formation.' publications.doi='10.1016/j.tig.2003.10.009' publications.firstAuthor='Raftery Laurel A' publications.id=1007688 publications.issue='12' publications.journal='Trends Genet.' publications.month='Dec' publications.pages='701-8' publications.pubMedId='14642751' publications.title='Gradients and thresholds: BMP response gradients unveiled in Drosophila embryos.' publications.volume='19' publications.year=2003\n",
      "Gene: publications.abstractText=\"Brinker (Brk), a novel protein with features of a transcriptional repressor, regulates the graded response to Decapentaplegic (Dpp) in appendage primordia of Drosophila. Here, we show that in the embryo brk also has differential effects on Dpp target genes, depending on the level of Dpp activity required for their activation. Low-level target genes, like dpp itself, tolloid and early zerknüllt, show strong ectopic expression in ventrolateral regions of brk mutant embryos; intermediate-level target genes like pannier show weak ectopic expression, while high-level target genes like u-shaped and rhomboid are not affected. Ectopic target gene activation in the absence of brk is independent of Dpp, Tkv and Medea, indicating that Dpp signaling normally antagonizes brk's repression of these target genes. brk is expressed like short gastrulation (sog) in ventrolateral regions of the embryo abutting the dpp domain. Here, both brk and sog antagonize the antineurogenic activity of Dpp so that only in brk sog double mutants is the neuroectoderm completely deleted.\" publications.doi=None publications.firstAuthor='Jaźwińska A' publications.id=1007741 publications.issue='15' publications.journal='Development' publications.month='Aug' publications.pages='3323-34' publications.pubMedId='10393112' publications.title='The role of brinker in mediating the graded response to Dpp in early Drosophila embryos.' publications.volume='126' publications.year=1999\n",
      "Gene: publications.abstractText='By using a hsp70-Ubx fusion gene, we have ectopically expressed a Ubx product in the embryonic head primordia and studied the developmental effects on the larval head. We find that after high and persistent levels of Ubx product, the head is replaced by three (C1, C2 and C3) abdominal-like denticle belts. The C2 and C3 belts are the homeotic transformations of parasegments 1 and 2, respectively, while the C1 belt probably derives from the transformation and subsequent fusion of the most anterior procephalic primordia. On the basis of their response to the Ubx product and other arguments, we propose that the larval head is made of two genetically distinct components; one is the procephalon and the anterior region of the mandibular lobe, and the other is part of the parasegmental trunk and includes parasegments 1 and 2. Our results also indicate that most or all the larval head structures derive from precursor cells of ventral origin.' publications.doi=None publications.firstAuthor='González-Reyes A' publications.id=1007689 publications.issue='4' publications.journal='Development' publications.month='Dec' publications.pages='1459-71' publications.pubMedId='1687461' publications.title='Organization of the Drosophila head as revealed by the ectopic expression of the Ultrabithorax product.' publications.volume='113' publications.year=1991\n",
      "Gene: publications.abstractText='CBP and the related p300 protein are widely used transcriptional co-activators in metazoans that interact with multiple transcription factors. Whether CBP/p300 occupies the genome equally with all factors or preferentially binds together with some factors is not known. We therefore compared Drosophila melanogaster CBP (nejire) ChIP-seq peaks with regions bound by 40 different transcription factors in early embryos, and we found high co-occupancy with the Rel-family protein Dorsal. Dorsal is required for CBP occupancy in the embryo, but only at regions where few other factors are present. CBP peaks in mutant embryos lacking nuclear Dorsal are best correlated with TGF-ß/Dpp-signaling and Smad-protein binding. Differences in CBP occupancy in mutant embryos reflect gene expression changes genome-wide, but CBP also occupies some non-expressed genes. The presence of CBP at silent genes does not result in histone acetylation. We find that Polycomb-repressed H3K27me3 chromatin does not preclude CBP binding, but restricts histone acetylation at CBP-bound genomic sites. We conclude that CBP occupancy in Drosophila embryos preferentially overlaps factors controlling dorso-ventral patterning and that CBP binds silent genes without causing histone hyperacetylation.' publications.doi='10.1371/journal.pgen.1002769' publications.firstAuthor='Holmqvist Per-Henrik' publications.id=1007693 publications.issue='6' publications.journal='PLoS Genet.' publications.month=None publications.pages='e1002769' publications.pubMedId='22737084' publications.title='Preferential genome targeting of the CBP co-activator by Rel and Smad proteins in early Drosophila melanogaster embryos.' publications.volume='8' publications.year=2012\n",
      "Gene: publications.abstractText='CDP/Cux/Cut proteins are an evolutionarily conserved family of proteins containing several DNA binding domains: one Cut homeodomain and one, two or three Cut repeats. In Drosophila melanogaster, genetic studies indicated that Cut functions as a determinant of cell-type specification in several tissues, notably in the peripheral nervous system, the wing margin and the Malpighian tubule. Moreover, Cut was found to be a target and an effector of the Notch signaling pathway. In vertebrates, the same functions appear to be fulfilled by two cut-related genes with distinct patterns of expression. Cloning of the cDNA for the CCAAT-displacement protein (CDP) revealed that it was the human homologue of Drosophila Cut. CDP was later found be the DNA binding protein of the previously characterized histone nuclear factor D (HiNF-D). CDP and its mouse counterpart, Cux, were also reported to interact with regulatory elements from a large number of genes, including matrix attachment regions (MARs). CDP/Cut proteins were found generally to function as transcriptional repressors, although a participation in transcriptional activation is suggested by some data. Repression by CDP/Cut involves competition for binding site occupancy and active repression via the recruitment of a histone deacetylase activity. Various combinations of Cut repeats and the Cut homeodomains can generate distinct DNA binding activities. These activities are elevated in proliferating cells and decrease during terminal differentiation. One activity, involving the Cut homeodomain, is upregulated in S phase. CDP/Cut function is regulated by several post-translational modification events including phosphorylation, dephosphorylation, and acetylation. The CUTL1 gene in human was mapped to 7q22, a chromosomal region that is frequently rearranged in various cancers.' publications.doi=None publications.firstAuthor='Nepveu A' publications.id=1000618 publications.issue='1-2' publications.journal='Gene' publications.month='May' publications.pages='1-15' publications.pubMedId='11403998' publications.title='Role of the multifunctional CDP/Cut/Cux homeodomain transcription factor in regulating differentiation, cell growth and development.' publications.volume='270' publications.year=2001\n",
      "Gene: publications.abstractText='Cascades of zygotic gene expression pattern the anterior-posterior (AP) and dorsal-ventral (DV) axes of the early Drosophila embryo. Here, we used the global run-on sequencing assay (GRO-seq) to map the genome-wide RNA polymerase distribution during early Drosophila embryogenesis, thus providing insights into how genes are regulated. We identify widespread promoter-proximal pausing yet show that the presence of paused polymerase does not necessarily equate to direct regulation through pause release to productive elongation. Our data reveal that a subset of early Zelda-activated genes is regulated at the level of polymerase recruitment, whereas other Zelda target and axis patterning genes are predominantly regulated through pause release. In contrast to other signaling pathways, we found that bone morphogenetic protein (BMP) target genes are collectively more highly paused than BMP pathway components and show that BMP target gene expression requires the pause-inducing negative elongation factor (NELF) complex. Our data also suggest that polymerase pausing allows plasticity in gene activation throughout embryogenesis, as transiently repressed and transcriptionally silenced genes maintain and lose promoter polymerases, respectively. Finally, we provide evidence that the major effect of pausing is on the levels, rather than timing, of transcription. These data are discussed in terms of the efficiency of transcriptional activation required across cell populations during developmental time constraints.' publications.doi='10.1101/gad.215459.113' publications.firstAuthor='Saunders Abbie' publications.id=1001000 publications.issue='10' publications.journal='Genes Dev.' publications.month='May' publications.pages='1146-58' publications.pubMedId='23699410' publications.title='Extensive polymerase pausing during Drosophila axis patterning enables high-level and pliable transcription.' publications.volume='27' publications.year=2013\n",
      "Gene: publications.abstractText='Cell fates in the anterior and posterior termini of the Drosophila embryo are programmed by multiple zygotic genes that are regulated in response to a maternally encoded signal transduction pathway. These genes specify terminal as distinct from central cell fates, program pattern along the anteroposterior and dorsoventral axes of the termini, and also control endoderm specification and terminal morphogenetic movements. Here, we use a genetic interaction test to dissect the zygotic components of the terminal genetic hierarchy. We show that two genes, lines and empty spiracles, act downstream of tailless to repress central and promote terminal cell fates along the anteroposterior axis of the termini. Genes that control dorsoventral pattern in the termini and genes that program terminal morphogenesis act in distinct branches of the genetic hierarchy that are independent of tailless.' publications.doi=None publications.firstAuthor='Strecker T R' publications.id=1007685 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Apr' publications.pages='422-6' publications.pubMedId='1551484' publications.title='Genetic control of cell fate in the termini of the Drosophila embryo.' publications.volume='150' publications.year=1992\n",
      "Gene: publications.abstractText='Charles Darwin has proposed the theory that evolution of live organisms is based on random variation and natural selection. Jacques Monod in his classic book Chance and Necessity, published 40 years ago, presented his thesis \"that the biosphere does not contain a predictable class of objects or events, but constitutes a particular occurrence, compatible indeed with the first principles, but not deducible from those principals and therefore, essentially unpredictable.\"Recent discoveries in eye evolution are in agreement with both of these theses. They confirm Darwin\\'s assumption of a simple eye prototype and lend strong support for the notion of a monophyletic origin of the various eye types. Considering the complexity of the underlying gene regulatory networks the unpredictability is obvious. The evolution of the Hox gene cluster and the specification of the body plan starting from an evolutionary prototype segment is discussed. In the course of evolution, a series of similar prototypic segments gradually undergoes cephalization anteriorly and caudalization posteriorly through diversification of the Hox genes.' publications.doi='10.1093/gbe/evr061' publications.firstAuthor='Gehring Walter J' publications.id=1007863 publications.issue=None publications.journal='Genome Biol Evol' publications.month=None publications.pages='1053-66' publications.pubMedId='21979158' publications.title='Chance and necessity in eye evolution.' publications.volume='3' publications.year=2011\n",
      "Gene: publications.abstractText='Chromatin domain insulators are thought to insulate adjacent genes, including their regulatory elements, from each other by organizing chromatin into functionally independent domains. Thus insulators should play a global role in gene regulation by keeping regulatory domains separated. However, this has never been demonstrated. We previously designed and characterized a transgene that is under GAL4 UAS control and encodes a dominant-negative form of the Boundary Element-Associated Factors BEAF-32A and BEAF-32B. The BID transgene encodes the BEAF self-interaction domain but lacks a DNA binding domain. Expression of BID in eye imaginal discs leads to a rough eye phenotype. Here we screen for dominant mutations that modify this eye phenotype. This assay provides evidence for cross-talk between different classes of insulators, and for a broad role of the BEAF proteins in maintaining patterns of gene expression during eye development. Most identified genes encode other insulator binding proteins, transcription factors involved in head development, or general transcription factors. Because it is unlikely that insulator function is limited to eye development, the present results support the hypothesis that insulators play a widespread role in maintaining global transcription programs.' publications.doi='10.1007/s00438-006-0187-8' publications.firstAuthor='Roy Swarnava' publications.id=1005516 publications.issue='3' publications.journal='Mol. Genet. Genomics' publications.month='Mar' publications.pages='273-86' publications.pubMedId='17143631' publications.title='A genetic screen supports a broad role for the Drosophila insulator proteins BEAF-32A and BEAF-32B in maintaining patterns of gene expression.' publications.volume='277' publications.year=2007\n",
      "Gene: publications.abstractText=\"Circulating carbohydrates are an essential energy source, perturbations in which are pathognomonic of various diseases, diabetes being the most prevalent. Yet many of the genes underlying diabetes and its characteristic hyperglycaemia remain elusive. Here we use physiological and genetic interrogations in D. melanogaster to uncover the 'glucome', the complete set of genes involved in glucose regulation in flies. Partial genomic screens of ∼1,000 genes yield ∼160 hyperglycaemia 'flyabetes' candidates that we classify using fat body- and muscle-specific knockdown and biochemical assays. The results highlight the minor glucose fraction as a physiological indicator of metabolism in Drosophila. The hits uncovered in our screen may have conserved functions in mammalian glucose homeostasis, as heterozygous and homozygous mutants of Ck1alpha in the murine adipose lineage, develop diabetes. Our findings demonstrate that glucose has a role in fly biology and that genetic screenings carried out in flies may increase our understanding of mammalian pathophysiology.\" publications.doi='10.1038/ncomms8102' publications.firstAuthor='Ugrankar Rupali' publications.id=1001814 publications.issue=None publications.journal='Nat Commun' publications.month=None publications.pages='7102' publications.pubMedId='25994086' publications.title='Drosophila glucome screening identifies Ck1alpha as a regulator of mammalian glucose metabolism.' publications.volume='6' publications.year=2015\n",
      "Gene: publications.abstractText='Cis-regulatory DNAs control the timing and sites of gene expression during metazoan development. Changes in gene expression are responsible for the morphological diversification of metazoan body plans. However, traditional methods for the identification and characterization of cis-regulatory DNAs are tedious. During the past year, computational methods have been used to identify novel cis-DNAs within the entire Drosophila genome. These methods change the way that cis-DNAs will be analyzed in future studies and offer the promise of unraveling complex gene networks.' publications.doi=None publications.firstAuthor='Markstein Michele' publications.id=1007887 publications.issue='5' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Oct' publications.pages='601-6' publications.pubMedId='12200166' publications.title='Decoding cis-regulatory DNAs in the Drosophila genome.' publications.volume='12' publications.year=2002\n",
      "Gene: publications.abstractText='Cis-regulatory modules (CRMs) function by binding sequence specific transcription factors, but the relationship between in vivo physical binding and the regulatory capacity of factor-bound DNA elements remains uncertain. We investigate this relationship for the well-studied Twist factor in Drosophila melanogaster embryos by analyzing genome-wide factor occupancy and testing the functional significance of Twist occupied regions and motifs within regions. Twist ChIP-seq data efficiently identified previously studied Twist-dependent CRMs and robustly predicted new CRM activity in transgenesis, with newly identified Twist-occupied regions supporting diverse spatiotemporal patterns (>74% positive, n = 31). Some, but not all, candidate CRMs require Twist for proper expression in the embryo. The Twist motifs most favored in genome ChIP data (in vivo) differed from those most favored by Systematic Evolution of Ligands by EXponential enrichment (SELEX) (in vitro). Furthermore, the majority of ChIP-seq signals could be parsimoniously explained by a CABVTG motif located within 50 bp of the ChIP summit and, of these, CACATG was most prevalent. Mutagenesis experiments demonstrated that different Twist E-box motif types are not fully interchangeable, suggesting that the ChIP-derived consensus (CABVTG) includes sites having distinct regulatory outputs. Further analysis of position, frequency of occurrence, and sequence conservation revealed significant enrichment and conservation of CABVTG E-box motifs near Twist ChIP-seq signal summits, preferential conservation of ±150 bp surrounding Twist occupied summits, and enrichment of GA- and CA-repeat sequences near Twist occupied summits. Our results show that high resolution in vivo occupancy data can be used to drive efficient discovery and dissection of global and local cis-regulatory logic.' publications.doi='10.1101/gr.104018.109' publications.firstAuthor='Ozdemir Anil' publications.id=1007715 publications.issue='4' publications.journal='Genome Res.' publications.month='Apr' publications.pages='566-77' publications.pubMedId='21383317' publications.title='High resolution mapping of Twist to DNA in Drosophila embryos: Efficient functional analysis and evolutionary conservation.' publications.volume='21' publications.year=2011\n",
      "Gene: publications.abstractText='Cis-regulatory modules that drive precise spatial-temporal patterns of gene expression are central to the process of metazoan development. We describea new computational strategy to annotate genomic sequences based on their \"pattern generating potential\"and to produce quantitative descriptions of transcriptional regulatory networks at the level of individual protein-module interactions. We use this approach to convert the qualitative understanding of interactions that regulate Drosophila segmentation into a network model in which a confidence value is associated with each transcription factor-module interaction. Sequence information from multiple Drosophila species is integrated with transcription factor binding specificities to determine conserved binding site frequencies across the genome. These binding site profiles are combined with transcription factor expression information to create a model to predict module activity patterns. This model is used to scan genomic sequences for the potential to generate all or part of the expression pattern of a nearby gene, obtained from available gene expression databases. Interactions between individual transcription factors and modules are inferred by a statistical method to quantify a factor\\'s contribution to the module\\'s pattern generating potential. We use these pattern generating potentials to systematically describe the location and function of known and novel cis-regulatory modules in the segmentation network, identifying many examples of modules predicted to have overlapping expression activities. Surprisingly, conserved transcription factor binding site frequencies were as effective as experimental measurements of occupancy in predicting module expression patterns or factor-module interactions. Thus, unlike previous module prediction methods, this method predicts not only the location of modules but also their spatial activity pattern and the factors that directly determine this pattern. As databases of transcription factor specificities and in vivo gene expression patterns grow, analysis of pattern generating potentials provides a general method to decode transcriptional regulatory sequences and networks.' publications.doi='10.1371/journal.pbio.1000456' publications.firstAuthor='Kazemian Majid' publications.id=1007756 publications.issue='8' publications.journal='PLoS Biol.' publications.month=None publications.pages=None publications.pubMedId='20808951' publications.title='Quantitative analysis of the Drosophila segmentation regulatory network using pattern generating potentials.' publications.volume='8' publications.year=2010\n",
      "Gene: publications.abstractText='Codon usage bias, the preferential use of particular codons within each codon family, is characteristic of synonymous base composition in many species, including Drosophila, yeast, and many bacteria. Preferential usage of particular codons in these species is maintained by natural selection acting largely at the level of translation. In Drosophila, as in bacteria, the rate of synonymous substitution per site is negatively correlated with the degree of codon usage bias, indicating stronger selection on codon usage in genes with high codon bias than in genes with low codon bias. Surprisingly, in these organisms, as well as in mammals, the rate of synonymous substitution is also positively correlated with the rate of nonsynonymous substitution. To investigate this correlation, we carried out a phylogenetic analysis of substitutions in 22 genes between two species of Drosophila, Drosophila pseudoobscura and D. subobscura, in codons that differ by one replacement and one synonymous change. We provide evidence for a relative excess of double substitutions in the same species lineage that cannot be explained by the simultaneous mutation of two adjacent bases. The synonymous changes in these codons also cannot be explained by a shift to a more preferred codon following a replacement substitution. We, therefore, interpret the excess of double codon substitutions within a lineage as being the result of relaxed constraints on both kinds of substitutions in particular codons.' publications.doi=None publications.firstAuthor='Comeron J M' publications.id=1007809 publications.issue='2' publications.journal='Genetics' publications.month='Oct' publications.pages='767-75' publications.pubMedId='9755207' publications.title='The correlation between synonymous and nonsynonymous substitutions in Drosophila: mutation, selection or relaxed constraints?' publications.volume='150' publications.year=1998\n",
      "Gene: publications.abstractText='Comparative genomic approaches to gene and cis-regulatory prediction are based on the principle that differential DNA sequence conservation reflects variation in functional constraint. Using this principle, we analyze noncoding sequence conservation in Drosophila for 40 loci with known or suspected cis-regulatory function encompassing >100 kb of DNA. We estimate the fraction of noncoding DNA conserved in both intergenic and intronic regions and describe the length distribution of ungapped conserved noncoding blocks. On average, 22%-26% of noncoding sequences surveyed are conserved in Drosophila, with median block length approximately 19 bp. We show that point substitution in conserved noncoding blocks exhibits transition bias as well as lineage effects in base composition, and occurs more than an order of magnitude more frequently than insertion/deletion (indel) substitution. Overall, patterns of noncoding DNA structure and evolution differ remarkably little between intergenic and intronic conserved blocks, suggesting that the effects of transcription per se contribute minimally to the constraints operating on these sequences. The results of this study have implications for the development of alignment and prediction algorithms specific to noncoding DNA, as well as for models of cis-regulatory DNA sequence evolution.' publications.doi='10.1101/gr.178701' publications.firstAuthor='Bergman C M' publications.id=1007849 publications.issue='8' publications.journal='Genome Res.' publications.month='Aug' publications.pages='1335-45' publications.pubMedId='11483574' publications.title='Analysis of conserved noncoding DNA in Drosophila reveals similar constraints in intergenic and intronic sequences.' publications.volume='11' publications.year=2001\n",
      "Gene: publications.abstractText='Comparisons between Hox genes in different arthropods suggest that the diversity of Antennapedia-class homeotic genes present in modern insects had already arisen before the divergence of insects and crustaceans, probably during the Cambrian. Hox gene duplications are therefore unlikely to have occurred concomitantly with trunk segment diversification in the lineage leading to insects. Available data suggest that domains of homeotic gene expression are also generally conserved among insects, but changes in Hox gene regulation may have played a significant role in segment diversification. Differences that have been documented alter specific aspects of Hox gene regulation within segments and correlate with alterations in segment morphology rather than overt homeotic transformations. The Drosophila Hox cluster contains several homeobox genes that are not homeotic genes--bicoid, fushi-tarazu and zen. the role of these genes during early development has been studied in some detail. It appears to be without parallel among the vertebrate Hox genes. No well conserved homologues of these genes have been found in other taxa, suggesting that they are evolving faster than the homeotic genes. Relatively divergent Antp-class genes isolated from other insects are probably homologues of fushi-tarazu, but these are almost unrecognisable outside of their homeodomains, and have accumulated approximately 10 times as many changes in their homeodomains as have homeotic genes in the same comparisons. They show conserved patterns of expression in the nervous system, but not during early development.' publications.doi=None publications.firstAuthor='Akam M' publications.id=1007898 publications.issue=None publications.journal='Dev. Suppl.' publications.month=None publications.pages='209-15' publications.pubMedId='7579521' publications.title='The evolving role of Hox genes in arthropods.' publications.volume=None publications.year=1994\n",
      "Gene: publications.abstractText='Computational methods attempting to identify instances of cis-regulatory modules (CRMs) in the genome face a challenging problem of searching for potentially interacting transcription factor binding sites while knowledge of the specific interactions involved remains limited. Without a comprehensive comparison of their performance, the reliability and accuracy of these tools remains unclear. Faced with a large number of different tools that address this problem, we summarized and categorized them based on search strategy and input data requirements. Twelve representative methods were chosen and applied to predict CRMs from the Drosophila CRM database REDfly, and across the human ENCODE regions. Our results show that the optimal choice of method varies depending on species and composition of the sequences in question. When discriminating CRMs from non-coding regions, those methods considering evolutionary conservation have a stronger predictive power than methods designed to be run on a single genome. Different CRM representations and search strategies rely on different CRM properties, and different methods can complement one another. For example, some favour homotypical clusters of binding sites, while others perform best on short CRMs. Furthermore, most methods appear to be sensitive to the composition and structure of the genome to which they are applied. We analyze the principal features that distinguish the methods that performed well, identify weaknesses leading to poor performance, and provide a guide for users. We also propose key considerations for the development and evaluation of future CRM-prediction methods.' publications.doi='10.1371/journal.pcbi.1001020' publications.firstAuthor='Su Jing' publications.id=1007788 publications.issue='12' publications.journal='PLoS Comput. Biol.' publications.month=None publications.pages='e1001020' publications.pubMedId='21152003' publications.title='Assessing computational methods of cis-regulatory module prediction.' publications.volume='6' publications.year=2010\n",
      "Gene: publications.abstractText=\"Current models describe male-specific fruitless (fruM) as a genetic 'switch' regulating sexual behavior in Drosophila melanogaster, and they postulate that female (F) and male (M) doublesex (dsx) products control body sexual morphology. In contradiction to this simple model, we show that dsx, as well as fruM and non-sex-specific retained (retn), affect both male and female sexual behaviors. In females, both retn and dsxF contribute to female receptivity, and both genes act to repress male-like courtship activity in the presence or absence of fruM. In males, consistent with the opposing functions of dsxM and dsxF, dsxM acts as a positive factor for male courtship. retn also acts counter to fruM in the development of the male-specific muscle of Lawrence. Molecularly, retn seems to regulate sexual behavior via a previously described complex that represses zerknullt. Thus, we show that fru and dsx together act as a 'switch' system regulating behavior in the context of other developmental genes, such as retn.\" publications.doi='10.1038/ng1908' publications.firstAuthor='Shirangi Troy R' publications.id=1007853 publications.issue='12' publications.journal='Nat. Genet.' publications.month='Dec' publications.pages='1435-9' publications.pubMedId='17086183' publications.title='A double-switch system regulates male courtship behavior in male and female Drosophila melanogaster.' publications.volume='38' publications.year=2006\n",
      "Gene: publications.abstractText='DREF, a transcription regulatory factor which specifically binds to the promoter-activating element DRE (DNA replication-related element) of DNA replication-related genes, was purified to homogeneity from nuclear extracts of Drosophila Kc cells. cDNA for DREF was isolated with the reverse-transcriptase polymerase chain reaction method using primers synthesized on the basis of partial amino acid sequences and following screening of cDNA libraries. Deduced from the nucleotide sequences of cDNA, DREF is a polypeptide of 701 amino acid residues with a molecular weight of 80,096, which contains three characteristic regions, rich in basic amino acids, proline, and acidic amino acids, respectively. Deletion analysis of bacterially expressed DREF fused with glutathione S-transferase (GST-DREF) indicated that a part of the N-terminal basic amino acid region (16-115 amino acids) is responsible for the specific binding to DRE. A polyclonal and four monoclonal antibodies were raised against the GST-DREF fusion protein. The antibodies inhibited specifically the transcription of DNA polymerase alpha promoter in vitro. Cotransfection experiments using Kc cells demonstrated that overproduction of DREF protein overcomes the repression of the proliferating cell nuclear antigen gene promoter by the zerknüllt gene product. These results confirmed that DREF is a trans-activating factor for DNA replication-related genes. Immunocytochemical analysis demonstrated the presence of DREF polypeptide in nuclei after the eighth nuclear division cycle, suggesting that nuclear accumulation of DREF is important for the coordinate zygotic expression of DNA replication-related genes carrying DRE sequences.' publications.doi=None publications.firstAuthor='Hirose F' publications.id=1007827 publications.issue='7' publications.journal='J. Biol. Chem.' publications.month='Feb' publications.pages='3930-7' publications.pubMedId='8632015' publications.title='Isolation and characterization of cDNA for DREF, a promoter-activating factor for Drosophila DNA replication-related genes.' publications.volume='271' publications.year=1996\n",
      "Gene: publications.abstractText=\"Decapentaplegic (Dpp), the fly homolog of the secreted mammalian BMP2/4 signaling molecules, is involved in almost all aspects of fly development. Dpp has critical functions at all developmental stages, from patterning of the eggshell to the determination of adult intestinal stem cell identity. Here, we focus on recent findings regarding the transcriptional regulatory logic of the pathway, on a new feedback regulator, Pentagone, and on Dpp's roles in scaling and growth of the Drosophila wing.\" publications.doi='10.1016/j.semcdb.2014.04.036' publications.firstAuthor='Hamaratoglu Fisun' publications.id=1007722 publications.issue=None publications.journal='Semin. Cell Dev. Biol.' publications.month='Aug' publications.pages='128-36' publications.pubMedId='24813173' publications.title='Dpp/BMP signaling in flies: from molecules to biology.' publications.volume='32' publications.year=2014\n",
      "Gene: publications.abstractText='Developing tissues are patterned by coordinated activities of signaling systems, which can be integrated by a regulatory region of a gene that binds multiple transcription factors or by a transcription factor that is modified by multiple enzymes. Based on a combination of genetic and imaging experiments in the early Drosophila embryo, we describe a signal integration mechanism that cannot be reduced to a single gene regulatory element or a single transcription factor. This mechanism relies on an enzymatic network formed by mitogen-activated protein kinase (MAPK) and its substrates. Specifically, anteriorly localized MAPK substrates, such as Bicoid, antagonize MAPK-dependent downregulation of Capicua, a repressor that is involved in gene regulation along the dorsoventral axis of the embryo. MAPK substrate competition provides a basis for ternary interaction of the anterior, dorsoventral, and terminal patterning systems. A mathematical model of this interaction can explain gene expression patterns with both anteroposterior and dorsoventral polarities.' publications.doi='10.1016/j.devcel.2011.05.009' publications.firstAuthor='Kim Yoosik' publications.id=1007720 publications.issue='6' publications.journal='Dev. Cell' publications.month='Jun' publications.pages='880-7' publications.pubMedId='21664584' publications.title='Gene regulation by MAPK substrate competition.' publications.volume='20' publications.year=2011\n",
      "Gene: publications.abstractText='Differentiation of distinct cell types at specific locations within a developing organism depends largely on the ability of cells to communicate. A major class of signalling proteins implicated in cell to cell communication is represented by members of the TGF beta superfamily. A corresponding class of transmembrane serine/threonine kinases has recently been discovered that act as cell surface receptors for ligands of the TGF beta superfamily. The product of the Drosophila gene decapentaplegic (dpp) encodes a TGF beta homolog that plays multiple roles during embryogenesis and the development of imaginal discs. Here we describe the complex expression pattern of thick veins (tkv), which encodes a receptor for dpp. We make use of tkv loss-of-function mutations to examine the consequences of the failure of embryonic cells to respond to dpp and/or other TGF beta homologs. We find that while maternal tkv product allows largely normal dorsoventral pattering of the embryo, zygotic tkv activity is indispensable for dorsal closure of the embryo after germ band retraction. Furthermore, tkv activity is crucial for patterning the visceral mesoderm; in the absence of functional tkv gene product, visceral mesoderm parasegment 7 cells fail to express Ultrabithorax, but instead accumulate Antennapedia protein. The tkv receptor is therefore involved in delimiting the expression domains of homeotic genes in the visceral mesoderm. Interestingly, tkv mutants fail to establish a proper tracheal network. Tracheal braches formed by cells migrating in dorsal or ventral directions are absent in tkv mutants. The requirements for tkv in dorsal closure, visceral mesoderm and trachea development assign novel functions to dpp or a closely related member of the TGF beta superfamily.' publications.doi=None publications.firstAuthor='Affolter M' publications.id=1007738 publications.issue='11' publications.journal='Development' publications.month='Nov' publications.pages='3105-17' publications.pubMedId='7720555' publications.title='Multiple requirements for the receptor serine/threonine kinase thick veins reveal novel functions of TGF beta homologs during Drosophila embryogenesis.' publications.volume='120' publications.year=1994\n",
      "Gene: publications.abstractText='Differentiation of the embryonic termini in Drosophila depends on signaling by the Tor RTK, which induces terminal gene expression by inactivating at the embryonic poles a uniformly distributed repressor activity that involves the Gro corepressor. Here, we identify a new gene, cic, that acts as a repressor of terminal genes regulated by the Tor pathway. cic also mediates repression along the dorsoventral axis, a process that requires the Dorsal morphogen and Gro, and which is also inhibited by Tor signaling at the termini. cic encodes an HMG-box transcription factor that interacts with Gro in vitro. We present evidence that Tor signaling regulates terminal patterning by inactivating Cic at the embryo poles. cic has been evolutionarily conserved, suggesting that Cic-like proteins may act as repressors regulated by RTK signaling in other organisms.' publications.doi=None publications.firstAuthor='Jiménez G' publications.id=1007737 publications.issue='2' publications.journal='Genes Dev.' publications.month='Jan' publications.pages='224-31' publications.pubMedId='10652276' publications.title='Relief of gene repression by torso RTK signaling: role of capicua in Drosophila terminal and dorsoventral patterning.' publications.volume='14' publications.year=2000\n",
      "Gene: publications.abstractText='Divergent morphology of species has largely been ascribed to genetic differences in the tissue-specific expression of proteins, which could be achieved by divergence in cis-regulatory elements or by altering the binding specificity of transcription factors (TFs). The relative importance of the latter has been difficult to assess, as previous systematic analyses of TF binding specificity have been performed using different methods in different species. To address this, we determined the binding specificities of 242 Drosophila TFs, and compared them to human and mouse data. This analysis revealed that TF binding specificities are highly conserved between Drosophila and mammals, and that for orthologous TFs, the similarity extends even to the level of very subtle dinucleotide binding preferences. The few human TFs with divergent specificities function in cell types not found in fruit flies, suggesting that evolution of TF specificities contributes to emergence of novel types of differentiated cells.' publications.doi='10.7554/eLife.04837' publications.firstAuthor='Nitta Kazuhiro R' publications.id=1000114 publications.issue=None publications.journal='Elife' publications.month=None publications.pages=None publications.pubMedId='25779349' publications.title='Conservation of transcription factor binding specificities across 600 million years of bilateria evolution.' publications.volume='4' publications.year=2015\n",
      "Gene: publications.abstractText=\"Diverse types of epithelial morphogenesis drive development. Similar cytoskeletal and cell adhesion machinery orchestrate these changes, but it is unclear how distinct tissue types are produced. Thus, it is important to define and compare different types of morphogenesis. We investigated cell flattening and elongation in the amnioserosa, a squamous epithelium formed at Drosophila gastrulation. Amnioserosa cells are initially columnar. Remarkably, they flatten and elongate autonomously by perpendicularly rotating the microtubule cytoskeleton--we call this 'rotary cell elongation'. Apical microtubule protrusion appears to initiate the rotation and microtubule inhibition perturbs the process. F-actin restrains and helps orient the microtubule protrusions. As amnioserosa cells elongate, they maintain their original cell-cell contacts and develop planar polarity. Myosin II localizes to anterior-posterior contacts, while the polarity protein Bazooka (PAR-3) localizes to dorsoventral contacts. Genetic analysis revealed that Myosin II and Bazooka cooperate to properly position adherens junctions. These results identify a specific cellular mechanism of squamous tissue morphogenesis and molecular interactions involved.\" publications.doi='10.1242/dev.019802' publications.firstAuthor='Pope Karen L' publications.id=1007798 publications.issue='13' publications.journal='Development' publications.month='Jul' publications.pages='2227-38' publications.pubMedId='18508861' publications.title='Control of cell flattening and junctional remodeling during squamous epithelial morphogenesis in Drosophila.' publications.volume='135' publications.year=2008\n",
      "Gene: publications.abstractText='Dorsal functions as both an activator and repressor of transcription to determine dorsoventral fate in the Drosophila melanogaster embryo. Repression by Dorsal requires the corepressor Groucho (Gro) and is mediated by silencers termed ventral repression regions (VRRs). A VRR in zerknüllt (zen) contains Dorsal binding sites as well as an essential element termed AT2. We have identified and purified an AT2 DNA binding activity in embryos and shown it to consist of cut (ct) and dead ringer (dri) gene products. Studies of loss-of-function mutations in ct and dri demonstrate that both genes are required for the activity of the AT2 site. Dorsal and Dri both bind Gro, acting cooperatively to recruit it to the DNA. Thus, ventral repression may require the formation of a multiprotein complex at the VRR. This complex includes Dorsal, Gro, and additional DNA binding proteins, which appear to convert Dorsal from an activator to a repressor by enabling it to recruit Gro to the template. By showing how binding site context can dramatically alter transcription factor function, these findings help clarify the mechanisms responsible for the regulatory specificity of transcription factors.' publications.doi=None publications.firstAuthor='Valentine S A' publications.id=1007782 publications.issue='11' publications.journal='Mol. Cell. Biol.' publications.month='Nov' publications.pages='6584-94' publications.pubMedId='9774673' publications.title='Dorsal-mediated repression requires the formation of a multiprotein repression complex at the ventral silencer.' publications.volume='18' publications.year=1998\n",
      "Gene: publications.abstractText='Dorsal-ventral patterning in vertebrate and invertebrate embryos is mediated by a conserved system of secreted proteins that establishes a bone morphogenetic protein (BMP) gradient. Although the Drosophila embryonic Decapentaplegic (Dpp) gradient has served as a model to understand how morphogen gradients are established, no role for the extracellular matrix has been previously described. Here we show that type IV collagen extracellular matrix proteins bind Dpp and regulate its signalling in both the Drosophila embryo and ovary. We provide evidence that the interaction between Dpp and type IV collagen augments Dpp signalling in the embryo by promoting gradient formation, yet it restricts the signalling range in the ovary through sequestration of the Dpp ligand. Together, these results identify a critical function of type IV collagens in modulating Dpp in the extracellular space during Drosophila development. On the basis of our findings that human type IV collagen binds BMP4, we predict that this role of type IV collagens will be conserved.' publications.doi='10.1038/nature07214' publications.firstAuthor='Wang Xiaomeng' publications.id=1007879 publications.issue='7209' publications.journal='Nature' publications.month='Sep' publications.pages='72-7' publications.pubMedId='18701888' publications.title='Type IV collagens regulate BMP signalling in Drosophila.' publications.volume='455' publications.year=2008\n",
      "Gene: publications.abstractText=\"Dorsoventral (DV) patterning of the Drosophila embryo is initiated by a broad Dorsal (Dl) nuclear gradient, which is regulated by a conserved signaling pathway that includes the Toll receptor and Pelle kinase. We investigate the consequences of expressing a constitutively activated form of the Toll receptor, Toll(10b), in anterior regions of the early embryo using the bicoid 3' UTR. Localized Toll(10b) products result in the formation of an ectopic, anteroposterior (AP) Dl nuclear gradient along the length of the embryo. The analysis of both authentic dorsal target genes and defined synthetic promoters suggeststhat the ectopic gradient is sufficient to generate the full repertory of DV patterning responses along the AP axis of the embryo. For example, mesoderm determinants are activated in the anterior third of the embryo, whereas neurogenic genes are expressed in central regions. These results raise the possibility that Toll signaling components diffuse in the plasma membrane or syncytial cytoplasm of the early embryo. This study also provides evidence that neurogenic repressors may be important for the establishment of the sharp mesoderm/neuroectoderm boundary in the early embryo.\" publications.doi=None publications.firstAuthor='Huang A M' publications.id=1002676 publications.issue='15' publications.journal='Genes Dev.' publications.month='Aug' publications.pages='1963-73' publications.pubMedId='9271119' publications.title='An anteroposterior Dorsal gradient in the Drosophila embryo.' publications.volume='11' publications.year=1997\n",
      "Gene: publications.abstractText='Dorsoventral patterning in Drosophila is initiated by the maternal regulatory factor dorsal (dl), which is a member of the Rel family of transcription factors. dl functions as a transcriptional activator and repressor to establish different territories of gene expression in the precellular embryo. Differential regulation of dl target genes may be essential for subdividing each tissue territory (the presumptive mesoderm, neuroectoderm, and dorsal ectoderm) into multiple cell types in older embryos. Different patterns of snail (sna) and decapentaplegic (dpp) expression help define the limits of inductive interactions between the mesoderm and dorsal ectoderm after gastrulation. Similarly, the differential regulation of short gastrulation (sog) and dpp may be decisive in the initial subdivision of the dorsal ectoderm, whereas different limits of gene expression within the neuroectoderm might provide the basis for the subsequent subdivision of this tissue into ventral and lateral regions.' publications.doi=None publications.firstAuthor='Rusch J' publications.id=1004078 publications.issue='4' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Aug' publications.pages='416-23' publications.pubMedId='8791536' publications.title='Threshold responses to the dorsal regulatory gradient and the subdivision of primary tissue territories in the Drosophila embryo.' publications.volume='6' publications.year=1996\n",
      "Gene: publications.abstractText='Dorsoventral patterning in Drosophila requires the Dorsal morphogen to act as both an activator and a repressor of transcription: an HMG1-like protein may serve to switch Dorsal from one to the other.' publications.doi=None publications.firstAuthor='Ip Y T' publications.id=1007784 publications.issue='1' publications.journal='Curr. Biol.' publications.month='Jan' publications.pages='1-3' publications.pubMedId='7697337' publications.title='Transcriptional regulation. Converting an activator into a repressor.' publications.volume='5' publications.year=1995\n",
      "Gene: publications.abstractText='Dorsoventral patterning of the Drosophila embryo is regulated by graded distribution of bone morphogenetic proteins (BMPs) composed of two ligands, decapentaplegic (Dpp) a BMP2/4 ortholog and screw (Scw) a BMP5/6/7/8 family member. scw(E1) encodes an unusual allele that was isolated as a dominant enhancer of partial loss-of-function mutations in dpp. However, the molecular mechanisms that underlie this genetic interaction remain to be addressed. Here we show that scw(E1) contains a mutation at the furin cleavage site within the prodomain that is crucial for ligand production. Furthermore, our data show that Scw(E1) preferentially forms heterodimers with Dpp rather than homotypic dimers, providing a possible explanation for the dominant negative phenotype of scw(E1) alleles. The unprocessed prodomain of Scw(E1) remains in a complex with the Dpp:Scw heterodimer, and thus could interfere with interaction of the ligand with the extracellular matrix, or the kinetics of processing/secretion of the ligand in vivo. These data reveal novel mechanisms by which post-translational regulation of Scw can modulate Dpp signaling activity.' publications.doi='10.1016/j.ydbio.2014.02.007' publications.firstAuthor='Künnapuu Jaana' publications.id=1007764 publications.issue='2' publications.journal='Dev. Biol.' publications.month='May' publications.pages='149-59' publications.pubMedId='24560644' publications.title='Cleavage of the Drosophila screw prodomain is critical for a dynamic BMP morphogen gradient in embryogenesis.' publications.volume='389' publications.year=2014\n",
      "Gene: publications.abstractText='Dorsoventral polarity of the Drosophila embryo requires maternal spätzle-Toll signaling to establish a nuclear gradient of Dorsal protein. The shape of this gradient is altered in embryos produced by females carrying dominant alleles of easter (ea(D)). The easter gene encodes a serine protease that generates processed Spätzle, which is proposed to act as the Toll ligand. By examining the expression domains of the zygotic genes zen, sog, rho and twist, which are targets of nuclear Dorsal, we show that the slope of the Dorsal gradient is progressively flattened in stronger ea(D) alleles. In the wild-type embryo, activated Easter is found in a high M(r) complex called Ea-X, which is hypothesized to contain a protease inhibitor. In ea(D) embryo extracts, we detect an Easter form corresponding to the free catalytic domain, which is never observed in wild type. These mutant ea(D) proteins retain protease activity, as determined by the production of processed Spätzle both in the embryo and in cultured Drosophila cells. These experiments suggest that the ea(D) mutations interfere with inactivation of catalytic Easter, and imply that this negative regulation is essential for generating the wild-type shape of the Dorsal gradient.' publications.doi=None publications.firstAuthor='Chang Andy J' publications.id=1007733 publications.issue='24' publications.journal='Development' publications.month='Dec' publications.pages='5635-45' publications.pubMedId='12421704' publications.title='Regulation of Easter activity is required for shaping the Dorsal gradient in the Drosophila embryo.' publications.volume='129' publications.year=2002\n",
      "Gene: publications.abstractText='Dpp signals are responsible for establishing a variety of cell identities in dorsal and lateral areas of the early Drosophila embryo, including the extra-embryonic amnioserosa as well as different ectodermal and mesodermal cell types. Although we have a reasonably clear picture of how Dpp signaling activity is modulated spatially and temporally during these processes, a better understanding of how these signals are executed requires the identification and characterization of a collection of downstream genes that uniquely respond to these signals. In the present study, we describe three novel genes, Dorsocross1, Dorsocross2 and Dorsocross3, which are expressed downstream of Dpp in the presumptive and definitive amnioserosa, dorsal ectoderm and dorsal mesoderm. We show that these genes are good candidates for being direct targets of the Dpp signaling cascade. Dorsocross expression in the dorsal ectoderm and mesoderm is metameric and requires a combination of Dpp and Wingless signals. In addition, a transverse stripe of expression in dorsoanterior areas of early embryos is independent of Dpp. The Dorsocross genes encode closely related proteins of the T-box domain family of transcription factors. All three genes are arranged in a gene cluster, are expressed in identical patterns in embryos, and appear to be genetically redundant. By generating mutants with a loss of all three Dorsocross genes, we demonstrate that Dorsocross gene activity is crucial for the completion of differentiation, cell proliferation arrest, and survival of amnioserosa cells. In addition, we show that the Dorsocross genes are required for normal patterning of the dorsolateral ectoderm and, in particular, the repression of wingless and the ladybird homeobox genes within this area of the germ band. These findings extend our knowledge of the regulatory pathways during amnioserosa development and the patterning of the dorsolateral embryonic germ band in response to Dpp signals.' publications.doi=None publications.firstAuthor='Reim Ingolf' publications.id=1007878 publications.issue='14' publications.journal='Development' publications.month='Jul' publications.pages='3187-204' publications.pubMedId='12783790' publications.title='The T-box-encoding Dorsocross genes function in amnioserosa development and the patterning of the dorsolateral germ band downstream of Dpp.' publications.volume='130' publications.year=2003\n",
      "Gene: publications.abstractText='Dramatic successes in identifying vertebrate homeobox genes closely related to their insect relatives have led to the recognition of classes within the homeodomain superfamily. To what extent are the homeodomain protein classes dedicated to specific functions during development? Although information on vertebrate gene functions is limited, existing evidence from mice and nematodes clearly supports conservation of function for the Hox genes. Less compelling, but still remarkable, is the conservation of other homeobox gene classes and of regulators of homeotic gene expression and function. It is too soon to say whether the cases of conservation are unique and exceptional, or the beginning of a profoundly unified view of gene regulation in animal development. In any case, new questions are raised by the data: how can the differences between mammals and insects be compatible with conservation of homeobox gene function? Did the evolution of animal form involve a proliferation of new homeodomain proteins, new modes of regulation of existing gene types, or new relationships with target genes, or is evolutionary change largely the province of other classes of genes? In this review, we summarize what is known about conservation of homeobox gene function.' publications.doi=None publications.firstAuthor='Manak J R' publications.id=1007790 publications.issue=None publications.journal='Dev. Suppl.' publications.month=None publications.pages='61-77' publications.pubMedId='7579525' publications.title='A class act: conservation of homeodomain protein functions.' publications.volume=None publications.year=1994\n",
      "Gene: publications.abstractText=\"Drosophila embryos lacking hindsight gene function have a normal body plan and undergo normal germ-band extension. However, they fail to retract their germ bands. hindsight encodes a large nuclear protein of 1920 amino acids that contains fourteen C2H2-type zinc fingers, and glutamine-rich and proline-rich domains, suggesting that it functions as a transcription factor. Initial embryonic expression of hindsight RNA and protein occurs in the endoderm (midgut) and extraembryonic membrane (amnioserosa) prior to germ-band extension and continues in these tissues beyond the completion of germ-band retraction. Expression also occurs in the developing tracheal system, central and peripheral nervous systems, and the ureter of the Malpighian tubules. Strikingly, hindsight is not expressed in the epidermal ectoderm which is the tissue that undergoes the cell shape changes and movements during germ-band retraction. The embryonic midgut can be eliminated without affecting germ-band retraction. However, elimination of the amnioserosa results in the failure of germ-band retraction, implicating amnioserosal expression of hindsight as crucial for this process. Ubiquitous expression of hindsight in the early embryo rescues germ-band retraction without producing dominant gain-of-function defects, suggesting that hindsight's role in germ-band retraction is permissive rather than instructive. Previous analyses have shown that hindsight is required for maintenance of the differentiated amnioserosa (Frank, L. C. and Rushlow, C. (1996) Development 122, 1343-1352). Two classes of models are consistent with the present data. First, hindsight's function in germ-band retraction may be limited to maintenance of the amnioserosa which then plays a physical role in the retraction process through contact with cells of the epidermal ectoderm. Second, hindsight might function bothto maintain the amnioserosa and to regulate chemical signaling from the amnioserosa to the epidermal ectoderm, thus coordinating the cell shape changes and movements that drive germ-band retraction.\" publications.doi=None publications.firstAuthor='Yip M L' publications.id=1004126 publications.issue='11' publications.journal='Development' publications.month='Jun' publications.pages='2129-41' publications.pubMedId='9187140' publications.title='Control of germ-band retraction in Drosophila by the zinc-finger protein HINDSIGHT.' publications.volume='124' publications.year=1997\n",
      "Gene: publications.abstractText='Drosophila melanogaster is a proven model system for many aspects of human biology. Here we present a two-hybrid-based protein-interaction map of the fly proteome. A total of 10,623 predicted transcripts were isolated and screened against standard and normalized complementary DNA libraries to produce a draft map of 7048 proteins and 20,405 interactions. A computational method of rating two-hybrid interaction confidence was developed to refine this draft map to a higher confidence map of 4679 proteins and 4780 interactions. Statistical modeling of the network showed two levels of organization: a short-range organization, presumably corresponding to multiprotein complexes, and a more global organization, presumably corresponding to intercomplex connections. The network recapitulated known pathways, extended pathways, and uncovered previously unknown pathway components. This map serves as a starting point for a systems biology modeling of multicellular organisms, including humans.' publications.doi='10.1126/science.1090289' publications.firstAuthor='Giot L' publications.id=1000762 publications.issue='5651' publications.journal='Science' publications.month='Dec' publications.pages='1727-36' publications.pubMedId='14605208' publications.title='A protein interaction map of Drosophila melanogaster.' publications.volume='302' publications.year=2003\n",
      "Gene: publications.abstractText='During embryogenesis Drosophila pole cells, like germ cells in many other vertebrates and invertebrates, actively migrate before assembling into the gonad. Five separate steps can be distinguished: an initial passive displacement of the germ cells by gastrulation movements, an amoeboid migratory phase during which the pole cells pass through the endoderm, migration on endoderm toward mesoderm, separation into two bilateral groups associated with the gonadal mesoderm precursors, followed by condensation into the gonad itself. We have analyzed gonad assembly in embryonic pattern mutants to determine whether distinct cues are required in this process. We show that the initiation of migration does not require the presence of the mesoderm, the eventual target of the germ cells. Rather, migration is triggered as a consequence of the differentiation of the endoderm. Examination of embryos mutant for maternal genes of the terminal group suggests that a primary role of the endoderm in this process is to allow the germ cell access to the interior of the embryo. In addition, we show that normal gonad assembly requires a region of the embryo that includes the posterior compartment of the fifth and the sixth abdominal segments.' publications.doi='10.1006/dbio.1994.1306' publications.firstAuthor='Warrior R' publications.id=1007791 publications.issue='1' publications.journal='Dev. Biol.' publications.month='Nov' publications.pages='180-94' publications.pubMedId='7958445' publications.title='Primordial germ cell migration and the assembly of the Drosophila embryonic gonad.' publications.volume='166' publications.year=1994\n",
      "Gene: publications.abstractText=\"During postblastoderm embryogenesis in Drosophila, cell cycles progress in an invariant spatiotemporal pattern. Most of these cycles are differentially timed by bursts of transcription of string (cdc25), a gene encoding a phosphatase that triggers mitosis by activating the Cdc2 kinase. An analysis of string expression in 36 pattern-formation mutants shows that known patterning genes act locally to influence string transcription. Embryonic expression of string gene fragments shows that the complete pattern of string transcription requires extensive cis-acting regulatory sequences (>15.3 kb), but that smaller segments of this regulatory region can drive proper temporal expression in defined spatial domains. We infer that string upstream sequences integrate many local signals to direct string's transcriptional program. Finally, we show that the spatiotemporal progression of string transcription is largely unaffected in mutant embryos specifically arrested in G2 of cycles 14, 15, or 16, or G1 of cycle 17. Thus, there is a regulatory hierarchy in which developmental inputs, not cell cycle inputs, control the timing of string transcription and hence cell cycle progression.\" publications.doi=None publications.firstAuthor='Edgar B A' publications.id=1007838 publications.issue='11' publications.journal='Development' publications.month='Nov' publications.pages='3131-43' publications.pubMedId='7720557' publications.title='Transcriptional regulation of string (cdc25): a link between developmental programming and the cell cycle.' publications.volume='120' publications.year=1994\n",
      "Gene: publications.abstractText='During the maternal-to-zygotic transition, a developing embryo integrates post-transcriptional regulation of maternal mRNAs with transcriptional activation of its own genome. By combining chromosomal ablation in Drosophila with microarray analysis, we characterized the basis of this integration. We show that the expression profile for at least one third of zygotically active genes is coupled to the concomitant degradation of the corresponding maternal mRNAs. The embryo uses transcription and degradation to generate localized patterns of expression, and zygotic transcription to degrade distinct classes of maternal transcripts. Although degradation does not appear to involve a simple regulatory code, the activation of the zygotic genome starts from intronless genes sharing a common cis-element. This cis-element interacts with a single protein, the Bicoid stability factor, and acts as a potent enhancer capable of timing the activity of an exogenous transactivator. We propose that this regulatory mode links morphogen gradients with temporal regulation during the maternal-to-zygotic transition.' publications.doi='10.1371/journal.pbio.0050117' publications.firstAuthor='De Renzis Stefano' publications.id=1007822 publications.issue='5' publications.journal='PLoS Biol.' publications.month='May' publications.pages='e117' publications.pubMedId='17456005' publications.title='Unmasking activation of the zygotic genome using chromosomal deletions in the Drosophila embryo.' publications.volume='5' publications.year=2007\n",
      "Gene: publications.abstractText='Early Drosophila development requires two receptor tyrosine kinase (RTK) pathways: the Torso and the Epidermal growth factor receptor (EGFR) pathways, which regulate terminal and dorsal-ventral patterning, respectively. Previous studies have shown that these pathways, either directly or indirectly, lead to post-transcriptional downregulation of the Capicua repressor in the early embryo and in the ovary. Here, we show that both regulatory effects are direct and depend on a MAPK docking site in Capicua that physically interacts with the MAPK Rolled. Capicua derivatives lacking this docking site cause dominant phenotypes similar to those resulting from loss of Torso and EGFR activities. Such phenotypes arise from inappropriate repression of genes normally expressed in response to Torso and EGFR signaling. Our results are consistent with a model whereby Capicua is the main nuclear effector of the Torso pathway, but only one of different effectors responding to EGFR signaling. Finally, we describe differences in the modes of Capicua downregulation by Torso and EGFR signaling, raising the possibility that such differences contribute to the tissue specificity of both signals.' publications.doi='10.1038/sj.emboj.7601532' publications.firstAuthor='Astigarraga Sergio' publications.id=1007686 publications.issue='3' publications.journal='EMBO J.' publications.month='Feb' publications.pages='668-77' publications.pubMedId='17255944' publications.title='A MAPK docking site is critical for downregulation of Capicua by Torso and EGFR RTK signaling.' publications.volume='26' publications.year=2007\n",
      "Gene: publications.abstractText='Embryogenesis in individuals with mutations or deficiencies of the genes in the polytene interval 84A-84B1,2 of Drosophila melanogaster was examined using scanning electron microscopy (SEM). The developmental function of this region of chromosome 3 is of particular interest since it contains the Antennapedia Gene Complex (ANT-C), a gene cluster that includes the homoeotic proboscipedia (pb), Sex combs reduced (Scr), and Antennapedia (Antp) loci. The results of SEM studies, clonal analyses, and temperature-shift experiments show that the fushi tarazu (ftz) and zerknullt (zen) genes, which map between pb and Scr, are involved in processes initiated during embryogenesis. The activity of ftz+ appears to be required within the first 4 hr of development for the establishment of the proper number of segments in the embryonic germ band. Individuals with ftz mutations or deficiencies produce only half the normal number of segments. Each of the segments is twice the normal width and is apparently comprised of cells that would normally form two separate metameres. The zen allele is required from about 2-4 hr of embryogenesis. Mutations of this gene result in disturbances of morphogenetic movements during gastrulation. The mutant phenotype is characterized by the absence of the optic lobe, defects in involution of the head segments, and in some cases, failure of germ band elongation. A requirement during embryogenesis for the activities of other genes residing in the 84A-84B1,2 polytene interval is suggested by the phenotypes of individuals heterozygous or homozygous for chromosomal deficiencies. Using the deficiencies Df(3R)AntpNs+R17, Df(3R)Scr, and Df(3R)ScxW+RX2, we examined the effects of deleting the distal portions or all of the 84A-84B1,2 interval. The defects in deletion heterozygotes suggest that the wild-type activity of some gene(s) other than zen, within or just adjacent to the 84B1,2 doublet, is required to complete normal head involution. Thedeletion of all the loci in the 84A5-84B1,2 interval results in grossly abnormal morphology and morphogenesis of the gnathocephalic appendages of the embryo. From these studies we conclude that mutations and deficiencies of genes associated with the ANT-C have profound effects on embryogenesis. The mutant phenotypes suggest, in addition to ensuring proper segment identity, the wild-type alleles of the 84A-84B1,2 genes are necessary for normal segmentation and elongation of the germ band and normal head involution.' publications.doi=None publications.firstAuthor='Wakimoto B T' publications.id=1005524 publications.issue='1' publications.journal='Dev. Biol.' publications.month='Mar' publications.pages='147-72' publications.pubMedId='6421639' publications.title='Defects in embryogenesis in mutants associated with the antennapedia gene complex of Drosophila melanogaster.' publications.volume='102' publications.year=1984\n",
      "Gene: publications.abstractText='Embryonic dorsal-ventral polarity in Drosophila is established through a series of successive steps and requires the functions of both maternal and zygotic genes. The graded distribution of the transcription factor dorsal in blastoderm nuclei represents the transition from the maternal to the zygotic program. This results in the activation of specific zygotic genes that act to create the regional pattern along this axis.' publications.doi=None publications.firstAuthor='Steward R' publications.id=1002691 publications.issue='4' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Aug' publications.pages='556-61' publications.pubMedId='8241766' publications.title='Dorsal-ventral polarity in the Drosophila embryo.' publications.volume='3' publications.year=1993\n",
      "Gene: publications.abstractText='Embryonic gene expression patterns are an indispensable part of modern developmental biology. Currently, investigators must visually inspect numerous images containing embryonic expression patterns to identify spatially similar patterns for inferring potential genetic interactions. The lack of a computational approach to identify pattern similarities is an impediment to advancement in developmental biology research because of the rapidly increasing amount of available embryonic gene expression data. Therefore, we have developed computational approaches to automate the comparison of gene expression patterns contained in images of early stage Drosophila melanogaster embryos (prior to the beginning of germ-band elongation); similarities and differences in gene expression patterns in these early stages have extensive developmental effects. Here we describe a basic expression search tool (BEST) to retrieve best matching expression patterns for a given query expression pattern and a computational device for gene interaction inference using gene expression pattern images and information on the associated genotypes and probes. Analysis of a prototype collection of Drosophila gene expression pattern images is presented to demonstrate the utility of these methods in identifying biologically meaningful matches and inferring gene interactions by direct image content analysis. In particular, the use of BEST searches for gene expression patterns is akin to that of BLAST searches for finding similar sequences. These computational developmental biology methodologies are likely to make the great wealth of embryonic gene expression pattern data easily accessible and to accelerate the discovery of developmental networks.' publications.doi=None publications.firstAuthor='Kumar Sudhir' publications.id=1007875 publications.issue='4' publications.journal='Genetics' publications.month='Dec' publications.pages='2037-47' publications.pubMedId='12524369' publications.title='BEST: a novel computational approach for comparing gene expression patterns from early stages of Drosophila melanogaster development.' publications.volume='162' publications.year=2002\n",
      "Gene: publications.abstractText='Epistasis analysis has been used in molecular genetics to infer cause and effect relationships among loci that decide a developmental fate. In this paper, in order to investigate the logical background of the rule that relates epistasis and causal relation, a mathematical epistasis analysis is formulated. An \"orderliness index\"is introduced by which the cause-and-effect relation can systematically be extracted from genotype-phenotype data. This method is applied to Drosophila melanogaster dorso-ventral axis formation and it is shown that the method of reconstructing the framework of the causal relation between a locus and its sign (activation or inhibition) as an \"orderliness diagram\"from eight groups of the genes. From the diagram it can be seen that the dorsal gene works as a morphogen. The difference between orderliness diagrams of the dorsal and ventral sides of the embryo shows that the cells on the ventral side receive the signal and the fate of the cells on the dorsal side is default in the wild-type embryo.' publications.doi='10.1006/jtbi.1995.0175' publications.firstAuthor='Takeda Y' publications.id=1007701 publications.issue='1' publications.journal='J. Theor. Biol.' publications.month='Sep' publications.pages='47-58' publications.pubMedId='7475106' publications.title='A generalized epistasis analysis as a method of reconstruction of developmental process and its application to Drosophila dorso-ventral axis formation.' publications.volume='176' publications.year=1995\n",
      "Gene: publications.abstractText='Establishment of dorsal-ventral polarity in the early Drosophila embryo requires a concentration gradient of the maternal morphogen dorsal (dl). This concentration gradient is established by selective nuclear transport of dl so that dl protein is present only in ventral nuclei. The activity of 11 genes is required for dl nuclear localization. One of these genes, Toll, encodes a transmembrane protein that appears to play the most direct role in regulating dl localization. We have examined the effects of Toll on dl in cotransfected Schneider cells to gain insight into the nature of the interaction between these proteins. We have found that Toll can enhance the nuclear localization of dl and, independently, the ability of dl to activate transcription once in the nucleus. We present evidence that the signaling pathway from Toll to dl involves protein kinase A (PKA) and that nuclear transport and activation of dl results from phosphorylation of dl by PKA. We discuss the significance of these results with respect both to Drosophila embryogenesis and to the regulation of the mammalian transcription factor NF-kappa B.' publications.doi=None publications.firstAuthor='Norris J L' publications.id=1007794 publications.issue='9' publications.journal='Genes Dev.' publications.month='Sep' publications.pages='1654-67' publications.pubMedId='1325392' publications.title='Selective nuclear transport of the Drosophila morphogen dorsal can be established by a signaling pathway involving the transmembrane protein Toll and protein kinase A.' publications.volume='6' publications.year=1992\n",
      "Gene: publications.abstractText='Evolutionary constraints on gene regulatory elements are poorly understood: Little is known about how the strength of transcription factor binding correlates with DNA sequence conservation, and whether transcription factor binding sites can evolve rapidly while retaining their function. Here we use the model of the NFKB/Rel-dependent gene regulation in divergent Drosophila species to examine the hypothesis that the functional properties of authentic transcription factor binding sites are under stronger evolutionary constraints than the genomic background. Using molecular modeling we compare tertiary structures of the Drosophila Rel family proteins Dorsal, Dif, and Relish and demonstrate that their DNA-binding and protein dimerization domains undergo distinct rates of evolution. The accumulated amino acid changes, however, are unlikely to affect DNA sequence recognition and affinity. We employ our recently developed microarray-based experimental platform and principal coordinates statistical analysis to quantitatively and systematically profile DNA binding affinities of three Drosophila Rel proteins to 10,368 variants of the NFKB recognition sequences. We then correlate the evolutionary divergence of gene regulatory regions with differences in DNA binding affinities. Genome-wide analyses reveal a significant increase in the number of conserved Rel binding sites in promoters of developmental and immune genes. Significantly, the affinity of Rel proteins to these sites was higher than to less conserved sites and was maintained by the conservation of the DNA binding site sequence (static conservation) or in some cases despite significantly diverged sequences (dynamic conservation). We discuss how two types of conservation may contribute to the stabilization and optimization of a functional gene regulatory code in evolution.' publications.doi='10.1101/gr.6490707' publications.firstAuthor='Copley Richard R' publications.id=1007785 publications.issue='9' publications.journal='Genome Res.' publications.month='Sep' publications.pages='1327-35' publications.pubMedId='17785540' publications.title='Functional conservation of Rel binding sites in drosophilid genomes.' publications.volume='17' publications.year=2007\n",
      "Gene: publications.abstractText='Extensive genetic and biochemical analysis of Drosophila melanogaster has made this system an important model for characterization of transcriptional regulatory elements and factors. Given the striking conservation of transcriptional controls in metazoans, general principles derived from studies of Drosophila are expected to continue to illuminate transcriptional regulation in other systems, including vertebrates. With improvement in technologies for genetic manipulation of insects, research in Drosophila will also aid the design of systems for controlled expression of genes in other hosts. This review focuses on recent advances from Drosophila in analysis of the functional components of transcriptional switches, including basal promoters, enhancers, boundary elements, and maintenance elements.' publications.doi=None publications.firstAuthor='Arnosti D N' publications.id=1005487 publications.issue='10' publications.journal='Insect Biochem. Mol. Biol.' publications.month='Oct' publications.pages='1257-73' publications.pubMedId='12225917' publications.title='Design and function of transcriptional switches in Drosophila.' publications.volume='32' publications.year=2002\n",
      "Gene: publications.abstractText='Formation of the dorsoventral axis in Drosophila melanogaster is mediated through control of the expression of several genes by the morphogen Dorsal. In the ventral part of the embryo Dorsal activates twist and represses zen amongst others. Recently, several proteins have been shown to assist Dorsal in the repression of zen, one of which is DSP1, a HMG box protein that was isolated as a putative co-repressor of Dorsal. In this report we used a DSP1 null mutant to ascertain in vivo the involvement of DSP1 in Dorsal-mediated repression of zen but not in the activation of twist. We show that Dorsal has the ability to interact with DSP1 in vitro as well as with rat HMG1. Using truncated versions of the proteins we located the domains of interaction as being the HMG boxes for DSP1 and HMG1 and the Rel domain for Dorsal. Finally, studies of the zen DNA binding properties of Dorsal and another related Rel protein (Gambif1 from Anopheles gambiae) revealed that their DNA binding affinities were increased in the presence of DSP1 and HMG1.' publications.doi=None publications.firstAuthor='Decoville M' publications.id=1007857 publications.issue='2' publications.journal='Nucleic Acids Res.' publications.month='Jan' publications.pages='454-62' publications.pubMedId='10606643' publications.title='HMG boxes of DSP1 protein interact with the rel homology domain of transcription factors.' publications.volume='28' publications.year=2000\n",
      "Gene: publications.abstractText='Functional assays in Drosophila melanogaster with orthologous transcription factors from other species suggest that changes in the protein-coding sequence may play a larger role in the evolution of transcription factor pathways than was previously believed. Interestingly, recent studies provide evidence that changes in transcription factor protein sequence can affect the regulation of only a subset of target genes, even in the same cells of a developing animal.' publications.doi=None publications.firstAuthor='Hsia Cheryl C' publications.id=1007711 publications.issue='2' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Apr' publications.pages='199-206' publications.pubMedId='12672498' publications.title='Evolution of transcription factor function.' publications.volume='13' publications.year=2003\n",
      "Gene: publications.abstractText='Functional studies seem now to confirm, as first suggested by E. Geoffroy Saint-Hilaire in 1822, that there was an inversion of the dorsoventral axis during animal evolution. A conserved system of extracellular signals provides positional information for the allocation of embryonic cells to specific tissue types both in Drosophila and vertebrates; the ventral region of Drosophila is homologous to the dorsal side of the vertebrate. Developmental studies are now revealing some of the characteristics of the ancestral animal that gave rise to the arthropod and mammalian lineages, for which we propose the name Urbilateria.' publications.doi='10.1038/380037a0' publications.firstAuthor='De Robertis E M' publications.id=1007772 publications.issue='6569' publications.journal='Nature' publications.month='Mar' publications.pages='37-40' publications.pubMedId='8598900' publications.title='A common plan for dorsoventral patterning in Bilateria.' publications.volume='380' publications.year=1996\n",
      "Gene: publications.abstractText='Genetic screens in Drosophila melanogaster have helped elucidate the process of axis formation during early embryogenesis. Axis formation in the D. melanogaster embryo involves the use of two fundamentally different mechanisms for generating morphogenetic activity: patterning the anteroposterior axis by diffusion of a transcription factor within the syncytial embryo and specification of the dorsoventral axis through a signal transduction cascade. Identification of Drosophila genes involved in axis formation provides a launch-pad for comparative studies that examine the evolution of axis specification in different insects. Additionally, there is similarity between axial patterning mechanisms elucidated genetically in Drosophila and those demonstrated for chordates such as Xenopus. In this review we examine the postfertilization mechanisms underlying axis specification in Drosophila. Comparative data are then used to ask whether aspects of axis formation might be derived or ancestral.' publications.doi='10.1146/annurev.genet.35.102401.090832' publications.firstAuthor='Lall S' publications.id=1002649 publications.issue=None publications.journal='Annu. Rev. Genet.' publications.month=None publications.pages='407-37' publications.pubMedId='11700289' publications.title='Conservation and divergence in molecular mechanisms of axis formation.' publications.volume='35' publications.year=2001\n",
      "Gene: publications.abstractText='Genetic studies have identified numerous sequence-specific transcription factors that control development, yet little is known about their in vivo distribution across animal genomes. We determined the genome-wide occupancy of the dorsoventral (DV) determinants Dorsal, Twist, and Snail in the Drosophila embryo using chromatin immunoprecipitation coupled with microarray analysis (ChIP-chip). The in vivo binding of these proteins correlate tightly with the limits of known enhancers. Our analysis predicts substantially more target genes than previous estimates, and includes Dpp signaling components and anteroposterior (AP) segmentation determinants. Thus, the ChIP-chip data uncover a much larger than expected regulatory network, which integrates diverse patterning processes during development.' publications.doi='10.1101/gad.1509607' publications.firstAuthor='Zeitlinger Julia' publications.id=1007897 publications.issue='4' publications.journal='Genes Dev.' publications.month='Feb' publications.pages='385-90' publications.pubMedId='17322397' publications.title='Whole-genome ChIP-chip analysis of Dorsal, Twist, and Snail suggests integration of diverse patterning processes in the Drosophila embryo.' publications.volume='21' publications.year=2007\n",
      "Gene: publications.abstractText='Genetic variation in brain size may provide the basis for the evolution of the brain and complex behaviours. The genetic substrate and the selective pressures acting on brain size are poorly understood. Here we use the Drosophila Genetic Reference Panel to map polymorphic variants affecting natural variation in mushroom body morphology. We identify 139 genes and 39 transcription factors and confirm effects on development and adult plasticity. We show correlations between morphology and aggression, sleep and lifespan. We propose that natural variation in adult brain size is controlled by interaction of the environment with gene networks controlling development and plasticity.' publications.doi='10.1038/ncomms10115' publications.firstAuthor='Zwarts Liesbeth' publications.id=1000539 publications.issue=None publications.journal='Nat Commun' publications.month=None publications.pages='10115' publications.pubMedId='26656654' publications.title='The genetic basis of natural variation in mushroom body size in Drosophila melanogaster.' publications.volume='6' publications.year=2015\n",
      "Gene: publications.abstractText='Genetics and molecular analyses have combined to yield insights into a functional cascade of transcription factors necessary to establish the molecular blueprint of the Drosophila body pattern in response to positional information in the egg. Recent progress in this field raises exciting questions regarding the molecular mechanisms involved, and their conservation in biological pattern-forming processes.' publications.doi=None publications.firstAuthor='Jäckle H' publications.id=1007744 publications.issue='3' publications.journal='Curr. Opin. Cell Biol.' publications.month='Jun' publications.pages='505-12' publications.pubMedId='8352969' publications.title='Transcriptional cascades in Drosophila.' publications.volume='5' publications.year=1993\n",
      "Gene: publications.abstractText='Genome sequencesfor most metazoans and plants are incomplete because of the presence of repeated DNA in the heterochromatin. The heterochromatic regions of Drosophila melanogaster contain 20 million bases (Mb) of sequence amenable to mapping, sequence assembly, and finishing. We describe the generation of 15 Mb of finished or improved heterochromatic sequence with the use of available clone resources and assembly methods. We also constructed a bacterial artificial chromosome-based physical map that spans 13 Mb of the pericentromeric heterochromatin and a cytogenetic map that positions 11 Mb in specific chromosomal locations. We have approached a complete assembly and mapping of the nonsatellite component of Drosophila heterochromatin. The strategy we describe is also applicable to generating substantially more information about heterochromatin in other species, including humans.' publications.doi='10.1126/science.1139816' publications.firstAuthor='Hoskins Roger A' publications.id=20254265 publications.issue='5831' publications.journal='Science' publications.month='Jun' publications.pages='1625-8' publications.pubMedId='17569867' publications.title='Sequence finishing and mapping of Drosophila melanogaster heterochromatin.' publications.volume='316' publications.year=2007\n",
      "Gene: publications.abstractText='Gradients of morphogens determine cell fates by specifying discrete thresholds of gene activities. In the Drosophila embryo, a BMP gradient subdivides the dorsal ectoderm into amnioserosa and dorsal epidermis, and also inhibits neuroectoderm formation. A number of genes are differentially expressed in response to the gradient, but how their borders of expression are established is not well understood. We present evidence that the BMP gradient, via the Smads, provides a two-fold input in regulating the amnioserosa-specific target genes such as Race. Peak levels of Smads in the presumptive amnioserosa set the expression domain of zen, and then Smads act in combination with Zen to directly activate Race. This situation resembles a feed-forward mechanism of transcriptional regulation. In addition, we demonstrate that ectopically expressed Zen can activate targets like Race in the presence of low level Smads, indicating that the role of the highest activity of the BMP gradient is to activate zen.' publications.doi='10.1242/dev.01722' publications.firstAuthor='Xu Mu' publications.id=1007761 publications.issue='7' publications.journal='Development' publications.month='Apr' publications.pages='1637-47' publications.pubMedId='15728670' publications.title='Peak levels of BMP in the Drosophila embryo control target genes by a feed-forward mechanism.' publications.volume='132' publications.year=2005\n",
      "Gene: publications.abstractText='Groucho (Gro) is a Drosophila corepressor required by numerous DNA-binding repressors, many of which are distributed in gradients and provide positional information during development. Gro contains well-conserved domains at its N- and C-termini, and a poorly conserved central region that includes the GP, CcN, and SP domains. All lethal point mutations in gro map to the conserved regions, leading to speculation that the unconserved central domains are dispensable. However, our sequence analysis suggests that the central domains are disordered leading us to suspect that the lack of lethal mutations in this region reflects a lack of order rather than an absence of essential functions. In support of this conclusion, genomic rescue experiments with Gro deletion variants demonstrate that the GP and CcN domains are required for viability. Misexpression assays using these same deletion variants show that the SP domain prevents unrestrained and promiscuous repression by Gro, while the GP and CcN domains are indispensable for repression. Deletion of the GP domain leads to loss of nuclear import, while deletion of the CcN domain leads to complete loss of repression. Changes in Gro activity levels reset the threshold concentrations at which graded repressors silence target gene expression. We conclude that co-regulators such as Gro are not simply permissive components of the repression machinery, but cooperate with graded DNA-binding factors in setting borders of gene expression. We suspect that disorder in the Gro central domains may provide the flexibility that allows this region to mediate multiple interactions required for repression.' publications.doi='10.1371/journal.pone.0030610' publications.firstAuthor='Turki-Judeh Wiam' publications.id=1007886 publications.issue='2' publications.journal='PLoS ONE' publications.month=None publications.pages='e30610' publications.pubMedId='22319573' publications.title='The unconserved groucho central region is essential for viability and modulates target gene specificity.' publications.volume='7' publications.year=2012\n",
      "Gene: publications.abstractText='HOT (highly occupied target) regions bound by many transcription factors are considered to be one of the most intriguing findings of the recent modENCODE reports, yet their functions have remained unclear. We tested 108 Drosophila melanogaster HOT regions in transgenic embryos with site-specifically integrated transcriptional reporters. In contrast to prior expectations, we found 102 (94%) to be active enhancers during embryogenesis and to display diverse spatial and temporal patterns, reminiscent of expression patternsfor important developmental genes. Remarkably, HOT regions strongly activate nearby genes and are required for endogenous gene expression, as we show using bacterial artificial chromosome (BAC) transgenesis. HOT enhancers have a distinct cis-regulatory signature with enriched sequence motifs for the global activators Vielfaltig, also known as Zelda, and Trithorax-like, also known as GAGA. This signature allows the prediction of HOT versus control regions from the DNA sequence alone.' publications.doi='10.1101/gad.188052.112' publications.firstAuthor='Kvon Evgeny Z' publications.id=1007709 publications.issue='9' publications.journal='Genes Dev.' publications.month='May' publications.pages='908-13' publications.pubMedId='22499593' publications.title='HOT regions function as patterned developmental enhancers and have a distinct cis-regulatory signature.' publications.volume='26' publications.year=2012\n",
      "Gene: publications.abstractText='Here, we describe one of the major maternal regulatory gradients, Dorsal, and threshold outputs of gene expression that result from the graded distribution of this transcription factor. The analysis of a large number of authentic and synthetic target genes suggests that the Dorsal gradient directly specifies at least four, and possibly as many as seven, different thresholds of gene activity and tissue differentiation. These thresholds initiate the differentiation of the three primary embryonic tissues: the mesoderm, neurogenic ectoderm, and dorsal ectoderm. Moreover, primary readouts of the Dorsal gradient create asymmetries that subdivide each tissue into multiple cell types during gastrulation. Dorsal patterning thresholds represent the culmination of one of the most complete gene regulation network known in development, which begins with the asymmetric positioning of the oocyte nucleus within the egg chamber and leads to the localized activation of the Toll-Dorsal signaling pathway in ventral regions of the early embryo.' publications.doi='10.1006/dbio.2002.0652' publications.firstAuthor='Stathopoulos Angelike' publications.id=1002707 publications.issue='1' publications.journal='Dev. Biol.' publications.month='Jun' publications.pages='57-67' publications.pubMedId='12027434' publications.title='Dorsal gradient networks in the Drosophila embryo.' publications.volume='246' publications.year=2002\n",
      "Gene: publications.abstractText='Holometabolous insects like Drosophila proceed through two phases of visual system development. The embryonic phase generates simple eyes of the larva. The postembryonic phase produces the adult specific compound eyes during late larval development and pupation. In primitive insects, by contrast, eye development persists seemingly continuously from embryogenesis through the end of postembryogenesis. Comparative literature suggests that the evolutionary transition from continuous to biphasic eye development occurred via transient developmental arrest. This review investigates how the developmental arrest model relates to the gene networks regulating larval and adult eye development in Drosophila, and embryonic compound eye development in primitive insects. Consistent with the developmental arrest model, the available data suggest that the determination of the anlage of the rudimentary Drosophila larval eye is homologous to the embryonic specification of the juvenile compound eye in directly developing insects while the Drosophila compound eye primordium is evolutionarily related to the yet little studied stem cell based postembryonic eye primordium of primitive insects.' publications.doi='10.1016/j.ydbio.2006.08.027' publications.firstAuthor='Friedrich Markus' publications.id=1007757 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Nov' publications.pages='310-29' publications.pubMedId='16973149' publications.title='Continuity versus split and reconstitution: exploring the molecular developmental corollaries of insect eye primordium evolution.' publications.volume='299' publications.year=2006\n",
      "Gene: publications.abstractText='Homeobox genes encode important developmental control proteins. The Drosophila fruit fly HOM complex genes are clustered in region 84-89 of chromosome 3. Probably due to large-scale genome duplication events, their human HOX orthologs belong to four paralogous regions. A series of 13 other homeobox genes are also clustered in region 88-94, on the same chromosome of Drosophila. We suggest that they also duplicated during vertebrate evolution and belong to paralogous regions in humans. These regions are on chromosome arms 4p, 5q, 10q, and 2p or 8p. We coined the term \"paralogon\"to designate paralogous regions in general. We propose to call these genes \"meta Hox\"genes. Like Hox genes, metaHox genes are present in one cluster in Drosophila and four clusters (metaHox A-D) in humans on the 4p/5q/10q paralogon.' publications.doi='10.1002/1097-010X(20001215)288:4<345::AID-JEZ7>3.0.CO;2-Y' publications.firstAuthor='Coulier F' publications.id=1007417 publications.issue='4' publications.journal='J. Exp. Zool.' publications.month='Dec' publications.pages='345-51' publications.pubMedId='11144283' publications.title='MetaHox gene clusters.' publications.volume='288' publications.year=2000\n",
      "Gene: publications.abstractText='Homeotic (Hox) genes are usually clustered and arranged in the same order as they are expressed along the anteroposterior body axis of metazoans. The mechanistic explanation for this colinearity has been elusive, and it may well be that a single and universal cause does not exist. The Hox-gene complex (HOM-C) has been rearranged differently in several Drosophila species, producing a striking diversity of Hox gene organizations. We investigated the genomic and functional consequences of the two HOM-C splits present in Drosophila buzzatii. Firstly, we sequenced two regions of the D. buzzatii genome, one containing the genes labial and abdominal A, and another one including proboscipedia, and compared their organization with that of D. melanogaster and D. pseudoobscura in order to map precisely the two splits. Then, a plethora of conserved noncoding sequences, which are putative enhancers, were identified around the three Hox genes closer to the splits. The position and order of these enhancers are conserved, with minor exceptions, between the three Drosophila species. Finally, we analyzed the expression patterns of the same three genes in embryos and imaginal discs of four Drosophila species with different Hox-gene organizations. The results show that their expression patterns are conserved despite the HOM-C splits. We conclude that, in Drosophila, Hox-gene clustering is not an absolute requirement for proper function. Rather, the organization of Hox genes is modular, and their clustering seems the result of phylogenetic inertia more than functional necessity.' publications.doi='10.1101/gr.3468605' publications.firstAuthor='Negre Bárbara' publications.id=1007635 publications.issue='5' publications.journal='Genome Res.' publications.month='May' publications.pages='692-700' publications.pubMedId='15867430' publications.title='Conservation of regulatory sequences and gene expression patterns in the disintegrating Drosophila Hox gene complex.' publications.volume='15' publications.year=2005\n",
      "Gene: publications.abstractText='Homeotic genes identify structures along the anterior to posterior axis during the development of most animals. These genes are clustered into complexes, and their positions within the cluster correlates with their time of expression and the positions of the anterioposterior boundaries of their expression domains. Functional analyses have revealed that this specific genetic order also coincides with a functional hierarchy among members of these complexes, so that the products of more posterior genes in the cluster tend to be prevalent over those of more anterior genes.' publications.doi=None publications.firstAuthor='Duboule D' publications.id=1007742 publications.issue='10' publications.journal='Trends Genet.' publications.month='Oct' publications.pages='358-64' publications.pubMedId='7985240' publications.title='Colinearity and functional hierarchy among genes of the homeotic complexes.' publications.volume='10' publications.year=1994\n",
      "Gene: publications.abstractText='Identifying the genomic regions bound by sequence-specific regulatory factors is central both to deciphering the complex DNA cis-regulatory code that controls transcription in metazoans and to determining the range of genes that shape animal morphogenesis. We used whole-genome tiling arrays to map sequences bound in Drosophila melanogaster embryos by the six maternal and gap transcription factors that initiate anterior-posterior patterning. We find that these sequence-specific DNA binding proteins bind with quantitatively different specificities to highly overlapping sets of several thousand genomic regions in blastoderm embryos. Specific high- and moderate-affinity in vitro recognition sequences for each factor are enriched in bound regions. This enrichment, however, is not sufficient to explain the pattern of binding in vivo and varies in a context-dependent manner, demonstrating that higher-order rules must govern targeting of transcription factors. The more highly bound regions include all of the over 40 well-characterized enhancers known to respond to these factors as well as several hundred putative new cis-regulatory modules clustered near developmental regulators and other genes with patterned expression at this stage of embryogenesis. The new targets include most of the microRNAs (miRNAs) transcribed in the blastoderm, as well as all major zygotically transcribed dorsal-ventral patterning genes, whose expression we show to be quantitatively modulated by anterior-posterior factors. In addition to these highly bound regions, there are several thousand regions that are reproducibly bound at lower levels. However, these poorly bound regions are, collectively, far more distant from genes transcribed in the blastoderm than highly bound regions; are preferentially found in protein-coding sequences; and are less conserved than highly bound regions. Together these observations suggest that many of these poorly bound regions are not involved in early-embryonic transcriptional regulation, and a significant proportion may be nonfunctional. Surprisingly, for five of the six factors, their recognition sites are not unambiguously more constrained evolutionarily than the immediate flanking DNA, even in more highly bound and presumably functional regions, indicating that comparative DNA sequence analysis is limited in its ability to identify functional transcription factor targets.' publications.doi='10.1371/journal.pbio.0060027' publications.firstAuthor='Li Xiao-yong' publications.id=1007729 publications.issue='2' publications.journal='PLoS Biol.' publications.month='Feb' publications.pages='e27' publications.pubMedId='18271625' publications.title='Transcription factors bind thousands of active and inactive regions in the Drosophila blastoderm.' publications.volume='6' publications.year=2008\n",
      "Gene: publications.abstractText='In Drosophila elaboration of positional identity along the anterior-posterior and dorsal-ventral embryonic body axes involves early zygotic gene functions that are expressed in response to maternal cues present in the unfertilized egg. Zygotic loci that are required for the specification of positional identity along the anterior-posterior body axis have been described in detail. Less is known about the zygotic loci responsible for differentiation of the dorsal-ventral pattern; however, several genes that might be involved have been identified. Zerknüllt (zen) is an example of a zygotic gene required for correct differentiation of dorsally derived embryonic tissues. On the basis of homoeo box cross homology, we have now isolated a gene, called S60, that derives from the zen region of the Antennapedia complex (ANT-C). Transcripts encoded by S60 transiently accumulate in the dorsal-most tissues of developing embryos. This pattern of expression suggests that S60 corresponds to zen. Since S60 contains a homoeo box, it is possible that differentiation of the anterior-posterior and dorsal-ventral embryonic patterns involves similar molecular mechanisms.' publications.doi='10.1038/323076a0' publications.firstAuthor='Doyle H J' publications.id=1007847 publications.issue='6083' publications.journal='Nature' publications.month=None publications.pages='76-9' publications.pubMedId='3755802' publications.title='Transcripts encoded by a homoeo box gene are restricted to dorsal tissues of Drosophila embryos.' publications.volume='323' publications.year=None\n",
      "Gene: publications.abstractText=\"In Drosophila embryos, dorsal-ventral polarity is defined by a signal transduction pathway that regulates nuclear import ofthe Dorsal protein. Dorsal protein's ability to act as a transcriptional activator of some zygotic genes and a repressor of others defines structure along the dorsal-ventral axis. Dorsal is a member of a group of proteins, the Rel-homologous proteins, whose activity is regulated at the level of nuclear localization. Dif, a more recently identified Drosophila Rel-homologue, has been proposed to act as a mediator of the immune response in Drosophila. In an effort to understand the function and regulation of Rel-homologous proteins in Drosophila, we have expressed Dif protein in Drosophila embryos derived from dorsal mutant mothers. We found that the Dif protein was capable of restoring embryonic dorsal-ventral pattern elements and was able to define polarity correctly with respect to the orientation of the egg shell. This, together with the observation that the ability of Dif to restore a dorsal-ventral axis depended on the signal transduction pathway that normally regulates Dorsal, suggests that Dif protein formed a nuclear concentration gradient similar to that seen for Dorsal. By studying the expression of Dorsal target genes we found that Dif could activate the zygotic genes that Dorsal activates and repress the genes repressed by Dorsal. Differences in the expression of these target genes, as well as the results from interaction studies carried out in yeast, suggest that Dif is not capable of synergizing with the basic helix-loop-helix transcription factors with which Dorsal normally interacts, and thereby lacks an important component of Dorsal-mediated pattern formation.\" publications.doi=None publications.firstAuthor='Stein D' publications.id=1007747 publications.issue='11' publications.journal='Development' publications.month='Jun' publications.pages='2159-69' publications.pubMedId='9570779' publications.title='The Dorsal-related immunity factor (Dif) can define the dorsal-ventral axis of polarity in the Drosophila embryo.' publications.volume='125' publications.year=1998\n",
      "Gene: publications.abstractText='In Drosophila, a Bcd protein gradient orchestrates patterning along the anteroposterior embryonic axis. However, studies of basal flies and other insects have revealed that bcd is a derived Hox3 gene found only in higher dipterans. To understand how bcd acquired its role in flies and how anteroposterior patterning mechanisms have evolved, I first review key features of bcd function in Drosophila: anterior localization and transcriptional and translation control of gene expression. I then discuss investigations of bcd in other higher dipterans that have provided insight into the evolution of regulatory interactions and the Bcd gradient. Finally, I review studies of Drosophila and other insects that address the evolution of bcd function and integration of bcd into ancestral regulatory mechanisms. I suggest further comparative studies may allow us to identify the intermediate steps in bcd evolution. This will make bcd a paradigm for the origin and evolution of genes and regulatory networks.' publications.doi='10.1002/bies.20285' publications.firstAuthor='McGregor Alistair P' publications.id=1007696 publications.issue='9' publications.journal='Bioessays' publications.month='Sep' publications.pages='904-13' publications.pubMedId='16108065' publications.title='How to get ahead: the origin, evolution and function of bicoid.' publications.volume='27' publications.year=2005\n",
      "Gene: publications.abstractText='In Drosophila, pattern formation at multiple stages of embryonic and imaginal development depends on the same intercellular signaling pathways. We have identified a novel gene, eyelid (eld), which is required for embryonic segmentation, development of the notum and wing margin, and photoreceptor differentiation. In these tissues, eld mutations have effects opposite to those caused by wingless (wg) mutations. eld encodes a widely expressed nuclear protein with a region homologous to a novel family of DNA-binding domains. Based on this homology and on the phenotypic analysis, we suggest that Eld could act as a transcription factor antagonistic to the Wg pathway.' publications.doi=None publications.firstAuthor='Treisman J E' publications.id=1007905 publications.issue='15' publications.journal='Genes Dev.' publications.month='Aug' publications.pages='1949-62' publications.pubMedId='9271118' publications.title='eyelid antagonizes wingless signaling during Drosophila development and has homology to the Bright family of DNA-binding proteins.' publications.volume='11' publications.year=1997\n",
      "Gene: publications.abstractText='In Drosophila, the dorsal (dl) morphogen gradient initiates the differentiation of the embryonic mesoderm and neuroectoderm by activating the expression of regulatory genes (e.g. twist and snail) in a concentration-dependent manner. dl also functions as a repressor that establishes the dorsal epidermis and amnioserosa by restricting regulatory genes such as dpp and zen to dorsal regions of the embryo. The ability of dl to function as both an activator and repressor distinguishes it from the bicoid morphogen, which appears to function solely as an activator. In an effort to determine how dl functions as a repressor we have performed a detailed characterization of a zen silencer element, called the VRE, which mediates ventral repression in response to the dl gradient. A minimal 110 bp VRE sequence is identified, which is able to silence the ventral expression of a heterologous promoter. This sequence contains two dl binding sites as well as binding sites for additional nuclear factors present in early embryos. Mutations in the latter binding sites convert the minimal VRE into an enhancer, which mediates transcriptional activation in ventral regions in response to dl. These results suggest that dl is intrinsically an activator, but is converted into a potent silencer when it interacts with neighboring corepressors.' publications.doi=None publications.firstAuthor='Jiang J' publications.id=1007846 publications.issue='8' publications.journal='EMBO J.' publications.month='Aug' publications.pages='3201-9' publications.pubMedId='8344257' publications.title='Conversion of a dorsal-dependent silencer into an enhancer: evidence for dorsal corepressors.' publications.volume='12' publications.year=1993\n",
      "Gene: publications.abstractText='In Drosophila, two TGF-beta growth factors, dpp and screw, function synergistically to subdivide the dorsal ectoderm into two embryonic tissues, the amnioserosa and dorsal epidermis. Previous studies have shown that peak dpp activity is required for the localized expression of zerknüllt (zen), which encodes a homeodomain transcription factor. We present evidence that zen directly activates the amnioserosa-specific expression of a downstream target gene, Race (Related to angiotensin converting enzyme). A 533 bp enhancer from the Race promoter region is shown to mediate selective expression in the amnioserosa, as well as the anterior and posterior midgut rudiments. This enhancer contains three zen protein binding sites, and mutations in these sites virtually abolish the expression of an otherwise normal Race-lacZ fusion gene in the amnioserosa, but not in the gut. Genetic epistasis experiments suggest that zen is not the sole activator of Race, although a hyperactivated form of zen (a zen-VP16 fusion protein) can partially complement reduced levels of dpp activity. These results suggest that dpp regulates multiple transcription factors, which function synergistically to specify the amnioserosa.' publications.doi=None publications.firstAuthor='Rusch J' publications.id=1004030 publications.issue='2' publications.journal='Development' publications.month='Jan' publications.pages='303-11' publications.pubMedId='9053307' publications.title='Regulation of a dpp target gene in the Drosophila embryo.' publications.volume='124' publications.year=1997\n",
      "Gene: publications.abstractText='In a developing Drosophila melanogaster embryo, mRNAs have a maternal origin, a zygotic origin, or both. During the maternal-zygotic transition, maternal products are degraded and gene expression comes under the control of the zygotic genome. To interrogate the function of mRNAs that are both maternally and zygotically expressed, it is common to examine the embryonic phenotypes derived from female germline mosaics. Recently, the development of RNAi vectors based on short hairpin RNAs (shRNAs) effective during oogenesis has provided an alternative to producing germline clones. Here, we evaluate the efficacies of: (1) maternally loaded shRNAs to knockdown zygotic transcripts and (2) maternally loaded Gal4 protein to drive zygotic shRNA expression. We show that, while Gal4-driven shRNAs in the female germline very effectively generate phenotypes for genes expressed maternally, maternally loaded shRNAs are not very effective at generating phenotypes for early zygotic genes. However, maternally loaded Gal4 protein is very efficient at generating phenotypes for zygotic genes expressed during mid-embryogenesis. We apply this powerful and simple method to unravel the embryonic functions of a number of pleiotropic genes.' publications.doi='10.1534/genetics.112.144915' publications.firstAuthor='Staller Max V' publications.id=1007903 publications.issue='1' publications.journal='Genetics' publications.month='Jan' publications.pages='51-61' publications.pubMedId='23105012' publications.title='Depleting gene activities in early Drosophila embryos with the \"maternal-Gal4-shRNA\"system.' publications.volume='193' publications.year=2013\n",
      "Gene: publications.abstractText='In a screen based on a rough eye phenotype caused by a dominant negative form of the BEAF-32A and BEAF-32B insulator proteins, we previously identified 17 proteins that genetically interact with BEAF. Eleven of these are developmental transcription factors, seven of which are encoded by the Antennapedia complex (ANT-C). While investigating potential reasons for the genetic interactions, we obtained evidence that BEAF plays a role in the regulation of genes in the ANT-C. BEAF does not localize near the transcription start sites of any genes in the ANT-C, indicating that BEAF does not locally affect regulation of these genes. Although BEAF affects chromatin structure or dynamics, we also found no evidence for a general change in binding to polytene chromosomes in the absence of BEAF. However, because we were unable to detect proteins encoded by ANT-C genes in salivary glands, the DREF and MLE proteins were used as proxies to examine binding. This does not rule out limited effects at particular binding sites or the possibility that BEAF might directly interact with certain transcription factors to affect their binding. In contrast, the embryonic expression levels and patterns of four examined ANT-C genes were altered (bcd, Dfd, ftz, pb). A control gene, Dref, was not affected. A full understanding of the regulation of ANT-C genes during development will have to take the role of BEAF into account.' publications.doi='10.1007/s00438-010-0591-y' publications.firstAuthor='Roy Swarnava' publications.id=1005511 publications.issue='2' publications.journal='Mol. Genet. Genomics' publications.month='Feb' publications.pages='113-23' publications.pubMedId='21132442' publications.title='Lack of the Drosophila BEAF insulator proteins alters regulation of genes in the Antennapedia complex.' publications.volume='285' publications.year=2011\n",
      "Gene: publications.abstractText='In all animals, the initial events of embryogenesis are controlled by maternal gene products that are deposited into the developing oocyte. At some point after fertilization, control of embryogenesis is transferred to the zygotic genome in a process called the maternal-to-zygotic transition. During this time, many maternal RNAs are degraded and transcription of zygotic RNAs ensues. There is a long-standing question as to which factors regulate these events. The recent findings that microRNAs and Smaug mediate maternal transcript degradation have shed new light on this aspect of the problem. However, the transcription factor(s) that activate the zygotic genome remain elusive. The discovery that many of the early transcribed genes in Drosophila share a cis-regulatory heptamer motif, CAGGTAG and related sequences, collectively referred to as TAGteam sites raised the possibility that a dedicated transcription factor could interact with these sites to activate transcription. Here we report that the zinc-finger protein Zelda (Zld; Zinc-finger early Drosophila activator) binds specifically to these sites and is capable of activating transcription in transient transfection assays. Mutant embryos lacking zld are defective in cellular blastoderm formation, and fail to activate many genes essential for cellularization, sex determination and pattern formation. Global expression profiling confirmed that Zld has an important role in the activation of the early zygotic genome and suggests that Zld may also regulate maternal RNA degradation during the maternal-to-zygotic transition.' publications.doi='10.1038/nature07388' publications.firstAuthor='Liang Hsiao-Lan' publications.id=1007807 publications.issue='7220' publications.journal='Nature' publications.month='Nov' publications.pages='400-3' publications.pubMedId='18931655' publications.title='The zinc-finger protein Zelda is a key activator of the early zygotic genome in Drosophila.' publications.volume='456' publications.year=2008\n",
      "Gene: publications.abstractText='In many developmental contexts, a locally produced morphogen specifies positional information by forming a concentration gradient over a field of cells. However, during embryonic dorsal-ventral patterning in Drosophila, two members of the bone morphogenetic protein (BMP) family, Decapentaplegic (Dpp) and Screw (Scw), are broadly transcribed but promote receptor-mediated signalling in a restricted subset of expressing cells. Here we use a novel immunostaining protocol to visualize receptor-bound BMPs and show that both proteins become localized to a sharp stripe of dorsal cells. We demonstrate that proper BMP localization involves two distinct processes. First, Dpp undergoes directed, long-range extracellular transport. Scw also undergoes long-range movement, but can do so independently of Dpp transport. Second, an intracellular positive feedback circuit promotes future ligand binding as a function of previous signalling strength. These data elicit a model in which extracellular Dpp transport initially creates a shallow gradient of BMP binding that is acted on by positive intracellular feedback to produce two stable states of BMP-receptor interactions, a spatial bistability in which BMP binding and signalling capabilities are high in dorsal-most cells and low in lateral cells.' publications.doi='10.1038/nature03318' publications.firstAuthor='Wang Yu-Chiun' publications.id=1007763 publications.issue='7030' publications.journal='Nature' publications.month='Mar' publications.pages='229-34' publications.pubMedId='15759004' publications.title='Spatial bistability of Dpp-receptor interactions during Drosophila dorsal-ventral patterning.' publications.volume='434' publications.year=2005\n",
      "Gene: publications.abstractText='In many organisms, transcription of the zygotic genome begins during the maternal-to-zygotic transition (MZT), which is characterized by a dramatic increase in global transcriptional activities and coincides with embryonic stem cell differentiation. In Drosophila, it has been shown that maternal morphogen gradients and ubiquitously distributed general transcription factors may cooperate to upregulate zygotic genes that are essential for pattern formation in the early embryo. Here, we show that Drosophila STAT (STAT92E) functions as a general transcription factor that, together with the transcription factor Zelda, induces transcription of a large number of early-transcribed zygotic genes during the MZT. STAT92E is present in the early embryo as a maternal product and is active around the MZT. DNA-binding motifs for STAT and Zelda are highly enriched in promoters of early zygotic genes but not in housekeeping genes. Loss of Stat92E in the early embryo, similarly to loss of zelda, preferentially down-regulates early zygotic genes important for pattern formation. We further show that STAT92E and Zelda synergistically regulate transcription. We conclude that STAT92E, in conjunction with Zelda, plays an important role in transcription of the zygotic genome at the onset of embryonic development.' publications.doi='10.1371/journal.pgen.1002086' publications.firstAuthor='Tsurumi Amy' publications.id=1007702 publications.issue='5' publications.journal='PLoS Genet.' publications.month='May' publications.pages='e1002086' publications.pubMedId='21637778' publications.title='STAT is an essential activator of the zygotic genome in the early Drosophila embryo.' publications.volume='7' publications.year=2011\n",
      "Gene: publications.abstractText='In past years, much attention has focused on the gene networks that regulate early developmental processes, but less attention has been paid to how multiple networks and processes are temporally coordinated. Recently the discovery of the transcriptional activator Zelda (Zld), which binds to CAGGTAG and related sequences present in the enhancers of many early-activated genes in Drosophila, hinted at a mechanism for how batteries of genes could be simultaneously activated. Here we use genome-wide binding and expression assays to identify Zld target genes in the early embryo with the goal of unraveling the gene circuitry regulated by Zld. We found that Zld binds to genes involved in early developmental processes such as cellularization, sex determination, neurogenesis, and pattern formation. In the absence of Zld, many target genes failed to be activated, while others, particularly the patterning genes, exhibited delayed transcriptional activation, some of which also showed weak and/or sporadic expression. These effects disrupted the normal sequence of patterning-gene interactions and resulted in highly altered spatial expression patterns, demonstrating the significance of a timing mechanism in early development. In addition, we observed prevalent overlap between Zld-bound regions and genomic \"hotspot\"regions, which are bound by many developmental transcription factors, especially the patterning factors. This, along with the finding that the most over-represented motif in hotspots, CAGGTA, is the Zld binding site, implicates Zld in promoting hotspot formation. We propose that Zld promotes timely and robust transcriptional activation of early-gene networks so that developmental events are coordinated and cell fates are established properly in the cellular blastoderm embryo.' publications.doi='10.1371/journal.pgen.1002339' publications.firstAuthor='Nien Chung-Yi' publications.id=1007816 publications.issue='10' publications.journal='PLoS Genet.' publications.month='Oct' publications.pages='e1002339' publications.pubMedId='22028675' publications.title='Temporal coordination of gene networks by Zelda in the early Drosophila embryo.' publications.volume='7' publications.year=2011\n",
      "Gene: publications.abstractText='In situ hybridization is an important technique for measuring the spatial expression patterns of mRNA in cells, tissues, and whole animals. However, mRNA levels cannot be compared across experiments using typical protocols. Here we present a semi-quantitative method to compare mRNA levels of a gene across multiple samples. This method yields an estimate of the error in the measurement to allow statistical comparison. Our method uses a typical in situ hybridization protocol to stain for a target gene and an internal standard, which we refer to as a co-stain. As a proof of concept, we apply this method to multiple lines of transgenic Drosophila embryos, harboring constructs that express reporter genes to different levels. We generated this test set by mutating enhancer sequences to contain different numbers of binding sites for Zelda, a transcriptional activator. We demonstrate that using a co-stain with in situ hybridization is an effective method to compare mRNA levels across samples. This method requires only minor modifications to existing in situ hybridization protocols and uses straightforward analysis techniques. This strategy can be broadly applied to detect quantitative, spatially resolved changes in mRNA levels.' publications.doi='10.1016/j.ymeth.2014.01.003' publications.firstAuthor='Wunderlich Zeba' publications.id=1007778 publications.issue='1' publications.journal='Methods' publications.month='Jun' publications.pages='233-41' publications.pubMedId='24434507' publications.title='Comparing mRNA levels using in situ hybridization of a target gene and co-stain.' publications.volume='68' publications.year=2014\n",
      "Gene: publications.abstractText='In the long-germ insect Drosophila melanogaster dorsoventral polarity is induced by localized Toll-receptor activation which leads to the formation of a nuclear gradient of the rel/ NF-kappaB protein Dorsal. Peak levels of nuclear Dorsal are found in a ventral stripe spanning the entire length of the blastoderm embryo allowing all segments and their dorsoventral subdivisions to be synchronously specified before gastrulation. We show that a nuclear Dorsal protein gradient of similar anteroposterior extension exists in the short-germ beetle, Tribolium castaneum, which forms most segments from a posterior growth zone after gastrulation. In contrast to Drosophila, (i) nuclear accumulation is first uniform and then becomes progressively restricted to a narrow ventral stripe, (ii) gradient refinement is accompanied by changes in the zygotic expression of the Tribolium Toll-receptor suggesting feedback regulation and, (iii) the gradient only transiently overlaps with the expression of a potential target, the Tribolium twist homolog, and does not repress Tribolium decapentaplegic. No nuclear Dorsal is seen in the cells of the growth zone of Tribolium embryos, indicating that here dorsoventral patterning occurs by a different mechanism. However, Dorsal is up-regulated and transiently forms a nuclear gradient in the serosa, a protective extraembryonic cell layer ultimately covering the whole embryo.' publications.doi=None publications.firstAuthor='Chen G' publications.id=1007690 publications.issue='23' publications.journal='Development' publications.month='Dec' publications.pages='5145-56' publications.pubMedId='11060240' publications.title='The maternal NF-kappaB/dorsal gradient of Tribolium castaneum: dynamics of early dorsoventral patterning in a short-germ beetle.' publications.volume='127' publications.year=2000\n",
      "Gene: publications.abstractText='Infection results in the rapid activation of immunity genes in the Drosophila fat body. Two classes of transcription factors have been implicated in this process: the REL-containing proteins, Dorsal, Dif, and Relish, and the GATA factor Serpent. Here we present evidence that REL-GATA synergy plays apervasive role in the immune response. SELEX assays identified consensus binding sites that permitted the characterization of several immunity regulatory DNAs. The distribution of REL and GATA sites within these DNAs suggests that most or all fat-specific immunity genes contain a common organization of regulatory elements: closely linked REL and GATA binding sites positioned in the same orientation and located near the transcription start site. Aspects of this \"regulatory code\"are essential for the immune response. These results suggest that immunity regulatory DNAs contain constrained organizational features, which may be a general property of eukaryotic enhancers.' publications.doi=None publications.firstAuthor='Senger Kate' publications.id=1007774 publications.issue='1' publications.journal='Mol. Cell' publications.month='Jan' publications.pages='19-32' publications.pubMedId='14731391' publications.title='Immunity regulatory DNAs share common organizational features in Drosophila.' publications.volume='13' publications.year=2004\n",
      "Gene: publications.abstractText='It is becoming increasingly clear that transcriptional repression is at least as important as transcriptional activation for establishing cell-type specific patterns of gene expression during embryogenesis. Recent studies in Drosophila suggest that repressors fall into two categories, short-range and long-range repressors. The former permit enhancer autonomy in modular promoters, whereas the latter function in a dominant fashion to silence multiple enhancers.' publications.doi=None publications.firstAuthor='Gray S' publications.id=1007724 publications.issue='3' publications.journal='Curr. Opin. Cell Biol.' publications.month='Jun' publications.pages='358-64' publications.pubMedId='8743887' publications.title='Transcriptional repression in development.' publications.volume='8' publications.year=1996\n",
      "Gene: publications.abstractText='It is expected that genes that are expressed early in development and have a complex expression pattern are under strong purifying selection and thus evolve slowly. Hox genes fulfill these criteria and thus, should have a low evolutionary rate. However, some observations point to a completely different scenario. Hox genes are usually highly conserved inside the homeobox, but very variable outside it. We have measured the rates of nucleotide divergence and indel fixation of three Hox genes, labial (lab), proboscipedia (pb) and abdominal-A (abd-A), and compared them with those of three genes derived by duplication from Hox3, bicoid (bcd), zerknüllt (zen) and zerknüllt-related (zen2), and 15 non-Hox genes in sets of orthologous sequences of three species of the genus Drosophila. These rates were compared to test the hypothesis that Hox genes evolve slowly. Our results show that the evolutionary rate of Hox genes is higher than that of non-Hox genes when both amino acid differences and indels are taken into account: 43.39% of the amino acid sequence is altered in Hox genes, versus 30.97% in non-Hox genes and 64.73% in Hox-derived genes. Microsatellites scattered along the coding sequence of Hox genes explain partially, but not fully, their fast sequence evolution. These results show that Hox genes have a higher evolutionary dynamics than other developmental genes, and emphasize the need to take into account indels in addition to nucleotide substitutions in order to accurately estimate evolutionary rates.' publications.doi='10.1186/1471-2148-6-106' publications.firstAuthor='Casillas Sònia' publications.id=1007632 publications.issue=None publications.journal='BMC Evol. Biol.' publications.month=None publications.pages='106' publications.pubMedId='17163987' publications.title='Fast sequence evolution of Hox and Hox-derived genes in the genus Drosophila.' publications.volume='6' publications.year=2006\n",
      "Gene: publications.abstractText='Like many DNA binding transcription factors, the Drosophila morphogen encoded by dorsal can both stimulate and repress promoter activity. In particular, this factor activates twist and represses zerknüllt on the ventral side of the early embryo. We find that when multiple copies of a dorsal binding site from the zerknüllt ventral repressor element are fused to a heterologous basal promoter, the resulting construct is activated by dorsal to give a ventral specific expression pattern. Thus, the ability of a dorsal binding site to mediate repression rather than activation is not an intrinsic property of the site, but depends upon its context. We also show that a hybrid promoter containing both the zerknüllt ventral repressor element and the twist ventral activator region is not ventrally active in the early embryo, demonstrating that repression is dominant over activation. Thus, the default mode of action of the dorsal protein is transcriptional activation. Additional factors may modify dorsal activity to bring about repression.' publications.doi=None publications.firstAuthor='Pan D' publications.id=1007841 publications.issue='5' publications.journal='EMBO J.' publications.month='May' publications.pages='1837-42' publications.pubMedId='1582412' publications.title='The same dorsal binding site mediates both activation and repression in a context-dependent manner.' publications.volume='11' publications.year=1992\n",
      "Gene: publications.abstractText='Major advances have been made in understanding the evolution of transcriptional regulation using microevolutionary and macroevolutionary experimental approaches. The roles of stabilising selection and compensatory changes in an enhancer region have been elucidated in Drosophila. The molecular dynamics of regulatory alleles have been studied in plants. Evidence is accumulating for the involvement of regulatory evolution in morphological changes between closely related species, as well as in major changes of body plans.' publications.doi=None publications.firstAuthor='Tautz D' publications.id=1007716 publications.issue='5' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Oct' publications.pages='575-9' publications.pubMedId='10980438' publications.title='Evolution of transcriptional regulation.' publications.volume='10' publications.year=2000\n",
      "Gene: publications.abstractText=\"Many proteins of the Rel family can act as both transcriptional activators and repressors. However, mechanism that discerns the 'activator/repressor' functions of Rel-proteins such as Dorsal (Drosophila homologue of mammalian NFκB) is not understood. Using genomic, biophysical and biochemical approaches, we demonstrate that the underlying principle of this functional specificity lies in the 'sequence-encoded structure' of the κB-DNA. We show that Dorsal-binding motifs exist in distinct activator and repressor conformations. Molecular dynamics of DNA-Dorsal complexes revealed that repressor κB-motifs typically have A-tract and flexible conformation that facilitates interaction with co-repressors. Deformable structure of repressor motifs, is due to changes in the hydrogen bonding in A:T pair in the 'A-tract' core. The sixth nucleotide in the nonameric κB-motif, 'A' (A(6)) in the repressor motifs and 'T' (T(6)) in the activator motifs, is critical to confer this functional specificity as A(6)\\u2009→\\u2009T(6) mutation transformed flexible repressor conformation into a rigid activator conformation. These results highlight that 'sequence encoded κB DNA-geometry' regulates gene expression by exerting allosteric effect on binding of Rel proteins which in turn regulates interaction with co-regulators. Further, we identified and characterized putative repressor motifs in Dl-target genes, which can potentially aid in functional annotation of Dorsal gene regulatory network.\" publications.doi='10.1093/nar/gkr672' publications.firstAuthor='Mrinal Nirotpal' publications.id=1007717 publications.issue='22' publications.journal='Nucleic Acids Res.' publications.month='Dec' publications.pages='9574-91' publications.pubMedId='21890896' publications.title='Role of sequence encoded κB DNA geometry in gene regulation by Dorsal.' publications.volume='39' publications.year=2011\n",
      "Gene: publications.abstractText='Massive zygotic transcription begins in many organisms during the midblastula transition when the cell cycle of the dividing egg slows down. A few genes are transcribed before this stage but how this differential activation is accomplished is still an open question. We have performed ChIP-seq experiments on tightly staged Drosophila embryos and show that massive recruitment of RNA polymerase II (Pol II) with widespread pausing occurs de novo during the midblastula transition. However, ∼100 genes are strongly occupied by Pol II before this timepoint and most of them do not show Pol II pausing, consistent with a requirement for rapid transcription during the fast nuclear cycles. This global change in Pol II pausing correlates with distinct core promoter elements and associates a TATA-enriched promoter with the rapid early transcription. This suggests that promoters are differentially used during the zygotic genome activation, presumably because they have distinct dynamic properties. DOI:http://dx.doi.org/10.7554/eLife.00861.001.' publications.doi='10.7554/eLife.00861' publications.firstAuthor='Chen Kai' publications.id=1003294 publications.issue=None publications.journal='Elife' publications.month=None publications.pages='e00861' publications.pubMedId='23951546' publications.title='A global change in RNA polymerase II pausing during the Drosophila midblastula transition.' publications.volume='2' publications.year=2013\n",
      "Gene: publications.abstractText=\"Maternal genes involved in dorsoventral (D/V) patterning of the Drosophila embryo interact to establish a stable nuclear concentration gradient of the Dorsal protein which acts as the morphogen along this axis. This protein belongs to the rel proto-oncogene and NF-KB transcriptional factor family and acts by controlling zygotic gene expression. In the ventral part of the embryo, dorsal specifically activates transcription of the gene twist and ventrally and laterally dorsal represses the expression of zerknüllt, a gene involved in the formation of dorsal derivatives. The extent of dorsal action is closely related to the affinity and the number of dorsal response elements present in these zygotic gene promoters. twist is one of the first zygotic genes necessary for mesoderm formation. It codes for a 'b-HLH' DNA-binding protein which can dimerize and bind to DNA in vitro and to polytene chromosomes in vivo. In addition, in cultured cells twist has been shown to be a transcriptional activator. Thus, the first events of embryonic development along the D/V axis are controlled at the transcriptional level.\" publications.doi=None publications.firstAuthor='Thisse C' publications.id=1007762 publications.issue=None publications.journal='Dev. Suppl.' publications.month=None publications.pages='173-81' publications.pubMedId='1299363' publications.title='Dorsoventral development of the Drosophila embryo is controlled by a cascade of transcriptional regulators.' publications.volume=None publications.year=1992\n",
      "Gene: publications.abstractText='Maternally contributed mRNAs and proteins control the initial stages of development following fertilization. During this time, most of the zygotic genome remains transcriptionally silent. The initiation of widespread zygotic transcription is coordinated with the degradation of maternally provided mRNAs at the maternal-to-zygotic transition (MZT). While most of the genome is silenced prior to the MZT, a small subset of zygotic genes essential for the future development of the organism is transcribed. Previous work in our laboratory and others identified the TAGteam element, a set of related heptameric DNA-sequences in the promoters of many early-expressed Drosophila genes required to drive their unusually early transcription. To understand how this unique subset of genes is regulated, we identified a TAGteam-binding factor Grainyhead (Grh). We demonstrated that Grh and the previously characterized transcriptional activator Zelda (Zld) bind to different TAGteam sequences with varying affinities, and that Grh competes with Zld for TAGteam occupancy. Moreover, overexpression of Grh in the early embryo causes defects in cell division, phenocopying Zld depletion. Our findings indicate that during early embryonic development the precise timing of gene expression is regulated by both the sequence of the TAGteam elements in the promoter and the relative levels of the transcription factors Grh and Zld.' publications.doi='10.1016/j.ydbio.2010.06.026' publications.firstAuthor='Harrison Melissa M' publications.id=1007813 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Sep' publications.pages='248-55' publications.pubMedId='20599892' publications.title='Grainyhead and Zelda compete for binding to the promoters of the earliest-expressed Drosophila genes.' publications.volume='345' publications.year=2010\n",
      "Gene: publications.abstractText='Members of the Eyes absent (Eya) protein family play important roles in tissue specification and patterning by serving as both transcriptional activators and protein tyrosine phosphatases. These activities are often carried out in the context of complexes containing members of the Six and/or Dach families of DNA binding proteins. eyes absent, the founding member of the Eya family is expressed dynamically within several embryonic, larval, and adult tissues of the fruit fly, Drosophila melanogaster. Loss-of-function mutations are known to result in disruptions of the embryonic head and central nervous system as well as the adult brain and visual system, including the compound eyes. In an effort to understand how eya is regulated during development, we have carried out a genetic screen designed to identify genes that lie upstream of eya and govern its expression. We have identified a large number of putative regulators, including members of several signaling pathways. Of particular interest is the identification of both yan/anterior open and pointed, two members of the EGF Receptor (EGFR) signaling cascade. The EGFR pathway is known to regulate the activity of Eya through phosphorylation via MAPK. Our findings suggest that this pathway is also used to influence eya transcriptional levels. Together these mechanisms provide a route for greater precision in regulating a factor that is critical for the formation of a wide range of diverse tissues.' publications.doi='10.1534/genetics.109.110122' publications.firstAuthor='Salzer Claire L' publications.id=1004677 publications.issue='1' publications.journal='Genetics' publications.month='Jan' publications.pages='185-97' publications.pubMedId='19884307' publications.title='The retinal determination gene eyes absent is regulated by the EGF receptor pathway throughout development in Drosophila.' publications.volume='184' publications.year=2010\n",
      "Gene: publications.abstractText='Metazoan genomes contain vast tracts of cis-regulatory DNA that have been identified typically through tedious functional assays. As a result, it has not been possible to uncover a cis-regulatory code that links primary DNA sequences to gene expression patterns. In an initial effort to determine whether coordinately regulated genes share a common \"grammar,\"we have examined the distribution of Dorsal recognition sequences in the Drosophila genome. Dorsal is one of the best-characterized sequence-specific transcription factors in Drosophila. The homeobox gene zerknullt (zen) is repressed directly by Dorsal, and this repression is mediated by a 600-bp silencer, the ventral repression element (VRE), which contains four optimal Dorsal binding sites. The arrangement and sequence of the Dorsal recognition sequences in the VRE were used to develop a computational algorithm to search the Drosophila genome for clusters of optimal Dorsal binding sites. There are 15 regions in the genome that contain three or more optimal sites within a span of 400 bp or less. Three of these regions are associated with known Dorsal target genes: sog, zen, and Brinker. The Dorsal binding cluster in sog is shown to mediate lateral stripes of gene expression in response to low levels of the Dorsal gradient. Two of the remaining 12 clusters are shown to be associated with genes that exhibit asymmetric patterns of expression across the dorsoventral axis. These results suggest that bioinformatics can be used to identify novel target genes and associated regulatory DNAs in a gene network.' publications.doi='10.1073/pnas.012591199' publications.firstAuthor='Markstein Michele' publications.id=1007894 publications.issue='2' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Jan' publications.pages='763-8' publications.pubMedId='11752406' publications.title='Genome-wide analysis of clustered Dorsal binding sites identifies putative target genes in the Drosophila embryo.' publications.volume='99' publications.year=2002\n",
      "Gene: publications.abstractText='Microarray assays and bioinformatics methods have identified many of the genes and associated regulatory DNAs that control the early phases of gastrulation in Drosophila. The localized activities of these genes are coordinated by a nuclear gradient of the maternal regulatory factor, Dorsal, that is established shortly after fertilization. At least half of the Dorsal target genes encode transcription factors or signaling components that lead to the restricted activation of FGF, EGF, and TGF-beta signaling pathways in the mesoderm, neurogenic ectoderm and dorsal ectoderm, respectively. Recent work has yielded insights into how these signaling pathways control gastrulation, particularly in the context of the Dorsal-mediated gene regulation network' publications.doi='10.1016/j.gde.2004.07.004' publications.firstAuthor='Stathopoulos Angelike' publications.id=1007818 publications.issue='5' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Oct' publications.pages='477-84' publications.pubMedId='15380237' publications.title='Whole-genome analysis of Drosophila gastrulation.' publications.volume='14' publications.year=2004\n",
      "Gene: publications.abstractText='Modern developmental biology relies heavily on the analysis of embryonic gene expression patterns. Investigators manually inspect hundreds or thousands of expression patterns to identify those that are spatially similar and to ultimately infer potential gene interactions. However, the rapid accumulation of gene expression pattern data over the last two decades, facilitated by high-throughput techniques, has produced a need for the development of efficient approaches for direct comparison of images, rather than their textual descriptions, to identify spatially similar expression patterns. The effectiveness of the Binary Feature Vector (BFV) and Invariant Moment Vector (IMV) based digital representations of the gene expression patterns in finding biologically meaningful patterns was compared for a small (226 images) and a large (1819 images) dataset. For eachdataset, an ordered list of images, with respect to a query image, was generated to identify overlapping and similar gene expression patterns, in a manner comparable to what a developmental biologist might do. The results showed that the BFV representation consistently outperforms the IMV representation in finding biologically meaningful matches when spatial overlap of the gene expression pattern and the genes involved are considered. Furthermore, we explored the value of conducting image-content based searches in a dataset where individual expression components (or domains) of multi-domain expression patterns were also included separately. We found that this technique improves performance of both IMV and BFV based searches. We conclude that the BFV representation consistently produces a more extensive and better list of biologically useful patterns than the IMV representation. The high quality of results obtained scales well as the search database becomes larger, which encourages efforts to build automated image query and retrieval systems for spatial gene expression patterns.' publications.doi='10.1186/1471-2105-5-202' publications.firstAuthor='Gurunathan Rajalakshmi' publications.id=1007799 publications.issue=None publications.journal='BMC Bioinformatics' publications.month=None publications.pages='202' publications.pubMedId='15603586' publications.title='Identifying spatially similar gene expression patterns in early stage fruit fly embryo images: binary feature versus invariant moment digital representations.' publications.volume='5' publications.year=2004\n",
      "Gene: publications.abstractText=\"Molecular developmental studies of fly and mouse embryos have shown that the identity of individual body segments is controlled by a suite of homeobox-containing genes called the Hox cluster. To examine the conservation of this patterning mechanism in other segmented phyla, we here describe four Hox gene homologs isolated from glossiphoniid leeches of the genus Helobdella. Based on sequence similarity and phylogenetic analysis, the leech genes Lox7, Lox6, Lox20, and Lox5 are deemed to be orthologs of the Drosophila genes lab, Dfd, Scr, and Antp, respectively. Sequence similarities between Lox5 and Antp outside the homeodomain and phylogenetic reconstructions suggest that the Antennapedia family of Hox genes (as defined by Bürglin, 1994) had already expanded to include at least two discrete Antp and Ubx/abdA precursors prior to the annelid/arthropod divergence. In situ hybridization reveals that the four Lox genes described in this study are all expressed at high levels within the segmented portion of the central nervous system (CNS), with variable levels of expression in the segmental mesoderm. Little or no expression was seen in peripheral ectoderm or endoderm, or in the unsegmented head region (prostomium). Each Lox gene has a distinct anterior expression boundary within one of the four rostral segments, and the anterior-posterior (AP) order of these expression boundaries is identical to that reported for the orthologous Hox gene products in fly and mouse. This finding supports the idea that the process of AP axis differentiation is conserved among the higher metazoan phyla with respect to the regional expression of individual Hox genes along that axis. One unusual feature of leech Hox genes is the observation that some genes are only expressed during later development -- beginning at the time of terminal cell differentiation -- whereas others begin expression at a much earlier stage, and their RNA ceases to be detectable shortly after the onset of expression of the 'late' Hox genes. The functional significance of this temporal disparity is unknown, but it is noteworthy that only the two 'early' Hox genes display high levels of mesodermal expression.\" publications.doi='10.1006/dbio.1997.8689' publications.firstAuthor='Kourakis M J' publications.id=1007858 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Oct' publications.pages='284-300' publications.pubMedId='9344545' publications.title='Conserved anterior boundaries of Hox gene expression in the central nervous system of the leech Helobdella.' publications.volume='190' publications.year=1997\n",
      "Gene: publications.abstractText='Morphogen gradients determine a range of cell fates by specifying multiple transcriptional threshold responses. In the dorsal ectoderm of the Drosophila embryo, a BMP gradient is translated into an activated Smad transcription factor gradient, which elicits at least three threshold responses - high, intermediate and low. However, the mechanism underlying differential response to Dpp is poorly understood, due in part to the insufficient number of well-studied target genes. We analyzed the regulation of the C15 gene, which can be activated in cells containing intermediate levels of Dpp. We show that C15 expression requires both dpp and zen, thus forming a genetic feed-forward loop. The C15 regulatory element contains clusters of Smad- and Zen-binding sites in close proximity. Mutational analysis shows that the number of intact Smad- and Zen-binding sites is essential for the C15 transcriptional response, and that the spatial limits of C15 expression are established through a repression mechanism in the dorsolateral cells of the embryo. Thus, the combinatorial action of Smad and Zen activators bound to a number of adjacent sites, and competing negative cues allows for proper gene response to lower than peak levels of the Dpp morphogen.' publications.doi='10.1242/dev.02689' publications.firstAuthor='Lin Meng-chi' publications.id=1007703 publications.issue='24' publications.journal='Development' publications.month='Dec' publications.pages='4805-13' publications.pubMedId='17092951' publications.title='Threshold response of C15 to the Dpp gradient in Drosophila is established by the cumulative effect of Smad and Zen activators and negative cues.' publications.volume='133' publications.year=2006\n",
      "Gene: publications.abstractText='Morphogenesis of epithelial tissues relies on the precise developmental control of cell polarity and architecture. In the early Drosophila embryo, the primary epithelium forms during cellularisation, following a tightly controlled genetic programme where specific sets of genes are upregulated. Some of them, for example, control membrane invagination between the nuclei anchored at the apical surface of the syncytium. We used microarrays to describe the global programme of gene expression underlying cellularisation and identified distinct classes of upregulated genes during this process. Fifty-seven genes were then tested functionally by RNAi. We found six genes affecting various aspects of cellular architecture: membrane growth, organelle transport or organisation and junction assembly. We focus here on charleston (char), a new regulator of nuclear morphogenesis and of apical nuclear anchoring. In char-depleted embryos, the nuclei fail to maintain their elongated shape and, instead, become rounded. In addition, together with a disruption of the centrosome-nuclear envelope interaction, the nuclei lose their regular apical anchoring. These nuclear defects perturb the regular columnar organisation of epithelial cells in the embryo. Although microtubules are required for both nuclear morphogenesis and anchoring, char does not control microtubule organisation and association to the nuclear envelope. We show that Char is lipid anchored at the nuclear envelope by a farnesylation group, and localises at the inner nuclear membrane together with Lamin. Our data suggest that Char forms a scaffold that regulates nuclear architecture to constrain nuclei in tight columnar epithelial cells. The upregulation of Char during cellularisation and gastrulation reveals the existence of an as yet unknown developmental control of nuclear morphology and anchoring in embryonic epithelia.' publications.doi='10.1242/dev.02251' publications.firstAuthor='Pilot Fanny' publications.id=1003296 publications.issue='4' publications.journal='Development' publications.month='Feb' publications.pages='711-23' publications.pubMedId='16421189' publications.title='Developmental control of nuclear morphogenesis and anchoring by charleston, identified in a functional genomic screen of Drosophila cellularisation.' publications.volume='133' publications.year=2006\n",
      "Gene: publications.abstractText='Morphogenetic movements are closely regulated by the expression of developmental genes. Here I examine whether developmental gene expression can in turn be mechanically regulated by morphogenetic movements. I have analyzed the effects of mechanical stress on the expression of Twist, which is normally expressed only in the most ventral cells of the cellular blastoderm embryo under the control of the Dorsal morphogen gradient. At embryogenesis gastrulation (stage 7), Twist is also expressed in the anterior foregut and stomodeal primordia. Submitting the early Drosophila embryo to a transient 10% uniaxial lateral deformation induces the ectopic expression of Twist around the entire dorsal-ventral axis and results in the ventralization of the embryo. This induction is independent of the Dorsal gradient and is triggered by mechanically induced Armadillo nuclear translocation. I also show that Twist is not expressed in the anterior foregut and stomodeal primordia at stage 7 in mutants that block the morphogenetic movement of germ-band extension. Because I can rescue the mutants with gentle compression of these cells, my interpretation is that the stomodeal-cell compression normally caused by the germ-band extension induces the expression of Twist. Correspondingly, laser ablation of dorsal cells in wild-type embryos relaxes stomodeal cell compression and reduces Twist expression in the stomodeal primordium. I also demonstrate that the induction of Twist in these cells depends on the nuclear translocation of Armadillo. I propose that anterior-gut formation is mechanically induced by the movement of germ-band extension through the induction of Twist expression in stomodeal cells.' publications.doi=None publications.firstAuthor='Farge Emmanuel' publications.id=1007884 publications.issue='16' publications.journal='Curr. Biol.' publications.month='Aug' publications.pages='1365-77' publications.pubMedId='12932320' publications.title='Mechanical induction of Twist in the Drosophila foregut/stomodeal primordium.' publications.volume='13' publications.year=2003\n",
      "Gene: publications.abstractText='Morphogens are secreted signaling molecules that form concentration gradients and control cell fate in developing tissues. During development, it is essential that morphogen range is strictly regulated in order for correct cell type specification to occur. One of the best characterized morphogens is Drosophila Decapentaplegic (Dpp), a BMP signaling molecule that patterns the dorsal ectoderm of the embryo by activating the Mad and Medea (Med) transcription factors. We demonstrate that there is a spatial and temporal expansion of the expression patterns of Dpp target genes in SUMO pathway mutant embryos. We identify Med as the primary SUMOylation target in the Dpp pathway, and show that failure to SUMOylate Med leads to the increased Dpp signaling range observed in the SUMO pathway mutant embryos. Med is SUMO modified in the nucleus, and we provide evidence that SUMOylation triggers Med nuclear export. Hence, Med SUMOylation provides a mechanism by which nuclei can continue to monitor the presence of extracellular Dpp signal to activate target gene expression for an appropriate duration. Overall, our results identify an unusual strategy for regulating morphogen range that, rather than impacting on the morphogen itself, targets an intracellular transducer.' publications.doi='10.1101/gad.494808' publications.firstAuthor='Miles Wayne O' publications.id=1007900 publications.issue='18' publications.journal='Genes Dev.' publications.month='Sep' publications.pages='2578-90' publications.pubMedId='18794353' publications.title='Medea SUMOylation restricts the signaling range of the Dpp morphogen in the Drosophila embryo.' publications.volume='22' publications.year=2008\n",
      "Gene: publications.abstractText=\"Most cell-specific enhancers are thought to lack an inherent organization, with critical binding sites distributed in a more or less random fashion. However, there are examples of fixed arrangements of binding sites, such as helical phasing, that promote the formation of higher-order protein complexes on the enhancer DNA template. Here, we investigate the regulatory 'grammar' of nearly 100 characterized enhancers for developmental control genes active in the early Drosophila embryo. The conservation of grammar is examined in seven divergent Drosophila genomes. Linked binding sites are observed for particular combinations of binding motifs, including Bicoid-Bicoid, Hunchback-Hunchback, Bicoid-Dorsal, Bicoid-Caudal and Dorsal-Twist. Direct evidence is presented for the importance of Bicoid-Dorsal linkage in the integration of the anterior-posterior and dorsal-ventral patterning systems. Hunchback-Hunchback interactions help explain unresolved aspects of segmentation, including the differential regulation of the eve stripe 3 + 7 and stripe 4 + 6 enhancers. We also present evidence that there is an under-representation of nucleosome positioning sequences in many enhancers, raising the possibility for a subtle higher-order structure extending across certain enhancers. We conclude that grammar of gene control regions is pervasively used in the patterning of the Drosophila embryo.\" publications.doi='10.1093/nar/gkp619' publications.firstAuthor='Papatsenko Dmitri' publications.id=1007740 publications.issue='17' publications.journal='Nucleic Acids Res.' publications.month='Sep' publications.pages='5665-77' publications.pubMedId='19651877' publications.title='Organization of developmental enhancers in the Drosophila embryo.' publications.volume='37' publications.year=2009\n",
      "Gene: publications.abstractText='Most eukaryotic genomes include a substantial repeat-rich fraction termed heterochromatin, which is concentrated in centric and telomeric regions. The repetitive nature of heterochromatic sequence makes it difficult to assemble and analyze. To better understand the heterochromatic component of the Drosophila melanogaster genome, we characterized and annotated portions of a whole-genome shotgun sequence assembly. WGS3, an improved whole-genome shotgun assembly, includes 20.7 Mb of draft-quality sequence not represented in the Release 3 sequence spanning the euchromatin. We annotated this sequence using the methods employed in the re-annotation of the Release 3 euchromatic sequence. This analysis predicted 297 protein-coding genes and six non-protein-coding genes, including known heterochromatic genes, and regions of similarity to known transposable elements. Bacterial artificial chromosome (BAC)-based fluorescence in situ hybridization analysis was used to correlate the genomic sequence with the cytogenetic map in order to refine the genomic definition of the centric heterochromatin; on the basis of our cytological definition, the annotated Release 3 euchromatic sequence extends into the centric heterochromatin on each chromosome arm. Whole-genome shotgun assembly produced a reliable draft-quality sequence of a significant part of the Drosophila heterochromatin. Annotation of this sequence defined the intron-exon structures of 30 known protein-coding genes and 267 protein-coding gene models. The cytogenetic mapping suggests that an additional 150 predicted genes are located in heterochromatin at the base of the Release 3 euchromatic sequence. Our analysis suggests strategies for improving the sequence and annotation of the heterochromatic portions of the Drosophila and other complex genomes.' publications.doi=None publications.firstAuthor='Hoskins Roger A' publications.id=1058977 publications.issue='12' publications.journal='Genome Biol.' publications.month=None publications.pages='RESEARCH0085' publications.pubMedId='12537574' publications.title='Heterochromatic sequences in a Drosophila whole-genome shotgun assembly.' publications.volume='3' publications.year=2002\n",
      "Gene: publications.abstractText='Most of our knowledge about the mechanisms of segmentation in arthropods comes from work on Drosophila melanogaster. In recent years it has become clear that this mechanism is far from universal, and different arthropod groups have distinct modes of segmentation that operate through divergent genetic mechanisms. We review recent data from a range of arthropods, identifying which features of the D. melanogaster segmentation cascade are present in the different groups, and discuss the evolutionary implications of their conserved and divergent aspects. A model is emerging, although slowly, for the way that arthropod segmentation mechanisms have evolved.' publications.doi='10.1038/nrg1724' publications.firstAuthor='Peel Andrew D' publications.id=1007719 publications.issue='12' publications.journal='Nat. Rev. Genet.' publications.month='Dec' publications.pages='905-16' publications.pubMedId='16341071' publications.title='Arthropod segmentation: beyond the Drosophila paradigm.' publications.volume='6' publications.year=2005\n",
      "Gene: publications.abstractText='Mutations of the homeotic gene proboscipedia (pb) of Drosophila cause striking transformations of the adult mouthparts, to legs or antennae. We report here an analysis of the gene structure of pb. Coding sequences across a 34 kb interval yield, by alternative splicing, four identified mRNA forms which differ immediately upstream of the homeobox. As a consequence, the homeodomain is expected to reside in four different contexts in the predicted protein isoforms. Mammalian homologs of pb, human HOX-2H and murine Hox-2.8, were identified based on the similarities of their homeodomains (95% identity) and several other conserved motifs. Examination of a collection of pb mutant alleles with antisera directed against the N-terminal region, the center or the C-terminal region of the protein showed that, surprisingly, several partial loss-of-function pb alleles appear to generate partially functional proteins truncated at their C-termini. This suggests that a significant portion of the protein contributes quantitatively to pb function, but is partially dispensable. Finally, evolutionary considerations suggest that pb may be one of several ancient genes which preceded the process yielding the modern homeotic gene complexes.' publications.doi=None publications.firstAuthor='Cribbs D L' publications.id=1007706 publications.issue='4' publications.journal='EMBO J.' publications.month='Apr' publications.pages='1437-49' publications.pubMedId='1348688' publications.title='Structural complexity and evolutionary conservation of the Drosophila homeotic gene proboscipedia.' publications.volume='11' publications.year=1992\n",
      "Gene: publications.abstractText='NURF is a conserved higher eukaryotic ISWI-containing chromatin remodeling complex that catalyzes ATP-dependent nucleosome sliding. By sliding nucleosomes, NURF is able to alter chromatin dynamics to control transcription and genome organization. Previous biochemical and genetic analysis of the specificity-subunit of Drosophila NURF (Nurf301/Enhancer of Bithorax (E(bx)) has defined NURF as a critical regulator of homeotic, heat-shock and steroid-responsive gene transcription. It has been speculated that NURF controls pathway specific transcription by co-operating with sequence-specific transcription factors to remodel chromatin at dedicated enhancers. However, conclusive in vivo demonstration of this is lacking and precise regulatory elements targeted by NURF are poorly defined. To address this, we have generated a comprehensive map of in vivo NURF activity, using MNase-sequencing to determine at base pair resolution NURF target nucleosomes, and ChIP-sequencing to define sites of NURF recruitment. Our data show that, besides anticipated roles at enhancers, NURF interacts physically and functionally with the TRF2/DREF basal transcription factor to organize nucleosomes downstream of active promoters. Moreover, we detect NURF remodeling and recruitment at distal insulator sites, where NURF functionally interacts with and co-localizes with DREF and insulator proteins including CP190 to establish nucleosome-depleted domains. This insulator function of NURF is most apparent at subclasses of insulators that mark the boundaries of chromatin domains, where multiple insulator proteins co-associate. By visualizing the complete repertoire of in vivo NURF chromatin targets, our data provide new insights into how chromatin remodeling can control genome organization and regulatory interactions.' publications.doi='10.1371/journal.pgen.1005969' publications.firstAuthor='Kwon So Yeon' publications.id=1007830 publications.issue='4' publications.journal='PLoS Genet.' publications.month='Apr' publications.pages='e1005969' publications.pubMedId='27046080' publications.title='Genome-Wide Mapping Targets of the Metazoan Chromatin Remodeling Factor NURF Reveals Nucleosome Remodeling at Enhancers, Core Promoters and Gene Insulators.' publications.volume='12' publications.year=2016\n",
      "Gene: publications.abstractText='Network motifs provided a\"conceptual tool\"for understanding the functional principles of biological networks, but such motifs have primarily been used to consider static network structures. Static networks, however, cannot be used to reveal time- and region-specific traits of biological systems. To overcome this limitation, we proposed the concept of a \"spatiotemporal network motif,\"a spatiotemporal sequence of network motifs of sub-networks which are active only at specific time points and body parts. On the basis of this concept, we analyzed the developmental gene regulatory network of the Drosophila melanogaster embryo. We identified spatiotemporal network motifs and investigated their distribution pattern in time and space. As a result, we found how key developmental processes are temporally and spatially regulated by the gene network. In particular, we found that nested feedback loops appeared frequently throughout the entire developmental process. From mathematical simulations, we found that mutual inhibition in the nested feedback loops contributes to the formation of spatial expression patterns. Taken together, the proposed concept and the simulations can be used to unravel the design principle of developmental gene regulatory networks.' publications.doi='10.1186/1752-0509-6-31' publications.firstAuthor='Kim Man-Sun' publications.id=1007708 publications.issue=None publications.journal='BMC Syst Biol' publications.month=None publications.pages='31' publications.pubMedId='22548745' publications.title='Spatiotemporal network motif reveals the biological traits of developmental gene regulatory networks in Drosophila melanogaster.' publications.volume='6' publications.year=2012\n",
      "Gene: publications.abstractText='Networks of regulatory relations between transcription factors (TF) and their target genes (TG)- implemented through TF binding sites (TFBS)- are key features of biology. An idealized approach to solving such networks consists of starting from a consensus TFBS or a position weight matrix (PWM) to generate a high accuracy list of candidate TGs for biological validation. Developing and evaluating such approaches remains a formidable challenge in regulatory bioinformatics. We perform a benchmark study on 34 Drosophila TFs to assess existing TFBS and cis-regulatory module (CRM) detection methods, with a strong focus on the use of multiple genomes. Particularly, for CRM-modelling we investigatethe addition of orthologous sites to a known PWM to construct phyloPWMs and we assess the added value of phylogenentic footprinting to predict contextual motifs around known TFBSs. For CRM-prediction, we compare motif conservation with network-level conservation approaches across multiple genomes. Choosing the optimal training and scoring strategies strongly enhances the performance of TG prediction for more than half of the tested TFs. Finally, we analyse a 35(th) TF, namely Eyeless, and find a significant overlap between predicted TGs and candidate TGs identified by microarray expression studies. In summary we identify several ways to optimize TF-specific TG predictions, some of which can be applied to all TFs, and others that can be applied only to particular TFs. The ability to model known TF-TG relations, together with the use of multiple genomes, results in a significant step forward in solving the architecture of gene regulatory networks.' publications.doi='10.1371/journal.pone.0001115' publications.firstAuthor='Aerts Stein' publications.id=1007731 publications.issue='11' publications.journal='PLoS ONE' publications.month=None publications.pages='e1115' publications.pubMedId='17973026' publications.title='Fine-tuning enhancer models to predict transcriptional targets across multiple genomes.' publications.volume='2' publications.year=2007\n",
      "Gene: publications.abstractText='Neurotactin (NRT), a member of the cholinesterase-homologous protein family, is a heterophilic cell adhesion molecule that is required for proper axon guidance during Drosophila development. In this study, we identify amalgam (AMA), a member of the immunoglobulin superfamily, as a ligand for the NRT receptor. Using transfected Schneider 2 cells and embryonic primary cultures, we demonstrate that AMA is a secreted protein. Furthermore, AMA is necessary for NRT-expressing cells both to aggregate with themselves and to associate with embryonic primary culture cells. Aggregation assays performed with truncated NRT molecules reveal that the integrity of the cholinesterase-like extracellular domain was not required either for AMA binding or for adhesion, with only amino acids 347-482 of the extracellular domain being necessary for both activities. Moreover, the NRT cytoplasmic domain is required for NRT-mediated adhesion, although not for AMA binding. Using an ama-deficient stock, we find that ama function is not essential for viability. Pupae deficient for ama do exhibit defasciculation defects of the ocellar nerves similar to those found in nrt mutants.' publications.doi='10.1093/emboj/19.17.4463' publications.firstAuthor='Frémion F' publications.id=1007728 publications.issue='17' publications.journal='EMBO J.' publications.month='Sep' publications.pages='4463-72' publications.pubMedId='10970840' publications.title='Amalgam is a ligand for the transmembrane receptor neurotactin and is required for neurotactin-mediated cell adhesion and axon fasciculation in Drosophila.' publications.volume='19' publications.year=2000\n",
      "Gene: publications.abstractText='One of the main challenges in evolutionary biology is to identify the molecular changes that underlie phenotypic differences that are of evolutionary significance. Comparative studies of early development have shown that changes in the spatio-temporal use of regulatory genes, as well as changes in the specificity of regulatory proteins, are correlated with important differences in morphology between phylogenetically distant species. However, it is not known how such changes take place in natural populations, and whether they result from a single, or many small, additive events. Extending this approach to the study of development of closely related species promises to enrich this debate.' publications.doi='10.1038/nrg947' publications.firstAuthor='Simpson Pat' publications.id=1007819 publications.issue='12' publications.journal='Nat. Rev. Genet.' publications.month='Dec' publications.pages='907-17' publications.pubMedId='12459721' publications.title='Evolution of development in closely related species of flies and worms.' publications.volume='3' publications.year=2002\n",
      "Gene: publications.abstractText='One of the most important questions in biology is how transcription factors (TFs) and cofactors control enhancer function and thus gene expression. Enhancer activation usually requires combinations of several TFs, indicating that TFs function synergistically and combinatorially. However, while TF binding has been extensively studied, little is known about how combinations of TFs and cofactors control enhancer function once they are bound. It is typically unclear which TFs participate in combinatorial enhancer activation, whether different TFs form functionally distinct groups, or if certain TFs might substitute for each other in defined enhancer contexts. Here we assess the potential regulatory contributions of TFs and cofactors to combinatorial enhancer control with enhancer complementation assays. We recruited GAL4-DNA-binding-domain fusions of 812 Drosophila TFs and cofactors to 24 enhancer contexts and measured enhancer activities by 82,752 luciferase assays in S2 cells. Most factors were functional in at least one context, yet their contributions differed between contexts and varied from repression to activation (up to 289-fold) for individual factors. Based on functional similarities across contexts, we define 15 groups of TFs that differ in developmental functions and protein sequence features. Similar TFs can substitute for each other, enabling enhancer re-engineering by exchanging TF motifs, and TF-cofactor pairs cooperate during enhancer control and interact physically. Overall, we show that activators and repressors can have diverse regulatory functions that typically depend on the enhancer context. The systematic functional characterization of TFs and cofactors should further our understanding of combinatorial enhancer control and gene regulation.' publications.doi='10.1038/nature15545' publications.firstAuthor='Stampfel Gerald' publications.id=1000807 publications.issue='7580' publications.journal='Nature' publications.month='Dec' publications.pages='147-51' publications.pubMedId='26550828' publications.title='Transcriptional regulators form diverse groups with context-dependent regulatory functions.' publications.volume='528' publications.year=2015\n",
      "Gene: publications.abstractText='Pattern formation in Drosophila is initiated by a small set of asymmetrically distributed maternal transcription factors that act as graded morphogens along the anterior-posterior and the dorsal-ventral axes of the embryo. Recent progress in the field provides first insight into the molecular mechanisms by which long-range positional information in the egg causes a series of localized zygotic transcription factors to position the developmental fate along the blastoderm.' publications.doi=None publications.firstAuthor='Hoch M' publications.id=1002690 publications.issue='4' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Aug' publications.pages='566-73' publications.pubMedId='8241768' publications.title='Transcriptional regulation and spatial patterning in Drosophila.' publications.volume='3' publications.year=1993\n",
      "Gene: publications.abstractText='Pattern formation in the developing embryo relies on key regulatory molecules, many of which are distributed in concentration gradients. For example, a gradient of BMP specifies cell fates along the dorsoventral axis in species ranging from flies to mammals. In Drosophila, a gradient of the BMP molecule Dpp gives rise to nested domains of target gene expression in the dorsal regionof the embryo; however, the mechanisms underlying the differential response are not well understood, partly owing to an insufficient number of well-studied targets. Here we analyze how the Dpp gradient regulates expression of pannier (pnr), a candidate low-level Dpp target gene. We predicted that the pnr enhancer would contain high-affinity binding sites for the Dpp effector Smad transcription factors, which would be occupied in the presence of low-level Dpp. Unexpectedly, the affinity of Smad sites in the pnr enhancer was similar to those in the Race enhancer, a high-level Dpp target gene, suggesting that the affinity threshold mechanism plays a minimal role in the regulation of pnr. Our results indicate that a mechanism involving a conserved bipartite motif that is predicted to bind a homeodomain factor in addition to Smads and the Brinker repressor, establishes the pnr expression domain. Furthermore, the pnr enhancer has a highly complex structure that integrates cues not only from the dorsoventral axis, but also from the anteroposterior and terminal patterning systems in the blastoderm embryo.' publications.doi='10.1242/dev.079772' publications.firstAuthor='Liang Hsiao-Lan' publications.id=1007710 publications.issue='11' publications.journal='Development' publications.month='Jun' publications.pages='1956-64' publications.pubMedId='22513375' publications.title='Response to the BMP gradient requires highly combinatorial inputs from multiple patterning systems in the Drosophila embryo.' publications.volume='139' publications.year=2012\n",
      "Gene: publications.abstractText='Pattern formation in the dorsal region of the Drosophila embryo depends on the activity of a small group of zygotically acting genes. dpp, a key gene in this group, encodes a TGF-beta-like product (Dpp) that has been proposed to function as a morphogen with peak levels of Dpp-specifying amnioserosa, the dorsal-most cell type, and lower Dpp levels specifying dorsal ectoderm. The short gastrulation gene also contributes to patterning the dorsal region, but unlike the other genes involved in this process, sog activity is only required in ventral cells. Genetic evidence indicates that sog functions to antagonize dpp activity. In this report we present further phenotypic characterization of sog mutant embryos in dorsal and lateral regions and describe the cloning of the sog locus. sog is expressed in a broad lateral stripe of cells that abuts the dorsal territory of dpp-expressing cells. sog is predicted to encode a protein with an internal signal sequence and a large extracellular domain containing four repeats of a novel motif defined by the spacing of 10 cysteine residues that is distantly related to domains present in thrombospondin and procollagen. We propose that one or more of these cysteine repeats can be liberated by proteolytic cleavage of the primary Sog protein. These putative soluble Sog peptides may then diffuse into the dorsal region to antagonize the activity of Dpp, leading to the subdivision of the dorsal territory into amnioserosa and dorsal ectoderm.' publications.doi=None publications.firstAuthor='Francois V' publications.id=1007736 publications.issue='21' publications.journal='Genes Dev.' publications.month='Nov' publications.pages='2602-16' publications.pubMedId='7958919' publications.title='Dorsal-ventral patterning of the Drosophila embryo depends on a putative negative growth factor encoded by the short gastrulation gene.' publications.volume='8' publications.year=1994\n",
      "Gene: publications.abstractText='Patterning of the Drosophila embryo requires not only the proper activation of determinants at specific times, but also their restriction to specific places. Recent studies on transcriptional repressors show how they delimit the gene expression patterns to ensure normal development.' publications.doi=None publications.firstAuthor='Ip Y T' publications.id=1007723 publications.issue='4' publications.journal='Curr. Biol.' publications.month='Apr' publications.pages='R216-8' publications.pubMedId='9162494' publications.title='Drosophila development. Delimiting patterns by repression.' publications.volume='7' publications.year=1997\n",
      "Gene: publications.abstractText='Patterning of the dorsal-ventral axis in the early Drosophila embryo depends on the nuclear distribution of the Dorsal transcription factor. Using live two-photon light-sheet microscopy, we quantified the nuclear Dorsal gradient in space and time and found that its amplitude and basal levels display oscillations throughout early embryonic development. These dynamics raise questions regarding how cells can reproducibly establish patterns of gene expression from a rapidly varying signal. We therefore quantified domains of Dorsal target genes, discovering their expression patterns are also dynamic. Computational modeling of this system reveals a correlation between Dorsal gradient dynamics and changes in target gene expression and suggests that these dynamics, together with time averaging of noise, results in the formation of graded gene expression borders in regions where the gradient is nearly flat. We propose that mRNA levels remain plastic during transient signaling events, allowing tissues to refine patterns in the face of genetic or environmental variation.' publications.doi='10.1016/j.devcel.2011.12.007' publications.firstAuthor='Reeves Gregory T' publications.id=1007697 publications.issue='3' publications.journal='Dev. Cell' publications.month='Mar' publications.pages='544-57' publications.pubMedId='22342544' publications.title='Dorsal-ventral gene expression in the Drosophila embryo reflects the dynamics and precision of the dorsal nuclear gradient.' publications.volume='22' publications.year=2012\n",
      "Gene: publications.abstractText='Patterning of the terminal regions of the Drosophila embryo is achieved by an exquisitely regulated signal that passes between the follicle cells of the ovary, and the developing embryo. This pathway, however, is missing or modified in other insects. Here we trace the evolution of this pathway by examining the origins and expression of its components. The three core components of this pathway: trunk, torso and torso-like have different evolutionary histories and have been assembled step-wise to form the canonical terminal patterning pathway of Drosophila and Tribolium. Trunk, torso and a gene unrelated to terminal patterning, prothoraciotrophic hormone (PTTH), show an intimately linked evolutionary history, with every holometabolous insect, except the honeybee, possessing both PTTH and torso genes. Trunk is more restricted in its phylogenetic distribution, present only in the Diptera and Tribolium and, surprisingly, in the chelicerate Ixodes scapularis, raising the possibility that trunk and torso evolved earlier than previously thought. In Drosophila torso-like restricts the activation of the terminal patterning pathway to the poles of the embryo. Torso-like evolved in the pan-crustacean lineage, but based on expression of components of the canonical terminal patterning system in the hemimetabolous insect Acyrthosiphon pisum and the holometabolous insect Apis mellifera, we find that the canonical terminal-patterning system is not active in these insects. We therefore propose that the ancestral function of torso-like is unrelated to terminal patterning and that torso-like has become co-opted into terminal patterning in the lineage leading to Coleoptera and Diptera. We also show that this co-option has not resulted in changes to the molecular function of this protein. Torso-like from the pea aphid, honeybee and Drosophila, despite being expressed in different patterns, are functionally equivalent. We propose that co-option of torso-like into restricting the activity of trunk and torso facilitated the final step in the evolution of this pathway; the capture of transcriptional control of target genes such as tailless and huckebein by this complex and novel patterning pathway.' publications.doi='10.1016/j.ydbio.2013.02.010' publications.firstAuthor='Duncan Elizabeth J' publications.id=1007833 publications.issue='1' publications.journal='Dev. Biol.' publications.month='May' publications.pages='245-61' publications.pubMedId='23438815' publications.title='Canonical terminal patterning is an evolutionary novelty.' publications.volume='377' publications.year=2013\n",
      "Gene: publications.abstractText='Phylostratigraphy is a method for dating the evolutionary emergence of a gene or gene family by identifying its homologs across the tree of life, typically by using BLAST searches. Applying this method to all genes in a species, or genomic phylostratigraphy, allows investigation of genome-wide patterns in new gene origination at different evolutionary times and thus has been extensively used. However, gene age estimation depends on the challenging task of detecting distant homologs via sequence similarity, which is expected to have differential accuracies for different genes. Here, we evaluate the accuracy of phylostratigraphy by realistic computer simulation with parameters estimated from genomic data, and investigate the impact of its error on findings of genome evolution. We show that 1) phylostratigraphy substantially underestimates gene age for a considerable fraction of genes, 2) the error is especially serious when the protein evolves rapidly, is short, and/or its most conserved block of sites is small, and 3) these errors create spurious nonuniform distributions of various gene properties among age groups, many of which cannot be predicted a priori. Given the high likelihood that conclusions about gene age are faulty, we advocate the use of realistic simulation to determine if observations from phylostratigraphy are explainable, at least qualitatively, by a null model of biased measurement, and in all cases, critical evaluation of results.' publications.doi='10.1093/molbev/msu286' publications.firstAuthor='Moyers Bryan A' publications.id=1000123 publications.issue='1' publications.journal='Mol. Biol. Evol.' publications.month='Jan' publications.pages='258-67' publications.pubMedId='25312911' publications.title='Phylostratigraphic bias creates spurious patterns of genome evolution.' publications.volume='32' publications.year=2015\n",
      "Gene: publications.abstractText='Polycomb-group (PcG) and Trithorax-group proteins together form a maintenance machinery that is responsible for stable heritable states of gene activity. While the best-studied target genes are the Hox genes of the Antennapedia and Bithorax complexes, a large number of key developmental genes are also Polycomb (Pc) targets, indicating a widespread role for this maintenance machinery in cell fate determination. We have studied the linkage between the binding of PcG proteins and the developmental regulation of gene expression using whole-genome mapping to identify sites bound by the PcG proteins, Pc and Pleiohomeotic (Pho), in the Drosophila embryo and in a more restricted tissue, the imaginal discs of the third thoracic segment. Our data provide support for the idea that Pho is a general component of the maintenance machinery, since the majority of Pc targets are also associated with Pho binding. We find, in general, considerable developmental stability of Pc and Pho binding at target genes and observe that Pc/Pho binding can be associated with both expressed and inactive genes. In particular, at the Hox complexes, both active and inactive genes have significant Pc and Pho binding. However, in comparison to inactive genes, the active Hox genes show reduced and altered binding profiles.During development, Pc target genes are not simply constantly associated with Pc/Pho binding, and we identify sets of genes with clear differential binding between embryo and imaginal disc. Using existing datasets, we show that for specific fate-determining genes of the haemocyte lineage, the active state is characterised by lack of Pc binding. Overall, our analysis suggests a dynamic relationship between Pc/Pho binding and gene transcription. Pc/Pho binding does not preclude transcription, but levels of Pc/Pho binding change during development, and loss of Pc/Pho binding can be associated with both stable gene activity and inactivity.' publications.doi='10.1371/journal.pgen.1000178' publications.firstAuthor='Kwong Camilla' publications.id=1007769 publications.issue='9' publications.journal='PLoS Genet.' publications.month=None publications.pages='e1000178' publications.pubMedId='18773083' publications.title='Stability and dynamics of polycomb target sites in Drosophila development.' publications.volume='4' publications.year=2008\n",
      "Gene: publications.abstractText='Population genetic theory predicts that maternal effect genes will evolve differently than genes expressed in both sexes because selection is only half as effective on autosomal genes expressed in one sex but not the other. Here, we use sequences of the tandem gene duplicates, bicoid (bcd) and zerknüllt (zen), to test the prediction that, with similar coefficients of purifying selection, a maternal effect gene evolves more rapidly than a zygotic gene because of this reduction in selective constraint. We find that the maternal effect gene, bcd, is evolving more rapidly than zygotically expressed, zen, providing the first direct confirmation of this prediction of maternal effect theory from molecular evidence. Our results extend current explanations for the accelerated rate of bcd evolution by providing an evolutionary mechanism, relaxed selective constraint, that allows bcd the evolutionary flexibility to escape the typical functional constraints of early developmental genes. We discuss general implications of our findings for the role of maternal effect genes in early developmental patterning.' publications.doi='10.1007/s10709-006-0031-4' publications.firstAuthor='Demuth Jeffery P' publications.id=1007758 publications.issue='1' publications.journal='Genetica' publications.month='Jan' publications.pages='37-43' publications.pubMedId='16955333' publications.title='Maternal expression increases the rate of bicoid evolution by relaxing selective constraint.' publications.volume='129' publications.year=2007\n",
      "Gene: publications.abstractText='Positional information in the dorsoventral axis of the Drosophila embryo is encoded by a BMP activity gradient formed by synergistic signaling between the BMP family members Decapentaplegic (DPP) and Screw (SCW). short gastrulation (sog), which is functionally homologous to Xenopus Chordin, is expressed in the ventrolateral regions of the embryo and has been shown to act as a local antagonist of BMP signaling. Here we demonstrate that SOG has a second function, which is to promote BMP signaling on the dorsal side of the embryo. We show that a weak, homozygous-viable sog mutant is enhanced to lethality by reduction in the activities of the Smad family members Mad or Medea, and that the lethality is caused by defects in the molecular specification and subsequent cellular differentiation of the dorsal-most cell type, the amnioserosa. While previous data had suggested that the negative function of SOG is directed against SCW, we present data that suggests that the positive activity of SOG is directed towards DPP. We demonstrate that Chordin shares the same apparent ligand specificity as does SOG, preferentially inhibiting SCW but not DPP activity. However, in Drosophila assays Chordin does not have the same capacity to elevate BMP signaling as does SOG, identifying a functional difference in the otherwise well conserved process of dorsoventral pattern formation in arthropods and chordates.' publications.doi=None publications.firstAuthor='Decotto E' publications.id=1007682 publications.issue='19' publications.journal='Development' publications.month='Oct' publications.pages='3831-41' publications.pubMedId='11585808' publications.title='A positive role for Short gastrulation in modulating BMP signaling during dorsoventral patterning in the Drosophila embryo.' publications.volume='128' publications.year=2001\n",
      "Gene: publications.abstractText='Previously, we mapped quantitative trait loci (QTL) affecting response to short-term selection for abdominal bristle number to seven suggestive regions that contain loci involved in bristle development and/or that have adult bristle number mutant phenotypes, and are thus candidates for bristle number QTL in natural populations. To test the hypothesis that the factors contributing to selection response genetically interact with these candidate loci, high and low chromosomes from selection lines were crossed to chromosomes containing wild-type or mutant alleles at the candidate loci, and the numbers of bristles were recorded in trans heterozygotes. Quantitative failure to complement, detected as a significant selection line*cross effect by analysis of variance, can be interpreted as evidence for allelism or epistasis between the factors on selected chromosomes and the candidate loci. Mutations at some candidate loci (bb, emc, h, Dl, Hairless) showed strong interactions with selected chromosomes, whereas others interacted weakly (ASC, abd, Scr) or not at all (N, mab, E(spl)). These results support the hypothesis that some candidate loci, initially identified through mutations of large effect on bristle number, either harbor or are close members in the same genetic pathway as variants that contribute to standing variation in bristle number.' publications.doi=None publications.firstAuthor='Long A D' publications.id=1007899 publications.issue='4' publications.journal='Genetics' publications.month='Dec' publications.pages='1497-510' publications.pubMedId='8978039' publications.title='Genetic interactions between naturally occurring alleles at quantitative trait loci and mutant alleles at candidate loci affecting bristle number in Drosophila melanogaster.' publications.volume='144' publications.year=1996\n",
      "Gene: publications.abstractText='Quantitative studies of embryogenesis require the ability to monitor pattern formation and morphogenesis in large numbers of embryos, at multiple time points and in diverse genetic backgrounds. We describe a simple approach that greatly facilitates these tasks for Drosophila melanogaster embryos, one of the most advanced models of developmental genetics. Based on passive hydrodynamics, we developed a microfluidic embryo-trap array that can be used to rapidly order and vertically orient hundreds of embryos. We describe the physical principles of the design and used this platform to quantitatively analyze multiple morphogen gradients in the dorsoventral patterning system. Our approach can also be used for live imaging and, with slight modifications, could be adapted for studies of pattern formation and morphogenesis in other model organisms.' publications.doi='10.1038/nmeth.1548' publications.firstAuthor='Chung Kwanghun' publications.id=1007787 publications.issue='2' publications.journal='Nat. Methods' publications.month='Feb' publications.pages='171-6' publications.pubMedId='21186361' publications.title='A microfluidic array for large-scale ordering and orientation of embryos.' publications.volume='8' publications.year=2011\n",
      "Gene: publications.abstractText='RNApolII-dependent transcription is repressed in primordial germ cells of many animals during early development and is thought to be important for maintenance of germline fate by preventing somatic differentiation. Germ cell transcriptional repression occurs concurrently with inhibition of phosphorylation in the carboxy-terminal domain (CTD) of RNApolII, as well as with chromatin remodeling. The precise mechanisms involved are unknown. Here, we present evidence that a noncoding RNA transcribed by the gene polar granule component (pgc) regulates transcriptional repression in Drosophila germ cells. Germ cells lacking pgc RNA express genes important for differentiation of nearby somatic cells and show premature phosphorylation of RNApolII. We further show that germ cells lacking pgc show increased levels of K4, but not K9 histone H3 methylation, and that the chromatin remodeling Swi/Snf complex is required for a second stage in germ cell transcriptional repression. We propose that a noncoding RNA controls transcription in early germ cells by blocking the transition from preinitiation to transcriptional elongation. We further show that repression of somatic differentiation signals mediated by the Torso receptor-tyrosine kinase is important for germline development.' publications.doi=None publications.firstAuthor='Martinho Rui Gonçalo' publications.id=1007732 publications.issue='2' publications.journal='Curr. Biol.' publications.month='Jan' publications.pages='159-65' publications.pubMedId='14738740' publications.title='A noncoding RNA is required for the repression of RNApolII-dependent transcription in primordial germ cells.' publications.volume='14' publications.year=2004\n",
      "Gene: publications.abstractText='Recent data have demonstrated that vRel, cRel, Dorsal, and NF-kappa B are members of a larger family of DNA-binding regulatory proteins. Rel proteins interact to form homo- and heterodimers that recognize specific sites on DNA, and it is likely that such protein-protein and protein-DNA interactions contribute to proper regulation of target gene expression by these proteins. Here we describe the use of a yeast transcription activation assay to study binding of three Rel family proteins to their native binding sites. These results show that the vRel and cRel proteins recognize two known NF-kappa B binding sites; the Dorsal protein does not recognize NF-kappa B sites, but does recognize related sites upstream of the Drosophila zerknüllt gene. Our experiments demonstrate that the members of this protein family recognize similar, but not identical, sites in the promoters of target genes, and we are able to identify a particular nucleotide that is apparently involved in the DNA-protein interaction. We exploit the properties of LexA fusion proteins to study the dimerization and DNA-contacting domains of cRel. Our results suggest that the cRel protein forms homodimers and that dimer formation may be necessary for cRel to bind DNA. Finally, our results show that transcription activation by these proteins is cooperative; such cooperativity may be important for correct temporal and spatial regulation of target gene expression.' publications.doi=None publications.firstAuthor='Kamens J' publications.id=1007839 publications.issue='10' publications.journal='New Biol.' publications.month='Oct' publications.pages='1005-13' publications.pubMedId='1768648' publications.title='A yeast transcription assay defines distinct rel and dorsal DNA recognition sequences.' publications.volume='3' publications.year=1991\n",
      "Gene: publications.abstractText='Retrogenes are processed copies of genes that are inserted into new genomic regions and that acquire new regulatory elements from the sequences in their surroundings. Here we use a comparative approach of phylogenetic footprinting and a non-comparative approach of measuring motif over-representation in retrogenes in order to describe putative elements present in cis-regulatory regions of 94 retrogenes recently described in Drosophila. The detailed examination of the motifs found in the core promoter regions of retrogenes reveals an abundance of the DNA replication-related element (DRE), the Initiator (Inr), and a new over-represented motif that we call the GCT motif. Parental genes also show an abundance of DRE and Inr motifs, but these do not seem to have been carried over with retrogenes. In particular, we also examined motifs upstream of retrogenes expressed in adult testis and were able to identify 6 additional over-represented motifs. Comparative analyses provide data on the conservation and origin of some of these motifs and reveal 15 additional conserved motifs in these retrogenes. Some of those conserved motifs are sequences bound by known transcription factors, while others are novel motifs. In this report we provide the first genome-wide data on which specific cis-regulatory regions can be recruited by retrogenes after they are inserted into new coding regions in the genome. Future experiments are needed to determine the function and role of the new elements presented here.' publications.doi='10.1016/j.ygeno.2008.09.006' publications.firstAuthor='Bai Yongsheng' publications.id=1007795 publications.issue='1' publications.journal='Genomics' publications.month='Jan' publications.pages='83-9' publications.pubMedId='18848618' publications.title='Quality of regulatory elements in Drosophila retrogenes.' publications.volume='93' publications.year=2009\n",
      "Gene: publications.abstractText='Seven zygotically active genes are required for normal patterning of the dorsal 40% of the Drosophila embryo. Among these genes, decapentaplegic (dpp) has the strongest mutant phenotype: in the absence of dpp, all cells in the dorsal and dorsolateral regions of the embryo adopt fates characteristic of more ventrally derived cells (Irish and Gelbart (1987) Genes Dev. 1, 868-879). Here we describe the phenotypes caused by alleles of another of this set of genes, tolloid, and show that tolloid is required for dorsal, but not dorsolateral, pattern. Extragenic suppressors of tolloid mutations were isolated that proved to be mutations that elevate dpp activity. We studied the relationship between tolloid and dpp by analyzing the phenotypes of tolloid embryos with elevated numbers of the dpp gene and found that doubling the dpp+ gene dosage completely suppressed weak tolloid mutants and partially suppressed the phenotypes of tolloid null mutants. We conclude that the function of tolloid is to increase dpp activity. We also examined the effect of doubling dpp+ gene dosage on the phenotypes caused by other mutations affecting dorsal development. Like tolloid, the phenotypes of mutant embryos lacking shrew gene function were suppressed by elevated dpp, indicating that shrew also acts upstream of dpp to increase dpp activity. In contrast, increasing the number of copies of the dpp gene enhanced the short gastrulation (sog) mutant phenotype, causing ventrolateral cells to adopt dorsal fates. This indicates that sog gene product normally blocks dpp activity ventrally. We propose that the tolloid, shrew and sog genes are required to generate a gradient of dpp activity, which directly specifies the pattern of the dorsal 40% of the embryo.' publications.doi=None publications.firstAuthor='Ferguson E L' publications.id=1007760 publications.issue='3' publications.journal='Development' publications.month='Mar' publications.pages='583-97' publications.pubMedId='1618130' publications.title='Localized enhancement and repression of the activity of the TGF-beta family member, decapentaplegic, is necessary for dorsal-ventral pattern formation in the Drosophila embryo.' publications.volume='114' publications.year=1992\n",
      "Gene: publications.abstractText='Signaling by Decapentaplegic (Dpp), a member of the TGFbeta superfamily of signaling molecules similar to vertebrate BMP2 and BMP4, has been implicated in many developmental processes in Drosophila melanogaster. Notably, Dpp acts as a long-range morphogen during imaginal disc growth and patterning. Genetic approaches led to the identification of a number of gene products that constitute the core signaling pathway. In addition to the ligand-activated heteromeric receptor complex and the signal-transducing intracellular Smad proteins, Dpp signaling requires two nuclear proteins, Schnurri (Shn) and Brinker (Brk), to prime cells for Dpp responsiveness. A complex interplay between the nuclear factors involved in Dpp signaling appears to control the transcriptional readout of the Dpp morphogen gradient. It remains to be seen whether similar molecular mechanisms operate in the nucleus in vertebrate systems.' publications.doi='10.1093/emboj/20.13.3298' publications.firstAuthor='Affolter M' publications.id=1004008 publications.issue='13' publications.journal='EMBO J.' publications.month='Jul' publications.pages='3298-305' publications.pubMedId='11432817' publications.title='Nuclear interpretation of Dpp signaling in Drosophila.' publications.volume='20' publications.year=2001\n",
      "Gene: publications.abstractText=\"Signaling molecules of the transforming growth factor beta (TGF-beta) family contribute to numerous developmental processes in a variety of organisms. However, our understanding of the mechanisms which regulate the activity of and mediate the response to TGF-beta family members remains incomplete. The product of the Drosophila decapentaplegic (dpp) locus is a well-characterized member of this family. We have taken a genetic approach to identify factors required for TGF-beta function in Drosophila by testing for genetic interactions between mutant alleles of dpp and a collection of chromosomal deficiencies. Our survey identified two deficiencies that act as maternal enhancers of recessive embryonic lethal alleles of dpp. The enhanced individuals die with weakly ventralized phenotypes. These phenotypes are consistent with a mechanism whereby the deficiencies deplete two maternally provided factors required for dpp's role in embryonic dorsal-ventral pattern formation. One of these deficiencies also appears todelete a factor required for dpp function in wing vein formation. These deficiencies remove material from the 54F-55A and 66B-66C polytene chromosomal regions, respectively. As neither of these regions has been previously implicated in dpp function, we propose that each of the deficiencies removes a novel factor or factors required for dpp function.\" publications.doi=None publications.firstAuthor='Nicholls R E' publications.id=1007890 publications.issue='1' publications.journal='Genetics' publications.month='May' publications.pages='203-15' publications.pubMedId='9584097' publications.title='Identification of chromosomal regions involved in decapentaplegic function in Drosophila.' publications.volume='149' publications.year=1998\n",
      "Gene: publications.abstractText='Similar to their human counterparts, the Drosophila Rbf1 and Rbf2 Retinoblastoma family members control cell cycle and developmentally regulated gene expression. Increasing evidence suggests that Rbf proteins rely on multiprotein complexes to control target gene transcription. We show here that the developmentally regulated COP9 signalosome (CSN) physically interacts with Rbf2 during embryogenesis. Furthermore, the CSN4 subunit of the COP9 signalosome co-occupies Rbf target gene promoters with Rbf1 and Rbf2, suggesting an active role for the COP9 signalosome in transcriptional regulation. The targeted knockdown of individual CSN subunits leads to diminished Rbf1 and Rbf2 levels and to altered cell cycle progression. The proteasome-mediated destruction of Rbf1 and Rbf2 is increased in cells and embryos with diminished COP9 activity, suggesting that the COP9 signalosome protects Rbf proteins during embryogenesis. Previous evidence has linked gene activation to protein turnover via the promoter-associated proteasome. Our findings suggest that Rbf repression may similarly involve the proteasome and the promoter-associated COP9 signalosome, serving to extend Rbf protein lifespan and enable appropriate programs of retinoblastoma gene control during development.' publications.doi='10.1091/mbc.E06-09-0790' publications.firstAuthor='Ullah Zakir' publications.id=1007749 publications.issue='4' publications.journal='Mol. Biol. Cell' publications.month='Apr' publications.pages='1179-86' publications.pubMedId='17251548' publications.title='Retinoblastoma protein regulation by the COP9 signalosome.' publications.volume='18' publications.year=2007\n",
      "Gene: publications.abstractText='Smad transcription factors lie at the core of one of the most versatile cytokine signaling pathways in metazoan biology-the transforming growth factor-beta (TGFbeta) pathway. Recent progress has shed light into the processes of Smad activation and deactivation, nucleocytoplasmic dynamics, and assembly of transcriptional complexes. A rich repertoire of regulatory devices exerts control over each step of the Smad pathway. This knowledge is enabling work on more complex questions about the organization, integration, and modulation of Smad-dependent transcriptional programs. We are beginning to uncover self-enabled gene response cascades, graded Smad response mechanisms, and Smad-dependent synexpression groups. Our growing understanding of TGFbeta signaling through the Smad pathway provides general principles for how animal cells translate complex inputs into concrete behavior.' publications.doi='10.1101/gad.1350705' publications.firstAuthor='Massagué Joan' publications.id=1007800 publications.issue='23' publications.journal='Genes Dev.' publications.month='Dec' publications.pages='2783-810' publications.pubMedId='16322555' publications.title='Smad transcription factors.' publications.volume='19' publications.year=2005\n",
      "Gene: publications.abstractText='Small differences in the levels of an extracellular signaling molecule can specify cell fate during development. Threshold responses are often determined at the level of transcription. Cell-specific and spatially localized patterns of gene expression depend on combinations of sequence-specific activators and repressors that bind to extensive cis-regulatory regions. Different mechanisms for integrating this complex regulatory information are discussed, particularly the role of coregulatory proteins, which are recruited to the DNA template by sequence-specific transcription factors. Recent studies suggest that a growing set of coactivators and corepressors mediate communication between diverse upstream regulatory proteins and the core RNA polymerase II transcription complex.' publications.doi=None publications.firstAuthor='Mannervik M' publications.id=1007892 publications.issue='5414' publications.journal='Science' publications.month='Apr' publications.pages='606-9' publications.pubMedId='10213677' publications.title='Transcriptional coregulators in development.' publications.volume='284' publications.year=1999\n",
      "Gene: publications.abstractText='The Bicoid-based anterior patterning system of Drosophila embryogenesis appears to be unique to higher dipterans. A new study suggests how this may have evolved out of an alternative mechanism based on cooperating Orthodenticle and Hunchback proteins, the two mechanisms intersecting at the level of downstream target genes.' publications.doi=None publications.firstAuthor='Lynch Jeremy' publications.id=1007748 publications.issue='14' publications.journal='Curr. Biol.' publications.month='Jul' publications.pages='R557-9' publications.pubMedId='12867048' publications.title='Evolution of development: beyond bicoid.' publications.volume='13' publications.year=2003\n",
      "Gene: publications.abstractText=\"The DRE/DREF system plays an important role in transcription of DNA replication genes such as those encoding the 180 and 73 kDa subunits of DNA polymerase alpha as well as that for encoding PCNA. In this study, we found two sequences homologous to DRE (5'-TATCGATA-3') in the 5'-flanking region (-370 to -357 with respect to the transcription initiation site) of the D-raf gene and confirmed transcriptional activity through gel mobility shift assays, transient CAT assays, and spatial patterns of lacZ expression in transgenic larval tissues carrying D-raf and lacZ fusion genes. Further, we demonstrated that the D-raf gene is another target of the Zerknüllt (Zen) protein with observation of D-raf repression by Zen protein in cultured cells and its ectopic expression in the dorsal region of the homozygous zen mutant embryo. The evidence of DRE/DREF involvement in regulation of the D-raf gene obtained in this study strongly supports the idea that the DRE/DREF system is responsible for the coordinated regulation of cell proliferation-related genes in Drosophila.\" publications.doi=None publications.firstAuthor='Ryu J R' publications.id=1007845 publications.issue='4' publications.journal='Nucleic Acids Res.' publications.month='Feb' publications.pages='794-9' publications.pubMedId='9016631' publications.title='Transcriptional regulation of the Drosophila-raf proto-oncogene by the DNA replication-related element (DRE)/DRE-binding factor (DREF) system.' publications.volume='25' publications.year=1997\n",
      "Gene: publications.abstractText='The Dorsal morphogen acts as both an activator and a repressor of transcription in the Drosophila embryo to regulate the expression of dorsal/ventral patterning genes. Circumstantial evidence has suggested that Dorsal is an intrinsic activator and that additional factors (corepressors) convert it into a repressor. These corepressors, however, have previously eluded definitive identification. We show here, via the analysis of embryos lacking the maternally encoded Groucho corepressor and via protein-binding assays, that recruitment of Groucho to the template by protein:protein interactions is required for the conversion of Dorsal from an activator to a repressor. Groucho is therefore a critical component of the dorsal/ventral patterning system.' publications.doi=None publications.firstAuthor='Dubnicoff T' publications.id=1007776 publications.issue='22' publications.journal='Genes Dev.' publications.month='Nov' publications.pages='2952-7' publications.pubMedId='9367978' publications.title='Conversion of dorsal from an activator to a repressor by the global corepressor Groucho.' publications.volume='11' publications.year=1997\n",
      "Gene: publications.abstractText='The Dorsal morphogen directs formation of the Drosophila dorsoventral axis by both activating and repressing transcription. It contains an N-terminal Rel homology domain (RHD), which is responsible for DNA binding and regulated nuclear import, and a C-terminal domain (CTD) that contains activation and repression motifs. To determine if the RHD has a direct role in transcriptional control, we analyzed a series of RHD mutations in S2 cells and embryos. Two classes ofmutations (termed class I and class II mutations) that alter activation without affecting DNA binding or nuclear import were identified. The two classes appear to define distinct protein interaction surfaces on opposite faces of the RHD. Class I mutations enhance an apparently inhibitory interaction between the RHD and the CTD and eliminate both activation and repression by Dorsal. In contrast, class II mutations result in increased activation in S2 cells but severely decreased activation in embryos and have little effect on repression. Analysis of the cuticles of class II mutant embryos suggests that, in the absence of Dorsal-mediated activation, Dorsal-mediated repression is not sufficient to pattern the embryo. These results provide some of the first evidence that the RHD plays an active role in transcriptional regulation in intact multicellular organisms.' publications.doi=None publications.firstAuthor='Jia Songtao' publications.id=1007882 publications.issue='14' publications.journal='Mol. Cell. Biol.' publications.month='Jul' publications.pages='5089-99' publications.pubMedId='12077338' publications.title='The Dorsal Rel homology domain plays an active role in transcriptional regulation.' publications.volume='22' publications.year=2002\n",
      "Gene: publications.abstractText='The Dorsal morphogen is a transcription factor that activates some genes and represses others to establish multiple domains of gene expression along the dorsal/ventral axis of the early Drosophila embryo. Repression by Dorsal appears to require accessory proteins that bind to corepression elements in Dorsal-dependent regulatory modules called ventral repression regions (VRRs). We have identified a corepression element in decapentaplegic (dpp), a zygotically active gene that is repressed by the Dorsal morphogen. This dpp repression element (DRE) is located within a previously identified VRR and close to essential Dorsal-binding sites. We have purified a factor from Drosophila embryo extracts that binds to the DRE but not to mutant forms of the DRE that fail to support efficient repression. This protein also binds to an apparently essential region in a VRR associated with the zerknüllt (zen) gene. One of the DREs in the dpp VRR overlaps the binding site for a potential activator protein suggesting that one mechanism of ventral repression may be the mutually exclusive binding of repressor and activator proteins. We have found the DRE-binding protein to be identical to NTF-1 (equivalent to Elf-1, the product of the grainyhead gene), a factor originally identified as an activator of the Ultrabithorax and Dopa decarboxylase promoters. NTF-1 mRNA is synthesized during oogenesis and deposited in the developing oocyte where it is available to contribute to ventral repression during early embryogenesis. Previous studies have shown that overexpression of NTF-1 in the postblastoderm embryo results in a phenotype that is consistent with a role for this factor in the repression of dpp later in embryogenesis.' publications.doi=None publications.firstAuthor='Huang J D' publications.id=1007712 publications.issue='24' publications.journal='Genes Dev.' publications.month='Dec' publications.pages='3177-89' publications.pubMedId='8543160' publications.title='Binding sites for transcription factor NTF-1/Elf-1 contribute to the ventral repression of decapentaplegic.' publications.volume='9' publications.year=1995\n",
      "Gene: publications.abstractText='The Drosophila Groucho (Gro) protein is the prototype for a large family of corepressors, examples of which are found in most metazoans. This family includes the human transducin-like Enhancer of split (TLE) proteins. As corepressors, Gro/TLE family proteins do not bind to DNA directly, but rather are recruited to the template by DNA-bound repressor proteins. Gro/TLE family proteins are required for many developmental processes, including lateral inhibition, segmentation, sex determination, dorsal/ventral pattern formation, terminal pattern formation, and eye development. These proteins are characterized by a conserved N-terminal glutamine-rich domain and a conserved C-terminal WD-repeat domain. The primary role of the glutamine-rich domain is apparently to mediate tetramerization, while the WD-repeat domain may mediate interactions with DNA-bound repressors. The glutamine rich and WD-repeat domains are separated by a less conserved region containing domains that have been implicated in transcriptional repression and nuclear localization. In addition to encoding full-length Gro/TLE family proteins, most metazoan genomes encode truncated family members that contain the N-terminal oligomerization domain, but lack the C-terminal WD-repeat domain. These truncated proteins may negatively regulate full-length Gro/TLE proteins, perhaps by sequestering them in non-productive complexes. Gro/TLE family proteins probably repress transcription by multiple mechanisms. For example, a glycine/proline-rich domain in the central variable region functions to recruit the histone deacetylase Rpd3 to the template. This histone deacetylase then presumably silences transcription by altering local chromatin structure. Other repression domains in Gro may function in a histone deacetylase-independent manner. Many aspects of Gro/TLE protein function remain to be explored, including the possible post-translational regulation of Gro/TLE activity as well as the mechanisms by which Gro/TLE proteins direct repression at a distance.' publications.doi=None publications.firstAuthor='Chen G' publications.id=1007743 publications.issue='1-2' publications.journal='Gene' publications.month='May' publications.pages='1-16' publications.pubMedId='10831834' publications.title='Groucho/TLE family proteins and transcriptional repression.' publications.volume='249' publications.year=2000\n",
      "Gene: publications.abstractText='The Drosophila Toll pathway takes part in both establishment of the embryonic dorsoventral axis and induction of the innate immune response in adults. Upon activation by the cytokine Spätzle, Toll interacts with the adaptor proteins DmMyD88 and Tube and the kinase Pelle and triggers degradation of the inhibitor Cactus, thus allowing the nuclear translocation of the transcription factor Dorsal/Dif. weckle (wek) was previously identified as a new dorsal group gene that encodes a putative zinc finger transcription factor. However, its role in the Toll pathway was unknown. Here, we isolated new wek alleles and demonstrated that cactus is epistatic to wek, which in turn is epistatic to Toll. Consistent with this, Wek localizes to the plasma membrane of embryos, independently of Toll signaling. Wek homodimerizes and associates with Toll. Moreover, Wek binds to and localizes DmMyD88 to the plasma membrane. Thus, Wek acts as an adaptor to assemble/stabilize a Toll/Wek/DmMyD88/Tube complex. Remarkably, unlike the DmMyD88/tube/pelle/cactus gene cassette of the Toll pathway, wek plays a minimal role, if any, in the immune defense against Gram-positive bacteria and fungi. We conclude that Wek is an adaptor to link Toll and DmMyD88 and is required for efficient recruitment of DmMyD88 to Toll. Unexpectedly, wek is dispensable for innate immune response, thus revealing differences in the Toll-mediated activation of Dorsal in the embryo and Dif in the fat body of adult flies.' publications.doi='10.1016/j.cub.2006.05.050' publications.firstAuthor='Chen Li-Ying' publications.id=1007796 publications.issue='12' publications.journal='Curr. Biol.' publications.month='Jun' publications.pages='1183-93' publications.pubMedId='16782008' publications.title='Weckle is a zinc finger adaptor of the toll pathway in dorsoventral patterning of the Drosophila embryo.' publications.volume='16' publications.year=2006\n",
      "Gene: publications.abstractText='The Drosophila decapentaplegic gene, the Xenopus activin genes and the genes encoding the mouse bone morphogenetic proteins are transforming growth factor-beta-related genes whose roles in development are the focus of current studies. They exhibit elaborate patterns of expression during development, and the protein products have potent effects on the differentiation of specific cell types.' publications.doi=None publications.firstAuthor='Hoffmann F M' publications.id=1007864 publications.issue='6' publications.journal='Curr. Opin. Cell Biol.' publications.month='Dec' publications.pages='947-52' publications.pubMedId='1687650' publications.title='Transforming growth factor-beta-related genes in Drosophila and vertebrate development.' publications.volume='3' publications.year=1991\n",
      "Gene: publications.abstractText='The Drosophila engrailed gene product (En) is a homeodomain-containing protein that contributes to segmental patterning. In transfection assays it acts as a transcriptional repressor. We show that En is an active repressor, blocking activation by mammalian and yeast activators that bind to sites some distance away from those bound by En. Active repression is distinct from the effects of passive homeodomain-containing proteins, which repress when competing with activators for binding sites and activate when competing with En. Active repression activity maps outside the En homeodomain, and this activity can be transferred to a heterologous DNA binding domain.' publications.doi=None publications.firstAuthor='Jaynes J B' publications.id=1007869 publications.issue='6' publications.journal='EMBO J.' publications.month='Jun' publications.pages='1427-33' publications.pubMedId='1673924' publications.title='Active repression of transcription by the engrailed homeodomain protein.' publications.volume='10' publications.year=1991\n",
      "Gene: publications.abstractText='The Drosophila eye field that gives rise to the visual system and dorsal head epidermis forms an unpaired anlage located in the dorsal head ectoderm. The eye field expresses and requires both Dpp and EGFR signaling for its development. As shown in previous studies, EGFR is required for cell maintenance in the developing visual system. Dpp initially switches on the early eye genes so and eya in the eye field. Consecutively, highlevels of Dpp in the dorsal midline inhibit these genes and promote development of head epidermis. We show that Dpp negatively regulates EGFR signaling, thereby increasing the amount of cell death in the dorsal midline. By this mechanism, Dpp controls the formation of a bilateral visual system and indirectly modulates cell death, which is essential for normal head morphogenesis. Loss of either Dpp or its downstream target, Zen, abolishes head epidermis fate and leads to the misexpression of dp-ERK in the dorsal midline. The resulting morphological phenotype consists of cyclopia, reduction of cell death, and failure of head involution. Ectopic expression of activated EGFR inhibits the Dpp target race and thereby causes cyclopia and defective head involution. We discuss possible mechanisms of Dpp and EGFR interaction in the embryo.' publications.doi=None publications.firstAuthor='Chang Ting' publications.id=1007859 publications.issue='1' publications.journal='Dev. Biol.' publications.month='Nov' publications.pages='103-13' publications.pubMedId='14568549' publications.title='Antagonistic relationship between Dpp and EGFR signaling in Drosophila head patterning.' publications.volume='263' publications.year=2003\n",
      "Gene: publications.abstractText='The Drosophila gene bicoid functions as the anterior body pattern organizer of Drosophila. Embryos lacking maternally expressed bicoid fail to develop anterior segments including head and thorax. In wild-type eggs, bicoid mRNA is localized in the anterior pole region and the bicoid protein forms an anterior-to-posterior concentration gradient. bicoid activity is required for transcriptional activation of zygotic segmentation genes and the translational suppression of uniformly distributed maternal caudal mRNA in the anterior region of the embryo. caudal genes as well as other homeobox genes or members of the Drosophila segmentation gene cascade have been found to be conserved in animal evolution. In contrast, bicoid homologs have been identified only in close relatives of the schizophoran fly Drosophila. This poses the question of how the bicoid gene evolved and adopted its unique function in organizing anterior-posterior polarity. We have cloned bicoid from a basal cyclorrhaphan fly, Megaselia abdita (Phoridae, Aschiza), and show that the gene originated from a recent duplication of the direct homolog of the vertebrate gene Hox3, termed zerknüllt, which specifies extraembryonic tissues in insects.' publications.doi=None publications.firstAuthor='Stauber M' publications.id=1007810 publications.issue='7' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Mar' publications.pages='3786-9' publications.pubMedId='10097115' publications.title='The anterior determinant bicoid of Drosophila is a derived Hox class 3 gene.' publications.volume='96' publications.year=1999\n",
      "Gene: publications.abstractText='The Drosophila gene buttonhead (btd) encodes a zinc-finger protein related to the human transcription factor Sp1. btd is expressed in the syncytial blastoderm embryo in a stripe covering the anlagen of the antennal, intercalary and mandibular head segments. btd has been characterized as a head gap gene, since these segments are deleted in btd mutant embryos. We report here that the cis-acting elements required for btd head stripe expression are contained in a 1 kb DNA fragment, located about 3 kb upstream of the promoter. The four maternal coordinate systems are necessary for correct btd head stripe expression, likely by acting through the 1 kb cis-acting control region. Expression of the btd head stripe depends on the anterior morphogen encoded by the gene bicoid (bcd). bcd-dependent activation also involves the activity of the morphogens of the posterior and dorsoventral systems, hunchback and dorsal, respectively, which act together to control the spatial limits of the expression domain. Finally, the terminal system takes part in the regulation of btd head stripe expression by enhancing activation at low levels of activity and repression at high levels of activity.' publications.doi=None publications.firstAuthor='Wimmer E A' publications.id=1007746 publications.issue='2' publications.journal='Mech. Dev.' publications.month='Oct' publications.pages='235-45' publications.pubMedId='8562425' publications.title='Trans- and cis-acting requirements for blastodermal expression of the head gap gene buttonhead.' publications.volume='53' publications.year=1995\n",
      "Gene: publications.abstractText='The Drosophila melanogaster genome was the first metazoan genome to have been sequenced by the whole-genome shotgun (WGS) method. Two issues relating to this achievement were widely debated in the genomics community: how correct is the sequence with respect to base-pair (bp) accuracy and frequency of assembly errors? And, how difficult is it to bring a WGS sequence to the accepted standard for finished sequence? We are now in a position to answer these questions. Our finishing process was designed to close gaps, improve sequence quality and validate the assembly. Sequence traces derived from the WGS and draft sequencing of individual bacterial artificial chromosomes (BACs) were assembled into BAC-sized segments. These segments were brought to high quality, and then joined to constitute the sequence of each chromosome arm. Overall assembly was verified by comparison to a physical map of fingerprinted BAC clones. In the current version of the 116.9 Mb euchromatic genome, called Release 3, the six euchromatic chromosome arms are represented by 13 scaffolds with a total of 37 sequence gaps. We compared Release 3 to Release 2; in autosomal regions of unique sequence, the error rate of Release 2 was one in 20,000 bp. The WGS strategy can efficiently produce a high-quality sequence of a metazoan genome while generating the reagents required for sequence finishing. However, the initial method of repeat assembly was flawed. The sequence we report here, Release 3, is a reliable resource for molecular genetic experimentation and computational analysis.' publications.doi=None publications.firstAuthor='Celniker Susan E' publications.id=1906654 publications.issue='12' publications.journal='Genome Biol.' publications.month=None publications.pages='RESEARCH0079' publications.pubMedId='12537568' publications.title='Finishing a whole-genome shotgun: release 3 of the Drosophila melanogaster euchromatic genome sequence.' publications.volume='3' publications.year=2002\n",
      "Gene: publications.abstractText='The Drosophila morphogen dorsal, KBF1, NF-kappa B, and the proto-oncogene c-rel belong to the rel family of transcription factors whose function is regulated post-translationally by selective nuclear import. In the early Drosophila embryo, dorsal protein is proposed to be retained in the cytoplasm through its interaction with cactus protein. The maternal dorsal group genes constitute a signal transduction pathway, which results in targeting cytoplasmic dorsal protein into the nuclei of the syncytial blastoderm embryo, in a ventral-to-dorsal gradient. The asymmetric transcriptional regulation of zygotic genes along the dorsoventral axis by the dorsal morphogen gradient establishes embryonic dorsoventral polarity. In the lymphocytes, the functional equivalent of cactus is I kappa B, which appears to retain NF-kappa B in the cytoplasm. This retention is relieved by extracellular signals in tissue culture. NF-kappa B and rel proteins each are known to function as oligomeric complexes. Here we present genetic and biochemical evidence for the existence and functional importance of an oligomeric dorsal complex in vivo.' publications.doi=None publications.firstAuthor='Govind S' publications.id=1007840 publications.issue='17' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Sep' publications.pages='7861-5' publications.pubMedId='1518806' publications.title='In vivo self-association of the Drosophila rel-protein dorsal.' publications.volume='89' publications.year=1992\n",
      "Gene: publications.abstractText='The Drosophila nonreceptor protein tyrosine phosphatase, Corkscrew (Csw), functions positively in multiple receptor tyrosine kinase (RTK) pathways, including signaling by the epidermal growth factor receptor (EGFR). Detailed phenotypic analyses of csw mutations have revealed that Csw activity is required in many of the same developmental processes that require EGFR function. However, it is still unclear where in the signaling hierarchy Csw functions relative to other proteins whose activities are also required downstream of the receptor. To address this issue, genetic interaction experiments were performed to place csw gene activity relative to the EGFR, spitz (spi), rhomboid (rho), daughter of sevenless (DOS), kinase-suppressor of ras (ksr), ras1, D-raf, pointed (pnt), and moleskin. We followed the EGFR-dependent formation of VA2 muscle precursor cells as a sensitive assay for these genetic interaction studies. First, we established that Csw has a positive function during mesoderm development. Second, we found that tissue-specific expression of a gain-of-function csw construct rescues loss-of-function mutations in other positive signaling genes upstream of rolled (rl)/MAPK in the EGFR pathway. Third, we were able to infer levels of EGFR signaling in various mutant backgrounds during myogenesis. This work extends previous studies of Csw during Torso and Sevenless RTK signaling to include an in-depth analysis of the role of Csw in the EGFR signaling pathway.' publications.doi=None publications.firstAuthor='Johnson Hamlet M R' publications.id=1007695 publications.issue='3' publications.journal='Genetics' publications.month='Nov' publications.pages='1073-87' publications.pubMedId='11729154' publications.title='Analysis of corkscrew signaling in the Drosophila epidermal growth factor receptor pathway during myogenesis.' publications.volume='159' publications.year=2001\n",
      "Gene: publications.abstractText=\"The Drosophila polypyrimidine tract-binding protein (dmPTB or hephaestus) plays an important role during embryogenesis. A loss of function mutation, heph(03429), results in varied defects in embryonic developmental processes, leading to embryonic lethality. However, the suite of molecular functions that are disrupted in the mutant remains unknown. We have used an unbiased high throughput sequencing approach to identify transcripts that are misregulated in this mutant. Misregulated transcripts show evidence of significantly altered patterns of splicing (exon skipping, 5' and 3' splice site switching), alternative 5' ends, and mRNA level changes (up and down regulation). These findings are independently supported by reverse-transcription-polymerase chain reaction (RT-PCR) analysis and in situ hybridization. We show that a group of genes, such as Zerknüllt, z600 and screw are among the most upregulated in the mutant and have been functionally linked to dorso-ventral patterning and/or dorsal closure processes. Thus, loss of dmPTB function results in specific misregulated transcripts, including those that provide the missing link between the loss of dmPTB function and observed developmental defects in embryogenesis. This study provides the first comprehensive repertoire of genes affected in vivo in the heph mutant in Drosophila and offers insight into the role of dmPTB during embryonic development.\" publications.doi='10.1371/journal.pone.0098585' publications.firstAuthor='Heimiller Joseph' publications.id=1002423 publications.issue='7' publications.journal='PLoS ONE' publications.month=None publications.pages='e98585' publications.pubMedId='25014769' publications.title='Drosophila polypyrimidine tract-binding protein (DmPTB) regulates dorso-ventral patterning genes in embryos.' publications.volume='9' publications.year=2014\n",
      "Gene: publications.abstractText=\"The Drosophila sex-determination switch gene Sex-lethal (Sxl) and the X-chromosome signal element genes (XSEs) that induce the female-specific expression of Sxl are transcribed extremely early in development when most of the genome of this organism is still silent. The DNA sequence CAGGTAG had been implicated in this pre-cellular blastoderm activation of sex-determination genes. A genome-wide computational search, reported here, suggested that CAGGTAG is not specific to early sex-determination genes, since it is over-represented upstream of most genes that are transcribed pre-cellular blastoderm, not just those involved in sex determination. The same search identified similarly over-represented, one-base-pair degenerate sequences as possible functional synonyms of CAGGTAG. We call these heptamers collectively, the TAGteam. Relevance of the TAGteam sequences to pre-cellular blastoderm transcription was established through analysis of TAGteam changes in Sxl, scute (an XSE), and the ;ventral repression element' of the pattern-formation gene zerknüllt. Decreasing the number of TAGteam sites retarded the onset of pre-blastoderm transcription, whereas increasing their number correlated with an advanced onset. Titration of repressors was thought to be the rate-limiting step determining the onset of such early transcription, but this TAGteam dose effect shows that activators must also play an important role in the timing of pre-blastoderm gene expression.\" publications.doi='10.1242/dev.02373' publications.firstAuthor='ten Bosch John R' publications.id=1007803 publications.issue='10' publications.journal='Development' publications.month='May' publications.pages='1967-77' publications.pubMedId='16624855' publications.title='The TAGteam DNA motif controls the timing of Drosophila pre-blastoderm transcription.' publications.volume='133' publications.year=2006\n",
      "Gene: publications.abstractText='The Drosophila short gastrulation gene (sog) encodes a large extracellular protein (Sog) that inhibits signaling by BMP-related ligands. Sog and its vertebrate counterpart Chordin contain four copies of a cysteine repeat (CR) motif defined by 10 cysteine residues spaced in a fixed pattern and a tryptophan residue situated between the first two cysteines. Here we present a structure-function analysis of the CR repeats in Sog, using a series of deletion and point mutation constructs, as well as constructs in which CR domains have been swapped. This analysis indicates that the CR domains are individually dispensable for Sog function but that they are not interchangeable. These studies reveal three different types of Sog activity: intact Sog, which inhibits signaling mediated by the ligand Glass bottom boat (Gbb), a more broadly active class of BMP antagonist referred to as Supersog, and a newly identified activity, which may promote rather than inhibit BMP signaling. Analysis of the activities of CR swap constructs indicates that the CR domains are required for full activity of the various forms of Sog but that the type of Sog activity is determined primarily by surrounding protein sequences. Cumulatively, our analysis suggests that CR domains interact physically with adjacent protein sequences to create forms of Sog with distinct BMP modulatory activities.' publications.doi=None publications.firstAuthor='Yu Kweon' publications.id=1004085 publications.issue='3' publications.journal='Genetics' publications.month='Mar' publications.pages='1323-36' publications.pubMedId='15082551' publications.title='Cysteine repeat domains and adjacent sequences determine distinct bone morphogenetic protein modulatory activities of the Drosophila Sog protein.' publications.volume='166' publications.year=2004\n",
      "Gene: publications.abstractText='The Drosophila trithorax group gene brahma (brm) encodes the ATPase subunit of a 2-MDa chromatin-remodeling complex. brm was identified in a screen for transcriptional activators of homeotic genes and subsequently shown to play a global role in transcription by RNA polymerase II. To gain insight into the targeting, function, and regulation of the BRM complex, we screened for mutations that genetically interact with a dominant-negative allele of brm (brm(K804R)). We first screened for dominant mutations that are lethal in combination with a brm(K804R) transgene under control of the brm promoter. In a distinct but related screen, we identified dominant mutations that modify eye defects resulting from expression of brm(K804R) in the eye-antennal imaginal disc. Mutations in three classes of genes were identified in our screens: genes encoding subunits of the BRM complex (brm, moira, and osa), other proteins directly involved in transcription (zerknullt and RpII140), and signaling molecules (Delta and vein). Expression of brm(K804R) in the adult sense organ precursor lineage causes phenotypes similar to those resulting from impaired Delta-Notch signaling. Our results suggest that signaling pathways may regulate the transcription of target genes by regulating the activity of the BRM complex.' publications.doi='10.1534/genetics.105.041327' publications.firstAuthor='Armstrong Jennifer A' publications.id=1007842 publications.issue='4' publications.journal='Genetics' publications.month='Aug' publications.pages='1761-74' publications.pubMedId='15944353' publications.title='Genetic screens for enhancers of brahma reveal functional interactions between the BRM chromatin-remodeling complex and the delta-notch signal transduction pathway in Drosophila.' publications.volume='170' publications.year=2005\n",
      "Gene: publications.abstractText='The Drosophila z600 gene is a member of an overlapping gene cluster located in the 71CD interval of chromosome 3. We have used several approaches to study the spatial and temporal expression of z600 during embryogenesis. Northern analysis shows that z600 is zygotically expressed, with gene transcripts accumulating transiently during early embryo-genesis. The localization of z600 transcripts in embryo sections by in situ hybridization reveals a dynamic pattern of RNA distribution. z600 RNA is distributed throughout the embryo during the midsyncytial blastoderm stage, becomes concentrated dorsally and posteriorly during cellularization, and persists in the dorsal-most region of the embryo during gastrulation and germ band extension. z600 transcript accumulation is altered in dorsalized or ventralized mutant embryos, suggesting that z600 is directly or indirectly regulated by genes specifying the dorsal-ventral pattern in the embryo. An analysis of germ line transformants harboring a z600-lacZ gene fusion demonstrates that a 276-bp DNA region contains the sequences responsible for dorsal activation, but not ventral repression, of z600 gene expression during early embryogenesis.' publications.doi=None publications.firstAuthor='Schulz R A' publications.id=1007860 publications.issue='1' publications.journal='Dev. Biol.' publications.month='Nov' publications.pages='211-21' publications.pubMedId='2478402' publications.title='Dorsal expression of the Drosophila z600 gene during early embryogenesis.' publications.volume='136' publications.year=1989\n",
      "Gene: publications.abstractText='The GATA transcriptionfactors are a family of C4 zinc finger-motif DNA-binding proteins that play defined roles in hematopoiesis as well as presumptive roles in other tissues where they are expressed (e.g., testis, neuronal and placental trophoblast cells) during vertebrate development. To investigate the possibility that GATA proteins may also be involved in Drosophila development, we have isolated and characterized a gene (dGATAa) encoding a factor that is quite similar to mammalian GATA factors. The dGATAa protein sequence contains the two zinc finger DNA-binding domain of the GATA class but bears no additional sequence similarity to any of the vertebrate GATA factors. Analysis of dGATAa gene transcription during Drosophila development revealed that its mRNA is expressed at high levels during early embryogenesis, with transcripts first appearing in the dorsal portion of the embryo just after cellularization. As development progresses, dGATAa mRNA is present at high levels in the dorsal epidermis, suggesting that dGATAa may be involved in determining dorsal cell fate. The pattern of expression in a variety of dorsoventral polarity mutants indicates that dGATAa lies downstream of the zygotic patterning genes decapentaplegic and zerknüllt.' publications.doi=None publications.firstAuthor='Winick J' publications.id=1007714 publications.issue='4' publications.journal='Development' publications.month='Dec' publications.pages='1055-65' publications.pubMedId='7916677' publications.title='A GATA family transcription factor is expressed along the embryonic dorsoventral axis in Drosophila melanogaster.' publications.volume='119' publications.year=1993\n",
      "Gene: publications.abstractText='The Hox genes play a role in anteroposterior axis specification of bilaterian animals that has been conserved for more than 600 million years. However, some of these genes have occasionally changed their roles in evolution. For example, the insect gene fushi tarazu (ftz), although localised in the Hox cluster, no longer acts as a Hox gene, but is involved in segmentation and nervous system development. Recent data of Mouchel-Vielh et al., and Hughes and Kaufman on ftz homologues in a crustacean and a myriapod, respectively, shed new light onto the evolution of this gene.' publications.doi='10.1002/bies.10180' publications.firstAuthor='Damen Wim G M' publications.id=1007874 publications.issue='11' publications.journal='Bioessays' publications.month='Nov' publications.pages='992-5' publications.pubMedId='12386929' publications.title='fushi tarazu: a Hox gene changes its role.' publications.volume='24' publications.year=2002\n",
      "Gene: publications.abstractText='The Ras-related Rap GTPases are highly conserved across diverse species but their normal biological function is not well understood. Initial studies in mammalian cells suggested a role for Rap as a Ras antagonist. More recent experiments indicate functions in calcium- and cAMP-mediated signaling and it has been proposed that protein kinase A-mediated phosphorylation activates Rap in vivo. We show that Ras1-mediated signaling pathways in Drosophila are not influenced by Rap1 levels, suggesting that Ras1 and Rap1 function via distinct pathways. Moreover, a mutation that abolishes the putative cAMP-dependent kinase phosphorylation site of Drosophila Rap1 can still rescue the Rap1 mutant phenotype. Our experiments show that Rap1 is not needed for cell proliferation and cell-fate specification but demonstrate a critical function for Rap1 in regulating normal morphogenesis in the eye disk, the ovary and the embryo. Rap1 mutations also disrupt cell migrations and cause abnormalities in cell shape. These findings indicate a role for Rap proteins as regulators of morphogenesis in vivo.' publications.doi='10.1093/emboj/18.3.605' publications.firstAuthor='Asha H' publications.id=1004639 publications.issue='3' publications.journal='EMBO J.' publications.month='Feb' publications.pages='605-15' publications.pubMedId='9927420' publications.title='The Rap1 GTPase functions as a regulator of morphogenesis in vivo.' publications.volume='18' publications.year=1999\n",
      "Gene: publications.abstractText='The Rel family of transcription factors participate in a diverse array of processes, including acute responses to injury and infection, lymphocyte differentiation, and embryonic patterning. These proteins show homology in an extended region spanning about 300 amino acids (the Rel homology domain [RHD]). The RHD mediates both DNA binding and interactions with a family of inhibitor proteins, including I kappa B alpha and cactus. Previous studies have shown that an N-terminal region of the RHD (containing the sequence motif RXXRXRXXC) is important for DNA binding, while the C-terminal nuclear localization sequence is important for inhibitor interactions. Here we present a structure-function analysis of the Drosophila dorsal RHD. These studies identify another sequence within the RHD (region I) that is essential for inhibitor interactions. There is a tight correlation between the conservation of region I sequences and the specificity of Rel-inhibitor interactions in both flies and mammals. Point mutations in the region I sequence can uncouple DNA binding and inhibitor interactions in vitro. The phenotypes associated with the expression of a modified dorsal protein in transgenic Drosophila embryos suggest a similar uncoupling in vivo. Recent crystallographic studies suggest that the region I sequence and the nuclear localization sequence might form a composite surface which interacts with inhibitor proteins.' publications.doi=None publications.firstAuthor='Tatei K' publications.id=1007823 publications.issue='7' publications.journal='Mol. Cell. Biol.' publications.month='Jul' publications.pages='3627-34' publications.pubMedId='7791770' publications.title='Specificity of Rel-inhibitor interactions in Drosophila embryos.' publications.volume='15' publications.year=1995\n",
      "Gene: publications.abstractText='The Rel family transcription factor Dorsal patterns the dorsoventral axis of the Drosophila embryo by activating genes such as twist and snail and repressing genes such as decapentaplegic and zerknüllt. Dorsal represses transcription by recruiting the co-repressor Groucho. However, repression occurs only when Dorsal-binding sites are close to binding sites for other factors that also bind Groucho. The need for additional factors to assist Dorsal in repression may result from the intrinsically weak interaction between Dorsal and Groucho. To test this idea, we generated a Dorsal variant containing a high-affinity Groucho recruitment motif at its C terminus. As predicted, this variant functions as a dedicated repressor, silencing decapentaplegic and zerknüllt while failing to activate twist and snail. We also converted Dorsal into a dedicated activator by replacing its weak Groucho-recruitment motif with heterologous activation domains. Although the dedicated activator alleles fail to repress decapentaplegic and zerknüllt in the syncytial blastoderm embryo, they are able to pattern the dorsoventral axis. This indicates that dorsoventral patterning is not dependent upon Dorsal-mediated repression, reflecting the existence of redundant mechanisms to block Decapentaplegic signaling.' publications.doi='10.1242/dev.02643' publications.firstAuthor='Ratnaparkhi Girish S' publications.id=1007865 publications.issue='22' publications.journal='Development' publications.month='Nov' publications.pages='4409-14' publications.pubMedId='17035291' publications.title='Uncoupling dorsal-mediated activation from dorsal-mediated repression in the Drosophila embryo.' publications.volume='133' publications.year=2006\n",
      "Gene: publications.abstractText='The TGF-β pathway is an evolutionarily conserved signal transduction module that mediates diverse biological processes in animals. In Drosophila, both the BMP and Activin branches are required for viability. Studies rooted in classical and molecular genetic approaches continue to uncover new developmental roles for TGF-β signaling. We present an overview of the secreted ligands, transmembrane receptors and cellular Smad transducer proteins that compose the core pathway in Drosophila. An assortment of tools have been developed to conduct tissue-specific loss- and gain-of-function experiments for these pathway components. We discuss the deployment of these reagents, with an emphasis on appropriate usage and limitations of the available tools. Throughout, we note reagents that are in need of further improvement or development, and signaling features requiring further study. A general theme is that comparison of phenotypes for ligands, receptors, and Smads can be used to map tissue interactions, and to separate canonical and non-canonical signaling activities. Core TGF-β signaling components are subject to multiple layers of regulation, and are coupled to context-specific inputs and outputs. In addition to fleshing out how TGF-β signaling serves the fruit fly, we anticipate that future studies will uncover new regulatory nodes and modes and will continue to advance paradigms for how TGF-β signaling regulates general developmental processes.' publications.doi='10.1016/j.ymeth.2014.03.016' publications.firstAuthor='Peterson Aidan J' publications.id=1004089 publications.issue='1' publications.journal='Methods' publications.month='Jun' publications.pages='183-93' publications.pubMedId='24680699' publications.title='Strategies for exploring TGF-β signaling in Drosophila.' publications.volume='68' publications.year=2014\n",
      "Gene: publications.abstractText='The TNF and TNFR superfamilies of proteins are conserved throughout evolution. The first invertebrate orthologs of TNF and TNFR, Eiger and Wengen, were identified in Drosophila, which enabled us to take advantage of its powerful genetics. Indeed, genetic studies on Eiger in the last decade have discovered their signaling mechanisms through activation of the JNK pathway and unveiled the role of Eiger-JNK signaling in a variety of cellular and tissue processes such as cell death, cell proliferation, tissue growth regulation, host defense, pain sensitization, and canalization. In this review, we will describe the in vivo signaling of Eiger and its physiological roles in fly development and homeostasis, and will discuss the evolution of the TNF/TNFR systems.' publications.doi='10.1016/j.smim.2014.05.003' publications.firstAuthor='Igaki Tatsushi' publications.id=1007806 publications.issue='3' publications.journal='Semin. Immunol.' publications.month='Jun' publications.pages='267-74' publications.pubMedId='24981286' publications.title='The Drosophila TNF ortholog Eiger: emerging physiological roles and evolution of the TNF system.' publications.volume='26' publications.year=2014\n",
      "Gene: publications.abstractText='The Transforming Growth Factor-beta superfamily member decapentaplegic (dpp) acts as an extracellular morphogen to pattern the embryonic ectoderm of the Drosophila embryo. To identify components of the dpp signaling pathway, we screened for mutations that act as dominant maternal enhancers of a weak allele of the dpp target gene zerknŁllt. In this screen, we recovered new alleles of the Mothers against dpp (Mad) and Medea genes. Phenotypic analysis of the new Medea mutations indicates that Medea, like Mad, is required for both embryonic and imaginal disc patterning. Genetic analysis suggests that Medea may have two independently mutable functions in patterning the embryonic ectoderm. Complete elimination of maternal and zygotic Medea activity in the early embryo results in a ventralized phenotype identical to that of null dpp mutants, indicating that Medea is required for all dpp-dependent signaling in embryonic dorsal-ventral patterning. Injection of mRNAs encoding DPP or a constitutively activated form of the DPP receptor, Thick veins, into embryos lacking all Medea activity failed to induce formation of any dorsal cell fates, demonstrating that Medea acts downstream of the thick veins receptor. We cloned Medea and found that it encodes a protein with striking sequence similarity to human SMAD4. Moreover, injection of human SMAD4 mRNA into embryos lacking all Medea activity conferred phenotypic rescue of the dorsal-ventral pattern, demonstrating conservation of function between the two gene products.' publications.doi=None publications.firstAuthor='Hudson J B' publications.id=1007786 publications.issue='8' publications.journal='Development' publications.month='Apr' publications.pages='1407-20' publications.pubMedId='9502722' publications.title='The Drosophila Medea gene is required downstream of dpp and encodes a functional homolog of human Smad4.' publications.volume='125' publications.year=1998\n",
      "Gene: publications.abstractText='The conservation of Homeotic (Hox) gene clustering and colinearity in many metazoans indicates that functional constraints operate on this genome organization. However, several studies have questioned its relevance in Drosophila. Here, we analyse the genomic organization of Hox and Hox-derived genes in 13 fruitfly species and the mosquito Anopheles gambiae. We found that at least seven different Homeotic complex (HOM-C) arrangements exist among Drosophila species, produced by three major splits, five microinversions and six gene transpositions. This dynamism contrasts with the stable organization of the complex in many other taxa. Although there is no evidence of an absolute requirement for Hox gene clustering in Drosophila, we found that strong functional constraints act on the individual genes.' publications.doi='10.1016/j.tig.2006.12.001' publications.firstAuthor='Negre Bárbara' publications.id=1007633 publications.issue='2' publications.journal='Trends Genet.' publications.month='Feb' publications.pages='55-9' publications.pubMedId='17188778' publications.title='HOM-C evolution in Drosophila: is there a need for Hox gene clustering?' publications.volume='23' publications.year=2007\n",
      "Gene: publications.abstractText='The developmental control genes containing an Antennapedia-type homeobox are clustered in insects and vertebrates. The evolution of these genes was studied by the construction of evolutionary trees and by statistical geometry in sequence space. The comparative analysis of the homeobox sequences reveals the subdivision of the Antennapedia-type homeobox genes into three classes early in metazoan evolution. This observation suggests an important function of these genes even in the most primitive metazoans. Subsequent duplication events generated a cluster of at least five homeobox genes in the last common ancestor of insects and vertebrates. These genes later independently gave rise to the 13 groups of paralogous genes in vertebrates and to the 11 Antennapedia-type genes in the Drosophila complexes.' publications.doi=None publications.firstAuthor='Schubert F R' publications.id=1007861 publications.issue='1' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Jan' publications.pages='143-7' publications.pubMedId='8093557' publications.title='The Antennapedia-type homeobox genes have evolved from three precursors separated early in metazoan evolution.' publications.volume='90' publications.year=1993\n",
      "Gene: publications.abstractText='The discovery of conserved protein domains found in many Drosophila and mammalian developmental gene products suggests that fundamental developmental processes are conserved throughout evolution. Our understanding of development has been enhanced by the discovery of the widespread role of the homeodomain (HD). The action of HD-containing proteins as transcriptional regulators is mediated through a helix-turn-helix motif which confers sequence specific DNA binding. Unexpectedly, the well conserved structural homology between the HD and the prokaryotic helix-turn-helix proteins contrasts with their divergent types of physical interaction with DNA. A C-terminal extension of the HD recognition helix has assumed the role that the N-terminus of the prokaryotic helix plays for specification of DNA binding preference. However, the HD appears also capable of recognizing DNA in an alternative way and its specificity in vivo may be modified by regions outside the helix-turn-helix motif. We propose that this intrinsic complexity of the HD, as well as its frequent association with other DNA binding domains, explains the functional specificity achieved by genes encoding highly related HDs.' publications.doi='10.1002/bies.950140302' publications.firstAuthor='Treisman J' publications.id=1007862 publications.issue='3' publications.journal='Bioessays' publications.month='Mar' publications.pages='145-50' publications.pubMedId='1350195' publications.title='The homeodomain: a new face for the helix-turn-helix?' publications.volume='14' publications.year=1992\n",
      "Gene: publications.abstractText=\"The discovery of the striking positional conservation between the Antennapedia and Bithorax homeotic gene complexes (ANT-C and BX-C) in Drosophila melanogaster and the murine Hox and human HOX clusters has had a substantial impact on our understanding of the evolution of development and its genetic regulation. Structural differences do exist among the mammalian Hox complexes and the ANT-C in D. melanogaster. To gain further insight into the evolutionary changes among these complexes, the ANT-C was cloned in the closely related species, Drosophila pseudoobscura. The overall structure of the ANT-C in D. pseudoobscura is highly similar to its D. melanogaster counterpart; however, two differences in the organization of the ANT-C have been identified. First, the z2 gene, a member of the ANT-C in D. melanogaster, is not present in the D. pseudoobscura ANT-C and is possibly absent from the D. pseudoobscura genome. Second, the orientation of the Deformed gene is inverted in D. pseudoobscura, providing it with a 5' to 3' direction of transcription identical to the remaining ANT-C homeobox genes with the exception of fushi tarazu. These differences demonstrate that subtle changes can occur in ANT-C structure during relatively short periods of evolutionary divergence, although the fundamental organization of the complex is conserved. These observations and others suggest that the complex is not absolutely rigid but that selective pressures have maintained this organization of genes for some functional reason that remains elusive.\" publications.doi=None publications.firstAuthor='Randazzo F M' publications.id=1007812 publications.issue='1' publications.journal='Genetics' publications.month='May' publications.pages='319-30' publications.pubMedId='8099892' publications.title='Structural changes in the antennapedia complex of Drosophila pseudoobscura.' publications.volume='134' publications.year=1993\n",
      "Gene: publications.abstractText='The dorsal (dl) morphogen gradient is responsible for initiating the differentiation of the mesoderm, neuroectoderm and dorsal ectoderm in the Drosophila embryo. dl encodes a sequence-specific DNA binding protein that belongs to the Rel family of transcription factors. Previous studies have shown that dl activates the mesoderm determinant twist (twi); here we use a combination of site-directed mutagenesis and P-transformation assays to demonstrate that it also functions as a direct transcriptional repressor of a second target gene, zerknüllt (zen). By exchanging dl binding sites between the promoters we show that activator sites from twi can mediate repression when placed in the context of the zen promoter, and that repressor sites from zen can mediate activation in the context of the twi promoter. This represents the first demonstration that common binding sites for any DNA binding protein can mediate both activation and repression in a developing embryo. Evidence is also presented that the affinities of dl binding sites are important for the efficiency of repression, but are not the sole determinants of the threshold response to the dl gradient.' publications.doi=None publications.firstAuthor='Jiang J' publications.id=1007856 publications.issue='8' publications.journal='EMBO J.' publications.month='Aug' publications.pages='3147-54' publications.pubMedId='1322296' publications.title='Individual dorsal morphogen binding sites mediate activation and repression in the Drosophila embryo.' publications.volume='11' publications.year=1992\n",
      "Gene: publications.abstractText='The dorsal (dl) nuclear gradient initiates the differentiation of the mesoderm, neuroectoderm, and dorsal ectoderm by activating and repressing gene expression in the early Drosophila embryo. This gradient is organized by a Toll signaling pathway that shares many common features with the mammalian IL-1 cytokine pathway. Here we present evidence that a second signaling pathway, controlled by the torso (tor) receptor tyrosine kinase, also modulates dl activity. Evidence is presented that the tor pathway selectively masks the ability of dl to repress gene expression but has only a slight effect on activation. Intracellular kinases that are thought to function downstream of tor, such as D-raf and the rolled MAP kinase, mediate this selective block in repression. Normally, the Toll and tor pathways are both active only at the embryonic poles, and consequently, target genes (zen and dpp) that are repressed in middle body regions are expressed at these sites. Constitutive activation of the tor pathway causes severe embryonic defects, including disruptions in gastrulation and mesoderm differentiation, as a result of misregulation of dl target genes. These results suggest that RTK signaling pathways can control gene expression by antirepression, and that multiple pathways can fine-tune the activities of a single transcription factor.' publications.doi=None publications.firstAuthor='Rusch J' publications.id=1007826 publications.issue='11' publications.journal='Genes Dev.' publications.month='Jun' publications.pages='1247-57' publications.pubMedId='7926728' publications.title='Regulation of the dorsal morphogen by the Toll and torso signaling pathways: a receptor tyrosine kinase selectively masks transcriptional repression.' publications.volume='8' publications.year=1994\n",
      "Gene: publications.abstractText='The dorsal (dl) protein gradient determines patterns of gene expression along the dorsal-ventral axis of the Drosophila embryo. dl protein is at peak levels in ventral nuclei of the embryo where it activates some genes (twist and snail) and represses others [zerknullt (zen), decapentaplegic and tolloid]. It is a member of the rel family of transcription factors and interacts with specific DNA sequences in the regulatory regions of its target genes. These sequences (dl binding sites), when taken from the context of either an activated or repressed promoter, mediate transcriptional activation of a heterologous promoter, but not repression. We found that T-rich sequences close to the dl binding sites in the silencer region of the zen promoter are conserved between three Drosophila species. Using this sequence information we defined a minimal element that can mediate repression of a heterologous promoter. This element interacts with at least two factors present in embryonic extracts, one of which is dl protein. The other factor binds to the T-rich site. Point mutations in either site abolish ventral repression in vivo. In addition, mutations in the T-rich site cause ectopic expression in ventral regions indicating that the minimal silencer was converted into an enhancer.' publications.doi=None publications.firstAuthor='Kirov N' publications.id=1007843 publications.issue='8' publications.journal='EMBO J.' publications.month='Aug' publications.pages='3193-9' publications.pubMedId='8344256' publications.title='Conversion of a silencer into an enhancer: evidence for a co-repressor in dorsal-mediated repression in Drosophila.' publications.volume='12' publications.year=1993\n",
      "Gene: publications.abstractText='The dorsal-ventral pattern of the Drosophila egg is established during oogenesis. Epidermal growth factor receptor (Egfr) signaling within the follicular epithelium is spatially regulated by the dorsally restricted distribution of its presumptive ligand, Gurken. As a consequence, pipe is transcribed in a broad ventral domain to initiate the Toll signaling pathway in the embryo, resulting in a gradient of Dorsal nuclear translocation. We show that expression of pipe RNA requires the action of fettucine (fet) in ovarian follicle cells. Loss of maternal fet activity produces a dorsalized eggshell and embryo. Although similar mutant phenotypes are observed with regulators of Egfr signaling, genetic analysis suggests that fet acts downstream of this event. The fet mutant phenotype is rescued by a transgene of capicua (cic), which encodes an HMG-box transcription factor. We show that Cic protein is initially expressed uniformly in ovarian follicle cell nuclei, and is subsequently downregulated on the dorsal side. Earlier studies described a requirement for cic in repressing zygotic target genes of both the torso and Toll pathways in the embryo. Our experiments reveal that cic controls dorsal-ventral patterning by regulating pipe expression in ovarian follicle cells, before its previously described role in interpreting the Dorsal gradient.' publications.doi=None publications.firstAuthor='Goff D J' publications.id=1007821 publications.issue='22' publications.journal='Development' publications.month='Nov' publications.pages='4553-62' publications.pubMedId='11714680' publications.title='Establishment of dorsal-ventral polarity of the Drosophila egg requires capicua action in ovarian follicle cells.' publications.volume='128' publications.year=2001\n",
      "Gene: publications.abstractText='The dorsal-ventral pattern of the Drosophila embryo is established by three sequential signaling pathways. Each pathway transmits spatial information by localizing the activity of an extracellular signal, which acts as a ligand for a broadly distributed transmembrane receptor. The components of the first two pathways are encoded by maternal effect genes, while the third pathway is specified by genes expressed in the zygote. During oogenesis, the oocyte transmits a signal to the surrounding follicle cells by the gurken-torpedo pathway. After fertilization, the initial asymmetry of the egg chamber is used by the spätzle-Toll pathway to generate within the embryo a nuclear gradient of the transcription factor Dorsal, which regulates the regional expression of a set of zygotic genes. On the dorsal side of the embryo, the decapentaplegic-punt/thick veins pathway then establishes patterning of the amnioserosa and dorsal ectoderm. Each pathway uses a distinct strategy to achieve spatial localization of signaling activity.' publications.doi='10.1146/annurev.ge.29.120195.002103' publications.firstAuthor='Morisato D' publications.id=1002618 publications.issue=None publications.journal='Annu. Rev. Genet.' publications.month=None publications.pages='371-99' publications.pubMedId='8825480' publications.title='Signaling pathways that establish the dorsal-ventral pattern of the Drosophila embryo.' publications.volume='29' publications.year=1995\n",
      "Gene: publications.abstractText='The earliest stages of development in most metazoans aredriven by maternally deposited proteins and mRNAs, with widespread transcriptional activation of the zygotic genome occurring hours after fertilization, at a period known as the maternal-to-zygotic transition (MZT). In Drosophila, the MZT is preceded by the transcription of a small number of genes that initiate sex determination, patterning, and other early developmental processes; and the zinc-finger protein Zelda (ZLD) plays a key role in their transcriptional activation. To better understand the mechanisms of ZLD activation and the range of its targets, we used chromatin immunoprecipitation coupled with high-throughput sequencing (ChIP-Seq) to map regions bound by ZLD before (mitotic cycle 8), during (mitotic cycle 13), and after (late mitotic cycle 14) the MZT. Although only a handful of genes are transcribed prior to mitotic cycle 10, we identified thousands of regions bound by ZLD in cycle 8 embryos, most of which remain bound through mitotic cycle 14. As expected, early ZLD-bound regions include the promoters and enhancers of genes transcribed at this early stage. However, we also observed ZLD bound at cycle 8 to the promoters of roughly a thousand genes whose first transcription does not occur until the MZT and to virtually all of the thousands of known and presumed enhancers bound at cycle 14 by transcription factors that regulate patterned gene activation during the MZT. The association between early ZLD binding and MZT activity is so strong that ZLD binding alone can be used to identify active promoters and regulatory sequences with high specificity and selectivity. This strong early association of ZLD with regions not active until the MZT suggests that ZLD is not only required for the earliest wave of transcription but also plays a major role in activating the genome at the MZT.' publications.doi='10.1371/journal.pgen.1002266' publications.firstAuthor='Harrison Melissa M' publications.id=1007815 publications.issue='10' publications.journal='PLoS Genet.' publications.month='Oct' publications.pages='e1002266' publications.pubMedId='22028662' publications.title='Zelda binding in the early Drosophila melanogaster embryo marks regions subsequently activated at the maternal-to-zygotic transition.' publications.volume='7' publications.year=2011\n",
      "Gene: publications.abstractText='The establishment of expression domains of developmentally regulated genes depends on cues provided by different concentrations of transcriptional activators and repressors. Here we analyze the regulation of the Drosophila gene zen, which is a target of the Decapentaplegic (Dpp) signaling pathway during cellular blastoderm formation. We show that low levels of the Dpp signal transducer p-Mad (phosphorylated Mad), together with the recently discovered negative regulator Brinker (Brk), define the spatial limits of zen transcription in a broad dorsal-on/ventral-off domain. The subsequent refinement of this pattern to the dorsal-most cells, however, correlates with high levels of p-Mad that accumulate in the same region during late blastoderm. Examination of the zen regulatory sequences revealed the presence of multiple Mad and Brk binding sites, and our results indicate that a full occupancy of the Mad sites due to high concentrations of nuclear Mad is the primary mechanism for refinement of zen. Interestingly, several Mad and Brk binding sites overlap, and we show that Mad and Brk cannot bind simultaneously to such sites. We propose a model whereby competition between Mad and Brk determines spatially restricted domains of expression of Dpp target genes.' publications.doi='10.1101/gad.861401' publications.firstAuthor='Rushlow C' publications.id=1007684 publications.issue='3' publications.journal='Genes Dev.' publications.month='Feb' publications.pages='340-51' publications.pubMedId='11159914' publications.title='Transcriptional regulation of the Drosophila gene zen by competing Smad and Brinker inputs.' publications.volume='15' publications.year=2001\n",
      "Gene: publications.abstractText='The establishment of the dorsal-ventral pattern in Drosophila embryos depends on a signal transduction process: a putative extracellular ligand released into the perivitelline space surrounding the embryo binds to the Toll receptor. Toll activation triggers the formation of the nuclear gradient of dorsal protein, the morphogen of the dorsal-ventral axis. Here, I analyse the dorsal protein distribution and the expression of zygotic dorsal-ventral genes in Toll- embryos that have been injected with wild-type cytoplasm under a variety of different injection conditions. Injections into two positions within a single embryo lead to the formation of two dorsal-ventral patterns in one embryo, allowing the analysis of interactions between pattern-forming processes. The results of single and double injections suggest that the spatial information for the embryonic dorsal-ventral axis is largely derived from spatial cues present in the extraembryonic compartment, which restrict the release of the putative Toll ligand. They argue against a Toll-dependent pattern-formation process employing local self-enhancement and lateral inhibition to enhance a weak initial asymmetry. The putative Toll ligand appears to originate from a ventrally restricted zone which extends along the entire anterior-posterior axis. Ligand diffusion or its graded release are required to determine the slope of the nuclear dorsal protein gradient. Both the Toll receptor and the putative ligand of Toll are in excess in wild-type embryos. Since spatial information for the embryonic dorsal-ventral axis is already present in the vitelline membrane or the perivitelline space, it is most likely generated during oogenesis. Oogenic pattern formation is also responsible for the perpendicular orientation the dorsal-ventral axis maintains with respect to the anterior-posterior axis.' publications.doi=None publications.firstAuthor='Roth S' publications.id=1007802 publications.issue='4' publications.journal='Development' publications.month='Apr' publications.pages='1385-96' publications.pubMedId='8404539' publications.title='Mechanisms of dorsal-ventral axis determination in Drosophila embryos revealed by cytoplasmic transplantations.' publications.volume='117' publications.year=1993\n",
      "Gene: publications.abstractText='The existence of a gene complex in the proximal right arm of chromosome 3 of Drosophila melanogaster involved in the development of the head and thorax was originally suggested by the phenotypes of several dominant homoeotic mutations and their revertants. A screen for mutations utilizing Df(3R) Antp(Ns+R17) (proximally broken in salivary region 84B1,2) yielded, among 102 recovered mutations, 17 localized by deficiency mapping to the putative homoeotic cluster. These fell into four complementation groups, two of which were characterized by homoeotic phenotypes. To explore the limits of the Antennapedia gene complex (ANT-C) more proximally, a second screen has been undertaken utilizing Df(3R)Scr, a deficiency of 84A1-B1,2.-Of 2832 chromosomes screened, 21 bearing alterations localized to polytene interval 84A-84B1,2 have been recovered. Sixteen are recessive lethals, and five showing reduced viability display a visible phenotype in surviving individuals. Complementation and phenotypic analyses revealed four complementation groups proximal to those identified in the previous screen, including two new alleles of the recessive homoeotic mutation, proboscipedia (pb). Ten of the new mutations correspond to complementation groups defined previously in the Df(3R)Antp(Ns+R17) screen four to the EbR11 group, two to the Scr group and four to the Antp group.-On the basis of the phenotypes of the 39 mutations localized to this region, plus their interactions with extant homoeotic mutations, we postulate that there are at least five functional sites comprising the ANT-C. Three have been demonstrated to be homoeotic in nature. The specific homoeotic transformations thus far observed suggest that these loci are critical for normal development of adult labial, maxillary and thoracic structures.' publications.doi=None publications.firstAuthor='Lewis R A' publications.id=1005519 publications.issue='2' publications.journal='Genetics' publications.month='Jun' publications.pages='383-97' publications.pubMedId='17249042' publications.title='Genetic Analysis of the Antennapedia Gene Complex (Ant-C) and Adjacent Chromosomal Regions of DROSOPHILA MELANOGASTER. II. Polytene Chromosome Segments 84A-84B1,2.' publications.volume='95' publications.year=1980\n",
      "Gene: publications.abstractText=\"The expression of the chloramphenicol acetyltransferase gene under control of the 1152-base pair 5'-flanking region (-1107 to +45 nucleotide positions with respect to the major transcription initiation site) of the Drosophila DNA polymerase alpha gene was repressed by cotransfection into Drosophila Kc cells with a zerknüllt (zen)-expressing plasmid as previously observed with the proliferating cell nuclear antigen (PCNA) gene promoter. The expression of the zen resulted in reduction of the abundance of mRNA for the transfected chloramphenicol acetyltransferase gene and also mRNAs for both DNA polymerase alpha and PCNA. Results obtained using various deletion derivatives of the promoter region and chemically synthesized oligonucleotides of the DNA replication-related element (DRE), a positive cis-acting element found in both DNA polymerase alpha and PCNA genes, revealed that the DRE sequences are responsible to repression by Zen protein. The nuclear extract of Kc cells transfected by the zen-expressing plasmid contained lesser amounts of the DRE-binding factor (DREF) than that of untransfected or mutant zen-transfected cells. These results suggest that the Zen protein represses expression of DNA replication-related genes by reducing DREF, although the detailed mechanism of the repression remains to be elucidated.\" publications.doi=None publications.firstAuthor='Hirose F' publications.id=1007848 publications.issue='4' publications.journal='J. Biol. Chem.' publications.month='Jan' publications.pages='2937-42' publications.pubMedId='7905482' publications.title='Repression of regulatory factor for Drosophila DNA replication-related gene promoters by zerknüllt homeodomain protein.' publications.volume='269' publications.year=1994\n",
      "Gene: publications.abstractText=\"The family of Hox-proteins has been a major focus of research for over 30 years. Hox-proteins are crucial to the correct development of bilateral organisms, however, some uncertainty remains as to which Hox-proteins are functionally equivalent across different species. Initial classification of Hox-proteins was based on phylogenetic analysis of the 60 amino acid homeodomain. This approach was successful in classifying Hox-proteins with differing homeodomains, but the relationships of Hox-proteins with nearly identical homeodomains, yet distinct biological functions, could not be resolved. Correspondingly, these 'problematic' proteins were classified into one large unresolved group. Other classifications used the relative location of the Hox-protein coding genes on the chromosome (synteny) to further resolve this group. Although widely used, this synteny-based classification is inconsistent with experimental evidence from functional equivalence studies. These inconsistencies led us to re-examine and derive a new classification for the Hox-protein family using all Hox-protein sequences available in the GenBank non-redundant protein database (NCBI-nr). We compare the use of the homeodomain, the homeodomain with conserved flanking regions (the YPWM and linker region), and full length Hox-protein sequences as a basis for classification of Hox-proteins. In contrast to previous attempts, our approach is able to resolve the relationships for the 'problematic' as well as ABD-B-like Hox-proteins. We highlight differences to previous classifications and clarify the relationships of Hox-proteins across the five major model organisms, Caenorhabditis elegans, Drosophila melanogaster, Branchiostoma floridae, Mus musculus and Danio rerio. Comparative and functional analysis of Hox-proteins, two fields crucial to understanding the development of bilateral organisms, have been hampered by difficulties in predicting functionally equivalent Hox-proteins across species. Our classification scheme offers a higher-resolution classification that is in accordance with phylogenetic as well as experimental data and, thereby, provides a novel basis for experiments, such as comparative and functional analyses of Hox-proteins.\" publications.doi='10.1371/journal.pone.0010820' publications.firstAuthor='Hueber Stefanie D' publications.id=1007880 publications.issue='5' publications.journal='PLoS ONE' publications.month=None publications.pages='e10820' publications.pubMedId='20520839' publications.title='Improving Hox protein classification across the major model organisms.' publications.volume='5' publications.year=2010\n",
      "Gene: publications.abstractText='The flour beetle Tribolium castaneum has become an important model organism for comparative studies of insect development. Many developmentally important genes have now been cloned from both Tribolium and Drosophila and their expression characteristics were studied. We analyze here the complete coding sequences of 17 homologous gene pairs from D. melanogaster and T. castaneum, most of which encode transcription factors. We find that the Tribolium genes are on average 30% shorter than their Drosophila homologues. This appears to be due largely to the almost-complete absence of trinucleotide repeats in the coding sequences of Tribolium as well as the generally lower degree of internal repetitiveness. Clusters of polar and other amino acids such as glutamine, proline, and serine, which are often considered to be important for transcriptional activation domains in Drosophila, are almost completely absent in Tribolium. Codon usage is generally less biased in Tribolium, although we find a similar tendency for the preference of G- or C-ending codons and a higher bias in conserved subregions of the proteins as in Drosophila. Most of the aminoacid substitutions in the DNA-binding domains of the transcription factors occur at residues that do not make a specific contact to DNA, suggesting that the recognition sequences are likely to be conserved between the two species.' publications.doi=None publications.firstAuthor='Schmid K J' publications.id=1007683 publications.issue='5' publications.journal='J. Mol. Evol.' publications.month='Nov' publications.pages='558-66' publications.pubMedId='10552037' publications.title='A comparison of homologous developmental genes from Drosophila and Tribolium reveals major differences in length and trinucleotide repeat content.' publications.volume='49' publications.year=1999\n",
      "Gene: publications.abstractText='The fly Drosophila melanogaster is one of the most intensively studied organisms in biology and serves as a model system for the investigation of many developmental and cellular processes common to higher eukaryotes, including humans. We have determined the nucleotide sequence of nearly all of the approximately 120-megabase euchromatic portion of the Drosophila genome using a whole-genome shotgun sequencing strategy supported by extensive clone-based sequence and a high-quality bacterial artificial chromosome physical map. Efforts are under way to close the remaining gaps; however, the sequence is of sufficient accuracy and contiguity to be declared substantially complete and to support an initial analysis of genome structure and preliminary gene annotation and interpretation. The genome encodes approximately 13,600 genes, somewhat fewer than the smaller Caenorhabditis elegans genome, but with comparable functional diversity.' publications.doi=None publications.firstAuthor='Adams M D' publications.id=1002171 publications.issue='5461' publications.journal='Science' publications.month='Mar' publications.pages='2185-95' publications.pubMedId='10731132' publications.title='The genome sequence of Drosophila melanogaster.' publications.volume='287' publications.year=2000\n",
      "Gene: publications.abstractText='The fork head (fkh) domain defines the DNA-binding region of a family of transcription factors which has been implicated in regulating cell fate decisions across species lines. We have cloned and molecularly characterized the crocodile (croc) gene which encodes a new family member from Drosophila. croc is expressed in the head anlagen of the blastoderm embryo under the control of the anterior, the dorsoventral and the terminal maternal organizer systems. The croc mutant phenotype indicates that the croc wild-type gene is required to function as an early patterning gene in the anterior-most blastoderm head segment anlage and for the establishment of a specific head skeletal structure that derives from the non-adjacent intercalary segment at a later stage of embryogenesis. As an early patterning gene, croc exerts unusual properties which do not allow it to be grouped among the established segmentation genes. A single-site mutation within the croc fkh domain, which causes a replacement of the first out of four conserved amino acid residues thought to be involved in the coordinate binding of Mg2+, abolishes the DNA binding of the protein in vitro. In view of the resulting lack-of-function mutant phenotype, it appears likely that metal binding by the affected region of the fkh domain is crucial for proper folding of the DNA-binding structure.' publications.doi=None publications.firstAuthor='Häcker U' publications.id=1007692 publications.issue='21' publications.journal='EMBO J.' publications.month='Nov' publications.pages='5306-17' publications.pubMedId='7489720' publications.title='The Drosophila fork head domain protein crocodile is required for the establishment of head structures.' publications.volume='14' publications.year=1995\n",
      "Gene: publications.abstractText='The gene regulatory network of a developmental process contains many mutually repressive interactions between two genes. They are often regulated by or regulate an additional factor, which constitute prominent network motifs, called regulated and regulating mutual loops. Our database analysis on the gene regulatory network for Drosophila melanogaster indicates that those with mutual repression are working specifically for the segmentation process. To clarify their biological roles, we mathematically study the response of the regulated mutual loop with mutual repression to input stimuli. We show that the mutual repression increases the response sensitivity without affecting the threshold input level to activate the target gene expression, as long as the network output is unique for a given input level. This high sensitivity of the motif can contribute to sharpening the spatial domain pattern without changing its position, assuring a robust developmental process. We also study transient dynamics that shows shift of domain boundary, agreeing with experimental observations. Importance of mutual repression is addressed by comparingwith other types of regulations.' publications.doi='10.1016/j.jtbi.2008.01.027' publications.firstAuthor='Ishihara Shuji' publications.id=1007779 publications.issue='1' publications.journal='J. Theor. Biol.' publications.month='May' publications.pages='131-44' publications.pubMedId='18342890' publications.title='Mutual interaction in network motifs robustly sharpens gene expression in developmental processes.' publications.volume='252' publications.year=2008\n",
      "Gene: publications.abstractText='The general consensus in the field is that limiting amounts of the transcription factor Dorsal establish dorsal boundaries of genes expressed along the dorsal-ventral (DV) axis of early Drosophila embryos, while repressors establish ventral boundaries. Yet recent studies have provided evidence that repressors act to specify the dorsal boundary of intermediate neuroblasts defective (ind), a gene expressed in a stripe along the DV axis in lateral regions of the embryo. Here we show that a short 12 base pair sequence (\"the A-box\") present twice within the ind CRM is both necessary and sufficient to support transcriptional repression in dorsal regions of embryos. To identify binding factors, we conducted affinity chromatography using the A-box element and found a number of DNA-binding proteins and chromatin-associated factors using mass spectroscopy. Only Grainyhead (Grh), a CP2 transcription factor with a unique DNA-binding domain, was found to bind the A-box sequence. Our results suggest that Grh acts as an activator to support expression of ind, which was surprising as we identified this factor using an element that mediates dorsally-localized repression. Grh and Dorsal both contribute to ind transcriptional activation. However, another recent study found that the repressor Capicua (Cic) also binds to the A-box sequence. While Cic was not identified through our A-box affinity chromatography, utilization of the same site, the A-box, by both factors Grh (activator) and Cic (repressor) may also support a \"switch-like\"response that helps to sharpen the ind dorsal boundary. Furthermore, our results also demonstrate that TGF-β signaling acts to refine ind CRM expression in an A-box independent manner in dorsal-most regions, suggesting that tiers of repression act in dorsal regions of the embryo.' publications.doi='10.1371/journal.pone.0029172' publications.firstAuthor='Garcia Mayra' publications.id=1007867 publications.issue='12' publications.journal='PLoS ONE' publications.month=None publications.pages='e29172' publications.pubMedId='22216201' publications.title='Lateral gene expression in Drosophila early embryos is supported by Grainyhead-mediated activation and tiers of dorsally-localized repression.' publications.volume='6' publications.year=2011\n",
      "Gene: publications.abstractText='The genomic and cDNA clones for a Drosophila melanogaster proliferating cell nuclear antigen (PCNA) (cyclin) were isolated and sequenced. The coding sequence for a 260-amino-acid residue polypeptide was interrupted by a single short intron of 60 base pairs (bp), and about 70% of the deduced amino acid sequence of the Drosophila PCNA was identical to the rat and human PCNA polypeptides, with conserved unique repeats of leucine in the C-terminal region. Genomic Southern blot hybridization analysis indicates the presence of a single gene for PCNA per genome. The PCNA mRNA was detected at a high level in adult ovaries, unfertilized eggs, and early embryos and at low levels in the other developmental stages. The major transcription initiation site (cap site) was localized at 89 bp upstream from the ATG codon. Neither a TATA box nor a CAAT box was found within the 600-bp region upstream of the cap site. Clusters of 10 bp of sequence similar to the binding sites for Drosophila proteins containing homeodomains were found in the region from -127 to -413. DNase I footprint analysis revealed that the Drosophila homeodomain proteins coded by even-skipped and zerknüllt genes can specifically bind to these sites. These results suggest that the expression of the PCNA gene is under the control of genes coding for homeodomain proteins.' publications.doi=None publications.firstAuthor='Yamaguchi M' publications.id=1007835 publications.issue='3' publications.journal='Mol. Cell. Biol.' publications.month='Mar' publications.pages='872-9' publications.pubMedId='1968224' publications.title=\"Drosophila proliferating cell nuclear antigen (cyclin) gene: structure, expression during development, and specific binding of homeodomain proteins to its 5'-flanking region.\" publications.volume='10' publications.year=1990\n",
      "Gene: publications.abstractText='The genomic program for development operates primarily by the regulated expression of genes encoding transcription factors and components of cell signaling pathways. This program is executed by cis-regulatory DNAs (e.g., enhancers and silencers) that control gene expression. The regulatory inputs and functional outputs of developmental control genes constitute network-like architectures. In this PNAS Special Feature are assembled papers on developmental gene regulatory networks governing the formation of various tissues and organs in nematodes, flies, sea urchins, frogs, and mammals. Here, we survey salient points of these networks, by using as reference those governing specification of the endomesoderm in sea urchin embryos and dorsal-ventral patterning in the Drosophila embryo.' publications.doi='10.1073/pnas.0408031102' publications.firstAuthor='Levine Michael' publications.id=1002604 publications.issue='14' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Apr' publications.pages='4936-42' publications.pubMedId='15788537' publications.title='Gene regulatory networks for development.' publications.volume='102' publications.year=2005\n",
      "Gene: publications.abstractText='The graded nuclear location of the transcription factor Dorsal along the dorsoventral axis of the early Drosophila embryo provides positional information for the determination of different cell fates. Nuclear uptake of Dorsal depends on a complex signalling pathway comprising two parts: an extracellular proteolytic cascade transmits the dorsoventral polarity of the egg chamber to the early embryo and generates a gradient of active Spätzle protein, the ligand of the receptor Toll; an intracellular cascade downstream of Toll relays this graded signal to embryonic nuclei. The slope of the Dorsal gradient is not determined by diffusion of extracellular or intracellular components from a local source, but results from self-organised patterning, in which positive and negative feedback is essential to create and maintain the ratio of key factors at different levels, thereby establishing and stabilising the graded spatial information for Dorsal nuclear uptake.' publications.doi='10.1016/j.cub.2005.10.026' publications.firstAuthor='Moussian Bernard' publications.id=1002675 publications.issue='21' publications.journal='Curr. Biol.' publications.month='Nov' publications.pages='R887-99' publications.pubMedId='16271864' publications.title='Dorsoventral axis formation in the Drosophila embryo--shaping and transducing a morphogen gradient.' publications.volume='15' publications.year=2005\n",
      "Gene: publications.abstractText='The homeo box, which encodes the DNA-binding homeo domain, is a DNA sequence motif present in several Drosophila developmental genes; it has been used to identify many homologous genes involved in mammalian development. The paired box is another conserved sequence motif, first identified in the paired (prd) and gooseberry (gsb) Drosophila homeo domain genes. It encodes a 128-amino-acid domain, the paired domain, which has since been found in other fly and mouse gene products, in association with the homeo domain or in its absence. We show that the paired box of the prd gene encodes a DNA-binding activity, independent of the DNA-binding activity of the Paired (Prd) homeo domain and with a different sequence specificity. The amino-terminal region of the paired domain, including one of the three predicted alpha-helices, is necessary and sufficient for binding. We investigate the binding of the Prd protein to two sites in the even-skipped promoter, which are composed of overlapping sequences bound by the homeo domain and by the paired domain. We also show that a mutation in the paired box of Prd, corresponding to the mutation in the paired box of the mouse Pax-1 gene thought to cause the undulated skeletal phenotype, destroys the ability of the Prd protein to bind to the paired domain-specific site. This supports the view that the undulated phenotype results from the inactivation of the DNA-binding activity of the paired domain of Pax-1.' publications.doi=None publications.firstAuthor='Treisman J' publications.id=1007770 publications.issue='4' publications.journal='Genes Dev.' publications.month='Apr' publications.pages='594-604' publications.pubMedId='1672661' publications.title='The paired box encodes a second DNA-binding domain in the paired homeo domain protein.' publications.volume='5' publications.year=1991\n",
      "Gene: publications.abstractText='The homeobox gene family of Drosophila appears to control a variety of position-specific patterning decisions during embryonic and imaginal development. Most of these patterning decisions determine groups of cells on the anterior-posterior axis of the Drosophila germ band. We have isolated a novel homeobox gene from Drosophila, designated H2.0. H2.0 has the most diverged homeobox so far characterized in metazoa, and, in contrast to all previously isolated homeobox genes, H2.0 exhibits a tissue-specific pattern of expression. The cells that accumulate transcripts for this novel gene correspond to the visceral musculature and its anlagen.' publications.doi=None publications.firstAuthor='Barad M' publications.id=1007759 publications.issue='7' publications.journal='EMBO J.' publications.month='Jul' publications.pages='2151-61' publications.pubMedId='2901348' publications.title='A novel, tissue-specific, Drosophila homeobox gene.' publications.volume='7' publications.year=1988\n",
      "Gene: publications.abstractText=\"The homoeobox gene zerknüllt (zen) plays an important role in the differentiation of dorsal tissues during Drosophila development. zen- embryos show transformations in the dorsal-most regions of the fate map, and lack several tissues that normally derive from these regions, including the amnioserosa and optic lobe. zen displays a simple dorsal on/ventral off pattern as early as cleavage cycle 10-11 (ref. 2). We have prepared a polyclonal antibody against a full-length zen protein, and used this to examine its pattern of expression in mutants that disrupt dorsal-ventral polarity. Most or all of the maternally expressed genes that are involved in this process have been previously identified and fall into two classes, so called 'dorsalizers' and 'ventralizers' (see refs 4-7, reviewed in ref. 8). On the basis of our analysis of zen expression in each of these maternal mutants we propose that one or more of the dorsalizing genes encodes a repressor which inhibits the expression of zen in ventral regions of developing embryos. The ventralizing gene cactus might play an important role in restricting the activity of this repressor to ventral regions, thereby permitting the activation of zen in those dorsal tissues where its function is critically required.\" publications.doi='10.1038/330583a0' publications.firstAuthor='Rushlow C' publications.id=1007855 publications.issue='6148' publications.journal='Nature' publications.month=None publications.pages='583-6' publications.pubMedId='2891036' publications.title='Maternal regulation of zerknüllt: a homoeobox gene controlling differentiation of dorsal tissues in Drosophila.' publications.volume='330' publications.year=None\n",
      "Gene: publications.abstractText='The immunoglobulin superfamily is a diverse group of proteins that are involved in various aspects of cell surface recognition. Here, we report the characterization of amalgam (ama), a gene in the Antennapedia complex (ANT-C) of D. melanogaster that exhibits amino acid similarity to vertebrate neural cell adhesion molecules and other members of the immunoglobulin superfamily. The putative 333 amino acid ama protein consists of a signal sequence, three immunoglobulin-like domains, and a short slightly hydrophobic carboxy-terminal region. Antibodies against the ama protein reveal that it accumulates on the surface of various mesodermal and neural cells during embryogenesis. The function of this protein remains elusive, as no mutations have been recovered for ama during saturation EMS mutagenesis of this chromosomal region.' publications.doi=None publications.firstAuthor='Seeger M A' publications.id=1007814 publications.issue='4' publications.journal='Cell' publications.month='Nov' publications.pages='589-600' publications.pubMedId='3141062' publications.title='Characterization of amalgam: a member of the immunoglobulin superfamily from Drosophila.' publications.volume='55' publications.year=1988\n",
      "Gene: publications.abstractText='The maternal Dorsal regulatory gradient initiates the differentiation of several tissues in the early Drosophila embryo. Whole-genome microarray assays identified as many as 40 new Dorsal target genes, which encode a broad spectrum of cell signaling proteins and transcription factors. Evidence is presented that a tissue-specific form of the NF-Y transcription complex is essential for the activation of gene expression in the mesoderm. Tissue-specific enhancers were identified for new Dorsal target genes, and bioinformatics methods identified conserved cis-regulatory elements for coordinately regulated genes that respond to similar thresholds of the Dorsal gradient. The new Dorsal target genes and enhancers represent one of the most extensive gene networks known for any developmental process.' publications.doi=None publications.firstAuthor='Stathopoulos Angelike' publications.id=1007766 publications.issue='5' publications.journal='Cell' publications.month='Nov' publications.pages='687-701' publications.pubMedId='12464180' publications.title='Whole-genome analysis of dorsal-ventral patterning in the Drosophila embryo.' publications.volume='111' publications.year=2002\n",
      "Gene: publications.abstractText='The maternal determinants of dorsoventral polarity of the Drosophila embryo are derived from somatic and germ-line components of the egg chamber. During oogenesis, asymmetry seems to be established by a signal transduction process. This process is thought to provide the developing embryo with a ventral signal responsible for determining the embryonic axis. Through a set of interactions that may involve signal transduction and proteolytic cascade events, positional information is generated in the form of a graded distribution of dorsal protein in blastoderm nuclei. Different levels of dorsal protein result in asymmetric expression of zygotic genes that ultimately specify cell fate.' publications.doi=None publications.firstAuthor='Govind S' publications.id=1002591 publications.issue='4' publications.journal='Trends Genet.' publications.month='Apr' publications.pages='119-25' publications.pubMedId='2068782' publications.title='Dorsoventral pattern formation in Drosophila: signal transduction and nuclear targeting.' publications.volume='7' publications.year=1991\n",
      "Gene: publications.abstractText='The maternal transcription factor Dorsal (Dl) functions as both an activator and a repressor in a context-dependent manner to control dorsal-ventral patterning in the Drosophila embryo. Previous studies have suggested that Dl is an intrinsic activator and its repressive activity requires additional corepressors that bind corepressor-binding sites near Dl-binding sites. However, the molecular identities of the corepressors have yet to be identified. Here, we present evidence that Capicua (Cic) is involved in Dl-mediated repression in the zerknüllt (zen) ventral repression element (VRE). Computational and genetic analyses indicate that a DNA-binding consensus sequence of Cic is highly analogous with previously identified corepressor-binding sequences and that Dl failed to repress zen expression in lateral regions of cic mutant embryos. Furthermore, electrophoretic mobility shift assay (EMSA) shows that Cic directly interacts with several corepressor-binding sites in the zen VRE. These results suggest that Cic may function as a corepressor by binding the VRE.' publications.doi=None publications.firstAuthor='Shin Dong-Hyeon' publications.id=1007834 publications.issue='9' publications.journal='BMB Rep' publications.month='Sep' publications.pages='518-23' publications.pubMedId='25059278' publications.title='Capicua is involved in Dorsal-mediated repression of zerknüllt expression in Drosophila embryo.' publications.volume='47' publications.year=2014\n",
      "Gene: publications.abstractText='The mechanisms that generate precise patterns of discrete cell types within developing fields are not well understood. One model for analyzing how cells interpret positional information in two dimensions is the regulation of proneural cluster formation within insect segments. Two adjacent proneural regulatory genes, achaete and scute, are expressed coincidently in cell clusters at reproducible anteroposterior (AP) and dorsoventral (DV) coordinates within the Drosophila embryo from which single neuroblasts later arise. Here, we show that the AP and DV position of these clusters is regulated through a common cis-acting region between the genes under the initial control of the products of the pair-rule and DV polarity genes and is later maintained by selected segment polarity genes. The combination of proneural gene activation/repression in AP stripes and repression within specific DV domains positions each cluster of achaete/scute expressing cells within segments; interactions between these cells then determine individual cell fates.' publications.doi=None publications.firstAuthor='Skeath J B' publications.id=1007824 publications.issue='12B' publications.journal='Genes Dev.' publications.month='Dec' publications.pages='2606-19' publications.pubMedId='1340472' publications.title='Gene regulation in two dimensions: the proneural achaete and scute genes are controlled by combinations of axis-patterning genes through a common intergenic control region.' publications.volume='6' publications.year=1992\n",
      "Gene: publications.abstractText='The members of the evolutionarily conserved Hox-gene complex, termed Hox genes, are required for specifying segmental identity during embryogenesis in various animal phyla. The Hox3 genes of winged insects have lost this ancestral function and are required for the development of extraembryonic epithelia, which do not contribute to any larval structure. Higher flies (Cyclorrhapha) such as Drosophila melanogaster contain Hox3 genes of two types, the zerknüllt type and the bicoid type. The zerknüllt gene is expressed zygotically on the dorsal side of the embryo and is required for establishing extraembryonic tissue. Its sister gene bicoid is expressed maternally and the transcripts are localized at the anterior pole of the mature egg. BICOID protein, which emerges from this localized source during early development, is required for embryonic patterning. All known direct bicoid homologues are confined to Cyclorrhaphan flies. Here, we describe Hox3 genes of the non-Cyclorrhaphan flies Empis livida (Empididae), Haematopota pluvialis (Tabanidae), and Clogmia albipunctata (Psychodidae). The gene sequences are more similar to zerknüllt homologues than to bicoid homologues, but they share expression characteristics of both genes. We propose that an ancestral Hox3 gene had been duplicated in the stem lineage of Cyclorrhaphan flies. During evolution, one of the gene copies lost maternal expression and evolved as zerknüllt, whereas the second copy lost zygotic expression and evolved as bicoid. Our finding correlates well with a partial reduction of zerknüllt-dependent extraembryonic tissue during Dipteran evolution.' publications.doi='10.1073/pnas.012292899' publications.firstAuthor='Stauber Michael' publications.id=1007745 publications.issue='1' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Jan' publications.pages='274-9' publications.pubMedId='11773616' publications.title='A single Hox3 gene with composite bicoid and zerknullt expression characteristics in non-Cyclorrhaphan flies.' publications.volume='99' publications.year=2002\n",
      "Gene: publications.abstractText='The organization of the anterior pattern in the Drosophila embryo is mediated by the maternal effect gene bicoid. bcd has been identified in an 8.7-kb genomic fragment by germ line transformants that completely rescue the mutant phenotype. The major transcript of 2.6 kb includes a homeobox with low homology to previously known homeoboxes, a PRD-repeat and a M-repeat. In situ hybridizations reveal that bcd is transcribed in the nurse cells. The mRNA is localized at the anterior tip of oocyte and early embryo until the cellular blastoderm stage. The localization of the transcript requires the function of the maternal effect genes exuperantia and swallow while transcript stability is reduced by functions depending on posterior group genes.' publications.doi=None publications.firstAuthor='Berleth T' publications.id=1007789 publications.issue='6' publications.journal='EMBO J.' publications.month='Jun' publications.pages='1749-56' publications.pubMedId='2901954' publications.title='The role of localization of bicoid RNA in organizing the anterior pattern of the Drosophila embryo.' publications.volume='7' publications.year=1988\n",
      "Gene: publications.abstractText='The origin of evolutionary novelty is believed to involve both positive selection and relaxed developmental constraint. In flies, the redesign of anterior patterning during embryogenesis is a major developmental innovation and the rapidly evolving Hox gene, bicoid (bcd), plays a critical role. We report evidence for relaxation of selective constraint acting on bicoid as a result of its maternal pattern of gene expression. Evolutionary theory predicts 2-fold greater sequence diversity for maternal effect genes than for zygotically expressed genes, because natural selection is only half as effective acting on autosomal genes expressed in one sex as it is on genes expressed in both sexes. We sample an individual from ten populations of Drosophila melanogaster and nine populations of D. simulans for polymorphism in the tandem gene duplicates bcd, which is maternally expressed, and zerknüllt (zen), which is zygotically expressed. In both species, we find the ratio of bcd to zen nucleotide diversity to be two or more in the coding regions but one in the noncoding regions, providing the first quantitative support for the theoretical prediction of relaxed selective constraint on maternal-effect genes resulting from sex-limited expression. Our results suggest that the accelerated rate of evolution observed for bcd is owing, at least partly, to variation generated by relaxed selective constraint.' publications.doi='10.1371/journal.pgen.0010057' publications.firstAuthor='Barker Michael S' publications.id=1007755 publications.issue='5' publications.journal='PLoS Genet.' publications.month='Nov' publications.pages='e57' publications.pubMedId='16299585' publications.title='Maternal expression relaxes constraint on innovation of the anterior determinant, bicoid.' publications.volume='1' publications.year=2005\n",
      "Gene: publications.abstractText=\"The passage of an individual's genome to future generations is essential for the maintenance of species and is mediated by highly specialized cells, the germ cells. Genetic studies in a number of model organisms have provided insight into the molecular mechanisms that control specification, migration and survival of early germ cells. Focusing on Drosophila, we will discuss the mechanisms by which germ cells initially form and remain transcriptionally silent while somatic cells are transcriptionally active. We will further discuss three separate attractiveand repellent guidance pathways, mediated by a G-protein coupled receptor, two lipid phosphate phosphohydrolases, and isoprenylation. We will compare and contrast these findings with those obtained in other organisms, in particular zebrafish and mice. While aspects of germ cell specification are strikingly different between these species, germ cell specific gene functions have been conserved. In particular, mechanisms that sense directional cues during germ cell migration seem to be shared between invertebrates and vertebrates.\" publications.doi='10.1016/j.cub.2004.07.018' publications.firstAuthor='Santos Ana C' publications.id=1007767 publications.issue='14' publications.journal='Curr. Biol.' publications.month='Jul' publications.pages='R578-89' publications.pubMedId='15268881' publications.title='Germ cell specification and migration in Drosophila and beyond.' publications.volume='14' publications.year=2004\n",
      "Gene: publications.abstractText='The position of the nucleus along the anterior rim of stage 8 Drosophila oocytes presages the dorsal side of the egg and the developing embryo. In this paper, we address the question of whether the oocyte has a previously determined dorsal side to which the nucleus is drawn, or whether nuclear position randomly determines the dorsal side. To do so, we have taken advantage of a genetic system in which Drosophila oocytes occasionally become binuclear. We find that (i) the two nuclei migrate independently to their respective positions on the anterior rim, sometimes selecting the same site, sometimes not, (ii) the two nuclei are equivalent in their ability to induce a dorsal-ventral pattern in the overlying follicular epithelium, and (iii) at any position around the anterior circumference of the egg chamber the follicle cell sheet is equally responsive to the Gurken signal associated with the oocyte nuclei. These results argue that the dorsal-ventral axis is determined arbitrarily by the randomly selected position of the nucleus on the anterior rim of the oocyte. Some of the binuclear eggs support embryonic development. However, despite the duplication of dorsal chorion structures, the majority of such embryos show normal dorsal-ventral patterning. Thus, processes exist in the ventral follicular epithelium or in the perivitelline space that compensate for the expansion of dorsal follicle cell fates and consequently allow the formation of a normal embryonic axis.' publications.doi=None publications.firstAuthor='Roth S' publications.id=1007870 publications.issue='5' publications.journal='Development' publications.month='Feb' publications.pages='927-34' publications.pubMedId='9927594' publications.title='Binuclear Drosophila oocytes: consequences and implications for dorsal-ventral patterning in oogenesis and embryogenesis.' publications.volume='126' publications.year=1999\n",
      "Gene: publications.abstractText='The pre-cellular Drosophila embryo contains 10 well characterized sequence-specific transcriptional repressors, which represent a broad spectrum of DNA-binding proteins. Previous studies have shown that two of the repressors, Hairy and Dorsal, recruit a common co-repressor protein, Groucho. Here we present evidence that three different repressors, Knirps, Krüppel and Snail, recruit a different co-repressor, dCtBP. Mutant embryos containing diminished levels of maternal dCtBP products exhibit both segmentation and dorsoventral patterning defects, which can be attributed to loss of Krüppel, Knirps and Snail activity. In contrast, the Dorsal and Hairy repressors retain at least some activity in dCtBP mutant embryos. dCtBP interacts with Krüppel, Knirps and Snail through a related sequence motif, PXDLSXK/H. This motif is essential for the repression activity of these proteins in transgenic embryos. We propose that dCtBP represents a major form of transcriptional repression in development, and that the Groucho and dCtBP co-repressors mediate separate pathways of repression.' publications.doi='10.1093/emboj/17.23.7009' publications.firstAuthor='Nibu Y' publications.id=1007750 publications.issue='23' publications.journal='EMBO J.' publications.month='Dec' publications.pages='7009-20' publications.pubMedId='9843507' publications.title='dCtBP mediates transcriptional repression by Knirps, Krüppel and Snail in the Drosophila embryo.' publications.volume='17' publications.year=1998\n",
      "Gene: publications.abstractText=\"The principles of embryonic pattern formation have been studied extensively in many systems using classical experimental approaches. In Drosophila, a powerful combination of genetics and transplantation experiments, as well as molecular biology, have helped to elucidate the mechanisms that operate during oogenesis and early embryogenesis to establish a set of positional cues required for axis determination in the early embryo. In systematic searches for maternal effect mutations a small number of about 30 genes have been identified that specifically affect the process of determination of the embryonic axes. These 'coordinate' genes define four systems that determine the anteroposterior (AP) axis (three systems) and the dorsoventral (DV) axis (one system) independently. In the anteroposterior axis, the anterior system determines the segmented region of head and thorax, the posterior system determines the segmented abdominal region, and the terminal system is responsible for the formation of the nonsegmented termini at the anterior and posterior egg tips, the acron and telson. In contrast, pattern along the dorsoventral axis is determined by one system only. Although all four systems use different biochemical mechanisms, they share several properties. (1) The product of one gene in each system is localized in a specific region of the freshly laid egg and functions as a spatial signal. (2) In each system, this spatial information finally results in the asymmetrical distribution of one gene product that functions as a transcription factor. (3) This transcription factor is distributed in a concentration gradient that defines the spatial limits of expression of one or more zygotic target genes. The combined action of these three anteroposterior systems as well as the dorsoventral system defines the expression of zygotic target genes in at least seven distinct regions along the anteroposterior and at least three in the dorsoventral axis. These longitudinal and transverse domains provide a coarse spatial prepattern which is then further refined by the action and interaction of zygotic pattern genes.\" publications.doi=None publications.firstAuthor='Nüsslein-Volhard C' publications.id=1002702 publications.issue=None publications.journal='Dev. Suppl.' publications.month=None publications.pages='1-10' publications.pubMedId='1742496' publications.title='Determination of the embryonic axes of Drosophila.' publications.volume='1' publications.year=1991\n",
      "Gene: publications.abstractText='The recent completion of the Drosophila melanogaster genomic sequence to high quality and the availability of a greatly expanded set of Drosophila cDNA sequences, aligning to 78% of the predicted euchromatic genes, afforded FlyBase the opportunity to significantly improve genomic annotations. We made the annotation process more rigorous by inspecting each gene visually, utilizing a comprehensive set of curation rules, requiring traceable evidence for each gene model, and comparing each predicted peptide to SWISS-PROT and TrEMBL sequences. Although the number of predicted protein-coding genes in Drosophila remains essentially unchanged, the revised annotation significantly improves gene models, resulting in structural changes to 85% of the transcripts and 45% of the predicted proteins. We annotated transposable elements and non-protein-coding RNAs as new features, and extended the annotation of untranslated (UTR) sequences and alternative transcripts to include more than 70% and 20% of genes, respectively. Finally, cDNA sequence provided evidence for dicistronic transcripts, neighboring genes with overlapping UTRs on the same DNA sequence strand, alternatively spliced genes that encode distinct, non-overlapping peptides, and numerous nested genes. Identification of so many unusual gene models not only suggests that some mechanisms for gene regulation are more prevalent than previously believed, but also underscores the complex challenges of eukaryotic gene prediction. At present, experimental data and human curation remain essential to generate high-quality genome annotations.' publications.doi=None publications.firstAuthor='Misra Sima' publications.id=1000884 publications.issue='12' publications.journal='Genome Biol.' publications.month=None publications.pages='RESEARCH0083' publications.pubMedId='12537572' publications.title='Annotation of the Drosophila melanogaster euchromatic genome: a systematic review.' publications.volume='3' publications.year=2002\n",
      "Gene: publications.abstractText='The repetitive DNA that constitutes most of the heterochromatic regions of metazoan genomes has hindered the comprehensive analysis of gene content and other functions. We have generated a detailed computational and manual annotation of 24 megabases of heterochromatic sequence in the Release 5 Drosophila melanogaster genome sequence. The heterochromatin contains a minimum of 230 to 254 protein-coding genes, which are conserved in other Drosophilids and more diverged species, as well as 32 pseudogenes and 13 noncoding RNAs. Improved methods revealed that more than 77% of this heterochromatin sequence, including introns and intergenic regions, is composed of fragmented and nested transposable elements and other repeated DNAs. Drosophila heterochromatin contains \"islands\"of highly conserved genes embedded in these \"oceans\"of complex repeats, which may require special expression and splicing mechanisms.' publications.doi='10.1126/science.1139815' publications.firstAuthor='Smith Christopher D' publications.id=1895606 publications.issue='5831' publications.journal='Science' publications.month='Jun' publications.pages='1586-91' publications.pubMedId='17569856' publications.title='The Release 5.1 annotation of Drosophila melanogaster heterochromatin.' publications.volume='316' publications.year=2007\n",
      "Gene: publications.abstractText='The retinal determination gene dachshund is distantly related to the family of Ski/Sno proto-oncogenes and influences the development of a wide range of tissues including the embryonic head, optic lobes, brain, central nervous system as well as the post-embryonic leg, wing, genital and eye-antennal discs. We were interested in the regulatory mechanisms that control the dynamic expression pattern of dachshund and in this report we set out to ascertain how the transcription of dachshund is modulated in the embryonic head and developing eye-antennal imaginal disc. We demonstrate that the TGFbeta signaling cascade, the transcription factor zerknullt and several other patterning genes prevent dachshund from being expressed inappropriately within the embryonic head. Additionally, we show that several members of the eye specification cascade influence the transcription of dachshund during normal and ectopic eye development. Our results suggest that dachshund is regulated by a complex combinatorial code of transcription factors and signaling pathways. Unraveling this code may lead to an understanding of how dachshund regulates the development of many diverse tissue types including the eye.' publications.doi='10.1016/j.ydbio.2006.05.004' publications.firstAuthor='Anderson Jason' publications.id=1007896 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Sep' publications.pages='536-49' publications.pubMedId='16780828' publications.title='Regulation of the retinal determination gene dachshund in the embryonic head and developing eye of Drosophila.' publications.volume='297' publications.year=2006\n",
      "Gene: publications.abstractText='The rho family of GTPases has been implicated in regulating changes in cell morphology in response to extracellular signals. We have cloned three widely expressed members of this family from Drosophila melanogaster; a rho homologue (Rho1) and two rac homologues (Rac1 and Rac2). Flies harbouring a Rho1 transgene that is specifically expressed in the eye exhibit a dramatic dose dependent disruption of normal eye development. Flies bearing at least two copies of the transgene display a severe rough eye phenotype characterized by missing secondary and tertiary pigment cells, a substantial reduction in the number of photoreceptor cells and a grossly abnormal morphology of the rhabdomeres. Cell fate determination in the imaginal disc occurs normally and abnormalities become manifest late in pupariation, coincident with the phase when the cells undergo major morphological changes. This phenotype is modified by mutations at several other loci that have been implicated in signal transduction, but not by mutations in ras pathway components.' publications.doi=None publications.firstAuthor='Hariharan I K' publications.id=1007713 publications.issue='2' publications.journal='EMBO J.' publications.month='Jan' publications.pages='292-302' publications.pubMedId='7835340' publications.title='Characterization of rho GTPase family homologues in Drosophila melanogaster: overexpressing Rho1 in retinal cells causes a late developmental defect.' publications.volume='14' publications.year=1995\n",
      "Gene: publications.abstractText='The short gastrulation (sog) gene is expressed in broad lateral stripes comprising the neuroectoderm of the Drosophila blastoderm embryo. sog encodes a predicted secreted protein that functions nonautonomously to antagonize the activity of the TGF-beta-like Decapentaplegic (Dpp) signaling pathway in the dorsal region of the embryo. Recently, it has been shown that sog and dpp are functionally equivalent to their respective Xenopus homologs chordin and BMP-4. In this report we provide the first direct evidence that sog plays a local role in the lateral region of the blastoderm embryo to oppose Dpp activity in the neuroectoderm. In the dorsal region, Dpp signaling both suppresses neurogenesis and maintains expression of genes that promote dorsal cell fates (dorsalization). We show that Dpp also can perform both of these functions in the neuroectoderm. In wild-type embryos, the ability of Dpp to induce expression of dorsal markers including itself (autoactivation) in the neuroectoderm is blocked by sog. We propose that Sog protects the neuroectoderm from an invasive positive feedback loop created by Dpp diffusion and autoactivation. We show that the two functions of Dpp signaling, neural suppression and dorsalization, are triggered by distinct thresholds of Dpp activity. Epistasis experiments reveal that all observed sog activity can be accounted for by Sog functioning as a dedicated Dpp antagonist. Finally, we provide evidence that Sog functions as a diffusible morphogen in the blastoderm embryo. These data strongly support the view that the primary phylogenetically conserved function of the Drosophila sog and dpp genes and the homologous Xenopus chordin and BMP-4 genes is to subdivide the primitive embryonic ectoderm into neural versus non-neural domains.' publications.doi=None publications.firstAuthor='Biehs B' publications.id=1007780 publications.issue='22' publications.journal='Genes Dev.' publications.month='Nov' publications.pages='2922-34' publications.pubMedId='8918893' publications.title='The Drosophila short gastrulation gene prevents Dpp from autoactivating and suppressing neurogenesis in the neuroectoderm.' publications.volume='10' publications.year=1996\n",
      "Gene: publications.abstractText='The spatial and temporal pattern of mitoses during the fourteenth nuclear cycle in a Drosophila embryo reflects differences in cell identities. We have analysed the domains of mitotic division in zygotic mutants that exhibit defects in larval cuticular pattern along the dorsoventral axis. This is a powerful means of fate mapping mutant embryos, as the altered position of mitotic domains in the dorsoventral pattern mutants correlate with their late cuticular phenotypes. In the mutants twist and snail, which fail to differentiate the ventrally derived mesoderm, mitoses specific to the mesoderm are absent. The lateral mesectodermal domain shows a partial ventral shift in twist mutants but a proportion of ventral cells do not behave characteristically, suggesting that twist has a positive role in the establishment of the mesoderm. In contrast, snail is required to repress mesectodermal fates in cells of the presumptive mesoderm. In the absence of both genes, the mesodermal and the mesectodermal anlage are deleted. Mutations at five loci delete specific pattern elements in the dorsal half of the embryo and cause partial ventralization. Mutations in the genes zerknüllt and shrew affect cell division only in the dorsalmost cells corresponding to the amnioserosa, while the genes tolloid, screw and decapentaplegic (dpp) affect divisions in both the prospective amnioserosa and the dorsal epidermis. We demonstrate that in each of these mutants dorsally placed mitotic domains are absent and this effect is correlated with an expansion and dorsal shift in the position of more ventral domains. The loss of activity in each of the five genes results in qualitatively similar alterations in the mitotic pattern; mutations with stronger ventralizing phenotypes affect increasingly greater subsets of the dorsal cells. Double mutant analysis indicates that these genes act in a concerted manner to specify dorsal fates. The correlation between phenotypic strength and the progressive loss of dorsal pattern elements in the ventralized mutants, suggests that one of these gene products, perhaps dpp, may provide positional information in a graded manner.' publications.doi=None publications.firstAuthor='Arora K' publications.id=1007868 publications.issue='4' publications.journal='Development' publications.month='Apr' publications.pages='1003-24' publications.pubMedId='1618145' publications.title='Altered mitotic domains reveal fate map changes in Drosophila embryos mutant for zygotic dorsoventral patterning genes.' publications.volume='114' publications.year=1992\n",
      "Gene: publications.abstractText='The synthesis of gene expression data and cis-regulatory analysis permits the elucidation of genomic regulatory networks. These networks provide a direct visualization of the functional interconnections among the regulatory genes and signaling components leading to cell-specific patterns of gene activity. Complex developmental processes are thereby illuminated in ways not revealed by the conventional analysis of individual genes. In this review, we describe emerging networks in several different model systems, and compare them with the gene regulatory network that controls dorsoventral patterning of the Drosophila embryo.' publications.doi='10.1016/j.devcel.2005.09.005' publications.firstAuthor='Stathopoulos Angelike' publications.id=1004041 publications.issue='4' publications.journal='Dev. Cell' publications.month='Oct' publications.pages='449-62' publications.pubMedId='16198288' publications.title='Genomic regulatory networks and animal development.' publications.volume='9' publications.year=2005\n",
      "Gene: publications.abstractText=\"The torpedo (DER) gene of Drosophila, which encodes a receptor tyrosine kinase of the EGF receptor subfamily, is essential for oogenesis, embryogenesis and imaginal disc development. To gain insight into the nature of the signals transduced by the torpedo product, we have characterized the gene's loss-of-function phenotype in the embryo. Through the induction of germline clones, we provide a genetic demonstration that maternal torpedo product does not contribute to zygotic development. Thus, the embryonic lethal phenotypes examined accurately reflect the consequences of eliminating all gene activity from the zygote. Temperature-shift experiments with the conditional allele topIF26 show that torpedo is required at two distinct times during embryonic development: the gene is first needed for germband retraction and for the production of anterior, posterior and ventral cuticle, then later for the secretion of ventral denticles. Since denticle formation can be severely disrupted in topIF26 animals without affecting cuticle production, the early and late requirements for torpedo appear to be functionally unrelated. torpedo, therefore, is required at multiple times in the development of the ventral epidermis, and may transduce qualitatively different signals. Since the early requirement for torpedo correlates with the first visible defect in embryonic development, increased cell death in the amnioserosa, cephalic ectoderm and ventral epidermis, the abnormalities in cuticle production and germband shortening seen in the mutant may be secondary consequences of a primary defect in cell viability. Given that the onset of cell death in torpedo embryos is not preceded by any obvious defects in mitogenesis, the establishment of cell identities or the maintenance of gene expression, it is possible that torpedo transduces a signal necessary for cell survival per se during early embryogenesis. During late embryogenesis, torpedo may mediate the reception of a second signal which regulates ventral epidermal cell differentiation.\" publications.doi=None publications.firstAuthor='Clifford R' publications.id=1007851 publications.issue='3' publications.journal='Development' publications.month='Jul' publications.pages='853-72' publications.pubMedId='1425358' publications.title='The torpedo (DER) receptor tyrosine kinase is required at multiple times during Drosophila embryogenesis.' publications.volume='115' publications.year=1992\n",
      "Gene: publications.abstractText=\"The toxicity of an abnormally long polyglutamine [poly(Q)]tract within specific proteins is the molecular lesion shared by Huntington's disease (HD) and several other hereditary neurodegenerative disorders. By a genetic screen in Drosophila, devised to uncover genes that suppress poly(Q) toxicity, we discovered a Drosophila homolog of human myeloid leukemia factor 1 (MLF1). Expression of the Drosophila homolog (dMLF) ameliorates the toxicity of poly(Q) expressed in the eye and central nervous system. In the retina, whether endogenously or ectopically expressed, dMLF co-localized with aggregates, suggesting that dMLF alone, or through an intermediary molecular partner, may suppress toxicity by sequestering poly(Q) and/or its aggregates.\" publications.doi=None publications.firstAuthor='Kazemi-Esfarjani Parsa' publications.id=1007866 publications.issue='21' publications.journal='Hum. Mol. Genet.' publications.month='Oct' publications.pages='2657-72' publications.pubMedId='12354791' publications.title='Suppression of polyglutamine toxicity by a Drosophila homolog of myeloid leukemia factor 1.' publications.volume='11' publications.year=2002\n",
      "Gene: publications.abstractText=\"The trithorax group (trxG) genes absent, small or homeotic discs 1 (ash1) and 2 (ash2) were isolated in a screen for mutants with abnormal imaginal discs. Mutations in either gene cause homeotic transformations but Hox genes are not their only targets. Although analysis of double mutants revealed that ash2 and ash1 mutations enhance each other's phenotypes, suggesting they are functionally related, it was shown that these proteins are subunits of distinct complexes. The analysis of wing imaginal disc transcriptomes from ash2 and ash1 mutants showed that they are highly similar. Functional annotation of regulated genes using Gene Ontology allowed identification of severely affected groups of genes that could be correlated to the wing phenotypes observed. Comparison of the differentially expressed genes with those from other genome-wide analyses revealed similarities between ASH2 and Sin3A, suggesting a putative functional relationship. Coimmunoprecipitation studies and immunolocalization on polytene chromosomes demonstrated that ASH2 and Sin3A interact with HCF (host-cell factor). The results of nucleosome western blots and clonal analysis indicated that ASH2 is necessary for trimethylation of the Lys4 on histone 3 (H3K4). The similarity between the transcriptomes of ash2 and ash1 mutants supports a model in which the two genes act together to maintain stable states of transcription. Like in humans, both ASH2 and Sin3A bind HCF. Finally, the reduction of H3K4 trimethylation in ash2 mutants is the first evidence in Drosophila regarding the molecular function of this trxG gene.\" publications.doi='10.1186/gb-2007-8-4-r67' publications.firstAuthor='Beltran Sergi' publications.id=1000626 publications.issue='4' publications.journal='Genome Biol.' publications.month=None publications.pages='R67' publications.pubMedId='17466076' publications.title='Functional dissection of the ash2 and ash1 transcriptomes provides insights into the transcriptional basis of wing phenotypes and reveals conserved protein interactions.' publications.volume='8' publications.year=2007\n",
      "Gene: publications.abstractText='The twisted gastrulation (tsg) gene is one of seven known zygotic genes that specify the fate of dorsal cells in Drosophila embryos. Mutations in these genes cause at least some of the cells on the dorsal half of the embryo to adopt more ventral cell fates leading to the proposal that most of these genes participate in establishing, maintaining, or modulating a gradient of a single signaling molecule DECAPENTAPLEGIC (DPP). We have examined the effects of tsg mutations on the development of cuticule elements, expression of a region specific enhancer trap, and patterns of mitotic domains. Mutations of tsg only affect the fate of a narrow strip of dorsal midline cells and do not affect dorsal ectoderm cells. However, the pattern of tsg expression is not coincident with the territories affected by tsg mutations. Structural analysis of the tsg gene reveals features of a secreted protein suggesting an extracellular site of action. The TSG protein bears a weak resemblance to human connective tissue growth factor (CTGF), a TGF-beta-induced protein. We propose that dorsal midline cell fate is specified by the combination of both a TSG and a DPP signal to which the dorsal midline cells are uniquely competent to respond.' publications.doi=None publications.firstAuthor='Mason E D' publications.id=1002687 publications.issue='13' publications.journal='Genes Dev.' publications.month='Jul' publications.pages='1489-501' publications.pubMedId='7958834' publications.title='Dorsal midline fate in Drosophila embryos requires twisted gastrulation, a gene encoding a secreted protein related to human connective tissue growth factor.' publications.volume='8' publications.year=1994\n",
      "Gene: publications.abstractText='The yellow (y) gene maps near the telomere of the X chromosome in Drosophila melanogaster but not in D. subobscura. Thus the strong reduction in the recombination rate associated with telomeric regions is not expected in D. subobscura. To study the divergence of a gene whose recombination rate differs between two species, the y gene of D. subobscura was sequenced. Sequence comparison between D. melanogaster and D. subobscura revealed several elements conserved in noncoding regions that may correspond to putative cis-acting regulatory sequences. Divergence in the y gene coding region between D. subobscura and D. melanogaster was compared with that found in other genes sequenced in both species. Both, yellow and scute exhibit an unusually high number of synonymous substitutions per site (ps). Also for these genes, the extent of codon bias differs between both species, being much higher in D. subobscura than in D. melanogaster. This pattern of divergence is consistent with the hitchhiking and background selection models that predict an increase in the fixation rate of slightly deleterious mutations and a decrease in the rate of fixation of slightly advantageous mutations in regions with low recombination rates such as in the y-sc gene region of D. melanogaster.' publications.doi=None publications.firstAuthor='Munté A' publications.id=1007885 publications.issue='1' publications.journal='Genetics' publications.month='Sep' publications.pages='165-75' publications.pubMedId='9286677' publications.title='Divergence of the yellow gene between Drosophila melanogaster and D. subobscura: recombination rate, codon bias and synonymous substitutions.' publications.volume='147' publications.year=1997\n",
      "Gene: publications.abstractText='This report focuses on dorso-ventral patterning in the segmented region of the Drosophila melanogaster embryo. According to the concept of positional information, this pattern results from the different response of cells to the Dorsal-protein morphogen. This protein shows a distribution gradient along the dorso-ventral axis, with the highest concentration on the ventral side. Using the generalized logical formalism developed by R. Thomas and co-workers, the different cellular responses were analysed in terms of the intracellular loops between the regulatory genes. Two positive loops were found to be involved, each constituting a switch which can be acted upon by the Dorsal morphogen to determine the different cell types that make up the embryonic dorso-ventral pattern. The novelty in this use of generalized logical formalism is the employment of a multilevel variable to represent a morphogen gradient. The proposed model accounts for the essential qualitative effects of the Dorsal gradient in the dorso-ventral determination process. Three main conclusions may be drawn. Firstly, the gene twist needs to have two functional threshold concentrations, one for autoactivation and the other for activation of the gene snail. Secondly, the autoactivation threshold must be smaller than that which activates snail. Thirdly, the action of the gene snail on the maintenance function of the gene twist is crucial for cells to be able to choose between the mesoderm or neuroectoderm developmental pathways. Furthermore, it is predicted that if the gene snail shows autoregulation, this will not be crucial for the determination of the embryonic D-V pattern. Copyright 1997 Academic Press Limited Copyright 1997 Academic Press Limited' publications.doi=None publications.firstAuthor='Sanchez null' publications.id=1007777 publications.issue='4' publications.journal='J. Theor. Biol.' publications.month='Dec' publications.pages='377-89' publications.pubMedId='9446747' publications.title='Establishement of the dorso-ventral pattern during embryonic development of drosophila melanogasater: a logical analysis' publications.volume='189' publications.year=1997\n",
      "Gene: publications.abstractText='Three genetic hierarchies control cell-fate specification in largely distinct regions of the antero-posterior axis of the Drosophila embryo, whereas a single hierarchy specifies dorso-ventral cell fates. Molecular genetic analysis of these hierarchies is leading to increased understanding of the nature of the regulatory circuitry that controls regional cell-fate specification.' publications.doi=None publications.firstAuthor='Lipshitz H D' publications.id=1002700 publications.issue='6' publications.journal='Curr. Opin. Cell Biol.' publications.month='Dec' publications.pages='966-75' publications.pubMedId='1814368' publications.title='Axis specification in the Drosophila embryo.' publications.volume='3' publications.year=1991\n",
      "Gene: publications.abstractText='To achieve the \"constancy of the wild-type,\"the developing organism must be buffered against stochastic fluctuations and environmental perturbations. This phenotypic buffering has been theorized to arise from a variety of genetic mechanisms and is widely thought to be adaptive and essential for viability. In the Drosophila blastoderm embryo, staining with antibodies against the active, phosphorylated form of the bone morphogenetic protein (BMP) signal transducer Mad, pMad, or visualization of the spatial pattern of BMP-receptor interactions reveals a spatially bistable pattern of BMP signaling centered on the dorsal midline. This signaling event is essential for the specification of dorsal cell fates, including the extraembryonic amnioserosa. BMP signaling is initiated by facilitated extracellular diffusion that localizes BMP ligands dorsally. BMP signaling then activates an intracellular positive feedback circuit that promotes future BMP-receptor interactions. Here, we identify a genetic network comprising three genes that canalizes this BMP signaling event. The BMP target eiger (egr) acts in the positive feedback circuit to promote signaling, while the BMP binding protein encoded by crossveinless-2 (cv-2) antagonizes signaling. Expression of both genes requires the early activity of the homeobox gene zerknüllt (zen). Two Drosophila species lacking early zen expression have high variability in BMP signaling. These data both detail a new mechanism that generates developmental canalization and identify an example of a species with noncanalized axial patterning.' publications.doi='10.1016/j.cub.2013.09.055' publications.firstAuthor='Gavin-Smyth Jackie' publications.id=1007805 publications.issue='22' publications.journal='Curr. Biol.' publications.month='Nov' publications.pages='2296-302' publications.pubMedId='24184102' publications.title='A genetic network conferring canalization to a bistable patterning system in Drosophila.' publications.volume='23' publications.year=2013\n",
      "Gene: publications.abstractText='Transcriptional enhancers are crucial regulators of gene expression and animal development and the characterization of their genomic organization, spatiotemporal activities and sequence properties is a key goal in modern biology. Here we characterize the in vivo activity of 7,705 Drosophila melanogaster enhancer candidates covering 13.5% of the non-coding non-repetitive genome throughout embryogenesis. 3,557 (46%) candidates are active, suggesting a high density with 50,000 to 100,000 developmental enhancers genome-wide. The vast majority of enhancers display specific spatial patterns that are highly dynamic during development. Most appear to regulate their neighbouring genes, suggesting that the cis-regulatory genome is organized locally into domains, which are supported by chromosomal domains, insulator binding and genome evolution. However, 12 to 21 per cent of enhancers appear to skip non-expressed neighbours and regulate a more distal gene. Finally, we computationally identify cis-regulatory motifs that are predictive and required for enhancer activity, as we validate experimentally. This work provides global insights into the organization of an animal regulatory genome and the make-up of enhancer sequences and confirms and generalizes principles from previous studies. All enhancer patterns are annotated manually with a controlled vocabulary and all results are available through a web interface (http://enhancers.starklab.org), including the raw images of all microscopy slides for manual inspection at arbitrary zoom levels.' publications.doi='10.1038/nature13395' publications.firstAuthor='Kvon Evgeny Z' publications.id=1003027 publications.issue='7512' publications.journal='Nature' publications.month='Aug' publications.pages='91-5' publications.pubMedId='24896182' publications.title='Genome-scale functional characterization of Drosophila developmental enhancers in vivo.' publications.volume='512' publications.year=2014\n",
      "Gene: publications.abstractText=\"Transcriptional repression is essential for the conversion of crude maternal gradients into sharp territories of tissue differentiation in the Drosophila embryo. Evidence will be presented suggesting that some of the embryonic repressors function through a short-range 'quenching' mechanism, whereby a repressor works over short distances (ca. 50 b.p.) to block neighbouring activators within a target enhancer. This type of repression can explain how different enhancers work autonomously within complex modular promoters. However, at least one of the repressors operating in the early embryo works through a long-range, or silencing, mechanism. The binding of a silencer to a given enhancer leads to the inactivation of all enhancers within a complex promoter. The analysis of chromatin boundary elements suggest that silencers and enhancers might work through distinct mechanisms. We speculate that silencers constrain the evolution of complex promoters.\" publications.doi='10.1098/rstb.1995.0111' publications.firstAuthor='Gray S' publications.id=1007730 publications.issue='1329' publications.journal='Philos. Trans. R. Soc. Lond., B, Biol. Sci.' publications.month='Sep' publications.pages='257-62' publications.pubMedId='8577836' publications.title='Transcriptional repression in the Drosophila embryo.' publications.volume='349' publications.year=1995\n",
      "Gene: publications.abstractText='Transcriptional repressors can be characterized by their range of action on promoters and enhancers. Short-range repressors interact over distances of 50-150 bp to inhibit, or quench, either upstream activators or the basal transcription complex. In contrast, long-range repressors act over several kilobases to silence basal promoters. We describe recent progress in characterizing the functional properties of one such long-range element in the Drosophila embryo and discuss the contrasting types of gene regulation that are made possible by short- and long-range repressors.' publications.doi=None publications.firstAuthor='Cai H N' publications.id=1007829 publications.issue='18' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Sep' publications.pages='9309-14' publications.pubMedId='8790326' publications.title='Long-range repression in the Drosophila embryo.' publications.volume='93' publications.year=1996\n",
      "Gene: publications.abstractText='Transcriptional repressors function primarily by recruiting co-repressors, which are accessory proteins that antagonize transcription by modifying chromatin structure. Although a repressor could function by recruiting just a single co-repressor, many can recruit more than one, with Drosophila Brinker (Brk) recruiting the co-repressors CtBP and Groucho (Gro), in addition to possessing a third repression domain, 3R. Previous studies indicated that Gro is sufficient for Brk to repress targets in the wing, questioning why it should need to recruit CtBP, a short-range co-repressor, when Gro is known to be able to function over longer distances. To resolve this we have used genomic engineering to generate a series of brk mutants that are unable to recruit Gro, CtBP and/or have 3R deleted. These reveal that although the recruitment of Gro is necessary and can be sufficient for Brk to make an almost morphologically wild-type fly, it is insufficient during oogenesis, where Brk must utilize CtBP and 3R to pattern the egg shell appropriately. Gro insufficiency during oogenesis can be explained by its downregulation in Brk-expressing cells through phosphorylation downstream of EGFR signaling.' publications.doi='10.1242/dev.099366' publications.firstAuthor='Upadhyai Priyanka' publications.id=1002044 publications.issue='20' publications.journal='Development' publications.month='Oct' publications.pages='4256-65' publications.pubMedId='24086079' publications.title='Brinker possesses multiple mechanisms for repression because its primary co-repressor, Groucho, may be unavailable in some cell types.' publications.volume='140' publications.year=2013\n",
      "Gene: publications.abstractText='Transposable elements (TEs) are mobile, repetitive sequences that make up significant fractions of metazoan genomes. Despite their near ubiquity and importance in genome and chromosome biology, most efforts to annotate TEs in genome sequences rely on the results of a single computational program, RepeatMasker. In contrast, recent advances in gene annotation indicate that high-quality gene models can be produced from combining multiple independent sources of computational evidence. To elevate the quality of TE annotations to a level comparable to that of gene models, we have developed a combined evidence-model TE annotation pipeline, analogous to systems used for gene annotation, by integrating results from multiple homology-based and de novo TE identification methods. As proof of principle, we have annotated \"TE models\"in Drosophila melanogaster Release 4 genomic sequences using the combined computational evidence derived from RepeatMasker, BLASTER, TBLASTX, all-by-all BLASTN, RECON, TE-HMM and the previous Release 3.1 annotation. Our system is designed for use with the Apollo genome annotation tool, allowing automatic results to be curated manually to produce reliable annotations. The euchromatic TE fraction of D. melanogaster is now estimated at 5.3% (cf. 3.86% in Release 3.1), and we found a substantially higher number of TEs (n = 6,013) than previously identified (n = 1,572). Most of the new TEs derive from small fragments of a few hundred nucleotides long and highly abundant families not previously annotated (e.g., INE-1). We also estimated that 518 TE copies (8.6%) are inserted into at least one other TE, forming a nest of elements. The pipeline allows rapid and thorough annotation of even the most complex TE models, including highly deleted and/or nested elements such as those often found in heterochromatic sequences. Our pipeline can be easily adapted to other genome sequences, such as those of the D. melanogaster heterochromatin or other species in the genus Drosophila.' publications.doi='10.1371/journal.pcbi.0010022' publications.firstAuthor='Quesneville Hadi' publications.id=1015598 publications.issue='2' publications.journal='PLoS Comput. Biol.' publications.month='Jul' publications.pages='166-75' publications.pubMedId='16110336' publications.title='Combined evidence annotation of transposable elements in genome sequences.' publications.volume='1' publications.year=2005\n",
      "Gene: publications.abstractText='Transposable elements are found in the genomes of nearly all eukaryotes. The recent completion of the Release 3 euchromatic genomic sequence of Drosophila melanogaster by the Berkeley Drosophila Genome Project has provided precise sequence for the repetitive elements in the Drosophila euchromatin. We have used this genomic sequence to describe the euchromatic transposable elements in the sequenced strain of this species. We identified 85 known and eight novel families of transposable element varying in copy number from one to 146. A total of 1,572 full and partial transposable elements were identified, comprising 3.86% of the sequence. More than two-thirds of the transposable elements are partial. The density of transposable elements increases an average of 4.7 times in the centromere-proximal regions of each of the major chromosome arms. We found that transposable elements are preferentially found outside genes; only 436 of 1,572 transposable elements are contained within the 61.4 Mb of sequence that is annotated as being transcribed.A large proportion of transposable elements is found nested within other elements of the same or different classes. Lastly, an analysis of structural variation from different families reveals distinct patterns of deletion for elements belonging to different classes. This analysis represents an initial characterization of the transposable elements in the Release 3 euchromatic genomic sequence of D. melanogaster for which comparison to the transposable elements of other organisms can begin to be made. These data have been made available on the Berkeley Drosophila Genome Project website for future analyses.' publications.doi=None publications.firstAuthor='Kaminker Joshua S' publications.id=1005240 publications.issue='12' publications.journal='Genome Biol.' publications.month=None publications.pages='RESEARCH0084' publications.pubMedId='12537573' publications.title='The transposable elements of the Drosophila melanogaster euchromatin: a genomics perspective.' publications.volume='3' publications.year=2002\n",
      "Gene: publications.abstractText='Two Drosophila Hox genes involved in segmentation, fushi tarazu and bicoid, appear to have acquired these roles by functional divergence from classical homeotic genes. Recent results indicate how genes with critical functions in development can evolve completely different functions among species.' publications.doi=None publications.firstAuthor='Gibson G' publications.id=1007735 publications.issue='12' publications.journal='Curr. Biol.' publications.month='Jun' publications.pages='R452-5' publications.pubMedId='10873798' publications.title='Evolution: hox genesand the cellared wine principle.' publications.volume='10' publications.year=2000\n",
      "Gene: publications.abstractText='Using X-ray mutagenesis we have induced and recovered phenotypic revertants of four dominant mutations thought to be associated with the Antennapedia complex of Drosophila melanogaster. These include seven revertants of Antennapedia-73b (Antp(73b)), six of Extra Sex Combs of Wakimoto (Scx(w)), three of Deformed (Dfd) and one of Humeral (Hu). Fifteen of the 17 revertants are associated with chromosomal aberrations and localize Antp(73b), Scx( w) and Hu to polytene chromosome bands 84B1,2. The Dfd lesion is apparently located in or adjacent to bands 84A4,5. Since all of the dominants are reverted by events that delete their respective chromosomal loci, we conclude that all four are the result of a gain-of-function lesions. Complementation analysis of the various revertant chromosomes has shown that Scx(w) and Hu are dominant allelic variants of the Antp locus. The Dfd lesion represents a dominant mutation at a locus just proximal to Antp and previously only occupied by recessive lethal mutations. Characterization of the revertants of Scx(w) and a comparison with the properties of the original mutation has revealed that the original lesion has effects on both the Antp and Sex Combs Reduced (Scr) loci and that these defects are in some cases separable by the reverting event.' publications.doi=None publications.firstAuthor='Hazelrigg T' publications.id=1000753 publications.issue='3' publications.journal='Genetics' publications.month='Nov' publications.pages='581-600' publications.pubMedId='17246168' publications.title='Revertants of Dominant Mutations Associated with the Antennapedia Gene Complex of DROSOPHILA MELANOGASTER: Cytology and Genetics.' publications.volume='105' publications.year=1983\n",
      "Gene: publications.abstractText='We address the problem of finding statistically significant associations between cis-regulatory motifs and functional gene sets, in order to understand the biological roles of transcription factors. We develop a computational framework for this task, whose features include a new statistical score for motif scanning, the use of different scores for predicting targets of different motifs, and new ways to deal with redundancies among significant motif-function associations. This framework is applied to the recently sequenced genome of the jewel wasp, Nasonia vitripennis, making use of the existing knowledge of motifs and gene annotations in another insect genome, that of the fruitfly. The framework uses cross-species comparison to improve the specificity of its predictions, and does so without relying upon non-coding sequence alignment. It is thereforewell suited for comparative genomics across large evolutionary divergences, where existing alignment-based methods are not applicable. We also apply the framework to find motifs associated with socially regulated gene sets in the honeybee, Apis mellifera, using comparisons with Nasonia, a solitary species, to identify honeybee-specific associations.' publications.doi='10.1371/journal.pcbi.1000652' publications.firstAuthor='Kim Jaebum' publications.id=1007904 publications.issue='1' publications.journal='PLoS Comput. Biol.' publications.month='Jan' publications.pages='e1000652' publications.pubMedId='20126523' publications.title='Functional characterization of transcription factor motifs using cross-species comparison across large evolutionary distances.' publications.volume='6' publications.year=2010\n",
      "Gene: publications.abstractText='We employed robotic methods and the whole-genome sequence of Drosophila melanogaster to facilitate a large-scale expression screen for spatially restricted transcripts in Drosophila embryos. In this screen, we identified a pair of genes, scylla (scyl) and charybde (chrb), that code for dorsal transcripts in early Drosophila embryos and are homologous to the human apoptotic gene RTP801. In Drosophila, both gene products are transcriptionally regulated targets of Dpp/Zen-mediated signal transduction and appear more generally to be downstream targets of homeobox regulation. Gene disruption studies revealed the functional redundancy of scyl and chrb, as well as their requirement for embryonic head involution. From the perspective of functional genomics, our studies demonstrate that global surveys of gene expression can complement traditional genetic screening methods for the identification of genes essential for development: beginning from their spatio-temporal expression profiles and extending to their downstream placement relative to dpp and zen, our studies reveal roles for the scyl and chrb gene products as links between patterning and cell death.' publications.doi='10.1016/j.ydbio.2005.12.014' publications.firstAuthor='Scuderi Anne' publications.id=1007888 publications.issue='1' publications.journal='Dev. Biol.' publications.month='Mar' publications.pages='110-22' publications.pubMedId='16423342' publications.title='scylla and charybde, homologues of the human apoptotic gene RTP801, are required for head involution in Drosophila.' publications.volume='291' publications.year=2006\n",
      "Gene: publications.abstractText='We have analyzed the contributions made by maternal and zygotic genes to the establishment of the expression patterns of four zygotic patterning genes: decapentaplegic (dpp), zerknüllt (zen), twist (twi), and snail (sna). All of these genes are initially expressed either dorsally or ventrally in the segmented region of the embryo, and at the poles. In the segmented region of the embryo, correct expression of these genes depends on cues from the maternal morphogen dorsal (dl). The dl gradient appears to be interpreted on three levels: dorsal cells express dpp and zen, but not twi and sna; lateral cells lack expression of all four genes; ventral cells express twi and sna, but not dpp and zen. dl appears to activate the expression of twi and sna and repress the expression of dpp and zen. Polar expression of dpp and zen requires the terminal system to override the repression by dl, while that of twi and sna requires the terminal system to augment activation by dl. The zygotic expression patterns established by the maternal genes appear to specify autonomous domains that carry out independent developmental programs, insofar as mutations in the genes that are expressed ventrally do not affect the initiation or ontogeny of the expression patterns of the genes that are expressed dorsally, and vice versa. However, interactions between the zygotic genes specific to a particular morphological domain appear to be important for further elaboration of the three levels specified by dl. Two of the genes, dpp and twi, are unaffected by mutations in any of the tested zygotic dorsal-ventral genes, suggesting that dpp and twi are the primary patterning genes for dorsal ectoderm and mesoderm, respectively.' publications.doi=None publications.firstAuthor='Ray R P' publications.id=1007768 publications.issue='1' publications.journal='Development' publications.month='Sep' publications.pages='35-54' publications.pubMedId='1765005' publications.title='The control of cell fate along the dorsal-ventral axis of the Drosophila embryo.' publications.volume='113' publications.year=1991\n",
      "Gene: publications.abstractText=\"We have analyzed the function of the Decapentaplegic (Dpp) and Hedgehog (Hh) signaling pathways in partitioning the dorsal head neurectoderm of the Drosophila embryo. This region, referred to as the anterior brain/eye anlage, gives rise to both the visual system and the protocerebrum. The anlage splits up into three main domains: the head midline ectoderm, protocerebral neurectoderm and visual primordium. Similar to their vertebrate counterparts, Hh and Dpp play an important role in the partitioning of the anterior brain/eye anlage. Dpp is secreted in the dorsal midline of the head. Lowering Dpp levels (in dpp heterozygotes or hypomorphic alleles) results in a 'cyclops' phenotype, where mid-dorsal head epidermis is transformed into dorsolateral structures, i.e. eye/optic lobe tissue, which causes a continuous visual primordium across the dorsal midline. Absence of Dpp results in the transformation of both dorsomedial and dorsolateral structures into brain neuroblasts. Regulatory genes that are required for eye/optic lobe fate, including sine oculis (so) and eyes absent (eya), are turned on in their respective domains by Dpp. The gene zerknuellt (zen), which is expressed in response to peak levels of Dpp in the dorsal midline, secondarily represses so and eya in the dorsomedial domain. Hh and its receptor/inhibitor, Patched (Ptc), are expressed in a transverse stripe along the posterior boundary of the eye field. As reported previously, Hh triggers the expression of determinants for larval eye (atonal) and adult eye (eyeless) in those cells of the eye field that are close to the Hh source. Eya and So, which are induced by Dpp, are epistatic to the Hh signal. Loss of Ptc, as well as overexpression of Hh, results in the ectopic induction of larval eye tissue in the dorsal midline (cyclopia). We discuss the similarities between vertebrate systems and Drosophila with regard to the fate map of the anterior brain/eye anlage, and its partitioning by Dpp and Hh signaling.\" publications.doi=None publications.firstAuthor='Chang T' publications.id=1007705 publications.issue='23' publications.journal='Development' publications.month='Dec' publications.pages='4691-704' publications.pubMedId='11731450' publications.title='Dpp and Hh signaling in the Drosophila embryonic eye field.' publications.volume='128' publications.year=2001\n",
      "Gene: publications.abstractText='We have characterized at the molecular level the zerknüllt (zen) region of the Drosophila subobscura Antennapedia complex. The sequence comparison between D. subobscura and D. melanogaster shows an irregular distribution of the conserved and diverged regions, with the homeobox and a putative activating domain completely conserved. Comparisons of the promoter sequence and pattern of expression of the gene during development suggest that the regulation of zen has been conserved during evolution. The conservation of zen expression in a subpopulation of the polar cells indicates the existence of an important role in such cells. We describe a transitory segmented pattern of expression of zen in both species, suggesting the existence of interactions with a pair rule gene. Some indirect clues indicate that the z2 gene might be absent from the D. subobscura genome. A chromosome walk initiated to reach the proboscipedia gene of D. subobscura reveals that the distance between pb and zen is at least four times the one described for D. melanogaster and for D. pseudoobscura. Finally, we present cytological evidence showing that the ANT-C is inverted in D. subobscura as compared to D. melanogaster.' publications.doi=None publications.firstAuthor='Terol J' publications.id=1007797 publications.issue='9' publications.journal='Chromosoma' publications.month='May' publications.pages='613-24' publications.pubMedId='7587584' publications.title='Molecular characterization of the zerknüllt region of the Antennapedia complex of D. subobscura.' publications.volume='103' publications.year=1995\n",
      "Gene: publications.abstractText='We have cloned, from a beetle and a locust, genes that are homologous to the class 3 Hox genes of vertebrates. Outside the homeobox they sharesequence motifs with the Drosophila zerknüllt (zen) and z2 genes, and like zen, are expressed only in extraembryonic membranes. We conclude that the zen genes of Drosophila derive from a Hox class 3 sequence that formed part of the common ancestral Hox cluster, but that in insects this (Hox) gene has lost its role in patterning the anterio-posterior axis of the embryo, and acquired a new function. In the lineage leading to Drosophila, the zen genes have diverged particularly rapidly.' publications.doi=None publications.firstAuthor='Falciani F' publications.id=1007825 publications.issue='16' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Aug' publications.pages='8479-84' publications.pubMedId='8710895' publications.title='Class 3 Hox genes in insects and the origin of zen.' publications.volume='93' publications.year=1996\n",
      "Gene: publications.abstractText=\"We have identified in the 5' untranslated region of the Drosophila copia retrotransposon, 3' to the left LTR, a sequence for transcriptional regulation by homeoproteins. Co-transfection assays using expression vectors for homeoproteins and reporter vectors containing the lacZ gene under the control of either the entire copia LTR with 5' untranslated sequence, or a minimal heterologous promoter flanked with a 130 bp fragment containing the copia untranslated region, disclosed both positive and negative modulations of promoter activity in Drosophila cells in culture: a 5-10 fold decrease with engrailed, even-skipped and zerknüllt in DH33 cells, and a 10-30 fold increase with fushi tarazu and zerknüllt in Schneider II cells. In all cases, the regulatory effects were abolished with reporter plasmids deleted for a 58 bp fragment encompassing the putative homeoprotein binding sites. Mobility shift assays with a purified homeodomain-containing peptide demonstrated direct interaction with the 58 bp fragment, with an affinity in the 1-10 nM range as reported with the same peptide for other well characterized homeodomain binding regulatory sites. Foot-printing experiments with the extended LTR demonstrated protection of 'consensus' sequences, located within the 58 bp fragment. These homeodomain binding sites could be involved in the developmental regulation of the copia retrotransposon.\" publications.doi=None publications.firstAuthor='Cavarec L' publications.id=1007699 publications.issue='22' publications.journal='Nucleic Acids Res.' publications.month='Nov' publications.pages='5041-9' publications.pubMedId='8255758' publications.title='The Drosophila copia retrotransposon contains binding sites for transcriptional regulation by homeoproteins.' publications.volume='21' publications.year=1993\n",
      "Gene: publications.abstractText='We have investigated Drosophila salivary gland determination by examining the effects of mutations in pattern forming genes on the salivary gland primordium. We find that the anterior-posterior extent of the primordium, a placode of columnar epithelial cells derived from parasegment 2, is established by the positive action of the homeotic gene Sex combs reduced (Scr). Embryos mutant for Scr lack a detectable placode, while ectopic Scr expression leads to the formation of ectopic salivary glands. In contrast, the dorsal-ventral extent of the placode is regulated negatively. Functions dependent on the decapentaplegic product place a dorsal limit on the placode, while dorsal-dependent genes act to limit the placode ventrally. We propose a model in which these pattern forming genes act early to determine the salivary gland anlage by regulating the expression of salivary gland determining genes, which in turn control genes that are involved in salivary gland morphogenesis.' publications.doi=None publications.firstAuthor='Panzer S' publications.id=1007734 publications.issue='1' publications.journal='Development' publications.month='Jan' publications.pages='49-57' publications.pubMedId='1349523' publications.title='Organogenesis in Drosophila melanogaster: embryonic salivary gland determination is controlled by homeotic and dorsoventral patterning genes.' publications.volume='114' publications.year=1992\n",
      "Gene: publications.abstractText='We have observed that zygotic transcription does not initiate at a single point in Drosophila embryos. Rather, a gene initiates transcription in a few nuclei of a fraction of embryos. During succeeding cycles, the frequency of transcribing embryos, and of nuclei transcribing in those embryos, gradually increases. For the fushi tarazu (ftz) gene, the timing of this process is regulated by the concentration of the maternally loaded, repressing transcription factor tramtrack (ttk). Altering the dose of Ttk protein in embryos shifts the activation of ftz transcription either forward or backward during development but does not effect Krüppel (Kr) activation. We have observed that the transcription of several genes, including ftz, is triggered in embryos at a critical nuclear density; therefore, we suggest that titration of transcription factors like ttk by the nucleocytoplasmic ratio triggers zygotic transcription in Drosophila.' publications.doi=None publications.firstAuthor='Pritchard D K' publications.id=1007817 publications.issue='9' publications.journal='Genes Dev.' publications.month='May' publications.pages='1131-42' publications.pubMedId='8654928' publications.title='Activation of transcription in Drosophila embryos is a gradual process mediated by the nucleocytoplasmic ratio.' publications.volume='10' publications.year=1996\n",
      "Gene: publications.abstractText='We have studied the ability of the Drosophila gap proteins Krüppel and hunchback to function as transcriptional regulators in cultured cells. Both proteins bind to specific sites in a 100-bp DNA fragment located upstream of the segment polarity gene engrailed, which also contains functional binding sites for a number of homeo box proteins. The hunchback protein is a strikingly concentration-dependent activator of transcription, capable of functioning both by itself and also synergistically with the pair-rule proteins fushi tarazu and paired. In contrast, Krüppel is a transcriptional repressor that can block transcription induced either by hunchback or by several different homeo box proteins. While repression of the homeo box protein activators requires a Krüppel-binding site on the DNA, repression of hunchback can occur efficiently in the absence of a Krüppel-binding site. We discuss the possible molecular mechanisms underlying these activities, as well as the potential significance of these results with respect to segmentation in Drosophila.' publications.doi=None publications.firstAuthor='Zuo P' publications.id=1007893 publications.issue='2' publications.journal='Genes Dev.' publications.month='Feb' publications.pages='254-64' publications.pubMedId='1671661' publications.title='Activation and repression of transcription by the gap proteins hunchback and Krüppel in cultured Drosophila cells.' publications.volume='5' publications.year=1991\n",
      "Gene: publications.abstractText='We have studied the transcriptional activity of the Drosophila homeodomain protein Engrailed (En) by using a transient expression assay employing Schneider L2 cells. En was found to very strongly repress promoters activated by a variety of different activator proteins. However, unlike another Drosophila homeodomain-containing repressor, Even-skipped (Eve), En was unable to repress the activity of several basal promoters in the absence of activator expression. These findings indicate that En is a specific repressor of activated transcription, and suggest that En may repress transcription by a different mechanism than Eve, perhaps by interfering with interactions between transcriptional activators and the general transcription machinery. By analyzing the properties of a variety of En mutants, we identified a minimal repression domain composed of 55 residues, which can function when fused to a heterologous DNA binding domain. Like repression domains identified in the Drosophila repressors Eve and Krüppel, the En repression domain is rich in alanine residues (26%), but unlike these other domains, is moderately charged (six arginine and three glutamic acid residues). Separate regions of En that may in some circumstances function in transcriptional activation were also identified.' publications.doi=None publications.firstAuthor='Han K' publications.id=1007700 publications.issue='7' publications.journal='EMBO J.' publications.month='Jul' publications.pages='2723-33' publications.pubMedId='8334991' publications.title='Functional domains of the Drosophila Engrailed protein.' publications.volume='12' publications.year=1993\n",
      "Gene: publications.abstractText='We have undertaken a developmental genetic analysis of labial (lab), the most proximal gene in the Antennapedia complex (ANT-C) of Drosophila melanogaster. The terminal phenotype of mutant embryos was examined in cuticle preparations, in thin sections, and by scanning electron microscopy. These preparations revealed a failure of head involution and the loss or disruption of several head structures, including the salivary glands and the H-piece and ventral arm of the cephalopharyngeal apparatus. Although these structures are presumed to derive from the gnathocephalic segments, we argue that the observed defects are likely to be a secondary consequence of a failure of head involution. A function for lab in the development of the adult head was inferred from the phenotype of animals bearing hypomorphic alleles and from clones of lab- tissue generated by mitotic recombination. Two aspects of the mutant phenotype were manifested. Ventrally, a deletion and/or disruption of tissue occurred in the maxillary palp and vibrissae regions. Dorsally, the posterior head appeared to be transformed to a thoracic-like identity. Mutations in lab, like those in the Deformed and proboscipedia loci of the ANT-C, reveal a homoeotic phenotype only in the adult stage of the life cycle.' publications.doi=None publications.firstAuthor='Merrill V K' publications.id=1005537 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Oct' publications.pages='376-91' publications.pubMedId='2570723' publications.title='A genetic and developmental analysis of mutations in labial, a gene necessary for proper head formation in Drosophila melanogaster.' publications.volume='135' publications.year=1989\n",
      "Gene: publications.abstractText='We have used a transient expression assay employing Drosophila tissue culture cells to study the potential of several Drosophila homeobox proteins to function as transcriptional regulators. A 96 bp fragment from the promoter region of the segment polarity gene engrailed, previously shown to contain five copies of a 10 bp consensus binding site for these proteins, enhanced transcription in the presence, but not the absence, of several different homeobox protein expression vectors. It is interesting that cotransfection with combinations of expression vectors encoding the homeobox proteins fushi tarazu, paired, and/or zen resulted in substantial synergistic increases in expression. In contrast, the products of the even-skipped and engrailed genes were found to repress, or quench, the activation induced by the other proteins. We discuss the implications of these results with respect to the role of homeobox genes in the control of embryonic development, and propose a \"multi-switch\"model whereby the activity of a target gene depends on the interactions of different homeobox proteins with multiple copies of a common binding site.' publications.doi=None publications.firstAuthor='Han K' publications.id=1007727 publications.issue='4' publications.journal='Cell' publications.month='Feb' publications.pages='573-83' publications.pubMedId='2563673' publications.title='Synergistic activation and repression of transcription by Drosophila homeobox proteins.' publications.volume='56' publications.year=1989\n",
      "Gene: publications.abstractText='We have used a transient expression assay employing Drosophila tissue culture cells to study the transcriptional repression activity of the homeo domain protein Even-skipped (Eve). Eve was found to repress all promoters that contained Eve-binding sites, including both TATA-containing and TATA-lacking minimal promoters, as well as promoters activated by several different classes of activator proteins. These findings suggest that the general transcription machinery can be a target of Eve. By analyzing properties of a variety of Eve mutants and chimeric fusion proteins, we have identified several features important for efficient repression. In addition to the DNA-binding domain, a potent repressor requires a repression domain, which can be as small as 27 residues. The minimal 57-residue Eve repression domain, as well as several others studied here, were all found to be proline rich and to contain a high percentage of hydrophobic residues. An intriguing feature of the strong repressors was that their DNA-binding activities, measured by gel retention assays with nuclear extracts, were significantly less than those of derivatives inactive in repression.' publications.doi=None publications.firstAuthor='Han K' publications.id=1007681 publications.issue='3' publications.journal='Genes Dev.' publications.month='Mar' publications.pages='491-503' publications.pubMedId='8095483' publications.title='Transcriptional repression by the Drosophila even-skipped protein: definition of a minimal repression domain.' publications.volume='7' publications.year=1993\n",
      "Gene: publications.abstractText='We identified Drosophila Smurf (DSmurf) as a negative regulator ofsignaling by the BMP2/4 ortholog DPP during embryonic dorsal-ventral patterning. DSmurf encodes a HECT domain ubiquitin-protein ligase, homologous to vertebrate Smurf1 and Smurf2, that binds the Smad1/5 ortholog MAD and likely promotes its proteolysis. The essential function of DSmurf is restricted to its action on the DPP pathway. DSmurf has two distinct, possibly mechanistically separate, functions in controlling DPP signaling. Prior to gastrulation, DSmurf mutations cause a spatial increase in the DPP gradient, as evidenced by ventrolateral expansion in expression domains of target genes representing all known signaling thresholds. After gastrulation, DSmurf mutations cause a temporal delay in downregulation of earlier DPP signals, resulting in a lethal defect in hindgut organogenesis.' publications.doi=None publications.firstAuthor='Podos S D' publications.id=1007872 publications.issue='4' publications.journal='Dev. Cell' publications.month='Oct' publications.pages='567-78' publications.pubMedId='11703946' publications.title='The DSmurf ubiquitin-protein ligase restricts BMP signaling spatially and temporally during Drosophila embryogenesis.' publications.volume='1' publications.year=2001\n",
      "Gene: publications.abstractText='We recently uncovered a novel genetic mechanism that generates the phenotypic uniformity, or canalization, of BMP signaling and cell fate specification during patterning of the dorsal-ventral (D/V) axis in D. melanogaster embryos. We went on to show that other wild-type Drosophila species lack this canalizing genetic circuitry and, consequently, have non-robust D/V patterning. In this review, we propose molecular mechanisms that may give rise to stereotyped BMP signaling, and we identify an additional species that could have decanalized D/V patterning. Extension of these analyses could in turn help explain why canalization is not a universal necessity for species survival.' publications.doi='10.4161/19336934.2014.983385' publications.firstAuthor='Gavin-Smyth Jackie' publications.id=82004051 publications.issue='3' publications.journal='Fly (Austin)' publications.month=None publications.pages='170-5' publications.pubMedId='25482877' publications.title='zen and the art of phenotypic maintenance: canalization of embryonic dorsal-ventral patterning in Drosophila.' publications.volume='8' publications.year=2014\n",
      "Gene: publications.abstractText='We report here the genetical and molecular characterization of a new Drosophila zygotic lethal locus, vrille (vri). Vri alleles act not only as dominant maternal enhancers of embryonic dorsoventral patterning defects caused by easter and decapentaplegic (dpp) mutations, but also as dominant zygotic enhancers of dpp alleles for phenotypes in wing. The vri gene encodes a new member of the bZIP family of transcription factors closely related to gene 9 of Xenopus laevis, induced by thyroid hormone during the tadpole tail resorption program, and NF-IL3A, a human T cell transcription factor that transactivates the interleukin-3 promoter. NF-IL3A shares 93% similarity and 60% identity with Vri for a stretch of 68 amino acids that includes the bZIP domain. Although all the alleles tested behave like antimorphs, the dominant enhancement is also seen with a nonsense mutation allele that prevents translation of the bZIP domain. Because of the strong domainant enhancement of dpp phenotypes by vri alleles in both embryo and wing, and also the similarity between the wing vein phenotypes caused by the vri and shortvein dpp alleles, we postulate that vri interacts either directly or indirectly with certain components of the dpp (a TGF beta homologue) signal transduction pathway.' publications.doi=None publications.firstAuthor='George H' publications.id=1007752 publications.issue='4' publications.journal='Genetics' publications.month='Aug' publications.pages='1345-63' publications.pubMedId='9258679' publications.title='The vrille gene of Drosophila is a maternal enhancer of decapentaplegic and encodes a new member of the bZIP family of transcription factors.' publications.volume='146' publications.year=1997\n",
      "Gene: publications.abstractText='We report the isolation and characterization of a putative angiotensin converting enzyme (ACE) in Drosophila, called Race. General interest in mammalian ACE stems from its association with high blood pressure; ACE has also been implicated in a variety of other physiological processes including the processing of neuropeptides and gut peristalsis. Mammalian ACE is a membrane associated zinc binding protease that converts angiotensin I (A I) into angiotensin II (A II). A II functions as a potent vasoconstrictor by triggering a G-coupled receptor system in the smooth muscles that line blood vessels. Drosophila Race is composed of 615 amino acid residues, and shares extensive sequence identity with mammalian ACE over its entire length (over 42% overall identity and greater than 60% similarity). Evidence is presented that Race might correspond to a target of the homeobox regulatory gene, zerknullt (zen). Soon after zen expression is restricted to the dorsal-most regions of the embryonic ectoderm, Race is activated in a coincident pattern and becomes associated with the amnioserosa during germ band elongation, shortening and heart morphogenesis. After germ band elongation, Race is also expressed in both the anterior and posterior midgut, where it persists throughout embryogenesis. Race expression is lost from the dorsal ectoderm in either zen- or dpp- mutants, although gut expression is unaffected. P-transformation assays and genetic complementation tests suggest that Race corresponds to a previously characterized lethal complementation group, 1(2)34Eb. Mutants die during larval/pupal development, and transheterozygotes for two different lethal alleles exhibit male sterility. We propose that Race might play a role in the contractions of the heart, gut, or testes and also suggest that Hox genes might be important for coordinating both developmental and physiological processes.' publications.doi=None publications.firstAuthor='Tatei K' publications.id=1007739 publications.issue='2-3' publications.journal='Mech. Dev.' publications.month='Jun' publications.pages='157-68' publications.pubMedId='7547464' publications.title='Race: a Drosophila homologue of the angiotensin converting enzyme.' publications.volume='51' publications.year=1995\n",
      "Gene: publications.abstractText='We show that the Drosophila protein DSP1, an HMG-1/2-like protein, binds DNA highly cooperatively with three members of the Rel family of transcriptional regulators (NF-kappaB, the p50 subunit of NF-kappaB, and the Rel domain of Dorsal). This cooperativity is apparent with DNA molecules bearing consensus Rel-protein-binding sites and is unaffected by the presence of a negative regulatory element, a sequence previously proposed to be important for mediating repression by these Rel proteins. The cooperativity observed in these DNA-binding assays is paralleled by interactions between protein pairs in the absence of DNA. We also show that in HeLa cells, as assayed by transient transfection, expression of DSP1 increases activation by Dorsal from the twist promoter and inhibits that activation from the zen promoter, consistent with the previously proposed idea that DSP1 can affect the action of Dorsal in a promoter-specific fashion.' publications.doi=None publications.firstAuthor='Brickman J M' publications.id=1007820 publications.issue='19' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Sep' publications.pages='10679-83' publications.pubMedId='10485885' publications.title='Interactions between an HMG-1 protein and members of the Rel family.' publications.volume='96' publications.year=1999\n",
      "Gene: publications.abstractText='While the role of the notochord and floor plate in patterning the dorsal-ventral (D/V) axis of the neural tube is clearly established, relatively little is known about the earliest stages of D/V regionalization. In an effort to examine more closely the initial, preneural plate stages of regionalization along the prospective D/V neural axis, we have performed a series of explant experiments employing xHB9, a novel marker of the motor neuron region in Xenopus. Using tissue recombinants and Keller explants we show that direct mesodermal contact is both necessary and sufficient for the initial induction of xHB9 in the motor neuron region. We also show that presumptive neural plate explants removed as early as midgastrulation and cultured in isolation are already specified to express xHB9 but do so in an inappropriate spatial pattern while identical explants are specified to express the floor plate marker vhh-1 with correct spatial patterning. Our data suggest that, in addition to floor plate signaling, continued interactions with the underlying mesoderm through neural tube stages are essential for proper spatial patterning of the motor neuron region.' publications.doi='10.1006/dbio.1997.8625' publications.firstAuthor='Saha M S' publications.id=1007828 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Jul' publications.pages='209-23' publications.pubMedId='9242418' publications.title='Dorsal-ventral patterning during neural induction in Xenopus: assessment of spinal cord regionalization with xHB9, a marker for the motor neuron region.' publications.volume='187' publications.year=1997\n",
      "Gene: publications.abstractText='Wingless signaling plays a central role during epidermal patterning in Drosophila. We have analyzed zygotic requirements for Wingless signaling in the embryonic ectoderm by generating synthetic deficiencies that uncover more than 99% of the genome. We found no genes required for initial wingless expression, other than previously identified segmentation genes. In contrast, maintenance of wingless expression shows a high degree of zygotic transcriptional requirements. Besides known genes, we have identified at least two additional genomic regions containing new genes involved in Wingless maintenance. We also assayed for the zygotic requirements for Wingless response and found that no single genomic region was required for the cytoplasmic accumulation of Armadillo in the receiving cells. Surprisingly, embryos homozygously deleted for the candidate Wingless receptor, Dfrizzled2, showed a normal Wingless response. However, the Armadillo response to Wingless was strongly reduced in double mutants of both known members of the frizzled family in Drosophila, frizzled and Dfrizzled2. Based on their expression pattern during embryogenesis, different Frizzled receptors may play unique but overlapping roles in development. In particular, we suggest that Frizzled and Dfrizzled2 are both required for Wingless autoregulation, but might be dispensable for late Engrailed maintenance. While Wingless signaling in embryos mutant for frizzled and Dfrizzled2 is affected, Wingless protein is still internalized into cells adjacent to wingless-expressing cells. Incorporation of Wingless protein may therefore involve cell surface molecules in addition to the genetically defined signaling receptors of the frizzled family.' publications.doi=None publications.firstAuthor='Müller H A' publications.id=1007751 publications.issue='3' publications.journal='Development' publications.month='Feb' publications.pages='577-86' publications.pubMedId='9876186' publications.title='Wingless signaling in the Drosophila embryo: zygotic requirements and the role of the frizzled genes.' publications.volume='126' publications.year=1999\n",
      "Gene: publications.abstractText='With their power to shape animal morphology, few genes have captured the imagination of biologists as the evolutionarily conserved members of the Hox clusters have done. Recent research has provided new insight into how Hox proteins cause morphological diversity at the organismal and evolutionary levels. Furthermore, an expanding collection of sequences that are directly regulated by Hox proteins provides information on the specificity of target-gene activation, which might allow the successful prediction of novel Hox-response genes. Finally, the recent discovery of microRNA genes within the Hox gene clusters indicates yet another level of control by Hox genes in development and evolution.' publications.doi='10.1038/nrg1726' publications.firstAuthor='Pearson Joseph C' publications.id=1007889 publications.issue='12' publications.journal='Nat. Rev. Genet.' publications.month='Dec' publications.pages='893-904' publications.pubMedId='16341070' publications.title='Modulating Hox gene functions during animal body patterning.' publications.volume='6' publications.year=2005\n",
      "Gene: publications.abstractText='Worldwide, acute, and chronic pain affects 20% of the adult population and represents an enormous financial and emotional burden. Using genome-wide neuronal-specific RNAi knockdown in Drosophila, we report a global screen for an innate behavior and identify hundreds of genes implicated in heat nociception, including the α2δ family calcium channel subunit straightjacket (stj). Mice mutant for the stj ortholog CACNA2D3 (α2δ3) also exhibit impaired behavioral heat pain sensitivity. In addition, in humans, α2δ3 SNP variants associate with reduced sensitivity to acute noxious heat and chronic back pain. Functional imaging in α2δ3 mutant mice revealed impaired transmission of thermal pain-evoked signals from the thalamus to higher-order pain centers. Intriguingly, in α2δ3 mutant mice, thermal pain and tactile stimulation triggered strong cross-activation, or synesthesia, of brain regions involved in vision, olfaction, and hearing.' publications.doi='10.1016/j.cell.2010.09.047' publications.firstAuthor='Neely G Gregory' publications.id=1000034 publications.issue='4' publications.journal='Cell' publications.month='Nov' publications.pages='628-38' publications.pubMedId='21074052' publications.title='A genome-wide Drosophila screen for heat nociception identifies α2δ3 as an evolutionarily conserved pain gene.' publications.volume='143' publications.year=2010\n",
      "Gene: publications.abstractText=\"cDNA cloning is a central technology in molecular biology. cDNA sequences are used to determine mRNA transcript structures, including splice junctions, open reading frames (ORFs) and 5'- and 3'-untranslated regions (UTRs). cDNA clones are valuable reagents for functional studies of genes and proteins. Expressed Sequence Tag (EST) sequencing is the method of choice for recovering cDNAs representing many of the transcripts encoded in a eukaryotic genome. However, EST sequencing samples a cDNA library at random, and it recovers transcripts with low expression levels inefficiently. We describe a PCR-based method for directed screening of plasmid cDNA libraries. We demonstrate its utility in a screen of libraries used in our Drosophila EST projects for 153 transcription factor genes that were not represented by full-length cDNA clones in our Drosophila Gene Collection. We recovered high-quality, full-length cDNAs for 72 genes and variously compromised clones for an additional 32 genes. The method can be used at any scale, from the isolation of cDNA clones for a particular gene of interest, to the improvement of large gene collections in model organisms and the human. Finally, we discuss the relative merits of directed cDNA library screening and RT-PCR approaches.\" publications.doi='10.1093/nar/gni184' publications.firstAuthor='Hoskins Roger A' publications.id=1000113 publications.issue='21' publications.journal='Nucleic Acids Res.' publications.month=None publications.pages='e185' publications.pubMedId='16326860' publications.title='Rapid and efficient cDNA library screening by self-ligation of inverse PCR products (SLIP).' publications.volume='33' publications.year=2005\n",
      "Gene: publications.abstractText=\"cappuccino and spire are unique Drosophila maternal-effect loci that participate in pattern formation in both the anteroposterior and dorsoventral axes of the early embryo. Mutant females produce embryos lacking pole cells, polar granules, and normal abdominal segmentation. They share these defects with the posterior group of maternal-effect genes. Although embryos are defective in abdominal segmentation, in double mutant combinations with Bicaudal D, abdominal segments can be formed in the anterior half of the egg. This indicates that embryos produced by mutant females contain the 'posterior determinant' required for abdominal segmentation (Nüsslein-Volhard et al. 1987) and suggests that the wild-type gene products are not required for production of the posterior determinant but, rather, for its localization or stabilization. The vasa protein, a component of polar granules, is not localized at the posterior pole of mutant egg chambers or embryos, providing additional support for the hypothesis that localization to or stabilization of substances at the posterior pole of the egg chamber is defective in mutant females. Females mutant for the strongest alleles also produce dorsalized embryos. Phenotypic analysis reveals that these dorsalized embryos also have abdominal segmentation defects. The mutant phenotypes can be ordered in a series of increasing severity. Pole cell formation is most sensitive to loss of functional gene products, followed by abdominal segmentation, whereas normal dorsoventral patterning is the least sensitive to loss of functional gene products. In addition, mutant females contain egg chambers that appear to be dorsalized, resulting in the production of eggs with dorsalized eggshells. Germ-line mosaics indicate that cappuccino and spire are required in the oocyte-nurse cell complex. This suggests that the eggshell phenotype results from altered pattern in the underlying germ cell. Also, we defined the epistatic relationships between several early patterning loci, on the basis of an analysis of the eggs and embryos produced by females doubly mutant for cappuccino or spire and other loci that affect the pattern of both the egg and the embryo. On the basis of our current knowledge of the genes involved in this process, we formulated a working model for the early steps in dorsoventral patterning.\" publications.doi=None publications.firstAuthor='Manseau L J' publications.id=1007698 publications.issue='9' publications.journal='Genes Dev.' publications.month='Sep' publications.pages='1437-52' publications.pubMedId='2514120' publications.title='cappuccino and spire: two unique maternal-effect loci required for both the anteroposterior and dorsoventral patterns of the Drosophila embryo.' publications.volume='3' publications.year=1989\n",
      "Gene: publications.abstractText='decapentaplegic (dpp) encodes a Drosophila transforming growth factor-beta homologue that functions as a morphogen in the developing embryo and in adult appendage formation. In the wing imaginal disc, a Dpp gradient governs patterning along the anteroposterior axis by inducing regional expression of diverse genes in a concentration-dependent manner. Recent studies show that responses to graded Dpp activity also require an input from a complementary and opposing gradient of Brinker (Brk), a transcriptional repressor protein encoded by a Dpp target gene. Here we show that Brk harbours a functional and transferable repression domain, through which it recruits the corepressors Groucho and CtBP. By analysing transcriptional outcomes arising from the genetic removal of these corepressors, and by ectopically expressing Brk variants in the embryo, we demonstrate that these corepressors are alternatively used by Brk for repressing some Dpp-responsive genes, whereas for repressing other distinct target genes they are not required. Our results show that Brk utilizes multiple means to repress its endogenous target genes, allowing repression of a multitude of complex Dpp target promoters.' publications.doi='10.1093/emboj/20.20.5725' publications.firstAuthor='Hasson P' publications.id=1007718 publications.issue='20' publications.journal='EMBO J.' publications.month='Oct' publications.pages='5725-36' publications.pubMedId='11598015' publications.title='Brinker requires two corepressors for maximal and versatile repression in Dpp signalling.' publications.volume='20' publications.year=2001\n",
      "Gene: publications.abstractText='even-skipped (eve) is a homeodomain-encoding gene that is a genetically defined repressor of Ultrabithorax (Ubx), fushi-tarazu (ftz), and wingless (wg). Here we report that purified eve protein represses transcription in vitro at the Ubx promoter, in a DNA binding site-dependent manner. eve protein represses transcription when bound either upstream or downstream of the RNA start site or when DNA binding sites are in either orientation. We also show that eve represses expression from the Ubx promoter in Drosophila tissue culture cells, again in a binding site-dependent manner. Deletion of eve DNA binding sites does not alter transcription in the absence of eve, and so repression is not likely to be the result of eve competitively inhibiting an activator protein from binding to the same DNA element. Instead, we propose that eve protein is probably interfering with the function of proteins bound at other locations in the promoter. The biochemical demonstration that a Drosophila homeodomain protein can directly regulate RNA synthesis strengthens the view that this class of regulators act as transcription factors to control development.' publications.doi=None publications.firstAuthor='Biggin M D' publications.id=1007854 publications.issue='3' publications.journal='Cell' publications.month='Aug' publications.pages='433-40' publications.pubMedId='2569362' publications.title='A purified Drosophila homeodomain protein represses transcription in vitro.' publications.volume='58' publications.year=1989\n",
      "Gene: publications.abstractText=\"mRNA-associated processes and gene structure in eukaryotes are typically treated as separate research subjects. Here, we bridge this separation and leverage the extensive multidisciplinary work on Drosophila melanogaster to examine the roles that capping, splicing, cleavage/polyadenylation, and telescripting (i.e, the protection of nascent transcripts from premature cleavage/polyadenylation by the splicing factor U1) might play in shaping exon-intron architecture in protein-coding genes. Our findings suggest that the distance between subsequent internal 5' splice sites (5'ss) in Drosophila genes is constrained such that telescripting effects are maximized, in theory, and thus nascent transcripts are less vulnerable to premature termination. Exceptionally weak 5'ss and constraints on intron-exon size at the gene 5' end also indicate that capping might enhance the recruitment of U1 and, in turn, promote telescripting at this location. Finally, a positive correlation between last exon length and last 5'ss strength suggests that optimal donor splice sites in the proximity of the pre-mRNA tail may inhibit the processing of downstream polyadenylation signals more than weak donor splice sites do. These findings corroborate and build upon previous experimental and computational studies on Drosophila genes. They support the possibility, hitherto scantly explored, that mRNA-associated processes impose significant constraints on the evolution of eukaryotic gene structure.\" publications.doi='10.1534/g3.116.029231' publications.firstAuthor='Lepennetier Gildas' publications.id=1000121 publications.issue='6' publications.journal='G3 (Bethesda)' publications.month='Jun' publications.pages='1617-26' publications.pubMedId='27172210' publications.title='mRNA-Associated Processes and Their Influence on Exon-Intron Structure in Drosophila melanogaster.' publications.volume='6' publications.year=2016\n",
      "Gene: publications.abstractText=\"twist is one of the earliest expressed zygotically active genes required for dorsal-ventral pattern formation in the Drosophila embryo. Genetic studies suggest that this gene is activated in the ventral part of the blastoderm by maternally expressed dorsal gene product. Using P-element-mediated germ-line transformation, we have mapped a small (260 bp) dorsal-dependent ventral activator region (VAR) in the 5'-flanking region of the twist promoter that can direct the early ventral expression of a heterologous promoter. The VAR contains binding sites for a number of proteins present in extracts of Drosophila embryos. One of these sites bears homology to known binding sites for the dorsal transcription factor and is specifically bound by bacterially expressed dorsal protein. Furthermore, a 37-bp deletion that removes the dorsal-binding sequences abolishes the ventral-specific activity of the twist promoter constructs. Our data also show that additional sequences within the VAR are required to render the dorsal-binding sites functional. Finally, reverse genetic and biochemical data suggest that the transcription factor, encoded by the zeste gene may help control the overall level, if not the pattern, of twist expression.\" publications.doi=None publications.firstAuthor='Pan D J' publications.id=1007837 publications.issue='10' publications.journal='Genes Dev.' publications.month='Oct' publications.pages='1892-901' publications.pubMedId='1655573' publications.title='Functional analysis of the Drosophila twist promoter reveals a dorsal-binding ventral activator region.' publications.volume='5' publications.year=1991\n",
      "Gene: publications.abstractText='wimp is a dominant maternal-effect mutation that interacts with a specific subset of early-acting maternal and zygotic Drosophila genes. We show that wimp is a change-of-function mutation, allelic to mutations of the 140-kD subunit of RNA polymerase, which causes reduced transcription of interacting genes. Loci that do not interact with wimp are expressed at normal levels. We discuss these results in terms of specific interactions between transcription factors and RNA polymerase. Embryos from wimp mothers show unaltered fate maps and develop normally, despite the reduction of transcript levels at least twofold. We suggest that spatial cues are determined by a balance of segmentation gene products rather than their absolute concentrations. We also demonstrate powerful genetic screens for otherwise undetected loci required for segmentation, sex determination, and other early functions.' publications.doi=None publications.firstAuthor='Parkhurst S M' publications.id=1007811 publications.issue='3' publications.journal='Genes Dev.' publications.month='Mar' publications.pages='341-57' publications.pubMedId='2001838' publications.title='wimp, a dominant maternal-effect mutation, reduces transcription of a specific subset of segmentation genes in Drosophila.' publications.volume='5' publications.year=1991\n",
      "Gene: publications.abstractText='zerknüllt (zen) is one of approximately 10 zygotically active genes that control the differentiation of the dorsal-ventral (D/V) pattern during early embryogenesis in Drosophila. Past genetic analyses suggest that maternal factors repress the expression of zen in ventral regions, thereby restricting zen products to dorsal and dorsal-lateral regions of precellular embryos. Subsequent interactions with other zygotic D/V regulatory genes refine the zen pattern, restricting expression to the dorsal-most ectoderm. Here we describe the use of zen promoter fusions and P-element transformation to identify cis elements that are responsible for the complex spatial pattern of zen expression. The zen promoter shows a two-tier organization: Distal sequences mediate its initial response to maternal factors, whereas proximal sequences are responsible for the refinement of the pattern in older embryos. The distal regulatory element has the property of a silencer (or anti-enhancer) element and can act over a distance to repress ventral expression of a heterologous promoter. Also, we discuss evidence that proximal promoter sequences interact with factors that may be modulated by a cell-cell communication pathway.' publications.doi=None publications.firstAuthor='Doyle H J' publications.id=1007901 publications.issue='10' publications.journal='Genes Dev.' publications.month='Oct' publications.pages='1518-33' publications.pubMedId='2612903' publications.title='Spatial regulation of zerknüllt: a dorsal-ventral patterning gene in Drosophila.' publications.volume='3' publications.year=1989\n",
      "Gene: publications.abstractText='zerknüllt (zen) is unique among the 18 known homeo box genes in Drosophila since it is required for the differentiation of the dorsal-ventral pattern, and does not appear to be involved in the process of segmentation. Here we show that the zen region of the Antennapedia complex (ANT-C) consists of two closely linked homeo box genes, designated z1 and z2. The z1 and z2 transcription units show essentially identical patterns of expression during early development, which are consistent with the timing and sites of zen+ gene activity. The putative proteins encoded by z1 and z2 are highly divergent and are related only by virtue of homeo box homology. We have used P-element-mediated germ line transformation to show that z1 alone can provide zen+ gene function, suggesting that the z2 gene might be dispensable. The occurrence of closely linked homeo box genes that display similar patterns of expression is not unique to the zen locus. Such gene duplications might provide important clues to the evolution of the homeo box gene family in Drosophila and other organisms.' publications.doi=None publications.firstAuthor='Rushlow C' publications.id=1007850 publications.issue='10' publications.journal='Genes Dev.' publications.month='Dec' publications.pages='1268-79' publications.pubMedId='2892759' publications.title='Molecular characterization of the zerknüllt region of the Antennapedia gene complex in Drosophila.' publications.volume='1' publications.year=1987\n",
      "Gene: publications.abstractText='β-Arrestins have been implicated in the regulation of multiple signalling pathways. However, their role in organism development is not well understood. In this study, we report a new in vivo function of the Drosophila β-arrestin Kurtz (Krz) in the regulation of two distinct developmental signalling modules: MAPK ERK and NF-κB, which transmit signals from the activated receptor tyrosine kinases (RTKs) and the Toll receptor, respectively. Analysis of the expression of effectors and target genes of Toll and the RTK Torso in krz maternal mutants reveals that Krz limits the activity of both pathways in the early embryo. Protein interaction studies suggest a previously uncharacterized mechanism for ERK inhibition: Krz can directly bind and sequester an inactive form of ERK, thus preventing its activation by the upstream kinase, MEK. A simultaneous dysregulation of different signalling systems in krz mutants results in an abnormal patterning of the embryo and severe developmental defects. Our findings uncover a new in vivo function of β-arrestins and present a new mechanism of ERK inhibition by the Drosophila β-arrestin Krz.' publications.doi='10.1038/emboj.2010.202' publications.firstAuthor='Tipping Marla' publications.id=1007801 publications.issue='19' publications.journal='EMBO J.' publications.month='Oct' publications.pages='3222-35' publications.pubMedId='20802461' publications.title='β-arrestin Kurtz inhibits MAPK and Toll signalling in Drosophila development.' publications.volume='29' publications.year=2010\n",
      "Gene: publications.abstractText=None publications.doi='10.1002/(SICI)1521-1878(200003)22:3<305::AID-BIES13>3.0.CO;2-R' publications.firstAuthor='Wilkins A S' publications.id=1007808 publications.issue='3' publications.journal='Bioessays' publications.month='Mar' publications.pages='305-7' publications.pubMedId='10684591' publications.title='Drosophila: flying high in Zürich.' publications.volume='22' publications.year=2000\n",
      "Gene: publications.abstractText=None publications.doi='10.1002/bies.950121107' publications.firstAuthor='White R J' publications.id=1007721 publications.issue='11' publications.journal='Bioessays' publications.month='Nov' publications.pages='537-9' publications.pubMedId='1982208' publications.title='Cell type-specific enhancement in the Drosophila embryo by consensus homeodomain binding sites.' publications.volume='12' publications.year=1990\n",
      "Gene: publications.abstractText=None publications.doi='10.1038/35056022' publications.firstAuthor='Casci T' publications.id=1007707 publications.issue='3' publications.journal='Nat. Rev. Genet.' publications.month='Mar' publications.pages='161' publications.pubMedId='11256063' publications.title='Patterning back to front.' publications.volume='2' publications.year=2001\n",
      "Gene: publications.abstractText=None publications.doi='10.1093/emboj/17.23.6769' publications.firstAuthor='Shilo B Z' publications.id=1007876 publications.issue='23' publications.journal='EMBO J.' publications.month='Dec' publications.pages='6769-71' publications.pubMedId='9835652' publications.title='Flies over Crete: Drosophila molecular biology. Kolymbari, Crete, July 12-18, 1998.' publications.volume='17' publications.year=1998\n",
      "Gene: publications.abstractText=None publications.doi='10.1101/gad.939601' publications.firstAuthor='Courey A J' publications.id=1007902 publications.issue='21' publications.journal='Genes Dev.' publications.month='Nov' publications.pages='2786-96' publications.pubMedId='11691830' publications.title='Transcriptional repression: the long and the short of it.' publications.volume='15' publications.year=2001\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Bier E' publications.id=1007775 publications.issue='5' publications.journal='Cell' publications.month='May' publications.pages='681-4' publications.pubMedId='9182755' publications.title='Anti-neural-inhibition: a conserved mechanism for neural induction.' publications.volume='89' publications.year=1997\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Budd P S' publications.id=1007871 publications.issue='3' publications.journal='Trends Genet.' publications.month='Mar' publications.pages='74-6' publications.pubMedId='1674389' publications.title='What do the regulators regulate? First glimpse downstream.' publications.volume='7' publications.year=1991\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Courey A J' publications.id=1007792 publications.issue='1' publications.journal='Biochim. Biophys. Acta' publications.month='Mar' publications.pages='1-18' publications.pubMedId='7893745' publications.title='The establishment and interpretation of transcription factor gradients in the Drosophila embryo.' publications.volume='1261' publications.year=1995\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Denell R' publications.id=1007804 publications.issue='3' publications.journal='Genetics' publications.month='Nov' publications.pages='549-52' publications.pubMedId='7851753' publications.title='Discovery and genetic definition of the Drosophila Antennapedia complex.' publications.volume='138' publications.year=1994\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Fraser S E' publications.id=1007694 publications.issue='1' publications.journal='Cell' publications.month='Jan' publications.pages='41-55' publications.pubMedId='10647930' publications.title='The molecular metamorphosis of experimental embryology.' publications.volume='100' publications.year=2000\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Hayashi S' publications.id=1007773 publications.issue='5' publications.journal='Cell' publications.month='Nov' publications.pages='883-94' publications.pubMedId='1979524' publications.title='What determines the specificity of action of Drosophila homeodomain proteins?' publications.volume='63' publications.year=1990\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Kaufman T C' publications.id=1005520 publications.issue=None publications.journal='Adv. Genet.' publications.month=None publications.pages='309-62' publications.pubMedId='1971986' publications.title='Molecular and genetic organization of the antennapedia gene complex of Drosophila melanogaster.' publications.volume='27' publications.year=1990\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='McGinnis W' publications.id=1007704 publications.issue='2' publications.journal='Cell' publications.month='Jan' publications.pages='283-302' publications.pubMedId='1346368' publications.title='Homeobox genes and axial patterning.' publications.volume='68' publications.year=1992\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Miklos G L' publications.id=1007726 publications.issue='4' publications.journal='Cell' publications.month='Aug' publications.pages='521-9' publications.pubMedId='8752207' publications.title='The role of the genome project in determining gene function: insights from model organisms.' publications.volume='86' publications.year=1996\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Read D' publications.id=1007891 publications.issue='2-3' publications.journal='Biochim. Biophys. Acta' publications.month='Dec' publications.pages='79-93' publications.pubMedId='1457465' publications.title='Transcriptional regulators of Drosophila embryogenesis.' publications.volume='1114' publications.year=1992\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Rushlow C' publications.id=1007765 publications.issue=None publications.journal='Adv. Genet.' publications.month=None publications.pages='277-307' publications.pubMedId='2112301' publications.title='Role of the zerknüllt gene in dorsal-ventral pattern formation in Drosophila.' publications.volume='27' publications.year=1990\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Zhou J' publications.id=1007844 publications.issue=None publications.journal='Cold Spring Harb. Symp. Quant. Biol.' publications.month=None publications.pages='307-12' publications.pubMedId='9598364' publications.title='The regulation of enhancer-promoter interactions in the Drosophila embryo.' publications.volume='62' publications.year=1997\n"
     ]
    }
   ],
   "source": [
    "for gene in query.results(row=\"rr\"):\n",
    "    print(gene)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Gene: publications.abstractText=\"A 631-bp fragment containing the 5'-flanking region of the Drosophila melanogaster proliferating-cell nuclear antigen (PCNA) gene was placed upstream of the chloramphenicol acetyltransferase (CAT) gene of a CAT vector. A transient expression assay of CAT activity in Drosophila Kc cells transfected with this plasmid and a set of 5'-deletion derivatives revealed that the promoter function resided within a 192-bp region (-168 to +24 with respect to the transcription initiation site). Cotransfection with a zerknüllt (zen)-expressing plasmid specifically repressed CAT expression. However, cotransfection with expression plasmids for a nonfunctional zen mutation, even-skipped, or bicoid showed no significant effect on CAT expression. RNase protection analysis revealed that the repression by zen was at the transcription step. The target sequence of zen was mapped within the 34-bp region (-119 to -86) of the PCNA gene promoter, even though it lacked zen protein-binding sites. Transgenic flies carrying the PCNA gene regulatory region (-607 to +137 or -168 to +137) fused with lacZ were established. When these flies were crossed with the zen mutant, ectopic expression of lacZ was observed in the dorsal region of gastrulating embryos carrying the transgene with either construct. These results indicate that zen indirectly represses PCNA gene expression, probably by regulating the expression of some transcription factor(s) that binds to the PCNA gene promoter.\" publications.doi=None publications.firstAuthor='Yamaguchi M' publications.id=1007881 publications.issue='10' publications.journal='Mol. Cell. Biol.' publications.month='Oct' publications.pages='4909-17' publications.pubMedId='1681423' publications.title='Repression of the Drosophila proliferating-cell nuclear antigen gene promoter by zerknüllt protein.' publications.volume='11' publications.year=1991\n",
      "Gene: publications.abstractText=\"A DNA replication-related element (DRE)-binding factor (DREF) has been revealed to be an important transcription factor for activating promoters of cell proliferation and differentiation related genes. The amino acid sequences of DREF are conserved in evolutionary separate Drosophila species, Drosophila melanogaster (Dm) and Drosophila virilis (Dv) in three regions. In the present study, evidence was obtained that there are several highly conserved regions in the 5' flanking region between the DmDREF and DvDREF genes. Band mobility shift assays using oligonucleotides corresponding to these conserved regions revealed that specific trans-acting factors can bind to at least three regions -554 to -543 (5'-TTTGTTCTTGCG), -81 to -70 (5'-GCCCACGTGGCT) and +225 to +234 (5'-GCAATCAGTG). Using a transient luciferase expression assay, we demonstrated that the region -554 to -543 functions as a negative regulatory element for DmDREF promoter activity, while the regions -77 to -70 (5'-ACGTGGCT) and +225 to +236 (5'-GCAATCAGTGTT) function as positive regulatory elements. In previous studies, we observed that expression of the homeodomain protein Zerknüllt (Zen) represses PCNA gene transcription, by reducing the DNA binding activity of DREF. Here we show Zen downregulates DREF gene promoter activity through action on the region between +241 and +254 (5'-AGAATACTCAACA). In addition, the DmDREF promoter contains five DREs. Using a double stranded RNA-mediated interference method, we generated evidence that expression of DmDREF could be auto-regulated by DREF through the third DRE located at +211 to +218. In living flies we obtained results consistent with those obtained in vitro and in cultured cells. The study thus indicates that DmDREF is effectively regulated via highly conserved regions between the DmDREF and DvDREF promoters, suggesting the existence of common regulatory factors, and that DmDREF can be positively regulated by itself via the third DRE located in its most highly conserved region.\" publications.doi=None publications.firstAuthor='Kwon Eunjeong' publications.id=1007877 publications.issue='2' publications.journal='Gene' publications.month='May' publications.pages='101-16' publications.pubMedId='12758126' publications.title='Transcription control of a gene for Drosophila transcription factor, DREF by DRE and cis-elements conserved between Drosophila melanogaster and virilis.' publications.volume='309' publications.year=2003\n",
      "Gene: publications.abstractText='A critical step in understanding the mechanisms of development is in defining the steps at the molecular, cellular, and organismal levels in the developmental program for a given organism-so that given the egg one can predict not only how the embryo will develop but also how that embryo evolved from its ancestors. Using methods employed by chemists and engineers in modeling hierarchical systems, I have integrated current theory and experiment into a calculational method that can model early Drosophila embryogenesis on a personal computer. This quantitative calculation tool is simple enough to be useful for experimentalists in designing experiments yet detailed enough for theoreticians to derive new insights on the evolution of developmental genetic networks. By integrating the strengths of theoretical and experimental methods into a single engineering model that can compute the cascade of genetic networks in a real organism, I provide a new calculational tool that can apply current theory to current experimental data to study the evolution of developmental programs.' publications.doi='10.1006/jtbi.1996.0328' publications.firstAuthor='Bodnar J W' publications.id=1007754 publications.issue='4' publications.journal='J. Theor. Biol.' publications.month='Oct' publications.pages='391-445' publications.pubMedId='9367733' publications.title='Programming the Drosophila embryo.' publications.volume='188' publications.year=1997\n",
      "Gene: publications.abstractText='A gradient of the maternal morphogen dorsal (dl) establishes dorsal-ventral (D-V) polarity in the early Drosophila embryo. The dl concentration gradient is initiated by regulated nuclear transport, and only protein that enters nuclei is active in the D-V patterning process. Here we show that dl is a DNA-binding protein that specifically interacts with distal sequences of the zerknüllt (zen) promoter, one of the genetic targets of the morphogen. These zen sequences have the properties of a silencer element and can act over long distances to repress the expression of a heterologous promoter. The dl protein recognizes a sequence motif similar to that of the mammalian transcriptional activator NF-kappa B, which was shown to contain extensive homology with dl and the oncoprotein rel. We present evidence that the DNA-binding activity of the dl protein is mediated by the region of homology (the rel domain) conserved in the rel and NF-kappa B proteins.' publications.doi=None publications.firstAuthor='Ip Y T' publications.id=1007836 publications.issue='2' publications.journal='Cell' publications.month='Jan' publications.pages='439-46' publications.pubMedId='1988156' publications.title='The dorsal morphogen is a sequence-specific DNA-binding protein that interacts with a long-range repression element in Drosophila.' publications.volume='64' publications.year=1991\n",
      "Gene: publications.abstractText='A gradient of the maternal morphogen dorsal (dl) initiates the differentiation of various tissues along the dorsal-ventral axis of early Drosophila embryos. dl is a sequence-specific DNA-binding protein that is related to the mammalian regulatory factor NF-kappa B. Previous studies suggest that dl can function as a transcriptional repressor. To determine how dl functions as an activator we have examined the promoter of the mesoderm determinant gene twist (twi). Genetic studies suggest that peak levels of dl protein in ventral regions of early embryos initiate twi expression. Using a combination of promoter fusion-P-transformation assays, and in vitro DNA-binding assays coupled with site-directed mutagenesis, we establish a direct link between dl-binding sites and twi expression in the early embryo. We also present evidence that the dorsal-ventral limits of twi expression depend on the number and affinity of dl-binding sites present in its promoter. A comparison of twi with a second dl target gene, zen, suggests a correlation between the affinities of dl-binding sites and response to different thresholds of dl morphogen.' publications.doi=None publications.firstAuthor='Jiang J' publications.id=1007687 publications.issue='10' publications.journal='Genes Dev.' publications.month='Oct' publications.pages='1881-91' publications.pubMedId='1655572' publications.title='The dorsal morphogen gradient regulates the mesoderm determinant twist in early Drosophila embryos.' publications.volume='5' publications.year=1991\n",
      "Gene: publications.abstractText=\"A member of a small group of genes in Drosophila that define the segmentation pattern of the early embryo even-skipped (eve), which plays a key role in a network of interactions among segmentation genes. It appears to control morphogenesis by regulating the expression of the segmentation gene engrailed (en), and by autoregulating its own expression (M. Frasch and M.L., in preparation). Here we show that these regulatory interactions could occur at the level of transcription as a full-length eve protein binds with high affinity to specific sequences located near the 5' ends of the eve and en genes. The en binding sites contain at least one copy of a 10-base pair consensus sequence: T-C-A-A-T-T-A-A-A-T. In contrast, the 5' eve binding sites are relatively G-C rich and do not share obvious similarities with the 10-base pair consensus sequence associated with en. Other homeo box proteins can recognize both classes of eve binding sites, lending support to the proposal that regulatory interactions among homeo box genes involve a competition of different homeo box proteins for similar cis regulatory sequences.\" publications.doi='10.1038/332858a0' publications.firstAuthor='Hoey T' publications.id=1007852 publications.issue='6167' publications.journal='Nature' publications.month='Apr' publications.pages='858-61' publications.pubMedId='2895896' publications.title='Divergent homeo box proteins recognize similar DNA sequences in Drosophila.' publications.volume='332' publications.year=1988\n",
      "Gene: publications.abstractText='A morphogen gradient is an important concept in developmental biology, because it describes a mechanism by which the emission of a signal from one part of an embryo can determine the location, differentiation and fate of many surrounding cells. The value of this idea has been clear for over half a century, but only recently have experimental systems and methods of analysis progressed to the point where we begin to understand how a cell can sense and respond to tiny changes in minute concentrations of extracellular signalling factors.' publications.doi='10.1038/35101500' publications.firstAuthor='Gurdon J B' publications.id=1004012 publications.issue='6858' publications.journal='Nature' publications.month='Oct' publications.pages='797-803' publications.pubMedId='11677596' publications.title='Morphogen gradient interpretation.' publications.volume='413' publications.year=2001\n",
      "Gene: publications.abstractText='A recent study reveals specific functions of Hox3/zerknüllt (zen) in the extraembryonic and embryonic primordia of the red flour beetle, Tribolium castaneum. The results shed light on the functional evolution of bicoid, a zen paralogue of higher flies, which determines anterior body parts.' publications.doi='10.1016/j.cub.2005.03.022' publications.firstAuthor='Schmidt-Ott Urs' publications.id=1007753 publications.issue='7' publications.journal='Curr. Biol.' publications.month='Apr' publications.pages='R245-7' publications.pubMedId='15823522' publications.title='Insect serosa: a head line in comparative developmental genetics.' publications.volume='15' publications.year=2005\n",
      "Gene: publications.abstractText=\"Although all bilaterian animals have a related set of Hox genes, the genomic organization of this gene complement comes in different flavors. In some unrelated species, Hox genes are clustered; in others, they are not. This indicates that the bilaterian ancestor had a clustered Hox gene family and that, subsequently, this genomic organization was either maintained or lost. Remarkably, the tightest organization is found in vertebrates, raising the embarrassingly finalistic possibility that vertebrates have maintained best this ancestral configuration. Alternatively, could they have co-evolved with an increased ;organization' of the Hox clusters, possibly linked to their genomic amplification, which would be at odds with our current perception of evolutionary mechanisms? When discussing the why's and how's of Hox gene clustering, we need to account for three points: the mechanisms of cluster evolution; the underlying biological constraints; and the developmental modes of the animals under consideration. By integrating these parameters, general conclusions emerge that can help solve the aforementioned dilemma.\" publications.doi='10.1242/dev.001065' publications.firstAuthor='Duboule Denis' publications.id=1007895 publications.issue='14' publications.journal='Development' publications.month='Jul' publications.pages='2549-60' publications.pubMedId='17553908' publications.title='The rise and fall of Hox gene clusters.' publications.volume='134' publications.year=2007\n",
      "Gene: publications.abstractText='Although genes involved in common developmental programs are usually scattered throughout the metazoan genome, there are some important examples of functionally interconnected regulatory genes that display close physical linkage. In particular the homeotic genes, which determine the identities of body parts, are clustered in the Hox complexes and clustering is thought to be crucial for the proper execution of their developmental programs. Here we describe the organization and functional properties of a more recently identified cluster of six homeobox genes at 93DE on the third chromosome of Drosophila. These genes, which include tinman, bagpipe, ladybird early, ladybird late, C15, and slouch, all participate in mesodermal patterning and differentiation programs and show multiple regulatory interactions among each other. We propose that their clustering, through unknown mechanisms, is functionally significant and discuss the similarities and differences between the 93DE homeobox gene cluster and the Hox complexes.' publications.doi='10.1002/1521-1878(200102)23:2<125::AID-BIES1019>3.0.CO;2-C' publications.firstAuthor='Jagla K' publications.id=1007883 publications.issue='2' publications.journal='Bioessays' publications.month='Feb' publications.pages='125-33' publications.pubMedId='11169585' publications.title='A cluster of Drosophila homeobox genes involved in mesoderm differentiation programs.' publications.volume='23' publications.year=2001\n",
      "Gene: publications.abstractText='Although the genetics of dorsal-ventral polarity which leads to mesoderm formation in Drosophila are understood in considerable detail, subsequent molecular mechanisms involved in patterning the mesoderm primordium into individual mesodermal subtypes are poorly understood. Two papers published recently suggest strongly that an inductive signal from dorsal ectoderm is involved in subdividing the underlying mesoderm, and present evidence that one of the signalling factors is Decapentaplegic (Dpp), a member of the bone morphogenetic protein subgroup of the Transforming Growth Factor-beta (TGF-beta) super family of proteins.' publications.doi='10.1002/bies.950170904' publications.firstAuthor='Venkatesh T V' publications.id=1007725 publications.issue='9' publications.journal='Bioessays' publications.month='Sep' publications.pages='754-7' publications.pubMedId='8763827' publications.title='How many signals does it take?' publications.volume='17' publications.year=1995\n",
      "Gene: publications.abstractText='Although the mechanisms of anterior-posterior axis formation are well understood in Drosophila, both embryological and molecular studies suggest significant variation in the mechanisms generating this axis within the Insecta class as a whole.' publications.doi=None publications.firstAuthor='Patel N H' publications.id=1007793 publications.issue='9' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Apr' publications.pages='4442-4' publications.pubMedId='10781040' publications.title=\"It's a bug's life.\" publications.volume='97' publications.year=2000\n",
      "Gene: publications.abstractText='Amino acid encoding genes contain character state information that may be useful for phylogenetic analysis on at least two levels. The nucleotide sequence and the translated amino acid sequences have both been employed separately as character states for cladistic studies of various taxa, including studies of the genealogy of genes in multigene families. In essence, amino acid sequences and nucleic acid sequences are two different ways of character coding the information in a gene. Silent positions in the nucleotide sequence (first or third positions in codons that can accrue change without changing the identity of the amino acid that the triplet codes for) may accrue change relatively rapidly and become saturated, losing the pattern of historical divergence. On the other hand, non-silent nucleotide alterations and their accompanying amino acid changes may evolve too slowly to reveal relationships among closely related taxa. In general, the dynamics of sequence change in silent and non-silent positions in protein coding genes result in homoplasy and lack of resolution, respectively. We suggest that the combination of nucleic acid and the translated amino acid coded character states into the same data matrix for phylogenetic analysis addresses some of the problems caused by the rapid change of silent nucleotide positions and overall slow rate of change of non-silent nucleotide positions and slowly changing amino acid positions. One major theoretical problem with this approach is the apparent non-independence of the two sources of characters. However, there are at least three possible outcomes when comparing protein coding nucleic acid sequences with their translated amino acids in a phylogenetic context on a codon by codon basis. First, the two character sets for a codon may be entirely congruent with respect to the information they convey about the relationships of a certain set of taxa. Second, one character set may display no information concerning a phylogenetic hypothesis while the other character set may impact information to a hypothesis. These two possibilities are cases of non-independence, however, we argue that congruence in such cases can be thought of as increasing the weight of the particular phylogenetic hypothesis that is supported by those characters. In the third case, the two sources of character information for a particular codon may be entirely incongruent with respect to phylogenetic hypotheses concerning the taxa examined. In this last case the two character sets are independent in that information from neither can predict the character states of the other. Examples of these possibilities are discussed and the general applicability of combining these two sources of information for protein coding genes is presented using sequences from the homeobox region of 46 homeobox genes from Drosophila melanogaster to develop a hypothesis of genealogical relationship of these genes in this large multigene family.' publications.doi=None publications.firstAuthor='Agosti D' publications.id=1007873 publications.issue=None publications.journal='Cladistics' publications.month=None publications.pages='65-82' publications.pubMedId='11541749' publications.title='On combining protein sequences and nucleic acid sequences in phylogenetic analysis: the homeobox protein case.' publications.volume='12' publications.year=1996\n",
      "Gene: publications.abstractText='Among the maternally active genes of Drosophila, cactus is the only one whose loss of function mutations specifically produce ventralized embryos. Its product inhibits nuclear translocation of the dorsal morphogen in the dorsal region of the embryo. Here we report the cloning of cactus and the sequencing of its maternal transcript. The identity of our clones was verified by induction of phenocopies with antisense RNA and rescue of the mutant phenotype with sense RNA. cactus is predicted to encode an acidic, cytoplasmic protein with seven ankyrin repeats. The sequence has similarity to the I kappa B proteins that inhibit the vertebrate transcription factor NF-kappa B. In analogy to results obtained with I kappa B and NF-kappa B, bacterially expressed cactus protein can inhibit DNA binding of dorsal protein in vitro.' publications.doi=None publications.firstAuthor='Geisler R' publications.id=1007832 publications.issue='4' publications.journal='Cell' publications.month='Nov' publications.pages='613-21' publications.pubMedId='1423618' publications.title='cactus, a gene involved in dorsoventral pattern formation of Drosophila, is related to the I kappa B gene family of vertebrates.' publications.volume='71' publications.year=1992\n",
      "Gene: publications.abstractText='An extracellular serine protease cascade generates the ligand that activates the Toll signaling pathway to establish dorsoventral polarity in the Drosophila embryo. We show here that this cascade is regulated by a serpin-type serine protease inhibitor, which plays an essential role in confining Toll signaling to the ventral side of the embryo. This role is strikingly analogous to the function of the mammalian serpin antithrombin in localizing the blood-clotting cascade, suggesting that serpin inhibition of protease activity may be a general mechanism for achieving spatial control in diverse biological processes.' publications.doi=None publications.firstAuthor='Hashimoto Carl' publications.id=1007691 publications.issue='6' publications.journal='Dev. Cell' publications.month='Dec' publications.pages='945-50' publications.pubMedId='14667416' publications.title='Spatial regulation of developmental signaling by a serpin.' publications.volume='5' publications.year=2003\n",
      "Gene: publications.abstractText='An old idea about the relationship between arthropod and vertebrate body plans has been given new life by studies of the signalling genes controlling dorsal and ventral development in Drosophila and Xenopus.' publications.doi=None publications.firstAuthor='Jones C M' publications.id=1007783 publications.issue='6' publications.journal='Curr. Biol.' publications.month='Jun' publications.pages='574-6' publications.pubMedId='7552158' publications.title='Inductive signals. Revolving vertebrates.' publications.volume='5' publications.year=1995\n",
      "Gene: publications.abstractText='An oligonucleotide based on the cdc 10/SWI6 repeats of the Drosophila Notch protein was used to isolate other Drosophila genes with these repeats. One of these genes is the cactus locus, 1 of 11 genes required maternally for the establishment in embryos of a gradient of nuclear localization of dorsal protein, a rel-like transcription factor. Previous work showed that in cactus mutants more dorsal protein enters the nucleus in dorsal regions, resulting in a ventralized phenotype. It is now shown that the cactus locus produces two proteins that differ at their carboxy termini; both contain six cdc 10/SWI6 repeats that are sufficient for binding to dorsal and for inhibiting the ability of dorsal to bind DNA. The site on dorsal to which cactus binds was localized to the rel homology domain, where it overlaps with, or is adjacent to, the nuclear localization signal. In vivo the bulk of the cactus protein associated with dorsal is phosphorylated. This, or the association with dorsal, appears to stabilize the maternal cactus protein.' publications.doi=None publications.firstAuthor='Kidd S' publications.id=1007831 publications.issue='4' publications.journal='Cell' publications.month='Nov' publications.pages='623-35' publications.pubMedId='1423619' publications.title='Characterization of the Drosophila cactus locus and analysis of interactions between cactus and dorsal proteins.' publications.volume='71' publications.year=1992\n",
      "Gene: publications.abstractText='At least 13 genes control the establishment of dorsoventral polarity in the Drosophila embryo and more than 30 genes control the anteroposterior pattern of body segments. Each group of genes is thought to control pattern formation along one body axis, independently of the other group. We have used the expression of the fushi tarazu (ftz) segmentation gene as a positional marker to investigate the relationship between the dorsoventral and anteroposterior axes. The ftz gene is normally expressed in seven transverse stripes. Changes in the striped pattern in embryos mutant for other genes (or progeny of females homozygous for maternal-effect mutations) can reveal alterations of cell fate resulting from such mutations. We show that in the absence of any of ten maternal-effect dorsoventral polarity gene functions, the characteristic stripes of ftz protein are altered. Normally there is a difference between ftz stripe spacing on the dorsal and ventral sides of the embryo; in dorsalized mutant embryos the ftz stripes appear to be altered so that dorsal-type spacing occurs on all sides of the embryo. These results indicate that cells respond to dorsoventral positional information in establishing early patterns of gene expression along the anteroposterior axis and that there may be more significant interactions between the different axes of positional information than previously determined.' publications.doi=None publications.firstAuthor='Carroll S B' publications.id=1002632 publications.issue='3' publications.journal='Development' publications.month='Mar' publications.pages='327-32' publications.pubMedId='3653004' publications.title='Genes that control dorsoventral polarity affect gene expression along the anteroposterior axis of the Drosophila embryo.' publications.volume='99' publications.year=1987\n",
      "Gene: publications.abstractText='Axis formation is a key step in development, but studies indicate that genes involved in insect axis formation are relatively fast evolving. Orthodenticle genes have conserved roles, often with hunchback, in maternal anterior patterning in several insect species. We show that two orthodenticle genes, otd1 and otd2, and hunchback act as maternal anterior patterning genes in the honeybee (Apis mellifera) but, unlike other insects, act to pattern the majority of the anteroposterior axis. These genes regulate the expression domains of anterior, central and posterior gap genes and may directly regulate the anterior gap gene giant. We show otd1 and hunchback also influence dorsoventral patterning by regulating zerknült (zen) as they do in Tribolium, but that zen does not regulate the expression of honeybee gap genes. This suggests that interactions between anteroposterior and dorsal-ventral patterning are ancestral in holometabolous insects. Honeybee axis formation, and the function of the conserved anterior patterning gene orthodenticle, displays unique characters that indicate that, even when conserved genes pattern the axis, their regulatory interactions differ within orders of insects, consistent with relatively fast evolution in axis formation pathways.' publications.doi='10.1242/dev.067926' publications.firstAuthor='Wilson Megan J' publications.id=1007771 publications.issue='16' publications.journal='Development' publications.month='Aug' publications.pages='3497-507' publications.pubMedId='21771808' publications.title='Diversity in insect axis formation: two orthodenticle genes and hunchback act in anterior patterning and influence dorsoventral organization in the honeybee (Apis mellifera).' publications.volume='138' publications.year=2011\n",
      "Gene: publications.abstractText='Bioinformatics methods have identified enhancers that mediate restricted expression in the Drosophila embryo. However, only a small fraction of the predicted enhancers actually work when tested in vivo. In the present study, co-regulated neurogenic enhancers that are activated by intermediate levels of the Dorsal regulatory gradient are shown to contain several shared sequence motifs. These motifs permitted the identification of new neurogenic enhancers with high precision: five out of seven predicted enhancers direct restricted expression within ventral regions of the neurogenic ectoderm. Mutations in some of the shared motifs disrupt enhancer function, and evidence is presented that the Twist and Su(H) regulatory proteins are essential for the specification of the ventral neurogenic ectoderm prior to gastrulation. The regulatory model of neurogenic gene expression defined in this study permitted the identification of a neurogenic enhancer in the distant Anopheles genome. We discuss the prospects for deciphering regulatory codes that link primary DNA sequence information with predicted patterns of gene expression.' publications.doi='10.1242/dev.01124' publications.firstAuthor='Markstein Michele' publications.id=1430828 publications.issue='10' publications.journal='Development' publications.month='May' publications.pages='2387-94' publications.pubMedId='15128669' publications.title='A regulatory code for neurogenic gene expression in the Drosophila embryo.' publications.volume='131' publications.year=2004\n",
      "Gene: publications.abstractText='Bone morphogenetic protein (BMP) signalling regulates embryonic dorsal-ventral cell fate decisions in flies, frogs and fish. BMP activity is controlled by several secreted factors including the antagonists chordin and short gastrulation (SOG). Here we show that a second secreted protein, Twisted gastrulation (Tsg), enhances the antagonistic activity of Sog/chordin. In Drosophila, visualization of BMP signalling using anti-phospho-Smad staining shows that the tsg and sog loss-of-function phenotypes are very similar. In S2 cells and imaginal discs, TSG and SOG together make a more effective inhibitor of BMP signalling than either of them alone. Blocking Tsg function in zebrafish with morpholino oligonucleotides causes ventralization similar to that produced by chordin mutants. Co-injection of sub-inhibitory levels of morpholines directed against both Tsg and chordin synergistically enhances the penetrance of the ventralized phenotype. We show that Tsgs from different species are functionally equivalent, and conclude that Tsg is a conserved protein that functions with SOG/chordin to antagonize BMP signalling.' publications.doi='10.1038/35068578' publications.firstAuthor='Ross J J' publications.id=1007781 publications.issue='6827' publications.journal='Nature' publications.month='Mar' publications.pages='479-83' publications.pubMedId='11260716' publications.title='Twisted gastrulation is a conserved extracellular BMP antagonist.' publications.volume='410' publications.year=2001\n",
      "Gene: publications.abstractText='Bone morphogenetic proteins (BMP) direct dorsal-ventral patterning in both invertebrate and vertebrate embryos, with strong evolutionary conservation of molecular components of the pathway. Dorsal-ventral patterning of the early Drosophila embryo is a powerful experimental system to probe mechanisms of BMP gradient formation and interpretation. Recent studies have found that spatial patterns of activated BMP signal transducers in Drosophila go through an unexpected transition: a shallow gradient of weak responses at mid-cellularization changes to a step gradient of stronger responses in cellularized embryos. The transition between two gradients of different shape yields new insights into the progression of Drosophila dorsal-ventral patterning and raises new issues about the mechanisms ofgradient formation.' publications.doi='10.1016/j.tig.2003.10.009' publications.firstAuthor='Raftery Laurel A' publications.id=1007688 publications.issue='12' publications.journal='Trends Genet.' publications.month='Dec' publications.pages='701-8' publications.pubMedId='14642751' publications.title='Gradients and thresholds: BMP response gradients unveiled in Drosophila embryos.' publications.volume='19' publications.year=2003\n",
      "Gene: publications.abstractText=\"Brinker (Brk), a novel protein with features of a transcriptional repressor, regulates the graded response to Decapentaplegic (Dpp) in appendage primordia of Drosophila. Here, we show that in the embryo brk also has differential effects on Dpp target genes, depending on the level of Dpp activity required for their activation. Low-level target genes, like dpp itself, tolloid and early zerknüllt, show strong ectopic expression in ventrolateral regions of brk mutant embryos; intermediate-level target genes like pannier show weak ectopic expression, while high-level target genes like u-shaped and rhomboid are not affected. Ectopic target gene activation in the absence of brk is independent of Dpp, Tkv and Medea, indicating that Dpp signaling normally antagonizes brk's repression of these target genes. brk is expressed like short gastrulation (sog) in ventrolateral regions of the embryo abutting the dpp domain. Here, both brk and sog antagonize the antineurogenic activity of Dpp so that only in brk sog double mutants is the neuroectoderm completely deleted.\" publications.doi=None publications.firstAuthor='Jaźwińska A' publications.id=1007741 publications.issue='15' publications.journal='Development' publications.month='Aug' publications.pages='3323-34' publications.pubMedId='10393112' publications.title='The role of brinker in mediating the graded response to Dpp in early Drosophila embryos.' publications.volume='126' publications.year=1999\n",
      "Gene: publications.abstractText='By using a hsp70-Ubx fusion gene, we have ectopically expressed a Ubx product in the embryonic head primordia and studied the developmental effects on the larval head. We find that after high and persistent levels of Ubx product, the head is replaced by three (C1, C2 and C3) abdominal-like denticle belts. The C2 and C3 belts are the homeotic transformations of parasegments 1 and 2, respectively, while the C1 belt probably derives from the transformation and subsequent fusion of the most anterior procephalic primordia. On the basis of their response to the Ubx product and other arguments, we propose that the larval head is made of two genetically distinct components; one is the procephalon and the anterior region of the mandibular lobe, and the other is part of the parasegmental trunk and includes parasegments 1 and 2. Our results also indicate that most or all the larval head structures derive from precursor cells of ventral origin.' publications.doi=None publications.firstAuthor='González-Reyes A' publications.id=1007689 publications.issue='4' publications.journal='Development' publications.month='Dec' publications.pages='1459-71' publications.pubMedId='1687461' publications.title='Organization of the Drosophila head as revealed by the ectopic expression of the Ultrabithorax product.' publications.volume='113' publications.year=1991\n",
      "Gene: publications.abstractText='CBP and the related p300 protein are widely used transcriptional co-activators in metazoans that interact with multiple transcription factors. Whether CBP/p300 occupies the genome equally with all factors or preferentially binds together with some factors is not known. We therefore compared Drosophila melanogaster CBP (nejire) ChIP-seq peaks with regions bound by 40 different transcription factors in early embryos, and we found high co-occupancy with the Rel-family protein Dorsal. Dorsal is required for CBP occupancy in the embryo, but only at regions where few other factors are present. CBP peaks in mutant embryos lacking nuclear Dorsal are best correlated with TGF-ß/Dpp-signaling and Smad-protein binding. Differences in CBP occupancy in mutant embryos reflect gene expression changes genome-wide, but CBP also occupies some non-expressed genes. The presence of CBP at silent genes does not result in histone acetylation. We find that Polycomb-repressed H3K27me3 chromatin does not preclude CBP binding, but restricts histone acetylation at CBP-bound genomic sites. We conclude that CBP occupancy in Drosophila embryos preferentially overlaps factors controlling dorso-ventral patterning and that CBP binds silent genes without causing histone hyperacetylation.' publications.doi='10.1371/journal.pgen.1002769' publications.firstAuthor='Holmqvist Per-Henrik' publications.id=1007693 publications.issue='6' publications.journal='PLoS Genet.' publications.month=None publications.pages='e1002769' publications.pubMedId='22737084' publications.title='Preferential genome targeting of the CBP co-activator by Rel and Smad proteins in early Drosophila melanogaster embryos.' publications.volume='8' publications.year=2012\n",
      "Gene: publications.abstractText='CDP/Cux/Cut proteins are an evolutionarily conserved family of proteins containing several DNA binding domains: one Cut homeodomain and one, two or three Cut repeats. In Drosophila melanogaster, genetic studies indicated that Cut functions as a determinant of cell-type specification in several tissues, notably in the peripheral nervous system, the wing margin and the Malpighian tubule. Moreover, Cut was found to be a target and an effector of the Notch signaling pathway. In vertebrates, the same functions appear to be fulfilled by two cut-related genes with distinct patterns of expression. Cloning of the cDNA for the CCAAT-displacement protein (CDP) revealed that it was the human homologue of Drosophila Cut. CDP was later found be the DNA binding protein of the previously characterized histone nuclear factor D (HiNF-D). CDP and its mouse counterpart, Cux, were also reported to interact with regulatory elements from a large number of genes, including matrix attachment regions (MARs). CDP/Cut proteins were found generally to function as transcriptional repressors, although a participation in transcriptional activation is suggested by some data. Repression by CDP/Cut involves competition for binding site occupancy and active repression via the recruitment of a histone deacetylase activity. Various combinations of Cut repeats and the Cut homeodomains can generate distinct DNA binding activities. These activities are elevated in proliferating cells and decrease during terminal differentiation. One activity, involving the Cut homeodomain, is upregulated in S phase. CDP/Cut function is regulated by several post-translational modification events including phosphorylation, dephosphorylation, and acetylation. The CUTL1 gene in human was mapped to 7q22, a chromosomal region that is frequently rearranged in various cancers.' publications.doi=None publications.firstAuthor='Nepveu A' publications.id=1000618 publications.issue='1-2' publications.journal='Gene' publications.month='May' publications.pages='1-15' publications.pubMedId='11403998' publications.title='Role of the multifunctional CDP/Cut/Cux homeodomain transcription factor in regulating differentiation, cell growth and development.' publications.volume='270' publications.year=2001\n",
      "Gene: publications.abstractText='Cascades of zygotic gene expression pattern the anterior-posterior (AP) and dorsal-ventral (DV) axes of the early Drosophila embryo. Here, we used the global run-on sequencing assay (GRO-seq) to map the genome-wide RNA polymerase distribution during early Drosophila embryogenesis, thus providing insights into how genes are regulated. We identify widespread promoter-proximal pausing yet show that the presence of paused polymerase does not necessarily equate to direct regulation through pause release to productive elongation. Our data reveal that a subset of early Zelda-activated genes is regulated at the level of polymerase recruitment, whereas other Zelda target and axis patterning genes are predominantly regulated through pause release. In contrast to other signaling pathways, we found that bone morphogenetic protein (BMP) target genes are collectively more highly paused than BMP pathway components and show that BMP target gene expression requires the pause-inducing negative elongation factor (NELF) complex. Our data also suggest that polymerase pausing allows plasticity in gene activation throughout embryogenesis, as transiently repressed and transcriptionally silenced genes maintain and lose promoter polymerases, respectively. Finally, we provide evidence that the major effect of pausing is on the levels, rather than timing, of transcription. These data are discussed in terms of the efficiency of transcriptional activation required across cell populations during developmental time constraints.' publications.doi='10.1101/gad.215459.113' publications.firstAuthor='Saunders Abbie' publications.id=1001000 publications.issue='10' publications.journal='Genes Dev.' publications.month='May' publications.pages='1146-58' publications.pubMedId='23699410' publications.title='Extensive polymerase pausing during Drosophila axis patterning enables high-level and pliable transcription.' publications.volume='27' publications.year=2013\n",
      "Gene: publications.abstractText='Cell fates in the anterior and posterior termini of the Drosophila embryo are programmed by multiple zygotic genes that are regulated in response to a maternally encoded signal transduction pathway. These genes specify terminal as distinct from central cell fates, program pattern along the anteroposterior and dorsoventral axes of the termini, and also control endoderm specification and terminal morphogenetic movements. Here, we use a genetic interaction test to dissect the zygotic components of the terminal genetic hierarchy. We show that two genes, lines and empty spiracles, act downstream of tailless to repress central and promote terminal cell fates along the anteroposterior axis of the termini. Genes that control dorsoventral pattern in the termini and genes that program terminal morphogenesis act in distinct branches of the genetic hierarchy that are independent of tailless.' publications.doi=None publications.firstAuthor='Strecker T R' publications.id=1007685 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Apr' publications.pages='422-6' publications.pubMedId='1551484' publications.title='Genetic control of cell fate in the termini of the Drosophila embryo.' publications.volume='150' publications.year=1992\n",
      "Gene: publications.abstractText='Charles Darwin has proposed the theory that evolution of live organisms is based on random variation and natural selection. Jacques Monod in his classic book Chance and Necessity, published 40 years ago, presented his thesis \"that the biosphere does not contain a predictable class of objects or events, but constitutes a particular occurrence, compatible indeed with the first principles, but not deducible from those principals and therefore, essentially unpredictable.\"Recent discoveries in eye evolution are in agreement with both of these theses. They confirm Darwin\\'s assumption of a simple eye prototype and lend strong support for the notion of a monophyletic origin of the various eye types. Considering the complexity of the underlying gene regulatory networks the unpredictability is obvious. The evolution of the Hox gene cluster and the specification of the body plan starting from an evolutionary prototype segment is discussed. In the course of evolution, a series of similar prototypic segments gradually undergoes cephalization anteriorly and caudalization posteriorly through diversification of the Hox genes.' publications.doi='10.1093/gbe/evr061' publications.firstAuthor='Gehring Walter J' publications.id=1007863 publications.issue=None publications.journal='Genome Biol Evol' publications.month=None publications.pages='1053-66' publications.pubMedId='21979158' publications.title='Chance and necessity in eye evolution.' publications.volume='3' publications.year=2011\n",
      "Gene: publications.abstractText='Chromatin domain insulators are thought to insulate adjacent genes, including their regulatory elements, from each other by organizing chromatin into functionally independent domains. Thus insulators should play a global role in gene regulation by keeping regulatory domains separated. However, this has never been demonstrated. We previously designed and characterized a transgene that is under GAL4 UAS control and encodes a dominant-negative form of the Boundary Element-Associated Factors BEAF-32A and BEAF-32B. The BID transgene encodes the BEAF self-interaction domain but lacks a DNA binding domain. Expression of BID in eye imaginal discs leads to a rough eye phenotype. Here we screen for dominant mutations that modify this eye phenotype. This assay provides evidence for cross-talk between different classes of insulators, and for a broad role of the BEAF proteins in maintaining patterns of gene expression during eye development. Most identified genes encode other insulator binding proteins, transcription factors involved in head development, or general transcription factors. Because it is unlikely that insulator function is limited to eye development, the present results support the hypothesis that insulators play a widespread role in maintaining global transcription programs.' publications.doi='10.1007/s00438-006-0187-8' publications.firstAuthor='Roy Swarnava' publications.id=1005516 publications.issue='3' publications.journal='Mol. Genet. Genomics' publications.month='Mar' publications.pages='273-86' publications.pubMedId='17143631' publications.title='A genetic screen supports a broad role for the Drosophila insulator proteins BEAF-32A and BEAF-32B in maintaining patterns of gene expression.' publications.volume='277' publications.year=2007\n",
      "Gene: publications.abstractText=\"Circulating carbohydrates are an essential energy source, perturbations in which are pathognomonic of various diseases, diabetes being the most prevalent. Yet many of the genes underlying diabetes and its characteristic hyperglycaemia remain elusive. Here we use physiological and genetic interrogations in D. melanogaster to uncover the 'glucome', the complete set of genes involved in glucose regulation in flies. Partial genomic screens of ∼1,000 genes yield ∼160 hyperglycaemia 'flyabetes' candidates that we classify using fat body- and muscle-specific knockdown and biochemical assays. The results highlight the minor glucose fraction as a physiological indicator of metabolism in Drosophila. The hits uncovered in our screen may have conserved functions in mammalian glucose homeostasis, as heterozygous and homozygous mutants of Ck1alpha in the murine adipose lineage, develop diabetes. Our findings demonstrate that glucose has a role in fly biology and that genetic screenings carried out in flies may increase our understanding of mammalian pathophysiology.\" publications.doi='10.1038/ncomms8102' publications.firstAuthor='Ugrankar Rupali' publications.id=1001814 publications.issue=None publications.journal='Nat Commun' publications.month=None publications.pages='7102' publications.pubMedId='25994086' publications.title='Drosophila glucome screening identifies Ck1alpha as a regulator of mammalian glucose metabolism.' publications.volume='6' publications.year=2015\n",
      "Gene: publications.abstractText='Cis-regulatory DNAs control the timing and sites of gene expression during metazoan development. Changes in gene expression are responsible for the morphological diversification of metazoan body plans. However, traditional methods for the identification and characterization of cis-regulatory DNAs are tedious. During the past year, computational methods have been used to identify novel cis-DNAs within the entire Drosophila genome. These methods change the way that cis-DNAs will be analyzed in future studies and offer the promise of unraveling complex gene networks.' publications.doi=None publications.firstAuthor='Markstein Michele' publications.id=1007887 publications.issue='5' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Oct' publications.pages='601-6' publications.pubMedId='12200166' publications.title='Decoding cis-regulatory DNAs in the Drosophila genome.' publications.volume='12' publications.year=2002\n",
      "Gene: publications.abstractText='Cis-regulatory modules (CRMs) function by binding sequence specific transcription factors, but the relationship between in vivo physical binding and the regulatory capacity of factor-bound DNA elements remains uncertain. We investigate this relationship for the well-studied Twist factor in Drosophila melanogaster embryos by analyzing genome-wide factor occupancy and testing the functional significance of Twist occupied regions and motifs within regions. Twist ChIP-seq data efficiently identified previously studied Twist-dependent CRMs and robustly predicted new CRM activity in transgenesis, with newly identified Twist-occupied regions supporting diverse spatiotemporal patterns (>74% positive, n = 31). Some, but not all, candidate CRMs require Twist for proper expression in the embryo. The Twist motifs most favored in genome ChIP data (in vivo) differed from those most favored by Systematic Evolution of Ligands by EXponential enrichment (SELEX) (in vitro). Furthermore, the majority of ChIP-seq signals could be parsimoniously explained by a CABVTG motif located within 50 bp of the ChIP summit and, of these, CACATG was most prevalent. Mutagenesis experiments demonstrated that different Twist E-box motif types are not fully interchangeable, suggesting that the ChIP-derived consensus (CABVTG) includes sites having distinct regulatory outputs. Further analysis of position, frequency of occurrence, and sequence conservation revealed significant enrichment and conservation of CABVTG E-box motifs near Twist ChIP-seq signal summits, preferential conservation of ±150 bp surrounding Twist occupied summits, and enrichment of GA- and CA-repeat sequences near Twist occupied summits. Our results show that high resolution in vivo occupancy data can be used to drive efficient discovery and dissection of global and local cis-regulatory logic.' publications.doi='10.1101/gr.104018.109' publications.firstAuthor='Ozdemir Anil' publications.id=1007715 publications.issue='4' publications.journal='Genome Res.' publications.month='Apr' publications.pages='566-77' publications.pubMedId='21383317' publications.title='High resolution mapping of Twist to DNA in Drosophila embryos: Efficient functional analysis and evolutionary conservation.' publications.volume='21' publications.year=2011\n",
      "Gene: publications.abstractText='Cis-regulatory modules that drive precise spatial-temporal patterns of gene expression are central to the process of metazoan development. We describea new computational strategy to annotate genomic sequences based on their \"pattern generating potential\"and to produce quantitative descriptions of transcriptional regulatory networks at the level of individual protein-module interactions. We use this approach to convert the qualitative understanding of interactions that regulate Drosophila segmentation into a network model in which a confidence value is associated with each transcription factor-module interaction. Sequence information from multiple Drosophila species is integrated with transcription factor binding specificities to determine conserved binding site frequencies across the genome. These binding site profiles are combined with transcription factor expression information to create a model to predict module activity patterns. This model is used to scan genomic sequences for the potential to generate all or part of the expression pattern of a nearby gene, obtained from available gene expression databases. Interactions between individual transcription factors and modules are inferred by a statistical method to quantify a factor\\'s contribution to the module\\'s pattern generating potential. We use these pattern generating potentials to systematically describe the location and function of known and novel cis-regulatory modules in the segmentation network, identifying many examples of modules predicted to have overlapping expression activities. Surprisingly, conserved transcription factor binding site frequencies were as effective as experimental measurements of occupancy in predicting module expression patterns or factor-module interactions. Thus, unlike previous module prediction methods, this method predicts not only the location of modules but also their spatial activity pattern and the factors that directly determine this pattern. As databases of transcription factor specificities and in vivo gene expression patterns grow, analysis of pattern generating potentials provides a general method to decode transcriptional regulatory sequences and networks.' publications.doi='10.1371/journal.pbio.1000456' publications.firstAuthor='Kazemian Majid' publications.id=1007756 publications.issue='8' publications.journal='PLoS Biol.' publications.month=None publications.pages=None publications.pubMedId='20808951' publications.title='Quantitative analysis of the Drosophila segmentation regulatory network using pattern generating potentials.' publications.volume='8' publications.year=2010\n",
      "Gene: publications.abstractText='Codon usage bias, the preferential use of particular codons within each codon family, is characteristic of synonymous base composition in many species, including Drosophila, yeast, and many bacteria. Preferential usage of particular codons in these species is maintained by natural selection acting largely at the level of translation. In Drosophila, as in bacteria, the rate of synonymous substitution per site is negatively correlated with the degree of codon usage bias, indicating stronger selection on codon usage in genes with high codon bias than in genes with low codon bias. Surprisingly, in these organisms, as well as in mammals, the rate of synonymous substitution is also positively correlated with the rate of nonsynonymous substitution. To investigate this correlation, we carried out a phylogenetic analysis of substitutions in 22 genes between two species of Drosophila, Drosophila pseudoobscura and D. subobscura, in codons that differ by one replacement and one synonymous change. We provide evidence for a relative excess of double substitutions in the same species lineage that cannot be explained by the simultaneous mutation of two adjacent bases. The synonymous changes in these codons also cannot be explained by a shift to a more preferred codon following a replacement substitution. We, therefore, interpret the excess of double codon substitutions within a lineage as being the result of relaxed constraints on both kinds of substitutions in particular codons.' publications.doi=None publications.firstAuthor='Comeron J M' publications.id=1007809 publications.issue='2' publications.journal='Genetics' publications.month='Oct' publications.pages='767-75' publications.pubMedId='9755207' publications.title='The correlation between synonymous and nonsynonymous substitutions in Drosophila: mutation, selection or relaxed constraints?' publications.volume='150' publications.year=1998\n",
      "Gene: publications.abstractText='Comparative genomic approaches to gene and cis-regulatory prediction are based on the principle that differential DNA sequence conservation reflects variation in functional constraint. Using this principle, we analyze noncoding sequence conservation in Drosophila for 40 loci with known or suspected cis-regulatory function encompassing >100 kb of DNA. We estimate the fraction of noncoding DNA conserved in both intergenic and intronic regions and describe the length distribution of ungapped conserved noncoding blocks. On average, 22%-26% of noncoding sequences surveyed are conserved in Drosophila, with median block length approximately 19 bp. We show that point substitution in conserved noncoding blocks exhibits transition bias as well as lineage effects in base composition, and occurs more than an order of magnitude more frequently than insertion/deletion (indel) substitution. Overall, patterns of noncoding DNA structure and evolution differ remarkably little between intergenic and intronic conserved blocks, suggesting that the effects of transcription per se contribute minimally to the constraints operating on these sequences. The results of this study have implications for the development of alignment and prediction algorithms specific to noncoding DNA, as well as for models of cis-regulatory DNA sequence evolution.' publications.doi='10.1101/gr.178701' publications.firstAuthor='Bergman C M' publications.id=1007849 publications.issue='8' publications.journal='Genome Res.' publications.month='Aug' publications.pages='1335-45' publications.pubMedId='11483574' publications.title='Analysis of conserved noncoding DNA in Drosophila reveals similar constraints in intergenic and intronic sequences.' publications.volume='11' publications.year=2001\n",
      "Gene: publications.abstractText='Comparisons between Hox genes in different arthropods suggest that the diversity of Antennapedia-class homeotic genes present in modern insects had already arisen before the divergence of insects and crustaceans, probably during the Cambrian. Hox gene duplications are therefore unlikely to have occurred concomitantly with trunk segment diversification in the lineage leading to insects. Available data suggest that domains of homeotic gene expression are also generally conserved among insects, but changes in Hox gene regulation may have played a significant role in segment diversification. Differences that have been documented alter specific aspects of Hox gene regulation within segments and correlate with alterations in segment morphology rather than overt homeotic transformations. The Drosophila Hox cluster contains several homeobox genes that are not homeotic genes--bicoid, fushi-tarazu and zen. the role of these genes during early development has been studied in some detail. It appears to be without parallel among the vertebrate Hox genes. No well conserved homologues of these genes have been found in other taxa, suggesting that they are evolving faster than the homeotic genes. Relatively divergent Antp-class genes isolated from other insects are probably homologues of fushi-tarazu, but these are almost unrecognisable outside of their homeodomains, and have accumulated approximately 10 times as many changes in their homeodomains as have homeotic genes in the same comparisons. They show conserved patterns of expression in the nervous system, but not during early development.' publications.doi=None publications.firstAuthor='Akam M' publications.id=1007898 publications.issue=None publications.journal='Dev. Suppl.' publications.month=None publications.pages='209-15' publications.pubMedId='7579521' publications.title='The evolving role of Hox genes in arthropods.' publications.volume=None publications.year=1994\n",
      "Gene: publications.abstractText='Computational methods attempting to identify instances of cis-regulatory modules (CRMs) in the genome face a challenging problem of searching for potentially interacting transcription factor binding sites while knowledge of the specific interactions involved remains limited. Without a comprehensive comparison of their performance, the reliability and accuracy of these tools remains unclear. Faced with a large number of different tools that address this problem, we summarized and categorized them based on search strategy and input data requirements. Twelve representative methods were chosen and applied to predict CRMs from the Drosophila CRM database REDfly, and across the human ENCODE regions. Our results show that the optimal choice of method varies depending on species and composition of the sequences in question. When discriminating CRMs from non-coding regions, those methods considering evolutionary conservation have a stronger predictive power than methods designed to be run on a single genome. Different CRM representations and search strategies rely on different CRM properties, and different methods can complement one another. For example, some favour homotypical clusters of binding sites, while others perform best on short CRMs. Furthermore, most methods appear to be sensitive to the composition and structure of the genome to which they are applied. We analyze the principal features that distinguish the methods that performed well, identify weaknesses leading to poor performance, and provide a guide for users. We also propose key considerations for the development and evaluation of future CRM-prediction methods.' publications.doi='10.1371/journal.pcbi.1001020' publications.firstAuthor='Su Jing' publications.id=1007788 publications.issue='12' publications.journal='PLoS Comput. Biol.' publications.month=None publications.pages='e1001020' publications.pubMedId='21152003' publications.title='Assessing computational methods of cis-regulatory module prediction.' publications.volume='6' publications.year=2010\n",
      "Gene: publications.abstractText=\"Current models describe male-specific fruitless (fruM) as a genetic 'switch' regulating sexual behavior in Drosophila melanogaster, and they postulate that female (F) and male (M) doublesex (dsx) products control body sexual morphology. In contradiction to this simple model, we show that dsx, as well as fruM and non-sex-specific retained (retn), affect both male and female sexual behaviors. In females, both retn and dsxF contribute to female receptivity, and both genes act to repress male-like courtship activity in the presence or absence of fruM. In males, consistent with the opposing functions of dsxM and dsxF, dsxM acts as a positive factor for male courtship. retn also acts counter to fruM in the development of the male-specific muscle of Lawrence. Molecularly, retn seems to regulate sexual behavior via a previously described complex that represses zerknullt. Thus, we show that fru and dsx together act as a 'switch' system regulating behavior in the context of other developmental genes, such as retn.\" publications.doi='10.1038/ng1908' publications.firstAuthor='Shirangi Troy R' publications.id=1007853 publications.issue='12' publications.journal='Nat. Genet.' publications.month='Dec' publications.pages='1435-9' publications.pubMedId='17086183' publications.title='A double-switch system regulates male courtship behavior in male and female Drosophila melanogaster.' publications.volume='38' publications.year=2006\n",
      "Gene: publications.abstractText='DREF, a transcription regulatory factor which specifically binds to the promoter-activating element DRE (DNA replication-related element) of DNA replication-related genes, was purified to homogeneity from nuclear extracts of Drosophila Kc cells. cDNA for DREF was isolated with the reverse-transcriptase polymerase chain reaction method using primers synthesized on the basis of partial amino acid sequences and following screening of cDNA libraries. Deduced from the nucleotide sequences of cDNA, DREF is a polypeptide of 701 amino acid residues with a molecular weight of 80,096, which contains three characteristic regions, rich in basic amino acids, proline, and acidic amino acids, respectively. Deletion analysis of bacterially expressed DREF fused with glutathione S-transferase (GST-DREF) indicated that a part of the N-terminal basic amino acid region (16-115 amino acids) is responsible for the specific binding to DRE. A polyclonal and four monoclonal antibodies were raised against the GST-DREF fusion protein. The antibodies inhibited specifically the transcription of DNA polymerase alpha promoter in vitro. Cotransfection experiments using Kc cells demonstrated that overproduction of DREF protein overcomes the repression of the proliferating cell nuclear antigen gene promoter by the zerknüllt gene product. These results confirmed that DREF is a trans-activating factor for DNA replication-related genes. Immunocytochemical analysis demonstrated the presence of DREF polypeptide in nuclei after the eighth nuclear division cycle, suggesting that nuclear accumulation of DREF is important for the coordinate zygotic expression of DNA replication-related genes carrying DRE sequences.' publications.doi=None publications.firstAuthor='Hirose F' publications.id=1007827 publications.issue='7' publications.journal='J. Biol. Chem.' publications.month='Feb' publications.pages='3930-7' publications.pubMedId='8632015' publications.title='Isolation and characterization of cDNA for DREF, a promoter-activating factor for Drosophila DNA replication-related genes.' publications.volume='271' publications.year=1996\n",
      "Gene: publications.abstractText=\"Decapentaplegic (Dpp), the fly homolog of the secreted mammalian BMP2/4 signaling molecules, is involved in almost all aspects of fly development. Dpp has critical functions at all developmental stages, from patterning of the eggshell to the determination of adult intestinal stem cell identity. Here, we focus on recent findings regarding the transcriptional regulatory logic of the pathway, on a new feedback regulator, Pentagone, and on Dpp's roles in scaling and growth of the Drosophila wing.\" publications.doi='10.1016/j.semcdb.2014.04.036' publications.firstAuthor='Hamaratoglu Fisun' publications.id=1007722 publications.issue=None publications.journal='Semin. Cell Dev. Biol.' publications.month='Aug' publications.pages='128-36' publications.pubMedId='24813173' publications.title='Dpp/BMP signaling in flies: from molecules to biology.' publications.volume='32' publications.year=2014\n",
      "Gene: publications.abstractText='Developing tissues are patterned by coordinated activities of signaling systems, which can be integrated by a regulatory region of a gene that binds multiple transcription factors or by a transcription factor that is modified by multiple enzymes. Based on a combination of genetic and imaging experiments in the early Drosophila embryo, we describe a signal integration mechanism that cannot be reduced to a single gene regulatory element or a single transcription factor. This mechanism relies on an enzymatic network formed by mitogen-activated protein kinase (MAPK) and its substrates. Specifically, anteriorly localized MAPK substrates, such as Bicoid, antagonize MAPK-dependent downregulation of Capicua, a repressor that is involved in gene regulation along the dorsoventral axis of the embryo. MAPK substrate competition provides a basis for ternary interaction of the anterior, dorsoventral, and terminal patterning systems. A mathematical model of this interaction can explain gene expression patterns with both anteroposterior and dorsoventral polarities.' publications.doi='10.1016/j.devcel.2011.05.009' publications.firstAuthor='Kim Yoosik' publications.id=1007720 publications.issue='6' publications.journal='Dev. Cell' publications.month='Jun' publications.pages='880-7' publications.pubMedId='21664584' publications.title='Gene regulation by MAPK substrate competition.' publications.volume='20' publications.year=2011\n",
      "Gene: publications.abstractText='Differentiation of distinct cell types at specific locations within a developing organism depends largely on the ability of cells to communicate. A major class of signalling proteins implicated in cell to cell communication is represented by members of the TGF beta superfamily. A corresponding class of transmembrane serine/threonine kinases has recently been discovered that act as cell surface receptors for ligands of the TGF beta superfamily. The product of the Drosophila gene decapentaplegic (dpp) encodes a TGF beta homolog that plays multiple roles during embryogenesis and the development of imaginal discs. Here we describe the complex expression pattern of thick veins (tkv), which encodes a receptor for dpp. We make use of tkv loss-of-function mutations to examine the consequences of the failure of embryonic cells to respond to dpp and/or other TGF beta homologs. We find that while maternal tkv product allows largely normal dorsoventral pattering of the embryo, zygotic tkv activity is indispensable for dorsal closure of the embryo after germ band retraction. Furthermore, tkv activity is crucial for patterning the visceral mesoderm; in the absence of functional tkv gene product, visceral mesoderm parasegment 7 cells fail to express Ultrabithorax, but instead accumulate Antennapedia protein. The tkv receptor is therefore involved in delimiting the expression domains of homeotic genes in the visceral mesoderm. Interestingly, tkv mutants fail to establish a proper tracheal network. Tracheal braches formed by cells migrating in dorsal or ventral directions are absent in tkv mutants. The requirements for tkv in dorsal closure, visceral mesoderm and trachea development assign novel functions to dpp or a closely related member of the TGF beta superfamily.' publications.doi=None publications.firstAuthor='Affolter M' publications.id=1007738 publications.issue='11' publications.journal='Development' publications.month='Nov' publications.pages='3105-17' publications.pubMedId='7720555' publications.title='Multiple requirements for the receptor serine/threonine kinase thick veins reveal novel functions of TGF beta homologs during Drosophila embryogenesis.' publications.volume='120' publications.year=1994\n",
      "Gene: publications.abstractText='Differentiation of the embryonic termini in Drosophila depends on signaling by the Tor RTK, which induces terminal gene expression by inactivating at the embryonic poles a uniformly distributed repressor activity that involves the Gro corepressor. Here, we identify a new gene, cic, that acts as a repressor of terminal genes regulated by the Tor pathway. cic also mediates repression along the dorsoventral axis, a process that requires the Dorsal morphogen and Gro, and which is also inhibited by Tor signaling at the termini. cic encodes an HMG-box transcription factor that interacts with Gro in vitro. We present evidence that Tor signaling regulates terminal patterning by inactivating Cic at the embryo poles. cic has been evolutionarily conserved, suggesting that Cic-like proteins may act as repressors regulated by RTK signaling in other organisms.' publications.doi=None publications.firstAuthor='Jiménez G' publications.id=1007737 publications.issue='2' publications.journal='Genes Dev.' publications.month='Jan' publications.pages='224-31' publications.pubMedId='10652276' publications.title='Relief of gene repression by torso RTK signaling: role of capicua in Drosophila terminal and dorsoventral patterning.' publications.volume='14' publications.year=2000\n",
      "Gene: publications.abstractText='Divergent morphology of species has largely been ascribed to genetic differences in the tissue-specific expression of proteins, which could be achieved by divergence in cis-regulatory elements or by altering the binding specificity of transcription factors (TFs). The relative importance of the latter has been difficult to assess, as previous systematic analyses of TF binding specificity have been performed using different methods in different species. To address this, we determined the binding specificities of 242 Drosophila TFs, and compared them to human and mouse data. This analysis revealed that TF binding specificities are highly conserved between Drosophila and mammals, and that for orthologous TFs, the similarity extends even to the level of very subtle dinucleotide binding preferences. The few human TFs with divergent specificities function in cell types not found in fruit flies, suggesting that evolution of TF specificities contributes to emergence of novel types of differentiated cells.' publications.doi='10.7554/eLife.04837' publications.firstAuthor='Nitta Kazuhiro R' publications.id=1000114 publications.issue=None publications.journal='Elife' publications.month=None publications.pages=None publications.pubMedId='25779349' publications.title='Conservation of transcription factor binding specificities across 600 million years of bilateria evolution.' publications.volume='4' publications.year=2015\n",
      "Gene: publications.abstractText=\"Diverse types of epithelial morphogenesis drive development. Similar cytoskeletal and cell adhesion machinery orchestrate these changes, but it is unclear how distinct tissue types are produced. Thus, it is important to define and compare different types of morphogenesis. We investigated cell flattening and elongation in the amnioserosa, a squamous epithelium formed at Drosophila gastrulation. Amnioserosa cells are initially columnar. Remarkably, they flatten and elongate autonomously by perpendicularly rotating the microtubule cytoskeleton--we call this 'rotary cell elongation'. Apical microtubule protrusion appears to initiate the rotation and microtubule inhibition perturbs the process. F-actin restrains and helps orient the microtubule protrusions. As amnioserosa cells elongate, they maintain their original cell-cell contacts and develop planar polarity. Myosin II localizes to anterior-posterior contacts, while the polarity protein Bazooka (PAR-3) localizes to dorsoventral contacts. Genetic analysis revealed that Myosin II and Bazooka cooperate to properly position adherens junctions. These results identify a specific cellular mechanism of squamous tissue morphogenesis and molecular interactions involved.\" publications.doi='10.1242/dev.019802' publications.firstAuthor='Pope Karen L' publications.id=1007798 publications.issue='13' publications.journal='Development' publications.month='Jul' publications.pages='2227-38' publications.pubMedId='18508861' publications.title='Control of cell flattening and junctional remodeling during squamous epithelial morphogenesis in Drosophila.' publications.volume='135' publications.year=2008\n",
      "Gene: publications.abstractText='Dorsal functions as both an activator and repressor of transcription to determine dorsoventral fate in the Drosophila melanogaster embryo. Repression by Dorsal requires the corepressor Groucho (Gro) and is mediated by silencers termed ventral repression regions (VRRs). A VRR in zerknüllt (zen) contains Dorsal binding sites as well as an essential element termed AT2. We have identified and purified an AT2 DNA binding activity in embryos and shown it to consist of cut (ct) and dead ringer (dri) gene products. Studies of loss-of-function mutations in ct and dri demonstrate that both genes are required for the activity of the AT2 site. Dorsal and Dri both bind Gro, acting cooperatively to recruit it to the DNA. Thus, ventral repression may require the formation of a multiprotein complex at the VRR. This complex includes Dorsal, Gro, and additional DNA binding proteins, which appear to convert Dorsal from an activator to a repressor by enabling it to recruit Gro to the template. By showing how binding site context can dramatically alter transcription factor function, these findings help clarify the mechanisms responsible for the regulatory specificity of transcription factors.' publications.doi=None publications.firstAuthor='Valentine S A' publications.id=1007782 publications.issue='11' publications.journal='Mol. Cell. Biol.' publications.month='Nov' publications.pages='6584-94' publications.pubMedId='9774673' publications.title='Dorsal-mediated repression requires the formation of a multiprotein repression complex at the ventral silencer.' publications.volume='18' publications.year=1998\n",
      "Gene: publications.abstractText='Dorsal-ventral patterning in vertebrate and invertebrate embryos is mediated by a conserved system of secreted proteins that establishes a bone morphogenetic protein (BMP) gradient. Although the Drosophila embryonic Decapentaplegic (Dpp) gradient has served as a model to understand how morphogen gradients are established, no role for the extracellular matrix has been previously described. Here we show that type IV collagen extracellular matrix proteins bind Dpp and regulate its signalling in both the Drosophila embryo and ovary. We provide evidence that the interaction between Dpp and type IV collagen augments Dpp signalling in the embryo by promoting gradient formation, yet it restricts the signalling range in the ovary through sequestration of the Dpp ligand. Together, these results identify a critical function of type IV collagens in modulating Dpp in the extracellular space during Drosophila development. On the basis of our findings that human type IV collagen binds BMP4, we predict that this role of type IV collagens will be conserved.' publications.doi='10.1038/nature07214' publications.firstAuthor='Wang Xiaomeng' publications.id=1007879 publications.issue='7209' publications.journal='Nature' publications.month='Sep' publications.pages='72-7' publications.pubMedId='18701888' publications.title='Type IV collagens regulate BMP signalling in Drosophila.' publications.volume='455' publications.year=2008\n",
      "Gene: publications.abstractText=\"Dorsoventral (DV) patterning of the Drosophila embryo is initiated by a broad Dorsal (Dl) nuclear gradient, which is regulated by a conserved signaling pathway that includes the Toll receptor and Pelle kinase. We investigate the consequences of expressing a constitutively activated form of the Toll receptor, Toll(10b), in anterior regions of the early embryo using the bicoid 3' UTR. Localized Toll(10b) products result in the formation of an ectopic, anteroposterior (AP) Dl nuclear gradient along the length of the embryo. The analysis of both authentic dorsal target genes and defined synthetic promoters suggeststhat the ectopic gradient is sufficient to generate the full repertory of DV patterning responses along the AP axis of the embryo. For example, mesoderm determinants are activated in the anterior third of the embryo, whereas neurogenic genes are expressed in central regions. These results raise the possibility that Toll signaling components diffuse in the plasma membrane or syncytial cytoplasm of the early embryo. This study also provides evidence that neurogenic repressors may be important for the establishment of the sharp mesoderm/neuroectoderm boundary in the early embryo.\" publications.doi=None publications.firstAuthor='Huang A M' publications.id=1002676 publications.issue='15' publications.journal='Genes Dev.' publications.month='Aug' publications.pages='1963-73' publications.pubMedId='9271119' publications.title='An anteroposterior Dorsal gradient in the Drosophila embryo.' publications.volume='11' publications.year=1997\n",
      "Gene: publications.abstractText='Dorsoventral patterning in Drosophila is initiated by the maternal regulatory factor dorsal (dl), which is a member of the Rel family of transcription factors. dl functions as a transcriptional activator and repressor to establish different territories of gene expression in the precellular embryo. Differential regulation of dl target genes may be essential for subdividing each tissue territory (the presumptive mesoderm, neuroectoderm, and dorsal ectoderm) into multiple cell types in older embryos. Different patterns of snail (sna) and decapentaplegic (dpp) expression help define the limits of inductive interactions between the mesoderm and dorsal ectoderm after gastrulation. Similarly, the differential regulation of short gastrulation (sog) and dpp may be decisive in the initial subdivision of the dorsal ectoderm, whereas different limits of gene expression within the neuroectoderm might provide the basis for the subsequent subdivision of this tissue into ventral and lateral regions.' publications.doi=None publications.firstAuthor='Rusch J' publications.id=1004078 publications.issue='4' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Aug' publications.pages='416-23' publications.pubMedId='8791536' publications.title='Threshold responses to the dorsal regulatory gradient and the subdivision of primary tissue territories in the Drosophila embryo.' publications.volume='6' publications.year=1996\n",
      "Gene: publications.abstractText='Dorsoventral patterning in Drosophila requires the Dorsal morphogen to act as both an activator and a repressor of transcription: an HMG1-like protein may serve to switch Dorsal from one to the other.' publications.doi=None publications.firstAuthor='Ip Y T' publications.id=1007784 publications.issue='1' publications.journal='Curr. Biol.' publications.month='Jan' publications.pages='1-3' publications.pubMedId='7697337' publications.title='Transcriptional regulation. Converting an activator into a repressor.' publications.volume='5' publications.year=1995\n",
      "Gene: publications.abstractText='Dorsoventral patterning of the Drosophila embryo is regulated by graded distribution of bone morphogenetic proteins (BMPs) composed of two ligands, decapentaplegic (Dpp) a BMP2/4 ortholog and screw (Scw) a BMP5/6/7/8 family member. scw(E1) encodes an unusual allele that was isolated as a dominant enhancer of partial loss-of-function mutations in dpp. However, the molecular mechanisms that underlie this genetic interaction remain to be addressed. Here we show that scw(E1) contains a mutation at the furin cleavage site within the prodomain that is crucial for ligand production. Furthermore, our data show that Scw(E1) preferentially forms heterodimers with Dpp rather than homotypic dimers, providing a possible explanation for the dominant negative phenotype of scw(E1) alleles. The unprocessed prodomain of Scw(E1) remains in a complex with the Dpp:Scw heterodimer, and thus could interfere with interaction of the ligand with the extracellular matrix, or the kinetics of processing/secretion of the ligand in vivo. These data reveal novel mechanisms by which post-translational regulation of Scw can modulate Dpp signaling activity.' publications.doi='10.1016/j.ydbio.2014.02.007' publications.firstAuthor='Künnapuu Jaana' publications.id=1007764 publications.issue='2' publications.journal='Dev. Biol.' publications.month='May' publications.pages='149-59' publications.pubMedId='24560644' publications.title='Cleavage of the Drosophila screw prodomain is critical for a dynamic BMP morphogen gradient in embryogenesis.' publications.volume='389' publications.year=2014\n",
      "Gene: publications.abstractText='Dorsoventral polarity of the Drosophila embryo requires maternal spätzle-Toll signaling to establish a nuclear gradient of Dorsal protein. The shape of this gradient is altered in embryos produced by females carrying dominant alleles of easter (ea(D)). The easter gene encodes a serine protease that generates processed Spätzle, which is proposed to act as the Toll ligand. By examining the expression domains of the zygotic genes zen, sog, rho and twist, which are targets of nuclear Dorsal, we show that the slope of the Dorsal gradient is progressively flattened in stronger ea(D) alleles. In the wild-type embryo, activated Easter is found in a high M(r) complex called Ea-X, which is hypothesized to contain a protease inhibitor. In ea(D) embryo extracts, we detect an Easter form corresponding to the free catalytic domain, which is never observed in wild type. These mutant ea(D) proteins retain protease activity, as determined by the production of processed Spätzle both in the embryo and in cultured Drosophila cells. These experiments suggest that the ea(D) mutations interfere with inactivation of catalytic Easter, and imply that this negative regulation is essential for generating the wild-type shape of the Dorsal gradient.' publications.doi=None publications.firstAuthor='Chang Andy J' publications.id=1007733 publications.issue='24' publications.journal='Development' publications.month='Dec' publications.pages='5635-45' publications.pubMedId='12421704' publications.title='Regulation of Easter activity is required for shaping the Dorsal gradient in the Drosophila embryo.' publications.volume='129' publications.year=2002\n",
      "Gene: publications.abstractText='Dpp signals are responsible for establishing a variety of cell identities in dorsal and lateral areas of the early Drosophila embryo, including the extra-embryonic amnioserosa as well as different ectodermal and mesodermal cell types. Although we have a reasonably clear picture of how Dpp signaling activity is modulated spatially and temporally during these processes, a better understanding of how these signals are executed requires the identification and characterization of a collection of downstream genes that uniquely respond to these signals. In the present study, we describe three novel genes, Dorsocross1, Dorsocross2 and Dorsocross3, which are expressed downstream of Dpp in the presumptive and definitive amnioserosa, dorsal ectoderm and dorsal mesoderm. We show that these genes are good candidates for being direct targets of the Dpp signaling cascade. Dorsocross expression in the dorsal ectoderm and mesoderm is metameric and requires a combination of Dpp and Wingless signals. In addition, a transverse stripe of expression in dorsoanterior areas of early embryos is independent of Dpp. The Dorsocross genes encode closely related proteins of the T-box domain family of transcription factors. All three genes are arranged in a gene cluster, are expressed in identical patterns in embryos, and appear to be genetically redundant. By generating mutants with a loss of all three Dorsocross genes, we demonstrate that Dorsocross gene activity is crucial for the completion of differentiation, cell proliferation arrest, and survival of amnioserosa cells. In addition, we show that the Dorsocross genes are required for normal patterning of the dorsolateral ectoderm and, in particular, the repression of wingless and the ladybird homeobox genes within this area of the germ band. These findings extend our knowledge of the regulatory pathways during amnioserosa development and the patterning of the dorsolateral embryonic germ band in response to Dpp signals.' publications.doi=None publications.firstAuthor='Reim Ingolf' publications.id=1007878 publications.issue='14' publications.journal='Development' publications.month='Jul' publications.pages='3187-204' publications.pubMedId='12783790' publications.title='The T-box-encoding Dorsocross genes function in amnioserosa development and the patterning of the dorsolateral germ band downstream of Dpp.' publications.volume='130' publications.year=2003\n",
      "Gene: publications.abstractText='Dramatic successes in identifying vertebrate homeobox genes closely related to their insect relatives have led to the recognition of classes within the homeodomain superfamily. To what extent are the homeodomain protein classes dedicated to specific functions during development? Although information on vertebrate gene functions is limited, existing evidence from mice and nematodes clearly supports conservation of function for the Hox genes. Less compelling, but still remarkable, is the conservation of other homeobox gene classes and of regulators of homeotic gene expression and function. It is too soon to say whether the cases of conservation are unique and exceptional, or the beginning of a profoundly unified view of gene regulation in animal development. In any case, new questions are raised by the data: how can the differences between mammals and insects be compatible with conservation of homeobox gene function? Did the evolution of animal form involve a proliferation of new homeodomain proteins, new modes of regulation of existing gene types, or new relationships with target genes, or is evolutionary change largely the province of other classes of genes? In this review, we summarize what is known about conservation of homeobox gene function.' publications.doi=None publications.firstAuthor='Manak J R' publications.id=1007790 publications.issue=None publications.journal='Dev. Suppl.' publications.month=None publications.pages='61-77' publications.pubMedId='7579525' publications.title='A class act: conservation of homeodomain protein functions.' publications.volume=None publications.year=1994\n",
      "Gene: publications.abstractText=\"Drosophila embryos lacking hindsight gene function have a normal body plan and undergo normal germ-band extension. However, they fail to retract their germ bands. hindsight encodes a large nuclear protein of 1920 amino acids that contains fourteen C2H2-type zinc fingers, and glutamine-rich and proline-rich domains, suggesting that it functions as a transcription factor. Initial embryonic expression of hindsight RNA and protein occurs in the endoderm (midgut) and extraembryonic membrane (amnioserosa) prior to germ-band extension and continues in these tissues beyond the completion of germ-band retraction. Expression also occurs in the developing tracheal system, central and peripheral nervous systems, and the ureter of the Malpighian tubules. Strikingly, hindsight is not expressed in the epidermal ectoderm which is the tissue that undergoes the cell shape changes and movements during germ-band retraction. The embryonic midgut can be eliminated without affecting germ-band retraction. However, elimination of the amnioserosa results in the failure of germ-band retraction, implicating amnioserosal expression of hindsight as crucial for this process. Ubiquitous expression of hindsight in the early embryo rescues germ-band retraction without producing dominant gain-of-function defects, suggesting that hindsight's role in germ-band retraction is permissive rather than instructive. Previous analyses have shown that hindsight is required for maintenance of the differentiated amnioserosa (Frank, L. C. and Rushlow, C. (1996) Development 122, 1343-1352). Two classes of models are consistent with the present data. First, hindsight's function in germ-band retraction may be limited to maintenance of the amnioserosa which then plays a physical role in the retraction process through contact with cells of the epidermal ectoderm. Second, hindsight might function bothto maintain the amnioserosa and to regulate chemical signaling from the amnioserosa to the epidermal ectoderm, thus coordinating the cell shape changes and movements that drive germ-band retraction.\" publications.doi=None publications.firstAuthor='Yip M L' publications.id=1004126 publications.issue='11' publications.journal='Development' publications.month='Jun' publications.pages='2129-41' publications.pubMedId='9187140' publications.title='Control of germ-band retraction in Drosophila by the zinc-finger protein HINDSIGHT.' publications.volume='124' publications.year=1997\n",
      "Gene: publications.abstractText='Drosophila melanogaster is a proven model system for many aspects of human biology. Here we present a two-hybrid-based protein-interaction map of the fly proteome. A total of 10,623 predicted transcripts were isolated and screened against standard and normalized complementary DNA libraries to produce a draft map of 7048 proteins and 20,405 interactions. A computational method of rating two-hybrid interaction confidence was developed to refine this draft map to a higher confidence map of 4679 proteins and 4780 interactions. Statistical modeling of the network showed two levels of organization: a short-range organization, presumably corresponding to multiprotein complexes, and a more global organization, presumably corresponding to intercomplex connections. The network recapitulated known pathways, extended pathways, and uncovered previously unknown pathway components. This map serves as a starting point for a systems biology modeling of multicellular organisms, including humans.' publications.doi='10.1126/science.1090289' publications.firstAuthor='Giot L' publications.id=1000762 publications.issue='5651' publications.journal='Science' publications.month='Dec' publications.pages='1727-36' publications.pubMedId='14605208' publications.title='A protein interaction map of Drosophila melanogaster.' publications.volume='302' publications.year=2003\n",
      "Gene: publications.abstractText='During embryogenesis Drosophila pole cells, like germ cells in many other vertebrates and invertebrates, actively migrate before assembling into the gonad. Five separate steps can be distinguished: an initial passive displacement of the germ cells by gastrulation movements, an amoeboid migratory phase during which the pole cells pass through the endoderm, migration on endoderm toward mesoderm, separation into two bilateral groups associated with the gonadal mesoderm precursors, followed by condensation into the gonad itself. We have analyzed gonad assembly in embryonic pattern mutants to determine whether distinct cues are required in this process. We show that the initiation of migration does not require the presence of the mesoderm, the eventual target of the germ cells. Rather, migration is triggered as a consequence of the differentiation of the endoderm. Examination of embryos mutant for maternal genes of the terminal group suggests that a primary role of the endoderm in this process is to allow the germ cell access to the interior of the embryo. In addition, we show that normal gonad assembly requires a region of the embryo that includes the posterior compartment of the fifth and the sixth abdominal segments.' publications.doi='10.1006/dbio.1994.1306' publications.firstAuthor='Warrior R' publications.id=1007791 publications.issue='1' publications.journal='Dev. Biol.' publications.month='Nov' publications.pages='180-94' publications.pubMedId='7958445' publications.title='Primordial germ cell migration and the assembly of the Drosophila embryonic gonad.' publications.volume='166' publications.year=1994\n",
      "Gene: publications.abstractText=\"During postblastoderm embryogenesis in Drosophila, cell cycles progress in an invariant spatiotemporal pattern. Most of these cycles are differentially timed by bursts of transcription of string (cdc25), a gene encoding a phosphatase that triggers mitosis by activating the Cdc2 kinase. An analysis of string expression in 36 pattern-formation mutants shows that known patterning genes act locally to influence string transcription. Embryonic expression of string gene fragments shows that the complete pattern of string transcription requires extensive cis-acting regulatory sequences (>15.3 kb), but that smaller segments of this regulatory region can drive proper temporal expression in defined spatial domains. We infer that string upstream sequences integrate many local signals to direct string's transcriptional program. Finally, we show that the spatiotemporal progression of string transcription is largely unaffected in mutant embryos specifically arrested in G2 of cycles 14, 15, or 16, or G1 of cycle 17. Thus, there is a regulatory hierarchy in which developmental inputs, not cell cycle inputs, control the timing of string transcription and hence cell cycle progression.\" publications.doi=None publications.firstAuthor='Edgar B A' publications.id=1007838 publications.issue='11' publications.journal='Development' publications.month='Nov' publications.pages='3131-43' publications.pubMedId='7720557' publications.title='Transcriptional regulation of string (cdc25): a link between developmental programming and the cell cycle.' publications.volume='120' publications.year=1994\n",
      "Gene: publications.abstractText='During the maternal-to-zygotic transition, a developing embryo integrates post-transcriptional regulation of maternal mRNAs with transcriptional activation of its own genome. By combining chromosomal ablation in Drosophila with microarray analysis, we characterized the basis of this integration. We show that the expression profile for at least one third of zygotically active genes is coupled to the concomitant degradation of the corresponding maternal mRNAs. The embryo uses transcription and degradation to generate localized patterns of expression, and zygotic transcription to degrade distinct classes of maternal transcripts. Although degradation does not appear to involve a simple regulatory code, the activation of the zygotic genome starts from intronless genes sharing a common cis-element. This cis-element interacts with a single protein, the Bicoid stability factor, and acts as a potent enhancer capable of timing the activity of an exogenous transactivator. We propose that this regulatory mode links morphogen gradients with temporal regulation during the maternal-to-zygotic transition.' publications.doi='10.1371/journal.pbio.0050117' publications.firstAuthor='De Renzis Stefano' publications.id=1007822 publications.issue='5' publications.journal='PLoS Biol.' publications.month='May' publications.pages='e117' publications.pubMedId='17456005' publications.title='Unmasking activation of the zygotic genome using chromosomal deletions in the Drosophila embryo.' publications.volume='5' publications.year=2007\n",
      "Gene: publications.abstractText='Early Drosophila development requires two receptor tyrosine kinase (RTK) pathways: the Torso and the Epidermal growth factor receptor (EGFR) pathways, which regulate terminal and dorsal-ventral patterning, respectively. Previous studies have shown that these pathways, either directly or indirectly, lead to post-transcriptional downregulation of the Capicua repressor in the early embryo and in the ovary. Here, we show that both regulatory effects are direct and depend on a MAPK docking site in Capicua that physically interacts with the MAPK Rolled. Capicua derivatives lacking this docking site cause dominant phenotypes similar to those resulting from loss of Torso and EGFR activities. Such phenotypes arise from inappropriate repression of genes normally expressed in response to Torso and EGFR signaling. Our results are consistent with a model whereby Capicua is the main nuclear effector of the Torso pathway, but only one of different effectors responding to EGFR signaling. Finally, we describe differences in the modes of Capicua downregulation by Torso and EGFR signaling, raising the possibility that such differences contribute to the tissue specificity of both signals.' publications.doi='10.1038/sj.emboj.7601532' publications.firstAuthor='Astigarraga Sergio' publications.id=1007686 publications.issue='3' publications.journal='EMBO J.' publications.month='Feb' publications.pages='668-77' publications.pubMedId='17255944' publications.title='A MAPK docking site is critical for downregulation of Capicua by Torso and EGFR RTK signaling.' publications.volume='26' publications.year=2007\n",
      "Gene: publications.abstractText='Embryogenesis in individuals with mutations or deficiencies of the genes in the polytene interval 84A-84B1,2 of Drosophila melanogaster was examined using scanning electron microscopy (SEM). The developmental function of this region of chromosome 3 is of particular interest since it contains the Antennapedia Gene Complex (ANT-C), a gene cluster that includes the homoeotic proboscipedia (pb), Sex combs reduced (Scr), and Antennapedia (Antp) loci. The results of SEM studies, clonal analyses, and temperature-shift experiments show that the fushi tarazu (ftz) and zerknullt (zen) genes, which map between pb and Scr, are involved in processes initiated during embryogenesis. The activity of ftz+ appears to be required within the first 4 hr of development for the establishment of the proper number of segments in the embryonic germ band. Individuals with ftz mutations or deficiencies produce only half the normal number of segments. Each of the segments is twice the normal width and is apparently comprised of cells that would normally form two separate metameres. The zen allele is required from about 2-4 hr of embryogenesis. Mutations of this gene result in disturbances of morphogenetic movements during gastrulation. The mutant phenotype is characterized by the absence of the optic lobe, defects in involution of the head segments, and in some cases, failure of germ band elongation. A requirement during embryogenesis for the activities of other genes residing in the 84A-84B1,2 polytene interval is suggested by the phenotypes of individuals heterozygous or homozygous for chromosomal deficiencies. Using the deficiencies Df(3R)AntpNs+R17, Df(3R)Scr, and Df(3R)ScxW+RX2, we examined the effects of deleting the distal portions or all of the 84A-84B1,2 interval. The defects in deletion heterozygotes suggest that the wild-type activity of some gene(s) other than zen, within or just adjacent to the 84B1,2 doublet, is required to complete normal head involution. Thedeletion of all the loci in the 84A5-84B1,2 interval results in grossly abnormal morphology and morphogenesis of the gnathocephalic appendages of the embryo. From these studies we conclude that mutations and deficiencies of genes associated with the ANT-C have profound effects on embryogenesis. The mutant phenotypes suggest, in addition to ensuring proper segment identity, the wild-type alleles of the 84A-84B1,2 genes are necessary for normal segmentation and elongation of the germ band and normal head involution.' publications.doi=None publications.firstAuthor='Wakimoto B T' publications.id=1005524 publications.issue='1' publications.journal='Dev. Biol.' publications.month='Mar' publications.pages='147-72' publications.pubMedId='6421639' publications.title='Defects in embryogenesis in mutants associated with the antennapedia gene complex of Drosophila melanogaster.' publications.volume='102' publications.year=1984\n",
      "Gene: publications.abstractText='Embryonic dorsal-ventral polarity in Drosophila is established through a series of successive steps and requires the functions of both maternal and zygotic genes. The graded distribution of the transcription factor dorsal in blastoderm nuclei represents the transition from the maternal to the zygotic program. This results in the activation of specific zygotic genes that act to create the regional pattern along this axis.' publications.doi=None publications.firstAuthor='Steward R' publications.id=1002691 publications.issue='4' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Aug' publications.pages='556-61' publications.pubMedId='8241766' publications.title='Dorsal-ventral polarity in the Drosophila embryo.' publications.volume='3' publications.year=1993\n",
      "Gene: publications.abstractText='Embryonic gene expression patterns are an indispensable part of modern developmental biology. Currently, investigators must visually inspect numerous images containing embryonic expression patterns to identify spatially similar patterns for inferring potential genetic interactions. The lack of a computational approach to identify pattern similarities is an impediment to advancement in developmental biology research because of the rapidly increasing amount of available embryonic gene expression data. Therefore, we have developed computational approaches to automate the comparison of gene expression patterns contained in images of early stage Drosophila melanogaster embryos (prior to the beginning of germ-band elongation); similarities and differences in gene expression patterns in these early stages have extensive developmental effects. Here we describe a basic expression search tool (BEST) to retrieve best matching expression patterns for a given query expression pattern and a computational device for gene interaction inference using gene expression pattern images and information on the associated genotypes and probes. Analysis of a prototype collection of Drosophila gene expression pattern images is presented to demonstrate the utility of these methods in identifying biologically meaningful matches and inferring gene interactions by direct image content analysis. In particular, the use of BEST searches for gene expression patterns is akin to that of BLAST searches for finding similar sequences. These computational developmental biology methodologies are likely to make the great wealth of embryonic gene expression pattern data easily accessible and to accelerate the discovery of developmental networks.' publications.doi=None publications.firstAuthor='Kumar Sudhir' publications.id=1007875 publications.issue='4' publications.journal='Genetics' publications.month='Dec' publications.pages='2037-47' publications.pubMedId='12524369' publications.title='BEST: a novel computational approach for comparing gene expression patterns from early stages of Drosophila melanogaster development.' publications.volume='162' publications.year=2002\n",
      "Gene: publications.abstractText='Epistasis analysis has been used in molecular genetics to infer cause and effect relationships among loci that decide a developmental fate. In this paper, in order to investigate the logical background of the rule that relates epistasis and causal relation, a mathematical epistasis analysis is formulated. An \"orderliness index\"is introduced by which the cause-and-effect relation can systematically be extracted from genotype-phenotype data. This method is applied to Drosophila melanogaster dorso-ventral axis formation and it is shown that the method of reconstructing the framework of the causal relation between a locus and its sign (activation or inhibition) as an \"orderliness diagram\"from eight groups of the genes. From the diagram it can be seen that the dorsal gene works as a morphogen. The difference between orderliness diagrams of the dorsal and ventral sides of the embryo shows that the cells on the ventral side receive the signal and the fate of the cells on the dorsal side is default in the wild-type embryo.' publications.doi='10.1006/jtbi.1995.0175' publications.firstAuthor='Takeda Y' publications.id=1007701 publications.issue='1' publications.journal='J. Theor. Biol.' publications.month='Sep' publications.pages='47-58' publications.pubMedId='7475106' publications.title='A generalized epistasis analysis as a method of reconstruction of developmental process and its application to Drosophila dorso-ventral axis formation.' publications.volume='176' publications.year=1995\n",
      "Gene: publications.abstractText='Establishment of dorsal-ventral polarity in the early Drosophila embryo requires a concentration gradient of the maternal morphogen dorsal (dl). This concentration gradient is established by selective nuclear transport of dl so that dl protein is present only in ventral nuclei. The activity of 11 genes is required for dl nuclear localization. One of these genes, Toll, encodes a transmembrane protein that appears to play the most direct role in regulating dl localization. We have examined the effects of Toll on dl in cotransfected Schneider cells to gain insight into the nature of the interaction between these proteins. We have found that Toll can enhance the nuclear localization of dl and, independently, the ability of dl to activate transcription once in the nucleus. We present evidence that the signaling pathway from Toll to dl involves protein kinase A (PKA) and that nuclear transport and activation of dl results from phosphorylation of dl by PKA. We discuss the significance of these results with respect both to Drosophila embryogenesis and to the regulation of the mammalian transcription factor NF-kappa B.' publications.doi=None publications.firstAuthor='Norris J L' publications.id=1007794 publications.issue='9' publications.journal='Genes Dev.' publications.month='Sep' publications.pages='1654-67' publications.pubMedId='1325392' publications.title='Selective nuclear transport of the Drosophila morphogen dorsal can be established by a signaling pathway involving the transmembrane protein Toll and protein kinase A.' publications.volume='6' publications.year=1992\n",
      "Gene: publications.abstractText='Evolutionary constraints on gene regulatory elements are poorly understood: Little is known about how the strength of transcription factor binding correlates with DNA sequence conservation, and whether transcription factor binding sites can evolve rapidly while retaining their function. Here we use the model of the NFKB/Rel-dependent gene regulation in divergent Drosophila species to examine the hypothesis that the functional properties of authentic transcription factor binding sites are under stronger evolutionary constraints than the genomic background. Using molecular modeling we compare tertiary structures of the Drosophila Rel family proteins Dorsal, Dif, and Relish and demonstrate that their DNA-binding and protein dimerization domains undergo distinct rates of evolution. The accumulated amino acid changes, however, are unlikely to affect DNA sequence recognition and affinity. We employ our recently developed microarray-based experimental platform and principal coordinates statistical analysis to quantitatively and systematically profile DNA binding affinities of three Drosophila Rel proteins to 10,368 variants of the NFKB recognition sequences. We then correlate the evolutionary divergence of gene regulatory regions with differences in DNA binding affinities. Genome-wide analyses reveal a significant increase in the number of conserved Rel binding sites in promoters of developmental and immune genes. Significantly, the affinity of Rel proteins to these sites was higher than to less conserved sites and was maintained by the conservation of the DNA binding site sequence (static conservation) or in some cases despite significantly diverged sequences (dynamic conservation). We discuss how two types of conservation may contribute to the stabilization and optimization of a functional gene regulatory code in evolution.' publications.doi='10.1101/gr.6490707' publications.firstAuthor='Copley Richard R' publications.id=1007785 publications.issue='9' publications.journal='Genome Res.' publications.month='Sep' publications.pages='1327-35' publications.pubMedId='17785540' publications.title='Functional conservation of Rel binding sites in drosophilid genomes.' publications.volume='17' publications.year=2007\n",
      "Gene: publications.abstractText='Extensive genetic and biochemical analysis of Drosophila melanogaster has made this system an important model for characterization of transcriptional regulatory elements and factors. Given the striking conservation of transcriptional controls in metazoans, general principles derived from studies of Drosophila are expected to continue to illuminate transcriptional regulation in other systems, including vertebrates. With improvement in technologies for genetic manipulation of insects, research in Drosophila will also aid the design of systems for controlled expression of genes in other hosts. This review focuses on recent advances from Drosophila in analysis of the functional components of transcriptional switches, including basal promoters, enhancers, boundary elements, and maintenance elements.' publications.doi=None publications.firstAuthor='Arnosti D N' publications.id=1005487 publications.issue='10' publications.journal='Insect Biochem. Mol. Biol.' publications.month='Oct' publications.pages='1257-73' publications.pubMedId='12225917' publications.title='Design and function of transcriptional switches in Drosophila.' publications.volume='32' publications.year=2002\n",
      "Gene: publications.abstractText='Formation of the dorsoventral axis in Drosophila melanogaster is mediated through control of the expression of several genes by the morphogen Dorsal. In the ventral part of the embryo Dorsal activates twist and represses zen amongst others. Recently, several proteins have been shown to assist Dorsal in the repression of zen, one of which is DSP1, a HMG box protein that was isolated as a putative co-repressor of Dorsal. In this report we used a DSP1 null mutant to ascertain in vivo the involvement of DSP1 in Dorsal-mediated repression of zen but not in the activation of twist. We show that Dorsal has the ability to interact with DSP1 in vitro as well as with rat HMG1. Using truncated versions of the proteins we located the domains of interaction as being the HMG boxes for DSP1 and HMG1 and the Rel domain for Dorsal. Finally, studies of the zen DNA binding properties of Dorsal and another related Rel protein (Gambif1 from Anopheles gambiae) revealed that their DNA binding affinities were increased in the presence of DSP1 and HMG1.' publications.doi=None publications.firstAuthor='Decoville M' publications.id=1007857 publications.issue='2' publications.journal='Nucleic Acids Res.' publications.month='Jan' publications.pages='454-62' publications.pubMedId='10606643' publications.title='HMG boxes of DSP1 protein interact with the rel homology domain of transcription factors.' publications.volume='28' publications.year=2000\n",
      "Gene: publications.abstractText='Functional assays in Drosophila melanogaster with orthologous transcription factors from other species suggest that changes in the protein-coding sequence may play a larger role in the evolution of transcription factor pathways than was previously believed. Interestingly, recent studies provide evidence that changes in transcription factor protein sequence can affect the regulation of only a subset of target genes, even in the same cells of a developing animal.' publications.doi=None publications.firstAuthor='Hsia Cheryl C' publications.id=1007711 publications.issue='2' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Apr' publications.pages='199-206' publications.pubMedId='12672498' publications.title='Evolution of transcription factor function.' publications.volume='13' publications.year=2003\n",
      "Gene: publications.abstractText='Functional studies seem now to confirm, as first suggested by E. Geoffroy Saint-Hilaire in 1822, that there was an inversion of the dorsoventral axis during animal evolution. A conserved system of extracellular signals provides positional information for the allocation of embryonic cells to specific tissue types both in Drosophila and vertebrates; the ventral region of Drosophila is homologous to the dorsal side of the vertebrate. Developmental studies are now revealing some of the characteristics of the ancestral animal that gave rise to the arthropod and mammalian lineages, for which we propose the name Urbilateria.' publications.doi='10.1038/380037a0' publications.firstAuthor='De Robertis E M' publications.id=1007772 publications.issue='6569' publications.journal='Nature' publications.month='Mar' publications.pages='37-40' publications.pubMedId='8598900' publications.title='A common plan for dorsoventral patterning in Bilateria.' publications.volume='380' publications.year=1996\n",
      "Gene: publications.abstractText='Genetic screens in Drosophila melanogaster have helped elucidate the process of axis formation during early embryogenesis. Axis formation in the D. melanogaster embryo involves the use of two fundamentally different mechanisms for generating morphogenetic activity: patterning the anteroposterior axis by diffusion of a transcription factor within the syncytial embryo and specification of the dorsoventral axis through a signal transduction cascade. Identification of Drosophila genes involved in axis formation provides a launch-pad for comparative studies that examine the evolution of axis specification in different insects. Additionally, there is similarity between axial patterning mechanisms elucidated genetically in Drosophila and those demonstrated for chordates such as Xenopus. In this review we examine the postfertilization mechanisms underlying axis specification in Drosophila. Comparative data are then used to ask whether aspects of axis formation might be derived or ancestral.' publications.doi='10.1146/annurev.genet.35.102401.090832' publications.firstAuthor='Lall S' publications.id=1002649 publications.issue=None publications.journal='Annu. Rev. Genet.' publications.month=None publications.pages='407-37' publications.pubMedId='11700289' publications.title='Conservation and divergence in molecular mechanisms of axis formation.' publications.volume='35' publications.year=2001\n",
      "Gene: publications.abstractText='Genetic studies have identified numerous sequence-specific transcription factors that control development, yet little is known about their in vivo distribution across animal genomes. We determined the genome-wide occupancy of the dorsoventral (DV) determinants Dorsal, Twist, and Snail in the Drosophila embryo using chromatin immunoprecipitation coupled with microarray analysis (ChIP-chip). The in vivo binding of these proteins correlate tightly with the limits of known enhancers. Our analysis predicts substantially more target genes than previous estimates, and includes Dpp signaling components and anteroposterior (AP) segmentation determinants. Thus, the ChIP-chip data uncover a much larger than expected regulatory network, which integrates diverse patterning processes during development.' publications.doi='10.1101/gad.1509607' publications.firstAuthor='Zeitlinger Julia' publications.id=1007897 publications.issue='4' publications.journal='Genes Dev.' publications.month='Feb' publications.pages='385-90' publications.pubMedId='17322397' publications.title='Whole-genome ChIP-chip analysis of Dorsal, Twist, and Snail suggests integration of diverse patterning processes in the Drosophila embryo.' publications.volume='21' publications.year=2007\n",
      "Gene: publications.abstractText='Genetic variation in brain size may provide the basis for the evolution of the brain and complex behaviours. The genetic substrate and the selective pressures acting on brain size are poorly understood. Here we use the Drosophila Genetic Reference Panel to map polymorphic variants affecting natural variation in mushroom body morphology. We identify 139 genes and 39 transcription factors and confirm effects on development and adult plasticity. We show correlations between morphology and aggression, sleep and lifespan. We propose that natural variation in adult brain size is controlled by interaction of the environment with gene networks controlling development and plasticity.' publications.doi='10.1038/ncomms10115' publications.firstAuthor='Zwarts Liesbeth' publications.id=1000539 publications.issue=None publications.journal='Nat Commun' publications.month=None publications.pages='10115' publications.pubMedId='26656654' publications.title='The genetic basis of natural variation in mushroom body size in Drosophila melanogaster.' publications.volume='6' publications.year=2015\n",
      "Gene: publications.abstractText='Genetics and molecular analyses have combined to yield insights into a functional cascade of transcription factors necessary to establish the molecular blueprint of the Drosophila body pattern in response to positional information in the egg. Recent progress in this field raises exciting questions regarding the molecular mechanisms involved, and their conservation in biological pattern-forming processes.' publications.doi=None publications.firstAuthor='Jäckle H' publications.id=1007744 publications.issue='3' publications.journal='Curr. Opin. Cell Biol.' publications.month='Jun' publications.pages='505-12' publications.pubMedId='8352969' publications.title='Transcriptional cascades in Drosophila.' publications.volume='5' publications.year=1993\n",
      "Gene: publications.abstractText='Genome sequencesfor most metazoans and plants are incomplete because of the presence of repeated DNA in the heterochromatin. The heterochromatic regions of Drosophila melanogaster contain 20 million bases (Mb) of sequence amenable to mapping, sequence assembly, and finishing. We describe the generation of 15 Mb of finished or improved heterochromatic sequence with the use of available clone resources and assembly methods. We also constructed a bacterial artificial chromosome-based physical map that spans 13 Mb of the pericentromeric heterochromatin and a cytogenetic map that positions 11 Mb in specific chromosomal locations. We have approached a complete assembly and mapping of the nonsatellite component of Drosophila heterochromatin. The strategy we describe is also applicable to generating substantially more information about heterochromatin in other species, including humans.' publications.doi='10.1126/science.1139816' publications.firstAuthor='Hoskins Roger A' publications.id=20254265 publications.issue='5831' publications.journal='Science' publications.month='Jun' publications.pages='1625-8' publications.pubMedId='17569867' publications.title='Sequence finishing and mapping of Drosophila melanogaster heterochromatin.' publications.volume='316' publications.year=2007\n",
      "Gene: publications.abstractText='Gradients of morphogens determine cell fates by specifying discrete thresholds of gene activities. In the Drosophila embryo, a BMP gradient subdivides the dorsal ectoderm into amnioserosa and dorsal epidermis, and also inhibits neuroectoderm formation. A number of genes are differentially expressed in response to the gradient, but how their borders of expression are established is not well understood. We present evidence that the BMP gradient, via the Smads, provides a two-fold input in regulating the amnioserosa-specific target genes such as Race. Peak levels of Smads in the presumptive amnioserosa set the expression domain of zen, and then Smads act in combination with Zen to directly activate Race. This situation resembles a feed-forward mechanism of transcriptional regulation. In addition, we demonstrate that ectopically expressed Zen can activate targets like Race in the presence of low level Smads, indicating that the role of the highest activity of the BMP gradient is to activate zen.' publications.doi='10.1242/dev.01722' publications.firstAuthor='Xu Mu' publications.id=1007761 publications.issue='7' publications.journal='Development' publications.month='Apr' publications.pages='1637-47' publications.pubMedId='15728670' publications.title='Peak levels of BMP in the Drosophila embryo control target genes by a feed-forward mechanism.' publications.volume='132' publications.year=2005\n",
      "Gene: publications.abstractText='Groucho (Gro) is a Drosophila corepressor required by numerous DNA-binding repressors, many of which are distributed in gradients and provide positional information during development. Gro contains well-conserved domains at its N- and C-termini, and a poorly conserved central region that includes the GP, CcN, and SP domains. All lethal point mutations in gro map to the conserved regions, leading to speculation that the unconserved central domains are dispensable. However, our sequence analysis suggests that the central domains are disordered leading us to suspect that the lack of lethal mutations in this region reflects a lack of order rather than an absence of essential functions. In support of this conclusion, genomic rescue experiments with Gro deletion variants demonstrate that the GP and CcN domains are required for viability. Misexpression assays using these same deletion variants show that the SP domain prevents unrestrained and promiscuous repression by Gro, while the GP and CcN domains are indispensable for repression. Deletion of the GP domain leads to loss of nuclear import, while deletion of the CcN domain leads to complete loss of repression. Changes in Gro activity levels reset the threshold concentrations at which graded repressors silence target gene expression. We conclude that co-regulators such as Gro are not simply permissive components of the repression machinery, but cooperate with graded DNA-binding factors in setting borders of gene expression. We suspect that disorder in the Gro central domains may provide the flexibility that allows this region to mediate multiple interactions required for repression.' publications.doi='10.1371/journal.pone.0030610' publications.firstAuthor='Turki-Judeh Wiam' publications.id=1007886 publications.issue='2' publications.journal='PLoS ONE' publications.month=None publications.pages='e30610' publications.pubMedId='22319573' publications.title='The unconserved groucho central region is essential for viability and modulates target gene specificity.' publications.volume='7' publications.year=2012\n",
      "Gene: publications.abstractText='HOT (highly occupied target) regions bound by many transcription factors are considered to be one of the most intriguing findings of the recent modENCODE reports, yet their functions have remained unclear. We tested 108 Drosophila melanogaster HOT regions in transgenic embryos with site-specifically integrated transcriptional reporters. In contrast to prior expectations, we found 102 (94%) to be active enhancers during embryogenesis and to display diverse spatial and temporal patterns, reminiscent of expression patternsfor important developmental genes. Remarkably, HOT regions strongly activate nearby genes and are required for endogenous gene expression, as we show using bacterial artificial chromosome (BAC) transgenesis. HOT enhancers have a distinct cis-regulatory signature with enriched sequence motifs for the global activators Vielfaltig, also known as Zelda, and Trithorax-like, also known as GAGA. This signature allows the prediction of HOT versus control regions from the DNA sequence alone.' publications.doi='10.1101/gad.188052.112' publications.firstAuthor='Kvon Evgeny Z' publications.id=1007709 publications.issue='9' publications.journal='Genes Dev.' publications.month='May' publications.pages='908-13' publications.pubMedId='22499593' publications.title='HOT regions function as patterned developmental enhancers and have a distinct cis-regulatory signature.' publications.volume='26' publications.year=2012\n",
      "Gene: publications.abstractText='Here, we describe one of the major maternal regulatory gradients, Dorsal, and threshold outputs of gene expression that result from the graded distribution of this transcription factor. The analysis of a large number of authentic and synthetic target genes suggests that the Dorsal gradient directly specifies at least four, and possibly as many as seven, different thresholds of gene activity and tissue differentiation. These thresholds initiate the differentiation of the three primary embryonic tissues: the mesoderm, neurogenic ectoderm, and dorsal ectoderm. Moreover, primary readouts of the Dorsal gradient create asymmetries that subdivide each tissue into multiple cell types during gastrulation. Dorsal patterning thresholds represent the culmination of one of the most complete gene regulation network known in development, which begins with the asymmetric positioning of the oocyte nucleus within the egg chamber and leads to the localized activation of the Toll-Dorsal signaling pathway in ventral regions of the early embryo.' publications.doi='10.1006/dbio.2002.0652' publications.firstAuthor='Stathopoulos Angelike' publications.id=1002707 publications.issue='1' publications.journal='Dev. Biol.' publications.month='Jun' publications.pages='57-67' publications.pubMedId='12027434' publications.title='Dorsal gradient networks in the Drosophila embryo.' publications.volume='246' publications.year=2002\n",
      "Gene: publications.abstractText='Holometabolous insects like Drosophila proceed through two phases of visual system development. The embryonic phase generates simple eyes of the larva. The postembryonic phase produces the adult specific compound eyes during late larval development and pupation. In primitive insects, by contrast, eye development persists seemingly continuously from embryogenesis through the end of postembryogenesis. Comparative literature suggests that the evolutionary transition from continuous to biphasic eye development occurred via transient developmental arrest. This review investigates how the developmental arrest model relates to the gene networks regulating larval and adult eye development in Drosophila, and embryonic compound eye development in primitive insects. Consistent with the developmental arrest model, the available data suggest that the determination of the anlage of the rudimentary Drosophila larval eye is homologous to the embryonic specification of the juvenile compound eye in directly developing insects while the Drosophila compound eye primordium is evolutionarily related to the yet little studied stem cell based postembryonic eye primordium of primitive insects.' publications.doi='10.1016/j.ydbio.2006.08.027' publications.firstAuthor='Friedrich Markus' publications.id=1007757 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Nov' publications.pages='310-29' publications.pubMedId='16973149' publications.title='Continuity versus split and reconstitution: exploring the molecular developmental corollaries of insect eye primordium evolution.' publications.volume='299' publications.year=2006\n",
      "Gene: publications.abstractText='Homeobox genes encode important developmental control proteins. The Drosophila fruit fly HOM complex genes are clustered in region 84-89 of chromosome 3. Probably due to large-scale genome duplication events, their human HOX orthologs belong to four paralogous regions. A series of 13 other homeobox genes are also clustered in region 88-94, on the same chromosome of Drosophila. We suggest that they also duplicated during vertebrate evolution and belong to paralogous regions in humans. These regions are on chromosome arms 4p, 5q, 10q, and 2p or 8p. We coined the term \"paralogon\"to designate paralogous regions in general. We propose to call these genes \"meta Hox\"genes. Like Hox genes, metaHox genes are present in one cluster in Drosophila and four clusters (metaHox A-D) in humans on the 4p/5q/10q paralogon.' publications.doi='10.1002/1097-010X(20001215)288:4<345::AID-JEZ7>3.0.CO;2-Y' publications.firstAuthor='Coulier F' publications.id=1007417 publications.issue='4' publications.journal='J. Exp. Zool.' publications.month='Dec' publications.pages='345-51' publications.pubMedId='11144283' publications.title='MetaHox gene clusters.' publications.volume='288' publications.year=2000\n",
      "Gene: publications.abstractText='Homeotic (Hox) genes are usually clustered and arranged in the same order as they are expressed along the anteroposterior body axis of metazoans. The mechanistic explanation for this colinearity has been elusive, and it may well be that a single and universal cause does not exist. The Hox-gene complex (HOM-C) has been rearranged differently in several Drosophila species, producing a striking diversity of Hox gene organizations. We investigated the genomic and functional consequences of the two HOM-C splits present in Drosophila buzzatii. Firstly, we sequenced two regions of the D. buzzatii genome, one containing the genes labial and abdominal A, and another one including proboscipedia, and compared their organization with that of D. melanogaster and D. pseudoobscura in order to map precisely the two splits. Then, a plethora of conserved noncoding sequences, which are putative enhancers, were identified around the three Hox genes closer to the splits. The position and order of these enhancers are conserved, with minor exceptions, between the three Drosophila species. Finally, we analyzed the expression patterns of the same three genes in embryos and imaginal discs of four Drosophila species with different Hox-gene organizations. The results show that their expression patterns are conserved despite the HOM-C splits. We conclude that, in Drosophila, Hox-gene clustering is not an absolute requirement for proper function. Rather, the organization of Hox genes is modular, and their clustering seems the result of phylogenetic inertia more than functional necessity.' publications.doi='10.1101/gr.3468605' publications.firstAuthor='Negre Bárbara' publications.id=1007635 publications.issue='5' publications.journal='Genome Res.' publications.month='May' publications.pages='692-700' publications.pubMedId='15867430' publications.title='Conservation of regulatory sequences and gene expression patterns in the disintegrating Drosophila Hox gene complex.' publications.volume='15' publications.year=2005\n",
      "Gene: publications.abstractText='Homeotic genes identify structures along the anterior to posterior axis during the development of most animals. These genes are clustered into complexes, and their positions within the cluster correlates with their time of expression and the positions of the anterioposterior boundaries of their expression domains. Functional analyses have revealed that this specific genetic order also coincides with a functional hierarchy among members of these complexes, so that the products of more posterior genes in the cluster tend to be prevalent over those of more anterior genes.' publications.doi=None publications.firstAuthor='Duboule D' publications.id=1007742 publications.issue='10' publications.journal='Trends Genet.' publications.month='Oct' publications.pages='358-64' publications.pubMedId='7985240' publications.title='Colinearity and functional hierarchy among genes of the homeotic complexes.' publications.volume='10' publications.year=1994\n",
      "Gene: publications.abstractText='Identifying the genomic regions bound by sequence-specific regulatory factors is central both to deciphering the complex DNA cis-regulatory code that controls transcription in metazoans and to determining the range of genes that shape animal morphogenesis. We used whole-genome tiling arrays to map sequences bound in Drosophila melanogaster embryos by the six maternal and gap transcription factors that initiate anterior-posterior patterning. We find that these sequence-specific DNA binding proteins bind with quantitatively different specificities to highly overlapping sets of several thousand genomic regions in blastoderm embryos. Specific high- and moderate-affinity in vitro recognition sequences for each factor are enriched in bound regions. This enrichment, however, is not sufficient to explain the pattern of binding in vivo and varies in a context-dependent manner, demonstrating that higher-order rules must govern targeting of transcription factors. The more highly bound regions include all of the over 40 well-characterized enhancers known to respond to these factors as well as several hundred putative new cis-regulatory modules clustered near developmental regulators and other genes with patterned expression at this stage of embryogenesis. The new targets include most of the microRNAs (miRNAs) transcribed in the blastoderm, as well as all major zygotically transcribed dorsal-ventral patterning genes, whose expression we show to be quantitatively modulated by anterior-posterior factors. In addition to these highly bound regions, there are several thousand regions that are reproducibly bound at lower levels. However, these poorly bound regions are, collectively, far more distant from genes transcribed in the blastoderm than highly bound regions; are preferentially found in protein-coding sequences; and are less conserved than highly bound regions. Together these observations suggest that many of these poorly bound regions are not involved in early-embryonic transcriptional regulation, and a significant proportion may be nonfunctional. Surprisingly, for five of the six factors, their recognition sites are not unambiguously more constrained evolutionarily than the immediate flanking DNA, even in more highly bound and presumably functional regions, indicating that comparative DNA sequence analysis is limited in its ability to identify functional transcription factor targets.' publications.doi='10.1371/journal.pbio.0060027' publications.firstAuthor='Li Xiao-yong' publications.id=1007729 publications.issue='2' publications.journal='PLoS Biol.' publications.month='Feb' publications.pages='e27' publications.pubMedId='18271625' publications.title='Transcription factors bind thousands of active and inactive regions in the Drosophila blastoderm.' publications.volume='6' publications.year=2008\n",
      "Gene: publications.abstractText='In Drosophila elaboration of positional identity along the anterior-posterior and dorsal-ventral embryonic body axes involves early zygotic gene functions that are expressed in response to maternal cues present in the unfertilized egg. Zygotic loci that are required for the specification of positional identity along the anterior-posterior body axis have been described in detail. Less is known about the zygotic loci responsible for differentiation of the dorsal-ventral pattern; however, several genes that might be involved have been identified. Zerknüllt (zen) is an example of a zygotic gene required for correct differentiation of dorsally derived embryonic tissues. On the basis of homoeo box cross homology, we have now isolated a gene, called S60, that derives from the zen region of the Antennapedia complex (ANT-C). Transcripts encoded by S60 transiently accumulate in the dorsal-most tissues of developing embryos. This pattern of expression suggests that S60 corresponds to zen. Since S60 contains a homoeo box, it is possible that differentiation of the anterior-posterior and dorsal-ventral embryonic patterns involves similar molecular mechanisms.' publications.doi='10.1038/323076a0' publications.firstAuthor='Doyle H J' publications.id=1007847 publications.issue='6083' publications.journal='Nature' publications.month=None publications.pages='76-9' publications.pubMedId='3755802' publications.title='Transcripts encoded by a homoeo box gene are restricted to dorsal tissues of Drosophila embryos.' publications.volume='323' publications.year=None\n",
      "Gene: publications.abstractText=\"In Drosophila embryos, dorsal-ventral polarity is defined by a signal transduction pathway that regulates nuclear import ofthe Dorsal protein. Dorsal protein's ability to act as a transcriptional activator of some zygotic genes and a repressor of others defines structure along the dorsal-ventral axis. Dorsal is a member of a group of proteins, the Rel-homologous proteins, whose activity is regulated at the level of nuclear localization. Dif, a more recently identified Drosophila Rel-homologue, has been proposed to act as a mediator of the immune response in Drosophila. In an effort to understand the function and regulation of Rel-homologous proteins in Drosophila, we have expressed Dif protein in Drosophila embryos derived from dorsal mutant mothers. We found that the Dif protein was capable of restoring embryonic dorsal-ventral pattern elements and was able to define polarity correctly with respect to the orientation of the egg shell. This, together with the observation that the ability of Dif to restore a dorsal-ventral axis depended on the signal transduction pathway that normally regulates Dorsal, suggests that Dif protein formed a nuclear concentration gradient similar to that seen for Dorsal. By studying the expression of Dorsal target genes we found that Dif could activate the zygotic genes that Dorsal activates and repress the genes repressed by Dorsal. Differences in the expression of these target genes, as well as the results from interaction studies carried out in yeast, suggest that Dif is not capable of synergizing with the basic helix-loop-helix transcription factors with which Dorsal normally interacts, and thereby lacks an important component of Dorsal-mediated pattern formation.\" publications.doi=None publications.firstAuthor='Stein D' publications.id=1007747 publications.issue='11' publications.journal='Development' publications.month='Jun' publications.pages='2159-69' publications.pubMedId='9570779' publications.title='The Dorsal-related immunity factor (Dif) can define the dorsal-ventral axis of polarity in the Drosophila embryo.' publications.volume='125' publications.year=1998\n",
      "Gene: publications.abstractText='In Drosophila, a Bcd protein gradient orchestrates patterning along the anteroposterior embryonic axis. However, studies of basal flies and other insects have revealed that bcd is a derived Hox3 gene found only in higher dipterans. To understand how bcd acquired its role in flies and how anteroposterior patterning mechanisms have evolved, I first review key features of bcd function in Drosophila: anterior localization and transcriptional and translation control of gene expression. I then discuss investigations of bcd in other higher dipterans that have provided insight into the evolution of regulatory interactions and the Bcd gradient. Finally, I review studies of Drosophila and other insects that address the evolution of bcd function and integration of bcd into ancestral regulatory mechanisms. I suggest further comparative studies may allow us to identify the intermediate steps in bcd evolution. This will make bcd a paradigm for the origin and evolution of genes and regulatory networks.' publications.doi='10.1002/bies.20285' publications.firstAuthor='McGregor Alistair P' publications.id=1007696 publications.issue='9' publications.journal='Bioessays' publications.month='Sep' publications.pages='904-13' publications.pubMedId='16108065' publications.title='How to get ahead: the origin, evolution and function of bicoid.' publications.volume='27' publications.year=2005\n",
      "Gene: publications.abstractText='In Drosophila, pattern formation at multiple stages of embryonic and imaginal development depends on the same intercellular signaling pathways. We have identified a novel gene, eyelid (eld), which is required for embryonic segmentation, development of the notum and wing margin, and photoreceptor differentiation. In these tissues, eld mutations have effects opposite to those caused by wingless (wg) mutations. eld encodes a widely expressed nuclear protein with a region homologous to a novel family of DNA-binding domains. Based on this homology and on the phenotypic analysis, we suggest that Eld could act as a transcription factor antagonistic to the Wg pathway.' publications.doi=None publications.firstAuthor='Treisman J E' publications.id=1007905 publications.issue='15' publications.journal='Genes Dev.' publications.month='Aug' publications.pages='1949-62' publications.pubMedId='9271118' publications.title='eyelid antagonizes wingless signaling during Drosophila development and has homology to the Bright family of DNA-binding proteins.' publications.volume='11' publications.year=1997\n",
      "Gene: publications.abstractText='In Drosophila, the dorsal (dl) morphogen gradient initiates the differentiation of the embryonic mesoderm and neuroectoderm by activating the expression of regulatory genes (e.g. twist and snail) in a concentration-dependent manner. dl also functions as a repressor that establishes the dorsal epidermis and amnioserosa by restricting regulatory genes such as dpp and zen to dorsal regions of the embryo. The ability of dl to function as both an activator and repressor distinguishes it from the bicoid morphogen, which appears to function solely as an activator. In an effort to determine how dl functions as a repressor we have performed a detailed characterization of a zen silencer element, called the VRE, which mediates ventral repression in response to the dl gradient. A minimal 110 bp VRE sequence is identified, which is able to silence the ventral expression of a heterologous promoter. This sequence contains two dl binding sites as well as binding sites for additional nuclear factors present in early embryos. Mutations in the latter binding sites convert the minimal VRE into an enhancer, which mediates transcriptional activation in ventral regions in response to dl. These results suggest that dl is intrinsically an activator, but is converted into a potent silencer when it interacts with neighboring corepressors.' publications.doi=None publications.firstAuthor='Jiang J' publications.id=1007846 publications.issue='8' publications.journal='EMBO J.' publications.month='Aug' publications.pages='3201-9' publications.pubMedId='8344257' publications.title='Conversion of a dorsal-dependent silencer into an enhancer: evidence for dorsal corepressors.' publications.volume='12' publications.year=1993\n",
      "Gene: publications.abstractText='In Drosophila, two TGF-beta growth factors, dpp and screw, function synergistically to subdivide the dorsal ectoderm into two embryonic tissues, the amnioserosa and dorsal epidermis. Previous studies have shown that peak dpp activity is required for the localized expression of zerknüllt (zen), which encodes a homeodomain transcription factor. We present evidence that zen directly activates the amnioserosa-specific expression of a downstream target gene, Race (Related to angiotensin converting enzyme). A 533 bp enhancer from the Race promoter region is shown to mediate selective expression in the amnioserosa, as well as the anterior and posterior midgut rudiments. This enhancer contains three zen protein binding sites, and mutations in these sites virtually abolish the expression of an otherwise normal Race-lacZ fusion gene in the amnioserosa, but not in the gut. Genetic epistasis experiments suggest that zen is not the sole activator of Race, although a hyperactivated form of zen (a zen-VP16 fusion protein) can partially complement reduced levels of dpp activity. These results suggest that dpp regulates multiple transcription factors, which function synergistically to specify the amnioserosa.' publications.doi=None publications.firstAuthor='Rusch J' publications.id=1004030 publications.issue='2' publications.journal='Development' publications.month='Jan' publications.pages='303-11' publications.pubMedId='9053307' publications.title='Regulation of a dpp target gene in the Drosophila embryo.' publications.volume='124' publications.year=1997\n",
      "Gene: publications.abstractText='In a developing Drosophila melanogaster embryo, mRNAs have a maternal origin, a zygotic origin, or both. During the maternal-zygotic transition, maternal products are degraded and gene expression comes under the control of the zygotic genome. To interrogate the function of mRNAs that are both maternally and zygotically expressed, it is common to examine the embryonic phenotypes derived from female germline mosaics. Recently, the development of RNAi vectors based on short hairpin RNAs (shRNAs) effective during oogenesis has provided an alternative to producing germline clones. Here, we evaluate the efficacies of: (1) maternally loaded shRNAs to knockdown zygotic transcripts and (2) maternally loaded Gal4 protein to drive zygotic shRNA expression. We show that, while Gal4-driven shRNAs in the female germline very effectively generate phenotypes for genes expressed maternally, maternally loaded shRNAs are not very effective at generating phenotypes for early zygotic genes. However, maternally loaded Gal4 protein is very efficient at generating phenotypes for zygotic genes expressed during mid-embryogenesis. We apply this powerful and simple method to unravel the embryonic functions of a number of pleiotropic genes.' publications.doi='10.1534/genetics.112.144915' publications.firstAuthor='Staller Max V' publications.id=1007903 publications.issue='1' publications.journal='Genetics' publications.month='Jan' publications.pages='51-61' publications.pubMedId='23105012' publications.title='Depleting gene activities in early Drosophila embryos with the \"maternal-Gal4-shRNA\"system.' publications.volume='193' publications.year=2013\n",
      "Gene: publications.abstractText='In a screen based on a rough eye phenotype caused by a dominant negative form of the BEAF-32A and BEAF-32B insulator proteins, we previously identified 17 proteins that genetically interact with BEAF. Eleven of these are developmental transcription factors, seven of which are encoded by the Antennapedia complex (ANT-C). While investigating potential reasons for the genetic interactions, we obtained evidence that BEAF plays a role in the regulation of genes in the ANT-C. BEAF does not localize near the transcription start sites of any genes in the ANT-C, indicating that BEAF does not locally affect regulation of these genes. Although BEAF affects chromatin structure or dynamics, we also found no evidence for a general change in binding to polytene chromosomes in the absence of BEAF. However, because we were unable to detect proteins encoded by ANT-C genes in salivary glands, the DREF and MLE proteins were used as proxies to examine binding. This does not rule out limited effects at particular binding sites or the possibility that BEAF might directly interact with certain transcription factors to affect their binding. In contrast, the embryonic expression levels and patterns of four examined ANT-C genes were altered (bcd, Dfd, ftz, pb). A control gene, Dref, was not affected. A full understanding of the regulation of ANT-C genes during development will have to take the role of BEAF into account.' publications.doi='10.1007/s00438-010-0591-y' publications.firstAuthor='Roy Swarnava' publications.id=1005511 publications.issue='2' publications.journal='Mol. Genet. Genomics' publications.month='Feb' publications.pages='113-23' publications.pubMedId='21132442' publications.title='Lack of the Drosophila BEAF insulator proteins alters regulation of genes in the Antennapedia complex.' publications.volume='285' publications.year=2011\n",
      "Gene: publications.abstractText='In all animals, the initial events of embryogenesis are controlled by maternal gene products that are deposited into the developing oocyte. At some point after fertilization, control of embryogenesis is transferred to the zygotic genome in a process called the maternal-to-zygotic transition. During this time, many maternal RNAs are degraded and transcription of zygotic RNAs ensues. There is a long-standing question as to which factors regulate these events. The recent findings that microRNAs and Smaug mediate maternal transcript degradation have shed new light on this aspect of the problem. However, the transcription factor(s) that activate the zygotic genome remain elusive. The discovery that many of the early transcribed genes in Drosophila share a cis-regulatory heptamer motif, CAGGTAG and related sequences, collectively referred to as TAGteam sites raised the possibility that a dedicated transcription factor could interact with these sites to activate transcription. Here we report that the zinc-finger protein Zelda (Zld; Zinc-finger early Drosophila activator) binds specifically to these sites and is capable of activating transcription in transient transfection assays. Mutant embryos lacking zld are defective in cellular blastoderm formation, and fail to activate many genes essential for cellularization, sex determination and pattern formation. Global expression profiling confirmed that Zld has an important role in the activation of the early zygotic genome and suggests that Zld may also regulate maternal RNA degradation during the maternal-to-zygotic transition.' publications.doi='10.1038/nature07388' publications.firstAuthor='Liang Hsiao-Lan' publications.id=1007807 publications.issue='7220' publications.journal='Nature' publications.month='Nov' publications.pages='400-3' publications.pubMedId='18931655' publications.title='The zinc-finger protein Zelda is a key activator of the early zygotic genome in Drosophila.' publications.volume='456' publications.year=2008\n",
      "Gene: publications.abstractText='In many developmental contexts, a locally produced morphogen specifies positional information by forming a concentration gradient over a field of cells. However, during embryonic dorsal-ventral patterning in Drosophila, two members of the bone morphogenetic protein (BMP) family, Decapentaplegic (Dpp) and Screw (Scw), are broadly transcribed but promote receptor-mediated signalling in a restricted subset of expressing cells. Here we use a novel immunostaining protocol to visualize receptor-bound BMPs and show that both proteins become localized to a sharp stripe of dorsal cells. We demonstrate that proper BMP localization involves two distinct processes. First, Dpp undergoes directed, long-range extracellular transport. Scw also undergoes long-range movement, but can do so independently of Dpp transport. Second, an intracellular positive feedback circuit promotes future ligand binding as a function of previous signalling strength. These data elicit a model in which extracellular Dpp transport initially creates a shallow gradient of BMP binding that is acted on by positive intracellular feedback to produce two stable states of BMP-receptor interactions, a spatial bistability in which BMP binding and signalling capabilities are high in dorsal-most cells and low in lateral cells.' publications.doi='10.1038/nature03318' publications.firstAuthor='Wang Yu-Chiun' publications.id=1007763 publications.issue='7030' publications.journal='Nature' publications.month='Mar' publications.pages='229-34' publications.pubMedId='15759004' publications.title='Spatial bistability of Dpp-receptor interactions during Drosophila dorsal-ventral patterning.' publications.volume='434' publications.year=2005\n",
      "Gene: publications.abstractText='In many organisms, transcription of the zygotic genome begins during the maternal-to-zygotic transition (MZT), which is characterized by a dramatic increase in global transcriptional activities and coincides with embryonic stem cell differentiation. In Drosophila, it has been shown that maternal morphogen gradients and ubiquitously distributed general transcription factors may cooperate to upregulate zygotic genes that are essential for pattern formation in the early embryo. Here, we show that Drosophila STAT (STAT92E) functions as a general transcription factor that, together with the transcription factor Zelda, induces transcription of a large number of early-transcribed zygotic genes during the MZT. STAT92E is present in the early embryo as a maternal product and is active around the MZT. DNA-binding motifs for STAT and Zelda are highly enriched in promoters of early zygotic genes but not in housekeeping genes. Loss of Stat92E in the early embryo, similarly to loss of zelda, preferentially down-regulates early zygotic genes important for pattern formation. We further show that STAT92E and Zelda synergistically regulate transcription. We conclude that STAT92E, in conjunction with Zelda, plays an important role in transcription of the zygotic genome at the onset of embryonic development.' publications.doi='10.1371/journal.pgen.1002086' publications.firstAuthor='Tsurumi Amy' publications.id=1007702 publications.issue='5' publications.journal='PLoS Genet.' publications.month='May' publications.pages='e1002086' publications.pubMedId='21637778' publications.title='STAT is an essential activator of the zygotic genome in the early Drosophila embryo.' publications.volume='7' publications.year=2011\n",
      "Gene: publications.abstractText='In past years, much attention has focused on the gene networks that regulate early developmental processes, but less attention has been paid to how multiple networks and processes are temporally coordinated. Recently the discovery of the transcriptional activator Zelda (Zld), which binds to CAGGTAG and related sequences present in the enhancers of many early-activated genes in Drosophila, hinted at a mechanism for how batteries of genes could be simultaneously activated. Here we use genome-wide binding and expression assays to identify Zld target genes in the early embryo with the goal of unraveling the gene circuitry regulated by Zld. We found that Zld binds to genes involved in early developmental processes such as cellularization, sex determination, neurogenesis, and pattern formation. In the absence of Zld, many target genes failed to be activated, while others, particularly the patterning genes, exhibited delayed transcriptional activation, some of which also showed weak and/or sporadic expression. These effects disrupted the normal sequence of patterning-gene interactions and resulted in highly altered spatial expression patterns, demonstrating the significance of a timing mechanism in early development. In addition, we observed prevalent overlap between Zld-bound regions and genomic \"hotspot\"regions, which are bound by many developmental transcription factors, especially the patterning factors. This, along with the finding that the most over-represented motif in hotspots, CAGGTA, is the Zld binding site, implicates Zld in promoting hotspot formation. We propose that Zld promotes timely and robust transcriptional activation of early-gene networks so that developmental events are coordinated and cell fates are established properly in the cellular blastoderm embryo.' publications.doi='10.1371/journal.pgen.1002339' publications.firstAuthor='Nien Chung-Yi' publications.id=1007816 publications.issue='10' publications.journal='PLoS Genet.' publications.month='Oct' publications.pages='e1002339' publications.pubMedId='22028675' publications.title='Temporal coordination of gene networks by Zelda in the early Drosophila embryo.' publications.volume='7' publications.year=2011\n",
      "Gene: publications.abstractText='In situ hybridization is an important technique for measuring the spatial expression patterns of mRNA in cells, tissues, and whole animals. However, mRNA levels cannot be compared across experiments using typical protocols. Here we present a semi-quantitative method to compare mRNA levels of a gene across multiple samples. This method yields an estimate of the error in the measurement to allow statistical comparison. Our method uses a typical in situ hybridization protocol to stain for a target gene and an internal standard, which we refer to as a co-stain. As a proof of concept, we apply this method to multiple lines of transgenic Drosophila embryos, harboring constructs that express reporter genes to different levels. We generated this test set by mutating enhancer sequences to contain different numbers of binding sites for Zelda, a transcriptional activator. We demonstrate that using a co-stain with in situ hybridization is an effective method to compare mRNA levels across samples. This method requires only minor modifications to existing in situ hybridization protocols and uses straightforward analysis techniques. This strategy can be broadly applied to detect quantitative, spatially resolved changes in mRNA levels.' publications.doi='10.1016/j.ymeth.2014.01.003' publications.firstAuthor='Wunderlich Zeba' publications.id=1007778 publications.issue='1' publications.journal='Methods' publications.month='Jun' publications.pages='233-41' publications.pubMedId='24434507' publications.title='Comparing mRNA levels using in situ hybridization of a target gene and co-stain.' publications.volume='68' publications.year=2014\n",
      "Gene: publications.abstractText='In the long-germ insect Drosophila melanogaster dorsoventral polarity is induced by localized Toll-receptor activation which leads to the formation of a nuclear gradient of the rel/ NF-kappaB protein Dorsal. Peak levels of nuclear Dorsal are found in a ventral stripe spanning the entire length of the blastoderm embryo allowing all segments and their dorsoventral subdivisions to be synchronously specified before gastrulation. We show that a nuclear Dorsal protein gradient of similar anteroposterior extension exists in the short-germ beetle, Tribolium castaneum, which forms most segments from a posterior growth zone after gastrulation. In contrast to Drosophila, (i) nuclear accumulation is first uniform and then becomes progressively restricted to a narrow ventral stripe, (ii) gradient refinement is accompanied by changes in the zygotic expression of the Tribolium Toll-receptor suggesting feedback regulation and, (iii) the gradient only transiently overlaps with the expression of a potential target, the Tribolium twist homolog, and does not repress Tribolium decapentaplegic. No nuclear Dorsal is seen in the cells of the growth zone of Tribolium embryos, indicating that here dorsoventral patterning occurs by a different mechanism. However, Dorsal is up-regulated and transiently forms a nuclear gradient in the serosa, a protective extraembryonic cell layer ultimately covering the whole embryo.' publications.doi=None publications.firstAuthor='Chen G' publications.id=1007690 publications.issue='23' publications.journal='Development' publications.month='Dec' publications.pages='5145-56' publications.pubMedId='11060240' publications.title='The maternal NF-kappaB/dorsal gradient of Tribolium castaneum: dynamics of early dorsoventral patterning in a short-germ beetle.' publications.volume='127' publications.year=2000\n",
      "Gene: publications.abstractText='Infection results in the rapid activation of immunity genes in the Drosophila fat body. Two classes of transcription factors have been implicated in this process: the REL-containing proteins, Dorsal, Dif, and Relish, and the GATA factor Serpent. Here we present evidence that REL-GATA synergy plays apervasive role in the immune response. SELEX assays identified consensus binding sites that permitted the characterization of several immunity regulatory DNAs. The distribution of REL and GATA sites within these DNAs suggests that most or all fat-specific immunity genes contain a common organization of regulatory elements: closely linked REL and GATA binding sites positioned in the same orientation and located near the transcription start site. Aspects of this \"regulatory code\"are essential for the immune response. These results suggest that immunity regulatory DNAs contain constrained organizational features, which may be a general property of eukaryotic enhancers.' publications.doi=None publications.firstAuthor='Senger Kate' publications.id=1007774 publications.issue='1' publications.journal='Mol. Cell' publications.month='Jan' publications.pages='19-32' publications.pubMedId='14731391' publications.title='Immunity regulatory DNAs share common organizational features in Drosophila.' publications.volume='13' publications.year=2004\n",
      "Gene: publications.abstractText='It is becoming increasingly clear that transcriptional repression is at least as important as transcriptional activation for establishing cell-type specific patterns of gene expression during embryogenesis. Recent studies in Drosophila suggest that repressors fall into two categories, short-range and long-range repressors. The former permit enhancer autonomy in modular promoters, whereas the latter function in a dominant fashion to silence multiple enhancers.' publications.doi=None publications.firstAuthor='Gray S' publications.id=1007724 publications.issue='3' publications.journal='Curr. Opin. Cell Biol.' publications.month='Jun' publications.pages='358-64' publications.pubMedId='8743887' publications.title='Transcriptional repression in development.' publications.volume='8' publications.year=1996\n",
      "Gene: publications.abstractText='It is expected that genes that are expressed early in development and have a complex expression pattern are under strong purifying selection and thus evolve slowly. Hox genes fulfill these criteria and thus, should have a low evolutionary rate. However, some observations point to a completely different scenario. Hox genes are usually highly conserved inside the homeobox, but very variable outside it. We have measured the rates of nucleotide divergence and indel fixation of three Hox genes, labial (lab), proboscipedia (pb) and abdominal-A (abd-A), and compared them with those of three genes derived by duplication from Hox3, bicoid (bcd), zerknüllt (zen) and zerknüllt-related (zen2), and 15 non-Hox genes in sets of orthologous sequences of three species of the genus Drosophila. These rates were compared to test the hypothesis that Hox genes evolve slowly. Our results show that the evolutionary rate of Hox genes is higher than that of non-Hox genes when both amino acid differences and indels are taken into account: 43.39% of the amino acid sequence is altered in Hox genes, versus 30.97% in non-Hox genes and 64.73% in Hox-derived genes. Microsatellites scattered along the coding sequence of Hox genes explain partially, but not fully, their fast sequence evolution. These results show that Hox genes have a higher evolutionary dynamics than other developmental genes, and emphasize the need to take into account indels in addition to nucleotide substitutions in order to accurately estimate evolutionary rates.' publications.doi='10.1186/1471-2148-6-106' publications.firstAuthor='Casillas Sònia' publications.id=1007632 publications.issue=None publications.journal='BMC Evol. Biol.' publications.month=None publications.pages='106' publications.pubMedId='17163987' publications.title='Fast sequence evolution of Hox and Hox-derived genes in the genus Drosophila.' publications.volume='6' publications.year=2006\n",
      "Gene: publications.abstractText='Like many DNA binding transcription factors, the Drosophila morphogen encoded by dorsal can both stimulate and repress promoter activity. In particular, this factor activates twist and represses zerknüllt on the ventral side of the early embryo. We find that when multiple copies of a dorsal binding site from the zerknüllt ventral repressor element are fused to a heterologous basal promoter, the resulting construct is activated by dorsal to give a ventral specific expression pattern. Thus, the ability of a dorsal binding site to mediate repression rather than activation is not an intrinsic property of the site, but depends upon its context. We also show that a hybrid promoter containing both the zerknüllt ventral repressor element and the twist ventral activator region is not ventrally active in the early embryo, demonstrating that repression is dominant over activation. Thus, the default mode of action of the dorsal protein is transcriptional activation. Additional factors may modify dorsal activity to bring about repression.' publications.doi=None publications.firstAuthor='Pan D' publications.id=1007841 publications.issue='5' publications.journal='EMBO J.' publications.month='May' publications.pages='1837-42' publications.pubMedId='1582412' publications.title='The same dorsal binding site mediates both activation and repression in a context-dependent manner.' publications.volume='11' publications.year=1992\n",
      "Gene: publications.abstractText='Major advances have been made in understanding the evolution of transcriptional regulation using microevolutionary and macroevolutionary experimental approaches. The roles of stabilising selection and compensatory changes in an enhancer region have been elucidated in Drosophila. The molecular dynamics of regulatory alleles have been studied in plants. Evidence is accumulating for the involvement of regulatory evolution in morphological changes between closely related species, as well as in major changes of body plans.' publications.doi=None publications.firstAuthor='Tautz D' publications.id=1007716 publications.issue='5' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Oct' publications.pages='575-9' publications.pubMedId='10980438' publications.title='Evolution of transcriptional regulation.' publications.volume='10' publications.year=2000\n",
      "Gene: publications.abstractText=\"Many proteins of the Rel family can act as both transcriptional activators and repressors. However, mechanism that discerns the 'activator/repressor' functions of Rel-proteins such as Dorsal (Drosophila homologue of mammalian NFκB) is not understood. Using genomic, biophysical and biochemical approaches, we demonstrate that the underlying principle of this functional specificity lies in the 'sequence-encoded structure' of the κB-DNA. We show that Dorsal-binding motifs exist in distinct activator and repressor conformations. Molecular dynamics of DNA-Dorsal complexes revealed that repressor κB-motifs typically have A-tract and flexible conformation that facilitates interaction with co-repressors. Deformable structure of repressor motifs, is due to changes in the hydrogen bonding in A:T pair in the 'A-tract' core. The sixth nucleotide in the nonameric κB-motif, 'A' (A(6)) in the repressor motifs and 'T' (T(6)) in the activator motifs, is critical to confer this functional specificity as A(6)\\u2009→\\u2009T(6) mutation transformed flexible repressor conformation into a rigid activator conformation. These results highlight that 'sequence encoded κB DNA-geometry' regulates gene expression by exerting allosteric effect on binding of Rel proteins which in turn regulates interaction with co-regulators. Further, we identified and characterized putative repressor motifs in Dl-target genes, which can potentially aid in functional annotation of Dorsal gene regulatory network.\" publications.doi='10.1093/nar/gkr672' publications.firstAuthor='Mrinal Nirotpal' publications.id=1007717 publications.issue='22' publications.journal='Nucleic Acids Res.' publications.month='Dec' publications.pages='9574-91' publications.pubMedId='21890896' publications.title='Role of sequence encoded κB DNA geometry in gene regulation by Dorsal.' publications.volume='39' publications.year=2011\n",
      "Gene: publications.abstractText='Massive zygotic transcription begins in many organisms during the midblastula transition when the cell cycle of the dividing egg slows down. A few genes are transcribed before this stage but how this differential activation is accomplished is still an open question. We have performed ChIP-seq experiments on tightly staged Drosophila embryos and show that massive recruitment of RNA polymerase II (Pol II) with widespread pausing occurs de novo during the midblastula transition. However, ∼100 genes are strongly occupied by Pol II before this timepoint and most of them do not show Pol II pausing, consistent with a requirement for rapid transcription during the fast nuclear cycles. This global change in Pol II pausing correlates with distinct core promoter elements and associates a TATA-enriched promoter with the rapid early transcription. This suggests that promoters are differentially used during the zygotic genome activation, presumably because they have distinct dynamic properties. DOI:http://dx.doi.org/10.7554/eLife.00861.001.' publications.doi='10.7554/eLife.00861' publications.firstAuthor='Chen Kai' publications.id=1003294 publications.issue=None publications.journal='Elife' publications.month=None publications.pages='e00861' publications.pubMedId='23951546' publications.title='A global change in RNA polymerase II pausing during the Drosophila midblastula transition.' publications.volume='2' publications.year=2013\n",
      "Gene: publications.abstractText=\"Maternal genes involved in dorsoventral (D/V) patterning of the Drosophila embryo interact to establish a stable nuclear concentration gradient of the Dorsal protein which acts as the morphogen along this axis. This protein belongs to the rel proto-oncogene and NF-KB transcriptional factor family and acts by controlling zygotic gene expression. In the ventral part of the embryo, dorsal specifically activates transcription of the gene twist and ventrally and laterally dorsal represses the expression of zerknüllt, a gene involved in the formation of dorsal derivatives. The extent of dorsal action is closely related to the affinity and the number of dorsal response elements present in these zygotic gene promoters. twist is one of the first zygotic genes necessary for mesoderm formation. It codes for a 'b-HLH' DNA-binding protein which can dimerize and bind to DNA in vitro and to polytene chromosomes in vivo. In addition, in cultured cells twist has been shown to be a transcriptional activator. Thus, the first events of embryonic development along the D/V axis are controlled at the transcriptional level.\" publications.doi=None publications.firstAuthor='Thisse C' publications.id=1007762 publications.issue=None publications.journal='Dev. Suppl.' publications.month=None publications.pages='173-81' publications.pubMedId='1299363' publications.title='Dorsoventral development of the Drosophila embryo is controlled by a cascade of transcriptional regulators.' publications.volume=None publications.year=1992\n",
      "Gene: publications.abstractText='Maternally contributed mRNAs and proteins control the initial stages of development following fertilization. During this time, most of the zygotic genome remains transcriptionally silent. The initiation of widespread zygotic transcription is coordinated with the degradation of maternally provided mRNAs at the maternal-to-zygotic transition (MZT). While most of the genome is silenced prior to the MZT, a small subset of zygotic genes essential for the future development of the organism is transcribed. Previous work in our laboratory and others identified the TAGteam element, a set of related heptameric DNA-sequences in the promoters of many early-expressed Drosophila genes required to drive their unusually early transcription. To understand how this unique subset of genes is regulated, we identified a TAGteam-binding factor Grainyhead (Grh). We demonstrated that Grh and the previously characterized transcriptional activator Zelda (Zld) bind to different TAGteam sequences with varying affinities, and that Grh competes with Zld for TAGteam occupancy. Moreover, overexpression of Grh in the early embryo causes defects in cell division, phenocopying Zld depletion. Our findings indicate that during early embryonic development the precise timing of gene expression is regulated by both the sequence of the TAGteam elements in the promoter and the relative levels of the transcription factors Grh and Zld.' publications.doi='10.1016/j.ydbio.2010.06.026' publications.firstAuthor='Harrison Melissa M' publications.id=1007813 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Sep' publications.pages='248-55' publications.pubMedId='20599892' publications.title='Grainyhead and Zelda compete for binding to the promoters of the earliest-expressed Drosophila genes.' publications.volume='345' publications.year=2010\n",
      "Gene: publications.abstractText='Members of the Eyes absent (Eya) protein family play important roles in tissue specification and patterning by serving as both transcriptional activators and protein tyrosine phosphatases. These activities are often carried out in the context of complexes containing members of the Six and/or Dach families of DNA binding proteins. eyes absent, the founding member of the Eya family is expressed dynamically within several embryonic, larval, and adult tissues of the fruit fly, Drosophila melanogaster. Loss-of-function mutations are known to result in disruptions of the embryonic head and central nervous system as well as the adult brain and visual system, including the compound eyes. In an effort to understand how eya is regulated during development, we have carried out a genetic screen designed to identify genes that lie upstream of eya and govern its expression. We have identified a large number of putative regulators, including members of several signaling pathways. Of particular interest is the identification of both yan/anterior open and pointed, two members of the EGF Receptor (EGFR) signaling cascade. The EGFR pathway is known to regulate the activity of Eya through phosphorylation via MAPK. Our findings suggest that this pathway is also used to influence eya transcriptional levels. Together these mechanisms provide a route for greater precision in regulating a factor that is critical for the formation of a wide range of diverse tissues.' publications.doi='10.1534/genetics.109.110122' publications.firstAuthor='Salzer Claire L' publications.id=1004677 publications.issue='1' publications.journal='Genetics' publications.month='Jan' publications.pages='185-97' publications.pubMedId='19884307' publications.title='The retinal determination gene eyes absent is regulated by the EGF receptor pathway throughout development in Drosophila.' publications.volume='184' publications.year=2010\n",
      "Gene: publications.abstractText='Metazoan genomes contain vast tracts of cis-regulatory DNA that have been identified typically through tedious functional assays. As a result, it has not been possible to uncover a cis-regulatory code that links primary DNA sequences to gene expression patterns. In an initial effort to determine whether coordinately regulated genes share a common \"grammar,\"we have examined the distribution of Dorsal recognition sequences in the Drosophila genome. Dorsal is one of the best-characterized sequence-specific transcription factors in Drosophila. The homeobox gene zerknullt (zen) is repressed directly by Dorsal, and this repression is mediated by a 600-bp silencer, the ventral repression element (VRE), which contains four optimal Dorsal binding sites. The arrangement and sequence of the Dorsal recognition sequences in the VRE were used to develop a computational algorithm to search the Drosophila genome for clusters of optimal Dorsal binding sites. There are 15 regions in the genome that contain three or more optimal sites within a span of 400 bp or less. Three of these regions are associated with known Dorsal target genes: sog, zen, and Brinker. The Dorsal binding cluster in sog is shown to mediate lateral stripes of gene expression in response to low levels of the Dorsal gradient. Two of the remaining 12 clusters are shown to be associated with genes that exhibit asymmetric patterns of expression across the dorsoventral axis. These results suggest that bioinformatics can be used to identify novel target genes and associated regulatory DNAs in a gene network.' publications.doi='10.1073/pnas.012591199' publications.firstAuthor='Markstein Michele' publications.id=1007894 publications.issue='2' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Jan' publications.pages='763-8' publications.pubMedId='11752406' publications.title='Genome-wide analysis of clustered Dorsal binding sites identifies putative target genes in the Drosophila embryo.' publications.volume='99' publications.year=2002\n",
      "Gene: publications.abstractText='Microarray assays and bioinformatics methods have identified many of the genes and associated regulatory DNAs that control the early phases of gastrulation in Drosophila. The localized activities of these genes are coordinated by a nuclear gradient of the maternal regulatory factor, Dorsal, that is established shortly after fertilization. At least half of the Dorsal target genes encode transcription factors or signaling components that lead to the restricted activation of FGF, EGF, and TGF-beta signaling pathways in the mesoderm, neurogenic ectoderm and dorsal ectoderm, respectively. Recent work has yielded insights into how these signaling pathways control gastrulation, particularly in the context of the Dorsal-mediated gene regulation network' publications.doi='10.1016/j.gde.2004.07.004' publications.firstAuthor='Stathopoulos Angelike' publications.id=1007818 publications.issue='5' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Oct' publications.pages='477-84' publications.pubMedId='15380237' publications.title='Whole-genome analysis of Drosophila gastrulation.' publications.volume='14' publications.year=2004\n",
      "Gene: publications.abstractText='Modern developmental biology relies heavily on the analysis of embryonic gene expression patterns. Investigators manually inspect hundreds or thousands of expression patterns to identify those that are spatially similar and to ultimately infer potential gene interactions. However, the rapid accumulation of gene expression pattern data over the last two decades, facilitated by high-throughput techniques, has produced a need for the development of efficient approaches for direct comparison of images, rather than their textual descriptions, to identify spatially similar expression patterns. The effectiveness of the Binary Feature Vector (BFV) and Invariant Moment Vector (IMV) based digital representations of the gene expression patterns in finding biologically meaningful patterns was compared for a small (226 images) and a large (1819 images) dataset. For eachdataset, an ordered list of images, with respect to a query image, was generated to identify overlapping and similar gene expression patterns, in a manner comparable to what a developmental biologist might do. The results showed that the BFV representation consistently outperforms the IMV representation in finding biologically meaningful matches when spatial overlap of the gene expression pattern and the genes involved are considered. Furthermore, we explored the value of conducting image-content based searches in a dataset where individual expression components (or domains) of multi-domain expression patterns were also included separately. We found that this technique improves performance of both IMV and BFV based searches. We conclude that the BFV representation consistently produces a more extensive and better list of biologically useful patterns than the IMV representation. The high quality of results obtained scales well as the search database becomes larger, which encourages efforts to build automated image query and retrieval systems for spatial gene expression patterns.' publications.doi='10.1186/1471-2105-5-202' publications.firstAuthor='Gurunathan Rajalakshmi' publications.id=1007799 publications.issue=None publications.journal='BMC Bioinformatics' publications.month=None publications.pages='202' publications.pubMedId='15603586' publications.title='Identifying spatially similar gene expression patterns in early stage fruit fly embryo images: binary feature versus invariant moment digital representations.' publications.volume='5' publications.year=2004\n",
      "Gene: publications.abstractText=\"Molecular developmental studies of fly and mouse embryos have shown that the identity of individual body segments is controlled by a suite of homeobox-containing genes called the Hox cluster. To examine the conservation of this patterning mechanism in other segmented phyla, we here describe four Hox gene homologs isolated from glossiphoniid leeches of the genus Helobdella. Based on sequence similarity and phylogenetic analysis, the leech genes Lox7, Lox6, Lox20, and Lox5 are deemed to be orthologs of the Drosophila genes lab, Dfd, Scr, and Antp, respectively. Sequence similarities between Lox5 and Antp outside the homeodomain and phylogenetic reconstructions suggest that the Antennapedia family of Hox genes (as defined by Bürglin, 1994) had already expanded to include at least two discrete Antp and Ubx/abdA precursors prior to the annelid/arthropod divergence. In situ hybridization reveals that the four Lox genes described in this study are all expressed at high levels within the segmented portion of the central nervous system (CNS), with variable levels of expression in the segmental mesoderm. Little or no expression was seen in peripheral ectoderm or endoderm, or in the unsegmented head region (prostomium). Each Lox gene has a distinct anterior expression boundary within one of the four rostral segments, and the anterior-posterior (AP) order of these expression boundaries is identical to that reported for the orthologous Hox gene products in fly and mouse. This finding supports the idea that the process of AP axis differentiation is conserved among the higher metazoan phyla with respect to the regional expression of individual Hox genes along that axis. One unusual feature of leech Hox genes is the observation that some genes are only expressed during later development -- beginning at the time of terminal cell differentiation -- whereas others begin expression at a much earlier stage, and their RNA ceases to be detectable shortly after the onset of expression of the 'late' Hox genes. The functional significance of this temporal disparity is unknown, but it is noteworthy that only the two 'early' Hox genes display high levels of mesodermal expression.\" publications.doi='10.1006/dbio.1997.8689' publications.firstAuthor='Kourakis M J' publications.id=1007858 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Oct' publications.pages='284-300' publications.pubMedId='9344545' publications.title='Conserved anterior boundaries of Hox gene expression in the central nervous system of the leech Helobdella.' publications.volume='190' publications.year=1997\n",
      "Gene: publications.abstractText='Morphogen gradients determine a range of cell fates by specifying multiple transcriptional threshold responses. In the dorsal ectoderm of the Drosophila embryo, a BMP gradient is translated into an activated Smad transcription factor gradient, which elicits at least three threshold responses - high, intermediate and low. However, the mechanism underlying differential response to Dpp is poorly understood, due in part to the insufficient number of well-studied target genes. We analyzed the regulation of the C15 gene, which can be activated in cells containing intermediate levels of Dpp. We show that C15 expression requires both dpp and zen, thus forming a genetic feed-forward loop. The C15 regulatory element contains clusters of Smad- and Zen-binding sites in close proximity. Mutational analysis shows that the number of intact Smad- and Zen-binding sites is essential for the C15 transcriptional response, and that the spatial limits of C15 expression are established through a repression mechanism in the dorsolateral cells of the embryo. Thus, the combinatorial action of Smad and Zen activators bound to a number of adjacent sites, and competing negative cues allows for proper gene response to lower than peak levels of the Dpp morphogen.' publications.doi='10.1242/dev.02689' publications.firstAuthor='Lin Meng-chi' publications.id=1007703 publications.issue='24' publications.journal='Development' publications.month='Dec' publications.pages='4805-13' publications.pubMedId='17092951' publications.title='Threshold response of C15 to the Dpp gradient in Drosophila is established by the cumulative effect of Smad and Zen activators and negative cues.' publications.volume='133' publications.year=2006\n",
      "Gene: publications.abstractText='Morphogenesis of epithelial tissues relies on the precise developmental control of cell polarity and architecture. In the early Drosophila embryo, the primary epithelium forms during cellularisation, following a tightly controlled genetic programme where specific sets of genes are upregulated. Some of them, for example, control membrane invagination between the nuclei anchored at the apical surface of the syncytium. We used microarrays to describe the global programme of gene expression underlying cellularisation and identified distinct classes of upregulated genes during this process. Fifty-seven genes were then tested functionally by RNAi. We found six genes affecting various aspects of cellular architecture: membrane growth, organelle transport or organisation and junction assembly. We focus here on charleston (char), a new regulator of nuclear morphogenesis and of apical nuclear anchoring. In char-depleted embryos, the nuclei fail to maintain their elongated shape and, instead, become rounded. In addition, together with a disruption of the centrosome-nuclear envelope interaction, the nuclei lose their regular apical anchoring. These nuclear defects perturb the regular columnar organisation of epithelial cells in the embryo. Although microtubules are required for both nuclear morphogenesis and anchoring, char does not control microtubule organisation and association to the nuclear envelope. We show that Char is lipid anchored at the nuclear envelope by a farnesylation group, and localises at the inner nuclear membrane together with Lamin. Our data suggest that Char forms a scaffold that regulates nuclear architecture to constrain nuclei in tight columnar epithelial cells. The upregulation of Char during cellularisation and gastrulation reveals the existence of an as yet unknown developmental control of nuclear morphology and anchoring in embryonic epithelia.' publications.doi='10.1242/dev.02251' publications.firstAuthor='Pilot Fanny' publications.id=1003296 publications.issue='4' publications.journal='Development' publications.month='Feb' publications.pages='711-23' publications.pubMedId='16421189' publications.title='Developmental control of nuclear morphogenesis and anchoring by charleston, identified in a functional genomic screen of Drosophila cellularisation.' publications.volume='133' publications.year=2006\n",
      "Gene: publications.abstractText='Morphogenetic movements are closely regulated by the expression of developmental genes. Here I examine whether developmental gene expression can in turn be mechanically regulated by morphogenetic movements. I have analyzed the effects of mechanical stress on the expression of Twist, which is normally expressed only in the most ventral cells of the cellular blastoderm embryo under the control of the Dorsal morphogen gradient. At embryogenesis gastrulation (stage 7), Twist is also expressed in the anterior foregut and stomodeal primordia. Submitting the early Drosophila embryo to a transient 10% uniaxial lateral deformation induces the ectopic expression of Twist around the entire dorsal-ventral axis and results in the ventralization of the embryo. This induction is independent of the Dorsal gradient and is triggered by mechanically induced Armadillo nuclear translocation. I also show that Twist is not expressed in the anterior foregut and stomodeal primordia at stage 7 in mutants that block the morphogenetic movement of germ-band extension. Because I can rescue the mutants with gentle compression of these cells, my interpretation is that the stomodeal-cell compression normally caused by the germ-band extension induces the expression of Twist. Correspondingly, laser ablation of dorsal cells in wild-type embryos relaxes stomodeal cell compression and reduces Twist expression in the stomodeal primordium. I also demonstrate that the induction of Twist in these cells depends on the nuclear translocation of Armadillo. I propose that anterior-gut formation is mechanically induced by the movement of germ-band extension through the induction of Twist expression in stomodeal cells.' publications.doi=None publications.firstAuthor='Farge Emmanuel' publications.id=1007884 publications.issue='16' publications.journal='Curr. Biol.' publications.month='Aug' publications.pages='1365-77' publications.pubMedId='12932320' publications.title='Mechanical induction of Twist in the Drosophila foregut/stomodeal primordium.' publications.volume='13' publications.year=2003\n",
      "Gene: publications.abstractText='Morphogens are secreted signaling molecules that form concentration gradients and control cell fate in developing tissues. During development, it is essential that morphogen range is strictly regulated in order for correct cell type specification to occur. One of the best characterized morphogens is Drosophila Decapentaplegic (Dpp), a BMP signaling molecule that patterns the dorsal ectoderm of the embryo by activating the Mad and Medea (Med) transcription factors. We demonstrate that there is a spatial and temporal expansion of the expression patterns of Dpp target genes in SUMO pathway mutant embryos. We identify Med as the primary SUMOylation target in the Dpp pathway, and show that failure to SUMOylate Med leads to the increased Dpp signaling range observed in the SUMO pathway mutant embryos. Med is SUMO modified in the nucleus, and we provide evidence that SUMOylation triggers Med nuclear export. Hence, Med SUMOylation provides a mechanism by which nuclei can continue to monitor the presence of extracellular Dpp signal to activate target gene expression for an appropriate duration. Overall, our results identify an unusual strategy for regulating morphogen range that, rather than impacting on the morphogen itself, targets an intracellular transducer.' publications.doi='10.1101/gad.494808' publications.firstAuthor='Miles Wayne O' publications.id=1007900 publications.issue='18' publications.journal='Genes Dev.' publications.month='Sep' publications.pages='2578-90' publications.pubMedId='18794353' publications.title='Medea SUMOylation restricts the signaling range of the Dpp morphogen in the Drosophila embryo.' publications.volume='22' publications.year=2008\n",
      "Gene: publications.abstractText=\"Most cell-specific enhancers are thought to lack an inherent organization, with critical binding sites distributed in a more or less random fashion. However, there are examples of fixed arrangements of binding sites, such as helical phasing, that promote the formation of higher-order protein complexes on the enhancer DNA template. Here, we investigate the regulatory 'grammar' of nearly 100 characterized enhancers for developmental control genes active in the early Drosophila embryo. The conservation of grammar is examined in seven divergent Drosophila genomes. Linked binding sites are observed for particular combinations of binding motifs, including Bicoid-Bicoid, Hunchback-Hunchback, Bicoid-Dorsal, Bicoid-Caudal and Dorsal-Twist. Direct evidence is presented for the importance of Bicoid-Dorsal linkage in the integration of the anterior-posterior and dorsal-ventral patterning systems. Hunchback-Hunchback interactions help explain unresolved aspects of segmentation, including the differential regulation of the eve stripe 3 + 7 and stripe 4 + 6 enhancers. We also present evidence that there is an under-representation of nucleosome positioning sequences in many enhancers, raising the possibility for a subtle higher-order structure extending across certain enhancers. We conclude that grammar of gene control regions is pervasively used in the patterning of the Drosophila embryo.\" publications.doi='10.1093/nar/gkp619' publications.firstAuthor='Papatsenko Dmitri' publications.id=1007740 publications.issue='17' publications.journal='Nucleic Acids Res.' publications.month='Sep' publications.pages='5665-77' publications.pubMedId='19651877' publications.title='Organization of developmental enhancers in the Drosophila embryo.' publications.volume='37' publications.year=2009\n",
      "Gene: publications.abstractText='Most eukaryotic genomes include a substantial repeat-rich fraction termed heterochromatin, which is concentrated in centric and telomeric regions. The repetitive nature of heterochromatic sequence makes it difficult to assemble and analyze. To better understand the heterochromatic component of the Drosophila melanogaster genome, we characterized and annotated portions of a whole-genome shotgun sequence assembly. WGS3, an improved whole-genome shotgun assembly, includes 20.7 Mb of draft-quality sequence not represented in the Release 3 sequence spanning the euchromatin. We annotated this sequence using the methods employed in the re-annotation of the Release 3 euchromatic sequence. This analysis predicted 297 protein-coding genes and six non-protein-coding genes, including known heterochromatic genes, and regions of similarity to known transposable elements. Bacterial artificial chromosome (BAC)-based fluorescence in situ hybridization analysis was used to correlate the genomic sequence with the cytogenetic map in order to refine the genomic definition of the centric heterochromatin; on the basis of our cytological definition, the annotated Release 3 euchromatic sequence extends into the centric heterochromatin on each chromosome arm. Whole-genome shotgun assembly produced a reliable draft-quality sequence of a significant part of the Drosophila heterochromatin. Annotation of this sequence defined the intron-exon structures of 30 known protein-coding genes and 267 protein-coding gene models. The cytogenetic mapping suggests that an additional 150 predicted genes are located in heterochromatin at the base of the Release 3 euchromatic sequence. Our analysis suggests strategies for improving the sequence and annotation of the heterochromatic portions of the Drosophila and other complex genomes.' publications.doi=None publications.firstAuthor='Hoskins Roger A' publications.id=1058977 publications.issue='12' publications.journal='Genome Biol.' publications.month=None publications.pages='RESEARCH0085' publications.pubMedId='12537574' publications.title='Heterochromatic sequences in a Drosophila whole-genome shotgun assembly.' publications.volume='3' publications.year=2002\n",
      "Gene: publications.abstractText='Most of our knowledge about the mechanisms of segmentation in arthropods comes from work on Drosophila melanogaster. In recent years it has become clear that this mechanism is far from universal, and different arthropod groups have distinct modes of segmentation that operate through divergent genetic mechanisms. We review recent data from a range of arthropods, identifying which features of the D. melanogaster segmentation cascade are present in the different groups, and discuss the evolutionary implications of their conserved and divergent aspects. A model is emerging, although slowly, for the way that arthropod segmentation mechanisms have evolved.' publications.doi='10.1038/nrg1724' publications.firstAuthor='Peel Andrew D' publications.id=1007719 publications.issue='12' publications.journal='Nat. Rev. Genet.' publications.month='Dec' publications.pages='905-16' publications.pubMedId='16341071' publications.title='Arthropod segmentation: beyond the Drosophila paradigm.' publications.volume='6' publications.year=2005\n",
      "Gene: publications.abstractText='Mutations of the homeotic gene proboscipedia (pb) of Drosophila cause striking transformations of the adult mouthparts, to legs or antennae. We report here an analysis of the gene structure of pb. Coding sequences across a 34 kb interval yield, by alternative splicing, four identified mRNA forms which differ immediately upstream of the homeobox. As a consequence, the homeodomain is expected to reside in four different contexts in the predicted protein isoforms. Mammalian homologs of pb, human HOX-2H and murine Hox-2.8, were identified based on the similarities of their homeodomains (95% identity) and several other conserved motifs. Examination of a collection of pb mutant alleles with antisera directed against the N-terminal region, the center or the C-terminal region of the protein showed that, surprisingly, several partial loss-of-function pb alleles appear to generate partially functional proteins truncated at their C-termini. This suggests that a significant portion of the protein contributes quantitatively to pb function, but is partially dispensable. Finally, evolutionary considerations suggest that pb may be one of several ancient genes which preceded the process yielding the modern homeotic gene complexes.' publications.doi=None publications.firstAuthor='Cribbs D L' publications.id=1007706 publications.issue='4' publications.journal='EMBO J.' publications.month='Apr' publications.pages='1437-49' publications.pubMedId='1348688' publications.title='Structural complexity and evolutionary conservation of the Drosophila homeotic gene proboscipedia.' publications.volume='11' publications.year=1992\n",
      "Gene: publications.abstractText='NURF is a conserved higher eukaryotic ISWI-containing chromatin remodeling complex that catalyzes ATP-dependent nucleosome sliding. By sliding nucleosomes, NURF is able to alter chromatin dynamics to control transcription and genome organization. Previous biochemical and genetic analysis of the specificity-subunit of Drosophila NURF (Nurf301/Enhancer of Bithorax (E(bx)) has defined NURF as a critical regulator of homeotic, heat-shock and steroid-responsive gene transcription. It has been speculated that NURF controls pathway specific transcription by co-operating with sequence-specific transcription factors to remodel chromatin at dedicated enhancers. However, conclusive in vivo demonstration of this is lacking and precise regulatory elements targeted by NURF are poorly defined. To address this, we have generated a comprehensive map of in vivo NURF activity, using MNase-sequencing to determine at base pair resolution NURF target nucleosomes, and ChIP-sequencing to define sites of NURF recruitment. Our data show that, besides anticipated roles at enhancers, NURF interacts physically and functionally with the TRF2/DREF basal transcription factor to organize nucleosomes downstream of active promoters. Moreover, we detect NURF remodeling and recruitment at distal insulator sites, where NURF functionally interacts with and co-localizes with DREF and insulator proteins including CP190 to establish nucleosome-depleted domains. This insulator function of NURF is most apparent at subclasses of insulators that mark the boundaries of chromatin domains, where multiple insulator proteins co-associate. By visualizing the complete repertoire of in vivo NURF chromatin targets, our data provide new insights into how chromatin remodeling can control genome organization and regulatory interactions.' publications.doi='10.1371/journal.pgen.1005969' publications.firstAuthor='Kwon So Yeon' publications.id=1007830 publications.issue='4' publications.journal='PLoS Genet.' publications.month='Apr' publications.pages='e1005969' publications.pubMedId='27046080' publications.title='Genome-Wide Mapping Targets of the Metazoan Chromatin Remodeling Factor NURF Reveals Nucleosome Remodeling at Enhancers, Core Promoters and Gene Insulators.' publications.volume='12' publications.year=2016\n",
      "Gene: publications.abstractText='Network motifs provided a\"conceptual tool\"for understanding the functional principles of biological networks, but such motifs have primarily been used to consider static network structures. Static networks, however, cannot be used to reveal time- and region-specific traits of biological systems. To overcome this limitation, we proposed the concept of a \"spatiotemporal network motif,\"a spatiotemporal sequence of network motifs of sub-networks which are active only at specific time points and body parts. On the basis of this concept, we analyzed the developmental gene regulatory network of the Drosophila melanogaster embryo. We identified spatiotemporal network motifs and investigated their distribution pattern in time and space. As a result, we found how key developmental processes are temporally and spatially regulated by the gene network. In particular, we found that nested feedback loops appeared frequently throughout the entire developmental process. From mathematical simulations, we found that mutual inhibition in the nested feedback loops contributes to the formation of spatial expression patterns. Taken together, the proposed concept and the simulations can be used to unravel the design principle of developmental gene regulatory networks.' publications.doi='10.1186/1752-0509-6-31' publications.firstAuthor='Kim Man-Sun' publications.id=1007708 publications.issue=None publications.journal='BMC Syst Biol' publications.month=None publications.pages='31' publications.pubMedId='22548745' publications.title='Spatiotemporal network motif reveals the biological traits of developmental gene regulatory networks in Drosophila melanogaster.' publications.volume='6' publications.year=2012\n",
      "Gene: publications.abstractText='Networks of regulatory relations between transcription factors (TF) and their target genes (TG)- implemented through TF binding sites (TFBS)- are key features of biology. An idealized approach to solving such networks consists of starting from a consensus TFBS or a position weight matrix (PWM) to generate a high accuracy list of candidate TGs for biological validation. Developing and evaluating such approaches remains a formidable challenge in regulatory bioinformatics. We perform a benchmark study on 34 Drosophila TFs to assess existing TFBS and cis-regulatory module (CRM) detection methods, with a strong focus on the use of multiple genomes. Particularly, for CRM-modelling we investigatethe addition of orthologous sites to a known PWM to construct phyloPWMs and we assess the added value of phylogenentic footprinting to predict contextual motifs around known TFBSs. For CRM-prediction, we compare motif conservation with network-level conservation approaches across multiple genomes. Choosing the optimal training and scoring strategies strongly enhances the performance of TG prediction for more than half of the tested TFs. Finally, we analyse a 35(th) TF, namely Eyeless, and find a significant overlap between predicted TGs and candidate TGs identified by microarray expression studies. In summary we identify several ways to optimize TF-specific TG predictions, some of which can be applied to all TFs, and others that can be applied only to particular TFs. The ability to model known TF-TG relations, together with the use of multiple genomes, results in a significant step forward in solving the architecture of gene regulatory networks.' publications.doi='10.1371/journal.pone.0001115' publications.firstAuthor='Aerts Stein' publications.id=1007731 publications.issue='11' publications.journal='PLoS ONE' publications.month=None publications.pages='e1115' publications.pubMedId='17973026' publications.title='Fine-tuning enhancer models to predict transcriptional targets across multiple genomes.' publications.volume='2' publications.year=2007\n",
      "Gene: publications.abstractText='Neurotactin (NRT), a member of the cholinesterase-homologous protein family, is a heterophilic cell adhesion molecule that is required for proper axon guidance during Drosophila development. In this study, we identify amalgam (AMA), a member of the immunoglobulin superfamily, as a ligand for the NRT receptor. Using transfected Schneider 2 cells and embryonic primary cultures, we demonstrate that AMA is a secreted protein. Furthermore, AMA is necessary for NRT-expressing cells both to aggregate with themselves and to associate with embryonic primary culture cells. Aggregation assays performed with truncated NRT molecules reveal that the integrity of the cholinesterase-like extracellular domain was not required either for AMA binding or for adhesion, with only amino acids 347-482 of the extracellular domain being necessary for both activities. Moreover, the NRT cytoplasmic domain is required for NRT-mediated adhesion, although not for AMA binding. Using an ama-deficient stock, we find that ama function is not essential for viability. Pupae deficient for ama do exhibit defasciculation defects of the ocellar nerves similar to those found in nrt mutants.' publications.doi='10.1093/emboj/19.17.4463' publications.firstAuthor='Frémion F' publications.id=1007728 publications.issue='17' publications.journal='EMBO J.' publications.month='Sep' publications.pages='4463-72' publications.pubMedId='10970840' publications.title='Amalgam is a ligand for the transmembrane receptor neurotactin and is required for neurotactin-mediated cell adhesion and axon fasciculation in Drosophila.' publications.volume='19' publications.year=2000\n",
      "Gene: publications.abstractText='One of the main challenges in evolutionary biology is to identify the molecular changes that underlie phenotypic differences that are of evolutionary significance. Comparative studies of early development have shown that changes in the spatio-temporal use of regulatory genes, as well as changes in the specificity of regulatory proteins, are correlated with important differences in morphology between phylogenetically distant species. However, it is not known how such changes take place in natural populations, and whether they result from a single, or many small, additive events. Extending this approach to the study of development of closely related species promises to enrich this debate.' publications.doi='10.1038/nrg947' publications.firstAuthor='Simpson Pat' publications.id=1007819 publications.issue='12' publications.journal='Nat. Rev. Genet.' publications.month='Dec' publications.pages='907-17' publications.pubMedId='12459721' publications.title='Evolution of development in closely related species of flies and worms.' publications.volume='3' publications.year=2002\n",
      "Gene: publications.abstractText='One of the most important questions in biology is how transcription factors (TFs) and cofactors control enhancer function and thus gene expression. Enhancer activation usually requires combinations of several TFs, indicating that TFs function synergistically and combinatorially. However, while TF binding has been extensively studied, little is known about how combinations of TFs and cofactors control enhancer function once they are bound. It is typically unclear which TFs participate in combinatorial enhancer activation, whether different TFs form functionally distinct groups, or if certain TFs might substitute for each other in defined enhancer contexts. Here we assess the potential regulatory contributions of TFs and cofactors to combinatorial enhancer control with enhancer complementation assays. We recruited GAL4-DNA-binding-domain fusions of 812 Drosophila TFs and cofactors to 24 enhancer contexts and measured enhancer activities by 82,752 luciferase assays in S2 cells. Most factors were functional in at least one context, yet their contributions differed between contexts and varied from repression to activation (up to 289-fold) for individual factors. Based on functional similarities across contexts, we define 15 groups of TFs that differ in developmental functions and protein sequence features. Similar TFs can substitute for each other, enabling enhancer re-engineering by exchanging TF motifs, and TF-cofactor pairs cooperate during enhancer control and interact physically. Overall, we show that activators and repressors can have diverse regulatory functions that typically depend on the enhancer context. The systematic functional characterization of TFs and cofactors should further our understanding of combinatorial enhancer control and gene regulation.' publications.doi='10.1038/nature15545' publications.firstAuthor='Stampfel Gerald' publications.id=1000807 publications.issue='7580' publications.journal='Nature' publications.month='Dec' publications.pages='147-51' publications.pubMedId='26550828' publications.title='Transcriptional regulators form diverse groups with context-dependent regulatory functions.' publications.volume='528' publications.year=2015\n",
      "Gene: publications.abstractText='Pattern formation in Drosophila is initiated by a small set of asymmetrically distributed maternal transcription factors that act as graded morphogens along the anterior-posterior and the dorsal-ventral axes of the embryo. Recent progress in the field provides first insight into the molecular mechanisms by which long-range positional information in the egg causes a series of localized zygotic transcription factors to position the developmental fate along the blastoderm.' publications.doi=None publications.firstAuthor='Hoch M' publications.id=1002690 publications.issue='4' publications.journal='Curr. Opin. Genet. Dev.' publications.month='Aug' publications.pages='566-73' publications.pubMedId='8241768' publications.title='Transcriptional regulation and spatial patterning in Drosophila.' publications.volume='3' publications.year=1993\n",
      "Gene: publications.abstractText='Pattern formation in the developing embryo relies on key regulatory molecules, many of which are distributed in concentration gradients. For example, a gradient of BMP specifies cell fates along the dorsoventral axis in species ranging from flies to mammals. In Drosophila, a gradient of the BMP molecule Dpp gives rise to nested domains of target gene expression in the dorsal regionof the embryo; however, the mechanisms underlying the differential response are not well understood, partly owing to an insufficient number of well-studied targets. Here we analyze how the Dpp gradient regulates expression of pannier (pnr), a candidate low-level Dpp target gene. We predicted that the pnr enhancer would contain high-affinity binding sites for the Dpp effector Smad transcription factors, which would be occupied in the presence of low-level Dpp. Unexpectedly, the affinity of Smad sites in the pnr enhancer was similar to those in the Race enhancer, a high-level Dpp target gene, suggesting that the affinity threshold mechanism plays a minimal role in the regulation of pnr. Our results indicate that a mechanism involving a conserved bipartite motif that is predicted to bind a homeodomain factor in addition to Smads and the Brinker repressor, establishes the pnr expression domain. Furthermore, the pnr enhancer has a highly complex structure that integrates cues not only from the dorsoventral axis, but also from the anteroposterior and terminal patterning systems in the blastoderm embryo.' publications.doi='10.1242/dev.079772' publications.firstAuthor='Liang Hsiao-Lan' publications.id=1007710 publications.issue='11' publications.journal='Development' publications.month='Jun' publications.pages='1956-64' publications.pubMedId='22513375' publications.title='Response to the BMP gradient requires highly combinatorial inputs from multiple patterning systems in the Drosophila embryo.' publications.volume='139' publications.year=2012\n",
      "Gene: publications.abstractText='Pattern formation in the dorsal region of the Drosophila embryo depends on the activity of a small group of zygotically acting genes. dpp, a key gene in this group, encodes a TGF-beta-like product (Dpp) that has been proposed to function as a morphogen with peak levels of Dpp-specifying amnioserosa, the dorsal-most cell type, and lower Dpp levels specifying dorsal ectoderm. The short gastrulation gene also contributes to patterning the dorsal region, but unlike the other genes involved in this process, sog activity is only required in ventral cells. Genetic evidence indicates that sog functions to antagonize dpp activity. In this report we present further phenotypic characterization of sog mutant embryos in dorsal and lateral regions and describe the cloning of the sog locus. sog is expressed in a broad lateral stripe of cells that abuts the dorsal territory of dpp-expressing cells. sog is predicted to encode a protein with an internal signal sequence and a large extracellular domain containing four repeats of a novel motif defined by the spacing of 10 cysteine residues that is distantly related to domains present in thrombospondin and procollagen. We propose that one or more of these cysteine repeats can be liberated by proteolytic cleavage of the primary Sog protein. These putative soluble Sog peptides may then diffuse into the dorsal region to antagonize the activity of Dpp, leading to the subdivision of the dorsal territory into amnioserosa and dorsal ectoderm.' publications.doi=None publications.firstAuthor='Francois V' publications.id=1007736 publications.issue='21' publications.journal='Genes Dev.' publications.month='Nov' publications.pages='2602-16' publications.pubMedId='7958919' publications.title='Dorsal-ventral patterning of the Drosophila embryo depends on a putative negative growth factor encoded by the short gastrulation gene.' publications.volume='8' publications.year=1994\n",
      "Gene: publications.abstractText='Patterning of the Drosophila embryo requires not only the proper activation of determinants at specific times, but also their restriction to specific places. Recent studies on transcriptional repressors show how they delimit the gene expression patterns to ensure normal development.' publications.doi=None publications.firstAuthor='Ip Y T' publications.id=1007723 publications.issue='4' publications.journal='Curr. Biol.' publications.month='Apr' publications.pages='R216-8' publications.pubMedId='9162494' publications.title='Drosophila development. Delimiting patterns by repression.' publications.volume='7' publications.year=1997\n",
      "Gene: publications.abstractText='Patterning of the dorsal-ventral axis in the early Drosophila embryo depends on the nuclear distribution of the Dorsal transcription factor. Using live two-photon light-sheet microscopy, we quantified the nuclear Dorsal gradient in space and time and found that its amplitude and basal levels display oscillations throughout early embryonic development. These dynamics raise questions regarding how cells can reproducibly establish patterns of gene expression from a rapidly varying signal. We therefore quantified domains of Dorsal target genes, discovering their expression patterns are also dynamic. Computational modeling of this system reveals a correlation between Dorsal gradient dynamics and changes in target gene expression and suggests that these dynamics, together with time averaging of noise, results in the formation of graded gene expression borders in regions where the gradient is nearly flat. We propose that mRNA levels remain plastic during transient signaling events, allowing tissues to refine patterns in the face of genetic or environmental variation.' publications.doi='10.1016/j.devcel.2011.12.007' publications.firstAuthor='Reeves Gregory T' publications.id=1007697 publications.issue='3' publications.journal='Dev. Cell' publications.month='Mar' publications.pages='544-57' publications.pubMedId='22342544' publications.title='Dorsal-ventral gene expression in the Drosophila embryo reflects the dynamics and precision of the dorsal nuclear gradient.' publications.volume='22' publications.year=2012\n",
      "Gene: publications.abstractText='Patterning of the terminal regions of the Drosophila embryo is achieved by an exquisitely regulated signal that passes between the follicle cells of the ovary, and the developing embryo. This pathway, however, is missing or modified in other insects. Here we trace the evolution of this pathway by examining the origins and expression of its components. The three core components of this pathway: trunk, torso and torso-like have different evolutionary histories and have been assembled step-wise to form the canonical terminal patterning pathway of Drosophila and Tribolium. Trunk, torso and a gene unrelated to terminal patterning, prothoraciotrophic hormone (PTTH), show an intimately linked evolutionary history, with every holometabolous insect, except the honeybee, possessing both PTTH and torso genes. Trunk is more restricted in its phylogenetic distribution, present only in the Diptera and Tribolium and, surprisingly, in the chelicerate Ixodes scapularis, raising the possibility that trunk and torso evolved earlier than previously thought. In Drosophila torso-like restricts the activation of the terminal patterning pathway to the poles of the embryo. Torso-like evolved in the pan-crustacean lineage, but based on expression of components of the canonical terminal patterning system in the hemimetabolous insect Acyrthosiphon pisum and the holometabolous insect Apis mellifera, we find that the canonical terminal-patterning system is not active in these insects. We therefore propose that the ancestral function of torso-like is unrelated to terminal patterning and that torso-like has become co-opted into terminal patterning in the lineage leading to Coleoptera and Diptera. We also show that this co-option has not resulted in changes to the molecular function of this protein. Torso-like from the pea aphid, honeybee and Drosophila, despite being expressed in different patterns, are functionally equivalent. We propose that co-option of torso-like into restricting the activity of trunk and torso facilitated the final step in the evolution of this pathway; the capture of transcriptional control of target genes such as tailless and huckebein by this complex and novel patterning pathway.' publications.doi='10.1016/j.ydbio.2013.02.010' publications.firstAuthor='Duncan Elizabeth J' publications.id=1007833 publications.issue='1' publications.journal='Dev. Biol.' publications.month='May' publications.pages='245-61' publications.pubMedId='23438815' publications.title='Canonical terminal patterning is an evolutionary novelty.' publications.volume='377' publications.year=2013\n",
      "Gene: publications.abstractText='Phylostratigraphy is a method for dating the evolutionary emergence of a gene or gene family by identifying its homologs across the tree of life, typically by using BLAST searches. Applying this method to all genes in a species, or genomic phylostratigraphy, allows investigation of genome-wide patterns in new gene origination at different evolutionary times and thus has been extensively used. However, gene age estimation depends on the challenging task of detecting distant homologs via sequence similarity, which is expected to have differential accuracies for different genes. Here, we evaluate the accuracy of phylostratigraphy by realistic computer simulation with parameters estimated from genomic data, and investigate the impact of its error on findings of genome evolution. We show that 1) phylostratigraphy substantially underestimates gene age for a considerable fraction of genes, 2) the error is especially serious when the protein evolves rapidly, is short, and/or its most conserved block of sites is small, and 3) these errors create spurious nonuniform distributions of various gene properties among age groups, many of which cannot be predicted a priori. Given the high likelihood that conclusions about gene age are faulty, we advocate the use of realistic simulation to determine if observations from phylostratigraphy are explainable, at least qualitatively, by a null model of biased measurement, and in all cases, critical evaluation of results.' publications.doi='10.1093/molbev/msu286' publications.firstAuthor='Moyers Bryan A' publications.id=1000123 publications.issue='1' publications.journal='Mol. Biol. Evol.' publications.month='Jan' publications.pages='258-67' publications.pubMedId='25312911' publications.title='Phylostratigraphic bias creates spurious patterns of genome evolution.' publications.volume='32' publications.year=2015\n",
      "Gene: publications.abstractText='Polycomb-group (PcG) and Trithorax-group proteins together form a maintenance machinery that is responsible for stable heritable states of gene activity. While the best-studied target genes are the Hox genes of the Antennapedia and Bithorax complexes, a large number of key developmental genes are also Polycomb (Pc) targets, indicating a widespread role for this maintenance machinery in cell fate determination. We have studied the linkage between the binding of PcG proteins and the developmental regulation of gene expression using whole-genome mapping to identify sites bound by the PcG proteins, Pc and Pleiohomeotic (Pho), in the Drosophila embryo and in a more restricted tissue, the imaginal discs of the third thoracic segment. Our data provide support for the idea that Pho is a general component of the maintenance machinery, since the majority of Pc targets are also associated with Pho binding. We find, in general, considerable developmental stability of Pc and Pho binding at target genes and observe that Pc/Pho binding can be associated with both expressed and inactive genes. In particular, at the Hox complexes, both active and inactive genes have significant Pc and Pho binding. However, in comparison to inactive genes, the active Hox genes show reduced and altered binding profiles.During development, Pc target genes are not simply constantly associated with Pc/Pho binding, and we identify sets of genes with clear differential binding between embryo and imaginal disc. Using existing datasets, we show that for specific fate-determining genes of the haemocyte lineage, the active state is characterised by lack of Pc binding. Overall, our analysis suggests a dynamic relationship between Pc/Pho binding and gene transcription. Pc/Pho binding does not preclude transcription, but levels of Pc/Pho binding change during development, and loss of Pc/Pho binding can be associated with both stable gene activity and inactivity.' publications.doi='10.1371/journal.pgen.1000178' publications.firstAuthor='Kwong Camilla' publications.id=1007769 publications.issue='9' publications.journal='PLoS Genet.' publications.month=None publications.pages='e1000178' publications.pubMedId='18773083' publications.title='Stability and dynamics of polycomb target sites in Drosophila development.' publications.volume='4' publications.year=2008\n",
      "Gene: publications.abstractText='Population genetic theory predicts that maternal effect genes will evolve differently than genes expressed in both sexes because selection is only half as effective on autosomal genes expressed in one sex but not the other. Here, we use sequences of the tandem gene duplicates, bicoid (bcd) and zerknüllt (zen), to test the prediction that, with similar coefficients of purifying selection, a maternal effect gene evolves more rapidly than a zygotic gene because of this reduction in selective constraint. We find that the maternal effect gene, bcd, is evolving more rapidly than zygotically expressed, zen, providing the first direct confirmation of this prediction of maternal effect theory from molecular evidence. Our results extend current explanations for the accelerated rate of bcd evolution by providing an evolutionary mechanism, relaxed selective constraint, that allows bcd the evolutionary flexibility to escape the typical functional constraints of early developmental genes. We discuss general implications of our findings for the role of maternal effect genes in early developmental patterning.' publications.doi='10.1007/s10709-006-0031-4' publications.firstAuthor='Demuth Jeffery P' publications.id=1007758 publications.issue='1' publications.journal='Genetica' publications.month='Jan' publications.pages='37-43' publications.pubMedId='16955333' publications.title='Maternal expression increases the rate of bicoid evolution by relaxing selective constraint.' publications.volume='129' publications.year=2007\n",
      "Gene: publications.abstractText='Positional information in the dorsoventral axis of the Drosophila embryo is encoded by a BMP activity gradient formed by synergistic signaling between the BMP family members Decapentaplegic (DPP) and Screw (SCW). short gastrulation (sog), which is functionally homologous to Xenopus Chordin, is expressed in the ventrolateral regions of the embryo and has been shown to act as a local antagonist of BMP signaling. Here we demonstrate that SOG has a second function, which is to promote BMP signaling on the dorsal side of the embryo. We show that a weak, homozygous-viable sog mutant is enhanced to lethality by reduction in the activities of the Smad family members Mad or Medea, and that the lethality is caused by defects in the molecular specification and subsequent cellular differentiation of the dorsal-most cell type, the amnioserosa. While previous data had suggested that the negative function of SOG is directed against SCW, we present data that suggests that the positive activity of SOG is directed towards DPP. We demonstrate that Chordin shares the same apparent ligand specificity as does SOG, preferentially inhibiting SCW but not DPP activity. However, in Drosophila assays Chordin does not have the same capacity to elevate BMP signaling as does SOG, identifying a functional difference in the otherwise well conserved process of dorsoventral pattern formation in arthropods and chordates.' publications.doi=None publications.firstAuthor='Decotto E' publications.id=1007682 publications.issue='19' publications.journal='Development' publications.month='Oct' publications.pages='3831-41' publications.pubMedId='11585808' publications.title='A positive role for Short gastrulation in modulating BMP signaling during dorsoventral patterning in the Drosophila embryo.' publications.volume='128' publications.year=2001\n",
      "Gene: publications.abstractText='Previously, we mapped quantitative trait loci (QTL) affecting response to short-term selection for abdominal bristle number to seven suggestive regions that contain loci involved in bristle development and/or that have adult bristle number mutant phenotypes, and are thus candidates for bristle number QTL in natural populations. To test the hypothesis that the factors contributing to selection response genetically interact with these candidate loci, high and low chromosomes from selection lines were crossed to chromosomes containing wild-type or mutant alleles at the candidate loci, and the numbers of bristles were recorded in trans heterozygotes. Quantitative failure to complement, detected as a significant selection line*cross effect by analysis of variance, can be interpreted as evidence for allelism or epistasis between the factors on selected chromosomes and the candidate loci. Mutations at some candidate loci (bb, emc, h, Dl, Hairless) showed strong interactions with selected chromosomes, whereas others interacted weakly (ASC, abd, Scr) or not at all (N, mab, E(spl)). These results support the hypothesis that some candidate loci, initially identified through mutations of large effect on bristle number, either harbor or are close members in the same genetic pathway as variants that contribute to standing variation in bristle number.' publications.doi=None publications.firstAuthor='Long A D' publications.id=1007899 publications.issue='4' publications.journal='Genetics' publications.month='Dec' publications.pages='1497-510' publications.pubMedId='8978039' publications.title='Genetic interactions between naturally occurring alleles at quantitative trait loci and mutant alleles at candidate loci affecting bristle number in Drosophila melanogaster.' publications.volume='144' publications.year=1996\n",
      "Gene: publications.abstractText='Quantitative studies of embryogenesis require the ability to monitor pattern formation and morphogenesis in large numbers of embryos, at multiple time points and in diverse genetic backgrounds. We describe a simple approach that greatly facilitates these tasks for Drosophila melanogaster embryos, one of the most advanced models of developmental genetics. Based on passive hydrodynamics, we developed a microfluidic embryo-trap array that can be used to rapidly order and vertically orient hundreds of embryos. We describe the physical principles of the design and used this platform to quantitatively analyze multiple morphogen gradients in the dorsoventral patterning system. Our approach can also be used for live imaging and, with slight modifications, could be adapted for studies of pattern formation and morphogenesis in other model organisms.' publications.doi='10.1038/nmeth.1548' publications.firstAuthor='Chung Kwanghun' publications.id=1007787 publications.issue='2' publications.journal='Nat. Methods' publications.month='Feb' publications.pages='171-6' publications.pubMedId='21186361' publications.title='A microfluidic array for large-scale ordering and orientation of embryos.' publications.volume='8' publications.year=2011\n",
      "Gene: publications.abstractText='RNApolII-dependent transcription is repressed in primordial germ cells of many animals during early development and is thought to be important for maintenance of germline fate by preventing somatic differentiation. Germ cell transcriptional repression occurs concurrently with inhibition of phosphorylation in the carboxy-terminal domain (CTD) of RNApolII, as well as with chromatin remodeling. The precise mechanisms involved are unknown. Here, we present evidence that a noncoding RNA transcribed by the gene polar granule component (pgc) regulates transcriptional repression in Drosophila germ cells. Germ cells lacking pgc RNA express genes important for differentiation of nearby somatic cells and show premature phosphorylation of RNApolII. We further show that germ cells lacking pgc show increased levels of K4, but not K9 histone H3 methylation, and that the chromatin remodeling Swi/Snf complex is required for a second stage in germ cell transcriptional repression. We propose that a noncoding RNA controls transcription in early germ cells by blocking the transition from preinitiation to transcriptional elongation. We further show that repression of somatic differentiation signals mediated by the Torso receptor-tyrosine kinase is important for germline development.' publications.doi=None publications.firstAuthor='Martinho Rui Gonçalo' publications.id=1007732 publications.issue='2' publications.journal='Curr. Biol.' publications.month='Jan' publications.pages='159-65' publications.pubMedId='14738740' publications.title='A noncoding RNA is required for the repression of RNApolII-dependent transcription in primordial germ cells.' publications.volume='14' publications.year=2004\n",
      "Gene: publications.abstractText='Recent data have demonstrated that vRel, cRel, Dorsal, and NF-kappa B are members of a larger family of DNA-binding regulatory proteins. Rel proteins interact to form homo- and heterodimers that recognize specific sites on DNA, and it is likely that such protein-protein and protein-DNA interactions contribute to proper regulation of target gene expression by these proteins. Here we describe the use of a yeast transcription activation assay to study binding of three Rel family proteins to their native binding sites. These results show that the vRel and cRel proteins recognize two known NF-kappa B binding sites; the Dorsal protein does not recognize NF-kappa B sites, but does recognize related sites upstream of the Drosophila zerknüllt gene. Our experiments demonstrate that the members of this protein family recognize similar, but not identical, sites in the promoters of target genes, and we are able to identify a particular nucleotide that is apparently involved in the DNA-protein interaction. We exploit the properties of LexA fusion proteins to study the dimerization and DNA-contacting domains of cRel. Our results suggest that the cRel protein forms homodimers and that dimer formation may be necessary for cRel to bind DNA. Finally, our results show that transcription activation by these proteins is cooperative; such cooperativity may be important for correct temporal and spatial regulation of target gene expression.' publications.doi=None publications.firstAuthor='Kamens J' publications.id=1007839 publications.issue='10' publications.journal='New Biol.' publications.month='Oct' publications.pages='1005-13' publications.pubMedId='1768648' publications.title='A yeast transcription assay defines distinct rel and dorsal DNA recognition sequences.' publications.volume='3' publications.year=1991\n",
      "Gene: publications.abstractText='Retrogenes are processed copies of genes that are inserted into new genomic regions and that acquire new regulatory elements from the sequences in their surroundings. Here we use a comparative approach of phylogenetic footprinting and a non-comparative approach of measuring motif over-representation in retrogenes in order to describe putative elements present in cis-regulatory regions of 94 retrogenes recently described in Drosophila. The detailed examination of the motifs found in the core promoter regions of retrogenes reveals an abundance of the DNA replication-related element (DRE), the Initiator (Inr), and a new over-represented motif that we call the GCT motif. Parental genes also show an abundance of DRE and Inr motifs, but these do not seem to have been carried over with retrogenes. In particular, we also examined motifs upstream of retrogenes expressed in adult testis and were able to identify 6 additional over-represented motifs. Comparative analyses provide data on the conservation and origin of some of these motifs and reveal 15 additional conserved motifs in these retrogenes. Some of those conserved motifs are sequences bound by known transcription factors, while others are novel motifs. In this report we provide the first genome-wide data on which specific cis-regulatory regions can be recruited by retrogenes after they are inserted into new coding regions in the genome. Future experiments are needed to determine the function and role of the new elements presented here.' publications.doi='10.1016/j.ygeno.2008.09.006' publications.firstAuthor='Bai Yongsheng' publications.id=1007795 publications.issue='1' publications.journal='Genomics' publications.month='Jan' publications.pages='83-9' publications.pubMedId='18848618' publications.title='Quality of regulatory elements in Drosophila retrogenes.' publications.volume='93' publications.year=2009\n",
      "Gene: publications.abstractText='Seven zygotically active genes are required for normal patterning of the dorsal 40% of the Drosophila embryo. Among these genes, decapentaplegic (dpp) has the strongest mutant phenotype: in the absence of dpp, all cells in the dorsal and dorsolateral regions of the embryo adopt fates characteristic of more ventrally derived cells (Irish and Gelbart (1987) Genes Dev. 1, 868-879). Here we describe the phenotypes caused by alleles of another of this set of genes, tolloid, and show that tolloid is required for dorsal, but not dorsolateral, pattern. Extragenic suppressors of tolloid mutations were isolated that proved to be mutations that elevate dpp activity. We studied the relationship between tolloid and dpp by analyzing the phenotypes of tolloid embryos with elevated numbers of the dpp gene and found that doubling the dpp+ gene dosage completely suppressed weak tolloid mutants and partially suppressed the phenotypes of tolloid null mutants. We conclude that the function of tolloid is to increase dpp activity. We also examined the effect of doubling dpp+ gene dosage on the phenotypes caused by other mutations affecting dorsal development. Like tolloid, the phenotypes of mutant embryos lacking shrew gene function were suppressed by elevated dpp, indicating that shrew also acts upstream of dpp to increase dpp activity. In contrast, increasing the number of copies of the dpp gene enhanced the short gastrulation (sog) mutant phenotype, causing ventrolateral cells to adopt dorsal fates. This indicates that sog gene product normally blocks dpp activity ventrally. We propose that the tolloid, shrew and sog genes are required to generate a gradient of dpp activity, which directly specifies the pattern of the dorsal 40% of the embryo.' publications.doi=None publications.firstAuthor='Ferguson E L' publications.id=1007760 publications.issue='3' publications.journal='Development' publications.month='Mar' publications.pages='583-97' publications.pubMedId='1618130' publications.title='Localized enhancement and repression of the activity of the TGF-beta family member, decapentaplegic, is necessary for dorsal-ventral pattern formation in the Drosophila embryo.' publications.volume='114' publications.year=1992\n",
      "Gene: publications.abstractText='Signaling by Decapentaplegic (Dpp), a member of the TGFbeta superfamily of signaling molecules similar to vertebrate BMP2 and BMP4, has been implicated in many developmental processes in Drosophila melanogaster. Notably, Dpp acts as a long-range morphogen during imaginal disc growth and patterning. Genetic approaches led to the identification of a number of gene products that constitute the core signaling pathway. In addition to the ligand-activated heteromeric receptor complex and the signal-transducing intracellular Smad proteins, Dpp signaling requires two nuclear proteins, Schnurri (Shn) and Brinker (Brk), to prime cells for Dpp responsiveness. A complex interplay between the nuclear factors involved in Dpp signaling appears to control the transcriptional readout of the Dpp morphogen gradient. It remains to be seen whether similar molecular mechanisms operate in the nucleus in vertebrate systems.' publications.doi='10.1093/emboj/20.13.3298' publications.firstAuthor='Affolter M' publications.id=1004008 publications.issue='13' publications.journal='EMBO J.' publications.month='Jul' publications.pages='3298-305' publications.pubMedId='11432817' publications.title='Nuclear interpretation of Dpp signaling in Drosophila.' publications.volume='20' publications.year=2001\n",
      "Gene: publications.abstractText=\"Signaling molecules of the transforming growth factor beta (TGF-beta) family contribute to numerous developmental processes in a variety of organisms. However, our understanding of the mechanisms which regulate the activity of and mediate the response to TGF-beta family members remains incomplete. The product of the Drosophila decapentaplegic (dpp) locus is a well-characterized member of this family. We have taken a genetic approach to identify factors required for TGF-beta function in Drosophila by testing for genetic interactions between mutant alleles of dpp and a collection of chromosomal deficiencies. Our survey identified two deficiencies that act as maternal enhancers of recessive embryonic lethal alleles of dpp. The enhanced individuals die with weakly ventralized phenotypes. These phenotypes are consistent with a mechanism whereby the deficiencies deplete two maternally provided factors required for dpp's role in embryonic dorsal-ventral pattern formation. One of these deficiencies also appears todelete a factor required for dpp function in wing vein formation. These deficiencies remove material from the 54F-55A and 66B-66C polytene chromosomal regions, respectively. As neither of these regions has been previously implicated in dpp function, we propose that each of the deficiencies removes a novel factor or factors required for dpp function.\" publications.doi=None publications.firstAuthor='Nicholls R E' publications.id=1007890 publications.issue='1' publications.journal='Genetics' publications.month='May' publications.pages='203-15' publications.pubMedId='9584097' publications.title='Identification of chromosomal regions involved in decapentaplegic function in Drosophila.' publications.volume='149' publications.year=1998\n",
      "Gene: publications.abstractText='Similar to their human counterparts, the Drosophila Rbf1 and Rbf2 Retinoblastoma family members control cell cycle and developmentally regulated gene expression. Increasing evidence suggests that Rbf proteins rely on multiprotein complexes to control target gene transcription. We show here that the developmentally regulated COP9 signalosome (CSN) physically interacts with Rbf2 during embryogenesis. Furthermore, the CSN4 subunit of the COP9 signalosome co-occupies Rbf target gene promoters with Rbf1 and Rbf2, suggesting an active role for the COP9 signalosome in transcriptional regulation. The targeted knockdown of individual CSN subunits leads to diminished Rbf1 and Rbf2 levels and to altered cell cycle progression. The proteasome-mediated destruction of Rbf1 and Rbf2 is increased in cells and embryos with diminished COP9 activity, suggesting that the COP9 signalosome protects Rbf proteins during embryogenesis. Previous evidence has linked gene activation to protein turnover via the promoter-associated proteasome. Our findings suggest that Rbf repression may similarly involve the proteasome and the promoter-associated COP9 signalosome, serving to extend Rbf protein lifespan and enable appropriate programs of retinoblastoma gene control during development.' publications.doi='10.1091/mbc.E06-09-0790' publications.firstAuthor='Ullah Zakir' publications.id=1007749 publications.issue='4' publications.journal='Mol. Biol. Cell' publications.month='Apr' publications.pages='1179-86' publications.pubMedId='17251548' publications.title='Retinoblastoma protein regulation by the COP9 signalosome.' publications.volume='18' publications.year=2007\n",
      "Gene: publications.abstractText='Smad transcription factors lie at the core of one of the most versatile cytokine signaling pathways in metazoan biology-the transforming growth factor-beta (TGFbeta) pathway. Recent progress has shed light into the processes of Smad activation and deactivation, nucleocytoplasmic dynamics, and assembly of transcriptional complexes. A rich repertoire of regulatory devices exerts control over each step of the Smad pathway. This knowledge is enabling work on more complex questions about the organization, integration, and modulation of Smad-dependent transcriptional programs. We are beginning to uncover self-enabled gene response cascades, graded Smad response mechanisms, and Smad-dependent synexpression groups. Our growing understanding of TGFbeta signaling through the Smad pathway provides general principles for how animal cells translate complex inputs into concrete behavior.' publications.doi='10.1101/gad.1350705' publications.firstAuthor='Massagué Joan' publications.id=1007800 publications.issue='23' publications.journal='Genes Dev.' publications.month='Dec' publications.pages='2783-810' publications.pubMedId='16322555' publications.title='Smad transcription factors.' publications.volume='19' publications.year=2005\n",
      "Gene: publications.abstractText='Small differences in the levels of an extracellular signaling molecule can specify cell fate during development. Threshold responses are often determined at the level of transcription. Cell-specific and spatially localized patterns of gene expression depend on combinations of sequence-specific activators and repressors that bind to extensive cis-regulatory regions. Different mechanisms for integrating this complex regulatory information are discussed, particularly the role of coregulatory proteins, which are recruited to the DNA template by sequence-specific transcription factors. Recent studies suggest that a growing set of coactivators and corepressors mediate communication between diverse upstream regulatory proteins and the core RNA polymerase II transcription complex.' publications.doi=None publications.firstAuthor='Mannervik M' publications.id=1007892 publications.issue='5414' publications.journal='Science' publications.month='Apr' publications.pages='606-9' publications.pubMedId='10213677' publications.title='Transcriptional coregulators in development.' publications.volume='284' publications.year=1999\n",
      "Gene: publications.abstractText='The Bicoid-based anterior patterning system of Drosophila embryogenesis appears to be unique to higher dipterans. A new study suggests how this may have evolved out of an alternative mechanism based on cooperating Orthodenticle and Hunchback proteins, the two mechanisms intersecting at the level of downstream target genes.' publications.doi=None publications.firstAuthor='Lynch Jeremy' publications.id=1007748 publications.issue='14' publications.journal='Curr. Biol.' publications.month='Jul' publications.pages='R557-9' publications.pubMedId='12867048' publications.title='Evolution of development: beyond bicoid.' publications.volume='13' publications.year=2003\n",
      "Gene: publications.abstractText=\"The DRE/DREF system plays an important role in transcription of DNA replication genes such as those encoding the 180 and 73 kDa subunits of DNA polymerase alpha as well as that for encoding PCNA. In this study, we found two sequences homologous to DRE (5'-TATCGATA-3') in the 5'-flanking region (-370 to -357 with respect to the transcription initiation site) of the D-raf gene and confirmed transcriptional activity through gel mobility shift assays, transient CAT assays, and spatial patterns of lacZ expression in transgenic larval tissues carrying D-raf and lacZ fusion genes. Further, we demonstrated that the D-raf gene is another target of the Zerknüllt (Zen) protein with observation of D-raf repression by Zen protein in cultured cells and its ectopic expression in the dorsal region of the homozygous zen mutant embryo. The evidence of DRE/DREF involvement in regulation of the D-raf gene obtained in this study strongly supports the idea that the DRE/DREF system is responsible for the coordinated regulation of cell proliferation-related genes in Drosophila.\" publications.doi=None publications.firstAuthor='Ryu J R' publications.id=1007845 publications.issue='4' publications.journal='Nucleic Acids Res.' publications.month='Feb' publications.pages='794-9' publications.pubMedId='9016631' publications.title='Transcriptional regulation of the Drosophila-raf proto-oncogene by the DNA replication-related element (DRE)/DRE-binding factor (DREF) system.' publications.volume='25' publications.year=1997\n",
      "Gene: publications.abstractText='The Dorsal morphogen acts as both an activator and a repressor of transcription in the Drosophila embryo to regulate the expression of dorsal/ventral patterning genes. Circumstantial evidence has suggested that Dorsal is an intrinsic activator and that additional factors (corepressors) convert it into a repressor. These corepressors, however, have previously eluded definitive identification. We show here, via the analysis of embryos lacking the maternally encoded Groucho corepressor and via protein-binding assays, that recruitment of Groucho to the template by protein:protein interactions is required for the conversion of Dorsal from an activator to a repressor. Groucho is therefore a critical component of the dorsal/ventral patterning system.' publications.doi=None publications.firstAuthor='Dubnicoff T' publications.id=1007776 publications.issue='22' publications.journal='Genes Dev.' publications.month='Nov' publications.pages='2952-7' publications.pubMedId='9367978' publications.title='Conversion of dorsal from an activator to a repressor by the global corepressor Groucho.' publications.volume='11' publications.year=1997\n",
      "Gene: publications.abstractText='The Dorsal morphogen directs formation of the Drosophila dorsoventral axis by both activating and repressing transcription. It contains an N-terminal Rel homology domain (RHD), which is responsible for DNA binding and regulated nuclear import, and a C-terminal domain (CTD) that contains activation and repression motifs. To determine if the RHD has a direct role in transcriptional control, we analyzed a series of RHD mutations in S2 cells and embryos. Two classes ofmutations (termed class I and class II mutations) that alter activation without affecting DNA binding or nuclear import were identified. The two classes appear to define distinct protein interaction surfaces on opposite faces of the RHD. Class I mutations enhance an apparently inhibitory interaction between the RHD and the CTD and eliminate both activation and repression by Dorsal. In contrast, class II mutations result in increased activation in S2 cells but severely decreased activation in embryos and have little effect on repression. Analysis of the cuticles of class II mutant embryos suggests that, in the absence of Dorsal-mediated activation, Dorsal-mediated repression is not sufficient to pattern the embryo. These results provide some of the first evidence that the RHD plays an active role in transcriptional regulation in intact multicellular organisms.' publications.doi=None publications.firstAuthor='Jia Songtao' publications.id=1007882 publications.issue='14' publications.journal='Mol. Cell. Biol.' publications.month='Jul' publications.pages='5089-99' publications.pubMedId='12077338' publications.title='The Dorsal Rel homology domain plays an active role in transcriptional regulation.' publications.volume='22' publications.year=2002\n",
      "Gene: publications.abstractText='The Dorsal morphogen is a transcription factor that activates some genes and represses others to establish multiple domains of gene expression along the dorsal/ventral axis of the early Drosophila embryo. Repression by Dorsal appears to require accessory proteins that bind to corepression elements in Dorsal-dependent regulatory modules called ventral repression regions (VRRs). We have identified a corepression element in decapentaplegic (dpp), a zygotically active gene that is repressed by the Dorsal morphogen. This dpp repression element (DRE) is located within a previously identified VRR and close to essential Dorsal-binding sites. We have purified a factor from Drosophila embryo extracts that binds to the DRE but not to mutant forms of the DRE that fail to support efficient repression. This protein also binds to an apparently essential region in a VRR associated with the zerknüllt (zen) gene. One of the DREs in the dpp VRR overlaps the binding site for a potential activator protein suggesting that one mechanism of ventral repression may be the mutually exclusive binding of repressor and activator proteins. We have found the DRE-binding protein to be identical to NTF-1 (equivalent to Elf-1, the product of the grainyhead gene), a factor originally identified as an activator of the Ultrabithorax and Dopa decarboxylase promoters. NTF-1 mRNA is synthesized during oogenesis and deposited in the developing oocyte where it is available to contribute to ventral repression during early embryogenesis. Previous studies have shown that overexpression of NTF-1 in the postblastoderm embryo results in a phenotype that is consistent with a role for this factor in the repression of dpp later in embryogenesis.' publications.doi=None publications.firstAuthor='Huang J D' publications.id=1007712 publications.issue='24' publications.journal='Genes Dev.' publications.month='Dec' publications.pages='3177-89' publications.pubMedId='8543160' publications.title='Binding sites for transcription factor NTF-1/Elf-1 contribute to the ventral repression of decapentaplegic.' publications.volume='9' publications.year=1995\n",
      "Gene: publications.abstractText='The Drosophila Groucho (Gro) protein is the prototype for a large family of corepressors, examples of which are found in most metazoans. This family includes the human transducin-like Enhancer of split (TLE) proteins. As corepressors, Gro/TLE family proteins do not bind to DNA directly, but rather are recruited to the template by DNA-bound repressor proteins. Gro/TLE family proteins are required for many developmental processes, including lateral inhibition, segmentation, sex determination, dorsal/ventral pattern formation, terminal pattern formation, and eye development. These proteins are characterized by a conserved N-terminal glutamine-rich domain and a conserved C-terminal WD-repeat domain. The primary role of the glutamine-rich domain is apparently to mediate tetramerization, while the WD-repeat domain may mediate interactions with DNA-bound repressors. The glutamine rich and WD-repeat domains are separated by a less conserved region containing domains that have been implicated in transcriptional repression and nuclear localization. In addition to encoding full-length Gro/TLE family proteins, most metazoan genomes encode truncated family members that contain the N-terminal oligomerization domain, but lack the C-terminal WD-repeat domain. These truncated proteins may negatively regulate full-length Gro/TLE proteins, perhaps by sequestering them in non-productive complexes. Gro/TLE family proteins probably repress transcription by multiple mechanisms. For example, a glycine/proline-rich domain in the central variable region functions to recruit the histone deacetylase Rpd3 to the template. This histone deacetylase then presumably silences transcription by altering local chromatin structure. Other repression domains in Gro may function in a histone deacetylase-independent manner. Many aspects of Gro/TLE protein function remain to be explored, including the possible post-translational regulation of Gro/TLE activity as well as the mechanisms by which Gro/TLE proteins direct repression at a distance.' publications.doi=None publications.firstAuthor='Chen G' publications.id=1007743 publications.issue='1-2' publications.journal='Gene' publications.month='May' publications.pages='1-16' publications.pubMedId='10831834' publications.title='Groucho/TLE family proteins and transcriptional repression.' publications.volume='249' publications.year=2000\n",
      "Gene: publications.abstractText='The Drosophila Toll pathway takes part in both establishment of the embryonic dorsoventral axis and induction of the innate immune response in adults. Upon activation by the cytokine Spätzle, Toll interacts with the adaptor proteins DmMyD88 and Tube and the kinase Pelle and triggers degradation of the inhibitor Cactus, thus allowing the nuclear translocation of the transcription factor Dorsal/Dif. weckle (wek) was previously identified as a new dorsal group gene that encodes a putative zinc finger transcription factor. However, its role in the Toll pathway was unknown. Here, we isolated new wek alleles and demonstrated that cactus is epistatic to wek, which in turn is epistatic to Toll. Consistent with this, Wek localizes to the plasma membrane of embryos, independently of Toll signaling. Wek homodimerizes and associates with Toll. Moreover, Wek binds to and localizes DmMyD88 to the plasma membrane. Thus, Wek acts as an adaptor to assemble/stabilize a Toll/Wek/DmMyD88/Tube complex. Remarkably, unlike the DmMyD88/tube/pelle/cactus gene cassette of the Toll pathway, wek plays a minimal role, if any, in the immune defense against Gram-positive bacteria and fungi. We conclude that Wek is an adaptor to link Toll and DmMyD88 and is required for efficient recruitment of DmMyD88 to Toll. Unexpectedly, wek is dispensable for innate immune response, thus revealing differences in the Toll-mediated activation of Dorsal in the embryo and Dif in the fat body of adult flies.' publications.doi='10.1016/j.cub.2006.05.050' publications.firstAuthor='Chen Li-Ying' publications.id=1007796 publications.issue='12' publications.journal='Curr. Biol.' publications.month='Jun' publications.pages='1183-93' publications.pubMedId='16782008' publications.title='Weckle is a zinc finger adaptor of the toll pathway in dorsoventral patterning of the Drosophila embryo.' publications.volume='16' publications.year=2006\n",
      "Gene: publications.abstractText='The Drosophila decapentaplegic gene, the Xenopus activin genes and the genes encoding the mouse bone morphogenetic proteins are transforming growth factor-beta-related genes whose roles in development are the focus of current studies. They exhibit elaborate patterns of expression during development, and the protein products have potent effects on the differentiation of specific cell types.' publications.doi=None publications.firstAuthor='Hoffmann F M' publications.id=1007864 publications.issue='6' publications.journal='Curr. Opin. Cell Biol.' publications.month='Dec' publications.pages='947-52' publications.pubMedId='1687650' publications.title='Transforming growth factor-beta-related genes in Drosophila and vertebrate development.' publications.volume='3' publications.year=1991\n",
      "Gene: publications.abstractText='The Drosophila engrailed gene product (En) is a homeodomain-containing protein that contributes to segmental patterning. In transfection assays it acts as a transcriptional repressor. We show that En is an active repressor, blocking activation by mammalian and yeast activators that bind to sites some distance away from those bound by En. Active repression is distinct from the effects of passive homeodomain-containing proteins, which repress when competing with activators for binding sites and activate when competing with En. Active repression activity maps outside the En homeodomain, and this activity can be transferred to a heterologous DNA binding domain.' publications.doi=None publications.firstAuthor='Jaynes J B' publications.id=1007869 publications.issue='6' publications.journal='EMBO J.' publications.month='Jun' publications.pages='1427-33' publications.pubMedId='1673924' publications.title='Active repression of transcription by the engrailed homeodomain protein.' publications.volume='10' publications.year=1991\n",
      "Gene: publications.abstractText='The Drosophila eye field that gives rise to the visual system and dorsal head epidermis forms an unpaired anlage located in the dorsal head ectoderm. The eye field expresses and requires both Dpp and EGFR signaling for its development. As shown in previous studies, EGFR is required for cell maintenance in the developing visual system. Dpp initially switches on the early eye genes so and eya in the eye field. Consecutively, highlevels of Dpp in the dorsal midline inhibit these genes and promote development of head epidermis. We show that Dpp negatively regulates EGFR signaling, thereby increasing the amount of cell death in the dorsal midline. By this mechanism, Dpp controls the formation of a bilateral visual system and indirectly modulates cell death, which is essential for normal head morphogenesis. Loss of either Dpp or its downstream target, Zen, abolishes head epidermis fate and leads to the misexpression of dp-ERK in the dorsal midline. The resulting morphological phenotype consists of cyclopia, reduction of cell death, and failure of head involution. Ectopic expression of activated EGFR inhibits the Dpp target race and thereby causes cyclopia and defective head involution. We discuss possible mechanisms of Dpp and EGFR interaction in the embryo.' publications.doi=None publications.firstAuthor='Chang Ting' publications.id=1007859 publications.issue='1' publications.journal='Dev. Biol.' publications.month='Nov' publications.pages='103-13' publications.pubMedId='14568549' publications.title='Antagonistic relationship between Dpp and EGFR signaling in Drosophila head patterning.' publications.volume='263' publications.year=2003\n",
      "Gene: publications.abstractText='The Drosophila gene bicoid functions as the anterior body pattern organizer of Drosophila. Embryos lacking maternally expressed bicoid fail to develop anterior segments including head and thorax. In wild-type eggs, bicoid mRNA is localized in the anterior pole region and the bicoid protein forms an anterior-to-posterior concentration gradient. bicoid activity is required for transcriptional activation of zygotic segmentation genes and the translational suppression of uniformly distributed maternal caudal mRNA in the anterior region of the embryo. caudal genes as well as other homeobox genes or members of the Drosophila segmentation gene cascade have been found to be conserved in animal evolution. In contrast, bicoid homologs have been identified only in close relatives of the schizophoran fly Drosophila. This poses the question of how the bicoid gene evolved and adopted its unique function in organizing anterior-posterior polarity. We have cloned bicoid from a basal cyclorrhaphan fly, Megaselia abdita (Phoridae, Aschiza), and show that the gene originated from a recent duplication of the direct homolog of the vertebrate gene Hox3, termed zerknüllt, which specifies extraembryonic tissues in insects.' publications.doi=None publications.firstAuthor='Stauber M' publications.id=1007810 publications.issue='7' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Mar' publications.pages='3786-9' publications.pubMedId='10097115' publications.title='The anterior determinant bicoid of Drosophila is a derived Hox class 3 gene.' publications.volume='96' publications.year=1999\n",
      "Gene: publications.abstractText='The Drosophila gene buttonhead (btd) encodes a zinc-finger protein related to the human transcription factor Sp1. btd is expressed in the syncytial blastoderm embryo in a stripe covering the anlagen of the antennal, intercalary and mandibular head segments. btd has been characterized as a head gap gene, since these segments are deleted in btd mutant embryos. We report here that the cis-acting elements required for btd head stripe expression are contained in a 1 kb DNA fragment, located about 3 kb upstream of the promoter. The four maternal coordinate systems are necessary for correct btd head stripe expression, likely by acting through the 1 kb cis-acting control region. Expression of the btd head stripe depends on the anterior morphogen encoded by the gene bicoid (bcd). bcd-dependent activation also involves the activity of the morphogens of the posterior and dorsoventral systems, hunchback and dorsal, respectively, which act together to control the spatial limits of the expression domain. Finally, the terminal system takes part in the regulation of btd head stripe expression by enhancing activation at low levels of activity and repression at high levels of activity.' publications.doi=None publications.firstAuthor='Wimmer E A' publications.id=1007746 publications.issue='2' publications.journal='Mech. Dev.' publications.month='Oct' publications.pages='235-45' publications.pubMedId='8562425' publications.title='Trans- and cis-acting requirements for blastodermal expression of the head gap gene buttonhead.' publications.volume='53' publications.year=1995\n",
      "Gene: publications.abstractText='The Drosophila melanogaster genome was the first metazoan genome to have been sequenced by the whole-genome shotgun (WGS) method. Two issues relating to this achievement were widely debated in the genomics community: how correct is the sequence with respect to base-pair (bp) accuracy and frequency of assembly errors? And, how difficult is it to bring a WGS sequence to the accepted standard for finished sequence? We are now in a position to answer these questions. Our finishing process was designed to close gaps, improve sequence quality and validate the assembly. Sequence traces derived from the WGS and draft sequencing of individual bacterial artificial chromosomes (BACs) were assembled into BAC-sized segments. These segments were brought to high quality, and then joined to constitute the sequence of each chromosome arm. Overall assembly was verified by comparison to a physical map of fingerprinted BAC clones. In the current version of the 116.9 Mb euchromatic genome, called Release 3, the six euchromatic chromosome arms are represented by 13 scaffolds with a total of 37 sequence gaps. We compared Release 3 to Release 2; in autosomal regions of unique sequence, the error rate of Release 2 was one in 20,000 bp. The WGS strategy can efficiently produce a high-quality sequence of a metazoan genome while generating the reagents required for sequence finishing. However, the initial method of repeat assembly was flawed. The sequence we report here, Release 3, is a reliable resource for molecular genetic experimentation and computational analysis.' publications.doi=None publications.firstAuthor='Celniker Susan E' publications.id=1906654 publications.issue='12' publications.journal='Genome Biol.' publications.month=None publications.pages='RESEARCH0079' publications.pubMedId='12537568' publications.title='Finishing a whole-genome shotgun: release 3 of the Drosophila melanogaster euchromatic genome sequence.' publications.volume='3' publications.year=2002\n",
      "Gene: publications.abstractText='The Drosophila morphogen dorsal, KBF1, NF-kappa B, and the proto-oncogene c-rel belong to the rel family of transcription factors whose function is regulated post-translationally by selective nuclear import. In the early Drosophila embryo, dorsal protein is proposed to be retained in the cytoplasm through its interaction with cactus protein. The maternal dorsal group genes constitute a signal transduction pathway, which results in targeting cytoplasmic dorsal protein into the nuclei of the syncytial blastoderm embryo, in a ventral-to-dorsal gradient. The asymmetric transcriptional regulation of zygotic genes along the dorsoventral axis by the dorsal morphogen gradient establishes embryonic dorsoventral polarity. In the lymphocytes, the functional equivalent of cactus is I kappa B, which appears to retain NF-kappa B in the cytoplasm. This retention is relieved by extracellular signals in tissue culture. NF-kappa B and rel proteins each are known to function as oligomeric complexes. Here we present genetic and biochemical evidence for the existence and functional importance of an oligomeric dorsal complex in vivo.' publications.doi=None publications.firstAuthor='Govind S' publications.id=1007840 publications.issue='17' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Sep' publications.pages='7861-5' publications.pubMedId='1518806' publications.title='In vivo self-association of the Drosophila rel-protein dorsal.' publications.volume='89' publications.year=1992\n",
      "Gene: publications.abstractText='The Drosophila nonreceptor protein tyrosine phosphatase, Corkscrew (Csw), functions positively in multiple receptor tyrosine kinase (RTK) pathways, including signaling by the epidermal growth factor receptor (EGFR). Detailed phenotypic analyses of csw mutations have revealed that Csw activity is required in many of the same developmental processes that require EGFR function. However, it is still unclear where in the signaling hierarchy Csw functions relative to other proteins whose activities are also required downstream of the receptor. To address this issue, genetic interaction experiments were performed to place csw gene activity relative to the EGFR, spitz (spi), rhomboid (rho), daughter of sevenless (DOS), kinase-suppressor of ras (ksr), ras1, D-raf, pointed (pnt), and moleskin. We followed the EGFR-dependent formation of VA2 muscle precursor cells as a sensitive assay for these genetic interaction studies. First, we established that Csw has a positive function during mesoderm development. Second, we found that tissue-specific expression of a gain-of-function csw construct rescues loss-of-function mutations in other positive signaling genes upstream of rolled (rl)/MAPK in the EGFR pathway. Third, we were able to infer levels of EGFR signaling in various mutant backgrounds during myogenesis. This work extends previous studies of Csw during Torso and Sevenless RTK signaling to include an in-depth analysis of the role of Csw in the EGFR signaling pathway.' publications.doi=None publications.firstAuthor='Johnson Hamlet M R' publications.id=1007695 publications.issue='3' publications.journal='Genetics' publications.month='Nov' publications.pages='1073-87' publications.pubMedId='11729154' publications.title='Analysis of corkscrew signaling in the Drosophila epidermal growth factor receptor pathway during myogenesis.' publications.volume='159' publications.year=2001\n",
      "Gene: publications.abstractText=\"The Drosophila polypyrimidine tract-binding protein (dmPTB or hephaestus) plays an important role during embryogenesis. A loss of function mutation, heph(03429), results in varied defects in embryonic developmental processes, leading to embryonic lethality. However, the suite of molecular functions that are disrupted in the mutant remains unknown. We have used an unbiased high throughput sequencing approach to identify transcripts that are misregulated in this mutant. Misregulated transcripts show evidence of significantly altered patterns of splicing (exon skipping, 5' and 3' splice site switching), alternative 5' ends, and mRNA level changes (up and down regulation). These findings are independently supported by reverse-transcription-polymerase chain reaction (RT-PCR) analysis and in situ hybridization. We show that a group of genes, such as Zerknüllt, z600 and screw are among the most upregulated in the mutant and have been functionally linked to dorso-ventral patterning and/or dorsal closure processes. Thus, loss of dmPTB function results in specific misregulated transcripts, including those that provide the missing link between the loss of dmPTB function and observed developmental defects in embryogenesis. This study provides the first comprehensive repertoire of genes affected in vivo in the heph mutant in Drosophila and offers insight into the role of dmPTB during embryonic development.\" publications.doi='10.1371/journal.pone.0098585' publications.firstAuthor='Heimiller Joseph' publications.id=1002423 publications.issue='7' publications.journal='PLoS ONE' publications.month=None publications.pages='e98585' publications.pubMedId='25014769' publications.title='Drosophila polypyrimidine tract-binding protein (DmPTB) regulates dorso-ventral patterning genes in embryos.' publications.volume='9' publications.year=2014\n",
      "Gene: publications.abstractText=\"The Drosophila sex-determination switch gene Sex-lethal (Sxl) and the X-chromosome signal element genes (XSEs) that induce the female-specific expression of Sxl are transcribed extremely early in development when most of the genome of this organism is still silent. The DNA sequence CAGGTAG had been implicated in this pre-cellular blastoderm activation of sex-determination genes. A genome-wide computational search, reported here, suggested that CAGGTAG is not specific to early sex-determination genes, since it is over-represented upstream of most genes that are transcribed pre-cellular blastoderm, not just those involved in sex determination. The same search identified similarly over-represented, one-base-pair degenerate sequences as possible functional synonyms of CAGGTAG. We call these heptamers collectively, the TAGteam. Relevance of the TAGteam sequences to pre-cellular blastoderm transcription was established through analysis of TAGteam changes in Sxl, scute (an XSE), and the ;ventral repression element' of the pattern-formation gene zerknüllt. Decreasing the number of TAGteam sites retarded the onset of pre-blastoderm transcription, whereas increasing their number correlated with an advanced onset. Titration of repressors was thought to be the rate-limiting step determining the onset of such early transcription, but this TAGteam dose effect shows that activators must also play an important role in the timing of pre-blastoderm gene expression.\" publications.doi='10.1242/dev.02373' publications.firstAuthor='ten Bosch John R' publications.id=1007803 publications.issue='10' publications.journal='Development' publications.month='May' publications.pages='1967-77' publications.pubMedId='16624855' publications.title='The TAGteam DNA motif controls the timing of Drosophila pre-blastoderm transcription.' publications.volume='133' publications.year=2006\n",
      "Gene: publications.abstractText='The Drosophila short gastrulation gene (sog) encodes a large extracellular protein (Sog) that inhibits signaling by BMP-related ligands. Sog and its vertebrate counterpart Chordin contain four copies of a cysteine repeat (CR) motif defined by 10 cysteine residues spaced in a fixed pattern and a tryptophan residue situated between the first two cysteines. Here we present a structure-function analysis of the CR repeats in Sog, using a series of deletion and point mutation constructs, as well as constructs in which CR domains have been swapped. This analysis indicates that the CR domains are individually dispensable for Sog function but that they are not interchangeable. These studies reveal three different types of Sog activity: intact Sog, which inhibits signaling mediated by the ligand Glass bottom boat (Gbb), a more broadly active class of BMP antagonist referred to as Supersog, and a newly identified activity, which may promote rather than inhibit BMP signaling. Analysis of the activities of CR swap constructs indicates that the CR domains are required for full activity of the various forms of Sog but that the type of Sog activity is determined primarily by surrounding protein sequences. Cumulatively, our analysis suggests that CR domains interact physically with adjacent protein sequences to create forms of Sog with distinct BMP modulatory activities.' publications.doi=None publications.firstAuthor='Yu Kweon' publications.id=1004085 publications.issue='3' publications.journal='Genetics' publications.month='Mar' publications.pages='1323-36' publications.pubMedId='15082551' publications.title='Cysteine repeat domains and adjacent sequences determine distinct bone morphogenetic protein modulatory activities of the Drosophila Sog protein.' publications.volume='166' publications.year=2004\n",
      "Gene: publications.abstractText='The Drosophila trithorax group gene brahma (brm) encodes the ATPase subunit of a 2-MDa chromatin-remodeling complex. brm was identified in a screen for transcriptional activators of homeotic genes and subsequently shown to play a global role in transcription by RNA polymerase II. To gain insight into the targeting, function, and regulation of the BRM complex, we screened for mutations that genetically interact with a dominant-negative allele of brm (brm(K804R)). We first screened for dominant mutations that are lethal in combination with a brm(K804R) transgene under control of the brm promoter. In a distinct but related screen, we identified dominant mutations that modify eye defects resulting from expression of brm(K804R) in the eye-antennal imaginal disc. Mutations in three classes of genes were identified in our screens: genes encoding subunits of the BRM complex (brm, moira, and osa), other proteins directly involved in transcription (zerknullt and RpII140), and signaling molecules (Delta and vein). Expression of brm(K804R) in the adult sense organ precursor lineage causes phenotypes similar to those resulting from impaired Delta-Notch signaling. Our results suggest that signaling pathways may regulate the transcription of target genes by regulating the activity of the BRM complex.' publications.doi='10.1534/genetics.105.041327' publications.firstAuthor='Armstrong Jennifer A' publications.id=1007842 publications.issue='4' publications.journal='Genetics' publications.month='Aug' publications.pages='1761-74' publications.pubMedId='15944353' publications.title='Genetic screens for enhancers of brahma reveal functional interactions between the BRM chromatin-remodeling complex and the delta-notch signal transduction pathway in Drosophila.' publications.volume='170' publications.year=2005\n",
      "Gene: publications.abstractText='The Drosophila z600 gene is a member of an overlapping gene cluster located in the 71CD interval of chromosome 3. We have used several approaches to study the spatial and temporal expression of z600 during embryogenesis. Northern analysis shows that z600 is zygotically expressed, with gene transcripts accumulating transiently during early embryo-genesis. The localization of z600 transcripts in embryo sections by in situ hybridization reveals a dynamic pattern of RNA distribution. z600 RNA is distributed throughout the embryo during the midsyncytial blastoderm stage, becomes concentrated dorsally and posteriorly during cellularization, and persists in the dorsal-most region of the embryo during gastrulation and germ band extension. z600 transcript accumulation is altered in dorsalized or ventralized mutant embryos, suggesting that z600 is directly or indirectly regulated by genes specifying the dorsal-ventral pattern in the embryo. An analysis of germ line transformants harboring a z600-lacZ gene fusion demonstrates that a 276-bp DNA region contains the sequences responsible for dorsal activation, but not ventral repression, of z600 gene expression during early embryogenesis.' publications.doi=None publications.firstAuthor='Schulz R A' publications.id=1007860 publications.issue='1' publications.journal='Dev. Biol.' publications.month='Nov' publications.pages='211-21' publications.pubMedId='2478402' publications.title='Dorsal expression of the Drosophila z600 gene during early embryogenesis.' publications.volume='136' publications.year=1989\n",
      "Gene: publications.abstractText='The GATA transcriptionfactors are a family of C4 zinc finger-motif DNA-binding proteins that play defined roles in hematopoiesis as well as presumptive roles in other tissues where they are expressed (e.g., testis, neuronal and placental trophoblast cells) during vertebrate development. To investigate the possibility that GATA proteins may also be involved in Drosophila development, we have isolated and characterized a gene (dGATAa) encoding a factor that is quite similar to mammalian GATA factors. The dGATAa protein sequence contains the two zinc finger DNA-binding domain of the GATA class but bears no additional sequence similarity to any of the vertebrate GATA factors. Analysis of dGATAa gene transcription during Drosophila development revealed that its mRNA is expressed at high levels during early embryogenesis, with transcripts first appearing in the dorsal portion of the embryo just after cellularization. As development progresses, dGATAa mRNA is present at high levels in the dorsal epidermis, suggesting that dGATAa may be involved in determining dorsal cell fate. The pattern of expression in a variety of dorsoventral polarity mutants indicates that dGATAa lies downstream of the zygotic patterning genes decapentaplegic and zerknüllt.' publications.doi=None publications.firstAuthor='Winick J' publications.id=1007714 publications.issue='4' publications.journal='Development' publications.month='Dec' publications.pages='1055-65' publications.pubMedId='7916677' publications.title='A GATA family transcription factor is expressed along the embryonic dorsoventral axis in Drosophila melanogaster.' publications.volume='119' publications.year=1993\n",
      "Gene: publications.abstractText='The Hox genes play a role in anteroposterior axis specification of bilaterian animals that has been conserved for more than 600 million years. However, some of these genes have occasionally changed their roles in evolution. For example, the insect gene fushi tarazu (ftz), although localised in the Hox cluster, no longer acts as a Hox gene, but is involved in segmentation and nervous system development. Recent data of Mouchel-Vielh et al., and Hughes and Kaufman on ftz homologues in a crustacean and a myriapod, respectively, shed new light onto the evolution of this gene.' publications.doi='10.1002/bies.10180' publications.firstAuthor='Damen Wim G M' publications.id=1007874 publications.issue='11' publications.journal='Bioessays' publications.month='Nov' publications.pages='992-5' publications.pubMedId='12386929' publications.title='fushi tarazu: a Hox gene changes its role.' publications.volume='24' publications.year=2002\n",
      "Gene: publications.abstractText='The Ras-related Rap GTPases are highly conserved across diverse species but their normal biological function is not well understood. Initial studies in mammalian cells suggested a role for Rap as a Ras antagonist. More recent experiments indicate functions in calcium- and cAMP-mediated signaling and it has been proposed that protein kinase A-mediated phosphorylation activates Rap in vivo. We show that Ras1-mediated signaling pathways in Drosophila are not influenced by Rap1 levels, suggesting that Ras1 and Rap1 function via distinct pathways. Moreover, a mutation that abolishes the putative cAMP-dependent kinase phosphorylation site of Drosophila Rap1 can still rescue the Rap1 mutant phenotype. Our experiments show that Rap1 is not needed for cell proliferation and cell-fate specification but demonstrate a critical function for Rap1 in regulating normal morphogenesis in the eye disk, the ovary and the embryo. Rap1 mutations also disrupt cell migrations and cause abnormalities in cell shape. These findings indicate a role for Rap proteins as regulators of morphogenesis in vivo.' publications.doi='10.1093/emboj/18.3.605' publications.firstAuthor='Asha H' publications.id=1004639 publications.issue='3' publications.journal='EMBO J.' publications.month='Feb' publications.pages='605-15' publications.pubMedId='9927420' publications.title='The Rap1 GTPase functions as a regulator of morphogenesis in vivo.' publications.volume='18' publications.year=1999\n",
      "Gene: publications.abstractText='The Rel family of transcription factors participate in a diverse array of processes, including acute responses to injury and infection, lymphocyte differentiation, and embryonic patterning. These proteins show homology in an extended region spanning about 300 amino acids (the Rel homology domain [RHD]). The RHD mediates both DNA binding and interactions with a family of inhibitor proteins, including I kappa B alpha and cactus. Previous studies have shown that an N-terminal region of the RHD (containing the sequence motif RXXRXRXXC) is important for DNA binding, while the C-terminal nuclear localization sequence is important for inhibitor interactions. Here we present a structure-function analysis of the Drosophila dorsal RHD. These studies identify another sequence within the RHD (region I) that is essential for inhibitor interactions. There is a tight correlation between the conservation of region I sequences and the specificity of Rel-inhibitor interactions in both flies and mammals. Point mutations in the region I sequence can uncouple DNA binding and inhibitor interactions in vitro. The phenotypes associated with the expression of a modified dorsal protein in transgenic Drosophila embryos suggest a similar uncoupling in vivo. Recent crystallographic studies suggest that the region I sequence and the nuclear localization sequence might form a composite surface which interacts with inhibitor proteins.' publications.doi=None publications.firstAuthor='Tatei K' publications.id=1007823 publications.issue='7' publications.journal='Mol. Cell. Biol.' publications.month='Jul' publications.pages='3627-34' publications.pubMedId='7791770' publications.title='Specificity of Rel-inhibitor interactions in Drosophila embryos.' publications.volume='15' publications.year=1995\n",
      "Gene: publications.abstractText='The Rel family transcription factor Dorsal patterns the dorsoventral axis of the Drosophila embryo by activating genes such as twist and snail and repressing genes such as decapentaplegic and zerknüllt. Dorsal represses transcription by recruiting the co-repressor Groucho. However, repression occurs only when Dorsal-binding sites are close to binding sites for other factors that also bind Groucho. The need for additional factors to assist Dorsal in repression may result from the intrinsically weak interaction between Dorsal and Groucho. To test this idea, we generated a Dorsal variant containing a high-affinity Groucho recruitment motif at its C terminus. As predicted, this variant functions as a dedicated repressor, silencing decapentaplegic and zerknüllt while failing to activate twist and snail. We also converted Dorsal into a dedicated activator by replacing its weak Groucho-recruitment motif with heterologous activation domains. Although the dedicated activator alleles fail to repress decapentaplegic and zerknüllt in the syncytial blastoderm embryo, they are able to pattern the dorsoventral axis. This indicates that dorsoventral patterning is not dependent upon Dorsal-mediated repression, reflecting the existence of redundant mechanisms to block Decapentaplegic signaling.' publications.doi='10.1242/dev.02643' publications.firstAuthor='Ratnaparkhi Girish S' publications.id=1007865 publications.issue='22' publications.journal='Development' publications.month='Nov' publications.pages='4409-14' publications.pubMedId='17035291' publications.title='Uncoupling dorsal-mediated activation from dorsal-mediated repression in the Drosophila embryo.' publications.volume='133' publications.year=2006\n",
      "Gene: publications.abstractText='The TGF-β pathway is an evolutionarily conserved signal transduction module that mediates diverse biological processes in animals. In Drosophila, both the BMP and Activin branches are required for viability. Studies rooted in classical and molecular genetic approaches continue to uncover new developmental roles for TGF-β signaling. We present an overview of the secreted ligands, transmembrane receptors and cellular Smad transducer proteins that compose the core pathway in Drosophila. An assortment of tools have been developed to conduct tissue-specific loss- and gain-of-function experiments for these pathway components. We discuss the deployment of these reagents, with an emphasis on appropriate usage and limitations of the available tools. Throughout, we note reagents that are in need of further improvement or development, and signaling features requiring further study. A general theme is that comparison of phenotypes for ligands, receptors, and Smads can be used to map tissue interactions, and to separate canonical and non-canonical signaling activities. Core TGF-β signaling components are subject to multiple layers of regulation, and are coupled to context-specific inputs and outputs. In addition to fleshing out how TGF-β signaling serves the fruit fly, we anticipate that future studies will uncover new regulatory nodes and modes and will continue to advance paradigms for how TGF-β signaling regulates general developmental processes.' publications.doi='10.1016/j.ymeth.2014.03.016' publications.firstAuthor='Peterson Aidan J' publications.id=1004089 publications.issue='1' publications.journal='Methods' publications.month='Jun' publications.pages='183-93' publications.pubMedId='24680699' publications.title='Strategies for exploring TGF-β signaling in Drosophila.' publications.volume='68' publications.year=2014\n",
      "Gene: publications.abstractText='The TNF and TNFR superfamilies of proteins are conserved throughout evolution. The first invertebrate orthologs of TNF and TNFR, Eiger and Wengen, were identified in Drosophila, which enabled us to take advantage of its powerful genetics. Indeed, genetic studies on Eiger in the last decade have discovered their signaling mechanisms through activation of the JNK pathway and unveiled the role of Eiger-JNK signaling in a variety of cellular and tissue processes such as cell death, cell proliferation, tissue growth regulation, host defense, pain sensitization, and canalization. In this review, we will describe the in vivo signaling of Eiger and its physiological roles in fly development and homeostasis, and will discuss the evolution of the TNF/TNFR systems.' publications.doi='10.1016/j.smim.2014.05.003' publications.firstAuthor='Igaki Tatsushi' publications.id=1007806 publications.issue='3' publications.journal='Semin. Immunol.' publications.month='Jun' publications.pages='267-74' publications.pubMedId='24981286' publications.title='The Drosophila TNF ortholog Eiger: emerging physiological roles and evolution of the TNF system.' publications.volume='26' publications.year=2014\n",
      "Gene: publications.abstractText='The Transforming Growth Factor-beta superfamily member decapentaplegic (dpp) acts as an extracellular morphogen to pattern the embryonic ectoderm of the Drosophila embryo. To identify components of the dpp signaling pathway, we screened for mutations that act as dominant maternal enhancers of a weak allele of the dpp target gene zerknŁllt. In this screen, we recovered new alleles of the Mothers against dpp (Mad) and Medea genes. Phenotypic analysis of the new Medea mutations indicates that Medea, like Mad, is required for both embryonic and imaginal disc patterning. Genetic analysis suggests that Medea may have two independently mutable functions in patterning the embryonic ectoderm. Complete elimination of maternal and zygotic Medea activity in the early embryo results in a ventralized phenotype identical to that of null dpp mutants, indicating that Medea is required for all dpp-dependent signaling in embryonic dorsal-ventral patterning. Injection of mRNAs encoding DPP or a constitutively activated form of the DPP receptor, Thick veins, into embryos lacking all Medea activity failed to induce formation of any dorsal cell fates, demonstrating that Medea acts downstream of the thick veins receptor. We cloned Medea and found that it encodes a protein with striking sequence similarity to human SMAD4. Moreover, injection of human SMAD4 mRNA into embryos lacking all Medea activity conferred phenotypic rescue of the dorsal-ventral pattern, demonstrating conservation of function between the two gene products.' publications.doi=None publications.firstAuthor='Hudson J B' publications.id=1007786 publications.issue='8' publications.journal='Development' publications.month='Apr' publications.pages='1407-20' publications.pubMedId='9502722' publications.title='The Drosophila Medea gene is required downstream of dpp and encodes a functional homolog of human Smad4.' publications.volume='125' publications.year=1998\n",
      "Gene: publications.abstractText='The conservation of Homeotic (Hox) gene clustering and colinearity in many metazoans indicates that functional constraints operate on this genome organization. However, several studies have questioned its relevance in Drosophila. Here, we analyse the genomic organization of Hox and Hox-derived genes in 13 fruitfly species and the mosquito Anopheles gambiae. We found that at least seven different Homeotic complex (HOM-C) arrangements exist among Drosophila species, produced by three major splits, five microinversions and six gene transpositions. This dynamism contrasts with the stable organization of the complex in many other taxa. Although there is no evidence of an absolute requirement for Hox gene clustering in Drosophila, we found that strong functional constraints act on the individual genes.' publications.doi='10.1016/j.tig.2006.12.001' publications.firstAuthor='Negre Bárbara' publications.id=1007633 publications.issue='2' publications.journal='Trends Genet.' publications.month='Feb' publications.pages='55-9' publications.pubMedId='17188778' publications.title='HOM-C evolution in Drosophila: is there a need for Hox gene clustering?' publications.volume='23' publications.year=2007\n",
      "Gene: publications.abstractText='The developmental control genes containing an Antennapedia-type homeobox are clustered in insects and vertebrates. The evolution of these genes was studied by the construction of evolutionary trees and by statistical geometry in sequence space. The comparative analysis of the homeobox sequences reveals the subdivision of the Antennapedia-type homeobox genes into three classes early in metazoan evolution. This observation suggests an important function of these genes even in the most primitive metazoans. Subsequent duplication events generated a cluster of at least five homeobox genes in the last common ancestor of insects and vertebrates. These genes later independently gave rise to the 13 groups of paralogous genes in vertebrates and to the 11 Antennapedia-type genes in the Drosophila complexes.' publications.doi=None publications.firstAuthor='Schubert F R' publications.id=1007861 publications.issue='1' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Jan' publications.pages='143-7' publications.pubMedId='8093557' publications.title='The Antennapedia-type homeobox genes have evolved from three precursors separated early in metazoan evolution.' publications.volume='90' publications.year=1993\n",
      "Gene: publications.abstractText='The discovery of conserved protein domains found in many Drosophila and mammalian developmental gene products suggests that fundamental developmental processes are conserved throughout evolution. Our understanding of development has been enhanced by the discovery of the widespread role of the homeodomain (HD). The action of HD-containing proteins as transcriptional regulators is mediated through a helix-turn-helix motif which confers sequence specific DNA binding. Unexpectedly, the well conserved structural homology between the HD and the prokaryotic helix-turn-helix proteins contrasts with their divergent types of physical interaction with DNA. A C-terminal extension of the HD recognition helix has assumed the role that the N-terminus of the prokaryotic helix plays for specification of DNA binding preference. However, the HD appears also capable of recognizing DNA in an alternative way and its specificity in vivo may be modified by regions outside the helix-turn-helix motif. We propose that this intrinsic complexity of the HD, as well as its frequent association with other DNA binding domains, explains the functional specificity achieved by genes encoding highly related HDs.' publications.doi='10.1002/bies.950140302' publications.firstAuthor='Treisman J' publications.id=1007862 publications.issue='3' publications.journal='Bioessays' publications.month='Mar' publications.pages='145-50' publications.pubMedId='1350195' publications.title='The homeodomain: a new face for the helix-turn-helix?' publications.volume='14' publications.year=1992\n",
      "Gene: publications.abstractText=\"The discovery of the striking positional conservation between the Antennapedia and Bithorax homeotic gene complexes (ANT-C and BX-C) in Drosophila melanogaster and the murine Hox and human HOX clusters has had a substantial impact on our understanding of the evolution of development and its genetic regulation. Structural differences do exist among the mammalian Hox complexes and the ANT-C in D. melanogaster. To gain further insight into the evolutionary changes among these complexes, the ANT-C was cloned in the closely related species, Drosophila pseudoobscura. The overall structure of the ANT-C in D. pseudoobscura is highly similar to its D. melanogaster counterpart; however, two differences in the organization of the ANT-C have been identified. First, the z2 gene, a member of the ANT-C in D. melanogaster, is not present in the D. pseudoobscura ANT-C and is possibly absent from the D. pseudoobscura genome. Second, the orientation of the Deformed gene is inverted in D. pseudoobscura, providing it with a 5' to 3' direction of transcription identical to the remaining ANT-C homeobox genes with the exception of fushi tarazu. These differences demonstrate that subtle changes can occur in ANT-C structure during relatively short periods of evolutionary divergence, although the fundamental organization of the complex is conserved. These observations and others suggest that the complex is not absolutely rigid but that selective pressures have maintained this organization of genes for some functional reason that remains elusive.\" publications.doi=None publications.firstAuthor='Randazzo F M' publications.id=1007812 publications.issue='1' publications.journal='Genetics' publications.month='May' publications.pages='319-30' publications.pubMedId='8099892' publications.title='Structural changes in the antennapedia complex of Drosophila pseudoobscura.' publications.volume='134' publications.year=1993\n",
      "Gene: publications.abstractText='The dorsal (dl) morphogen gradient is responsible for initiating the differentiation of the mesoderm, neuroectoderm and dorsal ectoderm in the Drosophila embryo. dl encodes a sequence-specific DNA binding protein that belongs to the Rel family of transcription factors. Previous studies have shown that dl activates the mesoderm determinant twist (twi); here we use a combination of site-directed mutagenesis and P-transformation assays to demonstrate that it also functions as a direct transcriptional repressor of a second target gene, zerknüllt (zen). By exchanging dl binding sites between the promoters we show that activator sites from twi can mediate repression when placed in the context of the zen promoter, and that repressor sites from zen can mediate activation in the context of the twi promoter. This represents the first demonstration that common binding sites for any DNA binding protein can mediate both activation and repression in a developing embryo. Evidence is also presented that the affinities of dl binding sites are important for the efficiency of repression, but are not the sole determinants of the threshold response to the dl gradient.' publications.doi=None publications.firstAuthor='Jiang J' publications.id=1007856 publications.issue='8' publications.journal='EMBO J.' publications.month='Aug' publications.pages='3147-54' publications.pubMedId='1322296' publications.title='Individual dorsal morphogen binding sites mediate activation and repression in the Drosophila embryo.' publications.volume='11' publications.year=1992\n",
      "Gene: publications.abstractText='The dorsal (dl) nuclear gradient initiates the differentiation of the mesoderm, neuroectoderm, and dorsal ectoderm by activating and repressing gene expression in the early Drosophila embryo. This gradient is organized by a Toll signaling pathway that shares many common features with the mammalian IL-1 cytokine pathway. Here we present evidence that a second signaling pathway, controlled by the torso (tor) receptor tyrosine kinase, also modulates dl activity. Evidence is presented that the tor pathway selectively masks the ability of dl to repress gene expression but has only a slight effect on activation. Intracellular kinases that are thought to function downstream of tor, such as D-raf and the rolled MAP kinase, mediate this selective block in repression. Normally, the Toll and tor pathways are both active only at the embryonic poles, and consequently, target genes (zen and dpp) that are repressed in middle body regions are expressed at these sites. Constitutive activation of the tor pathway causes severe embryonic defects, including disruptions in gastrulation and mesoderm differentiation, as a result of misregulation of dl target genes. These results suggest that RTK signaling pathways can control gene expression by antirepression, and that multiple pathways can fine-tune the activities of a single transcription factor.' publications.doi=None publications.firstAuthor='Rusch J' publications.id=1007826 publications.issue='11' publications.journal='Genes Dev.' publications.month='Jun' publications.pages='1247-57' publications.pubMedId='7926728' publications.title='Regulation of the dorsal morphogen by the Toll and torso signaling pathways: a receptor tyrosine kinase selectively masks transcriptional repression.' publications.volume='8' publications.year=1994\n",
      "Gene: publications.abstractText='The dorsal (dl) protein gradient determines patterns of gene expression along the dorsal-ventral axis of the Drosophila embryo. dl protein is at peak levels in ventral nuclei of the embryo where it activates some genes (twist and snail) and represses others [zerknullt (zen), decapentaplegic and tolloid]. It is a member of the rel family of transcription factors and interacts with specific DNA sequences in the regulatory regions of its target genes. These sequences (dl binding sites), when taken from the context of either an activated or repressed promoter, mediate transcriptional activation of a heterologous promoter, but not repression. We found that T-rich sequences close to the dl binding sites in the silencer region of the zen promoter are conserved between three Drosophila species. Using this sequence information we defined a minimal element that can mediate repression of a heterologous promoter. This element interacts with at least two factors present in embryonic extracts, one of which is dl protein. The other factor binds to the T-rich site. Point mutations in either site abolish ventral repression in vivo. In addition, mutations in the T-rich site cause ectopic expression in ventral regions indicating that the minimal silencer was converted into an enhancer.' publications.doi=None publications.firstAuthor='Kirov N' publications.id=1007843 publications.issue='8' publications.journal='EMBO J.' publications.month='Aug' publications.pages='3193-9' publications.pubMedId='8344256' publications.title='Conversion of a silencer into an enhancer: evidence for a co-repressor in dorsal-mediated repression in Drosophila.' publications.volume='12' publications.year=1993\n",
      "Gene: publications.abstractText='The dorsal-ventral pattern of the Drosophila egg is established during oogenesis. Epidermal growth factor receptor (Egfr) signaling within the follicular epithelium is spatially regulated by the dorsally restricted distribution of its presumptive ligand, Gurken. As a consequence, pipe is transcribed in a broad ventral domain to initiate the Toll signaling pathway in the embryo, resulting in a gradient of Dorsal nuclear translocation. We show that expression of pipe RNA requires the action of fettucine (fet) in ovarian follicle cells. Loss of maternal fet activity produces a dorsalized eggshell and embryo. Although similar mutant phenotypes are observed with regulators of Egfr signaling, genetic analysis suggests that fet acts downstream of this event. The fet mutant phenotype is rescued by a transgene of capicua (cic), which encodes an HMG-box transcription factor. We show that Cic protein is initially expressed uniformly in ovarian follicle cell nuclei, and is subsequently downregulated on the dorsal side. Earlier studies described a requirement for cic in repressing zygotic target genes of both the torso and Toll pathways in the embryo. Our experiments reveal that cic controls dorsal-ventral patterning by regulating pipe expression in ovarian follicle cells, before its previously described role in interpreting the Dorsal gradient.' publications.doi=None publications.firstAuthor='Goff D J' publications.id=1007821 publications.issue='22' publications.journal='Development' publications.month='Nov' publications.pages='4553-62' publications.pubMedId='11714680' publications.title='Establishment of dorsal-ventral polarity of the Drosophila egg requires capicua action in ovarian follicle cells.' publications.volume='128' publications.year=2001\n",
      "Gene: publications.abstractText='The dorsal-ventral pattern of the Drosophila embryo is established by three sequential signaling pathways. Each pathway transmits spatial information by localizing the activity of an extracellular signal, which acts as a ligand for a broadly distributed transmembrane receptor. The components of the first two pathways are encoded by maternal effect genes, while the third pathway is specified by genes expressed in the zygote. During oogenesis, the oocyte transmits a signal to the surrounding follicle cells by the gurken-torpedo pathway. After fertilization, the initial asymmetry of the egg chamber is used by the spätzle-Toll pathway to generate within the embryo a nuclear gradient of the transcription factor Dorsal, which regulates the regional expression of a set of zygotic genes. On the dorsal side of the embryo, the decapentaplegic-punt/thick veins pathway then establishes patterning of the amnioserosa and dorsal ectoderm. Each pathway uses a distinct strategy to achieve spatial localization of signaling activity.' publications.doi='10.1146/annurev.ge.29.120195.002103' publications.firstAuthor='Morisato D' publications.id=1002618 publications.issue=None publications.journal='Annu. Rev. Genet.' publications.month=None publications.pages='371-99' publications.pubMedId='8825480' publications.title='Signaling pathways that establish the dorsal-ventral pattern of the Drosophila embryo.' publications.volume='29' publications.year=1995\n",
      "Gene: publications.abstractText='The earliest stages of development in most metazoans aredriven by maternally deposited proteins and mRNAs, with widespread transcriptional activation of the zygotic genome occurring hours after fertilization, at a period known as the maternal-to-zygotic transition (MZT). In Drosophila, the MZT is preceded by the transcription of a small number of genes that initiate sex determination, patterning, and other early developmental processes; and the zinc-finger protein Zelda (ZLD) plays a key role in their transcriptional activation. To better understand the mechanisms of ZLD activation and the range of its targets, we used chromatin immunoprecipitation coupled with high-throughput sequencing (ChIP-Seq) to map regions bound by ZLD before (mitotic cycle 8), during (mitotic cycle 13), and after (late mitotic cycle 14) the MZT. Although only a handful of genes are transcribed prior to mitotic cycle 10, we identified thousands of regions bound by ZLD in cycle 8 embryos, most of which remain bound through mitotic cycle 14. As expected, early ZLD-bound regions include the promoters and enhancers of genes transcribed at this early stage. However, we also observed ZLD bound at cycle 8 to the promoters of roughly a thousand genes whose first transcription does not occur until the MZT and to virtually all of the thousands of known and presumed enhancers bound at cycle 14 by transcription factors that regulate patterned gene activation during the MZT. The association between early ZLD binding and MZT activity is so strong that ZLD binding alone can be used to identify active promoters and regulatory sequences with high specificity and selectivity. This strong early association of ZLD with regions not active until the MZT suggests that ZLD is not only required for the earliest wave of transcription but also plays a major role in activating the genome at the MZT.' publications.doi='10.1371/journal.pgen.1002266' publications.firstAuthor='Harrison Melissa M' publications.id=1007815 publications.issue='10' publications.journal='PLoS Genet.' publications.month='Oct' publications.pages='e1002266' publications.pubMedId='22028662' publications.title='Zelda binding in the early Drosophila melanogaster embryo marks regions subsequently activated at the maternal-to-zygotic transition.' publications.volume='7' publications.year=2011\n",
      "Gene: publications.abstractText='The establishment of expression domains of developmentally regulated genes depends on cues provided by different concentrations of transcriptional activators and repressors. Here we analyze the regulation of the Drosophila gene zen, which is a target of the Decapentaplegic (Dpp) signaling pathway during cellular blastoderm formation. We show that low levels of the Dpp signal transducer p-Mad (phosphorylated Mad), together with the recently discovered negative regulator Brinker (Brk), define the spatial limits of zen transcription in a broad dorsal-on/ventral-off domain. The subsequent refinement of this pattern to the dorsal-most cells, however, correlates with high levels of p-Mad that accumulate in the same region during late blastoderm. Examination of the zen regulatory sequences revealed the presence of multiple Mad and Brk binding sites, and our results indicate that a full occupancy of the Mad sites due to high concentrations of nuclear Mad is the primary mechanism for refinement of zen. Interestingly, several Mad and Brk binding sites overlap, and we show that Mad and Brk cannot bind simultaneously to such sites. We propose a model whereby competition between Mad and Brk determines spatially restricted domains of expression of Dpp target genes.' publications.doi='10.1101/gad.861401' publications.firstAuthor='Rushlow C' publications.id=1007684 publications.issue='3' publications.journal='Genes Dev.' publications.month='Feb' publications.pages='340-51' publications.pubMedId='11159914' publications.title='Transcriptional regulation of the Drosophila gene zen by competing Smad and Brinker inputs.' publications.volume='15' publications.year=2001\n",
      "Gene: publications.abstractText='The establishment of the dorsal-ventral pattern in Drosophila embryos depends on a signal transduction process: a putative extracellular ligand released into the perivitelline space surrounding the embryo binds to the Toll receptor. Toll activation triggers the formation of the nuclear gradient of dorsal protein, the morphogen of the dorsal-ventral axis. Here, I analyse the dorsal protein distribution and the expression of zygotic dorsal-ventral genes in Toll- embryos that have been injected with wild-type cytoplasm under a variety of different injection conditions. Injections into two positions within a single embryo lead to the formation of two dorsal-ventral patterns in one embryo, allowing the analysis of interactions between pattern-forming processes. The results of single and double injections suggest that the spatial information for the embryonic dorsal-ventral axis is largely derived from spatial cues present in the extraembryonic compartment, which restrict the release of the putative Toll ligand. They argue against a Toll-dependent pattern-formation process employing local self-enhancement and lateral inhibition to enhance a weak initial asymmetry. The putative Toll ligand appears to originate from a ventrally restricted zone which extends along the entire anterior-posterior axis. Ligand diffusion or its graded release are required to determine the slope of the nuclear dorsal protein gradient. Both the Toll receptor and the putative ligand of Toll are in excess in wild-type embryos. Since spatial information for the embryonic dorsal-ventral axis is already present in the vitelline membrane or the perivitelline space, it is most likely generated during oogenesis. Oogenic pattern formation is also responsible for the perpendicular orientation the dorsal-ventral axis maintains with respect to the anterior-posterior axis.' publications.doi=None publications.firstAuthor='Roth S' publications.id=1007802 publications.issue='4' publications.journal='Development' publications.month='Apr' publications.pages='1385-96' publications.pubMedId='8404539' publications.title='Mechanisms of dorsal-ventral axis determination in Drosophila embryos revealed by cytoplasmic transplantations.' publications.volume='117' publications.year=1993\n",
      "Gene: publications.abstractText='The existence of a gene complex in the proximal right arm of chromosome 3 of Drosophila melanogaster involved in the development of the head and thorax was originally suggested by the phenotypes of several dominant homoeotic mutations and their revertants. A screen for mutations utilizing Df(3R) Antp(Ns+R17) (proximally broken in salivary region 84B1,2) yielded, among 102 recovered mutations, 17 localized by deficiency mapping to the putative homoeotic cluster. These fell into four complementation groups, two of which were characterized by homoeotic phenotypes. To explore the limits of the Antennapedia gene complex (ANT-C) more proximally, a second screen has been undertaken utilizing Df(3R)Scr, a deficiency of 84A1-B1,2.-Of 2832 chromosomes screened, 21 bearing alterations localized to polytene interval 84A-84B1,2 have been recovered. Sixteen are recessive lethals, and five showing reduced viability display a visible phenotype in surviving individuals. Complementation and phenotypic analyses revealed four complementation groups proximal to those identified in the previous screen, including two new alleles of the recessive homoeotic mutation, proboscipedia (pb). Ten of the new mutations correspond to complementation groups defined previously in the Df(3R)Antp(Ns+R17) screen four to the EbR11 group, two to the Scr group and four to the Antp group.-On the basis of the phenotypes of the 39 mutations localized to this region, plus their interactions with extant homoeotic mutations, we postulate that there are at least five functional sites comprising the ANT-C. Three have been demonstrated to be homoeotic in nature. The specific homoeotic transformations thus far observed suggest that these loci are critical for normal development of adult labial, maxillary and thoracic structures.' publications.doi=None publications.firstAuthor='Lewis R A' publications.id=1005519 publications.issue='2' publications.journal='Genetics' publications.month='Jun' publications.pages='383-97' publications.pubMedId='17249042' publications.title='Genetic Analysis of the Antennapedia Gene Complex (Ant-C) and Adjacent Chromosomal Regions of DROSOPHILA MELANOGASTER. II. Polytene Chromosome Segments 84A-84B1,2.' publications.volume='95' publications.year=1980\n",
      "Gene: publications.abstractText=\"The expression of the chloramphenicol acetyltransferase gene under control of the 1152-base pair 5'-flanking region (-1107 to +45 nucleotide positions with respect to the major transcription initiation site) of the Drosophila DNA polymerase alpha gene was repressed by cotransfection into Drosophila Kc cells with a zerknüllt (zen)-expressing plasmid as previously observed with the proliferating cell nuclear antigen (PCNA) gene promoter. The expression of the zen resulted in reduction of the abundance of mRNA for the transfected chloramphenicol acetyltransferase gene and also mRNAs for both DNA polymerase alpha and PCNA. Results obtained using various deletion derivatives of the promoter region and chemically synthesized oligonucleotides of the DNA replication-related element (DRE), a positive cis-acting element found in both DNA polymerase alpha and PCNA genes, revealed that the DRE sequences are responsible to repression by Zen protein. The nuclear extract of Kc cells transfected by the zen-expressing plasmid contained lesser amounts of the DRE-binding factor (DREF) than that of untransfected or mutant zen-transfected cells. These results suggest that the Zen protein represses expression of DNA replication-related genes by reducing DREF, although the detailed mechanism of the repression remains to be elucidated.\" publications.doi=None publications.firstAuthor='Hirose F' publications.id=1007848 publications.issue='4' publications.journal='J. Biol. Chem.' publications.month='Jan' publications.pages='2937-42' publications.pubMedId='7905482' publications.title='Repression of regulatory factor for Drosophila DNA replication-related gene promoters by zerknüllt homeodomain protein.' publications.volume='269' publications.year=1994\n",
      "Gene: publications.abstractText=\"The family of Hox-proteins has been a major focus of research for over 30 years. Hox-proteins are crucial to the correct development of bilateral organisms, however, some uncertainty remains as to which Hox-proteins are functionally equivalent across different species. Initial classification of Hox-proteins was based on phylogenetic analysis of the 60 amino acid homeodomain. This approach was successful in classifying Hox-proteins with differing homeodomains, but the relationships of Hox-proteins with nearly identical homeodomains, yet distinct biological functions, could not be resolved. Correspondingly, these 'problematic' proteins were classified into one large unresolved group. Other classifications used the relative location of the Hox-protein coding genes on the chromosome (synteny) to further resolve this group. Although widely used, this synteny-based classification is inconsistent with experimental evidence from functional equivalence studies. These inconsistencies led us to re-examine and derive a new classification for the Hox-protein family using all Hox-protein sequences available in the GenBank non-redundant protein database (NCBI-nr). We compare the use of the homeodomain, the homeodomain with conserved flanking regions (the YPWM and linker region), and full length Hox-protein sequences as a basis for classification of Hox-proteins. In contrast to previous attempts, our approach is able to resolve the relationships for the 'problematic' as well as ABD-B-like Hox-proteins. We highlight differences to previous classifications and clarify the relationships of Hox-proteins across the five major model organisms, Caenorhabditis elegans, Drosophila melanogaster, Branchiostoma floridae, Mus musculus and Danio rerio. Comparative and functional analysis of Hox-proteins, two fields crucial to understanding the development of bilateral organisms, have been hampered by difficulties in predicting functionally equivalent Hox-proteins across species. Our classification scheme offers a higher-resolution classification that is in accordance with phylogenetic as well as experimental data and, thereby, provides a novel basis for experiments, such as comparative and functional analyses of Hox-proteins.\" publications.doi='10.1371/journal.pone.0010820' publications.firstAuthor='Hueber Stefanie D' publications.id=1007880 publications.issue='5' publications.journal='PLoS ONE' publications.month=None publications.pages='e10820' publications.pubMedId='20520839' publications.title='Improving Hox protein classification across the major model organisms.' publications.volume='5' publications.year=2010\n",
      "Gene: publications.abstractText='The flour beetle Tribolium castaneum has become an important model organism for comparative studies of insect development. Many developmentally important genes have now been cloned from both Tribolium and Drosophila and their expression characteristics were studied. We analyze here the complete coding sequences of 17 homologous gene pairs from D. melanogaster and T. castaneum, most of which encode transcription factors. We find that the Tribolium genes are on average 30% shorter than their Drosophila homologues. This appears to be due largely to the almost-complete absence of trinucleotide repeats in the coding sequences of Tribolium as well as the generally lower degree of internal repetitiveness. Clusters of polar and other amino acids such as glutamine, proline, and serine, which are often considered to be important for transcriptional activation domains in Drosophila, are almost completely absent in Tribolium. Codon usage is generally less biased in Tribolium, although we find a similar tendency for the preference of G- or C-ending codons and a higher bias in conserved subregions of the proteins as in Drosophila. Most of the aminoacid substitutions in the DNA-binding domains of the transcription factors occur at residues that do not make a specific contact to DNA, suggesting that the recognition sequences are likely to be conserved between the two species.' publications.doi=None publications.firstAuthor='Schmid K J' publications.id=1007683 publications.issue='5' publications.journal='J. Mol. Evol.' publications.month='Nov' publications.pages='558-66' publications.pubMedId='10552037' publications.title='A comparison of homologous developmental genes from Drosophila and Tribolium reveals major differences in length and trinucleotide repeat content.' publications.volume='49' publications.year=1999\n",
      "Gene: publications.abstractText='The fly Drosophila melanogaster is one of the most intensively studied organisms in biology and serves as a model system for the investigation of many developmental and cellular processes common to higher eukaryotes, including humans. We have determined the nucleotide sequence of nearly all of the approximately 120-megabase euchromatic portion of the Drosophila genome using a whole-genome shotgun sequencing strategy supported by extensive clone-based sequence and a high-quality bacterial artificial chromosome physical map. Efforts are under way to close the remaining gaps; however, the sequence is of sufficient accuracy and contiguity to be declared substantially complete and to support an initial analysis of genome structure and preliminary gene annotation and interpretation. The genome encodes approximately 13,600 genes, somewhat fewer than the smaller Caenorhabditis elegans genome, but with comparable functional diversity.' publications.doi=None publications.firstAuthor='Adams M D' publications.id=1002171 publications.issue='5461' publications.journal='Science' publications.month='Mar' publications.pages='2185-95' publications.pubMedId='10731132' publications.title='The genome sequence of Drosophila melanogaster.' publications.volume='287' publications.year=2000\n",
      "Gene: publications.abstractText='The fork head (fkh) domain defines the DNA-binding region of a family of transcription factors which has been implicated in regulating cell fate decisions across species lines. We have cloned and molecularly characterized the crocodile (croc) gene which encodes a new family member from Drosophila. croc is expressed in the head anlagen of the blastoderm embryo under the control of the anterior, the dorsoventral and the terminal maternal organizer systems. The croc mutant phenotype indicates that the croc wild-type gene is required to function as an early patterning gene in the anterior-most blastoderm head segment anlage and for the establishment of a specific head skeletal structure that derives from the non-adjacent intercalary segment at a later stage of embryogenesis. As an early patterning gene, croc exerts unusual properties which do not allow it to be grouped among the established segmentation genes. A single-site mutation within the croc fkh domain, which causes a replacement of the first out of four conserved amino acid residues thought to be involved in the coordinate binding of Mg2+, abolishes the DNA binding of the protein in vitro. In view of the resulting lack-of-function mutant phenotype, it appears likely that metal binding by the affected region of the fkh domain is crucial for proper folding of the DNA-binding structure.' publications.doi=None publications.firstAuthor='Häcker U' publications.id=1007692 publications.issue='21' publications.journal='EMBO J.' publications.month='Nov' publications.pages='5306-17' publications.pubMedId='7489720' publications.title='The Drosophila fork head domain protein crocodile is required for the establishment of head structures.' publications.volume='14' publications.year=1995\n",
      "Gene: publications.abstractText='The gene regulatory network of a developmental process contains many mutually repressive interactions between two genes. They are often regulated by or regulate an additional factor, which constitute prominent network motifs, called regulated and regulating mutual loops. Our database analysis on the gene regulatory network for Drosophila melanogaster indicates that those with mutual repression are working specifically for the segmentation process. To clarify their biological roles, we mathematically study the response of the regulated mutual loop with mutual repression to input stimuli. We show that the mutual repression increases the response sensitivity without affecting the threshold input level to activate the target gene expression, as long as the network output is unique for a given input level. This high sensitivity of the motif can contribute to sharpening the spatial domain pattern without changing its position, assuring a robust developmental process. We also study transient dynamics that shows shift of domain boundary, agreeing with experimental observations. Importance of mutual repression is addressed by comparingwith other types of regulations.' publications.doi='10.1016/j.jtbi.2008.01.027' publications.firstAuthor='Ishihara Shuji' publications.id=1007779 publications.issue='1' publications.journal='J. Theor. Biol.' publications.month='May' publications.pages='131-44' publications.pubMedId='18342890' publications.title='Mutual interaction in network motifs robustly sharpens gene expression in developmental processes.' publications.volume='252' publications.year=2008\n",
      "Gene: publications.abstractText='The general consensus in the field is that limiting amounts of the transcription factor Dorsal establish dorsal boundaries of genes expressed along the dorsal-ventral (DV) axis of early Drosophila embryos, while repressors establish ventral boundaries. Yet recent studies have provided evidence that repressors act to specify the dorsal boundary of intermediate neuroblasts defective (ind), a gene expressed in a stripe along the DV axis in lateral regions of the embryo. Here we show that a short 12 base pair sequence (\"the A-box\") present twice within the ind CRM is both necessary and sufficient to support transcriptional repression in dorsal regions of embryos. To identify binding factors, we conducted affinity chromatography using the A-box element and found a number of DNA-binding proteins and chromatin-associated factors using mass spectroscopy. Only Grainyhead (Grh), a CP2 transcription factor with a unique DNA-binding domain, was found to bind the A-box sequence. Our results suggest that Grh acts as an activator to support expression of ind, which was surprising as we identified this factor using an element that mediates dorsally-localized repression. Grh and Dorsal both contribute to ind transcriptional activation. However, another recent study found that the repressor Capicua (Cic) also binds to the A-box sequence. While Cic was not identified through our A-box affinity chromatography, utilization of the same site, the A-box, by both factors Grh (activator) and Cic (repressor) may also support a \"switch-like\"response that helps to sharpen the ind dorsal boundary. Furthermore, our results also demonstrate that TGF-β signaling acts to refine ind CRM expression in an A-box independent manner in dorsal-most regions, suggesting that tiers of repression act in dorsal regions of the embryo.' publications.doi='10.1371/journal.pone.0029172' publications.firstAuthor='Garcia Mayra' publications.id=1007867 publications.issue='12' publications.journal='PLoS ONE' publications.month=None publications.pages='e29172' publications.pubMedId='22216201' publications.title='Lateral gene expression in Drosophila early embryos is supported by Grainyhead-mediated activation and tiers of dorsally-localized repression.' publications.volume='6' publications.year=2011\n",
      "Gene: publications.abstractText='The genomic and cDNA clones for a Drosophila melanogaster proliferating cell nuclear antigen (PCNA) (cyclin) were isolated and sequenced. The coding sequence for a 260-amino-acid residue polypeptide was interrupted by a single short intron of 60 base pairs (bp), and about 70% of the deduced amino acid sequence of the Drosophila PCNA was identical to the rat and human PCNA polypeptides, with conserved unique repeats of leucine in the C-terminal region. Genomic Southern blot hybridization analysis indicates the presence of a single gene for PCNA per genome. The PCNA mRNA was detected at a high level in adult ovaries, unfertilized eggs, and early embryos and at low levels in the other developmental stages. The major transcription initiation site (cap site) was localized at 89 bp upstream from the ATG codon. Neither a TATA box nor a CAAT box was found within the 600-bp region upstream of the cap site. Clusters of 10 bp of sequence similar to the binding sites for Drosophila proteins containing homeodomains were found in the region from -127 to -413. DNase I footprint analysis revealed that the Drosophila homeodomain proteins coded by even-skipped and zerknüllt genes can specifically bind to these sites. These results suggest that the expression of the PCNA gene is under the control of genes coding for homeodomain proteins.' publications.doi=None publications.firstAuthor='Yamaguchi M' publications.id=1007835 publications.issue='3' publications.journal='Mol. Cell. Biol.' publications.month='Mar' publications.pages='872-9' publications.pubMedId='1968224' publications.title=\"Drosophila proliferating cell nuclear antigen (cyclin) gene: structure, expression during development, and specific binding of homeodomain proteins to its 5'-flanking region.\" publications.volume='10' publications.year=1990\n",
      "Gene: publications.abstractText='The genomic program for development operates primarily by the regulated expression of genes encoding transcription factors and components of cell signaling pathways. This program is executed by cis-regulatory DNAs (e.g., enhancers and silencers) that control gene expression. The regulatory inputs and functional outputs of developmental control genes constitute network-like architectures. In this PNAS Special Feature are assembled papers on developmental gene regulatory networks governing the formation of various tissues and organs in nematodes, flies, sea urchins, frogs, and mammals. Here, we survey salient points of these networks, by using as reference those governing specification of the endomesoderm in sea urchin embryos and dorsal-ventral patterning in the Drosophila embryo.' publications.doi='10.1073/pnas.0408031102' publications.firstAuthor='Levine Michael' publications.id=1002604 publications.issue='14' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Apr' publications.pages='4936-42' publications.pubMedId='15788537' publications.title='Gene regulatory networks for development.' publications.volume='102' publications.year=2005\n",
      "Gene: publications.abstractText='The graded nuclear location of the transcription factor Dorsal along the dorsoventral axis of the early Drosophila embryo provides positional information for the determination of different cell fates. Nuclear uptake of Dorsal depends on a complex signalling pathway comprising two parts: an extracellular proteolytic cascade transmits the dorsoventral polarity of the egg chamber to the early embryo and generates a gradient of active Spätzle protein, the ligand of the receptor Toll; an intracellular cascade downstream of Toll relays this graded signal to embryonic nuclei. The slope of the Dorsal gradient is not determined by diffusion of extracellular or intracellular components from a local source, but results from self-organised patterning, in which positive and negative feedback is essential to create and maintain the ratio of key factors at different levels, thereby establishing and stabilising the graded spatial information for Dorsal nuclear uptake.' publications.doi='10.1016/j.cub.2005.10.026' publications.firstAuthor='Moussian Bernard' publications.id=1002675 publications.issue='21' publications.journal='Curr. Biol.' publications.month='Nov' publications.pages='R887-99' publications.pubMedId='16271864' publications.title='Dorsoventral axis formation in the Drosophila embryo--shaping and transducing a morphogen gradient.' publications.volume='15' publications.year=2005\n",
      "Gene: publications.abstractText='The homeo box, which encodes the DNA-binding homeo domain, is a DNA sequence motif present in several Drosophila developmental genes; it has been used to identify many homologous genes involved in mammalian development. The paired box is another conserved sequence motif, first identified in the paired (prd) and gooseberry (gsb) Drosophila homeo domain genes. It encodes a 128-amino-acid domain, the paired domain, which has since been found in other fly and mouse gene products, in association with the homeo domain or in its absence. We show that the paired box of the prd gene encodes a DNA-binding activity, independent of the DNA-binding activity of the Paired (Prd) homeo domain and with a different sequence specificity. The amino-terminal region of the paired domain, including one of the three predicted alpha-helices, is necessary and sufficient for binding. We investigate the binding of the Prd protein to two sites in the even-skipped promoter, which are composed of overlapping sequences bound by the homeo domain and by the paired domain. We also show that a mutation in the paired box of Prd, corresponding to the mutation in the paired box of the mouse Pax-1 gene thought to cause the undulated skeletal phenotype, destroys the ability of the Prd protein to bind to the paired domain-specific site. This supports the view that the undulated phenotype results from the inactivation of the DNA-binding activity of the paired domain of Pax-1.' publications.doi=None publications.firstAuthor='Treisman J' publications.id=1007770 publications.issue='4' publications.journal='Genes Dev.' publications.month='Apr' publications.pages='594-604' publications.pubMedId='1672661' publications.title='The paired box encodes a second DNA-binding domain in the paired homeo domain protein.' publications.volume='5' publications.year=1991\n",
      "Gene: publications.abstractText='The homeobox gene family of Drosophila appears to control a variety of position-specific patterning decisions during embryonic and imaginal development. Most of these patterning decisions determine groups of cells on the anterior-posterior axis of the Drosophila germ band. We have isolated a novel homeobox gene from Drosophila, designated H2.0. H2.0 has the most diverged homeobox so far characterized in metazoa, and, in contrast to all previously isolated homeobox genes, H2.0 exhibits a tissue-specific pattern of expression. The cells that accumulate transcripts for this novel gene correspond to the visceral musculature and its anlagen.' publications.doi=None publications.firstAuthor='Barad M' publications.id=1007759 publications.issue='7' publications.journal='EMBO J.' publications.month='Jul' publications.pages='2151-61' publications.pubMedId='2901348' publications.title='A novel, tissue-specific, Drosophila homeobox gene.' publications.volume='7' publications.year=1988\n",
      "Gene: publications.abstractText=\"The homoeobox gene zerknüllt (zen) plays an important role in the differentiation of dorsal tissues during Drosophila development. zen- embryos show transformations in the dorsal-most regions of the fate map, and lack several tissues that normally derive from these regions, including the amnioserosa and optic lobe. zen displays a simple dorsal on/ventral off pattern as early as cleavage cycle 10-11 (ref. 2). We have prepared a polyclonal antibody against a full-length zen protein, and used this to examine its pattern of expression in mutants that disrupt dorsal-ventral polarity. Most or all of the maternally expressed genes that are involved in this process have been previously identified and fall into two classes, so called 'dorsalizers' and 'ventralizers' (see refs 4-7, reviewed in ref. 8). On the basis of our analysis of zen expression in each of these maternal mutants we propose that one or more of the dorsalizing genes encodes a repressor which inhibits the expression of zen in ventral regions of developing embryos. The ventralizing gene cactus might play an important role in restricting the activity of this repressor to ventral regions, thereby permitting the activation of zen in those dorsal tissues where its function is critically required.\" publications.doi='10.1038/330583a0' publications.firstAuthor='Rushlow C' publications.id=1007855 publications.issue='6148' publications.journal='Nature' publications.month=None publications.pages='583-6' publications.pubMedId='2891036' publications.title='Maternal regulation of zerknüllt: a homoeobox gene controlling differentiation of dorsal tissues in Drosophila.' publications.volume='330' publications.year=None\n",
      "Gene: publications.abstractText='The immunoglobulin superfamily is a diverse group of proteins that are involved in various aspects of cell surface recognition. Here, we report the characterization of amalgam (ama), a gene in the Antennapedia complex (ANT-C) of D. melanogaster that exhibits amino acid similarity to vertebrate neural cell adhesion molecules and other members of the immunoglobulin superfamily. The putative 333 amino acid ama protein consists of a signal sequence, three immunoglobulin-like domains, and a short slightly hydrophobic carboxy-terminal region. Antibodies against the ama protein reveal that it accumulates on the surface of various mesodermal and neural cells during embryogenesis. The function of this protein remains elusive, as no mutations have been recovered for ama during saturation EMS mutagenesis of this chromosomal region.' publications.doi=None publications.firstAuthor='Seeger M A' publications.id=1007814 publications.issue='4' publications.journal='Cell' publications.month='Nov' publications.pages='589-600' publications.pubMedId='3141062' publications.title='Characterization of amalgam: a member of the immunoglobulin superfamily from Drosophila.' publications.volume='55' publications.year=1988\n",
      "Gene: publications.abstractText='The maternal Dorsal regulatory gradient initiates the differentiation of several tissues in the early Drosophila embryo. Whole-genome microarray assays identified as many as 40 new Dorsal target genes, which encode a broad spectrum of cell signaling proteins and transcription factors. Evidence is presented that a tissue-specific form of the NF-Y transcription complex is essential for the activation of gene expression in the mesoderm. Tissue-specific enhancers were identified for new Dorsal target genes, and bioinformatics methods identified conserved cis-regulatory elements for coordinately regulated genes that respond to similar thresholds of the Dorsal gradient. The new Dorsal target genes and enhancers represent one of the most extensive gene networks known for any developmental process.' publications.doi=None publications.firstAuthor='Stathopoulos Angelike' publications.id=1007766 publications.issue='5' publications.journal='Cell' publications.month='Nov' publications.pages='687-701' publications.pubMedId='12464180' publications.title='Whole-genome analysis of dorsal-ventral patterning in the Drosophila embryo.' publications.volume='111' publications.year=2002\n",
      "Gene: publications.abstractText='The maternal determinants of dorsoventral polarity of the Drosophila embryo are derived from somatic and germ-line components of the egg chamber. During oogenesis, asymmetry seems to be established by a signal transduction process. This process is thought to provide the developing embryo with a ventral signal responsible for determining the embryonic axis. Through a set of interactions that may involve signal transduction and proteolytic cascade events, positional information is generated in the form of a graded distribution of dorsal protein in blastoderm nuclei. Different levels of dorsal protein result in asymmetric expression of zygotic genes that ultimately specify cell fate.' publications.doi=None publications.firstAuthor='Govind S' publications.id=1002591 publications.issue='4' publications.journal='Trends Genet.' publications.month='Apr' publications.pages='119-25' publications.pubMedId='2068782' publications.title='Dorsoventral pattern formation in Drosophila: signal transduction and nuclear targeting.' publications.volume='7' publications.year=1991\n",
      "Gene: publications.abstractText='The maternal transcription factor Dorsal (Dl) functions as both an activator and a repressor in a context-dependent manner to control dorsal-ventral patterning in the Drosophila embryo. Previous studies have suggested that Dl is an intrinsic activator and its repressive activity requires additional corepressors that bind corepressor-binding sites near Dl-binding sites. However, the molecular identities of the corepressors have yet to be identified. Here, we present evidence that Capicua (Cic) is involved in Dl-mediated repression in the zerknüllt (zen) ventral repression element (VRE). Computational and genetic analyses indicate that a DNA-binding consensus sequence of Cic is highly analogous with previously identified corepressor-binding sequences and that Dl failed to repress zen expression in lateral regions of cic mutant embryos. Furthermore, electrophoretic mobility shift assay (EMSA) shows that Cic directly interacts with several corepressor-binding sites in the zen VRE. These results suggest that Cic may function as a corepressor by binding the VRE.' publications.doi=None publications.firstAuthor='Shin Dong-Hyeon' publications.id=1007834 publications.issue='9' publications.journal='BMB Rep' publications.month='Sep' publications.pages='518-23' publications.pubMedId='25059278' publications.title='Capicua is involved in Dorsal-mediated repression of zerknüllt expression in Drosophila embryo.' publications.volume='47' publications.year=2014\n",
      "Gene: publications.abstractText='The mechanisms that generate precise patterns of discrete cell types within developing fields are not well understood. One model for analyzing how cells interpret positional information in two dimensions is the regulation of proneural cluster formation within insect segments. Two adjacent proneural regulatory genes, achaete and scute, are expressed coincidently in cell clusters at reproducible anteroposterior (AP) and dorsoventral (DV) coordinates within the Drosophila embryo from which single neuroblasts later arise. Here, we show that the AP and DV position of these clusters is regulated through a common cis-acting region between the genes under the initial control of the products of the pair-rule and DV polarity genes and is later maintained by selected segment polarity genes. The combination of proneural gene activation/repression in AP stripes and repression within specific DV domains positions each cluster of achaete/scute expressing cells within segments; interactions between these cells then determine individual cell fates.' publications.doi=None publications.firstAuthor='Skeath J B' publications.id=1007824 publications.issue='12B' publications.journal='Genes Dev.' publications.month='Dec' publications.pages='2606-19' publications.pubMedId='1340472' publications.title='Gene regulation in two dimensions: the proneural achaete and scute genes are controlled by combinations of axis-patterning genes through a common intergenic control region.' publications.volume='6' publications.year=1992\n",
      "Gene: publications.abstractText='The members of the evolutionarily conserved Hox-gene complex, termed Hox genes, are required for specifying segmental identity during embryogenesis in various animal phyla. The Hox3 genes of winged insects have lost this ancestral function and are required for the development of extraembryonic epithelia, which do not contribute to any larval structure. Higher flies (Cyclorrhapha) such as Drosophila melanogaster contain Hox3 genes of two types, the zerknüllt type and the bicoid type. The zerknüllt gene is expressed zygotically on the dorsal side of the embryo and is required for establishing extraembryonic tissue. Its sister gene bicoid is expressed maternally and the transcripts are localized at the anterior pole of the mature egg. BICOID protein, which emerges from this localized source during early development, is required for embryonic patterning. All known direct bicoid homologues are confined to Cyclorrhaphan flies. Here, we describe Hox3 genes of the non-Cyclorrhaphan flies Empis livida (Empididae), Haematopota pluvialis (Tabanidae), and Clogmia albipunctata (Psychodidae). The gene sequences are more similar to zerknüllt homologues than to bicoid homologues, but they share expression characteristics of both genes. We propose that an ancestral Hox3 gene had been duplicated in the stem lineage of Cyclorrhaphan flies. During evolution, one of the gene copies lost maternal expression and evolved as zerknüllt, whereas the second copy lost zygotic expression and evolved as bicoid. Our finding correlates well with a partial reduction of zerknüllt-dependent extraembryonic tissue during Dipteran evolution.' publications.doi='10.1073/pnas.012292899' publications.firstAuthor='Stauber Michael' publications.id=1007745 publications.issue='1' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Jan' publications.pages='274-9' publications.pubMedId='11773616' publications.title='A single Hox3 gene with composite bicoid and zerknullt expression characteristics in non-Cyclorrhaphan flies.' publications.volume='99' publications.year=2002\n",
      "Gene: publications.abstractText='The organization of the anterior pattern in the Drosophila embryo is mediated by the maternal effect gene bicoid. bcd has been identified in an 8.7-kb genomic fragment by germ line transformants that completely rescue the mutant phenotype. The major transcript of 2.6 kb includes a homeobox with low homology to previously known homeoboxes, a PRD-repeat and a M-repeat. In situ hybridizations reveal that bcd is transcribed in the nurse cells. The mRNA is localized at the anterior tip of oocyte and early embryo until the cellular blastoderm stage. The localization of the transcript requires the function of the maternal effect genes exuperantia and swallow while transcript stability is reduced by functions depending on posterior group genes.' publications.doi=None publications.firstAuthor='Berleth T' publications.id=1007789 publications.issue='6' publications.journal='EMBO J.' publications.month='Jun' publications.pages='1749-56' publications.pubMedId='2901954' publications.title='The role of localization of bicoid RNA in organizing the anterior pattern of the Drosophila embryo.' publications.volume='7' publications.year=1988\n",
      "Gene: publications.abstractText='The origin of evolutionary novelty is believed to involve both positive selection and relaxed developmental constraint. In flies, the redesign of anterior patterning during embryogenesis is a major developmental innovation and the rapidly evolving Hox gene, bicoid (bcd), plays a critical role. We report evidence for relaxation of selective constraint acting on bicoid as a result of its maternal pattern of gene expression. Evolutionary theory predicts 2-fold greater sequence diversity for maternal effect genes than for zygotically expressed genes, because natural selection is only half as effective acting on autosomal genes expressed in one sex as it is on genes expressed in both sexes. We sample an individual from ten populations of Drosophila melanogaster and nine populations of D. simulans for polymorphism in the tandem gene duplicates bcd, which is maternally expressed, and zerknüllt (zen), which is zygotically expressed. In both species, we find the ratio of bcd to zen nucleotide diversity to be two or more in the coding regions but one in the noncoding regions, providing the first quantitative support for the theoretical prediction of relaxed selective constraint on maternal-effect genes resulting from sex-limited expression. Our results suggest that the accelerated rate of evolution observed for bcd is owing, at least partly, to variation generated by relaxed selective constraint.' publications.doi='10.1371/journal.pgen.0010057' publications.firstAuthor='Barker Michael S' publications.id=1007755 publications.issue='5' publications.journal='PLoS Genet.' publications.month='Nov' publications.pages='e57' publications.pubMedId='16299585' publications.title='Maternal expression relaxes constraint on innovation of the anterior determinant, bicoid.' publications.volume='1' publications.year=2005\n",
      "Gene: publications.abstractText=\"The passage of an individual's genome to future generations is essential for the maintenance of species and is mediated by highly specialized cells, the germ cells. Genetic studies in a number of model organisms have provided insight into the molecular mechanisms that control specification, migration and survival of early germ cells. Focusing on Drosophila, we will discuss the mechanisms by which germ cells initially form and remain transcriptionally silent while somatic cells are transcriptionally active. We will further discuss three separate attractiveand repellent guidance pathways, mediated by a G-protein coupled receptor, two lipid phosphate phosphohydrolases, and isoprenylation. We will compare and contrast these findings with those obtained in other organisms, in particular zebrafish and mice. While aspects of germ cell specification are strikingly different between these species, germ cell specific gene functions have been conserved. In particular, mechanisms that sense directional cues during germ cell migration seem to be shared between invertebrates and vertebrates.\" publications.doi='10.1016/j.cub.2004.07.018' publications.firstAuthor='Santos Ana C' publications.id=1007767 publications.issue='14' publications.journal='Curr. Biol.' publications.month='Jul' publications.pages='R578-89' publications.pubMedId='15268881' publications.title='Germ cell specification and migration in Drosophila and beyond.' publications.volume='14' publications.year=2004\n",
      "Gene: publications.abstractText='The position of the nucleus along the anterior rim of stage 8 Drosophila oocytes presages the dorsal side of the egg and the developing embryo. In this paper, we address the question of whether the oocyte has a previously determined dorsal side to which the nucleus is drawn, or whether nuclear position randomly determines the dorsal side. To do so, we have taken advantage of a genetic system in which Drosophila oocytes occasionally become binuclear. We find that (i) the two nuclei migrate independently to their respective positions on the anterior rim, sometimes selecting the same site, sometimes not, (ii) the two nuclei are equivalent in their ability to induce a dorsal-ventral pattern in the overlying follicular epithelium, and (iii) at any position around the anterior circumference of the egg chamber the follicle cell sheet is equally responsive to the Gurken signal associated with the oocyte nuclei. These results argue that the dorsal-ventral axis is determined arbitrarily by the randomly selected position of the nucleus on the anterior rim of the oocyte. Some of the binuclear eggs support embryonic development. However, despite the duplication of dorsal chorion structures, the majority of such embryos show normal dorsal-ventral patterning. Thus, processes exist in the ventral follicular epithelium or in the perivitelline space that compensate for the expansion of dorsal follicle cell fates and consequently allow the formation of a normal embryonic axis.' publications.doi=None publications.firstAuthor='Roth S' publications.id=1007870 publications.issue='5' publications.journal='Development' publications.month='Feb' publications.pages='927-34' publications.pubMedId='9927594' publications.title='Binuclear Drosophila oocytes: consequences and implications for dorsal-ventral patterning in oogenesis and embryogenesis.' publications.volume='126' publications.year=1999\n",
      "Gene: publications.abstractText='The pre-cellular Drosophila embryo contains 10 well characterized sequence-specific transcriptional repressors, which represent a broad spectrum of DNA-binding proteins. Previous studies have shown that two of the repressors, Hairy and Dorsal, recruit a common co-repressor protein, Groucho. Here we present evidence that three different repressors, Knirps, Krüppel and Snail, recruit a different co-repressor, dCtBP. Mutant embryos containing diminished levels of maternal dCtBP products exhibit both segmentation and dorsoventral patterning defects, which can be attributed to loss of Krüppel, Knirps and Snail activity. In contrast, the Dorsal and Hairy repressors retain at least some activity in dCtBP mutant embryos. dCtBP interacts with Krüppel, Knirps and Snail through a related sequence motif, PXDLSXK/H. This motif is essential for the repression activity of these proteins in transgenic embryos. We propose that dCtBP represents a major form of transcriptional repression in development, and that the Groucho and dCtBP co-repressors mediate separate pathways of repression.' publications.doi='10.1093/emboj/17.23.7009' publications.firstAuthor='Nibu Y' publications.id=1007750 publications.issue='23' publications.journal='EMBO J.' publications.month='Dec' publications.pages='7009-20' publications.pubMedId='9843507' publications.title='dCtBP mediates transcriptional repression by Knirps, Krüppel and Snail in the Drosophila embryo.' publications.volume='17' publications.year=1998\n",
      "Gene: publications.abstractText=\"The principles of embryonic pattern formation have been studied extensively in many systems using classical experimental approaches. In Drosophila, a powerful combination of genetics and transplantation experiments, as well as molecular biology, have helped to elucidate the mechanisms that operate during oogenesis and early embryogenesis to establish a set of positional cues required for axis determination in the early embryo. In systematic searches for maternal effect mutations a small number of about 30 genes have been identified that specifically affect the process of determination of the embryonic axes. These 'coordinate' genes define four systems that determine the anteroposterior (AP) axis (three systems) and the dorsoventral (DV) axis (one system) independently. In the anteroposterior axis, the anterior system determines the segmented region of head and thorax, the posterior system determines the segmented abdominal region, and the terminal system is responsible for the formation of the nonsegmented termini at the anterior and posterior egg tips, the acron and telson. In contrast, pattern along the dorsoventral axis is determined by one system only. Although all four systems use different biochemical mechanisms, they share several properties. (1) The product of one gene in each system is localized in a specific region of the freshly laid egg and functions as a spatial signal. (2) In each system, this spatial information finally results in the asymmetrical distribution of one gene product that functions as a transcription factor. (3) This transcription factor is distributed in a concentration gradient that defines the spatial limits of expression of one or more zygotic target genes. The combined action of these three anteroposterior systems as well as the dorsoventral system defines the expression of zygotic target genes in at least seven distinct regions along the anteroposterior and at least three in the dorsoventral axis. These longitudinal and transverse domains provide a coarse spatial prepattern which is then further refined by the action and interaction of zygotic pattern genes.\" publications.doi=None publications.firstAuthor='Nüsslein-Volhard C' publications.id=1002702 publications.issue=None publications.journal='Dev. Suppl.' publications.month=None publications.pages='1-10' publications.pubMedId='1742496' publications.title='Determination of the embryonic axes of Drosophila.' publications.volume='1' publications.year=1991\n",
      "Gene: publications.abstractText='The recent completion of the Drosophila melanogaster genomic sequence to high quality and the availability of a greatly expanded set of Drosophila cDNA sequences, aligning to 78% of the predicted euchromatic genes, afforded FlyBase the opportunity to significantly improve genomic annotations. We made the annotation process more rigorous by inspecting each gene visually, utilizing a comprehensive set of curation rules, requiring traceable evidence for each gene model, and comparing each predicted peptide to SWISS-PROT and TrEMBL sequences. Although the number of predicted protein-coding genes in Drosophila remains essentially unchanged, the revised annotation significantly improves gene models, resulting in structural changes to 85% of the transcripts and 45% of the predicted proteins. We annotated transposable elements and non-protein-coding RNAs as new features, and extended the annotation of untranslated (UTR) sequences and alternative transcripts to include more than 70% and 20% of genes, respectively. Finally, cDNA sequence provided evidence for dicistronic transcripts, neighboring genes with overlapping UTRs on the same DNA sequence strand, alternatively spliced genes that encode distinct, non-overlapping peptides, and numerous nested genes. Identification of so many unusual gene models not only suggests that some mechanisms for gene regulation are more prevalent than previously believed, but also underscores the complex challenges of eukaryotic gene prediction. At present, experimental data and human curation remain essential to generate high-quality genome annotations.' publications.doi=None publications.firstAuthor='Misra Sima' publications.id=1000884 publications.issue='12' publications.journal='Genome Biol.' publications.month=None publications.pages='RESEARCH0083' publications.pubMedId='12537572' publications.title='Annotation of the Drosophila melanogaster euchromatic genome: a systematic review.' publications.volume='3' publications.year=2002\n",
      "Gene: publications.abstractText='The repetitive DNA that constitutes most of the heterochromatic regions of metazoan genomes has hindered the comprehensive analysis of gene content and other functions. We have generated a detailed computational and manual annotation of 24 megabases of heterochromatic sequence in the Release 5 Drosophila melanogaster genome sequence. The heterochromatin contains a minimum of 230 to 254 protein-coding genes, which are conserved in other Drosophilids and more diverged species, as well as 32 pseudogenes and 13 noncoding RNAs. Improved methods revealed that more than 77% of this heterochromatin sequence, including introns and intergenic regions, is composed of fragmented and nested transposable elements and other repeated DNAs. Drosophila heterochromatin contains \"islands\"of highly conserved genes embedded in these \"oceans\"of complex repeats, which may require special expression and splicing mechanisms.' publications.doi='10.1126/science.1139815' publications.firstAuthor='Smith Christopher D' publications.id=1895606 publications.issue='5831' publications.journal='Science' publications.month='Jun' publications.pages='1586-91' publications.pubMedId='17569856' publications.title='The Release 5.1 annotation of Drosophila melanogaster heterochromatin.' publications.volume='316' publications.year=2007\n",
      "Gene: publications.abstractText='The retinal determination gene dachshund is distantly related to the family of Ski/Sno proto-oncogenes and influences the development of a wide range of tissues including the embryonic head, optic lobes, brain, central nervous system as well as the post-embryonic leg, wing, genital and eye-antennal discs. We were interested in the regulatory mechanisms that control the dynamic expression pattern of dachshund and in this report we set out to ascertain how the transcription of dachshund is modulated in the embryonic head and developing eye-antennal imaginal disc. We demonstrate that the TGFbeta signaling cascade, the transcription factor zerknullt and several other patterning genes prevent dachshund from being expressed inappropriately within the embryonic head. Additionally, we show that several members of the eye specification cascade influence the transcription of dachshund during normal and ectopic eye development. Our results suggest that dachshund is regulated by a complex combinatorial code of transcription factors and signaling pathways. Unraveling this code may lead to an understanding of how dachshund regulates the development of many diverse tissue types including the eye.' publications.doi='10.1016/j.ydbio.2006.05.004' publications.firstAuthor='Anderson Jason' publications.id=1007896 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Sep' publications.pages='536-49' publications.pubMedId='16780828' publications.title='Regulation of the retinal determination gene dachshund in the embryonic head and developing eye of Drosophila.' publications.volume='297' publications.year=2006\n",
      "Gene: publications.abstractText='The rho family of GTPases has been implicated in regulating changes in cell morphology in response to extracellular signals. We have cloned three widely expressed members of this family from Drosophila melanogaster; a rho homologue (Rho1) and two rac homologues (Rac1 and Rac2). Flies harbouring a Rho1 transgene that is specifically expressed in the eye exhibit a dramatic dose dependent disruption of normal eye development. Flies bearing at least two copies of the transgene display a severe rough eye phenotype characterized by missing secondary and tertiary pigment cells, a substantial reduction in the number of photoreceptor cells and a grossly abnormal morphology of the rhabdomeres. Cell fate determination in the imaginal disc occurs normally and abnormalities become manifest late in pupariation, coincident with the phase when the cells undergo major morphological changes. This phenotype is modified by mutations at several other loci that have been implicated in signal transduction, but not by mutations in ras pathway components.' publications.doi=None publications.firstAuthor='Hariharan I K' publications.id=1007713 publications.issue='2' publications.journal='EMBO J.' publications.month='Jan' publications.pages='292-302' publications.pubMedId='7835340' publications.title='Characterization of rho GTPase family homologues in Drosophila melanogaster: overexpressing Rho1 in retinal cells causes a late developmental defect.' publications.volume='14' publications.year=1995\n",
      "Gene: publications.abstractText='The short gastrulation (sog) gene is expressed in broad lateral stripes comprising the neuroectoderm of the Drosophila blastoderm embryo. sog encodes a predicted secreted protein that functions nonautonomously to antagonize the activity of the TGF-beta-like Decapentaplegic (Dpp) signaling pathway in the dorsal region of the embryo. Recently, it has been shown that sog and dpp are functionally equivalent to their respective Xenopus homologs chordin and BMP-4. In this report we provide the first direct evidence that sog plays a local role in the lateral region of the blastoderm embryo to oppose Dpp activity in the neuroectoderm. In the dorsal region, Dpp signaling both suppresses neurogenesis and maintains expression of genes that promote dorsal cell fates (dorsalization). We show that Dpp also can perform both of these functions in the neuroectoderm. In wild-type embryos, the ability of Dpp to induce expression of dorsal markers including itself (autoactivation) in the neuroectoderm is blocked by sog. We propose that Sog protects the neuroectoderm from an invasive positive feedback loop created by Dpp diffusion and autoactivation. We show that the two functions of Dpp signaling, neural suppression and dorsalization, are triggered by distinct thresholds of Dpp activity. Epistasis experiments reveal that all observed sog activity can be accounted for by Sog functioning as a dedicated Dpp antagonist. Finally, we provide evidence that Sog functions as a diffusible morphogen in the blastoderm embryo. These data strongly support the view that the primary phylogenetically conserved function of the Drosophila sog and dpp genes and the homologous Xenopus chordin and BMP-4 genes is to subdivide the primitive embryonic ectoderm into neural versus non-neural domains.' publications.doi=None publications.firstAuthor='Biehs B' publications.id=1007780 publications.issue='22' publications.journal='Genes Dev.' publications.month='Nov' publications.pages='2922-34' publications.pubMedId='8918893' publications.title='The Drosophila short gastrulation gene prevents Dpp from autoactivating and suppressing neurogenesis in the neuroectoderm.' publications.volume='10' publications.year=1996\n",
      "Gene: publications.abstractText='The spatial and temporal pattern of mitoses during the fourteenth nuclear cycle in a Drosophila embryo reflects differences in cell identities. We have analysed the domains of mitotic division in zygotic mutants that exhibit defects in larval cuticular pattern along the dorsoventral axis. This is a powerful means of fate mapping mutant embryos, as the altered position of mitotic domains in the dorsoventral pattern mutants correlate with their late cuticular phenotypes. In the mutants twist and snail, which fail to differentiate the ventrally derived mesoderm, mitoses specific to the mesoderm are absent. The lateral mesectodermal domain shows a partial ventral shift in twist mutants but a proportion of ventral cells do not behave characteristically, suggesting that twist has a positive role in the establishment of the mesoderm. In contrast, snail is required to repress mesectodermal fates in cells of the presumptive mesoderm. In the absence of both genes, the mesodermal and the mesectodermal anlage are deleted. Mutations at five loci delete specific pattern elements in the dorsal half of the embryo and cause partial ventralization. Mutations in the genes zerknüllt and shrew affect cell division only in the dorsalmost cells corresponding to the amnioserosa, while the genes tolloid, screw and decapentaplegic (dpp) affect divisions in both the prospective amnioserosa and the dorsal epidermis. We demonstrate that in each of these mutants dorsally placed mitotic domains are absent and this effect is correlated with an expansion and dorsal shift in the position of more ventral domains. The loss of activity in each of the five genes results in qualitatively similar alterations in the mitotic pattern; mutations with stronger ventralizing phenotypes affect increasingly greater subsets of the dorsal cells. Double mutant analysis indicates that these genes act in a concerted manner to specify dorsal fates. The correlation between phenotypic strength and the progressive loss of dorsal pattern elements in the ventralized mutants, suggests that one of these gene products, perhaps dpp, may provide positional information in a graded manner.' publications.doi=None publications.firstAuthor='Arora K' publications.id=1007868 publications.issue='4' publications.journal='Development' publications.month='Apr' publications.pages='1003-24' publications.pubMedId='1618145' publications.title='Altered mitotic domains reveal fate map changes in Drosophila embryos mutant for zygotic dorsoventral patterning genes.' publications.volume='114' publications.year=1992\n",
      "Gene: publications.abstractText='The synthesis of gene expression data and cis-regulatory analysis permits the elucidation of genomic regulatory networks. These networks provide a direct visualization of the functional interconnections among the regulatory genes and signaling components leading to cell-specific patterns of gene activity. Complex developmental processes are thereby illuminated in ways not revealed by the conventional analysis of individual genes. In this review, we describe emerging networks in several different model systems, and compare them with the gene regulatory network that controls dorsoventral patterning of the Drosophila embryo.' publications.doi='10.1016/j.devcel.2005.09.005' publications.firstAuthor='Stathopoulos Angelike' publications.id=1004041 publications.issue='4' publications.journal='Dev. Cell' publications.month='Oct' publications.pages='449-62' publications.pubMedId='16198288' publications.title='Genomic regulatory networks and animal development.' publications.volume='9' publications.year=2005\n",
      "Gene: publications.abstractText=\"The torpedo (DER) gene of Drosophila, which encodes a receptor tyrosine kinase of the EGF receptor subfamily, is essential for oogenesis, embryogenesis and imaginal disc development. To gain insight into the nature of the signals transduced by the torpedo product, we have characterized the gene's loss-of-function phenotype in the embryo. Through the induction of germline clones, we provide a genetic demonstration that maternal torpedo product does not contribute to zygotic development. Thus, the embryonic lethal phenotypes examined accurately reflect the consequences of eliminating all gene activity from the zygote. Temperature-shift experiments with the conditional allele topIF26 show that torpedo is required at two distinct times during embryonic development: the gene is first needed for germband retraction and for the production of anterior, posterior and ventral cuticle, then later for the secretion of ventral denticles. Since denticle formation can be severely disrupted in topIF26 animals without affecting cuticle production, the early and late requirements for torpedo appear to be functionally unrelated. torpedo, therefore, is required at multiple times in the development of the ventral epidermis, and may transduce qualitatively different signals. Since the early requirement for torpedo correlates with the first visible defect in embryonic development, increased cell death in the amnioserosa, cephalic ectoderm and ventral epidermis, the abnormalities in cuticle production and germband shortening seen in the mutant may be secondary consequences of a primary defect in cell viability. Given that the onset of cell death in torpedo embryos is not preceded by any obvious defects in mitogenesis, the establishment of cell identities or the maintenance of gene expression, it is possible that torpedo transduces a signal necessary for cell survival per se during early embryogenesis. During late embryogenesis, torpedo may mediate the reception of a second signal which regulates ventral epidermal cell differentiation.\" publications.doi=None publications.firstAuthor='Clifford R' publications.id=1007851 publications.issue='3' publications.journal='Development' publications.month='Jul' publications.pages='853-72' publications.pubMedId='1425358' publications.title='The torpedo (DER) receptor tyrosine kinase is required at multiple times during Drosophila embryogenesis.' publications.volume='115' publications.year=1992\n",
      "Gene: publications.abstractText=\"The toxicity of an abnormally long polyglutamine [poly(Q)]tract within specific proteins is the molecular lesion shared by Huntington's disease (HD) and several other hereditary neurodegenerative disorders. By a genetic screen in Drosophila, devised to uncover genes that suppress poly(Q) toxicity, we discovered a Drosophila homolog of human myeloid leukemia factor 1 (MLF1). Expression of the Drosophila homolog (dMLF) ameliorates the toxicity of poly(Q) expressed in the eye and central nervous system. In the retina, whether endogenously or ectopically expressed, dMLF co-localized with aggregates, suggesting that dMLF alone, or through an intermediary molecular partner, may suppress toxicity by sequestering poly(Q) and/or its aggregates.\" publications.doi=None publications.firstAuthor='Kazemi-Esfarjani Parsa' publications.id=1007866 publications.issue='21' publications.journal='Hum. Mol. Genet.' publications.month='Oct' publications.pages='2657-72' publications.pubMedId='12354791' publications.title='Suppression of polyglutamine toxicity by a Drosophila homolog of myeloid leukemia factor 1.' publications.volume='11' publications.year=2002\n",
      "Gene: publications.abstractText=\"The trithorax group (trxG) genes absent, small or homeotic discs 1 (ash1) and 2 (ash2) were isolated in a screen for mutants with abnormal imaginal discs. Mutations in either gene cause homeotic transformations but Hox genes are not their only targets. Although analysis of double mutants revealed that ash2 and ash1 mutations enhance each other's phenotypes, suggesting they are functionally related, it was shown that these proteins are subunits of distinct complexes. The analysis of wing imaginal disc transcriptomes from ash2 and ash1 mutants showed that they are highly similar. Functional annotation of regulated genes using Gene Ontology allowed identification of severely affected groups of genes that could be correlated to the wing phenotypes observed. Comparison of the differentially expressed genes with those from other genome-wide analyses revealed similarities between ASH2 and Sin3A, suggesting a putative functional relationship. Coimmunoprecipitation studies and immunolocalization on polytene chromosomes demonstrated that ASH2 and Sin3A interact with HCF (host-cell factor). The results of nucleosome western blots and clonal analysis indicated that ASH2 is necessary for trimethylation of the Lys4 on histone 3 (H3K4). The similarity between the transcriptomes of ash2 and ash1 mutants supports a model in which the two genes act together to maintain stable states of transcription. Like in humans, both ASH2 and Sin3A bind HCF. Finally, the reduction of H3K4 trimethylation in ash2 mutants is the first evidence in Drosophila regarding the molecular function of this trxG gene.\" publications.doi='10.1186/gb-2007-8-4-r67' publications.firstAuthor='Beltran Sergi' publications.id=1000626 publications.issue='4' publications.journal='Genome Biol.' publications.month=None publications.pages='R67' publications.pubMedId='17466076' publications.title='Functional dissection of the ash2 and ash1 transcriptomes provides insights into the transcriptional basis of wing phenotypes and reveals conserved protein interactions.' publications.volume='8' publications.year=2007\n",
      "Gene: publications.abstractText='The twisted gastrulation (tsg) gene is one of seven known zygotic genes that specify the fate of dorsal cells in Drosophila embryos. Mutations in these genes cause at least some of the cells on the dorsal half of the embryo to adopt more ventral cell fates leading to the proposal that most of these genes participate in establishing, maintaining, or modulating a gradient of a single signaling molecule DECAPENTAPLEGIC (DPP). We have examined the effects of tsg mutations on the development of cuticule elements, expression of a region specific enhancer trap, and patterns of mitotic domains. Mutations of tsg only affect the fate of a narrow strip of dorsal midline cells and do not affect dorsal ectoderm cells. However, the pattern of tsg expression is not coincident with the territories affected by tsg mutations. Structural analysis of the tsg gene reveals features of a secreted protein suggesting an extracellular site of action. The TSG protein bears a weak resemblance to human connective tissue growth factor (CTGF), a TGF-beta-induced protein. We propose that dorsal midline cell fate is specified by the combination of both a TSG and a DPP signal to which the dorsal midline cells are uniquely competent to respond.' publications.doi=None publications.firstAuthor='Mason E D' publications.id=1002687 publications.issue='13' publications.journal='Genes Dev.' publications.month='Jul' publications.pages='1489-501' publications.pubMedId='7958834' publications.title='Dorsal midline fate in Drosophila embryos requires twisted gastrulation, a gene encoding a secreted protein related to human connective tissue growth factor.' publications.volume='8' publications.year=1994\n",
      "Gene: publications.abstractText='The yellow (y) gene maps near the telomere of the X chromosome in Drosophila melanogaster but not in D. subobscura. Thus the strong reduction in the recombination rate associated with telomeric regions is not expected in D. subobscura. To study the divergence of a gene whose recombination rate differs between two species, the y gene of D. subobscura was sequenced. Sequence comparison between D. melanogaster and D. subobscura revealed several elements conserved in noncoding regions that may correspond to putative cis-acting regulatory sequences. Divergence in the y gene coding region between D. subobscura and D. melanogaster was compared with that found in other genes sequenced in both species. Both, yellow and scute exhibit an unusually high number of synonymous substitutions per site (ps). Also for these genes, the extent of codon bias differs between both species, being much higher in D. subobscura than in D. melanogaster. This pattern of divergence is consistent with the hitchhiking and background selection models that predict an increase in the fixation rate of slightly deleterious mutations and a decrease in the rate of fixation of slightly advantageous mutations in regions with low recombination rates such as in the y-sc gene region of D. melanogaster.' publications.doi=None publications.firstAuthor='Munté A' publications.id=1007885 publications.issue='1' publications.journal='Genetics' publications.month='Sep' publications.pages='165-75' publications.pubMedId='9286677' publications.title='Divergence of the yellow gene between Drosophila melanogaster and D. subobscura: recombination rate, codon bias and synonymous substitutions.' publications.volume='147' publications.year=1997\n",
      "Gene: publications.abstractText='This report focuses on dorso-ventral patterning in the segmented region of the Drosophila melanogaster embryo. According to the concept of positional information, this pattern results from the different response of cells to the Dorsal-protein morphogen. This protein shows a distribution gradient along the dorso-ventral axis, with the highest concentration on the ventral side. Using the generalized logical formalism developed by R. Thomas and co-workers, the different cellular responses were analysed in terms of the intracellular loops between the regulatory genes. Two positive loops were found to be involved, each constituting a switch which can be acted upon by the Dorsal morphogen to determine the different cell types that make up the embryonic dorso-ventral pattern. The novelty in this use of generalized logical formalism is the employment of a multilevel variable to represent a morphogen gradient. The proposed model accounts for the essential qualitative effects of the Dorsal gradient in the dorso-ventral determination process. Three main conclusions may be drawn. Firstly, the gene twist needs to have two functional threshold concentrations, one for autoactivation and the other for activation of the gene snail. Secondly, the autoactivation threshold must be smaller than that which activates snail. Thirdly, the action of the gene snail on the maintenance function of the gene twist is crucial for cells to be able to choose between the mesoderm or neuroectoderm developmental pathways. Furthermore, it is predicted that if the gene snail shows autoregulation, this will not be crucial for the determination of the embryonic D-V pattern. Copyright 1997 Academic Press Limited Copyright 1997 Academic Press Limited' publications.doi=None publications.firstAuthor='Sanchez null' publications.id=1007777 publications.issue='4' publications.journal='J. Theor. Biol.' publications.month='Dec' publications.pages='377-89' publications.pubMedId='9446747' publications.title='Establishement of the dorso-ventral pattern during embryonic development of drosophila melanogasater: a logical analysis' publications.volume='189' publications.year=1997\n",
      "Gene: publications.abstractText='Three genetic hierarchies control cell-fate specification in largely distinct regions of the antero-posterior axis of the Drosophila embryo, whereas a single hierarchy specifies dorso-ventral cell fates. Molecular genetic analysis of these hierarchies is leading to increased understanding of the nature of the regulatory circuitry that controls regional cell-fate specification.' publications.doi=None publications.firstAuthor='Lipshitz H D' publications.id=1002700 publications.issue='6' publications.journal='Curr. Opin. Cell Biol.' publications.month='Dec' publications.pages='966-75' publications.pubMedId='1814368' publications.title='Axis specification in the Drosophila embryo.' publications.volume='3' publications.year=1991\n",
      "Gene: publications.abstractText='To achieve the \"constancy of the wild-type,\"the developing organism must be buffered against stochastic fluctuations and environmental perturbations. This phenotypic buffering has been theorized to arise from a variety of genetic mechanisms and is widely thought to be adaptive and essential for viability. In the Drosophila blastoderm embryo, staining with antibodies against the active, phosphorylated form of the bone morphogenetic protein (BMP) signal transducer Mad, pMad, or visualization of the spatial pattern of BMP-receptor interactions reveals a spatially bistable pattern of BMP signaling centered on the dorsal midline. This signaling event is essential for the specification of dorsal cell fates, including the extraembryonic amnioserosa. BMP signaling is initiated by facilitated extracellular diffusion that localizes BMP ligands dorsally. BMP signaling then activates an intracellular positive feedback circuit that promotes future BMP-receptor interactions. Here, we identify a genetic network comprising three genes that canalizes this BMP signaling event. The BMP target eiger (egr) acts in the positive feedback circuit to promote signaling, while the BMP binding protein encoded by crossveinless-2 (cv-2) antagonizes signaling. Expression of both genes requires the early activity of the homeobox gene zerknüllt (zen). Two Drosophila species lacking early zen expression have high variability in BMP signaling. These data both detail a new mechanism that generates developmental canalization and identify an example of a species with noncanalized axial patterning.' publications.doi='10.1016/j.cub.2013.09.055' publications.firstAuthor='Gavin-Smyth Jackie' publications.id=1007805 publications.issue='22' publications.journal='Curr. Biol.' publications.month='Nov' publications.pages='2296-302' publications.pubMedId='24184102' publications.title='A genetic network conferring canalization to a bistable patterning system in Drosophila.' publications.volume='23' publications.year=2013\n",
      "Gene: publications.abstractText='Transcriptional enhancers are crucial regulators of gene expression and animal development and the characterization of their genomic organization, spatiotemporal activities and sequence properties is a key goal in modern biology. Here we characterize the in vivo activity of 7,705 Drosophila melanogaster enhancer candidates covering 13.5% of the non-coding non-repetitive genome throughout embryogenesis. 3,557 (46%) candidates are active, suggesting a high density with 50,000 to 100,000 developmental enhancers genome-wide. The vast majority of enhancers display specific spatial patterns that are highly dynamic during development. Most appear to regulate their neighbouring genes, suggesting that the cis-regulatory genome is organized locally into domains, which are supported by chromosomal domains, insulator binding and genome evolution. However, 12 to 21 per cent of enhancers appear to skip non-expressed neighbours and regulate a more distal gene. Finally, we computationally identify cis-regulatory motifs that are predictive and required for enhancer activity, as we validate experimentally. This work provides global insights into the organization of an animal regulatory genome and the make-up of enhancer sequences and confirms and generalizes principles from previous studies. All enhancer patterns are annotated manually with a controlled vocabulary and all results are available through a web interface (http://enhancers.starklab.org), including the raw images of all microscopy slides for manual inspection at arbitrary zoom levels.' publications.doi='10.1038/nature13395' publications.firstAuthor='Kvon Evgeny Z' publications.id=1003027 publications.issue='7512' publications.journal='Nature' publications.month='Aug' publications.pages='91-5' publications.pubMedId='24896182' publications.title='Genome-scale functional characterization of Drosophila developmental enhancers in vivo.' publications.volume='512' publications.year=2014\n",
      "Gene: publications.abstractText=\"Transcriptional repression is essential for the conversion of crude maternal gradients into sharp territories of tissue differentiation in the Drosophila embryo. Evidence will be presented suggesting that some of the embryonic repressors function through a short-range 'quenching' mechanism, whereby a repressor works over short distances (ca. 50 b.p.) to block neighbouring activators within a target enhancer. This type of repression can explain how different enhancers work autonomously within complex modular promoters. However, at least one of the repressors operating in the early embryo works through a long-range, or silencing, mechanism. The binding of a silencer to a given enhancer leads to the inactivation of all enhancers within a complex promoter. The analysis of chromatin boundary elements suggest that silencers and enhancers might work through distinct mechanisms. We speculate that silencers constrain the evolution of complex promoters.\" publications.doi='10.1098/rstb.1995.0111' publications.firstAuthor='Gray S' publications.id=1007730 publications.issue='1329' publications.journal='Philos. Trans. R. Soc. Lond., B, Biol. Sci.' publications.month='Sep' publications.pages='257-62' publications.pubMedId='8577836' publications.title='Transcriptional repression in the Drosophila embryo.' publications.volume='349' publications.year=1995\n",
      "Gene: publications.abstractText='Transcriptional repressors can be characterized by their range of action on promoters and enhancers. Short-range repressors interact over distances of 50-150 bp to inhibit, or quench, either upstream activators or the basal transcription complex. In contrast, long-range repressors act over several kilobases to silence basal promoters. We describe recent progress in characterizing the functional properties of one such long-range element in the Drosophila embryo and discuss the contrasting types of gene regulation that are made possible by short- and long-range repressors.' publications.doi=None publications.firstAuthor='Cai H N' publications.id=1007829 publications.issue='18' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Sep' publications.pages='9309-14' publications.pubMedId='8790326' publications.title='Long-range repression in the Drosophila embryo.' publications.volume='93' publications.year=1996\n",
      "Gene: publications.abstractText='Transcriptional repressors function primarily by recruiting co-repressors, which are accessory proteins that antagonize transcription by modifying chromatin structure. Although a repressor could function by recruiting just a single co-repressor, many can recruit more than one, with Drosophila Brinker (Brk) recruiting the co-repressors CtBP and Groucho (Gro), in addition to possessing a third repression domain, 3R. Previous studies indicated that Gro is sufficient for Brk to repress targets in the wing, questioning why it should need to recruit CtBP, a short-range co-repressor, when Gro is known to be able to function over longer distances. To resolve this we have used genomic engineering to generate a series of brk mutants that are unable to recruit Gro, CtBP and/or have 3R deleted. These reveal that although the recruitment of Gro is necessary and can be sufficient for Brk to make an almost morphologically wild-type fly, it is insufficient during oogenesis, where Brk must utilize CtBP and 3R to pattern the egg shell appropriately. Gro insufficiency during oogenesis can be explained by its downregulation in Brk-expressing cells through phosphorylation downstream of EGFR signaling.' publications.doi='10.1242/dev.099366' publications.firstAuthor='Upadhyai Priyanka' publications.id=1002044 publications.issue='20' publications.journal='Development' publications.month='Oct' publications.pages='4256-65' publications.pubMedId='24086079' publications.title='Brinker possesses multiple mechanisms for repression because its primary co-repressor, Groucho, may be unavailable in some cell types.' publications.volume='140' publications.year=2013\n",
      "Gene: publications.abstractText='Transposable elements (TEs) are mobile, repetitive sequences that make up significant fractions of metazoan genomes. Despite their near ubiquity and importance in genome and chromosome biology, most efforts to annotate TEs in genome sequences rely on the results of a single computational program, RepeatMasker. In contrast, recent advances in gene annotation indicate that high-quality gene models can be produced from combining multiple independent sources of computational evidence. To elevate the quality of TE annotations to a level comparable to that of gene models, we have developed a combined evidence-model TE annotation pipeline, analogous to systems used for gene annotation, by integrating results from multiple homology-based and de novo TE identification methods. As proof of principle, we have annotated \"TE models\"in Drosophila melanogaster Release 4 genomic sequences using the combined computational evidence derived from RepeatMasker, BLASTER, TBLASTX, all-by-all BLASTN, RECON, TE-HMM and the previous Release 3.1 annotation. Our system is designed for use with the Apollo genome annotation tool, allowing automatic results to be curated manually to produce reliable annotations. The euchromatic TE fraction of D. melanogaster is now estimated at 5.3% (cf. 3.86% in Release 3.1), and we found a substantially higher number of TEs (n = 6,013) than previously identified (n = 1,572). Most of the new TEs derive from small fragments of a few hundred nucleotides long and highly abundant families not previously annotated (e.g., INE-1). We also estimated that 518 TE copies (8.6%) are inserted into at least one other TE, forming a nest of elements. The pipeline allows rapid and thorough annotation of even the most complex TE models, including highly deleted and/or nested elements such as those often found in heterochromatic sequences. Our pipeline can be easily adapted to other genome sequences, such as those of the D. melanogaster heterochromatin or other species in the genus Drosophila.' publications.doi='10.1371/journal.pcbi.0010022' publications.firstAuthor='Quesneville Hadi' publications.id=1015598 publications.issue='2' publications.journal='PLoS Comput. Biol.' publications.month='Jul' publications.pages='166-75' publications.pubMedId='16110336' publications.title='Combined evidence annotation of transposable elements in genome sequences.' publications.volume='1' publications.year=2005\n",
      "Gene: publications.abstractText='Transposable elements are found in the genomes of nearly all eukaryotes. The recent completion of the Release 3 euchromatic genomic sequence of Drosophila melanogaster by the Berkeley Drosophila Genome Project has provided precise sequence for the repetitive elements in the Drosophila euchromatin. We have used this genomic sequence to describe the euchromatic transposable elements in the sequenced strain of this species. We identified 85 known and eight novel families of transposable element varying in copy number from one to 146. A total of 1,572 full and partial transposable elements were identified, comprising 3.86% of the sequence. More than two-thirds of the transposable elements are partial. The density of transposable elements increases an average of 4.7 times in the centromere-proximal regions of each of the major chromosome arms. We found that transposable elements are preferentially found outside genes; only 436 of 1,572 transposable elements are contained within the 61.4 Mb of sequence that is annotated as being transcribed.A large proportion of transposable elements is found nested within other elements of the same or different classes. Lastly, an analysis of structural variation from different families reveals distinct patterns of deletion for elements belonging to different classes. This analysis represents an initial characterization of the transposable elements in the Release 3 euchromatic genomic sequence of D. melanogaster for which comparison to the transposable elements of other organisms can begin to be made. These data have been made available on the Berkeley Drosophila Genome Project website for future analyses.' publications.doi=None publications.firstAuthor='Kaminker Joshua S' publications.id=1005240 publications.issue='12' publications.journal='Genome Biol.' publications.month=None publications.pages='RESEARCH0084' publications.pubMedId='12537573' publications.title='The transposable elements of the Drosophila melanogaster euchromatin: a genomics perspective.' publications.volume='3' publications.year=2002\n",
      "Gene: publications.abstractText='Two Drosophila Hox genes involved in segmentation, fushi tarazu and bicoid, appear to have acquired these roles by functional divergence from classical homeotic genes. Recent results indicate how genes with critical functions in development can evolve completely different functions among species.' publications.doi=None publications.firstAuthor='Gibson G' publications.id=1007735 publications.issue='12' publications.journal='Curr. Biol.' publications.month='Jun' publications.pages='R452-5' publications.pubMedId='10873798' publications.title='Evolution: hox genesand the cellared wine principle.' publications.volume='10' publications.year=2000\n",
      "Gene: publications.abstractText='Using X-ray mutagenesis we have induced and recovered phenotypic revertants of four dominant mutations thought to be associated with the Antennapedia complex of Drosophila melanogaster. These include seven revertants of Antennapedia-73b (Antp(73b)), six of Extra Sex Combs of Wakimoto (Scx(w)), three of Deformed (Dfd) and one of Humeral (Hu). Fifteen of the 17 revertants are associated with chromosomal aberrations and localize Antp(73b), Scx( w) and Hu to polytene chromosome bands 84B1,2. The Dfd lesion is apparently located in or adjacent to bands 84A4,5. Since all of the dominants are reverted by events that delete their respective chromosomal loci, we conclude that all four are the result of a gain-of-function lesions. Complementation analysis of the various revertant chromosomes has shown that Scx(w) and Hu are dominant allelic variants of the Antp locus. The Dfd lesion represents a dominant mutation at a locus just proximal to Antp and previously only occupied by recessive lethal mutations. Characterization of the revertants of Scx(w) and a comparison with the properties of the original mutation has revealed that the original lesion has effects on both the Antp and Sex Combs Reduced (Scr) loci and that these defects are in some cases separable by the reverting event.' publications.doi=None publications.firstAuthor='Hazelrigg T' publications.id=1000753 publications.issue='3' publications.journal='Genetics' publications.month='Nov' publications.pages='581-600' publications.pubMedId='17246168' publications.title='Revertants of Dominant Mutations Associated with the Antennapedia Gene Complex of DROSOPHILA MELANOGASTER: Cytology and Genetics.' publications.volume='105' publications.year=1983\n",
      "Gene: publications.abstractText='We address the problem of finding statistically significant associations between cis-regulatory motifs and functional gene sets, in order to understand the biological roles of transcription factors. We develop a computational framework for this task, whose features include a new statistical score for motif scanning, the use of different scores for predicting targets of different motifs, and new ways to deal with redundancies among significant motif-function associations. This framework is applied to the recently sequenced genome of the jewel wasp, Nasonia vitripennis, making use of the existing knowledge of motifs and gene annotations in another insect genome, that of the fruitfly. The framework uses cross-species comparison to improve the specificity of its predictions, and does so without relying upon non-coding sequence alignment. It is thereforewell suited for comparative genomics across large evolutionary divergences, where existing alignment-based methods are not applicable. We also apply the framework to find motifs associated with socially regulated gene sets in the honeybee, Apis mellifera, using comparisons with Nasonia, a solitary species, to identify honeybee-specific associations.' publications.doi='10.1371/journal.pcbi.1000652' publications.firstAuthor='Kim Jaebum' publications.id=1007904 publications.issue='1' publications.journal='PLoS Comput. Biol.' publications.month='Jan' publications.pages='e1000652' publications.pubMedId='20126523' publications.title='Functional characterization of transcription factor motifs using cross-species comparison across large evolutionary distances.' publications.volume='6' publications.year=2010\n",
      "Gene: publications.abstractText='We employed robotic methods and the whole-genome sequence of Drosophila melanogaster to facilitate a large-scale expression screen for spatially restricted transcripts in Drosophila embryos. In this screen, we identified a pair of genes, scylla (scyl) and charybde (chrb), that code for dorsal transcripts in early Drosophila embryos and are homologous to the human apoptotic gene RTP801. In Drosophila, both gene products are transcriptionally regulated targets of Dpp/Zen-mediated signal transduction and appear more generally to be downstream targets of homeobox regulation. Gene disruption studies revealed the functional redundancy of scyl and chrb, as well as their requirement for embryonic head involution. From the perspective of functional genomics, our studies demonstrate that global surveys of gene expression can complement traditional genetic screening methods for the identification of genes essential for development: beginning from their spatio-temporal expression profiles and extending to their downstream placement relative to dpp and zen, our studies reveal roles for the scyl and chrb gene products as links between patterning and cell death.' publications.doi='10.1016/j.ydbio.2005.12.014' publications.firstAuthor='Scuderi Anne' publications.id=1007888 publications.issue='1' publications.journal='Dev. Biol.' publications.month='Mar' publications.pages='110-22' publications.pubMedId='16423342' publications.title='scylla and charybde, homologues of the human apoptotic gene RTP801, are required for head involution in Drosophila.' publications.volume='291' publications.year=2006\n",
      "Gene: publications.abstractText='We have analyzed the contributions made by maternal and zygotic genes to the establishment of the expression patterns of four zygotic patterning genes: decapentaplegic (dpp), zerknüllt (zen), twist (twi), and snail (sna). All of these genes are initially expressed either dorsally or ventrally in the segmented region of the embryo, and at the poles. In the segmented region of the embryo, correct expression of these genes depends on cues from the maternal morphogen dorsal (dl). The dl gradient appears to be interpreted on three levels: dorsal cells express dpp and zen, but not twi and sna; lateral cells lack expression of all four genes; ventral cells express twi and sna, but not dpp and zen. dl appears to activate the expression of twi and sna and repress the expression of dpp and zen. Polar expression of dpp and zen requires the terminal system to override the repression by dl, while that of twi and sna requires the terminal system to augment activation by dl. The zygotic expression patterns established by the maternal genes appear to specify autonomous domains that carry out independent developmental programs, insofar as mutations in the genes that are expressed ventrally do not affect the initiation or ontogeny of the expression patterns of the genes that are expressed dorsally, and vice versa. However, interactions between the zygotic genes specific to a particular morphological domain appear to be important for further elaboration of the three levels specified by dl. Two of the genes, dpp and twi, are unaffected by mutations in any of the tested zygotic dorsal-ventral genes, suggesting that dpp and twi are the primary patterning genes for dorsal ectoderm and mesoderm, respectively.' publications.doi=None publications.firstAuthor='Ray R P' publications.id=1007768 publications.issue='1' publications.journal='Development' publications.month='Sep' publications.pages='35-54' publications.pubMedId='1765005' publications.title='The control of cell fate along the dorsal-ventral axis of the Drosophila embryo.' publications.volume='113' publications.year=1991\n",
      "Gene: publications.abstractText=\"We have analyzed the function of the Decapentaplegic (Dpp) and Hedgehog (Hh) signaling pathways in partitioning the dorsal head neurectoderm of the Drosophila embryo. This region, referred to as the anterior brain/eye anlage, gives rise to both the visual system and the protocerebrum. The anlage splits up into three main domains: the head midline ectoderm, protocerebral neurectoderm and visual primordium. Similar to their vertebrate counterparts, Hh and Dpp play an important role in the partitioning of the anterior brain/eye anlage. Dpp is secreted in the dorsal midline of the head. Lowering Dpp levels (in dpp heterozygotes or hypomorphic alleles) results in a 'cyclops' phenotype, where mid-dorsal head epidermis is transformed into dorsolateral structures, i.e. eye/optic lobe tissue, which causes a continuous visual primordium across the dorsal midline. Absence of Dpp results in the transformation of both dorsomedial and dorsolateral structures into brain neuroblasts. Regulatory genes that are required for eye/optic lobe fate, including sine oculis (so) and eyes absent (eya), are turned on in their respective domains by Dpp. The gene zerknuellt (zen), which is expressed in response to peak levels of Dpp in the dorsal midline, secondarily represses so and eya in the dorsomedial domain. Hh and its receptor/inhibitor, Patched (Ptc), are expressed in a transverse stripe along the posterior boundary of the eye field. As reported previously, Hh triggers the expression of determinants for larval eye (atonal) and adult eye (eyeless) in those cells of the eye field that are close to the Hh source. Eya and So, which are induced by Dpp, are epistatic to the Hh signal. Loss of Ptc, as well as overexpression of Hh, results in the ectopic induction of larval eye tissue in the dorsal midline (cyclopia). We discuss the similarities between vertebrate systems and Drosophila with regard to the fate map of the anterior brain/eye anlage, and its partitioning by Dpp and Hh signaling.\" publications.doi=None publications.firstAuthor='Chang T' publications.id=1007705 publications.issue='23' publications.journal='Development' publications.month='Dec' publications.pages='4691-704' publications.pubMedId='11731450' publications.title='Dpp and Hh signaling in the Drosophila embryonic eye field.' publications.volume='128' publications.year=2001\n",
      "Gene: publications.abstractText='We have characterized at the molecular level the zerknüllt (zen) region of the Drosophila subobscura Antennapedia complex. The sequence comparison between D. subobscura and D. melanogaster shows an irregular distribution of the conserved and diverged regions, with the homeobox and a putative activating domain completely conserved. Comparisons of the promoter sequence and pattern of expression of the gene during development suggest that the regulation of zen has been conserved during evolution. The conservation of zen expression in a subpopulation of the polar cells indicates the existence of an important role in such cells. We describe a transitory segmented pattern of expression of zen in both species, suggesting the existence of interactions with a pair rule gene. Some indirect clues indicate that the z2 gene might be absent from the D. subobscura genome. A chromosome walk initiated to reach the proboscipedia gene of D. subobscura reveals that the distance between pb and zen is at least four times the one described for D. melanogaster and for D. pseudoobscura. Finally, we present cytological evidence showing that the ANT-C is inverted in D. subobscura as compared to D. melanogaster.' publications.doi=None publications.firstAuthor='Terol J' publications.id=1007797 publications.issue='9' publications.journal='Chromosoma' publications.month='May' publications.pages='613-24' publications.pubMedId='7587584' publications.title='Molecular characterization of the zerknüllt region of the Antennapedia complex of D. subobscura.' publications.volume='103' publications.year=1995\n",
      "Gene: publications.abstractText='We have cloned, from a beetle and a locust, genes that are homologous to the class 3 Hox genes of vertebrates. Outside the homeobox they sharesequence motifs with the Drosophila zerknüllt (zen) and z2 genes, and like zen, are expressed only in extraembryonic membranes. We conclude that the zen genes of Drosophila derive from a Hox class 3 sequence that formed part of the common ancestral Hox cluster, but that in insects this (Hox) gene has lost its role in patterning the anterio-posterior axis of the embryo, and acquired a new function. In the lineage leading to Drosophila, the zen genes have diverged particularly rapidly.' publications.doi=None publications.firstAuthor='Falciani F' publications.id=1007825 publications.issue='16' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Aug' publications.pages='8479-84' publications.pubMedId='8710895' publications.title='Class 3 Hox genes in insects and the origin of zen.' publications.volume='93' publications.year=1996\n",
      "Gene: publications.abstractText=\"We have identified in the 5' untranslated region of the Drosophila copia retrotransposon, 3' to the left LTR, a sequence for transcriptional regulation by homeoproteins. Co-transfection assays using expression vectors for homeoproteins and reporter vectors containing the lacZ gene under the control of either the entire copia LTR with 5' untranslated sequence, or a minimal heterologous promoter flanked with a 130 bp fragment containing the copia untranslated region, disclosed both positive and negative modulations of promoter activity in Drosophila cells in culture: a 5-10 fold decrease with engrailed, even-skipped and zerknüllt in DH33 cells, and a 10-30 fold increase with fushi tarazu and zerknüllt in Schneider II cells. In all cases, the regulatory effects were abolished with reporter plasmids deleted for a 58 bp fragment encompassing the putative homeoprotein binding sites. Mobility shift assays with a purified homeodomain-containing peptide demonstrated direct interaction with the 58 bp fragment, with an affinity in the 1-10 nM range as reported with the same peptide for other well characterized homeodomain binding regulatory sites. Foot-printing experiments with the extended LTR demonstrated protection of 'consensus' sequences, located within the 58 bp fragment. These homeodomain binding sites could be involved in the developmental regulation of the copia retrotransposon.\" publications.doi=None publications.firstAuthor='Cavarec L' publications.id=1007699 publications.issue='22' publications.journal='Nucleic Acids Res.' publications.month='Nov' publications.pages='5041-9' publications.pubMedId='8255758' publications.title='The Drosophila copia retrotransposon contains binding sites for transcriptional regulation by homeoproteins.' publications.volume='21' publications.year=1993\n",
      "Gene: publications.abstractText='We have investigated Drosophila salivary gland determination by examining the effects of mutations in pattern forming genes on the salivary gland primordium. We find that the anterior-posterior extent of the primordium, a placode of columnar epithelial cells derived from parasegment 2, is established by the positive action of the homeotic gene Sex combs reduced (Scr). Embryos mutant for Scr lack a detectable placode, while ectopic Scr expression leads to the formation of ectopic salivary glands. In contrast, the dorsal-ventral extent of the placode is regulated negatively. Functions dependent on the decapentaplegic product place a dorsal limit on the placode, while dorsal-dependent genes act to limit the placode ventrally. We propose a model in which these pattern forming genes act early to determine the salivary gland anlage by regulating the expression of salivary gland determining genes, which in turn control genes that are involved in salivary gland morphogenesis.' publications.doi=None publications.firstAuthor='Panzer S' publications.id=1007734 publications.issue='1' publications.journal='Development' publications.month='Jan' publications.pages='49-57' publications.pubMedId='1349523' publications.title='Organogenesis in Drosophila melanogaster: embryonic salivary gland determination is controlled by homeotic and dorsoventral patterning genes.' publications.volume='114' publications.year=1992\n",
      "Gene: publications.abstractText='We have observed that zygotic transcription does not initiate at a single point in Drosophila embryos. Rather, a gene initiates transcription in a few nuclei of a fraction of embryos. During succeeding cycles, the frequency of transcribing embryos, and of nuclei transcribing in those embryos, gradually increases. For the fushi tarazu (ftz) gene, the timing of this process is regulated by the concentration of the maternally loaded, repressing transcription factor tramtrack (ttk). Altering the dose of Ttk protein in embryos shifts the activation of ftz transcription either forward or backward during development but does not effect Krüppel (Kr) activation. We have observed that the transcription of several genes, including ftz, is triggered in embryos at a critical nuclear density; therefore, we suggest that titration of transcription factors like ttk by the nucleocytoplasmic ratio triggers zygotic transcription in Drosophila.' publications.doi=None publications.firstAuthor='Pritchard D K' publications.id=1007817 publications.issue='9' publications.journal='Genes Dev.' publications.month='May' publications.pages='1131-42' publications.pubMedId='8654928' publications.title='Activation of transcription in Drosophila embryos is a gradual process mediated by the nucleocytoplasmic ratio.' publications.volume='10' publications.year=1996\n",
      "Gene: publications.abstractText='We have studied the ability of the Drosophila gap proteins Krüppel and hunchback to function as transcriptional regulators in cultured cells. Both proteins bind to specific sites in a 100-bp DNA fragment located upstream of the segment polarity gene engrailed, which also contains functional binding sites for a number of homeo box proteins. The hunchback protein is a strikingly concentration-dependent activator of transcription, capable of functioning both by itself and also synergistically with the pair-rule proteins fushi tarazu and paired. In contrast, Krüppel is a transcriptional repressor that can block transcription induced either by hunchback or by several different homeo box proteins. While repression of the homeo box protein activators requires a Krüppel-binding site on the DNA, repression of hunchback can occur efficiently in the absence of a Krüppel-binding site. We discuss the possible molecular mechanisms underlying these activities, as well as the potential significance of these results with respect to segmentation in Drosophila.' publications.doi=None publications.firstAuthor='Zuo P' publications.id=1007893 publications.issue='2' publications.journal='Genes Dev.' publications.month='Feb' publications.pages='254-64' publications.pubMedId='1671661' publications.title='Activation and repression of transcription by the gap proteins hunchback and Krüppel in cultured Drosophila cells.' publications.volume='5' publications.year=1991\n",
      "Gene: publications.abstractText='We have studied the transcriptional activity of the Drosophila homeodomain protein Engrailed (En) by using a transient expression assay employing Schneider L2 cells. En was found to very strongly repress promoters activated by a variety of different activator proteins. However, unlike another Drosophila homeodomain-containing repressor, Even-skipped (Eve), En was unable to repress the activity of several basal promoters in the absence of activator expression. These findings indicate that En is a specific repressor of activated transcription, and suggest that En may repress transcription by a different mechanism than Eve, perhaps by interfering with interactions between transcriptional activators and the general transcription machinery. By analyzing the properties of a variety of En mutants, we identified a minimal repression domain composed of 55 residues, which can function when fused to a heterologous DNA binding domain. Like repression domains identified in the Drosophila repressors Eve and Krüppel, the En repression domain is rich in alanine residues (26%), but unlike these other domains, is moderately charged (six arginine and three glutamic acid residues). Separate regions of En that may in some circumstances function in transcriptional activation were also identified.' publications.doi=None publications.firstAuthor='Han K' publications.id=1007700 publications.issue='7' publications.journal='EMBO J.' publications.month='Jul' publications.pages='2723-33' publications.pubMedId='8334991' publications.title='Functional domains of the Drosophila Engrailed protein.' publications.volume='12' publications.year=1993\n",
      "Gene: publications.abstractText='We have undertaken a developmental genetic analysis of labial (lab), the most proximal gene in the Antennapedia complex (ANT-C) of Drosophila melanogaster. The terminal phenotype of mutant embryos was examined in cuticle preparations, in thin sections, and by scanning electron microscopy. These preparations revealed a failure of head involution and the loss or disruption of several head structures, including the salivary glands and the H-piece and ventral arm of the cephalopharyngeal apparatus. Although these structures are presumed to derive from the gnathocephalic segments, we argue that the observed defects are likely to be a secondary consequence of a failure of head involution. A function for lab in the development of the adult head was inferred from the phenotype of animals bearing hypomorphic alleles and from clones of lab- tissue generated by mitotic recombination. Two aspects of the mutant phenotype were manifested. Ventrally, a deletion and/or disruption of tissue occurred in the maxillary palp and vibrissae regions. Dorsally, the posterior head appeared to be transformed to a thoracic-like identity. Mutations in lab, like those in the Deformed and proboscipedia loci of the ANT-C, reveal a homoeotic phenotype only in the adult stage of the life cycle.' publications.doi=None publications.firstAuthor='Merrill V K' publications.id=1005537 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Oct' publications.pages='376-91' publications.pubMedId='2570723' publications.title='A genetic and developmental analysis of mutations in labial, a gene necessary for proper head formation in Drosophila melanogaster.' publications.volume='135' publications.year=1989\n",
      "Gene: publications.abstractText='We have used a transient expression assay employing Drosophila tissue culture cells to study the potential of several Drosophila homeobox proteins to function as transcriptional regulators. A 96 bp fragment from the promoter region of the segment polarity gene engrailed, previously shown to contain five copies of a 10 bp consensus binding site for these proteins, enhanced transcription in the presence, but not the absence, of several different homeobox protein expression vectors. It is interesting that cotransfection with combinations of expression vectors encoding the homeobox proteins fushi tarazu, paired, and/or zen resulted in substantial synergistic increases in expression. In contrast, the products of the even-skipped and engrailed genes were found to repress, or quench, the activation induced by the other proteins. We discuss the implications of these results with respect to the role of homeobox genes in the control of embryonic development, and propose a \"multi-switch\"model whereby the activity of a target gene depends on the interactions of different homeobox proteins with multiple copies of a common binding site.' publications.doi=None publications.firstAuthor='Han K' publications.id=1007727 publications.issue='4' publications.journal='Cell' publications.month='Feb' publications.pages='573-83' publications.pubMedId='2563673' publications.title='Synergistic activation and repression of transcription by Drosophila homeobox proteins.' publications.volume='56' publications.year=1989\n",
      "Gene: publications.abstractText='We have used a transient expression assay employing Drosophila tissue culture cells to study the transcriptional repression activity of the homeo domain protein Even-skipped (Eve). Eve was found to repress all promoters that contained Eve-binding sites, including both TATA-containing and TATA-lacking minimal promoters, as well as promoters activated by several different classes of activator proteins. These findings suggest that the general transcription machinery can be a target of Eve. By analyzing properties of a variety of Eve mutants and chimeric fusion proteins, we have identified several features important for efficient repression. In addition to the DNA-binding domain, a potent repressor requires a repression domain, which can be as small as 27 residues. The minimal 57-residue Eve repression domain, as well as several others studied here, were all found to be proline rich and to contain a high percentage of hydrophobic residues. An intriguing feature of the strong repressors was that their DNA-binding activities, measured by gel retention assays with nuclear extracts, were significantly less than those of derivatives inactive in repression.' publications.doi=None publications.firstAuthor='Han K' publications.id=1007681 publications.issue='3' publications.journal='Genes Dev.' publications.month='Mar' publications.pages='491-503' publications.pubMedId='8095483' publications.title='Transcriptional repression by the Drosophila even-skipped protein: definition of a minimal repression domain.' publications.volume='7' publications.year=1993\n",
      "Gene: publications.abstractText='We identified Drosophila Smurf (DSmurf) as a negative regulator ofsignaling by the BMP2/4 ortholog DPP during embryonic dorsal-ventral patterning. DSmurf encodes a HECT domain ubiquitin-protein ligase, homologous to vertebrate Smurf1 and Smurf2, that binds the Smad1/5 ortholog MAD and likely promotes its proteolysis. The essential function of DSmurf is restricted to its action on the DPP pathway. DSmurf has two distinct, possibly mechanistically separate, functions in controlling DPP signaling. Prior to gastrulation, DSmurf mutations cause a spatial increase in the DPP gradient, as evidenced by ventrolateral expansion in expression domains of target genes representing all known signaling thresholds. After gastrulation, DSmurf mutations cause a temporal delay in downregulation of earlier DPP signals, resulting in a lethal defect in hindgut organogenesis.' publications.doi=None publications.firstAuthor='Podos S D' publications.id=1007872 publications.issue='4' publications.journal='Dev. Cell' publications.month='Oct' publications.pages='567-78' publications.pubMedId='11703946' publications.title='The DSmurf ubiquitin-protein ligase restricts BMP signaling spatially and temporally during Drosophila embryogenesis.' publications.volume='1' publications.year=2001\n",
      "Gene: publications.abstractText='We recently uncovered a novel genetic mechanism that generates the phenotypic uniformity, or canalization, of BMP signaling and cell fate specification during patterning of the dorsal-ventral (D/V) axis in D. melanogaster embryos. We went on to show that other wild-type Drosophila species lack this canalizing genetic circuitry and, consequently, have non-robust D/V patterning. In this review, we propose molecular mechanisms that may give rise to stereotyped BMP signaling, and we identify an additional species that could have decanalized D/V patterning. Extension of these analyses could in turn help explain why canalization is not a universal necessity for species survival.' publications.doi='10.4161/19336934.2014.983385' publications.firstAuthor='Gavin-Smyth Jackie' publications.id=82004051 publications.issue='3' publications.journal='Fly (Austin)' publications.month=None publications.pages='170-5' publications.pubMedId='25482877' publications.title='zen and the art of phenotypic maintenance: canalization of embryonic dorsal-ventral patterning in Drosophila.' publications.volume='8' publications.year=2014\n",
      "Gene: publications.abstractText='We report here the genetical and molecular characterization of a new Drosophila zygotic lethal locus, vrille (vri). Vri alleles act not only as dominant maternal enhancers of embryonic dorsoventral patterning defects caused by easter and decapentaplegic (dpp) mutations, but also as dominant zygotic enhancers of dpp alleles for phenotypes in wing. The vri gene encodes a new member of the bZIP family of transcription factors closely related to gene 9 of Xenopus laevis, induced by thyroid hormone during the tadpole tail resorption program, and NF-IL3A, a human T cell transcription factor that transactivates the interleukin-3 promoter. NF-IL3A shares 93% similarity and 60% identity with Vri for a stretch of 68 amino acids that includes the bZIP domain. Although all the alleles tested behave like antimorphs, the dominant enhancement is also seen with a nonsense mutation allele that prevents translation of the bZIP domain. Because of the strong domainant enhancement of dpp phenotypes by vri alleles in both embryo and wing, and also the similarity between the wing vein phenotypes caused by the vri and shortvein dpp alleles, we postulate that vri interacts either directly or indirectly with certain components of the dpp (a TGF beta homologue) signal transduction pathway.' publications.doi=None publications.firstAuthor='George H' publications.id=1007752 publications.issue='4' publications.journal='Genetics' publications.month='Aug' publications.pages='1345-63' publications.pubMedId='9258679' publications.title='The vrille gene of Drosophila is a maternal enhancer of decapentaplegic and encodes a new member of the bZIP family of transcription factors.' publications.volume='146' publications.year=1997\n",
      "Gene: publications.abstractText='We report the isolation and characterization of a putative angiotensin converting enzyme (ACE) in Drosophila, called Race. General interest in mammalian ACE stems from its association with high blood pressure; ACE has also been implicated in a variety of other physiological processes including the processing of neuropeptides and gut peristalsis. Mammalian ACE is a membrane associated zinc binding protease that converts angiotensin I (A I) into angiotensin II (A II). A II functions as a potent vasoconstrictor by triggering a G-coupled receptor system in the smooth muscles that line blood vessels. Drosophila Race is composed of 615 amino acid residues, and shares extensive sequence identity with mammalian ACE over its entire length (over 42% overall identity and greater than 60% similarity). Evidence is presented that Race might correspond to a target of the homeobox regulatory gene, zerknullt (zen). Soon after zen expression is restricted to the dorsal-most regions of the embryonic ectoderm, Race is activated in a coincident pattern and becomes associated with the amnioserosa during germ band elongation, shortening and heart morphogenesis. After germ band elongation, Race is also expressed in both the anterior and posterior midgut, where it persists throughout embryogenesis. Race expression is lost from the dorsal ectoderm in either zen- or dpp- mutants, although gut expression is unaffected. P-transformation assays and genetic complementation tests suggest that Race corresponds to a previously characterized lethal complementation group, 1(2)34Eb. Mutants die during larval/pupal development, and transheterozygotes for two different lethal alleles exhibit male sterility. We propose that Race might play a role in the contractions of the heart, gut, or testes and also suggest that Hox genes might be important for coordinating both developmental and physiological processes.' publications.doi=None publications.firstAuthor='Tatei K' publications.id=1007739 publications.issue='2-3' publications.journal='Mech. Dev.' publications.month='Jun' publications.pages='157-68' publications.pubMedId='7547464' publications.title='Race: a Drosophila homologue of the angiotensin converting enzyme.' publications.volume='51' publications.year=1995\n",
      "Gene: publications.abstractText='We show that the Drosophila protein DSP1, an HMG-1/2-like protein, binds DNA highly cooperatively with three members of the Rel family of transcriptional regulators (NF-kappaB, the p50 subunit of NF-kappaB, and the Rel domain of Dorsal). This cooperativity is apparent with DNA molecules bearing consensus Rel-protein-binding sites and is unaffected by the presence of a negative regulatory element, a sequence previously proposed to be important for mediating repression by these Rel proteins. The cooperativity observed in these DNA-binding assays is paralleled by interactions between protein pairs in the absence of DNA. We also show that in HeLa cells, as assayed by transient transfection, expression of DSP1 increases activation by Dorsal from the twist promoter and inhibits that activation from the zen promoter, consistent with the previously proposed idea that DSP1 can affect the action of Dorsal in a promoter-specific fashion.' publications.doi=None publications.firstAuthor='Brickman J M' publications.id=1007820 publications.issue='19' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Sep' publications.pages='10679-83' publications.pubMedId='10485885' publications.title='Interactions between an HMG-1 protein and members of the Rel family.' publications.volume='96' publications.year=1999\n",
      "Gene: publications.abstractText='While the role of the notochord and floor plate in patterning the dorsal-ventral (D/V) axis of the neural tube is clearly established, relatively little is known about the earliest stages of D/V regionalization. In an effort to examine more closely the initial, preneural plate stages of regionalization along the prospective D/V neural axis, we have performed a series of explant experiments employing xHB9, a novel marker of the motor neuron region in Xenopus. Using tissue recombinants and Keller explants we show that direct mesodermal contact is both necessary and sufficient for the initial induction of xHB9 in the motor neuron region. We also show that presumptive neural plate explants removed as early as midgastrulation and cultured in isolation are already specified to express xHB9 but do so in an inappropriate spatial pattern while identical explants are specified to express the floor plate marker vhh-1 with correct spatial patterning. Our data suggest that, in addition to floor plate signaling, continued interactions with the underlying mesoderm through neural tube stages are essential for proper spatial patterning of the motor neuron region.' publications.doi='10.1006/dbio.1997.8625' publications.firstAuthor='Saha M S' publications.id=1007828 publications.issue='2' publications.journal='Dev. Biol.' publications.month='Jul' publications.pages='209-23' publications.pubMedId='9242418' publications.title='Dorsal-ventral patterning during neural induction in Xenopus: assessment of spinal cord regionalization with xHB9, a marker for the motor neuron region.' publications.volume='187' publications.year=1997\n",
      "Gene: publications.abstractText='Wingless signaling plays a central role during epidermal patterning in Drosophila. We have analyzed zygotic requirements for Wingless signaling in the embryonic ectoderm by generating synthetic deficiencies that uncover more than 99% of the genome. We found no genes required for initial wingless expression, other than previously identified segmentation genes. In contrast, maintenance of wingless expression shows a high degree of zygotic transcriptional requirements. Besides known genes, we have identified at least two additional genomic regions containing new genes involved in Wingless maintenance. We also assayed for the zygotic requirements for Wingless response and found that no single genomic region was required for the cytoplasmic accumulation of Armadillo in the receiving cells. Surprisingly, embryos homozygously deleted for the candidate Wingless receptor, Dfrizzled2, showed a normal Wingless response. However, the Armadillo response to Wingless was strongly reduced in double mutants of both known members of the frizzled family in Drosophila, frizzled and Dfrizzled2. Based on their expression pattern during embryogenesis, different Frizzled receptors may play unique but overlapping roles in development. In particular, we suggest that Frizzled and Dfrizzled2 are both required for Wingless autoregulation, but might be dispensable for late Engrailed maintenance. While Wingless signaling in embryos mutant for frizzled and Dfrizzled2 is affected, Wingless protein is still internalized into cells adjacent to wingless-expressing cells. Incorporation of Wingless protein may therefore involve cell surface molecules in addition to the genetically defined signaling receptors of the frizzled family.' publications.doi=None publications.firstAuthor='Müller H A' publications.id=1007751 publications.issue='3' publications.journal='Development' publications.month='Feb' publications.pages='577-86' publications.pubMedId='9876186' publications.title='Wingless signaling in the Drosophila embryo: zygotic requirements and the role of the frizzled genes.' publications.volume='126' publications.year=1999\n",
      "Gene: publications.abstractText='With their power to shape animal morphology, few genes have captured the imagination of biologists as the evolutionarily conserved members of the Hox clusters have done. Recent research has provided new insight into how Hox proteins cause morphological diversity at the organismal and evolutionary levels. Furthermore, an expanding collection of sequences that are directly regulated by Hox proteins provides information on the specificity of target-gene activation, which might allow the successful prediction of novel Hox-response genes. Finally, the recent discovery of microRNA genes within the Hox gene clusters indicates yet another level of control by Hox genes in development and evolution.' publications.doi='10.1038/nrg1726' publications.firstAuthor='Pearson Joseph C' publications.id=1007889 publications.issue='12' publications.journal='Nat. Rev. Genet.' publications.month='Dec' publications.pages='893-904' publications.pubMedId='16341070' publications.title='Modulating Hox gene functions during animal body patterning.' publications.volume='6' publications.year=2005\n",
      "Gene: publications.abstractText='Worldwide, acute, and chronic pain affects 20% of the adult population and represents an enormous financial and emotional burden. Using genome-wide neuronal-specific RNAi knockdown in Drosophila, we report a global screen for an innate behavior and identify hundreds of genes implicated in heat nociception, including the α2δ family calcium channel subunit straightjacket (stj). Mice mutant for the stj ortholog CACNA2D3 (α2δ3) also exhibit impaired behavioral heat pain sensitivity. In addition, in humans, α2δ3 SNP variants associate with reduced sensitivity to acute noxious heat and chronic back pain. Functional imaging in α2δ3 mutant mice revealed impaired transmission of thermal pain-evoked signals from the thalamus to higher-order pain centers. Intriguingly, in α2δ3 mutant mice, thermal pain and tactile stimulation triggered strong cross-activation, or synesthesia, of brain regions involved in vision, olfaction, and hearing.' publications.doi='10.1016/j.cell.2010.09.047' publications.firstAuthor='Neely G Gregory' publications.id=1000034 publications.issue='4' publications.journal='Cell' publications.month='Nov' publications.pages='628-38' publications.pubMedId='21074052' publications.title='A genome-wide Drosophila screen for heat nociception identifies α2δ3 as an evolutionarily conserved pain gene.' publications.volume='143' publications.year=2010\n",
      "Gene: publications.abstractText=\"cDNA cloning is a central technology in molecular biology. cDNA sequences are used to determine mRNA transcript structures, including splice junctions, open reading frames (ORFs) and 5'- and 3'-untranslated regions (UTRs). cDNA clones are valuable reagents for functional studies of genes and proteins. Expressed Sequence Tag (EST) sequencing is the method of choice for recovering cDNAs representing many of the transcripts encoded in a eukaryotic genome. However, EST sequencing samples a cDNA library at random, and it recovers transcripts with low expression levels inefficiently. We describe a PCR-based method for directed screening of plasmid cDNA libraries. We demonstrate its utility in a screen of libraries used in our Drosophila EST projects for 153 transcription factor genes that were not represented by full-length cDNA clones in our Drosophila Gene Collection. We recovered high-quality, full-length cDNAs for 72 genes and variously compromised clones for an additional 32 genes. The method can be used at any scale, from the isolation of cDNA clones for a particular gene of interest, to the improvement of large gene collections in model organisms and the human. Finally, we discuss the relative merits of directed cDNA library screening and RT-PCR approaches.\" publications.doi='10.1093/nar/gni184' publications.firstAuthor='Hoskins Roger A' publications.id=1000113 publications.issue='21' publications.journal='Nucleic Acids Res.' publications.month=None publications.pages='e185' publications.pubMedId='16326860' publications.title='Rapid and efficient cDNA library screening by self-ligation of inverse PCR products (SLIP).' publications.volume='33' publications.year=2005\n",
      "Gene: publications.abstractText=\"cappuccino and spire are unique Drosophila maternal-effect loci that participate in pattern formation in both the anteroposterior and dorsoventral axes of the early embryo. Mutant females produce embryos lacking pole cells, polar granules, and normal abdominal segmentation. They share these defects with the posterior group of maternal-effect genes. Although embryos are defective in abdominal segmentation, in double mutant combinations with Bicaudal D, abdominal segments can be formed in the anterior half of the egg. This indicates that embryos produced by mutant females contain the 'posterior determinant' required for abdominal segmentation (Nüsslein-Volhard et al. 1987) and suggests that the wild-type gene products are not required for production of the posterior determinant but, rather, for its localization or stabilization. The vasa protein, a component of polar granules, is not localized at the posterior pole of mutant egg chambers or embryos, providing additional support for the hypothesis that localization to or stabilization of substances at the posterior pole of the egg chamber is defective in mutant females. Females mutant for the strongest alleles also produce dorsalized embryos. Phenotypic analysis reveals that these dorsalized embryos also have abdominal segmentation defects. The mutant phenotypes can be ordered in a series of increasing severity. Pole cell formation is most sensitive to loss of functional gene products, followed by abdominal segmentation, whereas normal dorsoventral patterning is the least sensitive to loss of functional gene products. In addition, mutant females contain egg chambers that appear to be dorsalized, resulting in the production of eggs with dorsalized eggshells. Germ-line mosaics indicate that cappuccino and spire are required in the oocyte-nurse cell complex. This suggests that the eggshell phenotype results from altered pattern in the underlying germ cell. Also, we defined the epistatic relationships between several early patterning loci, on the basis of an analysis of the eggs and embryos produced by females doubly mutant for cappuccino or spire and other loci that affect the pattern of both the egg and the embryo. On the basis of our current knowledge of the genes involved in this process, we formulated a working model for the early steps in dorsoventral patterning.\" publications.doi=None publications.firstAuthor='Manseau L J' publications.id=1007698 publications.issue='9' publications.journal='Genes Dev.' publications.month='Sep' publications.pages='1437-52' publications.pubMedId='2514120' publications.title='cappuccino and spire: two unique maternal-effect loci required for both the anteroposterior and dorsoventral patterns of the Drosophila embryo.' publications.volume='3' publications.year=1989\n",
      "Gene: publications.abstractText='decapentaplegic (dpp) encodes a Drosophila transforming growth factor-beta homologue that functions as a morphogen in the developing embryo and in adult appendage formation. In the wing imaginal disc, a Dpp gradient governs patterning along the anteroposterior axis by inducing regional expression of diverse genes in a concentration-dependent manner. Recent studies show that responses to graded Dpp activity also require an input from a complementary and opposing gradient of Brinker (Brk), a transcriptional repressor protein encoded by a Dpp target gene. Here we show that Brk harbours a functional and transferable repression domain, through which it recruits the corepressors Groucho and CtBP. By analysing transcriptional outcomes arising from the genetic removal of these corepressors, and by ectopically expressing Brk variants in the embryo, we demonstrate that these corepressors are alternatively used by Brk for repressing some Dpp-responsive genes, whereas for repressing other distinct target genes they are not required. Our results show that Brk utilizes multiple means to repress its endogenous target genes, allowing repression of a multitude of complex Dpp target promoters.' publications.doi='10.1093/emboj/20.20.5725' publications.firstAuthor='Hasson P' publications.id=1007718 publications.issue='20' publications.journal='EMBO J.' publications.month='Oct' publications.pages='5725-36' publications.pubMedId='11598015' publications.title='Brinker requires two corepressors for maximal and versatile repression in Dpp signalling.' publications.volume='20' publications.year=2001\n",
      "Gene: publications.abstractText='even-skipped (eve) is a homeodomain-encoding gene that is a genetically defined repressor of Ultrabithorax (Ubx), fushi-tarazu (ftz), and wingless (wg). Here we report that purified eve protein represses transcription in vitro at the Ubx promoter, in a DNA binding site-dependent manner. eve protein represses transcription when bound either upstream or downstream of the RNA start site or when DNA binding sites are in either orientation. We also show that eve represses expression from the Ubx promoter in Drosophila tissue culture cells, again in a binding site-dependent manner. Deletion of eve DNA binding sites does not alter transcription in the absence of eve, and so repression is not likely to be the result of eve competitively inhibiting an activator protein from binding to the same DNA element. Instead, we propose that eve protein is probably interfering with the function of proteins bound at other locations in the promoter. The biochemical demonstration that a Drosophila homeodomain protein can directly regulate RNA synthesis strengthens the view that this class of regulators act as transcription factors to control development.' publications.doi=None publications.firstAuthor='Biggin M D' publications.id=1007854 publications.issue='3' publications.journal='Cell' publications.month='Aug' publications.pages='433-40' publications.pubMedId='2569362' publications.title='A purified Drosophila homeodomain protein represses transcription in vitro.' publications.volume='58' publications.year=1989\n",
      "Gene: publications.abstractText=\"mRNA-associated processes and gene structure in eukaryotes are typically treated as separate research subjects. Here, we bridge this separation and leverage the extensive multidisciplinary work on Drosophila melanogaster to examine the roles that capping, splicing, cleavage/polyadenylation, and telescripting (i.e, the protection of nascent transcripts from premature cleavage/polyadenylation by the splicing factor U1) might play in shaping exon-intron architecture in protein-coding genes. Our findings suggest that the distance between subsequent internal 5' splice sites (5'ss) in Drosophila genes is constrained such that telescripting effects are maximized, in theory, and thus nascent transcripts are less vulnerable to premature termination. Exceptionally weak 5'ss and constraints on intron-exon size at the gene 5' end also indicate that capping might enhance the recruitment of U1 and, in turn, promote telescripting at this location. Finally, a positive correlation between last exon length and last 5'ss strength suggests that optimal donor splice sites in the proximity of the pre-mRNA tail may inhibit the processing of downstream polyadenylation signals more than weak donor splice sites do. These findings corroborate and build upon previous experimental and computational studies on Drosophila genes. They support the possibility, hitherto scantly explored, that mRNA-associated processes impose significant constraints on the evolution of eukaryotic gene structure.\" publications.doi='10.1534/g3.116.029231' publications.firstAuthor='Lepennetier Gildas' publications.id=1000121 publications.issue='6' publications.journal='G3 (Bethesda)' publications.month='Jun' publications.pages='1617-26' publications.pubMedId='27172210' publications.title='mRNA-Associated Processes and Their Influence on Exon-Intron Structure in Drosophila melanogaster.' publications.volume='6' publications.year=2016\n",
      "Gene: publications.abstractText=\"twist is one of the earliest expressed zygotically active genes required for dorsal-ventral pattern formation in the Drosophila embryo. Genetic studies suggest that this gene is activated in the ventral part of the blastoderm by maternally expressed dorsal gene product. Using P-element-mediated germ-line transformation, we have mapped a small (260 bp) dorsal-dependent ventral activator region (VAR) in the 5'-flanking region of the twist promoter that can direct the early ventral expression of a heterologous promoter. The VAR contains binding sites for a number of proteins present in extracts of Drosophila embryos. One of these sites bears homology to known binding sites for the dorsal transcription factor and is specifically bound by bacterially expressed dorsal protein. Furthermore, a 37-bp deletion that removes the dorsal-binding sequences abolishes the ventral-specific activity of the twist promoter constructs. Our data also show that additional sequences within the VAR are required to render the dorsal-binding sites functional. Finally, reverse genetic and biochemical data suggest that the transcription factor, encoded by the zeste gene may help control the overall level, if not the pattern, of twist expression.\" publications.doi=None publications.firstAuthor='Pan D J' publications.id=1007837 publications.issue='10' publications.journal='Genes Dev.' publications.month='Oct' publications.pages='1892-901' publications.pubMedId='1655573' publications.title='Functional analysis of the Drosophila twist promoter reveals a dorsal-binding ventral activator region.' publications.volume='5' publications.year=1991\n",
      "Gene: publications.abstractText='wimp is a dominant maternal-effect mutation that interacts with a specific subset of early-acting maternal and zygotic Drosophila genes. We show that wimp is a change-of-function mutation, allelic to mutations of the 140-kD subunit of RNA polymerase, which causes reduced transcription of interacting genes. Loci that do not interact with wimp are expressed at normal levels. We discuss these results in terms of specific interactions between transcription factors and RNA polymerase. Embryos from wimp mothers show unaltered fate maps and develop normally, despite the reduction of transcript levels at least twofold. We suggest that spatial cues are determined by a balance of segmentation gene products rather than their absolute concentrations. We also demonstrate powerful genetic screens for otherwise undetected loci required for segmentation, sex determination, and other early functions.' publications.doi=None publications.firstAuthor='Parkhurst S M' publications.id=1007811 publications.issue='3' publications.journal='Genes Dev.' publications.month='Mar' publications.pages='341-57' publications.pubMedId='2001838' publications.title='wimp, a dominant maternal-effect mutation, reduces transcription of a specific subset of segmentation genes in Drosophila.' publications.volume='5' publications.year=1991\n",
      "Gene: publications.abstractText='zerknüllt (zen) is one of approximately 10 zygotically active genes that control the differentiation of the dorsal-ventral (D/V) pattern during early embryogenesis in Drosophila. Past genetic analyses suggest that maternal factors repress the expression of zen in ventral regions, thereby restricting zen products to dorsal and dorsal-lateral regions of precellular embryos. Subsequent interactions with other zygotic D/V regulatory genes refine the zen pattern, restricting expression to the dorsal-most ectoderm. Here we describe the use of zen promoter fusions and P-element transformation to identify cis elements that are responsible for the complex spatial pattern of zen expression. The zen promoter shows a two-tier organization: Distal sequences mediate its initial response to maternal factors, whereas proximal sequences are responsible for the refinement of the pattern in older embryos. The distal regulatory element has the property of a silencer (or anti-enhancer) element and can act over a distance to repress ventral expression of a heterologous promoter. Also, we discuss evidence that proximal promoter sequences interact with factors that may be modulated by a cell-cell communication pathway.' publications.doi=None publications.firstAuthor='Doyle H J' publications.id=1007901 publications.issue='10' publications.journal='Genes Dev.' publications.month='Oct' publications.pages='1518-33' publications.pubMedId='2612903' publications.title='Spatial regulation of zerknüllt: a dorsal-ventral patterning gene in Drosophila.' publications.volume='3' publications.year=1989\n",
      "Gene: publications.abstractText='zerknüllt (zen) is unique among the 18 known homeo box genes in Drosophila since it is required for the differentiation of the dorsal-ventral pattern, and does not appear to be involved in the process of segmentation. Here we show that the zen region of the Antennapedia complex (ANT-C) consists of two closely linked homeo box genes, designated z1 and z2. The z1 and z2 transcription units show essentially identical patterns of expression during early development, which are consistent with the timing and sites of zen+ gene activity. The putative proteins encoded by z1 and z2 are highly divergent and are related only by virtue of homeo box homology. We have used P-element-mediated germ line transformation to show that z1 alone can provide zen+ gene function, suggesting that the z2 gene might be dispensable. The occurrence of closely linked homeo box genes that display similar patterns of expression is not unique to the zen locus. Such gene duplications might provide important clues to the evolution of the homeo box gene family in Drosophila and other organisms.' publications.doi=None publications.firstAuthor='Rushlow C' publications.id=1007850 publications.issue='10' publications.journal='Genes Dev.' publications.month='Dec' publications.pages='1268-79' publications.pubMedId='2892759' publications.title='Molecular characterization of the zerknüllt region of the Antennapedia gene complex in Drosophila.' publications.volume='1' publications.year=1987\n",
      "Gene: publications.abstractText='β-Arrestins have been implicated in the regulation of multiple signalling pathways. However, their role in organism development is not well understood. In this study, we report a new in vivo function of the Drosophila β-arrestin Kurtz (Krz) in the regulation of two distinct developmental signalling modules: MAPK ERK and NF-κB, which transmit signals from the activated receptor tyrosine kinases (RTKs) and the Toll receptor, respectively. Analysis of the expression of effectors and target genes of Toll and the RTK Torso in krz maternal mutants reveals that Krz limits the activity of both pathways in the early embryo. Protein interaction studies suggest a previously uncharacterized mechanism for ERK inhibition: Krz can directly bind and sequester an inactive form of ERK, thus preventing its activation by the upstream kinase, MEK. A simultaneous dysregulation of different signalling systems in krz mutants results in an abnormal patterning of the embryo and severe developmental defects. Our findings uncover a new in vivo function of β-arrestins and present a new mechanism of ERK inhibition by the Drosophila β-arrestin Krz.' publications.doi='10.1038/emboj.2010.202' publications.firstAuthor='Tipping Marla' publications.id=1007801 publications.issue='19' publications.journal='EMBO J.' publications.month='Oct' publications.pages='3222-35' publications.pubMedId='20802461' publications.title='β-arrestin Kurtz inhibits MAPK and Toll signalling in Drosophila development.' publications.volume='29' publications.year=2010\n",
      "Gene: publications.abstractText=None publications.doi='10.1002/(SICI)1521-1878(200003)22:3<305::AID-BIES13>3.0.CO;2-R' publications.firstAuthor='Wilkins A S' publications.id=1007808 publications.issue='3' publications.journal='Bioessays' publications.month='Mar' publications.pages='305-7' publications.pubMedId='10684591' publications.title='Drosophila: flying high in Zürich.' publications.volume='22' publications.year=2000\n",
      "Gene: publications.abstractText=None publications.doi='10.1002/bies.950121107' publications.firstAuthor='White R J' publications.id=1007721 publications.issue='11' publications.journal='Bioessays' publications.month='Nov' publications.pages='537-9' publications.pubMedId='1982208' publications.title='Cell type-specific enhancement in the Drosophila embryo by consensus homeodomain binding sites.' publications.volume='12' publications.year=1990\n",
      "Gene: publications.abstractText=None publications.doi='10.1038/35056022' publications.firstAuthor='Casci T' publications.id=1007707 publications.issue='3' publications.journal='Nat. Rev. Genet.' publications.month='Mar' publications.pages='161' publications.pubMedId='11256063' publications.title='Patterning back to front.' publications.volume='2' publications.year=2001\n",
      "Gene: publications.abstractText=None publications.doi='10.1093/emboj/17.23.6769' publications.firstAuthor='Shilo B Z' publications.id=1007876 publications.issue='23' publications.journal='EMBO J.' publications.month='Dec' publications.pages='6769-71' publications.pubMedId='9835652' publications.title='Flies over Crete: Drosophila molecular biology. Kolymbari, Crete, July 12-18, 1998.' publications.volume='17' publications.year=1998\n",
      "Gene: publications.abstractText=None publications.doi='10.1101/gad.939601' publications.firstAuthor='Courey A J' publications.id=1007902 publications.issue='21' publications.journal='Genes Dev.' publications.month='Nov' publications.pages='2786-96' publications.pubMedId='11691830' publications.title='Transcriptional repression: the long and the short of it.' publications.volume='15' publications.year=2001\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Bier E' publications.id=1007775 publications.issue='5' publications.journal='Cell' publications.month='May' publications.pages='681-4' publications.pubMedId='9182755' publications.title='Anti-neural-inhibition: a conserved mechanism for neural induction.' publications.volume='89' publications.year=1997\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Budd P S' publications.id=1007871 publications.issue='3' publications.journal='Trends Genet.' publications.month='Mar' publications.pages='74-6' publications.pubMedId='1674389' publications.title='What do the regulators regulate? First glimpse downstream.' publications.volume='7' publications.year=1991\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Courey A J' publications.id=1007792 publications.issue='1' publications.journal='Biochim. Biophys. Acta' publications.month='Mar' publications.pages='1-18' publications.pubMedId='7893745' publications.title='The establishment and interpretation of transcription factor gradients in the Drosophila embryo.' publications.volume='1261' publications.year=1995\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Denell R' publications.id=1007804 publications.issue='3' publications.journal='Genetics' publications.month='Nov' publications.pages='549-52' publications.pubMedId='7851753' publications.title='Discovery and genetic definition of the Drosophila Antennapedia complex.' publications.volume='138' publications.year=1994\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Fraser S E' publications.id=1007694 publications.issue='1' publications.journal='Cell' publications.month='Jan' publications.pages='41-55' publications.pubMedId='10647930' publications.title='The molecular metamorphosis of experimental embryology.' publications.volume='100' publications.year=2000\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Hayashi S' publications.id=1007773 publications.issue='5' publications.journal='Cell' publications.month='Nov' publications.pages='883-94' publications.pubMedId='1979524' publications.title='What determines the specificity of action of Drosophila homeodomain proteins?' publications.volume='63' publications.year=1990\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Kaufman T C' publications.id=1005520 publications.issue=None publications.journal='Adv. Genet.' publications.month=None publications.pages='309-62' publications.pubMedId='1971986' publications.title='Molecular and genetic organization of the antennapedia gene complex of Drosophila melanogaster.' publications.volume='27' publications.year=1990\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='McGinnis W' publications.id=1007704 publications.issue='2' publications.journal='Cell' publications.month='Jan' publications.pages='283-302' publications.pubMedId='1346368' publications.title='Homeobox genes and axial patterning.' publications.volume='68' publications.year=1992\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Miklos G L' publications.id=1007726 publications.issue='4' publications.journal='Cell' publications.month='Aug' publications.pages='521-9' publications.pubMedId='8752207' publications.title='The role of the genome project in determining gene function: insights from model organisms.' publications.volume='86' publications.year=1996\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Read D' publications.id=1007891 publications.issue='2-3' publications.journal='Biochim. Biophys. Acta' publications.month='Dec' publications.pages='79-93' publications.pubMedId='1457465' publications.title='Transcriptional regulators of Drosophila embryogenesis.' publications.volume='1114' publications.year=1992\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Rushlow C' publications.id=1007765 publications.issue=None publications.journal='Adv. Genet.' publications.month=None publications.pages='277-307' publications.pubMedId='2112301' publications.title='Role of the zerknüllt gene in dorsal-ventral pattern formation in Drosophila.' publications.volume='27' publications.year=1990\n",
      "Gene: publications.abstractText=None publications.doi=None publications.firstAuthor='Zhou J' publications.id=1007844 publications.issue=None publications.journal='Cold Spring Harb. Symp. Quant. Biol.' publications.month=None publications.pages='307-12' publications.pubMedId='9598364' publications.title='The regulation of enhancer-promoter interactions in the Drosophila embryo.' publications.volume='62' publications.year=1997\n"
     ]
    }
   ],
   "source": [
    "for gene in query.rows():\n",
    "    print(gene)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Iterating through query.results(row=\"rr\") and Iterating through query.rows() are equivalent. Feel free to use whichever you feel more comfortable with. If you want to extract only specific columns, it may be easier to use \"list\" instead of rr. Let's say you want to extract column 2 & 3, i.e. publications.doi and publications.firstAuthor, then it can be done as follows. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "None Yamaguchi M\n",
      "None Kwon Eunjeong\n",
      "10.1006/jtbi.1996.0328 Bodnar J W\n",
      "None Ip Y T\n",
      "None Jiang J\n",
      "10.1038/332858a0 Hoey T\n",
      "10.1038/35101500 Gurdon J B\n",
      "10.1016/j.cub.2005.03.022 Schmidt-Ott Urs\n",
      "10.1242/dev.001065 Duboule Denis\n",
      "10.1002/1521-1878(200102)23:2<125::AID-BIES1019>3.0.CO;2-C Jagla K\n",
      "10.1002/bies.950170904 Venkatesh T V\n",
      "None Patel N H\n",
      "None Agosti D\n",
      "None Geisler R\n",
      "None Hashimoto Carl\n",
      "None Jones C M\n",
      "None Kidd S\n",
      "None Carroll S B\n",
      "10.1242/dev.067926 Wilson Megan J\n",
      "10.1242/dev.01124 Markstein Michele\n",
      "10.1038/35068578 Ross J J\n",
      "10.1016/j.tig.2003.10.009 Raftery Laurel A\n",
      "None Jaźwińska A\n",
      "None González-Reyes A\n",
      "10.1371/journal.pgen.1002769 Holmqvist Per-Henrik\n",
      "None Nepveu A\n",
      "10.1101/gad.215459.113 Saunders Abbie\n",
      "None Strecker T R\n",
      "10.1093/gbe/evr061 Gehring Walter J\n",
      "10.1007/s00438-006-0187-8 Roy Swarnava\n",
      "10.1038/ncomms8102 Ugrankar Rupali\n",
      "None Markstein Michele\n",
      "10.1101/gr.104018.109 Ozdemir Anil\n",
      "10.1371/journal.pbio.1000456 Kazemian Majid\n",
      "None Comeron J M\n",
      "10.1101/gr.178701 Bergman C M\n",
      "None Akam M\n",
      "10.1371/journal.pcbi.1001020 Su Jing\n",
      "10.1038/ng1908 Shirangi Troy R\n",
      "None Hirose F\n",
      "10.1016/j.semcdb.2014.04.036 Hamaratoglu Fisun\n",
      "10.1016/j.devcel.2011.05.009 Kim Yoosik\n",
      "None Affolter M\n",
      "None Jiménez G\n",
      "10.7554/eLife.04837 Nitta Kazuhiro R\n",
      "10.1242/dev.019802 Pope Karen L\n",
      "None Valentine S A\n",
      "10.1038/nature07214 Wang Xiaomeng\n",
      "None Huang A M\n",
      "None Rusch J\n",
      "None Ip Y T\n",
      "10.1016/j.ydbio.2014.02.007 Künnapuu Jaana\n",
      "None Chang Andy J\n",
      "None Reim Ingolf\n",
      "None Manak J R\n",
      "None Yip M L\n",
      "10.1126/science.1090289 Giot L\n",
      "10.1006/dbio.1994.1306 Warrior R\n",
      "None Edgar B A\n",
      "10.1371/journal.pbio.0050117 De Renzis Stefano\n",
      "10.1038/sj.emboj.7601532 Astigarraga Sergio\n",
      "None Wakimoto B T\n",
      "None Steward R\n",
      "None Kumar Sudhir\n",
      "10.1006/jtbi.1995.0175 Takeda Y\n",
      "None Norris J L\n",
      "10.1101/gr.6490707 Copley Richard R\n",
      "None Arnosti D N\n",
      "None Decoville M\n",
      "None Hsia Cheryl C\n",
      "10.1038/380037a0 De Robertis E M\n",
      "10.1146/annurev.genet.35.102401.090832 Lall S\n",
      "10.1101/gad.1509607 Zeitlinger Julia\n",
      "10.1038/ncomms10115 Zwarts Liesbeth\n",
      "None Jäckle H\n",
      "10.1126/science.1139816 Hoskins Roger A\n",
      "10.1242/dev.01722 Xu Mu\n",
      "10.1371/journal.pone.0030610 Turki-Judeh Wiam\n",
      "10.1101/gad.188052.112 Kvon Evgeny Z\n",
      "10.1006/dbio.2002.0652 Stathopoulos Angelike\n",
      "10.1016/j.ydbio.2006.08.027 Friedrich Markus\n",
      "10.1002/1097-010X(20001215)288:4<345::AID-JEZ7>3.0.CO;2-Y Coulier F\n",
      "10.1101/gr.3468605 Negre Bárbara\n",
      "None Duboule D\n",
      "10.1371/journal.pbio.0060027 Li Xiao-yong\n",
      "10.1038/323076a0 Doyle H J\n",
      "None Stein D\n",
      "10.1002/bies.20285 McGregor Alistair P\n",
      "None Treisman J E\n",
      "None Jiang J\n",
      "None Rusch J\n",
      "10.1534/genetics.112.144915 Staller Max V\n",
      "10.1007/s00438-010-0591-y Roy Swarnava\n",
      "10.1038/nature07388 Liang Hsiao-Lan\n",
      "10.1038/nature03318 Wang Yu-Chiun\n",
      "10.1371/journal.pgen.1002086 Tsurumi Amy\n",
      "10.1371/journal.pgen.1002339 Nien Chung-Yi\n",
      "10.1016/j.ymeth.2014.01.003 Wunderlich Zeba\n",
      "None Chen G\n",
      "None Senger Kate\n",
      "None Gray S\n",
      "10.1186/1471-2148-6-106 Casillas Sònia\n",
      "None Pan D\n",
      "None Tautz D\n",
      "10.1093/nar/gkr672 Mrinal Nirotpal\n",
      "10.7554/eLife.00861 Chen Kai\n",
      "None Thisse C\n",
      "10.1016/j.ydbio.2010.06.026 Harrison Melissa M\n",
      "10.1534/genetics.109.110122 Salzer Claire L\n",
      "10.1073/pnas.012591199 Markstein Michele\n",
      "10.1016/j.gde.2004.07.004 Stathopoulos Angelike\n",
      "10.1186/1471-2105-5-202 Gurunathan Rajalakshmi\n",
      "10.1006/dbio.1997.8689 Kourakis M J\n",
      "10.1242/dev.02689 Lin Meng-chi\n",
      "10.1242/dev.02251 Pilot Fanny\n",
      "None Farge Emmanuel\n",
      "10.1101/gad.494808 Miles Wayne O\n",
      "10.1093/nar/gkp619 Papatsenko Dmitri\n",
      "None Hoskins Roger A\n",
      "10.1038/nrg1724 Peel Andrew D\n",
      "None Cribbs D L\n",
      "10.1371/journal.pgen.1005969 Kwon So Yeon\n",
      "10.1186/1752-0509-6-31 Kim Man-Sun\n",
      "10.1371/journal.pone.0001115 Aerts Stein\n",
      "10.1093/emboj/19.17.4463 Frémion F\n",
      "10.1038/nrg947 Simpson Pat\n",
      "10.1038/nature15545 Stampfel Gerald\n",
      "None Hoch M\n",
      "10.1242/dev.079772 Liang Hsiao-Lan\n",
      "None Francois V\n",
      "None Ip Y T\n",
      "10.1016/j.devcel.2011.12.007 Reeves Gregory T\n",
      "10.1016/j.ydbio.2013.02.010 Duncan Elizabeth J\n",
      "10.1093/molbev/msu286 Moyers Bryan A\n",
      "10.1371/journal.pgen.1000178 Kwong Camilla\n",
      "10.1007/s10709-006-0031-4 Demuth Jeffery P\n",
      "None Decotto E\n",
      "None Long A D\n",
      "10.1038/nmeth.1548 Chung Kwanghun\n",
      "None Martinho Rui Gonçalo\n",
      "None Kamens J\n",
      "10.1016/j.ygeno.2008.09.006 Bai Yongsheng\n",
      "None Ferguson E L\n",
      "10.1093/emboj/20.13.3298 Affolter M\n",
      "None Nicholls R E\n",
      "10.1091/mbc.E06-09-0790 Ullah Zakir\n",
      "10.1101/gad.1350705 Massagué Joan\n",
      "None Mannervik M\n",
      "None Lynch Jeremy\n",
      "None Ryu J R\n",
      "None Dubnicoff T\n",
      "None Jia Songtao\n",
      "None Huang J D\n",
      "None Chen G\n",
      "10.1016/j.cub.2006.05.050 Chen Li-Ying\n",
      "None Hoffmann F M\n",
      "None Jaynes J B\n",
      "None Chang Ting\n",
      "None Stauber M\n",
      "None Wimmer E A\n",
      "None Celniker Susan E\n",
      "None Govind S\n",
      "None Johnson Hamlet M R\n",
      "10.1371/journal.pone.0098585 Heimiller Joseph\n",
      "10.1242/dev.02373 ten Bosch John R\n",
      "None Yu Kweon\n",
      "10.1534/genetics.105.041327 Armstrong Jennifer A\n",
      "None Schulz R A\n",
      "None Winick J\n",
      "10.1002/bies.10180 Damen Wim G M\n",
      "10.1093/emboj/18.3.605 Asha H\n",
      "None Tatei K\n",
      "10.1242/dev.02643 Ratnaparkhi Girish S\n",
      "10.1016/j.ymeth.2014.03.016 Peterson Aidan J\n",
      "10.1016/j.smim.2014.05.003 Igaki Tatsushi\n",
      "None Hudson J B\n",
      "10.1016/j.tig.2006.12.001 Negre Bárbara\n",
      "None Schubert F R\n",
      "10.1002/bies.950140302 Treisman J\n",
      "None Randazzo F M\n",
      "None Jiang J\n",
      "None Rusch J\n",
      "None Kirov N\n",
      "None Goff D J\n",
      "10.1146/annurev.ge.29.120195.002103 Morisato D\n",
      "10.1371/journal.pgen.1002266 Harrison Melissa M\n",
      "10.1101/gad.861401 Rushlow C\n",
      "None Roth S\n",
      "None Lewis R A\n",
      "None Hirose F\n",
      "10.1371/journal.pone.0010820 Hueber Stefanie D\n",
      "None Schmid K J\n",
      "None Adams M D\n",
      "None Häcker U\n",
      "10.1016/j.jtbi.2008.01.027 Ishihara Shuji\n",
      "10.1371/journal.pone.0029172 Garcia Mayra\n",
      "None Yamaguchi M\n",
      "10.1073/pnas.0408031102 Levine Michael\n",
      "10.1016/j.cub.2005.10.026 Moussian Bernard\n",
      "None Treisman J\n",
      "None Barad M\n",
      "10.1038/330583a0 Rushlow C\n",
      "None Seeger M A\n",
      "None Stathopoulos Angelike\n",
      "None Govind S\n",
      "None Shin Dong-Hyeon\n",
      "None Skeath J B\n",
      "10.1073/pnas.012292899 Stauber Michael\n",
      "None Berleth T\n",
      "10.1371/journal.pgen.0010057 Barker Michael S\n",
      "10.1016/j.cub.2004.07.018 Santos Ana C\n",
      "None Roth S\n",
      "10.1093/emboj/17.23.7009 Nibu Y\n",
      "None Nüsslein-Volhard C\n",
      "None Misra Sima\n",
      "10.1126/science.1139815 Smith Christopher D\n",
      "10.1016/j.ydbio.2006.05.004 Anderson Jason\n",
      "None Hariharan I K\n",
      "None Biehs B\n",
      "None Arora K\n",
      "10.1016/j.devcel.2005.09.005 Stathopoulos Angelike\n",
      "None Clifford R\n",
      "None Kazemi-Esfarjani Parsa\n",
      "10.1186/gb-2007-8-4-r67 Beltran Sergi\n",
      "None Mason E D\n",
      "None Munté A\n",
      "None Sanchez null\n",
      "None Lipshitz H D\n",
      "10.1016/j.cub.2013.09.055 Gavin-Smyth Jackie\n",
      "10.1038/nature13395 Kvon Evgeny Z\n",
      "10.1098/rstb.1995.0111 Gray S\n",
      "None Cai H N\n",
      "10.1242/dev.099366 Upadhyai Priyanka\n",
      "10.1371/journal.pcbi.0010022 Quesneville Hadi\n",
      "None Kaminker Joshua S\n",
      "None Gibson G\n",
      "None Hazelrigg T\n",
      "10.1371/journal.pcbi.1000652 Kim Jaebum\n",
      "10.1016/j.ydbio.2005.12.014 Scuderi Anne\n",
      "None Ray R P\n",
      "None Chang T\n",
      "None Terol J\n",
      "None Falciani F\n",
      "None Cavarec L\n",
      "None Panzer S\n",
      "None Pritchard D K\n",
      "None Zuo P\n",
      "None Han K\n",
      "None Merrill V K\n",
      "None Han K\n",
      "None Han K\n",
      "None Podos S D\n",
      "10.4161/19336934.2014.983385 Gavin-Smyth Jackie\n",
      "None George H\n",
      "None Tatei K\n",
      "None Brickman J M\n",
      "10.1006/dbio.1997.8625 Saha M S\n",
      "None Müller H A\n",
      "10.1038/nrg1726 Pearson Joseph C\n",
      "10.1016/j.cell.2010.09.047 Neely G Gregory\n",
      "10.1093/nar/gni184 Hoskins Roger A\n",
      "None Manseau L J\n",
      "10.1093/emboj/20.20.5725 Hasson P\n",
      "None Biggin M D\n",
      "10.1534/g3.116.029231 Lepennetier Gildas\n",
      "None Pan D J\n",
      "None Parkhurst S M\n",
      "None Doyle H J\n",
      "None Rushlow C\n",
      "10.1038/emboj.2010.202 Tipping Marla\n",
      "10.1002/(SICI)1521-1878(200003)22:3<305::AID-BIES13>3.0.CO;2-R Wilkins A S\n",
      "10.1002/bies.950121107 White R J\n",
      "10.1038/35056022 Casci T\n",
      "10.1093/emboj/17.23.6769 Shilo B Z\n",
      "10.1101/gad.939601 Courey A J\n",
      "None Bier E\n",
      "None Budd P S\n",
      "None Courey A J\n",
      "None Denell R\n",
      "None Fraser S E\n",
      "None Hayashi S\n",
      "None Kaufman T C\n",
      "None McGinnis W\n",
      "None Miklos G L\n",
      "None Read D\n",
      "None Rushlow C\n",
      "None Zhou J\n"
     ]
    }
   ],
   "source": [
    "for gene in query.results(\"list\"):\n",
    "    print(gene[1],gene[2])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "If you want to print only those rows where publications.doi is not None then you can add an if condition as shown below. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "10.1006/jtbi.1996.0328\n",
      "10.1038/332858a0\n",
      "10.1038/35101500\n",
      "10.1016/j.cub.2005.03.022\n",
      "10.1242/dev.001065\n",
      "10.1002/1521-1878(200102)23:2<125::AID-BIES1019>3.0.CO;2-C\n",
      "10.1002/bies.950170904\n",
      "10.1242/dev.067926\n",
      "10.1242/dev.01124\n",
      "10.1038/35068578\n",
      "10.1016/j.tig.2003.10.009\n",
      "10.1371/journal.pgen.1002769\n",
      "10.1101/gad.215459.113\n",
      "10.1093/gbe/evr061\n",
      "10.1007/s00438-006-0187-8\n",
      "10.1038/ncomms8102\n",
      "10.1101/gr.104018.109\n",
      "10.1371/journal.pbio.1000456\n",
      "10.1101/gr.178701\n",
      "10.1371/journal.pcbi.1001020\n",
      "10.1038/ng1908\n",
      "10.1016/j.semcdb.2014.04.036\n",
      "10.1016/j.devcel.2011.05.009\n",
      "10.7554/eLife.04837\n",
      "10.1242/dev.019802\n",
      "10.1038/nature07214\n",
      "10.1016/j.ydbio.2014.02.007\n",
      "10.1126/science.1090289\n",
      "10.1006/dbio.1994.1306\n",
      "10.1371/journal.pbio.0050117\n",
      "10.1038/sj.emboj.7601532\n",
      "10.1006/jtbi.1995.0175\n",
      "10.1101/gr.6490707\n",
      "10.1038/380037a0\n",
      "10.1146/annurev.genet.35.102401.090832\n",
      "10.1101/gad.1509607\n",
      "10.1038/ncomms10115\n",
      "10.1126/science.1139816\n",
      "10.1242/dev.01722\n",
      "10.1371/journal.pone.0030610\n",
      "10.1101/gad.188052.112\n",
      "10.1006/dbio.2002.0652\n",
      "10.1016/j.ydbio.2006.08.027\n",
      "10.1002/1097-010X(20001215)288:4<345::AID-JEZ7>3.0.CO;2-Y\n",
      "10.1101/gr.3468605\n",
      "10.1371/journal.pbio.0060027\n",
      "10.1038/323076a0\n",
      "10.1002/bies.20285\n",
      "10.1534/genetics.112.144915\n",
      "10.1007/s00438-010-0591-y\n",
      "10.1038/nature07388\n",
      "10.1038/nature03318\n",
      "10.1371/journal.pgen.1002086\n",
      "10.1371/journal.pgen.1002339\n",
      "10.1016/j.ymeth.2014.01.003\n",
      "10.1186/1471-2148-6-106\n",
      "10.1093/nar/gkr672\n",
      "10.7554/eLife.00861\n",
      "10.1016/j.ydbio.2010.06.026\n",
      "10.1534/genetics.109.110122\n",
      "10.1073/pnas.012591199\n",
      "10.1016/j.gde.2004.07.004\n",
      "10.1186/1471-2105-5-202\n",
      "10.1006/dbio.1997.8689\n",
      "10.1242/dev.02689\n",
      "10.1242/dev.02251\n",
      "10.1101/gad.494808\n",
      "10.1093/nar/gkp619\n",
      "10.1038/nrg1724\n",
      "10.1371/journal.pgen.1005969\n",
      "10.1186/1752-0509-6-31\n",
      "10.1371/journal.pone.0001115\n",
      "10.1093/emboj/19.17.4463\n",
      "10.1038/nrg947\n",
      "10.1038/nature15545\n",
      "10.1242/dev.079772\n",
      "10.1016/j.devcel.2011.12.007\n",
      "10.1016/j.ydbio.2013.02.010\n",
      "10.1093/molbev/msu286\n",
      "10.1371/journal.pgen.1000178\n",
      "10.1007/s10709-006-0031-4\n",
      "10.1038/nmeth.1548\n",
      "10.1016/j.ygeno.2008.09.006\n",
      "10.1093/emboj/20.13.3298\n",
      "10.1091/mbc.E06-09-0790\n",
      "10.1101/gad.1350705\n",
      "10.1016/j.cub.2006.05.050\n",
      "10.1371/journal.pone.0098585\n",
      "10.1242/dev.02373\n",
      "10.1534/genetics.105.041327\n",
      "10.1002/bies.10180\n",
      "10.1093/emboj/18.3.605\n",
      "10.1242/dev.02643\n",
      "10.1016/j.ymeth.2014.03.016\n",
      "10.1016/j.smim.2014.05.003\n",
      "10.1016/j.tig.2006.12.001\n",
      "10.1002/bies.950140302\n",
      "10.1146/annurev.ge.29.120195.002103\n",
      "10.1371/journal.pgen.1002266\n",
      "10.1101/gad.861401\n",
      "10.1371/journal.pone.0010820\n",
      "10.1016/j.jtbi.2008.01.027\n",
      "10.1371/journal.pone.0029172\n",
      "10.1073/pnas.0408031102\n",
      "10.1016/j.cub.2005.10.026\n",
      "10.1038/330583a0\n",
      "10.1073/pnas.012292899\n",
      "10.1371/journal.pgen.0010057\n",
      "10.1016/j.cub.2004.07.018\n",
      "10.1093/emboj/17.23.7009\n",
      "10.1126/science.1139815\n",
      "10.1016/j.ydbio.2006.05.004\n",
      "10.1016/j.devcel.2005.09.005\n",
      "10.1186/gb-2007-8-4-r67\n",
      "10.1016/j.cub.2013.09.055\n",
      "10.1038/nature13395\n",
      "10.1098/rstb.1995.0111\n",
      "10.1242/dev.099366\n",
      "10.1371/journal.pcbi.0010022\n",
      "10.1371/journal.pcbi.1000652\n",
      "10.1016/j.ydbio.2005.12.014\n",
      "10.4161/19336934.2014.983385\n",
      "10.1006/dbio.1997.8625\n",
      "10.1038/nrg1726\n",
      "10.1016/j.cell.2010.09.047\n",
      "10.1093/nar/gni184\n",
      "10.1093/emboj/20.20.5725\n",
      "10.1534/g3.116.029231\n",
      "10.1038/emboj.2010.202\n",
      "10.1002/(SICI)1521-1878(200003)22:3<305::AID-BIES13>3.0.CO;2-R\n",
      "10.1002/bies.950121107\n",
      "10.1038/35056022\n",
      "10.1093/emboj/17.23.6769\n",
      "10.1101/gad.939601\n"
     ]
    }
   ],
   "source": [
    "for row in query.results(row=\"list\"):\n",
    "    if row[1]!=None:\n",
    "        print(row[1])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "You can pass two more parameters while passing query.results(). These are start and size. Start represents the row number that you want to start processing from. By default this is set to 0 (first row). Size represents the number of rows that you want to print. Lets say we want to print rows 10 and 11 only. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Gene: publications.abstractText='Although the genetics of dorsal-ventral polarity which leads to mesoderm formation in Drosophila are understood in considerable detail, subsequent molecular mechanisms involved in patterning the mesoderm primordium into individual mesodermal subtypes are poorly understood. Two papers published recently suggest strongly that an inductive signal from dorsal ectoderm is involved in subdividing the underlying mesoderm, and present evidence that one of the signalling factors is Decapentaplegic (Dpp), a member of the bone morphogenetic protein subgroup of the Transforming Growth Factor-beta (TGF-beta) super family of proteins.' publications.doi='10.1002/bies.950170904' publications.firstAuthor='Venkatesh T V' publications.id=1007725 publications.issue='9' publications.journal='Bioessays' publications.month='Sep' publications.pages='754-7' publications.pubMedId='8763827' publications.title='How many signals does it take?' publications.volume='17' publications.year=1995\n",
      "Gene: publications.abstractText='Although the mechanisms of anterior-posterior axis formation are well understood in Drosophila, both embryological and molecular studies suggest significant variation in the mechanisms generating this axis within the Insecta class as a whole.' publications.doi=None publications.firstAuthor='Patel N H' publications.id=1007793 publications.issue='9' publications.journal='Proc. Natl. Acad. Sci. U.S.A.' publications.month='Apr' publications.pages='4442-4' publications.pubMedId='10781040' publications.title=\"It's a bug's life.\" publications.volume='97' publications.year=2000\n"
     ]
    }
   ],
   "source": [
    "for row in query.results(row=\"rr\",size=2,start=10):\n",
    "    print(row)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "If you prefer dealing with lists of strings, i.e. csv and tsv objects you can use them too. First, you need to import the csv library. If you want to read your results in a CSV format, you should create a csv.reader object. This has been shown below. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 46,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "import csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 47,
   "metadata": {
    "collapsed": true
   },
   "outputs": [],
   "source": [
    "csv_reader = csv.reader(query.results(row=\"csv\"), delimiter=\",\", quotechar='\"')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 48,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "A 631-bp fragment containing the 5'-flanking region of the Drosophila melanogaster proliferating-cell nuclear antigen (PCNA) gene was placed upstream of the chloramphenicol acetyltransferase (CAT) gene of a CAT vector. A transient expression assay of CAT activity in Drosophila Kc cells transfected with this plasmid and a set of 5'-deletion derivatives revealed that the promoter function resided within a 192-bp region (-168 to +24 with respect to the transcription initiation site). Cotransfection with a zerknüllt (zen)-expressing plasmid specifically repressed CAT expression. However, cotransfection with expression plasmids for a nonfunctional zen mutation, even-skipped, or bicoid showed no significant effect on CAT expression. RNase protection analysis revealed that the repression by zen was at the transcription step. The target sequence of zen was mapped within the 34-bp region (-119 to -86) of the PCNA gene promoter, even though it lacked zen protein-binding sites. Transgenic flies carrying the PCNA gene regulatory region (-607 to +137 or -168 to +137) fused with lacZ were established. When these flies were crossed with the zen mutant, ectopic expression of lacZ was observed in the dorsal region of gastrulating embryos carrying the transgene with either construct. These results indicate that zen indirectly represses PCNA gene expression, probably by regulating the expression of some transcription factor(s) that binds to the PCNA gene promoter.\n",
      "A DNA replication-related element (DRE)-binding factor (DREF) has been revealed to be an important transcription factor for activating promoters of cell proliferation and differentiation related genes. The amino acid sequences of DREF are conserved in evolutionary separate Drosophila species, Drosophila melanogaster (Dm) and Drosophila virilis (Dv) in three regions. In the present study, evidence was obtained that there are several highly conserved regions in the 5' flanking region between the DmDREF and DvDREF genes. Band mobility shift assays using oligonucleotides corresponding to these conserved regions revealed that specific trans-acting factors can bind to at least three regions -554 to -543 (5'-TTTGTTCTTGCG), -81 to -70 (5'-GCCCACGTGGCT) and +225 to +234 (5'-GCAATCAGTG). Using a transient luciferase expression assay, we demonstrated that the region -554 to -543 functions as a negative regulatory element for DmDREF promoter activity, while the regions -77 to -70 (5'-ACGTGGCT) and +225 to +236 (5'-GCAATCAGTGTT) function as positive regulatory elements. In previous studies, we observed that expression of the homeodomain protein Zerknüllt (Zen) represses PCNA gene transcription, by reducing the DNA binding activity of DREF. Here we show Zen downregulates DREF gene promoter activity through action on the region between +241 and +254 (5'-AGAATACTCAACA). In addition, the DmDREF promoter contains five DREs. Using a double stranded RNA-mediated interference method, we generated evidence that expression of DmDREF could be auto-regulated by DREF through the third DRE located at +211 to +218. In living flies we obtained results consistent with those obtained in vitro and in cultured cells. The study thus indicates that DmDREF is effectively regulated via highly conserved regions between the DmDREF and DvDREF promoters, suggesting the existence of common regulatory factors, and that DmDREF can be positively regulated by itself via the third DRE located in its most highly conserved region.\n",
      "A critical step in understanding the mechanisms of development is in defining the steps at the molecular, cellular, and organismal levels in the developmental program for a given organism-so that given the egg one can predict not only how the embryo will develop but also how that embryo evolved from its ancestors. Using methods employed by chemists and engineers in modeling hierarchical systems, I have integrated current theory and experiment into a calculational method that can model early Drosophila embryogenesis on a personal computer. This quantitative calculation tool is simple enough to be useful for experimentalists in designing experiments yet detailed enough for theoreticians to derive new insights on the evolution of developmental genetic networks. By integrating the strengths of theoretical and experimental methods into a single engineering model that can compute the cascade of genetic networks in a real organism, I provide a new calculational tool that can apply current theory to current experimental data to study the evolution of developmental programs.\n",
      "A gradient of the maternal morphogen dorsal (dl) establishes dorsal-ventral (D-V) polarity in the early Drosophila embryo. The dl concentration gradient is initiated by regulated nuclear transport, and only protein that enters nuclei is active in the D-V patterning process. Here we show that dl is a DNA-binding protein that specifically interacts with distal sequences of the zerknüllt (zen) promoter, one of the genetic targets of the morphogen. These zen sequences have the properties of a silencer element and can act over long distances to repress the expression of a heterologous promoter. The dl protein recognizes a sequence motif similar to that of the mammalian transcriptional activator NF-kappa B, which was shown to contain extensive homology with dl and the oncoprotein rel. We present evidence that the DNA-binding activity of the dl protein is mediated by the region of homology (the rel domain) conserved in the rel and NF-kappa B proteins.\n",
      "A gradient of the maternal morphogen dorsal (dl) initiates the differentiation of various tissues along the dorsal-ventral axis of early Drosophila embryos. dl is a sequence-specific DNA-binding protein that is related to the mammalian regulatory factor NF-kappa B. Previous studies suggest that dl can function as a transcriptional repressor. To determine how dl functions as an activator we have examined the promoter of the mesoderm determinant gene twist (twi). Genetic studies suggest that peak levels of dl protein in ventral regions of early embryos initiate twi expression. Using a combination of promoter fusion-P-transformation assays, and in vitro DNA-binding assays coupled with site-directed mutagenesis, we establish a direct link between dl-binding sites and twi expression in the early embryo. We also present evidence that the dorsal-ventral limits of twi expression depend on the number and affinity of dl-binding sites present in its promoter. A comparison of twi with a second dl target gene, zen, suggests a correlation between the affinities of dl-binding sites and response to different thresholds of dl morphogen.\n",
      "A member of a small group of genes in Drosophila that define the segmentation pattern of the early embryo even-skipped (eve), which plays a key role in a network of interactions among segmentation genes. It appears to control morphogenesis by regulating the expression of the segmentation gene engrailed (en), and by autoregulating its own expression (M. Frasch and M.L., in preparation). Here we show that these regulatory interactions could occur at the level of transcription as a full-length eve protein binds with high affinity to specific sequences located near the 5' ends of the eve and en genes. The en binding sites contain at least one copy of a 10-base pair consensus sequence: T-C-A-A-T-T-A-A-A-T. In contrast, the 5' eve binding sites are relatively G-C rich and do not share obvious similarities with the 10-base pair consensus sequence associated with en. Other homeo box proteins can recognize both classes of eve binding sites, lending support to the proposal that regulatory interactions among homeo box genes involve a competition of different homeo box proteins for similar cis regulatory sequences.\n",
      "A morphogen gradient is an important concept in developmental biology, because it describes a mechanism by which the emission of a signal from one part of an embryo can determine the location, differentiation and fate of many surrounding cells. The value of this idea has been clear for over half a century, but only recently have experimental systems and methods of analysis progressed to the point where we begin to understand how a cell can sense and respond to tiny changes in minute concentrations of extracellular signalling factors.\n",
      "A recent study reveals specific functions of Hox3/zerknüllt (zen) in the extraembryonic and embryonic primordia of the red flour beetle, Tribolium castaneum. The results shed light on the functional evolution of bicoid, a zen paralogue of higher flies, which determines anterior body parts.\n",
      "Although all bilaterian animals have a related set of Hox genes, the genomic organization of this gene complement comes in different flavors. In some unrelated species, Hox genes are clustered; in others, they are not. This indicates that the bilaterian ancestor had a clustered Hox gene family and that, subsequently, this genomic organization was either maintained or lost. Remarkably, the tightest organization is found in vertebrates, raising the embarrassingly finalistic possibility that vertebrates have maintained best this ancestral configuration. Alternatively, could they have co-evolved with an increased ;organization' of the Hox clusters, possibly linked to their genomic amplification, which would be at odds with our current perception of evolutionary mechanisms? When discussing the why's and how's of Hox gene clustering, we need to account for three points: the mechanisms of cluster evolution; the underlying biological constraints; and the developmental modes of the animals under consideration. By integrating these parameters, general conclusions emerge that can help solve the aforementioned dilemma.\n",
      "Although genes involved in common developmental programs are usually scattered throughout the metazoan genome, there are some important examples of functionally interconnected regulatory genes that display close physical linkage. In particular the homeotic genes, which determine the identities of body parts, are clustered in the Hox complexes and clustering is thought to be crucial for the proper execution of their developmental programs. Here we describe the organization and functional properties of a more recently identified cluster of six homeobox genes at 93DE on the third chromosome of Drosophila. These genes, which include tinman, bagpipe, ladybird early, ladybird late, C15, and slouch, all participate in mesodermal patterning and differentiation programs and show multiple regulatory interactions among each other. We propose that their clustering, through unknown mechanisms, is functionally significant and discuss the similarities and differences between the 93DE homeobox gene cluster and the Hox complexes.\n",
      "Although the genetics of dorsal-ventral polarity which leads to mesoderm formation in Drosophila are understood in considerable detail, subsequent molecular mechanisms involved in patterning the mesoderm primordium into individual mesodermal subtypes are poorly understood. Two papers published recently suggest strongly that an inductive signal from dorsal ectoderm is involved in subdividing the underlying mesoderm, and present evidence that one of the signalling factors is Decapentaplegic (Dpp), a member of the bone morphogenetic protein subgroup of the Transforming Growth Factor-beta (TGF-beta) super family of proteins.\n",
      "Although the mechanisms of anterior-posterior axis formation are well understood in Drosophila, both embryological and molecular studies suggest significant variation in the mechanisms generating this axis within the Insecta class as a whole.\n",
      "Amino acid encoding genes contain character state information that may be useful for phylogenetic analysis on at least two levels. The nucleotide sequence and the translated amino acid sequences have both been employed separately as character states for cladistic studies of various taxa, including studies of the genealogy of genes in multigene families. In essence, amino acid sequences and nucleic acid sequences are two different ways of character coding the information in a gene. Silent positions in the nucleotide sequence (first or third positions in codons that can accrue change without changing the identity of the amino acid that the triplet codes for) may accrue change relatively rapidly and become saturated, losing the pattern of historical divergence. On the other hand, non-silent nucleotide alterations and their accompanying amino acid changes may evolve too slowly to reveal relationships among closely related taxa. In general, the dynamics of sequence change in silent and non-silent positions in protein coding genes result in homoplasy and lack of resolution, respectively. We suggest that the combination of nucleic acid and the translated amino acid coded character states into the same data matrix for phylogenetic analysis addresses some of the problems caused by the rapid change of silent nucleotide positions and overall slow rate of change of non-silent nucleotide positions and slowly changing amino acid positions. One major theoretical problem with this approach is the apparent non-independence of the two sources of characters. However, there are at least three possible outcomes when comparing protein coding nucleic acid sequences with their translated amino acids in a phylogenetic context on a codon by codon basis. First, the two character sets for a codon may be entirely congruent with respect to the information they convey about the relationships of a certain set of taxa. Second, one character set may display no information concerning a phylogenetic hypothesis while the other character set may impact information to a hypothesis. These two possibilities are cases of non-independence, however, we argue that congruence in such cases can be thought of as increasing the weight of the particular phylogenetic hypothesis that is supported by those characters. In the third case, the two sources of character information for a particular codon may be entirely incongruent with respect to phylogenetic hypotheses concerning the taxa examined. In this last case the two character sets are independent in that information from neither can predict the character states of the other. Examples of these possibilities are discussed and the general applicability of combining these two sources of information for protein coding genes is presented using sequences from the homeobox region of 46 homeobox genes from Drosophila melanogaster to develop a hypothesis of genealogical relationship of these genes in this large multigene family.\n",
      "Among the maternally active genes of Drosophila, cactus is the only one whose loss of function mutations specifically produce ventralized embryos. Its product inhibits nuclear translocation of the dorsal morphogen in the dorsal region of the embryo. Here we report the cloning of cactus and the sequencing of its maternal transcript. The identity of our clones was verified by induction of phenocopies with antisense RNA and rescue of the mutant phenotype with sense RNA. cactus is predicted to encode an acidic, cytoplasmic protein with seven ankyrin repeats. The sequence has similarity to the I kappa B proteins that inhibit the vertebrate transcription factor NF-kappa B. In analogy to results obtained with I kappa B and NF-kappa B, bacterially expressed cactus protein can inhibit DNA binding of dorsal protein in vitro.\n",
      "An extracellular serine protease cascade generates the ligand that activates the Toll signaling pathway to establish dorsoventral polarity in the Drosophila embryo. We show here that this cascade is regulated by a serpin-type serine protease inhibitor, which plays an essential role in confining Toll signaling to the ventral side of the embryo. This role is strikingly analogous to the function of the mammalian serpin antithrombin in localizing the blood-clotting cascade, suggesting that serpin inhibition of protease activity may be a general mechanism for achieving spatial control in diverse biological processes.\n",
      "An old idea about the relationship between arthropod and vertebrate body plans has been given new life by studies of the signalling genes controlling dorsal and ventral development in Drosophila and Xenopus.\n",
      "An oligonucleotide based on the cdc 10/SWI6 repeats of the Drosophila Notch protein was used to isolate other Drosophila genes with these repeats. One of these genes is the cactus locus, 1 of 11 genes required maternally for the establishment in embryos of a gradient of nuclear localization of dorsal protein, a rel-like transcription factor. Previous work showed that in cactus mutants more dorsal protein enters the nucleus in dorsal regions, resulting in a ventralized phenotype. It is now shown that the cactus locus produces two proteins that differ at their carboxy termini; both contain six cdc 10/SWI6 repeats that are sufficient for binding to dorsal and for inhibiting the ability of dorsal to bind DNA. The site on dorsal to which cactus binds was localized to the rel homology domain, where it overlaps with, or is adjacent to, the nuclear localization signal. In vivo the bulk of the cactus protein associated with dorsal is phosphorylated. This, or the association with dorsal, appears to stabilize the maternal cactus protein.\n",
      "At least 13 genes control the establishment of dorsoventral polarity in the Drosophila embryo and more than 30 genes control the anteroposterior pattern of body segments. Each group of genes is thought to control pattern formation along one body axis, independently of the other group. We have used the expression of the fushi tarazu (ftz) segmentation gene as a positional marker to investigate the relationship between the dorsoventral and anteroposterior axes. The ftz gene is normally expressed in seven transverse stripes. Changes in the striped pattern in embryos mutant for other genes (or progeny of females homozygous for maternal-effect mutations) can reveal alterations of cell fate resulting from such mutations. We show that in the absence of any of ten maternal-effect dorsoventral polarity gene functions, the characteristic stripes of ftz protein are altered. Normally there is a difference between ftz stripe spacing on the dorsal and ventral sides of the embryo; in dorsalized mutant embryos the ftz stripes appear to be altered so that dorsal-type spacing occurs on all sides of the embryo. These results indicate that cells respond to dorsoventral positional information in establishing early patterns of gene expression along the anteroposterior axis and that there may be more significant interactions between the different axes of positional information than previously determined.\n",
      "Axis formation is a key step in development, but studies indicate that genes involved in insect axis formation are relatively fast evolving. Orthodenticle genes have conserved roles, often with hunchback, in maternal anterior patterning in several insect species. We show that two orthodenticle genes, otd1 and otd2, and hunchback act as maternal anterior patterning genes in the honeybee (Apis mellifera) but, unlike other insects, act to pattern the majority of the anteroposterior axis. These genes regulate the expression domains of anterior, central and posterior gap genes and may directly regulate the anterior gap gene giant. We show otd1 and hunchback also influence dorsoventral patterning by regulating zerknült (zen) as they do in Tribolium, but that zen does not regulate the expression of honeybee gap genes. This suggests that interactions between anteroposterior and dorsal-ventral patterning are ancestral in holometabolous insects. Honeybee axis formation, and the function of the conserved anterior patterning gene orthodenticle, displays unique characters that indicate that, even when conserved genes pattern the axis, their regulatory interactions differ within orders of insects, consistent with relatively fast evolution in axis formation pathways.\n",
      "Bioinformatics methods have identified enhancers that mediate restricted expression in the Drosophila embryo. However, only a small fraction of the predicted enhancers actually work when tested in vivo. In the present study, co-regulated neurogenic enhancers that are activated by intermediate levels of the Dorsal regulatory gradient are shown to contain several shared sequence motifs. These motifs permitted the identification of new neurogenic enhancers with high precision: five out of seven predicted enhancers direct restricted expression within ventral regions of the neurogenic ectoderm. Mutations in some of the shared motifs disrupt enhancer function, and evidence is presented that the Twist and Su(H) regulatory proteins are essential for the specification of the ventral neurogenic ectoderm prior to gastrulation. The regulatory model of neurogenic gene expression defined in this study permitted the identification of a neurogenic enhancer in the distant Anopheles genome. We discuss the prospects for deciphering regulatory codes that link primary DNA sequence information with predicted patterns of gene expression.\n",
      "Bone morphogenetic protein (BMP) signalling regulates embryonic dorsal-ventral cell fate decisions in flies, frogs and fish. BMP activity is controlled by several secreted factors including the antagonists chordin and short gastrulation (SOG). Here we show that a second secreted protein, Twisted gastrulation (Tsg), enhances the antagonistic activity of Sog/chordin. In Drosophila, visualization of BMP signalling using anti-phospho-Smad staining shows that the tsg and sog loss-of-function phenotypes are very similar. In S2 cells and imaginal discs, TSG and SOG together make a more effective inhibitor of BMP signalling than either of them alone. Blocking Tsg function in zebrafish with morpholino oligonucleotides causes ventralization similar to that produced by chordin mutants. Co-injection of sub-inhibitory levels of morpholines directed against both Tsg and chordin synergistically enhances the penetrance of the ventralized phenotype. We show that Tsgs from different species are functionally equivalent, and conclude that Tsg is a conserved protein that functions with SOG/chordin to antagonize BMP signalling.\n",
      "Bone morphogenetic proteins (BMP) direct dorsal-ventral patterning in both invertebrate and vertebrate embryos, with strong evolutionary conservation of molecular components of the pathway. Dorsal-ventral patterning of the early Drosophila embryo is a powerful experimental system to probe mechanisms of BMP gradient formation and interpretation. Recent studies have found that spatial patterns of activated BMP signal transducers in Drosophila go through an unexpected transition: a shallow gradient of weak responses at mid-cellularization changes to a step gradient of stronger responses in cellularized embryos. The transition between two gradients of different shape yields new insights into the progression of Drosophila dorsal-ventral patterning and raises new issues about the mechanisms ofgradient formation.\n",
      "Brinker (Brk), a novel protein with features of a transcriptional repressor, regulates the graded response to Decapentaplegic (Dpp) in appendage primordia of Drosophila. Here, we show that in the embryo brk also has differential effects on Dpp target genes, depending on the level of Dpp activity required for their activation. Low-level target genes, like dpp itself, tolloid and early zerknüllt, show strong ectopic expression in ventrolateral regions of brk mutant embryos; intermediate-level target genes like pannier show weak ectopic expression, while high-level target genes like u-shaped and rhomboid are not affected. Ectopic target gene activation in the absence of brk is independent of Dpp, Tkv and Medea, indicating that Dpp signaling normally antagonizes brk's repression of these target genes. brk is expressed like short gastrulation (sog) in ventrolateral regions of the embryo abutting the dpp domain. Here, both brk and sog antagonize the antineurogenic activity of Dpp so that only in brk sog double mutants is the neuroectoderm completely deleted.\n",
      "By using a hsp70-Ubx fusion gene, we have ectopically expressed a Ubx product in the embryonic head primordia and studied the developmental effects on the larval head. We find that after high and persistent levels of Ubx product, the head is replaced by three (C1, C2 and C3) abdominal-like denticle belts. The C2 and C3 belts are the homeotic transformations of parasegments 1 and 2, respectively, while the C1 belt probably derives from the transformation and subsequent fusion of the most anterior procephalic primordia. On the basis of their response to the Ubx product and other arguments, we propose that the larval head is made of two genetically distinct components; one is the procephalon and the anterior region of the mandibular lobe, and the other is part of the parasegmental trunk and includes parasegments 1 and 2. Our results also indicate that most or all the larval head structures derive from precursor cells of ventral origin.\n",
      "CBP and the related p300 protein are widely used transcriptional co-activators in metazoans that interact with multiple transcription factors. Whether CBP/p300 occupies the genome equally with all factors or preferentially binds together with some factors is not known. We therefore compared Drosophila melanogaster CBP (nejire) ChIP-seq peaks with regions bound by 40 different transcription factors in early embryos, and we found high co-occupancy with the Rel-family protein Dorsal. Dorsal is required for CBP occupancy in the embryo, but only at regions where few other factors are present. CBP peaks in mutant embryos lacking nuclear Dorsal are best correlated with TGF-ß/Dpp-signaling and Smad-protein binding. Differences in CBP occupancy in mutant embryos reflect gene expression changes genome-wide, but CBP also occupies some non-expressed genes. The presence of CBP at silent genes does not result in histone acetylation. We find that Polycomb-repressed H3K27me3 chromatin does not preclude CBP binding, but restricts histone acetylation at CBP-bound genomic sites. We conclude that CBP occupancy in Drosophila embryos preferentially overlaps factors controlling dorso-ventral patterning and that CBP binds silent genes without causing histone hyperacetylation.\n",
      "CDP/Cux/Cut proteins are an evolutionarily conserved family of proteins containing several DNA binding domains: one Cut homeodomain and one, two or three Cut repeats. In Drosophila melanogaster, genetic studies indicated that Cut functions as a determinant of cell-type specification in several tissues, notably in the peripheral nervous system, the wing margin and the Malpighian tubule. Moreover, Cut was found to be a target and an effector of the Notch signaling pathway. In vertebrates, the same functions appear to be fulfilled by two cut-related genes with distinct patterns of expression. Cloning of the cDNA for the CCAAT-displacement protein (CDP) revealed that it was the human homologue of Drosophila Cut. CDP was later found be the DNA binding protein of the previously characterized histone nuclear factor D (HiNF-D). CDP and its mouse counterpart, Cux, were also reported to interact with regulatory elements from a large number of genes, including matrix attachment regions (MARs). CDP/Cut proteins were found generally to function as transcriptional repressors, although a participation in transcriptional activation is suggested by some data. Repression by CDP/Cut involves competition for binding site occupancy and active repression via the recruitment of a histone deacetylase activity. Various combinations of Cut repeats and the Cut homeodomains can generate distinct DNA binding activities. These activities are elevated in proliferating cells and decrease during terminal differentiation. One activity, involving the Cut homeodomain, is upregulated in S phase. CDP/Cut function is regulated by several post-translational modification events including phosphorylation, dephosphorylation, and acetylation. The CUTL1 gene in human was mapped to 7q22, a chromosomal region that is frequently rearranged in various cancers.\n",
      "Cascades of zygotic gene expression pattern the anterior-posterior (AP) and dorsal-ventral (DV) axes of the early Drosophila embryo. Here, we used the global run-on sequencing assay (GRO-seq) to map the genome-wide RNA polymerase distribution during early Drosophila embryogenesis, thus providing insights into how genes are regulated. We identify widespread promoter-proximal pausing yet show that the presence of paused polymerase does not necessarily equate to direct regulation through pause release to productive elongation. Our data reveal that a subset of early Zelda-activated genes is regulated at the level of polymerase recruitment, whereas other Zelda target and axis patterning genes are predominantly regulated through pause release. In contrast to other signaling pathways, we found that bone morphogenetic protein (BMP) target genes are collectively more highly paused than BMP pathway components and show that BMP target gene expression requires the pause-inducing negative elongation factor (NELF) complex. Our data also suggest that polymerase pausing allows plasticity in gene activation throughout embryogenesis, as transiently repressed and transcriptionally silenced genes maintain and lose promoter polymerases, respectively. Finally, we provide evidence that the major effect of pausing is on the levels, rather than timing, of transcription. These data are discussed in terms of the efficiency of transcriptional activation required across cell populations during developmental time constraints.\n",
      "Cell fates in the anterior and posterior termini of the Drosophila embryo are programmed by multiple zygotic genes that are regulated in response to a maternally encoded signal transduction pathway. These genes specify terminal as distinct from central cell fates, program pattern along the anteroposterior and dorsoventral axes of the termini, and also control endoderm specification and terminal morphogenetic movements. Here, we use a genetic interaction test to dissect the zygotic components of the terminal genetic hierarchy. We show that two genes, lines and empty spiracles, act downstream of tailless to repress central and promote terminal cell fates along the anteroposterior axis of the termini. Genes that control dorsoventral pattern in the termini and genes that program terminal morphogenesis act in distinct branches of the genetic hierarchy that are independent of tailless.\n",
      "Charles Darwin has proposed the theory that evolution of live organisms is based on random variation and natural selection. Jacques Monod in his classic book Chance and Necessity, published 40 years ago, presented his thesis \"that the biosphere does not contain a predictable class of objects or events, but constitutes a particular occurrence, compatible indeed with the first principles, but not deducible from those principals and therefore, essentially unpredictable.\"Recent discoveries in eye evolution are in agreement with both of these theses. They confirm Darwin's assumption of a simple eye prototype and lend strong support for the notion of a monophyletic origin of the various eye types. Considering the complexity of the underlying gene regulatory networks the unpredictability is obvious. The evolution of the Hox gene cluster and the specification of the body plan starting from an evolutionary prototype segment is discussed. In the course of evolution, a series of similar prototypic segments gradually undergoes cephalization anteriorly and caudalization posteriorly through diversification of the Hox genes.\n",
      "Chromatin domain insulators are thought to insulate adjacent genes, including their regulatory elements, from each other by organizing chromatin into functionally independent domains. Thus insulators should play a global role in gene regulation by keeping regulatory domains separated. However, this has never been demonstrated. We previously designed and characterized a transgene that is under GAL4 UAS control and encodes a dominant-negative form of the Boundary Element-Associated Factors BEAF-32A and BEAF-32B. The BID transgene encodes the BEAF self-interaction domain but lacks a DNA binding domain. Expression of BID in eye imaginal discs leads to a rough eye phenotype. Here we screen for dominant mutations that modify this eye phenotype. This assay provides evidence for cross-talk between different classes of insulators, and for a broad role of the BEAF proteins in maintaining patterns of gene expression during eye development. Most identified genes encode other insulator binding proteins, transcription factors involved in head development, or general transcription factors. Because it is unlikely that insulator function is limited to eye development, the present results support the hypothesis that insulators play a widespread role in maintaining global transcription programs.\n",
      "Circulating carbohydrates are an essential energy source, perturbations in which are pathognomonic of various diseases, diabetes being the most prevalent. Yet many of the genes underlying diabetes and its characteristic hyperglycaemia remain elusive. Here we use physiological and genetic interrogations in D. melanogaster to uncover the 'glucome', the complete set of genes involved in glucose regulation in flies. Partial genomic screens of ∼1,000 genes yield ∼160 hyperglycaemia 'flyabetes' candidates that we classify using fat body- and muscle-specific knockdown and biochemical assays. The results highlight the minor glucose fraction as a physiological indicator of metabolism in Drosophila. The hits uncovered in our screen may have conserved functions in mammalian glucose homeostasis, as heterozygous and homozygous mutants of Ck1alpha in the murine adipose lineage, develop diabetes. Our findings demonstrate that glucose has a role in fly biology and that genetic screenings carried out in flies may increase our understanding of mammalian pathophysiology.\n",
      "Cis-regulatory DNAs control the timing and sites of gene expression during metazoan development. Changes in gene expression are responsible for the morphological diversification of metazoan body plans. However, traditional methods for the identification and characterization of cis-regulatory DNAs are tedious. During the past year, computational methods have been used to identify novel cis-DNAs within the entire Drosophila genome. These methods change the way that cis-DNAs will be analyzed in future studies and offer the promise of unraveling complex gene networks.\n",
      "Cis-regulatory modules (CRMs) function by binding sequence specific transcription factors, but the relationship between in vivo physical binding and the regulatory capacity of factor-bound DNA elements remains uncertain. We investigate this relationship for the well-studied Twist factor in Drosophila melanogaster embryos by analyzing genome-wide factor occupancy and testing the functional significance of Twist occupied regions and motifs within regions. Twist ChIP-seq data efficiently identified previously studied Twist-dependent CRMs and robustly predicted new CRM activity in transgenesis, with newly identified Twist-occupied regions supporting diverse spatiotemporal patterns (>74% positive, n = 31). Some, but not all, candidate CRMs require Twist for proper expression in the embryo. The Twist motifs most favored in genome ChIP data (in vivo) differed from those most favored by Systematic Evolution of Ligands by EXponential enrichment (SELEX) (in vitro). Furthermore, the majority of ChIP-seq signals could be parsimoniously explained by a CABVTG motif located within 50 bp of the ChIP summit and, of these, CACATG was most prevalent. Mutagenesis experiments demonstrated that different Twist E-box motif types are not fully interchangeable, suggesting that the ChIP-derived consensus (CABVTG) includes sites having distinct regulatory outputs. Further analysis of position, frequency of occurrence, and sequence conservation revealed significant enrichment and conservation of CABVTG E-box motifs near Twist ChIP-seq signal summits, preferential conservation of ±150 bp surrounding Twist occupied summits, and enrichment of GA- and CA-repeat sequences near Twist occupied summits. Our results show that high resolution in vivo occupancy data can be used to drive efficient discovery and dissection of global and local cis-regulatory logic.\n",
      "Cis-regulatory modules that drive precise spatial-temporal patterns of gene expression are central to the process of metazoan development. We describea new computational strategy to annotate genomic sequences based on their \"pattern generating potential\"and to produce quantitative descriptions of transcriptional regulatory networks at the level of individual protein-module interactions. We use this approach to convert the qualitative understanding of interactions that regulate Drosophila segmentation into a network model in which a confidence value is associated with each transcription factor-module interaction. Sequence information from multiple Drosophila species is integrated with transcription factor binding specificities to determine conserved binding site frequencies across the genome. These binding site profiles are combined with transcription factor expression information to create a model to predict module activity patterns. This model is used to scan genomic sequences for the potential to generate all or part of the expression pattern of a nearby gene, obtained from available gene expression databases. Interactions between individual transcription factors and modules are inferred by a statistical method to quantify a factor's contribution to the module's pattern generating potential. We use these pattern generating potentials to systematically describe the location and function of known and novel cis-regulatory modules in the segmentation network, identifying many examples of modules predicted to have overlapping expression activities. Surprisingly, conserved transcription factor binding site frequencies were as effective as experimental measurements of occupancy in predicting module expression patterns or factor-module interactions. Thus, unlike previous module prediction methods, this method predicts not only the location of modules but also their spatial activity pattern and the factors that directly determine this pattern. As databases of transcription factor specificities and in vivo gene expression patterns grow, analysis of pattern generating potentials provides a general method to decode transcriptional regulatory sequences and networks.\n",
      "Codon usage bias, the preferential use of particular codons within each codon family, is characteristic of synonymous base composition in many species, including Drosophila, yeast, and many bacteria. Preferential usage of particular codons in these species is maintained by natural selection acting largely at the level of translation. In Drosophila, as in bacteria, the rate of synonymous substitution per site is negatively correlated with the degree of codon usage bias, indicating stronger selection on codon usage in genes with high codon bias than in genes with low codon bias. Surprisingly, in these organisms, as well as in mammals, the rate of synonymous substitution is also positively correlated with the rate of nonsynonymous substitution. To investigate this correlation, we carried out a phylogenetic analysis of substitutions in 22 genes between two species of Drosophila, Drosophila pseudoobscura and D. subobscura, in codons that differ by one replacement and one synonymous change. We provide evidence for a relative excess of double substitutions in the same species lineage that cannot be explained by the simultaneous mutation of two adjacent bases. The synonymous changes in these codons also cannot be explained by a shift to a more preferred codon following a replacement substitution. We, therefore, interpret the excess of double codon substitutions within a lineage as being the result of relaxed constraints on both kinds of substitutions in particular codons.\n",
      "Comparative genomic approaches to gene and cis-regulatory prediction are based on the principle that differential DNA sequence conservation reflects variation in functional constraint. Using this principle, we analyze noncoding sequence conservation in Drosophila for 40 loci with known or suspected cis-regulatory function encompassing >100 kb of DNA. We estimate the fraction of noncoding DNA conserved in both intergenic and intronic regions and describe the length distribution of ungapped conserved noncoding blocks. On average, 22%-26% of noncoding sequences surveyed are conserved in Drosophila, with median block length approximately 19 bp. We show that point substitution in conserved noncoding blocks exhibits transition bias as well as lineage effects in base composition, and occurs more than an order of magnitude more frequently than insertion/deletion (indel) substitution. Overall, patterns of noncoding DNA structure and evolution differ remarkably little between intergenic and intronic conserved blocks, suggesting that the effects of transcription per se contribute minimally to the constraints operating on these sequences. The results of this study have implications for the development of alignment and prediction algorithms specific to noncoding DNA, as well as for models of cis-regulatory DNA sequence evolution.\n",
      "Comparisons between Hox genes in different arthropods suggest that the diversity of Antennapedia-class homeotic genes present in modern insects had already arisen before the divergence of insects and crustaceans, probably during the Cambrian. Hox gene duplications are therefore unlikely to have occurred concomitantly with trunk segment diversification in the lineage leading to insects. Available data suggest that domains of homeotic gene expression are also generally conserved among insects, but changes in Hox gene regulation may have played a significant role in segment diversification. Differences that have been documented alter specific aspects of Hox gene regulation within segments and correlate with alterations in segment morphology rather than overt homeotic transformations. The Drosophila Hox cluster contains several homeobox genes that are not homeotic genes--bicoid, fushi-tarazu and zen. the role of these genes during early development has been studied in some detail. It appears to be without parallel among the vertebrate Hox genes. No well conserved homologues of these genes have been found in other taxa, suggesting that they are evolving faster than the homeotic genes. Relatively divergent Antp-class genes isolated from other insects are probably homologues of fushi-tarazu, but these are almost unrecognisable outside of their homeodomains, and have accumulated approximately 10 times as many changes in their homeodomains as have homeotic genes in the same comparisons. They show conserved patterns of expression in the nervous system, but not during early development.\n",
      "Computational methods attempting to identify instances of cis-regulatory modules (CRMs) in the genome face a challenging problem of searching for potentially interacting transcription factor binding sites while knowledge of the specific interactions involved remains limited. Without a comprehensive comparison of their performance, the reliability and accuracy of these tools remains unclear. Faced with a large number of different tools that address this problem, we summarized and categorized them based on search strategy and input data requirements. Twelve representative methods were chosen and applied to predict CRMs from the Drosophila CRM database REDfly, and across the human ENCODE regions. Our results show that the optimal choice of method varies depending on species and composition of the sequences in question. When discriminating CRMs from non-coding regions, those methods considering evolutionary conservation have a stronger predictive power than methods designed to be run on a single genome. Different CRM representations and search strategies rely on different CRM properties, and different methods can complement one another. For example, some favour homotypical clusters of binding sites, while others perform best on short CRMs. Furthermore, most methods appear to be sensitive to the composition and structure of the genome to which they are applied. We analyze the principal features that distinguish the methods that performed well, identify weaknesses leading to poor performance, and provide a guide for users. We also propose key considerations for the development and evaluation of future CRM-prediction methods.\n",
      "Current models describe male-specific fruitless (fruM) as a genetic 'switch' regulating sexual behavior in Drosophila melanogaster, and they postulate that female (F) and male (M) doublesex (dsx) products control body sexual morphology. In contradiction to this simple model, we show that dsx, as well as fruM and non-sex-specific retained (retn), affect both male and female sexual behaviors. In females, both retn and dsxF contribute to female receptivity, and both genes act to repress male-like courtship activity in the presence or absence of fruM. In males, consistent with the opposing functions of dsxM and dsxF, dsxM acts as a positive factor for male courtship. retn also acts counter to fruM in the development of the male-specific muscle of Lawrence. Molecularly, retn seems to regulate sexual behavior via a previously described complex that represses zerknullt. Thus, we show that fru and dsx together act as a 'switch' system regulating behavior in the context of other developmental genes, such as retn.\n",
      "DREF, a transcription regulatory factor which specifically binds to the promoter-activating element DRE (DNA replication-related element) of DNA replication-related genes, was purified to homogeneity from nuclear extracts of Drosophila Kc cells. cDNA for DREF was isolated with the reverse-transcriptase polymerase chain reaction method using primers synthesized on the basis of partial amino acid sequences and following screening of cDNA libraries. Deduced from the nucleotide sequences of cDNA, DREF is a polypeptide of 701 amino acid residues with a molecular weight of 80,096, which contains three characteristic regions, rich in basic amino acids, proline, and acidic amino acids, respectively. Deletion analysis of bacterially expressed DREF fused with glutathione S-transferase (GST-DREF) indicated that a part of the N-terminal basic amino acid region (16-115 amino acids) is responsible for the specific binding to DRE. A polyclonal and four monoclonal antibodies were raised against the GST-DREF fusion protein. The antibodies inhibited specifically the transcription of DNA polymerase alpha promoter in vitro. Cotransfection experiments using Kc cells demonstrated that overproduction of DREF protein overcomes the repression of the proliferating cell nuclear antigen gene promoter by the zerknüllt gene product. These results confirmed that DREF is a trans-activating factor for DNA replication-related genes. Immunocytochemical analysis demonstrated the presence of DREF polypeptide in nuclei after the eighth nuclear division cycle, suggesting that nuclear accumulation of DREF is important for the coordinate zygotic expression of DNA replication-related genes carrying DRE sequences.\n",
      "Decapentaplegic (Dpp), the fly homolog of the secreted mammalian BMP2/4 signaling molecules, is involved in almost all aspects of fly development. Dpp has critical functions at all developmental stages, from patterning of the eggshell to the determination of adult intestinal stem cell identity. Here, we focus on recent findings regarding the transcriptional regulatory logic of the pathway, on a new feedback regulator, Pentagone, and on Dpp's roles in scaling and growth of the Drosophila wing.\n",
      "Developing tissues are patterned by coordinated activities of signaling systems, which can be integrated by a regulatory region of a gene that binds multiple transcription factors or by a transcription factor that is modified by multiple enzymes. Based on a combination of genetic and imaging experiments in the early Drosophila embryo, we describe a signal integration mechanism that cannot be reduced to a single gene regulatory element or a single transcription factor. This mechanism relies on an enzymatic network formed by mitogen-activated protein kinase (MAPK) and its substrates. Specifically, anteriorly localized MAPK substrates, such as Bicoid, antagonize MAPK-dependent downregulation of Capicua, a repressor that is involved in gene regulation along the dorsoventral axis of the embryo. MAPK substrate competition provides a basis for ternary interaction of the anterior, dorsoventral, and terminal patterning systems. A mathematical model of this interaction can explain gene expression patterns with both anteroposterior and dorsoventral polarities.\n",
      "Differentiation of distinct cell types at specific locations within a developing organism depends largely on the ability of cells to communicate. A major class of signalling proteins implicated in cell to cell communication is represented by members of the TGF beta superfamily. A corresponding class of transmembrane serine/threonine kinases has recently been discovered that act as cell surface receptors for ligands of the TGF beta superfamily. The product of the Drosophila gene decapentaplegic (dpp) encodes a TGF beta homolog that plays multiple roles during embryogenesis and the development of imaginal discs. Here we describe the complex expression pattern of thick veins (tkv), which encodes a receptor for dpp. We make use of tkv loss-of-function mutations to examine the consequences of the failure of embryonic cells to respond to dpp and/or other TGF beta homologs. We find that while maternal tkv product allows largely normal dorsoventral pattering of the embryo, zygotic tkv activity is indispensable for dorsal closure of the embryo after germ band retraction. Furthermore, tkv activity is crucial for patterning the visceral mesoderm; in the absence of functional tkv gene product, visceral mesoderm parasegment 7 cells fail to express Ultrabithorax, but instead accumulate Antennapedia protein. The tkv receptor is therefore involved in delimiting the expression domains of homeotic genes in the visceral mesoderm. Interestingly, tkv mutants fail to establish a proper tracheal network. Tracheal braches formed by cells migrating in dorsal or ventral directions are absent in tkv mutants. The requirements for tkv in dorsal closure, visceral mesoderm and trachea development assign novel functions to dpp or a closely related member of the TGF beta superfamily.\n",
      "Differentiation of the embryonic termini in Drosophila depends on signaling by the Tor RTK, which induces terminal gene expression by inactivating at the embryonic poles a uniformly distributed repressor activity that involves the Gro corepressor. Here, we identify a new gene, cic, that acts as a repressor of terminal genes regulated by the Tor pathway. cic also mediates repression along the dorsoventral axis, a process that requires the Dorsal morphogen and Gro, and which is also inhibited by Tor signaling at the termini. cic encodes an HMG-box transcription factor that interacts with Gro in vitro. We present evidence that Tor signaling regulates terminal patterning by inactivating Cic at the embryo poles. cic has been evolutionarily conserved, suggesting that Cic-like proteins may act as repressors regulated by RTK signaling in other organisms.\n",
      "Divergent morphology of species has largely been ascribed to genetic differences in the tissue-specific expression of proteins, which could be achieved by divergence in cis-regulatory elements or by altering the binding specificity of transcription factors (TFs). The relative importance of the latter has been difficult to assess, as previous systematic analyses of TF binding specificity have been performed using different methods in different species. To address this, we determined the binding specificities of 242 Drosophila TFs, and compared them to human and mouse data. This analysis revealed that TF binding specificities are highly conserved between Drosophila and mammals, and that for orthologous TFs, the similarity extends even to the level of very subtle dinucleotide binding preferences. The few human TFs with divergent specificities function in cell types not found in fruit flies, suggesting that evolution of TF specificities contributes to emergence of novel types of differentiated cells.\n",
      "Diverse types of epithelial morphogenesis drive development. Similar cytoskeletal and cell adhesion machinery orchestrate these changes, but it is unclear how distinct tissue types are produced. Thus, it is important to define and compare different types of morphogenesis. We investigated cell flattening and elongation in the amnioserosa, a squamous epithelium formed at Drosophila gastrulation. Amnioserosa cells are initially columnar. Remarkably, they flatten and elongate autonomously by perpendicularly rotating the microtubule cytoskeleton--we call this 'rotary cell elongation'. Apical microtubule protrusion appears to initiate the rotation and microtubule inhibition perturbs the process. F-actin restrains and helps orient the microtubule protrusions. As amnioserosa cells elongate, they maintain their original cell-cell contacts and develop planar polarity. Myosin II localizes to anterior-posterior contacts, while the polarity protein Bazooka (PAR-3) localizes to dorsoventral contacts. Genetic analysis revealed that Myosin II and Bazooka cooperate to properly position adherens junctions. These results identify a specific cellular mechanism of squamous tissue morphogenesis and molecular interactions involved.\n",
      "Dorsal functions as both an activator and repressor of transcription to determine dorsoventral fate in the Drosophila melanogaster embryo. Repression by Dorsal requires the corepressor Groucho (Gro) and is mediated by silencers termed ventral repression regions (VRRs). A VRR in zerknüllt (zen) contains Dorsal binding sites as well as an essential element termed AT2. We have identified and purified an AT2 DNA binding activity in embryos and shown it to consist of cut (ct) and dead ringer (dri) gene products. Studies of loss-of-function mutations in ct and dri demonstrate that both genes are required for the activity of the AT2 site. Dorsal and Dri both bind Gro, acting cooperatively to recruit it to the DNA. Thus, ventral repression may require the formation of a multiprotein complex at the VRR. This complex includes Dorsal, Gro, and additional DNA binding proteins, which appear to convert Dorsal from an activator to a repressor by enabling it to recruit Gro to the template. By showing how binding site context can dramatically alter transcription factor function, these findings help clarify the mechanisms responsible for the regulatory specificity of transcription factors.\n",
      "Dorsal-ventral patterning in vertebrate and invertebrate embryos is mediated by a conserved system of secreted proteins that establishes a bone morphogenetic protein (BMP) gradient. Although the Drosophila embryonic Decapentaplegic (Dpp) gradient has served as a model to understand how morphogen gradients are established, no role for the extracellular matrix has been previously described. Here we show that type IV collagen extracellular matrix proteins bind Dpp and regulate its signalling in both the Drosophila embryo and ovary. We provide evidence that the interaction between Dpp and type IV collagen augments Dpp signalling in the embryo by promoting gradient formation, yet it restricts the signalling range in the ovary through sequestration of the Dpp ligand. Together, these results identify a critical function of type IV collagens in modulating Dpp in the extracellular space during Drosophila development. On the basis of our findings that human type IV collagen binds BMP4, we predict that this role of type IV collagens will be conserved.\n",
      "Dorsoventral (DV) patterning of the Drosophila embryo is initiated by a broad Dorsal (Dl) nuclear gradient, which is regulated by a conserved signaling pathway that includes the Toll receptor and Pelle kinase. We investigate the consequences of expressing a constitutively activated form of the Toll receptor, Toll(10b), in anterior regions of the early embryo using the bicoid 3' UTR. Localized Toll(10b) products result in the formation of an ectopic, anteroposterior (AP) Dl nuclear gradient along the length of the embryo. The analysis of both authentic dorsal target genes and defined synthetic promoters suggeststhat the ectopic gradient is sufficient to generate the full repertory of DV patterning responses along the AP axis of the embryo. For example, mesoderm determinants are activated in the anterior third of the embryo, whereas neurogenic genes are expressed in central regions. These results raise the possibility that Toll signaling components diffuse in the plasma membrane or syncytial cytoplasm of the early embryo. This study also provides evidence that neurogenic repressors may be important for the establishment of the sharp mesoderm/neuroectoderm boundary in the early embryo.\n",
      "Dorsoventral patterning in Drosophila is initiated by the maternal regulatory factor dorsal (dl), which is a member of the Rel family of transcription factors. dl functions as a transcriptional activator and repressor to establish different territories of gene expression in the precellular embryo. Differential regulation of dl target genes may be essential for subdividing each tissue territory (the presumptive mesoderm, neuroectoderm, and dorsal ectoderm) into multiple cell types in older embryos. Different patterns of snail (sna) and decapentaplegic (dpp) expression help define the limits of inductive interactions between the mesoderm and dorsal ectoderm after gastrulation. Similarly, the differential regulation of short gastrulation (sog) and dpp may be decisive in the initial subdivision of the dorsal ectoderm, whereas different limits of gene expression within the neuroectoderm might provide the basis for the subsequent subdivision of this tissue into ventral and lateral regions.\n",
      "Dorsoventral patterning in Drosophila requires the Dorsal morphogen to act as both an activator and a repressor of transcription: an HMG1-like protein may serve to switch Dorsal from one to the other.\n",
      "Dorsoventral patterning of the Drosophila embryo is regulated by graded distribution of bone morphogenetic proteins (BMPs) composed of two ligands, decapentaplegic (Dpp) a BMP2/4 ortholog and screw (Scw) a BMP5/6/7/8 family member. scw(E1) encodes an unusual allele that was isolated as a dominant enhancer of partial loss-of-function mutations in dpp. However, the molecular mechanisms that underlie this genetic interaction remain to be addressed. Here we show that scw(E1) contains a mutation at the furin cleavage site within the prodomain that is crucial for ligand production. Furthermore, our data show that Scw(E1) preferentially forms heterodimers with Dpp rather than homotypic dimers, providing a possible explanation for the dominant negative phenotype of scw(E1) alleles. The unprocessed prodomain of Scw(E1) remains in a complex with the Dpp:Scw heterodimer, and thus could interfere with interaction of the ligand with the extracellular matrix, or the kinetics of processing/secretion of the ligand in vivo. These data reveal novel mechanisms by which post-translational regulation of Scw can modulate Dpp signaling activity.\n",
      "Dorsoventral polarity of the Drosophila embryo requires maternal spätzle-Toll signaling to establish a nuclear gradient of Dorsal protein. The shape of this gradient is altered in embryos produced by females carrying dominant alleles of easter (ea(D)). The easter gene encodes a serine protease that generates processed Spätzle, which is proposed to act as the Toll ligand. By examining the expression domains of the zygotic genes zen, sog, rho and twist, which are targets of nuclear Dorsal, we show that the slope of the Dorsal gradient is progressively flattened in stronger ea(D) alleles. In the wild-type embryo, activated Easter is found in a high M(r) complex called Ea-X, which is hypothesized to contain a protease inhibitor. In ea(D) embryo extracts, we detect an Easter form corresponding to the free catalytic domain, which is never observed in wild type. These mutant ea(D) proteins retain protease activity, as determined by the production of processed Spätzle both in the embryo and in cultured Drosophila cells. These experiments suggest that the ea(D) mutations interfere with inactivation of catalytic Easter, and imply that this negative regulation is essential for generating the wild-type shape of the Dorsal gradient.\n",
      "Dpp signals are responsible for establishing a variety of cell identities in dorsal and lateral areas of the early Drosophila embryo, including the extra-embryonic amnioserosa as well as different ectodermal and mesodermal cell types. Although we have a reasonably clear picture of how Dpp signaling activity is modulated spatially and temporally during these processes, a better understanding of how these signals are executed requires the identification and characterization of a collection of downstream genes that uniquely respond to these signals. In the present study, we describe three novel genes, Dorsocross1, Dorsocross2 and Dorsocross3, which are expressed downstream of Dpp in the presumptive and definitive amnioserosa, dorsal ectoderm and dorsal mesoderm. We show that these genes are good candidates for being direct targets of the Dpp signaling cascade. Dorsocross expression in the dorsal ectoderm and mesoderm is metameric and requires a combination of Dpp and Wingless signals. In addition, a transverse stripe of expression in dorsoanterior areas of early embryos is independent of Dpp. The Dorsocross genes encode closely related proteins of the T-box domain family of transcription factors. All three genes are arranged in a gene cluster, are expressed in identical patterns in embryos, and appear to be genetically redundant. By generating mutants with a loss of all three Dorsocross genes, we demonstrate that Dorsocross gene activity is crucial for the completion of differentiation, cell proliferation arrest, and survival of amnioserosa cells. In addition, we show that the Dorsocross genes are required for normal patterning of the dorsolateral ectoderm and, in particular, the repression of wingless and the ladybird homeobox genes within this area of the germ band. These findings extend our knowledge of the regulatory pathways during amnioserosa development and the patterning of the dorsolateral embryonic germ band in response to Dpp signals.\n",
      "Dramatic successes in identifying vertebrate homeobox genes closely related to their insect relatives have led to the recognition of classes within the homeodomain superfamily. To what extent are the homeodomain protein classes dedicated to specific functions during development? Although information on vertebrate gene functions is limited, existing evidence from mice and nematodes clearly supports conservation of function for the Hox genes. Less compelling, but still remarkable, is the conservation of other homeobox gene classes and of regulators of homeotic gene expression and function. It is too soon to say whether the cases of conservation are unique and exceptional, or the beginning of a profoundly unified view of gene regulation in animal development. In any case, new questions are raised by the data: how can the differences between mammals and insects be compatible with conservation of homeobox gene function? Did the evolution of animal form involve a proliferation of new homeodomain proteins, new modes of regulation of existing gene types, or new relationships with target genes, or is evolutionary change largely the province of other classes of genes? In this review, we summarize what is known about conservation of homeobox gene function.\n",
      "Drosophila embryos lacking hindsight gene function have a normal body plan and undergo normal germ-band extension. However, they fail to retract their germ bands. hindsight encodes a large nuclear protein of 1920 amino acids that contains fourteen C2H2-type zinc fingers, and glutamine-rich and proline-rich domains, suggesting that it functions as a transcription factor. Initial embryonic expression of hindsight RNA and protein occurs in the endoderm (midgut) and extraembryonic membrane (amnioserosa) prior to germ-band extension and continues in these tissues beyond the completion of germ-band retraction. Expression also occurs in the developing tracheal system, central and peripheral nervous systems, and the ureter of the Malpighian tubules. Strikingly, hindsight is not expressed in the epidermal ectoderm which is the tissue that undergoes the cell shape changes and movements during germ-band retraction. The embryonic midgut can be eliminated without affecting germ-band retraction. However, elimination of the amnioserosa results in the failure of germ-band retraction, implicating amnioserosal expression of hindsight as crucial for this process. Ubiquitous expression of hindsight in the early embryo rescues germ-band retraction without producing dominant gain-of-function defects, suggesting that hindsight's role in germ-band retraction is permissive rather than instructive. Previous analyses have shown that hindsight is required for maintenance of the differentiated amnioserosa (Frank, L. C. and Rushlow, C. (1996) Development 122, 1343-1352). Two classes of models are consistent with the present data. First, hindsight's function in germ-band retraction may be limited to maintenance of the amnioserosa which then plays a physical role in the retraction process through contact with cells of the epidermal ectoderm. Second, hindsight might function bothto maintain the amnioserosa and to regulate chemical signaling from the amnioserosa to the epidermal ectoderm, thus coordinating the cell shape changes and movements that drive germ-band retraction.\n",
      "Drosophila melanogaster is a proven model system for many aspects of human biology. Here we present a two-hybrid-based protein-interaction map of the fly proteome. A total of 10,623 predicted transcripts were isolated and screened against standard and normalized complementary DNA libraries to produce a draft map of 7048 proteins and 20,405 interactions. A computational method of rating two-hybrid interaction confidence was developed to refine this draft map to a higher confidence map of 4679 proteins and 4780 interactions. Statistical modeling of the network showed two levels of organization: a short-range organization, presumably corresponding to multiprotein complexes, and a more global organization, presumably corresponding to intercomplex connections. The network recapitulated known pathways, extended pathways, and uncovered previously unknown pathway components. This map serves as a starting point for a systems biology modeling of multicellular organisms, including humans.\n",
      "During embryogenesis Drosophila pole cells, like germ cells in many other vertebrates and invertebrates, actively migrate before assembling into the gonad. Five separate steps can be distinguished: an initial passive displacement of the germ cells by gastrulation movements, an amoeboid migratory phase during which the pole cells pass through the endoderm, migration on endoderm toward mesoderm, separation into two bilateral groups associated with the gonadal mesoderm precursors, followed by condensation into the gonad itself. We have analyzed gonad assembly in embryonic pattern mutants to determine whether distinct cues are required in this process. We show that the initiation of migration does not require the presence of the mesoderm, the eventual target of the germ cells. Rather, migration is triggered as a consequence of the differentiation of the endoderm. Examination of embryos mutant for maternal genes of the terminal group suggests that a primary role of the endoderm in this process is to allow the germ cell access to the interior of the embryo. In addition, we show that normal gonad assembly requires a region of the embryo that includes the posterior compartment of the fifth and the sixth abdominal segments.\n",
      "During postblastoderm embryogenesis in Drosophila, cell cycles progress in an invariant spatiotemporal pattern. Most of these cycles are differentially timed by bursts of transcription of string (cdc25), a gene encoding a phosphatase that triggers mitosis by activating the Cdc2 kinase. An analysis of string expression in 36 pattern-formation mutants shows that known patterning genes act locally to influence string transcription. Embryonic expression of string gene fragments shows that the complete pattern of string transcription requires extensive cis-acting regulatory sequences (>15.3 kb), but that smaller segments of this regulatory region can drive proper temporal expression in defined spatial domains. We infer that string upstream sequences integrate many local signals to direct string's transcriptional program. Finally, we show that the spatiotemporal progression of string transcription is largely unaffected in mutant embryos specifically arrested in G2 of cycles 14, 15, or 16, or G1 of cycle 17. Thus, there is a regulatory hierarchy in which developmental inputs, not cell cycle inputs, control the timing of string transcription and hence cell cycle progression.\n",
      "During the maternal-to-zygotic transition, a developing embryo integrates post-transcriptional regulation of maternal mRNAs with transcriptional activation of its own genome. By combining chromosomal ablation in Drosophila with microarray analysis, we characterized the basis of this integration. We show that the expression profile for at least one third of zygotically active genes is coupled to the concomitant degradation of the corresponding maternal mRNAs. The embryo uses transcription and degradation to generate localized patterns of expression, and zygotic transcription to degrade distinct classes of maternal transcripts. Although degradation does not appear to involve a simple regulatory code, the activation of the zygotic genome starts from intronless genes sharing a common cis-element. This cis-element interacts with a single protein, the Bicoid stability factor, and acts as a potent enhancer capable of timing the activity of an exogenous transactivator. We propose that this regulatory mode links morphogen gradients with temporal regulation during the maternal-to-zygotic transition.\n",
      "Early Drosophila development requires two receptor tyrosine kinase (RTK) pathways: the Torso and the Epidermal growth factor receptor (EGFR) pathways, which regulate terminal and dorsal-ventral patterning, respectively. Previous studies have shown that these pathways, either directly or indirectly, lead to post-transcriptional downregulation of the Capicua repressor in the early embryo and in the ovary. Here, we show that both regulatory effects are direct and depend on a MAPK docking site in Capicua that physically interacts with the MAPK Rolled. Capicua derivatives lacking this docking site cause dominant phenotypes similar to those resulting from loss of Torso and EGFR activities. Such phenotypes arise from inappropriate repression of genes normally expressed in response to Torso and EGFR signaling. Our results are consistent with a model whereby Capicua is the main nuclear effector of the Torso pathway, but only one of different effectors responding to EGFR signaling. Finally, we describe differences in the modes of Capicua downregulation by Torso and EGFR signaling, raising the possibility that such differences contribute to the tissue specificity of both signals.\n",
      "Embryogenesis in individuals with mutations or deficiencies of the genes in the polytene interval 84A-84B1,2 of Drosophila melanogaster was examined using scanning electron microscopy (SEM). The developmental function of this region of chromosome 3 is of particular interest since it contains the Antennapedia Gene Complex (ANT-C), a gene cluster that includes the homoeotic proboscipedia (pb), Sex combs reduced (Scr), and Antennapedia (Antp) loci. The results of SEM studies, clonal analyses, and temperature-shift experiments show that the fushi tarazu (ftz) and zerknullt (zen) genes, which map between pb and Scr, are involved in processes initiated during embryogenesis. The activity of ftz+ appears to be required within the first 4 hr of development for the establishment of the proper number of segments in the embryonic germ band. Individuals with ftz mutations or deficiencies produce only half the normal number of segments. Each of the segments is twice the normal width and is apparently comprised of cells that would normally form two separate metameres. The zen allele is required from about 2-4 hr of embryogenesis. Mutations of this gene result in disturbances of morphogenetic movements during gastrulation. The mutant phenotype is characterized by the absence of the optic lobe, defects in involution of the head segments, and in some cases, failure of germ band elongation. A requirement during embryogenesis for the activities of other genes residing in the 84A-84B1,2 polytene interval is suggested by the phenotypes of individuals heterozygous or homozygous for chromosomal deficiencies. Using the deficiencies Df(3R)AntpNs+R17, Df(3R)Scr, and Df(3R)ScxW+RX2, we examined the effects of deleting the distal portions or all of the 84A-84B1,2 interval. The defects in deletion heterozygotes suggest that the wild-type activity of some gene(s) other than zen, within or just adjacent to the 84B1,2 doublet, is required to complete normal head involution. Thedeletion of all the loci in the 84A5-84B1,2 interval results in grossly abnormal morphology and morphogenesis of the gnathocephalic appendages of the embryo. From these studies we conclude that mutations and deficiencies of genes associated with the ANT-C have profound effects on embryogenesis. The mutant phenotypes suggest, in addition to ensuring proper segment identity, the wild-type alleles of the 84A-84B1,2 genes are necessary for normal segmentation and elongation of the germ band and normal head involution.\n",
      "Embryonic dorsal-ventral polarity in Drosophila is established through a series of successive steps and requires the functions of both maternal and zygotic genes. The graded distribution of the transcription factor dorsal in blastoderm nuclei represents the transition from the maternal to the zygotic program. This results in the activation of specific zygotic genes that act to create the regional pattern along this axis.\n",
      "Embryonic gene expression patterns are an indispensable part of modern developmental biology. Currently, investigators must visually inspect numerous images containing embryonic expression patterns to identify spatially similar patterns for inferring potential genetic interactions. The lack of a computational approach to identify pattern similarities is an impediment to advancement in developmental biology research because of the rapidly increasing amount of available embryonic gene expression data. Therefore, we have developed computational approaches to automate the comparison of gene expression patterns contained in images of early stage Drosophila melanogaster embryos (prior to the beginning of germ-band elongation); similarities and differences in gene expression patterns in these early stages have extensive developmental effects. Here we describe a basic expression search tool (BEST) to retrieve best matching expression patterns for a given query expression pattern and a computational device for gene interaction inference using gene expression pattern images and information on the associated genotypes and probes. Analysis of a prototype collection of Drosophila gene expression pattern images is presented to demonstrate the utility of these methods in identifying biologically meaningful matches and inferring gene interactions by direct image content analysis. In particular, the use of BEST searches for gene expression patterns is akin to that of BLAST searches for finding similar sequences. These computational developmental biology methodologies are likely to make the great wealth of embryonic gene expression pattern data easily accessible and to accelerate the discovery of developmental networks.\n",
      "Epistasis analysis has been used in molecular genetics to infer cause and effect relationships among loci that decide a developmental fate. In this paper, in order to investigate the logical background of the rule that relates epistasis and causal relation, a mathematical epistasis analysis is formulated. An \"orderliness index\"is introduced by which the cause-and-effect relation can systematically be extracted from genotype-phenotype data. This method is applied to Drosophila melanogaster dorso-ventral axis formation and it is shown that the method of reconstructing the framework of the causal relation between a locus and its sign (activation or inhibition) as an \"orderliness diagram\"from eight groups of the genes. From the diagram it can be seen that the dorsal gene works as a morphogen. The difference between orderliness diagrams of the dorsal and ventral sides of the embryo shows that the cells on the ventral side receive the signal and the fate of the cells on the dorsal side is default in the wild-type embryo.\n",
      "Establishment of dorsal-ventral polarity in the early Drosophila embryo requires a concentration gradient of the maternal morphogen dorsal (dl). This concentration gradient is established by selective nuclear transport of dl so that dl protein is present only in ventral nuclei. The activity of 11 genes is required for dl nuclear localization. One of these genes, Toll, encodes a transmembrane protein that appears to play the most direct role in regulating dl localization. We have examined the effects of Toll on dl in cotransfected Schneider cells to gain insight into the nature of the interaction between these proteins. We have found that Toll can enhance the nuclear localization of dl and, independently, the ability of dl to activate transcription once in the nucleus. We present evidence that the signaling pathway from Toll to dl involves protein kinase A (PKA) and that nuclear transport and activation of dl results from phosphorylation of dl by PKA. We discuss the significance of these results with respect both to Drosophila embryogenesis and to the regulation of the mammalian transcription factor NF-kappa B.\n",
      "Evolutionary constraints on gene regulatory elements are poorly understood: Little is known about how the strength of transcription factor binding correlates with DNA sequence conservation, and whether transcription factor binding sites can evolve rapidly while retaining their function. Here we use the model of the NFKB/Rel-dependent gene regulation in divergent Drosophila species to examine the hypothesis that the functional properties of authentic transcription factor binding sites are under stronger evolutionary constraints than the genomic background. Using molecular modeling we compare tertiary structures of the Drosophila Rel family proteins Dorsal, Dif, and Relish and demonstrate that their DNA-binding and protein dimerization domains undergo distinct rates of evolution. The accumulated amino acid changes, however, are unlikely to affect DNA sequence recognition and affinity. We employ our recently developed microarray-based experimental platform and principal coordinates statistical analysis to quantitatively and systematically profile DNA binding affinities of three Drosophila Rel proteins to 10,368 variants of the NFKB recognition sequences. We then correlate the evolutionary divergence of gene regulatory regions with differences in DNA binding affinities. Genome-wide analyses reveal a significant increase in the number of conserved Rel binding sites in promoters of developmental and immune genes. Significantly, the affinity of Rel proteins to these sites was higher than to less conserved sites and was maintained by the conservation of the DNA binding site sequence (static conservation) or in some cases despite significantly diverged sequences (dynamic conservation). We discuss how two types of conservation may contribute to the stabilization and optimization of a functional gene regulatory code in evolution.\n",
      "Extensive genetic and biochemical analysis of Drosophila melanogaster has made this system an important model for characterization of transcriptional regulatory elements and factors. Given the striking conservation of transcriptional controls in metazoans, general principles derived from studies of Drosophila are expected to continue to illuminate transcriptional regulation in other systems, including vertebrates. With improvement in technologies for genetic manipulation of insects, research in Drosophila will also aid the design of systems for controlled expression of genes in other hosts. This review focuses on recent advances from Drosophila in analysis of the functional components of transcriptional switches, including basal promoters, enhancers, boundary elements, and maintenance elements.\n",
      "Formation of the dorsoventral axis in Drosophila melanogaster is mediated through control of the expression of several genes by the morphogen Dorsal. In the ventral part of the embryo Dorsal activates twist and represses zen amongst others. Recently, several proteins have been shown to assist Dorsal in the repression of zen, one of which is DSP1, a HMG box protein that was isolated as a putative co-repressor of Dorsal. In this report we used a DSP1 null mutant to ascertain in vivo the involvement of DSP1 in Dorsal-mediated repression of zen but not in the activation of twist. We show that Dorsal has the ability to interact with DSP1 in vitro as well as with rat HMG1. Using truncated versions of the proteins we located the domains of interaction as being the HMG boxes for DSP1 and HMG1 and the Rel domain for Dorsal. Finally, studies of the zen DNA binding properties of Dorsal and another related Rel protein (Gambif1 from Anopheles gambiae) revealed that their DNA binding affinities were increased in the presence of DSP1 and HMG1.\n",
      "Functional assays in Drosophila melanogaster with orthologous transcription factors from other species suggest that changes in the protein-coding sequence may play a larger role in the evolution of transcription factor pathways than was previously believed. Interestingly, recent studies provide evidence that changes in transcription factor protein sequence can affect the regulation of only a subset of target genes, even in the same cells of a developing animal.\n",
      "Functional studies seem now to confirm, as first suggested by E. Geoffroy Saint-Hilaire in 1822, that there was an inversion of the dorsoventral axis during animal evolution. A conserved system of extracellular signals provides positional information for the allocation of embryonic cells to specific tissue types both in Drosophila and vertebrates; the ventral region of Drosophila is homologous to the dorsal side of the vertebrate. Developmental studies are now revealing some of the characteristics of the ancestral animal that gave rise to the arthropod and mammalian lineages, for which we propose the name Urbilateria.\n",
      "Genetic screens in Drosophila melanogaster have helped elucidate the process of axis formation during early embryogenesis. Axis formation in the D. melanogaster embryo involves the use of two fundamentally different mechanisms for generating morphogenetic activity: patterning the anteroposterior axis by diffusion of a transcription factor within the syncytial embryo and specification of the dorsoventral axis through a signal transduction cascade. Identification of Drosophila genes involved in axis formation provides a launch-pad for comparative studies that examine the evolution of axis specification in different insects. Additionally, there is similarity between axial patterning mechanisms elucidated genetically in Drosophila and those demonstrated for chordates such as Xenopus. In this review we examine the postfertilization mechanisms underlying axis specification in Drosophila. Comparative data are then used to ask whether aspects of axis formation might be derived or ancestral.\n",
      "Genetic studies have identified numerous sequence-specific transcription factors that control development, yet little is known about their in vivo distribution across animal genomes. We determined the genome-wide occupancy of the dorsoventral (DV) determinants Dorsal, Twist, and Snail in the Drosophila embryo using chromatin immunoprecipitation coupled with microarray analysis (ChIP-chip). The in vivo binding of these proteins correlate tightly with the limits of known enhancers. Our analysis predicts substantially more target genes than previous estimates, and includes Dpp signaling components and anteroposterior (AP) segmentation determinants. Thus, the ChIP-chip data uncover a much larger than expected regulatory network, which integrates diverse patterning processes during development.\n",
      "Genetic variation in brain size may provide the basis for the evolution of the brain and complex behaviours. The genetic substrate and the selective pressures acting on brain size are poorly understood. Here we use the Drosophila Genetic Reference Panel to map polymorphic variants affecting natural variation in mushroom body morphology. We identify 139 genes and 39 transcription factors and confirm effects on development and adult plasticity. We show correlations between morphology and aggression, sleep and lifespan. We propose that natural variation in adult brain size is controlled by interaction of the environment with gene networks controlling development and plasticity.\n",
      "Genetics and molecular analyses have combined to yield insights into a functional cascade of transcription factors necessary to establish the molecular blueprint of the Drosophila body pattern in response to positional information in the egg. Recent progress in this field raises exciting questions regarding the molecular mechanisms involved, and their conservation in biological pattern-forming processes.\n",
      "Genome sequencesfor most metazoans and plants are incomplete because of the presence of repeated DNA in the heterochromatin. The heterochromatic regions of Drosophila melanogaster contain 20 million bases (Mb) of sequence amenable to mapping, sequence assembly, and finishing. We describe the generation of 15 Mb of finished or improved heterochromatic sequence with the use of available clone resources and assembly methods. We also constructed a bacterial artificial chromosome-based physical map that spans 13 Mb of the pericentromeric heterochromatin and a cytogenetic map that positions 11 Mb in specific chromosomal locations. We have approached a complete assembly and mapping of the nonsatellite component of Drosophila heterochromatin. The strategy we describe is also applicable to generating substantially more information about heterochromatin in other species, including humans.\n",
      "Gradients of morphogens determine cell fates by specifying discrete thresholds of gene activities. In the Drosophila embryo, a BMP gradient subdivides the dorsal ectoderm into amnioserosa and dorsal epidermis, and also inhibits neuroectoderm formation. A number of genes are differentially expressed in response to the gradient, but how their borders of expression are established is not well understood. We present evidence that the BMP gradient, via the Smads, provides a two-fold input in regulating the amnioserosa-specific target genes such as Race. Peak levels of Smads in the presumptive amnioserosa set the expression domain of zen, and then Smads act in combination with Zen to directly activate Race. This situation resembles a feed-forward mechanism of transcriptional regulation. In addition, we demonstrate that ectopically expressed Zen can activate targets like Race in the presence of low level Smads, indicating that the role of the highest activity of the BMP gradient is to activate zen.\n",
      "Groucho (Gro) is a Drosophila corepressor required by numerous DNA-binding repressors, many of which are distributed in gradients and provide positional information during development. Gro contains well-conserved domains at its N- and C-termini, and a poorly conserved central region that includes the GP, CcN, and SP domains. All lethal point mutations in gro map to the conserved regions, leading to speculation that the unconserved central domains are dispensable. However, our sequence analysis suggests that the central domains are disordered leading us to suspect that the lack of lethal mutations in this region reflects a lack of order rather than an absence of essential functions. In support of this conclusion, genomic rescue experiments with Gro deletion variants demonstrate that the GP and CcN domains are required for viability. Misexpression assays using these same deletion variants show that the SP domain prevents unrestrained and promiscuous repression by Gro, while the GP and CcN domains are indispensable for repression. Deletion of the GP domain leads to loss of nuclear import, while deletion of the CcN domain leads to complete loss of repression. Changes in Gro activity levels reset the threshold concentrations at which graded repressors silence target gene expression. We conclude that co-regulators such as Gro are not simply permissive components of the repression machinery, but cooperate with graded DNA-binding factors in setting borders of gene expression. We suspect that disorder in the Gro central domains may provide the flexibility that allows this region to mediate multiple interactions required for repression.\n",
      "HOT (highly occupied target) regions bound by many transcription factors are considered to be one of the most intriguing findings of the recent modENCODE reports, yet their functions have remained unclear. We tested 108 Drosophila melanogaster HOT regions in transgenic embryos with site-specifically integrated transcriptional reporters. In contrast to prior expectations, we found 102 (94%) to be active enhancers during embryogenesis and to display diverse spatial and temporal patterns, reminiscent of expression patternsfor important developmental genes. Remarkably, HOT regions strongly activate nearby genes and are required for endogenous gene expression, as we show using bacterial artificial chromosome (BAC) transgenesis. HOT enhancers have a distinct cis-regulatory signature with enriched sequence motifs for the global activators Vielfaltig, also known as Zelda, and Trithorax-like, also known as GAGA. This signature allows the prediction of HOT versus control regions from the DNA sequence alone.\n",
      "Here, we describe one of the major maternal regulatory gradients, Dorsal, and threshold outputs of gene expression that result from the graded distribution of this transcription factor. The analysis of a large number of authentic and synthetic target genes suggests that the Dorsal gradient directly specifies at least four, and possibly as many as seven, different thresholds of gene activity and tissue differentiation. These thresholds initiate the differentiation of the three primary embryonic tissues: the mesoderm, neurogenic ectoderm, and dorsal ectoderm. Moreover, primary readouts of the Dorsal gradient create asymmetries that subdivide each tissue into multiple cell types during gastrulation. Dorsal patterning thresholds represent the culmination of one of the most complete gene regulation network known in development, which begins with the asymmetric positioning of the oocyte nucleus within the egg chamber and leads to the localized activation of the Toll-Dorsal signaling pathway in ventral regions of the early embryo.\n",
      "Holometabolous insects like Drosophila proceed through two phases of visual system development. The embryonic phase generates simple eyes of the larva. The postembryonic phase produces the adult specific compound eyes during late larval development and pupation. In primitive insects, by contrast, eye development persists seemingly continuously from embryogenesis through the end of postembryogenesis. Comparative literature suggests that the evolutionary transition from continuous to biphasic eye development occurred via transient developmental arrest. This review investigates how the developmental arrest model relates to the gene networks regulating larval and adult eye development in Drosophila, and embryonic compound eye development in primitive insects. Consistent with the developmental arrest model, the available data suggest that the determination of the anlage of the rudimentary Drosophila larval eye is homologous to the embryonic specification of the juvenile compound eye in directly developing insects while the Drosophila compound eye primordium is evolutionarily related to the yet little studied stem cell based postembryonic eye primordium of primitive insects.\n",
      "Homeobox genes encode important developmental control proteins. The Drosophila fruit fly HOM complex genes are clustered in region 84-89 of chromosome 3. Probably due to large-scale genome duplication events, their human HOX orthologs belong to four paralogous regions. A series of 13 other homeobox genes are also clustered in region 88-94, on the same chromosome of Drosophila. We suggest that they also duplicated during vertebrate evolution and belong to paralogous regions in humans. These regions are on chromosome arms 4p, 5q, 10q, and 2p or 8p. We coined the term \"paralogon\"to designate paralogous regions in general. We propose to call these genes \"meta Hox\"genes. Like Hox genes, metaHox genes are present in one cluster in Drosophila and four clusters (metaHox A-D) in humans on the 4p/5q/10q paralogon.\n",
      "Homeotic (Hox) genes are usually clustered and arranged in the same order as they are expressed along the anteroposterior body axis of metazoans. The mechanistic explanation for this colinearity has been elusive, and it may well be that a single and universal cause does not exist. The Hox-gene complex (HOM-C) has been rearranged differently in several Drosophila species, producing a striking diversity of Hox gene organizations. We investigated the genomic and functional consequences of the two HOM-C splits present in Drosophila buzzatii. Firstly, we sequenced two regions of the D. buzzatii genome, one containing the genes labial and abdominal A, and another one including proboscipedia, and compared their organization with that of D. melanogaster and D. pseudoobscura in order to map precisely the two splits. Then, a plethora of conserved noncoding sequences, which are putative enhancers, were identified around the three Hox genes closer to the splits. The position and order of these enhancers are conserved, with minor exceptions, between the three Drosophila species. Finally, we analyzed the expression patterns of the same three genes in embryos and imaginal discs of four Drosophila species with different Hox-gene organizations. The results show that their expression patterns are conserved despite the HOM-C splits. We conclude that, in Drosophila, Hox-gene clustering is not an absolute requirement for proper function. Rather, the organization of Hox genes is modular, and their clustering seems the result of phylogenetic inertia more than functional necessity.\n",
      "Homeotic genes identify structures along the anterior to posterior axis during the development of most animals. These genes are clustered into complexes, and their positions within the cluster correlates with their time of expression and the positions of the anterioposterior boundaries of their expression domains. Functional analyses have revealed that this specific genetic order also coincides with a functional hierarchy among members of these complexes, so that the products of more posterior genes in the cluster tend to be prevalent over those of more anterior genes.\n",
      "Identifying the genomic regions bound by sequence-specific regulatory factors is central both to deciphering the complex DNA cis-regulatory code that controls transcription in metazoans and to determining the range of genes that shape animal morphogenesis. We used whole-genome tiling arrays to map sequences bound in Drosophila melanogaster embryos by the six maternal and gap transcription factors that initiate anterior-posterior patterning. We find that these sequence-specific DNA binding proteins bind with quantitatively different specificities to highly overlapping sets of several thousand genomic regions in blastoderm embryos. Specific high- and moderate-affinity in vitro recognition sequences for each factor are enriched in bound regions. This enrichment, however, is not sufficient to explain the pattern of binding in vivo and varies in a context-dependent manner, demonstrating that higher-order rules must govern targeting of transcription factors. The more highly bound regions include all of the over 40 well-characterized enhancers known to respond to these factors as well as several hundred putative new cis-regulatory modules clustered near developmental regulators and other genes with patterned expression at this stage of embryogenesis. The new targets include most of the microRNAs (miRNAs) transcribed in the blastoderm, as well as all major zygotically transcribed dorsal-ventral patterning genes, whose expression we show to be quantitatively modulated by anterior-posterior factors. In addition to these highly bound regions, there are several thousand regions that are reproducibly bound at lower levels. However, these poorly bound regions are, collectively, far more distant from genes transcribed in the blastoderm than highly bound regions; are preferentially found in protein-coding sequences; and are less conserved than highly bound regions. Together these observations suggest that many of these poorly bound regions are not involved in early-embryonic transcriptional regulation, and a significant proportion may be nonfunctional. Surprisingly, for five of the six factors, their recognition sites are not unambiguously more constrained evolutionarily than the immediate flanking DNA, even in more highly bound and presumably functional regions, indicating that comparative DNA sequence analysis is limited in its ability to identify functional transcription factor targets.\n",
      "In Drosophila elaboration of positional identity along the anterior-posterior and dorsal-ventral embryonic body axes involves early zygotic gene functions that are expressed in response to maternal cues present in the unfertilized egg. Zygotic loci that are required for the specification of positional identity along the anterior-posterior body axis have been described in detail. Less is known about the zygotic loci responsible for differentiation of the dorsal-ventral pattern; however, several genes that might be involved have been identified. Zerknüllt (zen) is an example of a zygotic gene required for correct differentiation of dorsally derived embryonic tissues. On the basis of homoeo box cross homology, we have now isolated a gene, called S60, that derives from the zen region of the Antennapedia complex (ANT-C). Transcripts encoded by S60 transiently accumulate in the dorsal-most tissues of developing embryos. This pattern of expression suggests that S60 corresponds to zen. Since S60 contains a homoeo box, it is possible that differentiation of the anterior-posterior and dorsal-ventral embryonic patterns involves similar molecular mechanisms.\n",
      "In Drosophila embryos, dorsal-ventral polarity is defined by a signal transduction pathway that regulates nuclear import ofthe Dorsal protein. Dorsal protein's ability to act as a transcriptional activator of some zygotic genes and a repressor of others defines structure along the dorsal-ventral axis. Dorsal is a member of a group of proteins, the Rel-homologous proteins, whose activity is regulated at the level of nuclear localization. Dif, a more recently identified Drosophila Rel-homologue, has been proposed to act as a mediator of the immune response in Drosophila. In an effort to understand the function and regulation of Rel-homologous proteins in Drosophila, we have expressed Dif protein in Drosophila embryos derived from dorsal mutant mothers. We found that the Dif protein was capable of restoring embryonic dorsal-ventral pattern elements and was able to define polarity correctly with respect to the orientation of the egg shell. This, together with the observation that the ability of Dif to restore a dorsal-ventral axis depended on the signal transduction pathway that normally regulates Dorsal, suggests that Dif protein formed a nuclear concentration gradient similar to that seen for Dorsal. By studying the expression of Dorsal target genes we found that Dif could activate the zygotic genes that Dorsal activates and repress the genes repressed by Dorsal. Differences in the expression of these target genes, as well as the results from interaction studies carried out in yeast, suggest that Dif is not capable of synergizing with the basic helix-loop-helix transcription factors with which Dorsal normally interacts, and thereby lacks an important component of Dorsal-mediated pattern formation.\n",
      "In Drosophila, a Bcd protein gradient orchestrates patterning along the anteroposterior embryonic axis. However, studies of basal flies and other insects have revealed that bcd is a derived Hox3 gene found only in higher dipterans. To understand how bcd acquired its role in flies and how anteroposterior patterning mechanisms have evolved, I first review key features of bcd function in Drosophila: anterior localization and transcriptional and translation control of gene expression. I then discuss investigations of bcd in other higher dipterans that have provided insight into the evolution of regulatory interactions and the Bcd gradient. Finally, I review studies of Drosophila and other insects that address the evolution of bcd function and integration of bcd into ancestral regulatory mechanisms. I suggest further comparative studies may allow us to identify the intermediate steps in bcd evolution. This will make bcd a paradigm for the origin and evolution of genes and regulatory networks.\n",
      "In Drosophila, pattern formation at multiple stages of embryonic and imaginal development depends on the same intercellular signaling pathways. We have identified a novel gene, eyelid (eld), which is required for embryonic segmentation, development of the notum and wing margin, and photoreceptor differentiation. In these tissues, eld mutations have effects opposite to those caused by wingless (wg) mutations. eld encodes a widely expressed nuclear protein with a region homologous to a novel family of DNA-binding domains. Based on this homology and on the phenotypic analysis, we suggest that Eld could act as a transcription factor antagonistic to the Wg pathway.\n",
      "In Drosophila, the dorsal (dl) morphogen gradient initiates the differentiation of the embryonic mesoderm and neuroectoderm by activating the expression of regulatory genes (e.g. twist and snail) in a concentration-dependent manner. dl also functions as a repressor that establishes the dorsal epidermis and amnioserosa by restricting regulatory genes such as dpp and zen to dorsal regions of the embryo. The ability of dl to function as both an activator and repressor distinguishes it from the bicoid morphogen, which appears to function solely as an activator. In an effort to determine how dl functions as a repressor we have performed a detailed characterization of a zen silencer element, called the VRE, which mediates ventral repression in response to the dl gradient. A minimal 110 bp VRE sequence is identified, which is able to silence the ventral expression of a heterologous promoter. This sequence contains two dl binding sites as well as binding sites for additional nuclear factors present in early embryos. Mutations in the latter binding sites convert the minimal VRE into an enhancer, which mediates transcriptional activation in ventral regions in response to dl. These results suggest that dl is intrinsically an activator, but is converted into a potent silencer when it interacts with neighboring corepressors.\n",
      "In Drosophila, two TGF-beta growth factors, dpp and screw, function synergistically to subdivide the dorsal ectoderm into two embryonic tissues, the amnioserosa and dorsal epidermis. Previous studies have shown that peak dpp activity is required for the localized expression of zerknüllt (zen), which encodes a homeodomain transcription factor. We present evidence that zen directly activates the amnioserosa-specific expression of a downstream target gene, Race (Related to angiotensin converting enzyme). A 533 bp enhancer from the Race promoter region is shown to mediate selective expression in the amnioserosa, as well as the anterior and posterior midgut rudiments. This enhancer contains three zen protein binding sites, and mutations in these sites virtually abolish the expression of an otherwise normal Race-lacZ fusion gene in the amnioserosa, but not in the gut. Genetic epistasis experiments suggest that zen is not the sole activator of Race, although a hyperactivated form of zen (a zen-VP16 fusion protein) can partially complement reduced levels of dpp activity. These results suggest that dpp regulates multiple transcription factors, which function synergistically to specify the amnioserosa.\n",
      "In a developing Drosophila melanogaster embryo, mRNAs have a maternal origin, a zygotic origin, or both. During the maternal-zygotic transition, maternal products are degraded and gene expression comes under the control of the zygotic genome. To interrogate the function of mRNAs that are both maternally and zygotically expressed, it is common to examine the embryonic phenotypes derived from female germline mosaics. Recently, the development of RNAi vectors based on short hairpin RNAs (shRNAs) effective during oogenesis has provided an alternative to producing germline clones. Here, we evaluate the efficacies of: (1) maternally loaded shRNAs to knockdown zygotic transcripts and (2) maternally loaded Gal4 protein to drive zygotic shRNA expression. We show that, while Gal4-driven shRNAs in the female germline very effectively generate phenotypes for genes expressed maternally, maternally loaded shRNAs are not very effective at generating phenotypes for early zygotic genes. However, maternally loaded Gal4 protein is very efficient at generating phenotypes for zygotic genes expressed during mid-embryogenesis. We apply this powerful and simple method to unravel the embryonic functions of a number of pleiotropic genes.\n",
      "In a screen based on a rough eye phenotype caused by a dominant negative form of the BEAF-32A and BEAF-32B insulator proteins, we previously identified 17 proteins that genetically interact with BEAF. Eleven of these are developmental transcription factors, seven of which are encoded by the Antennapedia complex (ANT-C). While investigating potential reasons for the genetic interactions, we obtained evidence that BEAF plays a role in the regulation of genes in the ANT-C. BEAF does not localize near the transcription start sites of any genes in the ANT-C, indicating that BEAF does not locally affect regulation of these genes. Although BEAF affects chromatin structure or dynamics, we also found no evidence for a general change in binding to polytene chromosomes in the absence of BEAF. However, because we were unable to detect proteins encoded by ANT-C genes in salivary glands, the DREF and MLE proteins were used as proxies to examine binding. This does not rule out limited effects at particular binding sites or the possibility that BEAF might directly interact with certain transcription factors to affect their binding. In contrast, the embryonic expression levels and patterns of four examined ANT-C genes were altered (bcd, Dfd, ftz, pb). A control gene, Dref, was not affected. A full understanding of the regulation of ANT-C genes during development will have to take the role of BEAF into account.\n",
      "In all animals, the initial events of embryogenesis are controlled by maternal gene products that are deposited into the developing oocyte. At some point after fertilization, control of embryogenesis is transferred to the zygotic genome in a process called the maternal-to-zygotic transition. During this time, many maternal RNAs are degraded and transcription of zygotic RNAs ensues. There is a long-standing question as to which factors regulate these events. The recent findings that microRNAs and Smaug mediate maternal transcript degradation have shed new light on this aspect of the problem. However, the transcription factor(s) that activate the zygotic genome remain elusive. The discovery that many of the early transcribed genes in Drosophila share a cis-regulatory heptamer motif, CAGGTAG and related sequences, collectively referred to as TAGteam sites raised the possibility that a dedicated transcription factor could interact with these sites to activate transcription. Here we report that the zinc-finger protein Zelda (Zld; Zinc-finger early Drosophila activator) binds specifically to these sites and is capable of activating transcription in transient transfection assays. Mutant embryos lacking zld are defective in cellular blastoderm formation, and fail to activate many genes essential for cellularization, sex determination and pattern formation. Global expression profiling confirmed that Zld has an important role in the activation of the early zygotic genome and suggests that Zld may also regulate maternal RNA degradation during the maternal-to-zygotic transition.\n",
      "In many developmental contexts, a locally produced morphogen specifies positional information by forming a concentration gradient over a field of cells. However, during embryonic dorsal-ventral patterning in Drosophila, two members of the bone morphogenetic protein (BMP) family, Decapentaplegic (Dpp) and Screw (Scw), are broadly transcribed but promote receptor-mediated signalling in a restricted subset of expressing cells. Here we use a novel immunostaining protocol to visualize receptor-bound BMPs and show that both proteins become localized to a sharp stripe of dorsal cells. We demonstrate that proper BMP localization involves two distinct processes. First, Dpp undergoes directed, long-range extracellular transport. Scw also undergoes long-range movement, but can do so independently of Dpp transport. Second, an intracellular positive feedback circuit promotes future ligand binding as a function of previous signalling strength. These data elicit a model in which extracellular Dpp transport initially creates a shallow gradient of BMP binding that is acted on by positive intracellular feedback to produce two stable states of BMP-receptor interactions, a spatial bistability in which BMP binding and signalling capabilities are high in dorsal-most cells and low in lateral cells.\n",
      "In many organisms, transcription of the zygotic genome begins during the maternal-to-zygotic transition (MZT), which is characterized by a dramatic increase in global transcriptional activities and coincides with embryonic stem cell differentiation. In Drosophila, it has been shown that maternal morphogen gradients and ubiquitously distributed general transcription factors may cooperate to upregulate zygotic genes that are essential for pattern formation in the early embryo. Here, we show that Drosophila STAT (STAT92E) functions as a general transcription factor that, together with the transcription factor Zelda, induces transcription of a large number of early-transcribed zygotic genes during the MZT. STAT92E is present in the early embryo as a maternal product and is active around the MZT. DNA-binding motifs for STAT and Zelda are highly enriched in promoters of early zygotic genes but not in housekeeping genes. Loss of Stat92E in the early embryo, similarly to loss of zelda, preferentially down-regulates early zygotic genes important for pattern formation. We further show that STAT92E and Zelda synergistically regulate transcription. We conclude that STAT92E, in conjunction with Zelda, plays an important role in transcription of the zygotic genome at the onset of embryonic development.\n",
      "In past years, much attention has focused on the gene networks that regulate early developmental processes, but less attention has been paid to how multiple networks and processes are temporally coordinated. Recently the discovery of the transcriptional activator Zelda (Zld), which binds to CAGGTAG and related sequences present in the enhancers of many early-activated genes in Drosophila, hinted at a mechanism for how batteries of genes could be simultaneously activated. Here we use genome-wide binding and expression assays to identify Zld target genes in the early embryo with the goal of unraveling the gene circuitry regulated by Zld. We found that Zld binds to genes involved in early developmental processes such as cellularization, sex determination, neurogenesis, and pattern formation. In the absence of Zld, many target genes failed to be activated, while others, particularly the patterning genes, exhibited delayed transcriptional activation, some of which also showed weak and/or sporadic expression. These effects disrupted the normal sequence of patterning-gene interactions and resulted in highly altered spatial expression patterns, demonstrating the significance of a timing mechanism in early development. In addition, we observed prevalent overlap between Zld-bound regions and genomic \"hotspot\"regions, which are bound by many developmental transcription factors, especially the patterning factors. This, along with the finding that the most over-represented motif in hotspots, CAGGTA, is the Zld binding site, implicates Zld in promoting hotspot formation. We propose that Zld promotes timely and robust transcriptional activation of early-gene networks so that developmental events are coordinated and cell fates are established properly in the cellular blastoderm embryo.\n",
      "In situ hybridization is an important technique for measuring the spatial expression patterns of mRNA in cells, tissues, and whole animals. However, mRNA levels cannot be compared across experiments using typical protocols. Here we present a semi-quantitative method to compare mRNA levels of a gene across multiple samples. This method yields an estimate of the error in the measurement to allow statistical comparison. Our method uses a typical in situ hybridization protocol to stain for a target gene and an internal standard, which we refer to as a co-stain. As a proof of concept, we apply this method to multiple lines of transgenic Drosophila embryos, harboring constructs that express reporter genes to different levels. We generated this test set by mutating enhancer sequences to contain different numbers of binding sites for Zelda, a transcriptional activator. We demonstrate that using a co-stain with in situ hybridization is an effective method to compare mRNA levels across samples. This method requires only minor modifications to existing in situ hybridization protocols and uses straightforward analysis techniques. This strategy can be broadly applied to detect quantitative, spatially resolved changes in mRNA levels.\n",
      "In the long-germ insect Drosophila melanogaster dorsoventral polarity is induced by localized Toll-receptor activation which leads to the formation of a nuclear gradient of the rel/ NF-kappaB protein Dorsal. Peak levels of nuclear Dorsal are found in a ventral stripe spanning the entire length of the blastoderm embryo allowing all segments and their dorsoventral subdivisions to be synchronously specified before gastrulation. We show that a nuclear Dorsal protein gradient of similar anteroposterior extension exists in the short-germ beetle, Tribolium castaneum, which forms most segments from a posterior growth zone after gastrulation. In contrast to Drosophila, (i) nuclear accumulation is first uniform and then becomes progressively restricted to a narrow ventral stripe, (ii) gradient refinement is accompanied by changes in the zygotic expression of the Tribolium Toll-receptor suggesting feedback regulation and, (iii) the gradient only transiently overlaps with the expression of a potential target, the Tribolium twist homolog, and does not repress Tribolium decapentaplegic. No nuclear Dorsal is seen in the cells of the growth zone of Tribolium embryos, indicating that here dorsoventral patterning occurs by a different mechanism. However, Dorsal is up-regulated and transiently forms a nuclear gradient in the serosa, a protective extraembryonic cell layer ultimately covering the whole embryo.\n",
      "Infection results in the rapid activation of immunity genes in the Drosophila fat body. Two classes of transcription factors have been implicated in this process: the REL-containing proteins, Dorsal, Dif, and Relish, and the GATA factor Serpent. Here we present evidence that REL-GATA synergy plays apervasive role in the immune response. SELEX assays identified consensus binding sites that permitted the characterization of several immunity regulatory DNAs. The distribution of REL and GATA sites within these DNAs suggests that most or all fat-specific immunity genes contain a common organization of regulatory elements: closely linked REL and GATA binding sites positioned in the same orientation and located near the transcription start site. Aspects of this \"regulatory code\"are essential for the immune response. These results suggest that immunity regulatory DNAs contain constrained organizational features, which may be a general property of eukaryotic enhancers.\n",
      "It is becoming increasingly clear that transcriptional repression is at least as important as transcriptional activation for establishing cell-type specific patterns of gene expression during embryogenesis. Recent studies in Drosophila suggest that repressors fall into two categories, short-range and long-range repressors. The former permit enhancer autonomy in modular promoters, whereas the latter function in a dominant fashion to silence multiple enhancers.\n",
      "It is expected that genes that are expressed early in development and have a complex expression pattern are under strong purifying selection and thus evolve slowly. Hox genes fulfill these criteria and thus, should have a low evolutionary rate. However, some observations point to a completely different scenario. Hox genes are usually highly conserved inside the homeobox, but very variable outside it. We have measured the rates of nucleotide divergence and indel fixation of three Hox genes, labial (lab), proboscipedia (pb) and abdominal-A (abd-A), and compared them with those of three genes derived by duplication from Hox3, bicoid (bcd), zerknüllt (zen) and zerknüllt-related (zen2), and 15 non-Hox genes in sets of orthologous sequences of three species of the genus Drosophila. These rates were compared to test the hypothesis that Hox genes evolve slowly. Our results show that the evolutionary rate of Hox genes is higher than that of non-Hox genes when both amino acid differences and indels are taken into account: 43.39% of the amino acid sequence is altered in Hox genes, versus 30.97% in non-Hox genes and 64.73% in Hox-derived genes. Microsatellites scattered along the coding sequence of Hox genes explain partially, but not fully, their fast sequence evolution. These results show that Hox genes have a higher evolutionary dynamics than other developmental genes, and emphasize the need to take into account indels in addition to nucleotide substitutions in order to accurately estimate evolutionary rates.\n",
      "Like many DNA binding transcription factors, the Drosophila morphogen encoded by dorsal can both stimulate and repress promoter activity. In particular, this factor activates twist and represses zerknüllt on the ventral side of the early embryo. We find that when multiple copies of a dorsal binding site from the zerknüllt ventral repressor element are fused to a heterologous basal promoter, the resulting construct is activated by dorsal to give a ventral specific expression pattern. Thus, the ability of a dorsal binding site to mediate repression rather than activation is not an intrinsic property of the site, but depends upon its context. We also show that a hybrid promoter containing both the zerknüllt ventral repressor element and the twist ventral activator region is not ventrally active in the early embryo, demonstrating that repression is dominant over activation. Thus, the default mode of action of the dorsal protein is transcriptional activation. Additional factors may modify dorsal activity to bring about repression.\n",
      "Major advances have been made in understanding the evolution of transcriptional regulation using microevolutionary and macroevolutionary experimental approaches. The roles of stabilising selection and compensatory changes in an enhancer region have been elucidated in Drosophila. The molecular dynamics of regulatory alleles have been studied in plants. Evidence is accumulating for the involvement of regulatory evolution in morphological changes between closely related species, as well as in major changes of body plans.\n",
      "Many proteins of the Rel family can act as both transcriptional activators and repressors. However, mechanism that discerns the 'activator/repressor' functions of Rel-proteins such as Dorsal (Drosophila homologue of mammalian NFκB) is not understood. Using genomic, biophysical and biochemical approaches, we demonstrate that the underlying principle of this functional specificity lies in the 'sequence-encoded structure' of the κB-DNA. We show that Dorsal-binding motifs exist in distinct activator and repressor conformations. Molecular dynamics of DNA-Dorsal complexes revealed that repressor κB-motifs typically have A-tract and flexible conformation that facilitates interaction with co-repressors. Deformable structure of repressor motifs, is due to changes in the hydrogen bonding in A:T pair in the 'A-tract' core. The sixth nucleotide in the nonameric κB-motif, 'A' (A(6)) in the repressor motifs and 'T' (T(6)) in the activator motifs, is critical to confer this functional specificity as A(6) → T(6) mutation transformed flexible repressor conformation into a rigid activator conformation. These results highlight that 'sequence encoded κB DNA-geometry' regulates gene expression by exerting allosteric effect on binding of Rel proteins which in turn regulates interaction with co-regulators. Further, we identified and characterized putative repressor motifs in Dl-target genes, which can potentially aid in functional annotation of Dorsal gene regulatory network.\n",
      "Massive zygotic transcription begins in many organisms during the midblastula transition when the cell cycle of the dividing egg slows down. A few genes are transcribed before this stage but how this differential activation is accomplished is still an open question. We have performed ChIP-seq experiments on tightly staged Drosophila embryos and show that massive recruitment of RNA polymerase II (Pol II) with widespread pausing occurs de novo during the midblastula transition. However, ∼100 genes are strongly occupied by Pol II before this timepoint and most of them do not show Pol II pausing, consistent with a requirement for rapid transcription during the fast nuclear cycles. This global change in Pol II pausing correlates with distinct core promoter elements and associates a TATA-enriched promoter with the rapid early transcription. This suggests that promoters are differentially used during the zygotic genome activation, presumably because they have distinct dynamic properties. DOI:http://dx.doi.org/10.7554/eLife.00861.001.\n",
      "Maternal genes involved in dorsoventral (D/V) patterning of the Drosophila embryo interact to establish a stable nuclear concentration gradient of the Dorsal protein which acts as the morphogen along this axis. This protein belongs to the rel proto-oncogene and NF-KB transcriptional factor family and acts by controlling zygotic gene expression. In the ventral part of the embryo, dorsal specifically activates transcription of the gene twist and ventrally and laterally dorsal represses the expression of zerknüllt, a gene involved in the formation of dorsal derivatives. The extent of dorsal action is closely related to the affinity and the number of dorsal response elements present in these zygotic gene promoters. twist is one of the first zygotic genes necessary for mesoderm formation. It codes for a 'b-HLH' DNA-binding protein which can dimerize and bind to DNA in vitro and to polytene chromosomes in vivo. In addition, in cultured cells twist has been shown to be a transcriptional activator. Thus, the first events of embryonic development along the D/V axis are controlled at the transcriptional level.\n",
      "Maternally contributed mRNAs and proteins control the initial stages of development following fertilization. During this time, most of the zygotic genome remains transcriptionally silent. The initiation of widespread zygotic transcription is coordinated with the degradation of maternally provided mRNAs at the maternal-to-zygotic transition (MZT). While most of the genome is silenced prior to the MZT, a small subset of zygotic genes essential for the future development of the organism is transcribed. Previous work in our laboratory and others identified the TAGteam element, a set of related heptameric DNA-sequences in the promoters of many early-expressed Drosophila genes required to drive their unusually early transcription. To understand how this unique subset of genes is regulated, we identified a TAGteam-binding factor Grainyhead (Grh). We demonstrated that Grh and the previously characterized transcriptional activator Zelda (Zld) bind to different TAGteam sequences with varying affinities, and that Grh competes with Zld for TAGteam occupancy. Moreover, overexpression of Grh in the early embryo causes defects in cell division, phenocopying Zld depletion. Our findings indicate that during early embryonic development the precise timing of gene expression is regulated by both the sequence of the TAGteam elements in the promoter and the relative levels of the transcription factors Grh and Zld.\n",
      "Members of the Eyes absent (Eya) protein family play important roles in tissue specification and patterning by serving as both transcriptional activators and protein tyrosine phosphatases. These activities are often carried out in the context of complexes containing members of the Six and/or Dach families of DNA binding proteins. eyes absent, the founding member of the Eya family is expressed dynamically within several embryonic, larval, and adult tissues of the fruit fly, Drosophila melanogaster. Loss-of-function mutations are known to result in disruptions of the embryonic head and central nervous system as well as the adult brain and visual system, including the compound eyes. In an effort to understand how eya is regulated during development, we have carried out a genetic screen designed to identify genes that lie upstream of eya and govern its expression. We have identified a large number of putative regulators, including members of several signaling pathways. Of particular interest is the identification of both yan/anterior open and pointed, two members of the EGF Receptor (EGFR) signaling cascade. The EGFR pathway is known to regulate the activity of Eya through phosphorylation via MAPK. Our findings suggest that this pathway is also used to influence eya transcriptional levels. Together these mechanisms provide a route for greater precision in regulating a factor that is critical for the formation of a wide range of diverse tissues.\n",
      "Metazoan genomes contain vast tracts of cis-regulatory DNA that have been identified typically through tedious functional assays. As a result, it has not been possible to uncover a cis-regulatory code that links primary DNA sequences to gene expression patterns. In an initial effort to determine whether coordinately regulated genes share a common \"grammar,\"we have examined the distribution of Dorsal recognition sequences in the Drosophila genome. Dorsal is one of the best-characterized sequence-specific transcription factors in Drosophila. The homeobox gene zerknullt (zen) is repressed directly by Dorsal, and this repression is mediated by a 600-bp silencer, the ventral repression element (VRE), which contains four optimal Dorsal binding sites. The arrangement and sequence of the Dorsal recognition sequences in the VRE were used to develop a computational algorithm to search the Drosophila genome for clusters of optimal Dorsal binding sites. There are 15 regions in the genome that contain three or more optimal sites within a span of 400 bp or less. Three of these regions are associated with known Dorsal target genes: sog, zen, and Brinker. The Dorsal binding cluster in sog is shown to mediate lateral stripes of gene expression in response to low levels of the Dorsal gradient. Two of the remaining 12 clusters are shown to be associated with genes that exhibit asymmetric patterns of expression across the dorsoventral axis. These results suggest that bioinformatics can be used to identify novel target genes and associated regulatory DNAs in a gene network.\n",
      "Microarray assays and bioinformatics methods have identified many of the genes and associated regulatory DNAs that control the early phases of gastrulation in Drosophila. The localized activities of these genes are coordinated by a nuclear gradient of the maternal regulatory factor, Dorsal, that is established shortly after fertilization. At least half of the Dorsal target genes encode transcription factors or signaling components that lead to the restricted activation of FGF, EGF, and TGF-beta signaling pathways in the mesoderm, neurogenic ectoderm and dorsal ectoderm, respectively. Recent work has yielded insights into how these signaling pathways control gastrulation, particularly in the context of the Dorsal-mediated gene regulation network\n",
      "Modern developmental biology relies heavily on the analysis of embryonic gene expression patterns. Investigators manually inspect hundreds or thousands of expression patterns to identify those that are spatially similar and to ultimately infer potential gene interactions. However, the rapid accumulation of gene expression pattern data over the last two decades, facilitated by high-throughput techniques, has produced a need for the development of efficient approaches for direct comparison of images, rather than their textual descriptions, to identify spatially similar expression patterns. The effectiveness of the Binary Feature Vector (BFV) and Invariant Moment Vector (IMV) based digital representations of the gene expression patterns in finding biologically meaningful patterns was compared for a small (226 images) and a large (1819 images) dataset. For eachdataset, an ordered list of images, with respect to a query image, was generated to identify overlapping and similar gene expression patterns, in a manner comparable to what a developmental biologist might do. The results showed that the BFV representation consistently outperforms the IMV representation in finding biologically meaningful matches when spatial overlap of the gene expression pattern and the genes involved are considered. Furthermore, we explored the value of conducting image-content based searches in a dataset where individual expression components (or domains) of multi-domain expression patterns were also included separately. We found that this technique improves performance of both IMV and BFV based searches. We conclude that the BFV representation consistently produces a more extensive and better list of biologically useful patterns than the IMV representation. The high quality of results obtained scales well as the search database becomes larger, which encourages efforts to build automated image query and retrieval systems for spatial gene expression patterns.\n",
      "Molecular developmental studies of fly and mouse embryos have shown that the identity of individual body segments is controlled by a suite of homeobox-containing genes called the Hox cluster. To examine the conservation of this patterning mechanism in other segmented phyla, we here describe four Hox gene homologs isolated from glossiphoniid leeches of the genus Helobdella. Based on sequence similarity and phylogenetic analysis, the leech genes Lox7, Lox6, Lox20, and Lox5 are deemed to be orthologs of the Drosophila genes lab, Dfd, Scr, and Antp, respectively. Sequence similarities between Lox5 and Antp outside the homeodomain and phylogenetic reconstructions suggest that the Antennapedia family of Hox genes (as defined by Bürglin, 1994) had already expanded to include at least two discrete Antp and Ubx/abdA precursors prior to the annelid/arthropod divergence. In situ hybridization reveals that the four Lox genes described in this study are all expressed at high levels within the segmented portion of the central nervous system (CNS), with variable levels of expression in the segmental mesoderm. Little or no expression was seen in peripheral ectoderm or endoderm, or in the unsegmented head region (prostomium). Each Lox gene has a distinct anterior expression boundary within one of the four rostral segments, and the anterior-posterior (AP) order of these expression boundaries is identical to that reported for the orthologous Hox gene products in fly and mouse. This finding supports the idea that the process of AP axis differentiation is conserved among the higher metazoan phyla with respect to the regional expression of individual Hox genes along that axis. One unusual feature of leech Hox genes is the observation that some genes are only expressed during later development -- beginning at the time of terminal cell differentiation -- whereas others begin expression at a much earlier stage, and their RNA ceases to be detectable shortly after the onset of expression of the 'late' Hox genes. The functional significance of this temporal disparity is unknown, but it is noteworthy that only the two 'early' Hox genes display high levels of mesodermal expression.\n",
      "Morphogen gradients determine a range of cell fates by specifying multiple transcriptional threshold responses. In the dorsal ectoderm of the Drosophila embryo, a BMP gradient is translated into an activated Smad transcription factor gradient, which elicits at least three threshold responses - high, intermediate and low. However, the mechanism underlying differential response to Dpp is poorly understood, due in part to the insufficient number of well-studied target genes. We analyzed the regulation of the C15 gene, which can be activated in cells containing intermediate levels of Dpp. We show that C15 expression requires both dpp and zen, thus forming a genetic feed-forward loop. The C15 regulatory element contains clusters of Smad- and Zen-binding sites in close proximity. Mutational analysis shows that the number of intact Smad- and Zen-binding sites is essential for the C15 transcriptional response, and that the spatial limits of C15 expression are established through a repression mechanism in the dorsolateral cells of the embryo. Thus, the combinatorial action of Smad and Zen activators bound to a number of adjacent sites, and competing negative cues allows for proper gene response to lower than peak levels of the Dpp morphogen.\n",
      "Morphogenesis of epithelial tissues relies on the precise developmental control of cell polarity and architecture. In the early Drosophila embryo, the primary epithelium forms during cellularisation, following a tightly controlled genetic programme where specific sets of genes are upregulated. Some of them, for example, control membrane invagination between the nuclei anchored at the apical surface of the syncytium. We used microarrays to describe the global programme of gene expression underlying cellularisation and identified distinct classes of upregulated genes during this process. Fifty-seven genes were then tested functionally by RNAi. We found six genes affecting various aspects of cellular architecture: membrane growth, organelle transport or organisation and junction assembly. We focus here on charleston (char), a new regulator of nuclear morphogenesis and of apical nuclear anchoring. In char-depleted embryos, the nuclei fail to maintain their elongated shape and, instead, become rounded. In addition, together with a disruption of the centrosome-nuclear envelope interaction, the nuclei lose their regular apical anchoring. These nuclear defects perturb the regular columnar organisation of epithelial cells in the embryo. Although microtubules are required for both nuclear morphogenesis and anchoring, char does not control microtubule organisation and association to the nuclear envelope. We show that Char is lipid anchored at the nuclear envelope by a farnesylation group, and localises at the inner nuclear membrane together with Lamin. Our data suggest that Char forms a scaffold that regulates nuclear architecture to constrain nuclei in tight columnar epithelial cells. The upregulation of Char during cellularisation and gastrulation reveals the existence of an as yet unknown developmental control of nuclear morphology and anchoring in embryonic epithelia.\n",
      "Morphogenetic movements are closely regulated by the expression of developmental genes. Here I examine whether developmental gene expression can in turn be mechanically regulated by morphogenetic movements. I have analyzed the effects of mechanical stress on the expression of Twist, which is normally expressed only in the most ventral cells of the cellular blastoderm embryo under the control of the Dorsal morphogen gradient. At embryogenesis gastrulation (stage 7), Twist is also expressed in the anterior foregut and stomodeal primordia. Submitting the early Drosophila embryo to a transient 10% uniaxial lateral deformation induces the ectopic expression of Twist around the entire dorsal-ventral axis and results in the ventralization of the embryo. This induction is independent of the Dorsal gradient and is triggered by mechanically induced Armadillo nuclear translocation. I also show that Twist is not expressed in the anterior foregut and stomodeal primordia at stage 7 in mutants that block the morphogenetic movement of germ-band extension. Because I can rescue the mutants with gentle compression of these cells, my interpretation is that the stomodeal-cell compression normally caused by the germ-band extension induces the expression of Twist. Correspondingly, laser ablation of dorsal cells in wild-type embryos relaxes stomodeal cell compression and reduces Twist expression in the stomodeal primordium. I also demonstrate that the induction of Twist in these cells depends on the nuclear translocation of Armadillo. I propose that anterior-gut formation is mechanically induced by the movement of germ-band extension through the induction of Twist expression in stomodeal cells.\n",
      "Morphogens are secreted signaling molecules that form concentration gradients and control cell fate in developing tissues. During development, it is essential that morphogen range is strictly regulated in order for correct cell type specification to occur. One of the best characterized morphogens is Drosophila Decapentaplegic (Dpp), a BMP signaling molecule that patterns the dorsal ectoderm of the embryo by activating the Mad and Medea (Med) transcription factors. We demonstrate that there is a spatial and temporal expansion of the expression patterns of Dpp target genes in SUMO pathway mutant embryos. We identify Med as the primary SUMOylation target in the Dpp pathway, and show that failure to SUMOylate Med leads to the increased Dpp signaling range observed in the SUMO pathway mutant embryos. Med is SUMO modified in the nucleus, and we provide evidence that SUMOylation triggers Med nuclear export. Hence, Med SUMOylation provides a mechanism by which nuclei can continue to monitor the presence of extracellular Dpp signal to activate target gene expression for an appropriate duration. Overall, our results identify an unusual strategy for regulating morphogen range that, rather than impacting on the morphogen itself, targets an intracellular transducer.\n",
      "Most cell-specific enhancers are thought to lack an inherent organization, with critical binding sites distributed in a more or less random fashion. However, there are examples of fixed arrangements of binding sites, such as helical phasing, that promote the formation of higher-order protein complexes on the enhancer DNA template. Here, we investigate the regulatory 'grammar' of nearly 100 characterized enhancers for developmental control genes active in the early Drosophila embryo. The conservation of grammar is examined in seven divergent Drosophila genomes. Linked binding sites are observed for particular combinations of binding motifs, including Bicoid-Bicoid, Hunchback-Hunchback, Bicoid-Dorsal, Bicoid-Caudal and Dorsal-Twist. Direct evidence is presented for the importance of Bicoid-Dorsal linkage in the integration of the anterior-posterior and dorsal-ventral patterning systems. Hunchback-Hunchback interactions help explain unresolved aspects of segmentation, including the differential regulation of the eve stripe 3 + 7 and stripe 4 + 6 enhancers. We also present evidence that there is an under-representation of nucleosome positioning sequences in many enhancers, raising the possibility for a subtle higher-order structure extending across certain enhancers. We conclude that grammar of gene control regions is pervasively used in the patterning of the Drosophila embryo.\n",
      "Most eukaryotic genomes include a substantial repeat-rich fraction termed heterochromatin, which is concentrated in centric and telomeric regions. The repetitive nature of heterochromatic sequence makes it difficult to assemble and analyze. To better understand the heterochromatic component of the Drosophila melanogaster genome, we characterized and annotated portions of a whole-genome shotgun sequence assembly. WGS3, an improved whole-genome shotgun assembly, includes 20.7 Mb of draft-quality sequence not represented in the Release 3 sequence spanning the euchromatin. We annotated this sequence using the methods employed in the re-annotation of the Release 3 euchromatic sequence. This analysis predicted 297 protein-coding genes and six non-protein-coding genes, including known heterochromatic genes, and regions of similarity to known transposable elements. Bacterial artificial chromosome (BAC)-based fluorescence in situ hybridization analysis was used to correlate the genomic sequence with the cytogenetic map in order to refine the genomic definition of the centric heterochromatin; on the basis of our cytological definition, the annotated Release 3 euchromatic sequence extends into the centric heterochromatin on each chromosome arm. Whole-genome shotgun assembly produced a reliable draft-quality sequence of a significant part of the Drosophila heterochromatin. Annotation of this sequence defined the intron-exon structures of 30 known protein-coding genes and 267 protein-coding gene models. The cytogenetic mapping suggests that an additional 150 predicted genes are located in heterochromatin at the base of the Release 3 euchromatic sequence. Our analysis suggests strategies for improving the sequence and annotation of the heterochromatic portions of the Drosophila and other complex genomes.\n",
      "Most of our knowledge about the mechanisms of segmentation in arthropods comes from work on Drosophila melanogaster. In recent years it has become clear that this mechanism is far from universal, and different arthropod groups have distinct modes of segmentation that operate through divergent genetic mechanisms. We review recent data from a range of arthropods, identifying which features of the D. melanogaster segmentation cascade are present in the different groups, and discuss the evolutionary implications of their conserved and divergent aspects. A model is emerging, although slowly, for the way that arthropod segmentation mechanisms have evolved.\n",
      "Mutations of the homeotic gene proboscipedia (pb) of Drosophila cause striking transformations of the adult mouthparts, to legs or antennae. We report here an analysis of the gene structure of pb. Coding sequences across a 34 kb interval yield, by alternative splicing, four identified mRNA forms which differ immediately upstream of the homeobox. As a consequence, the homeodomain is expected to reside in four different contexts in the predicted protein isoforms. Mammalian homologs of pb, human HOX-2H and murine Hox-2.8, were identified based on the similarities of their homeodomains (95% identity) and several other conserved motifs. Examination of a collection of pb mutant alleles with antisera directed against the N-terminal region, the center or the C-terminal region of the protein showed that, surprisingly, several partial loss-of-function pb alleles appear to generate partially functional proteins truncated at their C-termini. This suggests that a significant portion of the protein contributes quantitatively to pb function, but is partially dispensable. Finally, evolutionary considerations suggest that pb may be one of several ancient genes which preceded the process yielding the modern homeotic gene complexes.\n",
      "NURF is a conserved higher eukaryotic ISWI-containing chromatin remodeling complex that catalyzes ATP-dependent nucleosome sliding. By sliding nucleosomes, NURF is able to alter chromatin dynamics to control transcription and genome organization. Previous biochemical and genetic analysis of the specificity-subunit of Drosophila NURF (Nurf301/Enhancer of Bithorax (E(bx)) has defined NURF as a critical regulator of homeotic, heat-shock and steroid-responsive gene transcription. It has been speculated that NURF controls pathway specific transcription by co-operating with sequence-specific transcription factors to remodel chromatin at dedicated enhancers. However, conclusive in vivo demonstration of this is lacking and precise regulatory elements targeted by NURF are poorly defined. To address this, we have generated a comprehensive map of in vivo NURF activity, using MNase-sequencing to determine at base pair resolution NURF target nucleosomes, and ChIP-sequencing to define sites of NURF recruitment. Our data show that, besides anticipated roles at enhancers, NURF interacts physically and functionally with the TRF2/DREF basal transcription factor to organize nucleosomes downstream of active promoters. Moreover, we detect NURF remodeling and recruitment at distal insulator sites, where NURF functionally interacts with and co-localizes with DREF and insulator proteins including CP190 to establish nucleosome-depleted domains. This insulator function of NURF is most apparent at subclasses of insulators that mark the boundaries of chromatin domains, where multiple insulator proteins co-associate. By visualizing the complete repertoire of in vivo NURF chromatin targets, our data provide new insights into how chromatin remodeling can control genome organization and regulatory interactions.\n",
      "Network motifs provided a\"conceptual tool\"for understanding the functional principles of biological networks, but such motifs have primarily been used to consider static network structures. Static networks, however, cannot be used to reveal time- and region-specific traits of biological systems. To overcome this limitation, we proposed the concept of a \"spatiotemporal network motif,\"a spatiotemporal sequence of network motifs of sub-networks which are active only at specific time points and body parts. On the basis of this concept, we analyzed the developmental gene regulatory network of the Drosophila melanogaster embryo. We identified spatiotemporal network motifs and investigated their distribution pattern in time and space. As a result, we found how key developmental processes are temporally and spatially regulated by the gene network. In particular, we found that nested feedback loops appeared frequently throughout the entire developmental process. From mathematical simulations, we found that mutual inhibition in the nested feedback loops contributes to the formation of spatial expression patterns. Taken together, the proposed concept and the simulations can be used to unravel the design principle of developmental gene regulatory networks.\n",
      "Networks of regulatory relations between transcription factors (TF) and their target genes (TG)- implemented through TF binding sites (TFBS)- are key features of biology. An idealized approach to solving such networks consists of starting from a consensus TFBS or a position weight matrix (PWM) to generate a high accuracy list of candidate TGs for biological validation. Developing and evaluating such approaches remains a formidable challenge in regulatory bioinformatics. We perform a benchmark study on 34 Drosophila TFs to assess existing TFBS and cis-regulatory module (CRM) detection methods, with a strong focus on the use of multiple genomes. Particularly, for CRM-modelling we investigatethe addition of orthologous sites to a known PWM to construct phyloPWMs and we assess the added value of phylogenentic footprinting to predict contextual motifs around known TFBSs. For CRM-prediction, we compare motif conservation with network-level conservation approaches across multiple genomes. Choosing the optimal training and scoring strategies strongly enhances the performance of TG prediction for more than half of the tested TFs. Finally, we analyse a 35(th) TF, namely Eyeless, and find a significant overlap between predicted TGs and candidate TGs identified by microarray expression studies. In summary we identify several ways to optimize TF-specific TG predictions, some of which can be applied to all TFs, and others that can be applied only to particular TFs. The ability to model known TF-TG relations, together with the use of multiple genomes, results in a significant step forward in solving the architecture of gene regulatory networks.\n",
      "Neurotactin (NRT), a member of the cholinesterase-homologous protein family, is a heterophilic cell adhesion molecule that is required for proper axon guidance during Drosophila development. In this study, we identify amalgam (AMA), a member of the immunoglobulin superfamily, as a ligand for the NRT receptor. Using transfected Schneider 2 cells and embryonic primary cultures, we demonstrate that AMA is a secreted protein. Furthermore, AMA is necessary for NRT-expressing cells both to aggregate with themselves and to associate with embryonic primary culture cells. Aggregation assays performed with truncated NRT molecules reveal that the integrity of the cholinesterase-like extracellular domain was not required either for AMA binding or for adhesion, with only amino acids 347-482 of the extracellular domain being necessary for both activities. Moreover, the NRT cytoplasmic domain is required for NRT-mediated adhesion, although not for AMA binding. Using an ama-deficient stock, we find that ama function is not essential for viability. Pupae deficient for ama do exhibit defasciculation defects of the ocellar nerves similar to those found in nrt mutants.\n",
      "One of the main challenges in evolutionary biology is to identify the molecular changes that underlie phenotypic differences that are of evolutionary significance. Comparative studies of early development have shown that changes in the spatio-temporal use of regulatory genes, as well as changes in the specificity of regulatory proteins, are correlated with important differences in morphology between phylogenetically distant species. However, it is not known how such changes take place in natural populations, and whether they result from a single, or many small, additive events. Extending this approach to the study of development of closely related species promises to enrich this debate.\n",
      "One of the most important questions in biology is how transcription factors (TFs) and cofactors control enhancer function and thus gene expression. Enhancer activation usually requires combinations of several TFs, indicating that TFs function synergistically and combinatorially. However, while TF binding has been extensively studied, little is known about how combinations of TFs and cofactors control enhancer function once they are bound. It is typically unclear which TFs participate in combinatorial enhancer activation, whether different TFs form functionally distinct groups, or if certain TFs might substitute for each other in defined enhancer contexts. Here we assess the potential regulatory contributions of TFs and cofactors to combinatorial enhancer control with enhancer complementation assays. We recruited GAL4-DNA-binding-domain fusions of 812 Drosophila TFs and cofactors to 24 enhancer contexts and measured enhancer activities by 82,752 luciferase assays in S2 cells. Most factors were functional in at least one context, yet their contributions differed between contexts and varied from repression to activation (up to 289-fold) for individual factors. Based on functional similarities across contexts, we define 15 groups of TFs that differ in developmental functions and protein sequence features. Similar TFs can substitute for each other, enabling enhancer re-engineering by exchanging TF motifs, and TF-cofactor pairs cooperate during enhancer control and interact physically. Overall, we show that activators and repressors can have diverse regulatory functions that typically depend on the enhancer context. The systematic functional characterization of TFs and cofactors should further our understanding of combinatorial enhancer control and gene regulation.\n",
      "Pattern formation in Drosophila is initiated by a small set of asymmetrically distributed maternal transcription factors that act as graded morphogens along the anterior-posterior and the dorsal-ventral axes of the embryo. Recent progress in the field provides first insight into the molecular mechanisms by which long-range positional information in the egg causes a series of localized zygotic transcription factors to position the developmental fate along the blastoderm.\n",
      "Pattern formation in the developing embryo relies on key regulatory molecules, many of which are distributed in concentration gradients. For example, a gradient of BMP specifies cell fates along the dorsoventral axis in species ranging from flies to mammals. In Drosophila, a gradient of the BMP molecule Dpp gives rise to nested domains of target gene expression in the dorsal regionof the embryo; however, the mechanisms underlying the differential response are not well understood, partly owing to an insufficient number of well-studied targets. Here we analyze how the Dpp gradient regulates expression of pannier (pnr), a candidate low-level Dpp target gene. We predicted that the pnr enhancer would contain high-affinity binding sites for the Dpp effector Smad transcription factors, which would be occupied in the presence of low-level Dpp. Unexpectedly, the affinity of Smad sites in the pnr enhancer was similar to those in the Race enhancer, a high-level Dpp target gene, suggesting that the affinity threshold mechanism plays a minimal role in the regulation of pnr. Our results indicate that a mechanism involving a conserved bipartite motif that is predicted to bind a homeodomain factor in addition to Smads and the Brinker repressor, establishes the pnr expression domain. Furthermore, the pnr enhancer has a highly complex structure that integrates cues not only from the dorsoventral axis, but also from the anteroposterior and terminal patterning systems in the blastoderm embryo.\n",
      "Pattern formation in the dorsal region of the Drosophila embryo depends on the activity of a small group of zygotically acting genes. dpp, a key gene in this group, encodes a TGF-beta-like product (Dpp) that has been proposed to function as a morphogen with peak levels of Dpp-specifying amnioserosa, the dorsal-most cell type, and lower Dpp levels specifying dorsal ectoderm. The short gastrulation gene also contributes to patterning the dorsal region, but unlike the other genes involved in this process, sog activity is only required in ventral cells. Genetic evidence indicates that sog functions to antagonize dpp activity. In this report we present further phenotypic characterization of sog mutant embryos in dorsal and lateral regions and describe the cloning of the sog locus. sog is expressed in a broad lateral stripe of cells that abuts the dorsal territory of dpp-expressing cells. sog is predicted to encode a protein with an internal signal sequence and a large extracellular domain containing four repeats of a novel motif defined by the spacing of 10 cysteine residues that is distantly related to domains present in thrombospondin and procollagen. We propose that one or more of these cysteine repeats can be liberated by proteolytic cleavage of the primary Sog protein. These putative soluble Sog peptides may then diffuse into the dorsal region to antagonize the activity of Dpp, leading to the subdivision of the dorsal territory into amnioserosa and dorsal ectoderm.\n",
      "Patterning of the Drosophila embryo requires not only the proper activation of determinants at specific times, but also their restriction to specific places. Recent studies on transcriptional repressors show how they delimit the gene expression patterns to ensure normal development.\n",
      "Patterning of the dorsal-ventral axis in the early Drosophila embryo depends on the nuclear distribution of the Dorsal transcription factor. Using live two-photon light-sheet microscopy, we quantified the nuclear Dorsal gradient in space and time and found that its amplitude and basal levels display oscillations throughout early embryonic development. These dynamics raise questions regarding how cells can reproducibly establish patterns of gene expression from a rapidly varying signal. We therefore quantified domains of Dorsal target genes, discovering their expression patterns are also dynamic. Computational modeling of this system reveals a correlation between Dorsal gradient dynamics and changes in target gene expression and suggests that these dynamics, together with time averaging of noise, results in the formation of graded gene expression borders in regions where the gradient is nearly flat. We propose that mRNA levels remain plastic during transient signaling events, allowing tissues to refine patterns in the face of genetic or environmental variation.\n",
      "Patterning of the terminal regions of the Drosophila embryo is achieved by an exquisitely regulated signal that passes between the follicle cells of the ovary, and the developing embryo. This pathway, however, is missing or modified in other insects. Here we trace the evolution of this pathway by examining the origins and expression of its components. The three core components of this pathway: trunk, torso and torso-like have different evolutionary histories and have been assembled step-wise to form the canonical terminal patterning pathway of Drosophila and Tribolium. Trunk, torso and a gene unrelated to terminal patterning, prothoraciotrophic hormone (PTTH), show an intimately linked evolutionary history, with every holometabolous insect, except the honeybee, possessing both PTTH and torso genes. Trunk is more restricted in its phylogenetic distribution, present only in the Diptera and Tribolium and, surprisingly, in the chelicerate Ixodes scapularis, raising the possibility that trunk and torso evolved earlier than previously thought. In Drosophila torso-like restricts the activation of the terminal patterning pathway to the poles of the embryo. Torso-like evolved in the pan-crustacean lineage, but based on expression of components of the canonical terminal patterning system in the hemimetabolous insect Acyrthosiphon pisum and the holometabolous insect Apis mellifera, we find that the canonical terminal-patterning system is not active in these insects. We therefore propose that the ancestral function of torso-like is unrelated to terminal patterning and that torso-like has become co-opted into terminal patterning in the lineage leading to Coleoptera and Diptera. We also show that this co-option has not resulted in changes to the molecular function of this protein. Torso-like from the pea aphid, honeybee and Drosophila, despite being expressed in different patterns, are functionally equivalent. We propose that co-option of torso-like into restricting the activity of trunk and torso facilitated the final step in the evolution of this pathway; the capture of transcriptional control of target genes such as tailless and huckebein by this complex and novel patterning pathway.\n",
      "Phylostratigraphy is a method for dating the evolutionary emergence of a gene or gene family by identifying its homologs across the tree of life, typically by using BLAST searches. Applying this method to all genes in a species, or genomic phylostratigraphy, allows investigation of genome-wide patterns in new gene origination at different evolutionary times and thus has been extensively used. However, gene age estimation depends on the challenging task of detecting distant homologs via sequence similarity, which is expected to have differential accuracies for different genes. Here, we evaluate the accuracy of phylostratigraphy by realistic computer simulation with parameters estimated from genomic data, and investigate the impact of its error on findings of genome evolution. We show that 1) phylostratigraphy substantially underestimates gene age for a considerable fraction of genes, 2) the error is especially serious when the protein evolves rapidly, is short, and/or its most conserved block of sites is small, and 3) these errors create spurious nonuniform distributions of various gene properties among age groups, many of which cannot be predicted a priori. Given the high likelihood that conclusions about gene age are faulty, we advocate the use of realistic simulation to determine if observations from phylostratigraphy are explainable, at least qualitatively, by a null model of biased measurement, and in all cases, critical evaluation of results.\n",
      "Polycomb-group (PcG) and Trithorax-group proteins together form a maintenance machinery that is responsible for stable heritable states of gene activity. While the best-studied target genes are the Hox genes of the Antennapedia and Bithorax complexes, a large number of key developmental genes are also Polycomb (Pc) targets, indicating a widespread role for this maintenance machinery in cell fate determination. We have studied the linkage between the binding of PcG proteins and the developmental regulation of gene expression using whole-genome mapping to identify sites bound by the PcG proteins, Pc and Pleiohomeotic (Pho), in the Drosophila embryo and in a more restricted tissue, the imaginal discs of the third thoracic segment. Our data provide support for the idea that Pho is a general component of the maintenance machinery, since the majority of Pc targets are also associated with Pho binding. We find, in general, considerable developmental stability of Pc and Pho binding at target genes and observe that Pc/Pho binding can be associated with both expressed and inactive genes. In particular, at the Hox complexes, both active and inactive genes have significant Pc and Pho binding. However, in comparison to inactive genes, the active Hox genes show reduced and altered binding profiles.During development, Pc target genes are not simply constantly associated with Pc/Pho binding, and we identify sets of genes with clear differential binding between embryo and imaginal disc. Using existing datasets, we show that for specific fate-determining genes of the haemocyte lineage, the active state is characterised by lack of Pc binding. Overall, our analysis suggests a dynamic relationship between Pc/Pho binding and gene transcription. Pc/Pho binding does not preclude transcription, but levels of Pc/Pho binding change during development, and loss of Pc/Pho binding can be associated with both stable gene activity and inactivity.\n",
      "Population genetic theory predicts that maternal effect genes will evolve differently than genes expressed in both sexes because selection is only half as effective on autosomal genes expressed in one sex but not the other. Here, we use sequences of the tandem gene duplicates, bicoid (bcd) and zerknüllt (zen), to test the prediction that, with similar coefficients of purifying selection, a maternal effect gene evolves more rapidly than a zygotic gene because of this reduction in selective constraint. We find that the maternal effect gene, bcd, is evolving more rapidly than zygotically expressed, zen, providing the first direct confirmation of this prediction of maternal effect theory from molecular evidence. Our results extend current explanations for the accelerated rate of bcd evolution by providing an evolutionary mechanism, relaxed selective constraint, that allows bcd the evolutionary flexibility to escape the typical functional constraints of early developmental genes. We discuss general implications of our findings for the role of maternal effect genes in early developmental patterning.\n",
      "Positional information in the dorsoventral axis of the Drosophila embryo is encoded by a BMP activity gradient formed by synergistic signaling between the BMP family members Decapentaplegic (DPP) and Screw (SCW). short gastrulation (sog), which is functionally homologous to Xenopus Chordin, is expressed in the ventrolateral regions of the embryo and has been shown to act as a local antagonist of BMP signaling. Here we demonstrate that SOG has a second function, which is to promote BMP signaling on the dorsal side of the embryo. We show that a weak, homozygous-viable sog mutant is enhanced to lethality by reduction in the activities of the Smad family members Mad or Medea, and that the lethality is caused by defects in the molecular specification and subsequent cellular differentiation of the dorsal-most cell type, the amnioserosa. While previous data had suggested that the negative function of SOG is directed against SCW, we present data that suggests that the positive activity of SOG is directed towards DPP. We demonstrate that Chordin shares the same apparent ligand specificity as does SOG, preferentially inhibiting SCW but not DPP activity. However, in Drosophila assays Chordin does not have the same capacity to elevate BMP signaling as does SOG, identifying a functional difference in the otherwise well conserved process of dorsoventral pattern formation in arthropods and chordates.\n",
      "Previously, we mapped quantitative trait loci (QTL) affecting response to short-term selection for abdominal bristle number to seven suggestive regions that contain loci involved in bristle development and/or that have adult bristle number mutant phenotypes, and are thus candidates for bristle number QTL in natural populations. To test the hypothesis that the factors contributing to selection response genetically interact with these candidate loci, high and low chromosomes from selection lines were crossed to chromosomes containing wild-type or mutant alleles at the candidate loci, and the numbers of bristles were recorded in trans heterozygotes. Quantitative failure to complement, detected as a significant selection line*cross effect by analysis of variance, can be interpreted as evidence for allelism or epistasis between the factors on selected chromosomes and the candidate loci. Mutations at some candidate loci (bb, emc, h, Dl, Hairless) showed strong interactions with selected chromosomes, whereas others interacted weakly (ASC, abd, Scr) or not at all (N, mab, E(spl)). These results support the hypothesis that some candidate loci, initially identified through mutations of large effect on bristle number, either harbor or are close members in the same genetic pathway as variants that contribute to standing variation in bristle number.\n",
      "Quantitative studies of embryogenesis require the ability to monitor pattern formation and morphogenesis in large numbers of embryos, at multiple time points and in diverse genetic backgrounds. We describe a simple approach that greatly facilitates these tasks for Drosophila melanogaster embryos, one of the most advanced models of developmental genetics. Based on passive hydrodynamics, we developed a microfluidic embryo-trap array that can be used to rapidly order and vertically orient hundreds of embryos. We describe the physical principles of the design and used this platform to quantitatively analyze multiple morphogen gradients in the dorsoventral patterning system. Our approach can also be used for live imaging and, with slight modifications, could be adapted for studies of pattern formation and morphogenesis in other model organisms.\n",
      "RNApolII-dependent transcription is repressed in primordial germ cells of many animals during early development and is thought to be important for maintenance of germline fate by preventing somatic differentiation. Germ cell transcriptional repression occurs concurrently with inhibition of phosphorylation in the carboxy-terminal domain (CTD) of RNApolII, as well as with chromatin remodeling. The precise mechanisms involved are unknown. Here, we present evidence that a noncoding RNA transcribed by the gene polar granule component (pgc) regulates transcriptional repression in Drosophila germ cells. Germ cells lacking pgc RNA express genes important for differentiation of nearby somatic cells and show premature phosphorylation of RNApolII. We further show that germ cells lacking pgc show increased levels of K4, but not K9 histone H3 methylation, and that the chromatin remodeling Swi/Snf complex is required for a second stage in germ cell transcriptional repression. We propose that a noncoding RNA controls transcription in early germ cells by blocking the transition from preinitiation to transcriptional elongation. We further show that repression of somatic differentiation signals mediated by the Torso receptor-tyrosine kinase is important for germline development.\n",
      "Recent data have demonstrated that vRel, cRel, Dorsal, and NF-kappa B are members of a larger family of DNA-binding regulatory proteins. Rel proteins interact to form homo- and heterodimers that recognize specific sites on DNA, and it is likely that such protein-protein and protein-DNA interactions contribute to proper regulation of target gene expression by these proteins. Here we describe the use of a yeast transcription activation assay to study binding of three Rel family proteins to their native binding sites. These results show that the vRel and cRel proteins recognize two known NF-kappa B binding sites; the Dorsal protein does not recognize NF-kappa B sites, but does recognize related sites upstream of the Drosophila zerknüllt gene. Our experiments demonstrate that the members of this protein family recognize similar, but not identical, sites in the promoters of target genes, and we are able to identify a particular nucleotide that is apparently involved in the DNA-protein interaction. We exploit the properties of LexA fusion proteins to study the dimerization and DNA-contacting domains of cRel. Our results suggest that the cRel protein forms homodimers and that dimer formation may be necessary for cRel to bind DNA. Finally, our results show that transcription activation by these proteins is cooperative; such cooperativity may be important for correct temporal and spatial regulation of target gene expression.\n",
      "Retrogenes are processed copies of genes that are inserted into new genomic regions and that acquire new regulatory elements from the sequences in their surroundings. Here we use a comparative approach of phylogenetic footprinting and a non-comparative approach of measuring motif over-representation in retrogenes in order to describe putative elements present in cis-regulatory regions of 94 retrogenes recently described in Drosophila. The detailed examination of the motifs found in the core promoter regions of retrogenes reveals an abundance of the DNA replication-related element (DRE), the Initiator (Inr), and a new over-represented motif that we call the GCT motif. Parental genes also show an abundance of DRE and Inr motifs, but these do not seem to have been carried over with retrogenes. In particular, we also examined motifs upstream of retrogenes expressed in adult testis and were able to identify 6 additional over-represented motifs. Comparative analyses provide data on the conservation and origin of some of these motifs and reveal 15 additional conserved motifs in these retrogenes. Some of those conserved motifs are sequences bound by known transcription factors, while others are novel motifs. In this report we provide the first genome-wide data on which specific cis-regulatory regions can be recruited by retrogenes after they are inserted into new coding regions in the genome. Future experiments are needed to determine the function and role of the new elements presented here.\n",
      "Seven zygotically active genes are required for normal patterning of the dorsal 40% of the Drosophila embryo. Among these genes, decapentaplegic (dpp) has the strongest mutant phenotype: in the absence of dpp, all cells in the dorsal and dorsolateral regions of the embryo adopt fates characteristic of more ventrally derived cells (Irish and Gelbart (1987) Genes Dev. 1, 868-879). Here we describe the phenotypes caused by alleles of another of this set of genes, tolloid, and show that tolloid is required for dorsal, but not dorsolateral, pattern. Extragenic suppressors of tolloid mutations were isolated that proved to be mutations that elevate dpp activity. We studied the relationship between tolloid and dpp by analyzing the phenotypes of tolloid embryos with elevated numbers of the dpp gene and found that doubling the dpp+ gene dosage completely suppressed weak tolloid mutants and partially suppressed the phenotypes of tolloid null mutants. We conclude that the function of tolloid is to increase dpp activity. We also examined the effect of doubling dpp+ gene dosage on the phenotypes caused by other mutations affecting dorsal development. Like tolloid, the phenotypes of mutant embryos lacking shrew gene function were suppressed by elevated dpp, indicating that shrew also acts upstream of dpp to increase dpp activity. In contrast, increasing the number of copies of the dpp gene enhanced the short gastrulation (sog) mutant phenotype, causing ventrolateral cells to adopt dorsal fates. This indicates that sog gene product normally blocks dpp activity ventrally. We propose that the tolloid, shrew and sog genes are required to generate a gradient of dpp activity, which directly specifies the pattern of the dorsal 40% of the embryo.\n",
      "Signaling by Decapentaplegic (Dpp), a member of the TGFbeta superfamily of signaling molecules similar to vertebrate BMP2 and BMP4, has been implicated in many developmental processes in Drosophila melanogaster. Notably, Dpp acts as a long-range morphogen during imaginal disc growth and patterning. Genetic approaches led to the identification of a number of gene products that constitute the core signaling pathway. In addition to the ligand-activated heteromeric receptor complex and the signal-transducing intracellular Smad proteins, Dpp signaling requires two nuclear proteins, Schnurri (Shn) and Brinker (Brk), to prime cells for Dpp responsiveness. A complex interplay between the nuclear factors involved in Dpp signaling appears to control the transcriptional readout of the Dpp morphogen gradient. It remains to be seen whether similar molecular mechanisms operate in the nucleus in vertebrate systems.\n",
      "Signaling molecules of the transforming growth factor beta (TGF-beta) family contribute to numerous developmental processes in a variety of organisms. However, our understanding of the mechanisms which regulate the activity of and mediate the response to TGF-beta family members remains incomplete. The product of the Drosophila decapentaplegic (dpp) locus is a well-characterized member of this family. We have taken a genetic approach to identify factors required for TGF-beta function in Drosophila by testing for genetic interactions between mutant alleles of dpp and a collection of chromosomal deficiencies. Our survey identified two deficiencies that act as maternal enhancers of recessive embryonic lethal alleles of dpp. The enhanced individuals die with weakly ventralized phenotypes. These phenotypes are consistent with a mechanism whereby the deficiencies deplete two maternally provided factors required for dpp's role in embryonic dorsal-ventral pattern formation. One of these deficiencies also appears todelete a factor required for dpp function in wing vein formation. These deficiencies remove material from the 54F-55A and 66B-66C polytene chromosomal regions, respectively. As neither of these regions has been previously implicated in dpp function, we propose that each of the deficiencies removes a novel factor or factors required for dpp function.\n",
      "Similar to their human counterparts, the Drosophila Rbf1 and Rbf2 Retinoblastoma family members control cell cycle and developmentally regulated gene expression. Increasing evidence suggests that Rbf proteins rely on multiprotein complexes to control target gene transcription. We show here that the developmentally regulated COP9 signalosome (CSN) physically interacts with Rbf2 during embryogenesis. Furthermore, the CSN4 subunit of the COP9 signalosome co-occupies Rbf target gene promoters with Rbf1 and Rbf2, suggesting an active role for the COP9 signalosome in transcriptional regulation. The targeted knockdown of individual CSN subunits leads to diminished Rbf1 and Rbf2 levels and to altered cell cycle progression. The proteasome-mediated destruction of Rbf1 and Rbf2 is increased in cells and embryos with diminished COP9 activity, suggesting that the COP9 signalosome protects Rbf proteins during embryogenesis. Previous evidence has linked gene activation to protein turnover via the promoter-associated proteasome. Our findings suggest that Rbf repression may similarly involve the proteasome and the promoter-associated COP9 signalosome, serving to extend Rbf protein lifespan and enable appropriate programs of retinoblastoma gene control during development.\n",
      "Smad transcription factors lie at the core of one of the most versatile cytokine signaling pathways in metazoan biology-the transforming growth factor-beta (TGFbeta) pathway. Recent progress has shed light into the processes of Smad activation and deactivation, nucleocytoplasmic dynamics, and assembly of transcriptional complexes. A rich repertoire of regulatory devices exerts control over each step of the Smad pathway. This knowledge is enabling work on more complex questions about the organization, integration, and modulation of Smad-dependent transcriptional programs. We are beginning to uncover self-enabled gene response cascades, graded Smad response mechanisms, and Smad-dependent synexpression groups. Our growing understanding of TGFbeta signaling through the Smad pathway provides general principles for how animal cells translate complex inputs into concrete behavior.\n",
      "Small differences in the levels of an extracellular signaling molecule can specify cell fate during development. Threshold responses are often determined at the level of transcription. Cell-specific and spatially localized patterns of gene expression depend on combinations of sequence-specific activators and repressors that bind to extensive cis-regulatory regions. Different mechanisms for integrating this complex regulatory information are discussed, particularly the role of coregulatory proteins, which are recruited to the DNA template by sequence-specific transcription factors. Recent studies suggest that a growing set of coactivators and corepressors mediate communication between diverse upstream regulatory proteins and the core RNA polymerase II transcription complex.\n",
      "The Bicoid-based anterior patterning system of Drosophila embryogenesis appears to be unique to higher dipterans. A new study suggests how this may have evolved out of an alternative mechanism based on cooperating Orthodenticle and Hunchback proteins, the two mechanisms intersecting at the level of downstream target genes.\n",
      "The DRE/DREF system plays an important role in transcription of DNA replication genes such as those encoding the 180 and 73 kDa subunits of DNA polymerase alpha as well as that for encoding PCNA. In this study, we found two sequences homologous to DRE (5'-TATCGATA-3') in the 5'-flanking region (-370 to -357 with respect to the transcription initiation site) of the D-raf gene and confirmed transcriptional activity through gel mobility shift assays, transient CAT assays, and spatial patterns of lacZ expression in transgenic larval tissues carrying D-raf and lacZ fusion genes. Further, we demonstrated that the D-raf gene is another target of the Zerknüllt (Zen) protein with observation of D-raf repression by Zen protein in cultured cells and its ectopic expression in the dorsal region of the homozygous zen mutant embryo. The evidence of DRE/DREF involvement in regulation of the D-raf gene obtained in this study strongly supports the idea that the DRE/DREF system is responsible for the coordinated regulation of cell proliferation-related genes in Drosophila.\n",
      "The Dorsal morphogen acts as both an activator and a repressor of transcription in the Drosophila embryo to regulate the expression of dorsal/ventral patterning genes. Circumstantial evidence has suggested that Dorsal is an intrinsic activator and that additional factors (corepressors) convert it into a repressor. These corepressors, however, have previously eluded definitive identification. We show here, via the analysis of embryos lacking the maternally encoded Groucho corepressor and via protein-binding assays, that recruitment of Groucho to the template by protein:protein interactions is required for the conversion of Dorsal from an activator to a repressor. Groucho is therefore a critical component of the dorsal/ventral patterning system.\n",
      "The Dorsal morphogen directs formation of the Drosophila dorsoventral axis by both activating and repressing transcription. It contains an N-terminal Rel homology domain (RHD), which is responsible for DNA binding and regulated nuclear import, and a C-terminal domain (CTD) that contains activation and repression motifs. To determine if the RHD has a direct role in transcriptional control, we analyzed a series of RHD mutations in S2 cells and embryos. Two classes ofmutations (termed class I and class II mutations) that alter activation without affecting DNA binding or nuclear import were identified. The two classes appear to define distinct protein interaction surfaces on opposite faces of the RHD. Class I mutations enhance an apparently inhibitory interaction between the RHD and the CTD and eliminate both activation and repression by Dorsal. In contrast, class II mutations result in increased activation in S2 cells but severely decreased activation in embryos and have little effect on repression. Analysis of the cuticles of class II mutant embryos suggests that, in the absence of Dorsal-mediated activation, Dorsal-mediated repression is not sufficient to pattern the embryo. These results provide some of the first evidence that the RHD plays an active role in transcriptional regulation in intact multicellular organisms.\n",
      "The Dorsal morphogen is a transcription factor that activates some genes and represses others to establish multiple domains of gene expression along the dorsal/ventral axis of the early Drosophila embryo. Repression by Dorsal appears to require accessory proteins that bind to corepression elements in Dorsal-dependent regulatory modules called ventral repression regions (VRRs). We have identified a corepression element in decapentaplegic (dpp), a zygotically active gene that is repressed by the Dorsal morphogen. This dpp repression element (DRE) is located within a previously identified VRR and close to essential Dorsal-binding sites. We have purified a factor from Drosophila embryo extracts that binds to the DRE but not to mutant forms of the DRE that fail to support efficient repression. This protein also binds to an apparently essential region in a VRR associated with the zerknüllt (zen) gene. One of the DREs in the dpp VRR overlaps the binding site for a potential activator protein suggesting that one mechanism of ventral repression may be the mutually exclusive binding of repressor and activator proteins. We have found the DRE-binding protein to be identical to NTF-1 (equivalent to Elf-1, the product of the grainyhead gene), a factor originally identified as an activator of the Ultrabithorax and Dopa decarboxylase promoters. NTF-1 mRNA is synthesized during oogenesis and deposited in the developing oocyte where it is available to contribute to ventral repression during early embryogenesis. Previous studies have shown that overexpression of NTF-1 in the postblastoderm embryo results in a phenotype that is consistent with a role for this factor in the repression of dpp later in embryogenesis.\n",
      "The Drosophila Groucho (Gro) protein is the prototype for a large family of corepressors, examples of which are found in most metazoans. This family includes the human transducin-like Enhancer of split (TLE) proteins. As corepressors, Gro/TLE family proteins do not bind to DNA directly, but rather are recruited to the template by DNA-bound repressor proteins. Gro/TLE family proteins are required for many developmental processes, including lateral inhibition, segmentation, sex determination, dorsal/ventral pattern formation, terminal pattern formation, and eye development. These proteins are characterized by a conserved N-terminal glutamine-rich domain and a conserved C-terminal WD-repeat domain. The primary role of the glutamine-rich domain is apparently to mediate tetramerization, while the WD-repeat domain may mediate interactions with DNA-bound repressors. The glutamine rich and WD-repeat domains are separated by a less conserved region containing domains that have been implicated in transcriptional repression and nuclear localization. In addition to encoding full-length Gro/TLE family proteins, most metazoan genomes encode truncated family members that contain the N-terminal oligomerization domain, but lack the C-terminal WD-repeat domain. These truncated proteins may negatively regulate full-length Gro/TLE proteins, perhaps by sequestering them in non-productive complexes. Gro/TLE family proteins probably repress transcription by multiple mechanisms. For example, a glycine/proline-rich domain in the central variable region functions to recruit the histone deacetylase Rpd3 to the template. This histone deacetylase then presumably silences transcription by altering local chromatin structure. Other repression domains in Gro may function in a histone deacetylase-independent manner. Many aspects of Gro/TLE protein function remain to be explored, including the possible post-translational regulation of Gro/TLE activity as well as the mechanisms by which Gro/TLE proteins direct repression at a distance.\n",
      "The Drosophila Toll pathway takes part in both establishment of the embryonic dorsoventral axis and induction of the innate immune response in adults. Upon activation by the cytokine Spätzle, Toll interacts with the adaptor proteins DmMyD88 and Tube and the kinase Pelle and triggers degradation of the inhibitor Cactus, thus allowing the nuclear translocation of the transcription factor Dorsal/Dif. weckle (wek) was previously identified as a new dorsal group gene that encodes a putative zinc finger transcription factor. However, its role in the Toll pathway was unknown. Here, we isolated new wek alleles and demonstrated that cactus is epistatic to wek, which in turn is epistatic to Toll. Consistent with this, Wek localizes to the plasma membrane of embryos, independently of Toll signaling. Wek homodimerizes and associates with Toll. Moreover, Wek binds to and localizes DmMyD88 to the plasma membrane. Thus, Wek acts as an adaptor to assemble/stabilize a Toll/Wek/DmMyD88/Tube complex. Remarkably, unlike the DmMyD88/tube/pelle/cactus gene cassette of the Toll pathway, wek plays a minimal role, if any, in the immune defense against Gram-positive bacteria and fungi. We conclude that Wek is an adaptor to link Toll and DmMyD88 and is required for efficient recruitment of DmMyD88 to Toll. Unexpectedly, wek is dispensable for innate immune response, thus revealing differences in the Toll-mediated activation of Dorsal in the embryo and Dif in the fat body of adult flies.\n",
      "The Drosophila decapentaplegic gene, the Xenopus activin genes and the genes encoding the mouse bone morphogenetic proteins are transforming growth factor-beta-related genes whose roles in development are the focus of current studies. They exhibit elaborate patterns of expression during development, and the protein products have potent effects on the differentiation of specific cell types.\n",
      "The Drosophila engrailed gene product (En) is a homeodomain-containing protein that contributes to segmental patterning. In transfection assays it acts as a transcriptional repressor. We show that En is an active repressor, blocking activation by mammalian and yeast activators that bind to sites some distance away from those bound by En. Active repression is distinct from the effects of passive homeodomain-containing proteins, which repress when competing with activators for binding sites and activate when competing with En. Active repression activity maps outside the En homeodomain, and this activity can be transferred to a heterologous DNA binding domain.\n",
      "The Drosophila eye field that gives rise to the visual system and dorsal head epidermis forms an unpaired anlage located in the dorsal head ectoderm. The eye field expresses and requires both Dpp and EGFR signaling for its development. As shown in previous studies, EGFR is required for cell maintenance in the developing visual system. Dpp initially switches on the early eye genes so and eya in the eye field. Consecutively, highlevels of Dpp in the dorsal midline inhibit these genes and promote development of head epidermis. We show that Dpp negatively regulates EGFR signaling, thereby increasing the amount of cell death in the dorsal midline. By this mechanism, Dpp controls the formation of a bilateral visual system and indirectly modulates cell death, which is essential for normal head morphogenesis. Loss of either Dpp or its downstream target, Zen, abolishes head epidermis fate and leads to the misexpression of dp-ERK in the dorsal midline. The resulting morphological phenotype consists of cyclopia, reduction of cell death, and failure of head involution. Ectopic expression of activated EGFR inhibits the Dpp target race and thereby causes cyclopia and defective head involution. We discuss possible mechanisms of Dpp and EGFR interaction in the embryo.\n",
      "The Drosophila gene bicoid functions as the anterior body pattern organizer of Drosophila. Embryos lacking maternally expressed bicoid fail to develop anterior segments including head and thorax. In wild-type eggs, bicoid mRNA is localized in the anterior pole region and the bicoid protein forms an anterior-to-posterior concentration gradient. bicoid activity is required for transcriptional activation of zygotic segmentation genes and the translational suppression of uniformly distributed maternal caudal mRNA in the anterior region of the embryo. caudal genes as well as other homeobox genes or members of the Drosophila segmentation gene cascade have been found to be conserved in animal evolution. In contrast, bicoid homologs have been identified only in close relatives of the schizophoran fly Drosophila. This poses the question of how the bicoid gene evolved and adopted its unique function in organizing anterior-posterior polarity. We have cloned bicoid from a basal cyclorrhaphan fly, Megaselia abdita (Phoridae, Aschiza), and show that the gene originated from a recent duplication of the direct homolog of the vertebrate gene Hox3, termed zerknüllt, which specifies extraembryonic tissues in insects.\n",
      "The Drosophila gene buttonhead (btd) encodes a zinc-finger protein related to the human transcription factor Sp1. btd is expressed in the syncytial blastoderm embryo in a stripe covering the anlagen of the antennal, intercalary and mandibular head segments. btd has been characterized as a head gap gene, since these segments are deleted in btd mutant embryos. We report here that the cis-acting elements required for btd head stripe expression are contained in a 1 kb DNA fragment, located about 3 kb upstream of the promoter. The four maternal coordinate systems are necessary for correct btd head stripe expression, likely by acting through the 1 kb cis-acting control region. Expression of the btd head stripe depends on the anterior morphogen encoded by the gene bicoid (bcd). bcd-dependent activation also involves the activity of the morphogens of the posterior and dorsoventral systems, hunchback and dorsal, respectively, which act together to control the spatial limits of the expression domain. Finally, the terminal system takes part in the regulation of btd head stripe expression by enhancing activation at low levels of activity and repression at high levels of activity.\n",
      "The Drosophila melanogaster genome was the first metazoan genome to have been sequenced by the whole-genome shotgun (WGS) method. Two issues relating to this achievement were widely debated in the genomics community: how correct is the sequence with respect to base-pair (bp) accuracy and frequency of assembly errors? And, how difficult is it to bring a WGS sequence to the accepted standard for finished sequence? We are now in a position to answer these questions. Our finishing process was designed to close gaps, improve sequence quality and validate the assembly. Sequence traces derived from the WGS and draft sequencing of individual bacterial artificial chromosomes (BACs) were assembled into BAC-sized segments. These segments were brought to high quality, and then joined to constitute the sequence of each chromosome arm. Overall assembly was verified by comparison to a physical map of fingerprinted BAC clones. In the current version of the 116.9 Mb euchromatic genome, called Release 3, the six euchromatic chromosome arms are represented by 13 scaffolds with a total of 37 sequence gaps. We compared Release 3 to Release 2; in autosomal regions of unique sequence, the error rate of Release 2 was one in 20,000 bp. The WGS strategy can efficiently produce a high-quality sequence of a metazoan genome while generating the reagents required for sequence finishing. However, the initial method of repeat assembly was flawed. The sequence we report here, Release 3, is a reliable resource for molecular genetic experimentation and computational analysis.\n",
      "The Drosophila morphogen dorsal, KBF1, NF-kappa B, and the proto-oncogene c-rel belong to the rel family of transcription factors whose function is regulated post-translationally by selective nuclear import. In the early Drosophila embryo, dorsal protein is proposed to be retained in the cytoplasm through its interaction with cactus protein. The maternal dorsal group genes constitute a signal transduction pathway, which results in targeting cytoplasmic dorsal protein into the nuclei of the syncytial blastoderm embryo, in a ventral-to-dorsal gradient. The asymmetric transcriptional regulation of zygotic genes along the dorsoventral axis by the dorsal morphogen gradient establishes embryonic dorsoventral polarity. In the lymphocytes, the functional equivalent of cactus is I kappa B, which appears to retain NF-kappa B in the cytoplasm. This retention is relieved by extracellular signals in tissue culture. NF-kappa B and rel proteins each are known to function as oligomeric complexes. Here we present genetic and biochemical evidence for the existence and functional importance of an oligomeric dorsal complex in vivo.\n",
      "The Drosophila nonreceptor protein tyrosine phosphatase, Corkscrew (Csw), functions positively in multiple receptor tyrosine kinase (RTK) pathways, including signaling by the epidermal growth factor receptor (EGFR). Detailed phenotypic analyses of csw mutations have revealed that Csw activity is required in many of the same developmental processes that require EGFR function. However, it is still unclear where in the signaling hierarchy Csw functions relative to other proteins whose activities are also required downstream of the receptor. To address this issue, genetic interaction experiments were performed to place csw gene activity relative to the EGFR, spitz (spi), rhomboid (rho), daughter of sevenless (DOS), kinase-suppressor of ras (ksr), ras1, D-raf, pointed (pnt), and moleskin. We followed the EGFR-dependent formation of VA2 muscle precursor cells as a sensitive assay for these genetic interaction studies. First, we established that Csw has a positive function during mesoderm development. Second, we found that tissue-specific expression of a gain-of-function csw construct rescues loss-of-function mutations in other positive signaling genes upstream of rolled (rl)/MAPK in the EGFR pathway. Third, we were able to infer levels of EGFR signaling in various mutant backgrounds during myogenesis. This work extends previous studies of Csw during Torso and Sevenless RTK signaling to include an in-depth analysis of the role of Csw in the EGFR signaling pathway.\n",
      "The Drosophila polypyrimidine tract-binding protein (dmPTB or hephaestus) plays an important role during embryogenesis. A loss of function mutation, heph(03429), results in varied defects in embryonic developmental processes, leading to embryonic lethality. However, the suite of molecular functions that are disrupted in the mutant remains unknown. We have used an unbiased high throughput sequencing approach to identify transcripts that are misregulated in this mutant. Misregulated transcripts show evidence of significantly altered patterns of splicing (exon skipping, 5' and 3' splice site switching), alternative 5' ends, and mRNA level changes (up and down regulation). These findings are independently supported by reverse-transcription-polymerase chain reaction (RT-PCR) analysis and in situ hybridization. We show that a group of genes, such as Zerknüllt, z600 and screw are among the most upregulated in the mutant and have been functionally linked to dorso-ventral patterning and/or dorsal closure processes. Thus, loss of dmPTB function results in specific misregulated transcripts, including those that provide the missing link between the loss of dmPTB function and observed developmental defects in embryogenesis. This study provides the first comprehensive repertoire of genes affected in vivo in the heph mutant in Drosophila and offers insight into the role of dmPTB during embryonic development.\n",
      "The Drosophila sex-determination switch gene Sex-lethal (Sxl) and the X-chromosome signal element genes (XSEs) that induce the female-specific expression of Sxl are transcribed extremely early in development when most of the genome of this organism is still silent. The DNA sequence CAGGTAG had been implicated in this pre-cellular blastoderm activation of sex-determination genes. A genome-wide computational search, reported here, suggested that CAGGTAG is not specific to early sex-determination genes, since it is over-represented upstream of most genes that are transcribed pre-cellular blastoderm, not just those involved in sex determination. The same search identified similarly over-represented, one-base-pair degenerate sequences as possible functional synonyms of CAGGTAG. We call these heptamers collectively, the TAGteam. Relevance of the TAGteam sequences to pre-cellular blastoderm transcription was established through analysis of TAGteam changes in Sxl, scute (an XSE), and the ;ventral repression element' of the pattern-formation gene zerknüllt. Decreasing the number of TAGteam sites retarded the onset of pre-blastoderm transcription, whereas increasing their number correlated with an advanced onset. Titration of repressors was thought to be the rate-limiting step determining the onset of such early transcription, but this TAGteam dose effect shows that activators must also play an important role in the timing of pre-blastoderm gene expression.\n",
      "The Drosophila short gastrulation gene (sog) encodes a large extracellular protein (Sog) that inhibits signaling by BMP-related ligands. Sog and its vertebrate counterpart Chordin contain four copies of a cysteine repeat (CR) motif defined by 10 cysteine residues spaced in a fixed pattern and a tryptophan residue situated between the first two cysteines. Here we present a structure-function analysis of the CR repeats in Sog, using a series of deletion and point mutation constructs, as well as constructs in which CR domains have been swapped. This analysis indicates that the CR domains are individually dispensable for Sog function but that they are not interchangeable. These studies reveal three different types of Sog activity: intact Sog, which inhibits signaling mediated by the ligand Glass bottom boat (Gbb), a more broadly active class of BMP antagonist referred to as Supersog, and a newly identified activity, which may promote rather than inhibit BMP signaling. Analysis of the activities of CR swap constructs indicates that the CR domains are required for full activity of the various forms of Sog but that the type of Sog activity is determined primarily by surrounding protein sequences. Cumulatively, our analysis suggests that CR domains interact physically with adjacent protein sequences to create forms of Sog with distinct BMP modulatory activities.\n",
      "The Drosophila trithorax group gene brahma (brm) encodes the ATPase subunit of a 2-MDa chromatin-remodeling complex. brm was identified in a screen for transcriptional activators of homeotic genes and subsequently shown to play a global role in transcription by RNA polymerase II. To gain insight into the targeting, function, and regulation of the BRM complex, we screened for mutations that genetically interact with a dominant-negative allele of brm (brm(K804R)). We first screened for dominant mutations that are lethal in combination with a brm(K804R) transgene under control of the brm promoter. In a distinct but related screen, we identified dominant mutations that modify eye defects resulting from expression of brm(K804R) in the eye-antennal imaginal disc. Mutations in three classes of genes were identified in our screens: genes encoding subunits of the BRM complex (brm, moira, and osa), other proteins directly involved in transcription (zerknullt and RpII140), and signaling molecules (Delta and vein). Expression of brm(K804R) in the adult sense organ precursor lineage causes phenotypes similar to those resulting from impaired Delta-Notch signaling. Our results suggest that signaling pathways may regulate the transcription of target genes by regulating the activity of the BRM complex.\n",
      "The Drosophila z600 gene is a member of an overlapping gene cluster located in the 71CD interval of chromosome 3. We have used several approaches to study the spatial and temporal expression of z600 during embryogenesis. Northern analysis shows that z600 is zygotically expressed, with gene transcripts accumulating transiently during early embryo-genesis. The localization of z600 transcripts in embryo sections by in situ hybridization reveals a dynamic pattern of RNA distribution. z600 RNA is distributed throughout the embryo during the midsyncytial blastoderm stage, becomes concentrated dorsally and posteriorly during cellularization, and persists in the dorsal-most region of the embryo during gastrulation and germ band extension. z600 transcript accumulation is altered in dorsalized or ventralized mutant embryos, suggesting that z600 is directly or indirectly regulated by genes specifying the dorsal-ventral pattern in the embryo. An analysis of germ line transformants harboring a z600-lacZ gene fusion demonstrates that a 276-bp DNA region contains the sequences responsible for dorsal activation, but not ventral repression, of z600 gene expression during early embryogenesis.\n",
      "The GATA transcriptionfactors are a family of C4 zinc finger-motif DNA-binding proteins that play defined roles in hematopoiesis as well as presumptive roles in other tissues where they are expressed (e.g., testis, neuronal and placental trophoblast cells) during vertebrate development. To investigate the possibility that GATA proteins may also be involved in Drosophila development, we have isolated and characterized a gene (dGATAa) encoding a factor that is quite similar to mammalian GATA factors. The dGATAa protein sequence contains the two zinc finger DNA-binding domain of the GATA class but bears no additional sequence similarity to any of the vertebrate GATA factors. Analysis of dGATAa gene transcription during Drosophila development revealed that its mRNA is expressed at high levels during early embryogenesis, with transcripts first appearing in the dorsal portion of the embryo just after cellularization. As development progresses, dGATAa mRNA is present at high levels in the dorsal epidermis, suggesting that dGATAa may be involved in determining dorsal cell fate. The pattern of expression in a variety of dorsoventral polarity mutants indicates that dGATAa lies downstream of the zygotic patterning genes decapentaplegic and zerknüllt.\n",
      "The Hox genes play a role in anteroposterior axis specification of bilaterian animals that has been conserved for more than 600 million years. However, some of these genes have occasionally changed their roles in evolution. For example, the insect gene fushi tarazu (ftz), although localised in the Hox cluster, no longer acts as a Hox gene, but is involved in segmentation and nervous system development. Recent data of Mouchel-Vielh et al., and Hughes and Kaufman on ftz homologues in a crustacean and a myriapod, respectively, shed new light onto the evolution of this gene.\n",
      "The Ras-related Rap GTPases are highly conserved across diverse species but their normal biological function is not well understood. Initial studies in mammalian cells suggested a role for Rap as a Ras antagonist. More recent experiments indicate functions in calcium- and cAMP-mediated signaling and it has been proposed that protein kinase A-mediated phosphorylation activates Rap in vivo. We show that Ras1-mediated signaling pathways in Drosophila are not influenced by Rap1 levels, suggesting that Ras1 and Rap1 function via distinct pathways. Moreover, a mutation that abolishes the putative cAMP-dependent kinase phosphorylation site of Drosophila Rap1 can still rescue the Rap1 mutant phenotype. Our experiments show that Rap1 is not needed for cell proliferation and cell-fate specification but demonstrate a critical function for Rap1 in regulating normal morphogenesis in the eye disk, the ovary and the embryo. Rap1 mutations also disrupt cell migrations and cause abnormalities in cell shape. These findings indicate a role for Rap proteins as regulators of morphogenesis in vivo.\n",
      "The Rel family of transcription factors participate in a diverse array of processes, including acute responses to injury and infection, lymphocyte differentiation, and embryonic patterning. These proteins show homology in an extended region spanning about 300 amino acids (the Rel homology domain [RHD]). The RHD mediates both DNA binding and interactions with a family of inhibitor proteins, including I kappa B alpha and cactus. Previous studies have shown that an N-terminal region of the RHD (containing the sequence motif RXXRXRXXC) is important for DNA binding, while the C-terminal nuclear localization sequence is important for inhibitor interactions. Here we present a structure-function analysis of the Drosophila dorsal RHD. These studies identify another sequence within the RHD (region I) that is essential for inhibitor interactions. There is a tight correlation between the conservation of region I sequences and the specificity of Rel-inhibitor interactions in both flies and mammals. Point mutations in the region I sequence can uncouple DNA binding and inhibitor interactions in vitro. The phenotypes associated with the expression of a modified dorsal protein in transgenic Drosophila embryos suggest a similar uncoupling in vivo. Recent crystallographic studies suggest that the region I sequence and the nuclear localization sequence might form a composite surface which interacts with inhibitor proteins.\n",
      "The Rel family transcription factor Dorsal patterns the dorsoventral axis of the Drosophila embryo by activating genes such as twist and snail and repressing genes such as decapentaplegic and zerknüllt. Dorsal represses transcription by recruiting the co-repressor Groucho. However, repression occurs only when Dorsal-binding sites are close to binding sites for other factors that also bind Groucho. The need for additional factors to assist Dorsal in repression may result from the intrinsically weak interaction between Dorsal and Groucho. To test this idea, we generated a Dorsal variant containing a high-affinity Groucho recruitment motif at its C terminus. As predicted, this variant functions as a dedicated repressor, silencing decapentaplegic and zerknüllt while failing to activate twist and snail. We also converted Dorsal into a dedicated activator by replacing its weak Groucho-recruitment motif with heterologous activation domains. Although the dedicated activator alleles fail to repress decapentaplegic and zerknüllt in the syncytial blastoderm embryo, they are able to pattern the dorsoventral axis. This indicates that dorsoventral patterning is not dependent upon Dorsal-mediated repression, reflecting the existence of redundant mechanisms to block Decapentaplegic signaling.\n",
      "The TGF-β pathway is an evolutionarily conserved signal transduction module that mediates diverse biological processes in animals. In Drosophila, both the BMP and Activin branches are required for viability. Studies rooted in classical and molecular genetic approaches continue to uncover new developmental roles for TGF-β signaling. We present an overview of the secreted ligands, transmembrane receptors and cellular Smad transducer proteins that compose the core pathway in Drosophila. An assortment of tools have been developed to conduct tissue-specific loss- and gain-of-function experiments for these pathway components. We discuss the deployment of these reagents, with an emphasis on appropriate usage and limitations of the available tools. Throughout, we note reagents that are in need of further improvement or development, and signaling features requiring further study. A general theme is that comparison of phenotypes for ligands, receptors, and Smads can be used to map tissue interactions, and to separate canonical and non-canonical signaling activities. Core TGF-β signaling components are subject to multiple layers of regulation, and are coupled to context-specific inputs and outputs. In addition to fleshing out how TGF-β signaling serves the fruit fly, we anticipate that future studies will uncover new regulatory nodes and modes and will continue to advance paradigms for how TGF-β signaling regulates general developmental processes.\n",
      "The TNF and TNFR superfamilies of proteins are conserved throughout evolution. The first invertebrate orthologs of TNF and TNFR, Eiger and Wengen, were identified in Drosophila, which enabled us to take advantage of its powerful genetics. Indeed, genetic studies on Eiger in the last decade have discovered their signaling mechanisms through activation of the JNK pathway and unveiled the role of Eiger-JNK signaling in a variety of cellular and tissue processes such as cell death, cell proliferation, tissue growth regulation, host defense, pain sensitization, and canalization. In this review, we will describe the in vivo signaling of Eiger and its physiological roles in fly development and homeostasis, and will discuss the evolution of the TNF/TNFR systems.\n",
      "The Transforming Growth Factor-beta superfamily member decapentaplegic (dpp) acts as an extracellular morphogen to pattern the embryonic ectoderm of the Drosophila embryo. To identify components of the dpp signaling pathway, we screened for mutations that act as dominant maternal enhancers of a weak allele of the dpp target gene zerknŁllt. In this screen, we recovered new alleles of the Mothers against dpp (Mad) and Medea genes. Phenotypic analysis of the new Medea mutations indicates that Medea, like Mad, is required for both embryonic and imaginal disc patterning. Genetic analysis suggests that Medea may have two independently mutable functions in patterning the embryonic ectoderm. Complete elimination of maternal and zygotic Medea activity in the early embryo results in a ventralized phenotype identical to that of null dpp mutants, indicating that Medea is required for all dpp-dependent signaling in embryonic dorsal-ventral patterning. Injection of mRNAs encoding DPP or a constitutively activated form of the DPP receptor, Thick veins, into embryos lacking all Medea activity failed to induce formation of any dorsal cell fates, demonstrating that Medea acts downstream of the thick veins receptor. We cloned Medea and found that it encodes a protein with striking sequence similarity to human SMAD4. Moreover, injection of human SMAD4 mRNA into embryos lacking all Medea activity conferred phenotypic rescue of the dorsal-ventral pattern, demonstrating conservation of function between the two gene products.\n",
      "The conservation of Homeotic (Hox) gene clustering and colinearity in many metazoans indicates that functional constraints operate on this genome organization. However, several studies have questioned its relevance in Drosophila. Here, we analyse the genomic organization of Hox and Hox-derived genes in 13 fruitfly species and the mosquito Anopheles gambiae. We found that at least seven different Homeotic complex (HOM-C) arrangements exist among Drosophila species, produced by three major splits, five microinversions and six gene transpositions. This dynamism contrasts with the stable organization of the complex in many other taxa. Although there is no evidence of an absolute requirement for Hox gene clustering in Drosophila, we found that strong functional constraints act on the individual genes.\n",
      "The developmental control genes containing an Antennapedia-type homeobox are clustered in insects and vertebrates. The evolution of these genes was studied by the construction of evolutionary trees and by statistical geometry in sequence space. The comparative analysis of the homeobox sequences reveals the subdivision of the Antennapedia-type homeobox genes into three classes early in metazoan evolution. This observation suggests an important function of these genes even in the most primitive metazoans. Subsequent duplication events generated a cluster of at least five homeobox genes in the last common ancestor of insects and vertebrates. These genes later independently gave rise to the 13 groups of paralogous genes in vertebrates and to the 11 Antennapedia-type genes in the Drosophila complexes.\n",
      "The discovery of conserved protein domains found in many Drosophila and mammalian developmental gene products suggests that fundamental developmental processes are conserved throughout evolution. Our understanding of development has been enhanced by the discovery of the widespread role of the homeodomain (HD). The action of HD-containing proteins as transcriptional regulators is mediated through a helix-turn-helix motif which confers sequence specific DNA binding. Unexpectedly, the well conserved structural homology between the HD and the prokaryotic helix-turn-helix proteins contrasts with their divergent types of physical interaction with DNA. A C-terminal extension of the HD recognition helix has assumed the role that the N-terminus of the prokaryotic helix plays for specification of DNA binding preference. However, the HD appears also capable of recognizing DNA in an alternative way and its specificity in vivo may be modified by regions outside the helix-turn-helix motif. We propose that this intrinsic complexity of the HD, as well as its frequent association with other DNA binding domains, explains the functional specificity achieved by genes encoding highly related HDs.\n",
      "The discovery of the striking positional conservation between the Antennapedia and Bithorax homeotic gene complexes (ANT-C and BX-C) in Drosophila melanogaster and the murine Hox and human HOX clusters has had a substantial impact on our understanding of the evolution of development and its genetic regulation. Structural differences do exist among the mammalian Hox complexes and the ANT-C in D. melanogaster. To gain further insight into the evolutionary changes among these complexes, the ANT-C was cloned in the closely related species, Drosophila pseudoobscura. The overall structure of the ANT-C in D. pseudoobscura is highly similar to its D. melanogaster counterpart; however, two differences in the organization of the ANT-C have been identified. First, the z2 gene, a member of the ANT-C in D. melanogaster, is not present in the D. pseudoobscura ANT-C and is possibly absent from the D. pseudoobscura genome. Second, the orientation of the Deformed gene is inverted in D. pseudoobscura, providing it with a 5' to 3' direction of transcription identical to the remaining ANT-C homeobox genes with the exception of fushi tarazu. These differences demonstrate that subtle changes can occur in ANT-C structure during relatively short periods of evolutionary divergence, although the fundamental organization of the complex is conserved. These observations and others suggest that the complex is not absolutely rigid but that selective pressures have maintained this organization of genes for some functional reason that remains elusive.\n",
      "The dorsal (dl) morphogen gradient is responsible for initiating the differentiation of the mesoderm, neuroectoderm and dorsal ectoderm in the Drosophila embryo. dl encodes a sequence-specific DNA binding protein that belongs to the Rel family of transcription factors. Previous studies have shown that dl activates the mesoderm determinant twist (twi); here we use a combination of site-directed mutagenesis and P-transformation assays to demonstrate that it also functions as a direct transcriptional repressor of a second target gene, zerknüllt (zen). By exchanging dl binding sites between the promoters we show that activator sites from twi can mediate repression when placed in the context of the zen promoter, and that repressor sites from zen can mediate activation in the context of the twi promoter. This represents the first demonstration that common binding sites for any DNA binding protein can mediate both activation and repression in a developing embryo. Evidence is also presented that the affinities of dl binding sites are important for the efficiency of repression, but are not the sole determinants of the threshold response to the dl gradient.\n",
      "The dorsal (dl) nuclear gradient initiates the differentiation of the mesoderm, neuroectoderm, and dorsal ectoderm by activating and repressing gene expression in the early Drosophila embryo. This gradient is organized by a Toll signaling pathway that shares many common features with the mammalian IL-1 cytokine pathway. Here we present evidence that a second signaling pathway, controlled by the torso (tor) receptor tyrosine kinase, also modulates dl activity. Evidence is presented that the tor pathway selectively masks the ability of dl to repress gene expression but has only a slight effect on activation. Intracellular kinases that are thought to function downstream of tor, such as D-raf and the rolled MAP kinase, mediate this selective block in repression. Normally, the Toll and tor pathways are both active only at the embryonic poles, and consequently, target genes (zen and dpp) that are repressed in middle body regions are expressed at these sites. Constitutive activation of the tor pathway causes severe embryonic defects, including disruptions in gastrulation and mesoderm differentiation, as a result of misregulation of dl target genes. These results suggest that RTK signaling pathways can control gene expression by antirepression, and that multiple pathways can fine-tune the activities of a single transcription factor.\n",
      "The dorsal (dl) protein gradient determines patterns of gene expression along the dorsal-ventral axis of the Drosophila embryo. dl protein is at peak levels in ventral nuclei of the embryo where it activates some genes (twist and snail) and represses others [zerknullt (zen), decapentaplegic and tolloid]. It is a member of the rel family of transcription factors and interacts with specific DNA sequences in the regulatory regions of its target genes. These sequences (dl binding sites), when taken from the context of either an activated or repressed promoter, mediate transcriptional activation of a heterologous promoter, but not repression. We found that T-rich sequences close to the dl binding sites in the silencer region of the zen promoter are conserved between three Drosophila species. Using this sequence information we defined a minimal element that can mediate repression of a heterologous promoter. This element interacts with at least two factors present in embryonic extracts, one of which is dl protein. The other factor binds to the T-rich site. Point mutations in either site abolish ventral repression in vivo. In addition, mutations in the T-rich site cause ectopic expression in ventral regions indicating that the minimal silencer was converted into an enhancer.\n",
      "The dorsal-ventral pattern of the Drosophila egg is established during oogenesis. Epidermal growth factor receptor (Egfr) signaling within the follicular epithelium is spatially regulated by the dorsally restricted distribution of its presumptive ligand, Gurken. As a consequence, pipe is transcribed in a broad ventral domain to initiate the Toll signaling pathway in the embryo, resulting in a gradient of Dorsal nuclear translocation. We show that expression of pipe RNA requires the action of fettucine (fet) in ovarian follicle cells. Loss of maternal fet activity produces a dorsalized eggshell and embryo. Although similar mutant phenotypes are observed with regulators of Egfr signaling, genetic analysis suggests that fet acts downstream of this event. The fet mutant phenotype is rescued by a transgene of capicua (cic), which encodes an HMG-box transcription factor. We show that Cic protein is initially expressed uniformly in ovarian follicle cell nuclei, and is subsequently downregulated on the dorsal side. Earlier studies described a requirement for cic in repressing zygotic target genes of both the torso and Toll pathways in the embryo. Our experiments reveal that cic controls dorsal-ventral patterning by regulating pipe expression in ovarian follicle cells, before its previously described role in interpreting the Dorsal gradient.\n",
      "The dorsal-ventral pattern of the Drosophila embryo is established by three sequential signaling pathways. Each pathway transmits spatial information by localizing the activity of an extracellular signal, which acts as a ligand for a broadly distributed transmembrane receptor. The components of the first two pathways are encoded by maternal effect genes, while the third pathway is specified by genes expressed in the zygote. During oogenesis, the oocyte transmits a signal to the surrounding follicle cells by the gurken-torpedo pathway. After fertilization, the initial asymmetry of the egg chamber is used by the spätzle-Toll pathway to generate within the embryo a nuclear gradient of the transcription factor Dorsal, which regulates the regional expression of a set of zygotic genes. On the dorsal side of the embryo, the decapentaplegic-punt/thick veins pathway then establishes patterning of the amnioserosa and dorsal ectoderm. Each pathway uses a distinct strategy to achieve spatial localization of signaling activity.\n",
      "The earliest stages of development in most metazoans aredriven by maternally deposited proteins and mRNAs, with widespread transcriptional activation of the zygotic genome occurring hours after fertilization, at a period known as the maternal-to-zygotic transition (MZT). In Drosophila, the MZT is preceded by the transcription of a small number of genes that initiate sex determination, patterning, and other early developmental processes; and the zinc-finger protein Zelda (ZLD) plays a key role in their transcriptional activation. To better understand the mechanisms of ZLD activation and the range of its targets, we used chromatin immunoprecipitation coupled with high-throughput sequencing (ChIP-Seq) to map regions bound by ZLD before (mitotic cycle 8), during (mitotic cycle 13), and after (late mitotic cycle 14) the MZT. Although only a handful of genes are transcribed prior to mitotic cycle 10, we identified thousands of regions bound by ZLD in cycle 8 embryos, most of which remain bound through mitotic cycle 14. As expected, early ZLD-bound regions include the promoters and enhancers of genes transcribed at this early stage. However, we also observed ZLD bound at cycle 8 to the promoters of roughly a thousand genes whose first transcription does not occur until the MZT and to virtually all of the thousands of known and presumed enhancers bound at cycle 14 by transcription factors that regulate patterned gene activation during the MZT. The association between early ZLD binding and MZT activity is so strong that ZLD binding alone can be used to identify active promoters and regulatory sequences with high specificity and selectivity. This strong early association of ZLD with regions not active until the MZT suggests that ZLD is not only required for the earliest wave of transcription but also plays a major role in activating the genome at the MZT.\n",
      "The establishment of expression domains of developmentally regulated genes depends on cues provided by different concentrations of transcriptional activators and repressors. Here we analyze the regulation of the Drosophila gene zen, which is a target of the Decapentaplegic (Dpp) signaling pathway during cellular blastoderm formation. We show that low levels of the Dpp signal transducer p-Mad (phosphorylated Mad), together with the recently discovered negative regulator Brinker (Brk), define the spatial limits of zen transcription in a broad dorsal-on/ventral-off domain. The subsequent refinement of this pattern to the dorsal-most cells, however, correlates with high levels of p-Mad that accumulate in the same region during late blastoderm. Examination of the zen regulatory sequences revealed the presence of multiple Mad and Brk binding sites, and our results indicate that a full occupancy of the Mad sites due to high concentrations of nuclear Mad is the primary mechanism for refinement of zen. Interestingly, several Mad and Brk binding sites overlap, and we show that Mad and Brk cannot bind simultaneously to such sites. We propose a model whereby competition between Mad and Brk determines spatially restricted domains of expression of Dpp target genes.\n",
      "The establishment of the dorsal-ventral pattern in Drosophila embryos depends on a signal transduction process: a putative extracellular ligand released into the perivitelline space surrounding the embryo binds to the Toll receptor. Toll activation triggers the formation of the nuclear gradient of dorsal protein, the morphogen of the dorsal-ventral axis. Here, I analyse the dorsal protein distribution and the expression of zygotic dorsal-ventral genes in Toll- embryos that have been injected with wild-type cytoplasm under a variety of different injection conditions. Injections into two positions within a single embryo lead to the formation of two dorsal-ventral patterns in one embryo, allowing the analysis of interactions between pattern-forming processes. The results of single and double injections suggest that the spatial information for the embryonic dorsal-ventral axis is largely derived from spatial cues present in the extraembryonic compartment, which restrict the release of the putative Toll ligand. They argue against a Toll-dependent pattern-formation process employing local self-enhancement and lateral inhibition to enhance a weak initial asymmetry. The putative Toll ligand appears to originate from a ventrally restricted zone which extends along the entire anterior-posterior axis. Ligand diffusion or its graded release are required to determine the slope of the nuclear dorsal protein gradient. Both the Toll receptor and the putative ligand of Toll are in excess in wild-type embryos. Since spatial information for the embryonic dorsal-ventral axis is already present in the vitelline membrane or the perivitelline space, it is most likely generated during oogenesis. Oogenic pattern formation is also responsible for the perpendicular orientation the dorsal-ventral axis maintains with respect to the anterior-posterior axis.\n",
      "The existence of a gene complex in the proximal right arm of chromosome 3 of Drosophila melanogaster involved in the development of the head and thorax was originally suggested by the phenotypes of several dominant homoeotic mutations and their revertants. A screen for mutations utilizing Df(3R) Antp(Ns+R17) (proximally broken in salivary region 84B1,2) yielded, among 102 recovered mutations, 17 localized by deficiency mapping to the putative homoeotic cluster. These fell into four complementation groups, two of which were characterized by homoeotic phenotypes. To explore the limits of the Antennapedia gene complex (ANT-C) more proximally, a second screen has been undertaken utilizing Df(3R)Scr, a deficiency of 84A1-B1,2.-Of 2832 chromosomes screened, 21 bearing alterations localized to polytene interval 84A-84B1,2 have been recovered. Sixteen are recessive lethals, and five showing reduced viability display a visible phenotype in surviving individuals. Complementation and phenotypic analyses revealed four complementation groups proximal to those identified in the previous screen, including two new alleles of the recessive homoeotic mutation, proboscipedia (pb). Ten of the new mutations correspond to complementation groups defined previously in the Df(3R)Antp(Ns+R17) screen four to the EbR11 group, two to the Scr group and four to the Antp group.-On the basis of the phenotypes of the 39 mutations localized to this region, plus their interactions with extant homoeotic mutations, we postulate that there are at least five functional sites comprising the ANT-C. Three have been demonstrated to be homoeotic in nature. The specific homoeotic transformations thus far observed suggest that these loci are critical for normal development of adult labial, maxillary and thoracic structures.\n",
      "The expression of the chloramphenicol acetyltransferase gene under control of the 1152-base pair 5'-flanking region (-1107 to +45 nucleotide positions with respect to the major transcription initiation site) of the Drosophila DNA polymerase alpha gene was repressed by cotransfection into Drosophila Kc cells with a zerknüllt (zen)-expressing plasmid as previously observed with the proliferating cell nuclear antigen (PCNA) gene promoter. The expression of the zen resulted in reduction of the abundance of mRNA for the transfected chloramphenicol acetyltransferase gene and also mRNAs for both DNA polymerase alpha and PCNA. Results obtained using various deletion derivatives of the promoter region and chemically synthesized oligonucleotides of the DNA replication-related element (DRE), a positive cis-acting element found in both DNA polymerase alpha and PCNA genes, revealed that the DRE sequences are responsible to repression by Zen protein. The nuclear extract of Kc cells transfected by the zen-expressing plasmid contained lesser amounts of the DRE-binding factor (DREF) than that of untransfected or mutant zen-transfected cells. These results suggest that the Zen protein represses expression of DNA replication-related genes by reducing DREF, although the detailed mechanism of the repression remains to be elucidated.\n",
      "The family of Hox-proteins has been a major focus of research for over 30 years. Hox-proteins are crucial to the correct development of bilateral organisms, however, some uncertainty remains as to which Hox-proteins are functionally equivalent across different species. Initial classification of Hox-proteins was based on phylogenetic analysis of the 60 amino acid homeodomain. This approach was successful in classifying Hox-proteins with differing homeodomains, but the relationships of Hox-proteins with nearly identical homeodomains, yet distinct biological functions, could not be resolved. Correspondingly, these 'problematic' proteins were classified into one large unresolved group. Other classifications used the relative location of the Hox-protein coding genes on the chromosome (synteny) to further resolve this group. Although widely used, this synteny-based classification is inconsistent with experimental evidence from functional equivalence studies. These inconsistencies led us to re-examine and derive a new classification for the Hox-protein family using all Hox-protein sequences available in the GenBank non-redundant protein database (NCBI-nr). We compare the use of the homeodomain, the homeodomain with conserved flanking regions (the YPWM and linker region), and full length Hox-protein sequences as a basis for classification of Hox-proteins. In contrast to previous attempts, our approach is able to resolve the relationships for the 'problematic' as well as ABD-B-like Hox-proteins. We highlight differences to previous classifications and clarify the relationships of Hox-proteins across the five major model organisms, Caenorhabditis elegans, Drosophila melanogaster, Branchiostoma floridae, Mus musculus and Danio rerio. Comparative and functional analysis of Hox-proteins, two fields crucial to understanding the development of bilateral organisms, have been hampered by difficulties in predicting functionally equivalent Hox-proteins across species. Our classification scheme offers a higher-resolution classification that is in accordance with phylogenetic as well as experimental data and, thereby, provides a novel basis for experiments, such as comparative and functional analyses of Hox-proteins.\n",
      "The flour beetle Tribolium castaneum has become an important model organism for comparative studies of insect development. Many developmentally important genes have now been cloned from both Tribolium and Drosophila and their expression characteristics were studied. We analyze here the complete coding sequences of 17 homologous gene pairs from D. melanogaster and T. castaneum, most of which encode transcription factors. We find that the Tribolium genes are on average 30% shorter than their Drosophila homologues. This appears to be due largely to the almost-complete absence of trinucleotide repeats in the coding sequences of Tribolium as well as the generally lower degree of internal repetitiveness. Clusters of polar and other amino acids such as glutamine, proline, and serine, which are often considered to be important for transcriptional activation domains in Drosophila, are almost completely absent in Tribolium. Codon usage is generally less biased in Tribolium, although we find a similar tendency for the preference of G- or C-ending codons and a higher bias in conserved subregions of the proteins as in Drosophila. Most of the aminoacid substitutions in the DNA-binding domains of the transcription factors occur at residues that do not make a specific contact to DNA, suggesting that the recognition sequences are likely to be conserved between the two species.\n",
      "The fly Drosophila melanogaster is one of the most intensively studied organisms in biology and serves as a model system for the investigation of many developmental and cellular processes common to higher eukaryotes, including humans. We have determined the nucleotide sequence of nearly all of the approximately 120-megabase euchromatic portion of the Drosophila genome using a whole-genome shotgun sequencing strategy supported by extensive clone-based sequence and a high-quality bacterial artificial chromosome physical map. Efforts are under way to close the remaining gaps; however, the sequence is of sufficient accuracy and contiguity to be declared substantially complete and to support an initial analysis of genome structure and preliminary gene annotation and interpretation. The genome encodes approximately 13,600 genes, somewhat fewer than the smaller Caenorhabditis elegans genome, but with comparable functional diversity.\n",
      "The fork head (fkh) domain defines the DNA-binding region of a family of transcription factors which has been implicated in regulating cell fate decisions across species lines. We have cloned and molecularly characterized the crocodile (croc) gene which encodes a new family member from Drosophila. croc is expressed in the head anlagen of the blastoderm embryo under the control of the anterior, the dorsoventral and the terminal maternal organizer systems. The croc mutant phenotype indicates that the croc wild-type gene is required to function as an early patterning gene in the anterior-most blastoderm head segment anlage and for the establishment of a specific head skeletal structure that derives from the non-adjacent intercalary segment at a later stage of embryogenesis. As an early patterning gene, croc exerts unusual properties which do not allow it to be grouped among the established segmentation genes. A single-site mutation within the croc fkh domain, which causes a replacement of the first out of four conserved amino acid residues thought to be involved in the coordinate binding of Mg2+, abolishes the DNA binding of the protein in vitro. In view of the resulting lack-of-function mutant phenotype, it appears likely that metal binding by the affected region of the fkh domain is crucial for proper folding of the DNA-binding structure.\n",
      "The gene regulatory network of a developmental process contains many mutually repressive interactions between two genes. They are often regulated by or regulate an additional factor, which constitute prominent network motifs, called regulated and regulating mutual loops. Our database analysis on the gene regulatory network for Drosophila melanogaster indicates that those with mutual repression are working specifically for the segmentation process. To clarify their biological roles, we mathematically study the response of the regulated mutual loop with mutual repression to input stimuli. We show that the mutual repression increases the response sensitivity without affecting the threshold input level to activate the target gene expression, as long as the network output is unique for a given input level. This high sensitivity of the motif can contribute to sharpening the spatial domain pattern without changing its position, assuring a robust developmental process. We also study transient dynamics that shows shift of domain boundary, agreeing with experimental observations. Importance of mutual repression is addressed by comparingwith other types of regulations.\n",
      "The general consensus in the field is that limiting amounts of the transcription factor Dorsal establish dorsal boundaries of genes expressed along the dorsal-ventral (DV) axis of early Drosophila embryos, while repressors establish ventral boundaries. Yet recent studies have provided evidence that repressors act to specify the dorsal boundary of intermediate neuroblasts defective (ind), a gene expressed in a stripe along the DV axis in lateral regions of the embryo. Here we show that a short 12 base pair sequence (\"the A-box\") present twice within the ind CRM is both necessary and sufficient to support transcriptional repression in dorsal regions of embryos. To identify binding factors, we conducted affinity chromatography using the A-box element and found a number of DNA-binding proteins and chromatin-associated factors using mass spectroscopy. Only Grainyhead (Grh), a CP2 transcription factor with a unique DNA-binding domain, was found to bind the A-box sequence. Our results suggest that Grh acts as an activator to support expression of ind, which was surprising as we identified this factor using an element that mediates dorsally-localized repression. Grh and Dorsal both contribute to ind transcriptional activation. However, another recent study found that the repressor Capicua (Cic) also binds to the A-box sequence. While Cic was not identified through our A-box affinity chromatography, utilization of the same site, the A-box, by both factors Grh (activator) and Cic (repressor) may also support a \"switch-like\"response that helps to sharpen the ind dorsal boundary. Furthermore, our results also demonstrate that TGF-β signaling acts to refine ind CRM expression in an A-box independent manner in dorsal-most regions, suggesting that tiers of repression act in dorsal regions of the embryo.\n",
      "The genomic and cDNA clones for a Drosophila melanogaster proliferating cell nuclear antigen (PCNA) (cyclin) were isolated and sequenced. The coding sequence for a 260-amino-acid residue polypeptide was interrupted by a single short intron of 60 base pairs (bp), and about 70% of the deduced amino acid sequence of the Drosophila PCNA was identical to the rat and human PCNA polypeptides, with conserved unique repeats of leucine in the C-terminal region. Genomic Southern blot hybridization analysis indicates the presence of a single gene for PCNA per genome. The PCNA mRNA was detected at a high level in adult ovaries, unfertilized eggs, and early embryos and at low levels in the other developmental stages. The major transcription initiation site (cap site) was localized at 89 bp upstream from the ATG codon. Neither a TATA box nor a CAAT box was found within the 600-bp region upstream of the cap site. Clusters of 10 bp of sequence similar to the binding sites for Drosophila proteins containing homeodomains were found in the region from -127 to -413. DNase I footprint analysis revealed that the Drosophila homeodomain proteins coded by even-skipped and zerknüllt genes can specifically bind to these sites. These results suggest that the expression of the PCNA gene is under the control of genes coding for homeodomain proteins.\n",
      "The genomic program for development operates primarily by the regulated expression of genes encoding transcription factors and components of cell signaling pathways. This program is executed by cis-regulatory DNAs (e.g., enhancers and silencers) that control gene expression. The regulatory inputs and functional outputs of developmental control genes constitute network-like architectures. In this PNAS Special Feature are assembled papers on developmental gene regulatory networks governing the formation of various tissues and organs in nematodes, flies, sea urchins, frogs, and mammals. Here, we survey salient points of these networks, by using as reference those governing specification of the endomesoderm in sea urchin embryos and dorsal-ventral patterning in the Drosophila embryo.\n",
      "The graded nuclear location of the transcription factor Dorsal along the dorsoventral axis of the early Drosophila embryo provides positional information for the determination of different cell fates. Nuclear uptake of Dorsal depends on a complex signalling pathway comprising two parts: an extracellular proteolytic cascade transmits the dorsoventral polarity of the egg chamber to the early embryo and generates a gradient of active Spätzle protein, the ligand of the receptor Toll; an intracellular cascade downstream of Toll relays this graded signal to embryonic nuclei. The slope of the Dorsal gradient is not determined by diffusion of extracellular or intracellular components from a local source, but results from self-organised patterning, in which positive and negative feedback is essential to create and maintain the ratio of key factors at different levels, thereby establishing and stabilising the graded spatial information for Dorsal nuclear uptake.\n",
      "The homeo box, which encodes the DNA-binding homeo domain, is a DNA sequence motif present in several Drosophila developmental genes; it has been used to identify many homologous genes involved in mammalian development. The paired box is another conserved sequence motif, first identified in the paired (prd) and gooseberry (gsb) Drosophila homeo domain genes. It encodes a 128-amino-acid domain, the paired domain, which has since been found in other fly and mouse gene products, in association with the homeo domain or in its absence. We show that the paired box of the prd gene encodes a DNA-binding activity, independent of the DNA-binding activity of the Paired (Prd) homeo domain and with a different sequence specificity. The amino-terminal region of the paired domain, including one of the three predicted alpha-helices, is necessary and sufficient for binding. We investigate the binding of the Prd protein to two sites in the even-skipped promoter, which are composed of overlapping sequences bound by the homeo domain and by the paired domain. We also show that a mutation in the paired box of Prd, corresponding to the mutation in the paired box of the mouse Pax-1 gene thought to cause the undulated skeletal phenotype, destroys the ability of the Prd protein to bind to the paired domain-specific site. This supports the view that the undulated phenotype results from the inactivation of the DNA-binding activity of the paired domain of Pax-1.\n",
      "The homeobox gene family of Drosophila appears to control a variety of position-specific patterning decisions during embryonic and imaginal development. Most of these patterning decisions determine groups of cells on the anterior-posterior axis of the Drosophila germ band. We have isolated a novel homeobox gene from Drosophila, designated H2.0. H2.0 has the most diverged homeobox so far characterized in metazoa, and, in contrast to all previously isolated homeobox genes, H2.0 exhibits a tissue-specific pattern of expression. The cells that accumulate transcripts for this novel gene correspond to the visceral musculature and its anlagen.\n",
      "The homoeobox gene zerknüllt (zen) plays an important role in the differentiation of dorsal tissues during Drosophila development. zen- embryos show transformations in the dorsal-most regions of the fate map, and lack several tissues that normally derive from these regions, including the amnioserosa and optic lobe. zen displays a simple dorsal on/ventral off pattern as early as cleavage cycle 10-11 (ref. 2). We have prepared a polyclonal antibody against a full-length zen protein, and used this to examine its pattern of expression in mutants that disrupt dorsal-ventral polarity. Most or all of the maternally expressed genes that are involved in this process have been previously identified and fall into two classes, so called 'dorsalizers' and 'ventralizers' (see refs 4-7, reviewed in ref. 8). On the basis of our analysis of zen expression in each of these maternal mutants we propose that one or more of the dorsalizing genes encodes a repressor which inhibits the expression of zen in ventral regions of developing embryos. The ventralizing gene cactus might play an important role in restricting the activity of this repressor to ventral regions, thereby permitting the activation of zen in those dorsal tissues where its function is critically required.\n",
      "The immunoglobulin superfamily is a diverse group of proteins that are involved in various aspects of cell surface recognition. Here, we report the characterization of amalgam (ama), a gene in the Antennapedia complex (ANT-C) of D. melanogaster that exhibits amino acid similarity to vertebrate neural cell adhesion molecules and other members of the immunoglobulin superfamily. The putative 333 amino acid ama protein consists of a signal sequence, three immunoglobulin-like domains, and a short slightly hydrophobic carboxy-terminal region. Antibodies against the ama protein reveal that it accumulates on the surface of various mesodermal and neural cells during embryogenesis. The function of this protein remains elusive, as no mutations have been recovered for ama during saturation EMS mutagenesis of this chromosomal region.\n",
      "The maternal Dorsal regulatory gradient initiates the differentiation of several tissues in the early Drosophila embryo. Whole-genome microarray assays identified as many as 40 new Dorsal target genes, which encode a broad spectrum of cell signaling proteins and transcription factors. Evidence is presented that a tissue-specific form of the NF-Y transcription complex is essential for the activation of gene expression in the mesoderm. Tissue-specific enhancers were identified for new Dorsal target genes, and bioinformatics methods identified conserved cis-regulatory elements for coordinately regulated genes that respond to similar thresholds of the Dorsal gradient. The new Dorsal target genes and enhancers represent one of the most extensive gene networks known for any developmental process.\n",
      "The maternal determinants of dorsoventral polarity of the Drosophila embryo are derived from somatic and germ-line components of the egg chamber. During oogenesis, asymmetry seems to be established by a signal transduction process. This process is thought to provide the developing embryo with a ventral signal responsible for determining the embryonic axis. Through a set of interactions that may involve signal transduction and proteolytic cascade events, positional information is generated in the form of a graded distribution of dorsal protein in blastoderm nuclei. Different levels of dorsal protein result in asymmetric expression of zygotic genes that ultimately specify cell fate.\n",
      "The maternal transcription factor Dorsal (Dl) functions as both an activator and a repressor in a context-dependent manner to control dorsal-ventral patterning in the Drosophila embryo. Previous studies have suggested that Dl is an intrinsic activator and its repressive activity requires additional corepressors that bind corepressor-binding sites near Dl-binding sites. However, the molecular identities of the corepressors have yet to be identified. Here, we present evidence that Capicua (Cic) is involved in Dl-mediated repression in the zerknüllt (zen) ventral repression element (VRE). Computational and genetic analyses indicate that a DNA-binding consensus sequence of Cic is highly analogous with previously identified corepressor-binding sequences and that Dl failed to repress zen expression in lateral regions of cic mutant embryos. Furthermore, electrophoretic mobility shift assay (EMSA) shows that Cic directly interacts with several corepressor-binding sites in the zen VRE. These results suggest that Cic may function as a corepressor by binding the VRE.\n",
      "The mechanisms that generate precise patterns of discrete cell types within developing fields are not well understood. One model for analyzing how cells interpret positional information in two dimensions is the regulation of proneural cluster formation within insect segments. Two adjacent proneural regulatory genes, achaete and scute, are expressed coincidently in cell clusters at reproducible anteroposterior (AP) and dorsoventral (DV) coordinates within the Drosophila embryo from which single neuroblasts later arise. Here, we show that the AP and DV position of these clusters is regulated through a common cis-acting region between the genes under the initial control of the products of the pair-rule and DV polarity genes and is later maintained by selected segment polarity genes. The combination of proneural gene activation/repression in AP stripes and repression within specific DV domains positions each cluster of achaete/scute expressing cells within segments; interactions between these cells then determine individual cell fates.\n",
      "The members of the evolutionarily conserved Hox-gene complex, termed Hox genes, are required for specifying segmental identity during embryogenesis in various animal phyla. The Hox3 genes of winged insects have lost this ancestral function and are required for the development of extraembryonic epithelia, which do not contribute to any larval structure. Higher flies (Cyclorrhapha) such as Drosophila melanogaster contain Hox3 genes of two types, the zerknüllt type and the bicoid type. The zerknüllt gene is expressed zygotically on the dorsal side of the embryo and is required for establishing extraembryonic tissue. Its sister gene bicoid is expressed maternally and the transcripts are localized at the anterior pole of the mature egg. BICOID protein, which emerges from this localized source during early development, is required for embryonic patterning. All known direct bicoid homologues are confined to Cyclorrhaphan flies. Here, we describe Hox3 genes of the non-Cyclorrhaphan flies Empis livida (Empididae), Haematopota pluvialis (Tabanidae), and Clogmia albipunctata (Psychodidae). The gene sequences are more similar to zerknüllt homologues than to bicoid homologues, but they share expression characteristics of both genes. We propose that an ancestral Hox3 gene had been duplicated in the stem lineage of Cyclorrhaphan flies. During evolution, one of the gene copies lost maternal expression and evolved as zerknüllt, whereas the second copy lost zygotic expression and evolved as bicoid. Our finding correlates well with a partial reduction of zerknüllt-dependent extraembryonic tissue during Dipteran evolution.\n",
      "The organization of the anterior pattern in the Drosophila embryo is mediated by the maternal effect gene bicoid. bcd has been identified in an 8.7-kb genomic fragment by germ line transformants that completely rescue the mutant phenotype. The major transcript of 2.6 kb includes a homeobox with low homology to previously known homeoboxes, a PRD-repeat and a M-repeat. In situ hybridizations reveal that bcd is transcribed in the nurse cells. The mRNA is localized at the anterior tip of oocyte and early embryo until the cellular blastoderm stage. The localization of the transcript requires the function of the maternal effect genes exuperantia and swallow while transcript stability is reduced by functions depending on posterior group genes.\n",
      "The origin of evolutionary novelty is believed to involve both positive selection and relaxed developmental constraint. In flies, the redesign of anterior patterning during embryogenesis is a major developmental innovation and the rapidly evolving Hox gene, bicoid (bcd), plays a critical role. We report evidence for relaxation of selective constraint acting on bicoid as a result of its maternal pattern of gene expression. Evolutionary theory predicts 2-fold greater sequence diversity for maternal effect genes than for zygotically expressed genes, because natural selection is only half as effective acting on autosomal genes expressed in one sex as it is on genes expressed in both sexes. We sample an individual from ten populations of Drosophila melanogaster and nine populations of D. simulans for polymorphism in the tandem gene duplicates bcd, which is maternally expressed, and zerknüllt (zen), which is zygotically expressed. In both species, we find the ratio of bcd to zen nucleotide diversity to be two or more in the coding regions but one in the noncoding regions, providing the first quantitative support for the theoretical prediction of relaxed selective constraint on maternal-effect genes resulting from sex-limited expression. Our results suggest that the accelerated rate of evolution observed for bcd is owing, at least partly, to variation generated by relaxed selective constraint.\n",
      "The passage of an individual's genome to future generations is essential for the maintenance of species and is mediated by highly specialized cells, the germ cells. Genetic studies in a number of model organisms have provided insight into the molecular mechanisms that control specification, migration and survival of early germ cells. Focusing on Drosophila, we will discuss the mechanisms by which germ cells initially form and remain transcriptionally silent while somatic cells are transcriptionally active. We will further discuss three separate attractiveand repellent guidance pathways, mediated by a G-protein coupled receptor, two lipid phosphate phosphohydrolases, and isoprenylation. We will compare and contrast these findings with those obtained in other organisms, in particular zebrafish and mice. While aspects of germ cell specification are strikingly different between these species, germ cell specific gene functions have been conserved. In particular, mechanisms that sense directional cues during germ cell migration seem to be shared between invertebrates and vertebrates.\n",
      "The position of the nucleus along the anterior rim of stage 8 Drosophila oocytes presages the dorsal side of the egg and the developing embryo. In this paper, we address the question of whether the oocyte has a previously determined dorsal side to which the nucleus is drawn, or whether nuclear position randomly determines the dorsal side. To do so, we have taken advantage of a genetic system in which Drosophila oocytes occasionally become binuclear. We find that (i) the two nuclei migrate independently to their respective positions on the anterior rim, sometimes selecting the same site, sometimes not, (ii) the two nuclei are equivalent in their ability to induce a dorsal-ventral pattern in the overlying follicular epithelium, and (iii) at any position around the anterior circumference of the egg chamber the follicle cell sheet is equally responsive to the Gurken signal associated with the oocyte nuclei. These results argue that the dorsal-ventral axis is determined arbitrarily by the randomly selected position of the nucleus on the anterior rim of the oocyte. Some of the binuclear eggs support embryonic development. However, despite the duplication of dorsal chorion structures, the majority of such embryos show normal dorsal-ventral patterning. Thus, processes exist in the ventral follicular epithelium or in the perivitelline space that compensate for the expansion of dorsal follicle cell fates and consequently allow the formation of a normal embryonic axis.\n",
      "The pre-cellular Drosophila embryo contains 10 well characterized sequence-specific transcriptional repressors, which represent a broad spectrum of DNA-binding proteins. Previous studies have shown that two of the repressors, Hairy and Dorsal, recruit a common co-repressor protein, Groucho. Here we present evidence that three different repressors, Knirps, Krüppel and Snail, recruit a different co-repressor, dCtBP. Mutant embryos containing diminished levels of maternal dCtBP products exhibit both segmentation and dorsoventral patterning defects, which can be attributed to loss of Krüppel, Knirps and Snail activity. In contrast, the Dorsal and Hairy repressors retain at least some activity in dCtBP mutant embryos. dCtBP interacts with Krüppel, Knirps and Snail through a related sequence motif, PXDLSXK/H. This motif is essential for the repression activity of these proteins in transgenic embryos. We propose that dCtBP represents a major form of transcriptional repression in development, and that the Groucho and dCtBP co-repressors mediate separate pathways of repression.\n",
      "The principles of embryonic pattern formation have been studied extensively in many systems using classical experimental approaches. In Drosophila, a powerful combination of genetics and transplantation experiments, as well as molecular biology, have helped to elucidate the mechanisms that operate during oogenesis and early embryogenesis to establish a set of positional cues required for axis determination in the early embryo. In systematic searches for maternal effect mutations a small number of about 30 genes have been identified that specifically affect the process of determination of the embryonic axes. These 'coordinate' genes define four systems that determine the anteroposterior (AP) axis (three systems) and the dorsoventral (DV) axis (one system) independently. In the anteroposterior axis, the anterior system determines the segmented region of head and thorax, the posterior system determines the segmented abdominal region, and the terminal system is responsible for the formation of the nonsegmented termini at the anterior and posterior egg tips, the acron and telson. In contrast, pattern along the dorsoventral axis is determined by one system only. Although all four systems use different biochemical mechanisms, they share several properties. (1) The product of one gene in each system is localized in a specific region of the freshly laid egg and functions as a spatial signal. (2) In each system, this spatial information finally results in the asymmetrical distribution of one gene product that functions as a transcription factor. (3) This transcription factor is distributed in a concentration gradient that defines the spatial limits of expression of one or more zygotic target genes. The combined action of these three anteroposterior systems as well as the dorsoventral system defines the expression of zygotic target genes in at least seven distinct regions along the anteroposterior and at least three in the dorsoventral axis. These longitudinal and transverse domains provide a coarse spatial prepattern which is then further refined by the action and interaction of zygotic pattern genes.\n",
      "The recent completion of the Drosophila melanogaster genomic sequence to high quality and the availability of a greatly expanded set of Drosophila cDNA sequences, aligning to 78% of the predicted euchromatic genes, afforded FlyBase the opportunity to significantly improve genomic annotations. We made the annotation process more rigorous by inspecting each gene visually, utilizing a comprehensive set of curation rules, requiring traceable evidence for each gene model, and comparing each predicted peptide to SWISS-PROT and TrEMBL sequences. Although the number of predicted protein-coding genes in Drosophila remains essentially unchanged, the revised annotation significantly improves gene models, resulting in structural changes to 85% of the transcripts and 45% of the predicted proteins. We annotated transposable elements and non-protein-coding RNAs as new features, and extended the annotation of untranslated (UTR) sequences and alternative transcripts to include more than 70% and 20% of genes, respectively. Finally, cDNA sequence provided evidence for dicistronic transcripts, neighboring genes with overlapping UTRs on the same DNA sequence strand, alternatively spliced genes that encode distinct, non-overlapping peptides, and numerous nested genes. Identification of so many unusual gene models not only suggests that some mechanisms for gene regulation are more prevalent than previously believed, but also underscores the complex challenges of eukaryotic gene prediction. At present, experimental data and human curation remain essential to generate high-quality genome annotations.\n",
      "The repetitive DNA that constitutes most of the heterochromatic regions of metazoan genomes has hindered the comprehensive analysis of gene content and other functions. We have generated a detailed computational and manual annotation of 24 megabases of heterochromatic sequence in the Release 5 Drosophila melanogaster genome sequence. The heterochromatin contains a minimum of 230 to 254 protein-coding genes, which are conserved in other Drosophilids and more diverged species, as well as 32 pseudogenes and 13 noncoding RNAs. Improved methods revealed that more than 77% of this heterochromatin sequence, including introns and intergenic regions, is composed of fragmented and nested transposable elements and other repeated DNAs. Drosophila heterochromatin contains \"islands\"of highly conserved genes embedded in these \"oceans\"of complex repeats, which may require special expression and splicing mechanisms.\n",
      "The retinal determination gene dachshund is distantly related to the family of Ski/Sno proto-oncogenes and influences the development of a wide range of tissues including the embryonic head, optic lobes, brain, central nervous system as well as the post-embryonic leg, wing, genital and eye-antennal discs. We were interested in the regulatory mechanisms that control the dynamic expression pattern of dachshund and in this report we set out to ascertain how the transcription of dachshund is modulated in the embryonic head and developing eye-antennal imaginal disc. We demonstrate that the TGFbeta signaling cascade, the transcription factor zerknullt and several other patterning genes prevent dachshund from being expressed inappropriately within the embryonic head. Additionally, we show that several members of the eye specification cascade influence the transcription of dachshund during normal and ectopic eye development. Our results suggest that dachshund is regulated by a complex combinatorial code of transcription factors and signaling pathways. Unraveling this code may lead to an understanding of how dachshund regulates the development of many diverse tissue types including the eye.\n",
      "The rho family of GTPases has been implicated in regulating changes in cell morphology in response to extracellular signals. We have cloned three widely expressed members of this family from Drosophila melanogaster; a rho homologue (Rho1) and two rac homologues (Rac1 and Rac2). Flies harbouring a Rho1 transgene that is specifically expressed in the eye exhibit a dramatic dose dependent disruption of normal eye development. Flies bearing at least two copies of the transgene display a severe rough eye phenotype characterized by missing secondary and tertiary pigment cells, a substantial reduction in the number of photoreceptor cells and a grossly abnormal morphology of the rhabdomeres. Cell fate determination in the imaginal disc occurs normally and abnormalities become manifest late in pupariation, coincident with the phase when the cells undergo major morphological changes. This phenotype is modified by mutations at several other loci that have been implicated in signal transduction, but not by mutations in ras pathway components.\n",
      "The short gastrulation (sog) gene is expressed in broad lateral stripes comprising the neuroectoderm of the Drosophila blastoderm embryo. sog encodes a predicted secreted protein that functions nonautonomously to antagonize the activity of the TGF-beta-like Decapentaplegic (Dpp) signaling pathway in the dorsal region of the embryo. Recently, it has been shown that sog and dpp are functionally equivalent to their respective Xenopus homologs chordin and BMP-4. In this report we provide the first direct evidence that sog plays a local role in the lateral region of the blastoderm embryo to oppose Dpp activity in the neuroectoderm. In the dorsal region, Dpp signaling both suppresses neurogenesis and maintains expression of genes that promote dorsal cell fates (dorsalization). We show that Dpp also can perform both of these functions in the neuroectoderm. In wild-type embryos, the ability of Dpp to induce expression of dorsal markers including itself (autoactivation) in the neuroectoderm is blocked by sog. We propose that Sog protects the neuroectoderm from an invasive positive feedback loop created by Dpp diffusion and autoactivation. We show that the two functions of Dpp signaling, neural suppression and dorsalization, are triggered by distinct thresholds of Dpp activity. Epistasis experiments reveal that all observed sog activity can be accounted for by Sog functioning as a dedicated Dpp antagonist. Finally, we provide evidence that Sog functions as a diffusible morphogen in the blastoderm embryo. These data strongly support the view that the primary phylogenetically conserved function of the Drosophila sog and dpp genes and the homologous Xenopus chordin and BMP-4 genes is to subdivide the primitive embryonic ectoderm into neural versus non-neural domains.\n",
      "The spatial and temporal pattern of mitoses during the fourteenth nuclear cycle in a Drosophila embryo reflects differences in cell identities. We have analysed the domains of mitotic division in zygotic mutants that exhibit defects in larval cuticular pattern along the dorsoventral axis. This is a powerful means of fate mapping mutant embryos, as the altered position of mitotic domains in the dorsoventral pattern mutants correlate with their late cuticular phenotypes. In the mutants twist and snail, which fail to differentiate the ventrally derived mesoderm, mitoses specific to the mesoderm are absent. The lateral mesectodermal domain shows a partial ventral shift in twist mutants but a proportion of ventral cells do not behave characteristically, suggesting that twist has a positive role in the establishment of the mesoderm. In contrast, snail is required to repress mesectodermal fates in cells of the presumptive mesoderm. In the absence of both genes, the mesodermal and the mesectodermal anlage are deleted. Mutations at five loci delete specific pattern elements in the dorsal half of the embryo and cause partial ventralization. Mutations in the genes zerknüllt and shrew affect cell division only in the dorsalmost cells corresponding to the amnioserosa, while the genes tolloid, screw and decapentaplegic (dpp) affect divisions in both the prospective amnioserosa and the dorsal epidermis. We demonstrate that in each of these mutants dorsally placed mitotic domains are absent and this effect is correlated with an expansion and dorsal shift in the position of more ventral domains. The loss of activity in each of the five genes results in qualitatively similar alterations in the mitotic pattern; mutations with stronger ventralizing phenotypes affect increasingly greater subsets of the dorsal cells. Double mutant analysis indicates that these genes act in a concerted manner to specify dorsal fates. The correlation between phenotypic strength and the progressive loss of dorsal pattern elements in the ventralized mutants, suggests that one of these gene products, perhaps dpp, may provide positional information in a graded manner.\n",
      "The synthesis of gene expression data and cis-regulatory analysis permits the elucidation of genomic regulatory networks. These networks provide a direct visualization of the functional interconnections among the regulatory genes and signaling components leading to cell-specific patterns of gene activity. Complex developmental processes are thereby illuminated in ways not revealed by the conventional analysis of individual genes. In this review, we describe emerging networks in several different model systems, and compare them with the gene regulatory network that controls dorsoventral patterning of the Drosophila embryo.\n",
      "The torpedo (DER) gene of Drosophila, which encodes a receptor tyrosine kinase of the EGF receptor subfamily, is essential for oogenesis, embryogenesis and imaginal disc development. To gain insight into the nature of the signals transduced by the torpedo product, we have characterized the gene's loss-of-function phenotype in the embryo. Through the induction of germline clones, we provide a genetic demonstration that maternal torpedo product does not contribute to zygotic development. Thus, the embryonic lethal phenotypes examined accurately reflect the consequences of eliminating all gene activity from the zygote. Temperature-shift experiments with the conditional allele topIF26 show that torpedo is required at two distinct times during embryonic development: the gene is first needed for germband retraction and for the production of anterior, posterior and ventral cuticle, then later for the secretion of ventral denticles. Since denticle formation can be severely disrupted in topIF26 animals without affecting cuticle production, the early and late requirements for torpedo appear to be functionally unrelated. torpedo, therefore, is required at multiple times in the development of the ventral epidermis, and may transduce qualitatively different signals. Since the early requirement for torpedo correlates with the first visible defect in embryonic development, increased cell death in the amnioserosa, cephalic ectoderm and ventral epidermis, the abnormalities in cuticle production and germband shortening seen in the mutant may be secondary consequences of a primary defect in cell viability. Given that the onset of cell death in torpedo embryos is not preceded by any obvious defects in mitogenesis, the establishment of cell identities or the maintenance of gene expression, it is possible that torpedo transduces a signal necessary for cell survival per se during early embryogenesis. During late embryogenesis, torpedo may mediate the reception of a second signal which regulates ventral epidermal cell differentiation.\n",
      "The toxicity of an abnormally long polyglutamine [poly(Q)]tract within specific proteins is the molecular lesion shared by Huntington's disease (HD) and several other hereditary neurodegenerative disorders. By a genetic screen in Drosophila, devised to uncover genes that suppress poly(Q) toxicity, we discovered a Drosophila homolog of human myeloid leukemia factor 1 (MLF1). Expression of the Drosophila homolog (dMLF) ameliorates the toxicity of poly(Q) expressed in the eye and central nervous system. In the retina, whether endogenously or ectopically expressed, dMLF co-localized with aggregates, suggesting that dMLF alone, or through an intermediary molecular partner, may suppress toxicity by sequestering poly(Q) and/or its aggregates.\n",
      "The trithorax group (trxG) genes absent, small or homeotic discs 1 (ash1) and 2 (ash2) were isolated in a screen for mutants with abnormal imaginal discs. Mutations in either gene cause homeotic transformations but Hox genes are not their only targets. Although analysis of double mutants revealed that ash2 and ash1 mutations enhance each other's phenotypes, suggesting they are functionally related, it was shown that these proteins are subunits of distinct complexes. The analysis of wing imaginal disc transcriptomes from ash2 and ash1 mutants showed that they are highly similar. Functional annotation of regulated genes using Gene Ontology allowed identification of severely affected groups of genes that could be correlated to the wing phenotypes observed. Comparison of the differentially expressed genes with those from other genome-wide analyses revealed similarities between ASH2 and Sin3A, suggesting a putative functional relationship. Coimmunoprecipitation studies and immunolocalization on polytene chromosomes demonstrated that ASH2 and Sin3A interact with HCF (host-cell factor). The results of nucleosome western blots and clonal analysis indicated that ASH2 is necessary for trimethylation of the Lys4 on histone 3 (H3K4). The similarity between the transcriptomes of ash2 and ash1 mutants supports a model in which the two genes act together to maintain stable states of transcription. Like in humans, both ASH2 and Sin3A bind HCF. Finally, the reduction of H3K4 trimethylation in ash2 mutants is the first evidence in Drosophila regarding the molecular function of this trxG gene.\n",
      "The twisted gastrulation (tsg) gene is one of seven known zygotic genes that specify the fate of dorsal cells in Drosophila embryos. Mutations in these genes cause at least some of the cells on the dorsal half of the embryo to adopt more ventral cell fates leading to the proposal that most of these genes participate in establishing, maintaining, or modulating a gradient of a single signaling molecule DECAPENTAPLEGIC (DPP). We have examined the effects of tsg mutations on the development of cuticule elements, expression of a region specific enhancer trap, and patterns of mitotic domains. Mutations of tsg only affect the fate of a narrow strip of dorsal midline cells and do not affect dorsal ectoderm cells. However, the pattern of tsg expression is not coincident with the territories affected by tsg mutations. Structural analysis of the tsg gene reveals features of a secreted protein suggesting an extracellular site of action. The TSG protein bears a weak resemblance to human connective tissue growth factor (CTGF), a TGF-beta-induced protein. We propose that dorsal midline cell fate is specified by the combination of both a TSG and a DPP signal to which the dorsal midline cells are uniquely competent to respond.\n",
      "The yellow (y) gene maps near the telomere of the X chromosome in Drosophila melanogaster but not in D. subobscura. Thus the strong reduction in the recombination rate associated with telomeric regions is not expected in D. subobscura. To study the divergence of a gene whose recombination rate differs between two species, the y gene of D. subobscura was sequenced. Sequence comparison between D. melanogaster and D. subobscura revealed several elements conserved in noncoding regions that may correspond to putative cis-acting regulatory sequences. Divergence in the y gene coding region between D. subobscura and D. melanogaster was compared with that found in other genes sequenced in both species. Both, yellow and scute exhibit an unusually high number of synonymous substitutions per site (ps). Also for these genes, the extent of codon bias differs between both species, being much higher in D. subobscura than in D. melanogaster. This pattern of divergence is consistent with the hitchhiking and background selection models that predict an increase in the fixation rate of slightly deleterious mutations and a decrease in the rate of fixation of slightly advantageous mutations in regions with low recombination rates such as in the y-sc gene region of D. melanogaster.\n",
      "This report focuses on dorso-ventral patterning in the segmented region of the Drosophila melanogaster embryo. According to the concept of positional information, this pattern results from the different response of cells to the Dorsal-protein morphogen. This protein shows a distribution gradient along the dorso-ventral axis, with the highest concentration on the ventral side. Using the generalized logical formalism developed by R. Thomas and co-workers, the different cellular responses were analysed in terms of the intracellular loops between the regulatory genes. Two positive loops were found to be involved, each constituting a switch which can be acted upon by the Dorsal morphogen to determine the different cell types that make up the embryonic dorso-ventral pattern. The novelty in this use of generalized logical formalism is the employment of a multilevel variable to represent a morphogen gradient. The proposed model accounts for the essential qualitative effects of the Dorsal gradient in the dorso-ventral determination process. Three main conclusions may be drawn. Firstly, the gene twist needs to have two functional threshold concentrations, one for autoactivation and the other for activation of the gene snail. Secondly, the autoactivation threshold must be smaller than that which activates snail. Thirdly, the action of the gene snail on the maintenance function of the gene twist is crucial for cells to be able to choose between the mesoderm or neuroectoderm developmental pathways. Furthermore, it is predicted that if the gene snail shows autoregulation, this will not be crucial for the determination of the embryonic D-V pattern. Copyright 1997 Academic Press Limited Copyright 1997 Academic Press Limited\n",
      "Three genetic hierarchies control cell-fate specification in largely distinct regions of the antero-posterior axis of the Drosophila embryo, whereas a single hierarchy specifies dorso-ventral cell fates. Molecular genetic analysis of these hierarchies is leading to increased understanding of the nature of the regulatory circuitry that controls regional cell-fate specification.\n",
      "To achieve the \"constancy of the wild-type,\"the developing organism must be buffered against stochastic fluctuations and environmental perturbations. This phenotypic buffering has been theorized to arise from a variety of genetic mechanisms and is widely thought to be adaptive and essential for viability. In the Drosophila blastoderm embryo, staining with antibodies against the active, phosphorylated form of the bone morphogenetic protein (BMP) signal transducer Mad, pMad, or visualization of the spatial pattern of BMP-receptor interactions reveals a spatially bistable pattern of BMP signaling centered on the dorsal midline. This signaling event is essential for the specification of dorsal cell fates, including the extraembryonic amnioserosa. BMP signaling is initiated by facilitated extracellular diffusion that localizes BMP ligands dorsally. BMP signaling then activates an intracellular positive feedback circuit that promotes future BMP-receptor interactions. Here, we identify a genetic network comprising three genes that canalizes this BMP signaling event. The BMP target eiger (egr) acts in the positive feedback circuit to promote signaling, while the BMP binding protein encoded by crossveinless-2 (cv-2) antagonizes signaling. Expression of both genes requires the early activity of the homeobox gene zerknüllt (zen). Two Drosophila species lacking early zen expression have high variability in BMP signaling. These data both detail a new mechanism that generates developmental canalization and identify an example of a species with noncanalized axial patterning.\n",
      "Transcriptional enhancers are crucial regulators of gene expression and animal development and the characterization of their genomic organization, spatiotemporal activities and sequence properties is a key goal in modern biology. Here we characterize the in vivo activity of 7,705 Drosophila melanogaster enhancer candidates covering 13.5% of the non-coding non-repetitive genome throughout embryogenesis. 3,557 (46%) candidates are active, suggesting a high density with 50,000 to 100,000 developmental enhancers genome-wide. The vast majority of enhancers display specific spatial patterns that are highly dynamic during development. Most appear to regulate their neighbouring genes, suggesting that the cis-regulatory genome is organized locally into domains, which are supported by chromosomal domains, insulator binding and genome evolution. However, 12 to 21 per cent of enhancers appear to skip non-expressed neighbours and regulate a more distal gene. Finally, we computationally identify cis-regulatory motifs that are predictive and required for enhancer activity, as we validate experimentally. This work provides global insights into the organization of an animal regulatory genome and the make-up of enhancer sequences and confirms and generalizes principles from previous studies. All enhancer patterns are annotated manually with a controlled vocabulary and all results are available through a web interface (http://enhancers.starklab.org), including the raw images of all microscopy slides for manual inspection at arbitrary zoom levels.\n",
      "Transcriptional repression is essential for the conversion of crude maternal gradients into sharp territories of tissue differentiation in the Drosophila embryo. Evidence will be presented suggesting that some of the embryonic repressors function through a short-range 'quenching' mechanism, whereby a repressor works over short distances (ca. 50 b.p.) to block neighbouring activators within a target enhancer. This type of repression can explain how different enhancers work autonomously within complex modular promoters. However, at least one of the repressors operating in the early embryo works through a long-range, or silencing, mechanism. The binding of a silencer to a given enhancer leads to the inactivation of all enhancers within a complex promoter. The analysis of chromatin boundary elements suggest that silencers and enhancers might work through distinct mechanisms. We speculate that silencers constrain the evolution of complex promoters.\n",
      "Transcriptional repressors can be characterized by their range of action on promoters and enhancers. Short-range repressors interact over distances of 50-150 bp to inhibit, or quench, either upstream activators or the basal transcription complex. In contrast, long-range repressors act over several kilobases to silence basal promoters. We describe recent progress in characterizing the functional properties of one such long-range element in the Drosophila embryo and discuss the contrasting types of gene regulation that are made possible by short- and long-range repressors.\n",
      "Transcriptional repressors function primarily by recruiting co-repressors, which are accessory proteins that antagonize transcription by modifying chromatin structure. Although a repressor could function by recruiting just a single co-repressor, many can recruit more than one, with Drosophila Brinker (Brk) recruiting the co-repressors CtBP and Groucho (Gro), in addition to possessing a third repression domain, 3R. Previous studies indicated that Gro is sufficient for Brk to repress targets in the wing, questioning why it should need to recruit CtBP, a short-range co-repressor, when Gro is known to be able to function over longer distances. To resolve this we have used genomic engineering to generate a series of brk mutants that are unable to recruit Gro, CtBP and/or have 3R deleted. These reveal that although the recruitment of Gro is necessary and can be sufficient for Brk to make an almost morphologically wild-type fly, it is insufficient during oogenesis, where Brk must utilize CtBP and 3R to pattern the egg shell appropriately. Gro insufficiency during oogenesis can be explained by its downregulation in Brk-expressing cells through phosphorylation downstream of EGFR signaling.\n",
      "Transposable elements (TEs) are mobile, repetitive sequences that make up significant fractions of metazoan genomes. Despite their near ubiquity and importance in genome and chromosome biology, most efforts to annotate TEs in genome sequences rely on the results of a single computational program, RepeatMasker. In contrast, recent advances in gene annotation indicate that high-quality gene models can be produced from combining multiple independent sources of computational evidence. To elevate the quality of TE annotations to a level comparable to that of gene models, we have developed a combined evidence-model TE annotation pipeline, analogous to systems used for gene annotation, by integrating results from multiple homology-based and de novo TE identification methods. As proof of principle, we have annotated \"TE models\"in Drosophila melanogaster Release 4 genomic sequences using the combined computational evidence derived from RepeatMasker, BLASTER, TBLASTX, all-by-all BLASTN, RECON, TE-HMM and the previous Release 3.1 annotation. Our system is designed for use with the Apollo genome annotation tool, allowing automatic results to be curated manually to produce reliable annotations. The euchromatic TE fraction of D. melanogaster is now estimated at 5.3% (cf. 3.86% in Release 3.1), and we found a substantially higher number of TEs (n = 6,013) than previously identified (n = 1,572). Most of the new TEs derive from small fragments of a few hundred nucleotides long and highly abundant families not previously annotated (e.g., INE-1). We also estimated that 518 TE copies (8.6%) are inserted into at least one other TE, forming a nest of elements. The pipeline allows rapid and thorough annotation of even the most complex TE models, including highly deleted and/or nested elements such as those often found in heterochromatic sequences. Our pipeline can be easily adapted to other genome sequences, such as those of the D. melanogaster heterochromatin or other species in the genus Drosophila.\n",
      "Transposable elements are found in the genomes of nearly all eukaryotes. The recent completion of the Release 3 euchromatic genomic sequence of Drosophila melanogaster by the Berkeley Drosophila Genome Project has provided precise sequence for the repetitive elements in the Drosophila euchromatin. We have used this genomic sequence to describe the euchromatic transposable elements in the sequenced strain of this species. We identified 85 known and eight novel families of transposable element varying in copy number from one to 146. A total of 1,572 full and partial transposable elements were identified, comprising 3.86% of the sequence. More than two-thirds of the transposable elements are partial. The density of transposable elements increases an average of 4.7 times in the centromere-proximal regions of each of the major chromosome arms. We found that transposable elements are preferentially found outside genes; only 436 of 1,572 transposable elements are contained within the 61.4 Mb of sequence that is annotated as being transcribed.A large proportion of transposable elements is found nested within other elements of the same or different classes. Lastly, an analysis of structural variation from different families reveals distinct patterns of deletion for elements belonging to different classes. This analysis represents an initial characterization of the transposable elements in the Release 3 euchromatic genomic sequence of D. melanogaster for which comparison to the transposable elements of other organisms can begin to be made. These data have been made available on the Berkeley Drosophila Genome Project website for future analyses.\n",
      "Two Drosophila Hox genes involved in segmentation, fushi tarazu and bicoid, appear to have acquired these roles by functional divergence from classical homeotic genes. Recent results indicate how genes with critical functions in development can evolve completely different functions among species.\n",
      "Using X-ray mutagenesis we have induced and recovered phenotypic revertants of four dominant mutations thought to be associated with the Antennapedia complex of Drosophila melanogaster. These include seven revertants of Antennapedia-73b (Antp(73b)), six of Extra Sex Combs of Wakimoto (Scx(w)), three of Deformed (Dfd) and one of Humeral (Hu). Fifteen of the 17 revertants are associated with chromosomal aberrations and localize Antp(73b), Scx( w) and Hu to polytene chromosome bands 84B1,2. The Dfd lesion is apparently located in or adjacent to bands 84A4,5. Since all of the dominants are reverted by events that delete their respective chromosomal loci, we conclude that all four are the result of a gain-of-function lesions. Complementation analysis of the various revertant chromosomes has shown that Scx(w) and Hu are dominant allelic variants of the Antp locus. The Dfd lesion represents a dominant mutation at a locus just proximal to Antp and previously only occupied by recessive lethal mutations. Characterization of the revertants of Scx(w) and a comparison with the properties of the original mutation has revealed that the original lesion has effects on both the Antp and Sex Combs Reduced (Scr) loci and that these defects are in some cases separable by the reverting event.\n",
      "We address the problem of finding statistically significant associations between cis-regulatory motifs and functional gene sets, in order to understand the biological roles of transcription factors. We develop a computational framework for this task, whose features include a new statistical score for motif scanning, the use of different scores for predicting targets of different motifs, and new ways to deal with redundancies among significant motif-function associations. This framework is applied to the recently sequenced genome of the jewel wasp, Nasonia vitripennis, making use of the existing knowledge of motifs and gene annotations in another insect genome, that of the fruitfly. The framework uses cross-species comparison to improve the specificity of its predictions, and does so without relying upon non-coding sequence alignment. It is thereforewell suited for comparative genomics across large evolutionary divergences, where existing alignment-based methods are not applicable. We also apply the framework to find motifs associated with socially regulated gene sets in the honeybee, Apis mellifera, using comparisons with Nasonia, a solitary species, to identify honeybee-specific associations.\n",
      "We employed robotic methods and the whole-genome sequence of Drosophila melanogaster to facilitate a large-scale expression screen for spatially restricted transcripts in Drosophila embryos. In this screen, we identified a pair of genes, scylla (scyl) and charybde (chrb), that code for dorsal transcripts in early Drosophila embryos and are homologous to the human apoptotic gene RTP801. In Drosophila, both gene products are transcriptionally regulated targets of Dpp/Zen-mediated signal transduction and appear more generally to be downstream targets of homeobox regulation. Gene disruption studies revealed the functional redundancy of scyl and chrb, as well as their requirement for embryonic head involution. From the perspective of functional genomics, our studies demonstrate that global surveys of gene expression can complement traditional genetic screening methods for the identification of genes essential for development: beginning from their spatio-temporal expression profiles and extending to their downstream placement relative to dpp and zen, our studies reveal roles for the scyl and chrb gene products as links between patterning and cell death.\n",
      "We have analyzed the contributions made by maternal and zygotic genes to the establishment of the expression patterns of four zygotic patterning genes: decapentaplegic (dpp), zerknüllt (zen), twist (twi), and snail (sna). All of these genes are initially expressed either dorsally or ventrally in the segmented region of the embryo, and at the poles. In the segmented region of the embryo, correct expression of these genes depends on cues from the maternal morphogen dorsal (dl). The dl gradient appears to be interpreted on three levels: dorsal cells express dpp and zen, but not twi and sna; lateral cells lack expression of all four genes; ventral cells express twi and sna, but not dpp and zen. dl appears to activate the expression of twi and sna and repress the expression of dpp and zen. Polar expression of dpp and zen requires the terminal system to override the repression by dl, while that of twi and sna requires the terminal system to augment activation by dl. The zygotic expression patterns established by the maternal genes appear to specify autonomous domains that carry out independent developmental programs, insofar as mutations in the genes that are expressed ventrally do not affect the initiation or ontogeny of the expression patterns of the genes that are expressed dorsally, and vice versa. However, interactions between the zygotic genes specific to a particular morphological domain appear to be important for further elaboration of the three levels specified by dl. Two of the genes, dpp and twi, are unaffected by mutations in any of the tested zygotic dorsal-ventral genes, suggesting that dpp and twi are the primary patterning genes for dorsal ectoderm and mesoderm, respectively.\n",
      "We have analyzed the function of the Decapentaplegic (Dpp) and Hedgehog (Hh) signaling pathways in partitioning the dorsal head neurectoderm of the Drosophila embryo. This region, referred to as the anterior brain/eye anlage, gives rise to both the visual system and the protocerebrum. The anlage splits up into three main domains: the head midline ectoderm, protocerebral neurectoderm and visual primordium. Similar to their vertebrate counterparts, Hh and Dpp play an important role in the partitioning of the anterior brain/eye anlage. Dpp is secreted in the dorsal midline of the head. Lowering Dpp levels (in dpp heterozygotes or hypomorphic alleles) results in a 'cyclops' phenotype, where mid-dorsal head epidermis is transformed into dorsolateral structures, i.e. eye/optic lobe tissue, which causes a continuous visual primordium across the dorsal midline. Absence of Dpp results in the transformation of both dorsomedial and dorsolateral structures into brain neuroblasts. Regulatory genes that are required for eye/optic lobe fate, including sine oculis (so) and eyes absent (eya), are turned on in their respective domains by Dpp. The gene zerknuellt (zen), which is expressed in response to peak levels of Dpp in the dorsal midline, secondarily represses so and eya in the dorsomedial domain. Hh and its receptor/inhibitor, Patched (Ptc), are expressed in a transverse stripe along the posterior boundary of the eye field. As reported previously, Hh triggers the expression of determinants for larval eye (atonal) and adult eye (eyeless) in those cells of the eye field that are close to the Hh source. Eya and So, which are induced by Dpp, are epistatic to the Hh signal. Loss of Ptc, as well as overexpression of Hh, results in the ectopic induction of larval eye tissue in the dorsal midline (cyclopia). We discuss the similarities between vertebrate systems and Drosophila with regard to the fate map of the anterior brain/eye anlage, and its partitioning by Dpp and Hh signaling.\n",
      "We have characterized at the molecular level the zerknüllt (zen) region of the Drosophila subobscura Antennapedia complex. The sequence comparison between D. subobscura and D. melanogaster shows an irregular distribution of the conserved and diverged regions, with the homeobox and a putative activating domain completely conserved. Comparisons of the promoter sequence and pattern of expression of the gene during development suggest that the regulation of zen has been conserved during evolution. The conservation of zen expression in a subpopulation of the polar cells indicates the existence of an important role in such cells. We describe a transitory segmented pattern of expression of zen in both species, suggesting the existence of interactions with a pair rule gene. Some indirect clues indicate that the z2 gene might be absent from the D. subobscura genome. A chromosome walk initiated to reach the proboscipedia gene of D. subobscura reveals that the distance between pb and zen is at least four times the one described for D. melanogaster and for D. pseudoobscura. Finally, we present cytological evidence showing that the ANT-C is inverted in D. subobscura as compared to D. melanogaster.\n",
      "We have cloned, from a beetle and a locust, genes that are homologous to the class 3 Hox genes of vertebrates. Outside the homeobox they sharesequence motifs with the Drosophila zerknüllt (zen) and z2 genes, and like zen, are expressed only in extraembryonic membranes. We conclude that the zen genes of Drosophila derive from a Hox class 3 sequence that formed part of the common ancestral Hox cluster, but that in insects this (Hox) gene has lost its role in patterning the anterio-posterior axis of the embryo, and acquired a new function. In the lineage leading to Drosophila, the zen genes have diverged particularly rapidly.\n",
      "We have identified in the 5' untranslated region of the Drosophila copia retrotransposon, 3' to the left LTR, a sequence for transcriptional regulation by homeoproteins. Co-transfection assays using expression vectors for homeoproteins and reporter vectors containing the lacZ gene under the control of either the entire copia LTR with 5' untranslated sequence, or a minimal heterologous promoter flanked with a 130 bp fragment containing the copia untranslated region, disclosed both positive and negative modulations of promoter activity in Drosophila cells in culture: a 5-10 fold decrease with engrailed, even-skipped and zerknüllt in DH33 cells, and a 10-30 fold increase with fushi tarazu and zerknüllt in Schneider II cells. In all cases, the regulatory effects were abolished with reporter plasmids deleted for a 58 bp fragment encompassing the putative homeoprotein binding sites. Mobility shift assays with a purified homeodomain-containing peptide demonstrated direct interaction with the 58 bp fragment, with an affinity in the 1-10 nM range as reported with the same peptide for other well characterized homeodomain binding regulatory sites. Foot-printing experiments with the extended LTR demonstrated protection of 'consensus' sequences, located within the 58 bp fragment. These homeodomain binding sites could be involved in the developmental regulation of the copia retrotransposon.\n",
      "We have investigated Drosophila salivary gland determination by examining the effects of mutations in pattern forming genes on the salivary gland primordium. We find that the anterior-posterior extent of the primordium, a placode of columnar epithelial cells derived from parasegment 2, is established by the positive action of the homeotic gene Sex combs reduced (Scr). Embryos mutant for Scr lack a detectable placode, while ectopic Scr expression leads to the formation of ectopic salivary glands. In contrast, the dorsal-ventral extent of the placode is regulated negatively. Functions dependent on the decapentaplegic product place a dorsal limit on the placode, while dorsal-dependent genes act to limit the placode ventrally. We propose a model in which these pattern forming genes act early to determine the salivary gland anlage by regulating the expression of salivary gland determining genes, which in turn control genes that are involved in salivary gland morphogenesis.\n",
      "We have observed that zygotic transcription does not initiate at a single point in Drosophila embryos. Rather, a gene initiates transcription in a few nuclei of a fraction of embryos. During succeeding cycles, the frequency of transcribing embryos, and of nuclei transcribing in those embryos, gradually increases. For the fushi tarazu (ftz) gene, the timing of this process is regulated by the concentration of the maternally loaded, repressing transcription factor tramtrack (ttk). Altering the dose of Ttk protein in embryos shifts the activation of ftz transcription either forward or backward during development but does not effect Krüppel (Kr) activation. We have observed that the transcription of several genes, including ftz, is triggered in embryos at a critical nuclear density; therefore, we suggest that titration of transcription factors like ttk by the nucleocytoplasmic ratio triggers zygotic transcription in Drosophila.\n",
      "We have studied the ability of the Drosophila gap proteins Krüppel and hunchback to function as transcriptional regulators in cultured cells. Both proteins bind to specific sites in a 100-bp DNA fragment located upstream of the segment polarity gene engrailed, which also contains functional binding sites for a number of homeo box proteins. The hunchback protein is a strikingly concentration-dependent activator of transcription, capable of functioning both by itself and also synergistically with the pair-rule proteins fushi tarazu and paired. In contrast, Krüppel is a transcriptional repressor that can block transcription induced either by hunchback or by several different homeo box proteins. While repression of the homeo box protein activators requires a Krüppel-binding site on the DNA, repression of hunchback can occur efficiently in the absence of a Krüppel-binding site. We discuss the possible molecular mechanisms underlying these activities, as well as the potential significance of these results with respect to segmentation in Drosophila.\n",
      "We have studied the transcriptional activity of the Drosophila homeodomain protein Engrailed (En) by using a transient expression assay employing Schneider L2 cells. En was found to very strongly repress promoters activated by a variety of different activator proteins. However, unlike another Drosophila homeodomain-containing repressor, Even-skipped (Eve), En was unable to repress the activity of several basal promoters in the absence of activator expression. These findings indicate that En is a specific repressor of activated transcription, and suggest that En may repress transcription by a different mechanism than Eve, perhaps by interfering with interactions between transcriptional activators and the general transcription machinery. By analyzing the properties of a variety of En mutants, we identified a minimal repression domain composed of 55 residues, which can function when fused to a heterologous DNA binding domain. Like repression domains identified in the Drosophila repressors Eve and Krüppel, the En repression domain is rich in alanine residues (26%), but unlike these other domains, is moderately charged (six arginine and three glutamic acid residues). Separate regions of En that may in some circumstances function in transcriptional activation were also identified.\n",
      "We have undertaken a developmental genetic analysis of labial (lab), the most proximal gene in the Antennapedia complex (ANT-C) of Drosophila melanogaster. The terminal phenotype of mutant embryos was examined in cuticle preparations, in thin sections, and by scanning electron microscopy. These preparations revealed a failure of head involution and the loss or disruption of several head structures, including the salivary glands and the H-piece and ventral arm of the cephalopharyngeal apparatus. Although these structures are presumed to derive from the gnathocephalic segments, we argue that the observed defects are likely to be a secondary consequence of a failure of head involution. A function for lab in the development of the adult head was inferred from the phenotype of animals bearing hypomorphic alleles and from clones of lab- tissue generated by mitotic recombination. Two aspects of the mutant phenotype were manifested. Ventrally, a deletion and/or disruption of tissue occurred in the maxillary palp and vibrissae regions. Dorsally, the posterior head appeared to be transformed to a thoracic-like identity. Mutations in lab, like those in the Deformed and proboscipedia loci of the ANT-C, reveal a homoeotic phenotype only in the adult stage of the life cycle.\n",
      "We have used a transient expression assay employing Drosophila tissue culture cells to study the potential of several Drosophila homeobox proteins to function as transcriptional regulators. A 96 bp fragment from the promoter region of the segment polarity gene engrailed, previously shown to contain five copies of a 10 bp consensus binding site for these proteins, enhanced transcription in the presence, but not the absence, of several different homeobox protein expression vectors. It is interesting that cotransfection with combinations of expression vectors encoding the homeobox proteins fushi tarazu, paired, and/or zen resulted in substantial synergistic increases in expression. In contrast, the products of the even-skipped and engrailed genes were found to repress, or quench, the activation induced by the other proteins. We discuss the implications of these results with respect to the role of homeobox genes in the control of embryonic development, and propose a \"multi-switch\"model whereby the activity of a target gene depends on the interactions of different homeobox proteins with multiple copies of a common binding site.\n",
      "We have used a transient expression assay employing Drosophila tissue culture cells to study the transcriptional repression activity of the homeo domain protein Even-skipped (Eve). Eve was found to repress all promoters that contained Eve-binding sites, including both TATA-containing and TATA-lacking minimal promoters, as well as promoters activated by several different classes of activator proteins. These findings suggest that the general transcription machinery can be a target of Eve. By analyzing properties of a variety of Eve mutants and chimeric fusion proteins, we have identified several features important for efficient repression. In addition to the DNA-binding domain, a potent repressor requires a repression domain, which can be as small as 27 residues. The minimal 57-residue Eve repression domain, as well as several others studied here, were all found to be proline rich and to contain a high percentage of hydrophobic residues. An intriguing feature of the strong repressors was that their DNA-binding activities, measured by gel retention assays with nuclear extracts, were significantly less than those of derivatives inactive in repression.\n",
      "We identified Drosophila Smurf (DSmurf) as a negative regulator ofsignaling by the BMP2/4 ortholog DPP during embryonic dorsal-ventral patterning. DSmurf encodes a HECT domain ubiquitin-protein ligase, homologous to vertebrate Smurf1 and Smurf2, that binds the Smad1/5 ortholog MAD and likely promotes its proteolysis. The essential function of DSmurf is restricted to its action on the DPP pathway. DSmurf has two distinct, possibly mechanistically separate, functions in controlling DPP signaling. Prior to gastrulation, DSmurf mutations cause a spatial increase in the DPP gradient, as evidenced by ventrolateral expansion in expression domains of target genes representing all known signaling thresholds. After gastrulation, DSmurf mutations cause a temporal delay in downregulation of earlier DPP signals, resulting in a lethal defect in hindgut organogenesis.\n",
      "We recently uncovered a novel genetic mechanism that generates the phenotypic uniformity, or canalization, of BMP signaling and cell fate specification during patterning of the dorsal-ventral (D/V) axis in D. melanogaster embryos. We went on to show that other wild-type Drosophila species lack this canalizing genetic circuitry and, consequently, have non-robust D/V patterning. In this review, we propose molecular mechanisms that may give rise to stereotyped BMP signaling, and we identify an additional species that could have decanalized D/V patterning. Extension of these analyses could in turn help explain why canalization is not a universal necessity for species survival.\n",
      "We report here the genetical and molecular characterization of a new Drosophila zygotic lethal locus, vrille (vri). Vri alleles act not only as dominant maternal enhancers of embryonic dorsoventral patterning defects caused by easter and decapentaplegic (dpp) mutations, but also as dominant zygotic enhancers of dpp alleles for phenotypes in wing. The vri gene encodes a new member of the bZIP family of transcription factors closely related to gene 9 of Xenopus laevis, induced by thyroid hormone during the tadpole tail resorption program, and NF-IL3A, a human T cell transcription factor that transactivates the interleukin-3 promoter. NF-IL3A shares 93% similarity and 60% identity with Vri for a stretch of 68 amino acids that includes the bZIP domain. Although all the alleles tested behave like antimorphs, the dominant enhancement is also seen with a nonsense mutation allele that prevents translation of the bZIP domain. Because of the strong domainant enhancement of dpp phenotypes by vri alleles in both embryo and wing, and also the similarity between the wing vein phenotypes caused by the vri and shortvein dpp alleles, we postulate that vri interacts either directly or indirectly with certain components of the dpp (a TGF beta homologue) signal transduction pathway.\n",
      "We report the isolation and characterization of a putative angiotensin converting enzyme (ACE) in Drosophila, called Race. General interest in mammalian ACE stems from its association with high blood pressure; ACE has also been implicated in a variety of other physiological processes including the processing of neuropeptides and gut peristalsis. Mammalian ACE is a membrane associated zinc binding protease that converts angiotensin I (A I) into angiotensin II (A II). A II functions as a potent vasoconstrictor by triggering a G-coupled receptor system in the smooth muscles that line blood vessels. Drosophila Race is composed of 615 amino acid residues, and shares extensive sequence identity with mammalian ACE over its entire length (over 42% overall identity and greater than 60% similarity). Evidence is presented that Race might correspond to a target of the homeobox regulatory gene, zerknullt (zen). Soon after zen expression is restricted to the dorsal-most regions of the embryonic ectoderm, Race is activated in a coincident pattern and becomes associated with the amnioserosa during germ band elongation, shortening and heart morphogenesis. After germ band elongation, Race is also expressed in both the anterior and posterior midgut, where it persists throughout embryogenesis. Race expression is lost from the dorsal ectoderm in either zen- or dpp- mutants, although gut expression is unaffected. P-transformation assays and genetic complementation tests suggest that Race corresponds to a previously characterized lethal complementation group, 1(2)34Eb. Mutants die during larval/pupal development, and transheterozygotes for two different lethal alleles exhibit male sterility. We propose that Race might play a role in the contractions of the heart, gut, or testes and also suggest that Hox genes might be important for coordinating both developmental and physiological processes.\n",
      "We show that the Drosophila protein DSP1, an HMG-1/2-like protein, binds DNA highly cooperatively with three members of the Rel family of transcriptional regulators (NF-kappaB, the p50 subunit of NF-kappaB, and the Rel domain of Dorsal). This cooperativity is apparent with DNA molecules bearing consensus Rel-protein-binding sites and is unaffected by the presence of a negative regulatory element, a sequence previously proposed to be important for mediating repression by these Rel proteins. The cooperativity observed in these DNA-binding assays is paralleled by interactions between protein pairs in the absence of DNA. We also show that in HeLa cells, as assayed by transient transfection, expression of DSP1 increases activation by Dorsal from the twist promoter and inhibits that activation from the zen promoter, consistent with the previously proposed idea that DSP1 can affect the action of Dorsal in a promoter-specific fashion.\n",
      "While the role of the notochord and floor plate in patterning the dorsal-ventral (D/V) axis of the neural tube is clearly established, relatively little is known about the earliest stages of D/V regionalization. In an effort to examine more closely the initial, preneural plate stages of regionalization along the prospective D/V neural axis, we have performed a series of explant experiments employing xHB9, a novel marker of the motor neuron region in Xenopus. Using tissue recombinants and Keller explants we show that direct mesodermal contact is both necessary and sufficient for the initial induction of xHB9 in the motor neuron region. We also show that presumptive neural plate explants removed as early as midgastrulation and cultured in isolation are already specified to express xHB9 but do so in an inappropriate spatial pattern while identical explants are specified to express the floor plate marker vhh-1 with correct spatial patterning. Our data suggest that, in addition to floor plate signaling, continued interactions with the underlying mesoderm through neural tube stages are essential for proper spatial patterning of the motor neuron region.\n",
      "Wingless signaling plays a central role during epidermal patterning in Drosophila. We have analyzed zygotic requirements for Wingless signaling in the embryonic ectoderm by generating synthetic deficiencies that uncover more than 99% of the genome. We found no genes required for initial wingless expression, other than previously identified segmentation genes. In contrast, maintenance of wingless expression shows a high degree of zygotic transcriptional requirements. Besides known genes, we have identified at least two additional genomic regions containing new genes involved in Wingless maintenance. We also assayed for the zygotic requirements for Wingless response and found that no single genomic region was required for the cytoplasmic accumulation of Armadillo in the receiving cells. Surprisingly, embryos homozygously deleted for the candidate Wingless receptor, Dfrizzled2, showed a normal Wingless response. However, the Armadillo response to Wingless was strongly reduced in double mutants of both known members of the frizzled family in Drosophila, frizzled and Dfrizzled2. Based on their expression pattern during embryogenesis, different Frizzled receptors may play unique but overlapping roles in development. In particular, we suggest that Frizzled and Dfrizzled2 are both required for Wingless autoregulation, but might be dispensable for late Engrailed maintenance. While Wingless signaling in embryos mutant for frizzled and Dfrizzled2 is affected, Wingless protein is still internalized into cells adjacent to wingless-expressing cells. Incorporation of Wingless protein may therefore involve cell surface molecules in addition to the genetically defined signaling receptors of the frizzled family.\n",
      "With their power to shape animal morphology, few genes have captured the imagination of biologists as the evolutionarily conserved members of the Hox clusters have done. Recent research has provided new insight into how Hox proteins cause morphological diversity at the organismal and evolutionary levels. Furthermore, an expanding collection of sequences that are directly regulated by Hox proteins provides information on the specificity of target-gene activation, which might allow the successful prediction of novel Hox-response genes. Finally, the recent discovery of microRNA genes within the Hox gene clusters indicates yet another level of control by Hox genes in development and evolution.\n",
      "Worldwide, acute, and chronic pain affects 20% of the adult population and represents an enormous financial and emotional burden. Using genome-wide neuronal-specific RNAi knockdown in Drosophila, we report a global screen for an innate behavior and identify hundreds of genes implicated in heat nociception, including the α2δ family calcium channel subunit straightjacket (stj). Mice mutant for the stj ortholog CACNA2D3 (α2δ3) also exhibit impaired behavioral heat pain sensitivity. In addition, in humans, α2δ3 SNP variants associate with reduced sensitivity to acute noxious heat and chronic back pain. Functional imaging in α2δ3 mutant mice revealed impaired transmission of thermal pain-evoked signals from the thalamus to higher-order pain centers. Intriguingly, in α2δ3 mutant mice, thermal pain and tactile stimulation triggered strong cross-activation, or synesthesia, of brain regions involved in vision, olfaction, and hearing.\n",
      "cDNA cloning is a central technology in molecular biology. cDNA sequences are used to determine mRNA transcript structures, including splice junctions, open reading frames (ORFs) and 5'- and 3'-untranslated regions (UTRs). cDNA clones are valuable reagents for functional studies of genes and proteins. Expressed Sequence Tag (EST) sequencing is the method of choice for recovering cDNAs representing many of the transcripts encoded in a eukaryotic genome. However, EST sequencing samples a cDNA library at random, and it recovers transcripts with low expression levels inefficiently. We describe a PCR-based method for directed screening of plasmid cDNA libraries. We demonstrate its utility in a screen of libraries used in our Drosophila EST projects for 153 transcription factor genes that were not represented by full-length cDNA clones in our Drosophila Gene Collection. We recovered high-quality, full-length cDNAs for 72 genes and variously compromised clones for an additional 32 genes. The method can be used at any scale, from the isolation of cDNA clones for a particular gene of interest, to the improvement of large gene collections in model organisms and the human. Finally, we discuss the relative merits of directed cDNA library screening and RT-PCR approaches.\n",
      "cappuccino and spire are unique Drosophila maternal-effect loci that participate in pattern formation in both the anteroposterior and dorsoventral axes of the early embryo. Mutant females produce embryos lacking pole cells, polar granules, and normal abdominal segmentation. They share these defects with the posterior group of maternal-effect genes. Although embryos are defective in abdominal segmentation, in double mutant combinations with Bicaudal D, abdominal segments can be formed in the anterior half of the egg. This indicates that embryos produced by mutant females contain the 'posterior determinant' required for abdominal segmentation (Nüsslein-Volhard et al. 1987) and suggests that the wild-type gene products are not required for production of the posterior determinant but, rather, for its localization or stabilization. The vasa protein, a component of polar granules, is not localized at the posterior pole of mutant egg chambers or embryos, providing additional support for the hypothesis that localization to or stabilization of substances at the posterior pole of the egg chamber is defective in mutant females. Females mutant for the strongest alleles also produce dorsalized embryos. Phenotypic analysis reveals that these dorsalized embryos also have abdominal segmentation defects. The mutant phenotypes can be ordered in a series of increasing severity. Pole cell formation is most sensitive to loss of functional gene products, followed by abdominal segmentation, whereas normal dorsoventral patterning is the least sensitive to loss of functional gene products. In addition, mutant females contain egg chambers that appear to be dorsalized, resulting in the production of eggs with dorsalized eggshells. Germ-line mosaics indicate that cappuccino and spire are required in the oocyte-nurse cell complex. This suggests that the eggshell phenotype results from altered pattern in the underlying germ cell. Also, we defined the epistatic relationships between several early patterning loci, on the basis of an analysis of the eggs and embryos produced by females doubly mutant for cappuccino or spire and other loci that affect the pattern of both the egg and the embryo. On the basis of our current knowledge of the genes involved in this process, we formulated a working model for the early steps in dorsoventral patterning.\n",
      "decapentaplegic (dpp) encodes a Drosophila transforming growth factor-beta homologue that functions as a morphogen in the developing embryo and in adult appendage formation. In the wing imaginal disc, a Dpp gradient governs patterning along the anteroposterior axis by inducing regional expression of diverse genes in a concentration-dependent manner. Recent studies show that responses to graded Dpp activity also require an input from a complementary and opposing gradient of Brinker (Brk), a transcriptional repressor protein encoded by a Dpp target gene. Here we show that Brk harbours a functional and transferable repression domain, through which it recruits the corepressors Groucho and CtBP. By analysing transcriptional outcomes arising from the genetic removal of these corepressors, and by ectopically expressing Brk variants in the embryo, we demonstrate that these corepressors are alternatively used by Brk for repressing some Dpp-responsive genes, whereas for repressing other distinct target genes they are not required. Our results show that Brk utilizes multiple means to repress its endogenous target genes, allowing repression of a multitude of complex Dpp target promoters.\n",
      "even-skipped (eve) is a homeodomain-encoding gene that is a genetically defined repressor of Ultrabithorax (Ubx), fushi-tarazu (ftz), and wingless (wg). Here we report that purified eve protein represses transcription in vitro at the Ubx promoter, in a DNA binding site-dependent manner. eve protein represses transcription when bound either upstream or downstream of the RNA start site or when DNA binding sites are in either orientation. We also show that eve represses expression from the Ubx promoter in Drosophila tissue culture cells, again in a binding site-dependent manner. Deletion of eve DNA binding sites does not alter transcription in the absence of eve, and so repression is not likely to be the result of eve competitively inhibiting an activator protein from binding to the same DNA element. Instead, we propose that eve protein is probably interfering with the function of proteins bound at other locations in the promoter. The biochemical demonstration that a Drosophila homeodomain protein can directly regulate RNA synthesis strengthens the view that this class of regulators act as transcription factors to control development.\n",
      "mRNA-associated processes and gene structure in eukaryotes are typically treated as separate research subjects. Here, we bridge this separation and leverage the extensive multidisciplinary work on Drosophila melanogaster to examine the roles that capping, splicing, cleavage/polyadenylation, and telescripting (i.e, the protection of nascent transcripts from premature cleavage/polyadenylation by the splicing factor U1) might play in shaping exon-intron architecture in protein-coding genes. Our findings suggest that the distance between subsequent internal 5' splice sites (5'ss) in Drosophila genes is constrained such that telescripting effects are maximized, in theory, and thus nascent transcripts are less vulnerable to premature termination. Exceptionally weak 5'ss and constraints on intron-exon size at the gene 5' end also indicate that capping might enhance the recruitment of U1 and, in turn, promote telescripting at this location. Finally, a positive correlation between last exon length and last 5'ss strength suggests that optimal donor splice sites in the proximity of the pre-mRNA tail may inhibit the processing of downstream polyadenylation signals more than weak donor splice sites do. These findings corroborate and build upon previous experimental and computational studies on Drosophila genes. They support the possibility, hitherto scantly explored, that mRNA-associated processes impose significant constraints on the evolution of eukaryotic gene structure.\n",
      "twist is one of the earliest expressed zygotically active genes required for dorsal-ventral pattern formation in the Drosophila embryo. Genetic studies suggest that this gene is activated in the ventral part of the blastoderm by maternally expressed dorsal gene product. Using P-element-mediated germ-line transformation, we have mapped a small (260 bp) dorsal-dependent ventral activator region (VAR) in the 5'-flanking region of the twist promoter that can direct the early ventral expression of a heterologous promoter. The VAR contains binding sites for a number of proteins present in extracts of Drosophila embryos. One of these sites bears homology to known binding sites for the dorsal transcription factor and is specifically bound by bacterially expressed dorsal protein. Furthermore, a 37-bp deletion that removes the dorsal-binding sequences abolishes the ventral-specific activity of the twist promoter constructs. Our data also show that additional sequences within the VAR are required to render the dorsal-binding sites functional. Finally, reverse genetic and biochemical data suggest that the transcription factor, encoded by the zeste gene may help control the overall level, if not the pattern, of twist expression.\n",
      "wimp is a dominant maternal-effect mutation that interacts with a specific subset of early-acting maternal and zygotic Drosophila genes. We show that wimp is a change-of-function mutation, allelic to mutations of the 140-kD subunit of RNA polymerase, which causes reduced transcription of interacting genes. Loci that do not interact with wimp are expressed at normal levels. We discuss these results in terms of specific interactions between transcription factors and RNA polymerase. Embryos from wimp mothers show unaltered fate maps and develop normally, despite the reduction of transcript levels at least twofold. We suggest that spatial cues are determined by a balance of segmentation gene products rather than their absolute concentrations. We also demonstrate powerful genetic screens for otherwise undetected loci required for segmentation, sex determination, and other early functions.\n",
      "zerknüllt (zen) is one of approximately 10 zygotically active genes that control the differentiation of the dorsal-ventral (D/V) pattern during early embryogenesis in Drosophila. Past genetic analyses suggest that maternal factors repress the expression of zen in ventral regions, thereby restricting zen products to dorsal and dorsal-lateral regions of precellular embryos. Subsequent interactions with other zygotic D/V regulatory genes refine the zen pattern, restricting expression to the dorsal-most ectoderm. Here we describe the use of zen promoter fusions and P-element transformation to identify cis elements that are responsible for the complex spatial pattern of zen expression. The zen promoter shows a two-tier organization: Distal sequences mediate its initial response to maternal factors, whereas proximal sequences are responsible for the refinement of the pattern in older embryos. The distal regulatory element has the property of a silencer (or anti-enhancer) element and can act over a distance to repress ventral expression of a heterologous promoter. Also, we discuss evidence that proximal promoter sequences interact with factors that may be modulated by a cell-cell communication pathway.\n",
      "zerknüllt (zen) is unique among the 18 known homeo box genes in Drosophila since it is required for the differentiation of the dorsal-ventral pattern, and does not appear to be involved in the process of segmentation. Here we show that the zen region of the Antennapedia complex (ANT-C) consists of two closely linked homeo box genes, designated z1 and z2. The z1 and z2 transcription units show essentially identical patterns of expression during early development, which are consistent with the timing and sites of zen+ gene activity. The putative proteins encoded by z1 and z2 are highly divergent and are related only by virtue of homeo box homology. We have used P-element-mediated germ line transformation to show that z1 alone can provide zen+ gene function, suggesting that the z2 gene might be dispensable. The occurrence of closely linked homeo box genes that display similar patterns of expression is not unique to the zen locus. Such gene duplications might provide important clues to the evolution of the homeo box gene family in Drosophila and other organisms.\n",
      "β-Arrestins have been implicated in the regulation of multiple signalling pathways. However, their role in organism development is not well understood. In this study, we report a new in vivo function of the Drosophila β-arrestin Kurtz (Krz) in the regulation of two distinct developmental signalling modules: MAPK ERK and NF-κB, which transmit signals from the activated receptor tyrosine kinases (RTKs) and the Toll receptor, respectively. Analysis of the expression of effectors and target genes of Toll and the RTK Torso in krz maternal mutants reveals that Krz limits the activity of both pathways in the early embryo. Protein interaction studies suggest a previously uncharacterized mechanism for ERK inhibition: Krz can directly bind and sequester an inactive form of ERK, thus preventing its activation by the upstream kinase, MEK. A simultaneous dysregulation of different signalling systems in krz mutants results in an abnormal patterning of the embryo and severe developmental defects. Our findings uncover a new in vivo function of β-arrestins and present a new mechanism of ERK inhibition by the Drosophila β-arrestin Krz.\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for row in csv_reader:\n",
    "    print(row[0])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "If you have used the csv library before try writing your results into a csv file using the writer class. If you have not used it before, trying going through the documentation first and then writing code on your own. It is pretty self explanatory. The documentation can be found at: https://docs.python.org/2/library/csv.html"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "The last thing that we will look at, in this tutorial, is the summarize method. This method proves to be particularly useful when we want some basic statistics regarding a particular column. We will look at the statistics of the length of genes present in the list : List of the most enriched genes in the adult fly brain. We begin by creating a query. This is followed by adding views and the list constraint. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "metadata": {
    "collapsed": false
   },
   "outputs": [],
   "source": [
    "query2=service.new_query()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<intermine.query.Query at 0x7f9e100b57b8>"
      ]
     },
     "execution_count": 13,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "query2.select(\"Gene.*\",\"organism.*\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "<ListConstraint: Gene IN PL FlyAtlas_brain_top>"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "query2.add_constraint(\"Gene\",\"IN\",\"PL FlyAtlas_brain_top\")"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We then print out the first 10 rows of results. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Gene: briefDescription=None cytoLocation='10A3-10A3' description=None id=1107310 length=2075 name=None primaryIdentifier='FBgn0030259' score=None scoreType=None secondaryIdentifier='CG1545' symbol='CG1545' organism.commonName='fruit fly' organism.genus='Drosophila' organism.id=1000001 organism.name='Drosophila melanogaster' organism.shortName='D. melanogaster' organism.species='melanogaster' organism.taxonId=7227\n",
      "Gene: briefDescription=None cytoLocation='11D8-11D8' description=None id=1039485 length=90456 name='radish' primaryIdentifier='FBgn0265597' score=None scoreType=None secondaryIdentifier='CG44424' symbol='rad' organism.commonName='fruit fly' organism.genus='Drosophila' organism.id=1000001 organism.name='Drosophila melanogaster' organism.shortName='D. melanogaster' organism.species='melanogaster' organism.taxonId=7227\n",
      "Gene: briefDescription=None cytoLocation='14A1-14A3' description=None id=1040836 length=26224 name='mind-meld' primaryIdentifier='FBgn0259110' score=None scoreType=None secondaryIdentifier='CG42252' symbol='mmd' organism.commonName='fruit fly' organism.genus='Drosophila' organism.id=1000001 organism.name='Drosophila melanogaster' organism.shortName='D. melanogaster' organism.species='melanogaster' organism.taxonId=7227\n",
      "Gene: briefDescription=None cytoLocation='16F3-16F5' description=None id=1059829 length=138941 name='Shaker' primaryIdentifier='FBgn0003380' score=None scoreType=None secondaryIdentifier='CG12348' symbol='Sh' organism.commonName='fruit fly' organism.genus='Drosophila' organism.id=1000001 organism.name='Drosophila melanogaster' organism.shortName='D. melanogaster' organism.species='melanogaster' organism.taxonId=7227\n",
      "Gene: briefDescription=None cytoLocation='18C2-18C3' description=None id=1076481 length=21373 name='nicotinic Acetylcholine Receptor alpha7' primaryIdentifier='FBgn0086778' score=None scoreType=None secondaryIdentifier='CG32538' symbol='nAChRalpha7' organism.commonName='fruit fly' organism.genus='Drosophila' organism.id=1000001 organism.name='Drosophila melanogaster' organism.shortName='D. melanogaster' organism.species='melanogaster' organism.taxonId=7227\n",
      "Gene: briefDescription=None cytoLocation='19A4-19A4' description=None id=1174409 length=58027 name='Dopamine 2-like receptor' primaryIdentifier='FBgn0053517' score=None scoreType=None secondaryIdentifier='CG33517' symbol='Dop2R' organism.commonName='fruit fly' organism.genus='Drosophila' organism.id=1000001 organism.name='Drosophila melanogaster' organism.shortName='D. melanogaster' organism.species='melanogaster' organism.taxonId=7227\n",
      "Gene: briefDescription=None cytoLocation='24C9-24D1' description=None id=1008362 length=88786 name='friend of echinoid' primaryIdentifier='FBgn0051774' score=None scoreType=None secondaryIdentifier='CG31774' symbol='fred' organism.commonName='fruit fly' organism.genus='Drosophila' organism.id=1000001 organism.name='Drosophila melanogaster' organism.shortName='D. melanogaster' organism.species='melanogaster' organism.taxonId=7227\n",
      "Gene: briefDescription=None cytoLocation='25B4-25B4' description=None id=1699681 length=1564 name=None primaryIdentifier='FBgn0031650' score=None scoreType=None secondaryIdentifier='CG14044' symbol='CG14044' organism.commonName='fruit fly' organism.genus='Drosophila' organism.id=1000001 organism.name='Drosophila melanogaster' organism.shortName='D. melanogaster' organism.species='melanogaster' organism.taxonId=7227\n",
      "Gene: briefDescription=None cytoLocation='30D1-30E1' description=None id=1433896 length=92934 name='nicotinic Acetylcholine Receptor alpha6' primaryIdentifier='FBgn0032151' score=None scoreType=None secondaryIdentifier='CG4128' symbol='nAChRalpha6' organism.commonName='fruit fly' organism.genus='Drosophila' organism.id=1000001 organism.name='Drosophila melanogaster' organism.shortName='D. melanogaster' organism.species='melanogaster' organism.taxonId=7227\n",
      "Gene: briefDescription=None cytoLocation='42D4-42D6' description=None id=1181524 length=10005 name=None primaryIdentifier='FBgn0033108' score=None scoreType=None secondaryIdentifier='CG15236' symbol='CG15236' organism.commonName='fruit fly' organism.genus='Drosophila' organism.id=1000001 organism.name='Drosophila melanogaster' organism.shortName='D. melanogaster' organism.species='melanogaster' organism.taxonId=7227\n"
     ]
    }
   ],
   "source": [
    "for row in query2.rows(size=10):\n",
    "    print(row)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "We can look at the summary of the length of each gene. This contains some useful information such as the average length and the maximum and minimum length. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'count': 17.0, 'bucket': 1.0, 'max': 169106.0, 'stdev': 41489.79692298, 'average': 31601.891891891893, 'min': 808.0, 'buckets': 20.0}\n"
     ]
    }
   ],
   "source": [
    "print(query2.summarise(\"length\"))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    " Also, another property that comes in handy while printing results about a query that we can print only the columns we want in that result. Suppose for now, I only want the ID of Gene's, here's what I do: "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {
    "collapsed": false
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "1109635\n",
      "1040181\n",
      "1041556\n",
      "1061141\n",
      "1078562\n",
      "1071407\n",
      "1008394\n",
      "1700377\n",
      "1434993\n",
      "1182992\n"
     ]
    }
   ],
   "source": [
    " for row in query2.rows(size=10):\n",
    " print(row[\"Gene.id\"])"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This brings us to the end of the fifth tutorial. The next tutorial will be about further management of results. "
   ]
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python [conda root]",
   "language": "python",
   "name": "conda-root-py"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 1
}